{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "import requests\n",
    "import json\n",
    "import os\n",
    "from tqdm.notebook import tqdm"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_pfams_ids(path):\n",
    "    data = []\n",
    "    with open(pfam_family_file) as f:\n",
    "        data = f.readlines()\n",
    "    return [d.strip() for d in data]\n",
    "\n",
    "pfam_family_file = os.path.join(\"dataset\", \"relevant_pfam_ids.txt\")\n",
    "\n",
    "pfam_ids = get_pfams_ids(pfam_family_file)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [],
   "source": [
    "def get_description(id):\n",
    "    response = requests.get(f\"https://www.ebi.ac.uk/interpro/api/entry/pfam/{id}\")\n",
    "    return response.json()\n",
    "\n",
    "def get_description_mock(id):\n",
    "    return {\"metadata\" : {\"description\" : \"mock\"}}\n",
    "\n",
    "def collate_all_families(pfam_familiy_ids, mock=True):\n",
    "    dataset = {}\n",
    "    for pfam_id in tqdm(pfam_familiy_ids):\n",
    "        if mock:\n",
    "            dataset[pfam_id] = get_description_mock(pfam_id)\n",
    "        else:\n",
    "            dataset[pfam_id] = get_description(pfam_id)    \n",
    "\n",
    "    return dataset\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "application/vnd.jupyter.widget-view+json": {
       "model_id": "ba7307677ba64ab4a71c0ed913e24b06",
       "version_major": 2,
       "version_minor": 0
      },
      "text/plain": [
       "  0%|          | 0/1000 [00:00<?, ?it/s]"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    }
   ],
   "source": [
    "with open(os.path.join(\"dataset\", \"pfam_data.json\"), \"w\") as f:\n",
    "    json.dump(collate_all_families(pfam_ids, mock=False), f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "metadata": {},
   "outputs": [],
   "source": [
    "with open(os.path.join(\"dataset\", \"pfam_data.json\")) as f:\n",
    "    data = json.load(f)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'PF14748': {'metadata': {'accession': 'PF14748',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029036',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pyrroline-5-carboxylate reductase dimerisation',\n",
       "    'short': 'P5CR_dimer'},\n",
       "   'description': ['<p>Pyrroline-5-carboxylate reductase consists of two domains, an N-terminal catalytic domain (Pfam:PF03807) and a C-terminal dimerisation domain. This is the dimerisation domain [PMID:16233902].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0106', 'name': '6PGD_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 146,\n",
       "    'interactions': 0,\n",
       "    'matches': 54124,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54073,\n",
       "    'proteomes': 10226,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46727, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 41193},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50093,\n",
       "    'alignment:full': 12141,\n",
       "    'alignment:seed': 894},\n",
       "   'cross_references': {}}},\n",
       " 'PF00252': {'metadata': {'accession': 'PF00252',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR016180',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L16p/L10e', 'short': 'Ribosomal_L16'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 174,\n",
       "    'interactions': 0,\n",
       "    'matches': 69541,\n",
       "    'pathways': 0,\n",
       "    'proteins': 69502,\n",
       "    'proteomes': 11058,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 59649, 'rosettafold': 0},\n",
       "    'structures': 1158,\n",
       "    'taxa': 68160},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 60693,\n",
       "    'alignment:full': 12650,\n",
       "    'alignment:seed': 441},\n",
       "   'cross_references': {}}},\n",
       " 'PF01428': {'metadata': {'accession': 'PF01428',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000058',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AN1-like Zinc finger', 'short': 'zf-AN1'},\n",
       "   'description': ['<p>Zinc finger at the C-terminus of An1 Swiss:Q91889, a ubiquitin-like protein in Xenopus laevis. The following pattern describes the zinc finger. C-X2-C-X(9-12)-C-X(1-2)-C-X4-C-X2-H-X5-H-X-C Where X can be any amino acid, and numbers in brackets indicate the number of residues.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001828': {'PMID': 8390387,\n",
       "     'ISBN': None,\n",
       "     'volume': '128',\n",
       "     'issue': '2',\n",
       "     'year': 1993,\n",
       "     'title': 'Two related localized mRNAs from Xenopus laevis encode ubiquitin-like fusion proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '181-8',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Linnen JM', 'Bailey CP', 'Weeks DL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(93)90561-G'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 213,\n",
       "    'interactions': 0,\n",
       "    'matches': 23933,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18985,\n",
       "    'proteomes': 2321,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15575, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 9092},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19108,\n",
       "    'alignment:full': 10605,\n",
       "    'alignment:seed': 404},\n",
       "   'cross_references': {}}},\n",
       " 'PF04237': {'metadata': {'accession': 'PF04237',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007351',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YjbR', 'short': 'YjbR'},\n",
       "   'description': ['<p>YjbR has a CyaY-like fold [PMID:23044854].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0631', 'name': 'YgaC_TfoX-N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 53,\n",
       "    'interactions': 0,\n",
       "    'matches': 29310,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29268,\n",
       "    'proteomes': 4433,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25724, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 18676},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28017,\n",
       "    'alignment:full': 6110,\n",
       "    'alignment:seed': 492},\n",
       "   'cross_references': {}}},\n",
       " 'PF03015': {'metadata': {'accession': 'PF03015',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR033640',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Male sterility protein', 'short': 'Sterile'},\n",
       "   'description': ['<p>This family represents the C-terminal region of the male sterility protein in a number of arabidopsis and drosophila.  A sequence-related jojoba acyl CoA reductase is also included.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007457': {'PMID': 9351246,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '3',\n",
       "     'year': 1997,\n",
       "     'title': 'The Arabidopsis MALE STERILITY 2 protein shares similarity with reductases in elongation/condensation complexes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '615-23',\n",
       "     'medline_journal': 'Plant J',\n",
       "     'ISO_journal': 'Plant J.',\n",
       "     'authors': ['Aarts MG',\n",
       "      'Hodge R',\n",
       "      'Kalantidis K',\n",
       "      'Florack D',\n",
       "      'Wilson ZA',\n",
       "      'Mulligan BJ',\n",
       "      'Stiekema WJ',\n",
       "      'Scott R',\n",
       "      'Pereira A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-313X.1997.00615.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 185,\n",
       "    'interactions': 0,\n",
       "    'matches': 9725,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9561,\n",
       "    'proteomes': 1188,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7590, 'rosettafold': 1},\n",
       "    'structures': 1,\n",
       "    'taxa': 5159},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7886,\n",
       "    'alignment:full': 4423,\n",
       "    'alignment:seed': 311},\n",
       "   'cross_references': {}}},\n",
       " 'PF14842': {'metadata': {'accession': 'PF14842',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR028263',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FliG N-terminal domain', 'short': 'FliG_N'},\n",
       "   'description': ['<p>This is the N-terminal domain of the flagellar rotor protein FliG [PMID:20676082].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0436', 'name': 'FliG'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 20769,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20764,\n",
       "    'proteomes': 4151,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17970, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 17708},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19445,\n",
       "    'alignment:full': 4223,\n",
       "    'alignment:seed': 308},\n",
       "   'cross_references': {}}},\n",
       " 'PF02739': {'metadata': {'accession': 'PF02739',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020046',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"5'-3' exonuclease, N-terminal resolvase-like domain\",\n",
       "    'short': '5_3_exonuc_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019069': {'PMID': 8717047,\n",
       "     'ISBN': None,\n",
       "     'volume': '382',\n",
       "     'issue': '6588',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure of Taq polymerase with DNA at the polymerase active site.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '278-81',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Eom SH', 'Wang J', 'Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/382278a0'},\n",
       "    'PUB00019070': {'PMID': 8657312,\n",
       "     'ISBN': None,\n",
       "     'volume': '382',\n",
       "     'issue': '6586',\n",
       "     'year': 1996,\n",
       "     'title': \"A helical arch allowing single-stranded DNA to thread through T5 5'-exonuclease.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '90-3',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Ceska TA', 'Sayers JR', 'Stier G', 'Suck D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/382090a0'}},\n",
       "   'set_info': {'accession': 'CL0280', 'name': 'PIN'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 134,\n",
       "    'interactions': 0,\n",
       "    'matches': 64051,\n",
       "    'pathways': 0,\n",
       "    'proteins': 64038,\n",
       "    'proteomes': 9285,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 53782, 'rosettafold': 0},\n",
       "    'structures': 29,\n",
       "    'taxa': 39850},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 59625,\n",
       "    'alignment:full': 11311,\n",
       "    'alignment:seed': 716},\n",
       "   'cross_references': {}}},\n",
       " 'PF00391': {'metadata': {'accession': 'PF00391',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008279',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PEP-utilising enzyme, mobile domain',\n",
       "    'short': 'PEP-utilizers'},\n",
       "   'description': ['<p>This domain is a \"swivelling\" beta/beta/alpha domain which is thought to be mobile in all proteins known to contain it.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004887': {'PMID': 8610096,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '7',\n",
       "     'year': 1996,\n",
       "     'title': 'Swiveling-domain mechanism for enzymatic phosphotransfer between remote reaction sites.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2652-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Herzberg O',\n",
       "      'Chen CC',\n",
       "      'Kapadia G',\n",
       "      'McGuire M',\n",
       "      'Carroll LJ',\n",
       "      'Noh SJ',\n",
       "      'Dunaway-Mariano D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.7.2652'}},\n",
       "   'set_info': {'accession': 'CL0364', 'name': 'Leu-IlvD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 330,\n",
       "    'interactions': 0,\n",
       "    'matches': 112459,\n",
       "    'pathways': 0,\n",
       "    'proteins': 112278,\n",
       "    'proteomes': 8765,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 95425, 'rosettafold': 0},\n",
       "    'structures': 39,\n",
       "    'taxa': 38976},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 104224,\n",
       "    'alignment:full': 20244,\n",
       "    'alignment:seed': 1219},\n",
       "   'cross_references': {}}},\n",
       " 'PF03600': {'metadata': {'accession': 'PF03600',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004680',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Citrate transporter', 'short': 'CitMHS'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0182', 'name': 'IT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 190,\n",
       "    'interactions': 0,\n",
       "    'matches': 70527,\n",
       "    'pathways': 0,\n",
       "    'proteins': 63954,\n",
       "    'proteomes': 7221,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 54738, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 31894},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 72719,\n",
       "    'alignment:full': 15680,\n",
       "    'alignment:seed': 518},\n",
       "   'cross_references': {}}},\n",
       " 'PF17801': {'metadata': {'accession': 'PF17801',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041233',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Alpha galactosidase C-terminal beta sandwich domain',\n",
       "    'short': 'Melibiase_C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of alpha galactosidase enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0369', 'name': 'GHD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 413,\n",
       "    'interactions': 0,\n",
       "    'matches': 15536,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15502,\n",
       "    'proteomes': 2314,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13086, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 8778},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13513,\n",
       "    'alignment:full': 5631,\n",
       "    'alignment:seed': 325},\n",
       "   'cross_references': {}}},\n",
       " 'PF06245': {'metadata': {'accession': 'PF06245',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008323',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1015)',\n",
       "    'short': 'DUF1015'},\n",
       "   'description': ['<p>Family of proteins with unknown function found in archaea and bacteria.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 27,\n",
       "    'interactions': 0,\n",
       "    'matches': 17057,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16939,\n",
       "    'proteomes': 2843,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13879, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11853},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14460,\n",
       "    'alignment:full': 2712,\n",
       "    'alignment:seed': 305},\n",
       "   'cross_references': {}}},\n",
       " 'PF17803': {'metadata': {'accession': 'PF17803',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040853',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial cadherin-like domain', 'short': 'Cadherin_4'},\n",
       "   'description': ['<p>This entry contains numerous bacterial cadherin-like domains found in extracelullar proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 3648,\n",
       "    'interactions': 0,\n",
       "    'matches': 16839,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7404,\n",
       "    'proteomes': 903,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 2894, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 5366},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16574,\n",
       "    'alignment:full': 3877,\n",
       "    'alignment:seed': 694},\n",
       "   'cross_references': {}}},\n",
       " 'PF02207': {'metadata': {'accession': 'PF02207',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003126',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative zinc finger in N-recognin (UBR box)',\n",
       "    'short': 'zf-UBR'},\n",
       "   'description': ['<p>This region is found in E3 ubiquitin ligases that recognise N-recognins [PMID:16055722].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00033696': {'PMID': 16055722,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '16',\n",
       "     'year': 2005,\n",
       "     'title': 'A family of mammalian E3 ubiquitin ligases that contain the UBR box motif and recognize N-degrons.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7120-36',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Tasaki T',\n",
       "      'Mulder LC',\n",
       "      'Iwamatsu A',\n",
       "      'Lee MJ',\n",
       "      'Davydov IV',\n",
       "      'Varshavsky A',\n",
       "      'Muesing M',\n",
       "      'Kwon YT.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/MCB.25.16.7120-7136.2005'},\n",
       "    'PUB00033697': {'PMID': 14585983,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '22',\n",
       "     'year': 2003,\n",
       "     'title': 'Female lethality and apoptosis of spermatocytes in mice lacking the UBR2 ubiquitin ligase of the N-end rule pathway.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8255-71',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Kwon YT',\n",
       "      'Xia Z',\n",
       "      'An JY',\n",
       "      'Tasaki T',\n",
       "      'Davydov IV',\n",
       "      'Seo JW',\n",
       "      'Sheng J',\n",
       "      'Xie Y',\n",
       "      'Varshavsky A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/MCB.23.22.8255-8271.2003'},\n",
       "    'PUB00033698': {'PMID': 11689692,\n",
       "     'ISBN': None,\n",
       "     'volume': '21',\n",
       "     'issue': '23',\n",
       "     'year': 2001,\n",
       "     'title': 'Construction and analysis of mouse strains lacking the ubiquitin ligase UBR1 (E3alpha) of the N-end rule pathway.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8007-21',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Kwon YT',\n",
       "      'Xia Z',\n",
       "      'Davydov IV',\n",
       "      'Lecker SH',\n",
       "      'Varshavsky A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/MCB.21.23.8007-8021.2001'},\n",
       "    'PUB00033699': {'PMID': 9653112,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '14',\n",
       "     'year': 1998,\n",
       "     'title': 'The mouse and human genes encoding the recognition component of the N-end rule pathway.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7898-903',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Kwon YT',\n",
       "      'Reiss Y',\n",
       "      'Fried VA',\n",
       "      'Hershko A',\n",
       "      'Yoon JK',\n",
       "      'Gonda DK',\n",
       "      'Sangan P',\n",
       "      'Copeland NG',\n",
       "      'Jenkins NA',\n",
       "      'Varshavsky A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.95.14.7898'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 198,\n",
       "    'interactions': 0,\n",
       "    'matches': 13690,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13660,\n",
       "    'proteomes': 2111,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4571, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 7639},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10289,\n",
       "    'alignment:full': 6512,\n",
       "    'alignment:seed': 269},\n",
       "   'cross_references': {}}},\n",
       " 'PF03595': {'metadata': {'accession': 'PF03595',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004695',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Voltage-dependent anion channel', 'short': 'SLAC1'},\n",
       "   'description': ['<p>This family of transporters has ten alpha helical transmembrane segments [PMID:20981093].  The structure of a bacterial homologue of SLAC1 shows it to have a trimeric arrangement. The pore is composed of five helices with a conserved Phe residue involved in gating. One homologue, Mae1 from the yeast Schizosaccharomyces pombe, functions as a malate uptake transporter; another, Ssu1 from Saccharomyces cerevisiae and other fungi including Aspergillus fumigatus, is characterised as a sulfite efflux pump; and TehA from Escherichia coli is identified as a tellurite resistance protein by virtue of its association in the tehA/tehB operon. In plants, this family is found in the stomatal guard cells functioning as an anion-transporting pore [PMID:18305484]. Many homologues are incorrectly annotated as tellurite resistance or dicarboxylate transporter (TDT) proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 128,\n",
       "    'interactions': 0,\n",
       "    'matches': 21264,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21096,\n",
       "    'proteomes': 3317,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18438, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 14942},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20197,\n",
       "    'alignment:full': 6485,\n",
       "    'alignment:seed': 431},\n",
       "   'cross_references': {}}},\n",
       " 'PF04253': {'metadata': {'accession': 'PF04253',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007365',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transferrin receptor-like dimerisation domain',\n",
       "    'short': 'TFR_dimer'},\n",
       "   'description': ['<p>This domain is involved in dimerisation of the transferrin receptor as shown in its crystal structure.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 106,\n",
       "    'interactions': 0,\n",
       "    'matches': 10706,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10664,\n",
       "    'proteomes': 2010,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8274, 'rosettafold': 0},\n",
       "    'structures': 104,\n",
       "    'taxa': 7376},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8143,\n",
       "    'alignment:full': 4478,\n",
       "    'alignment:seed': 279},\n",
       "   'cross_references': {}}},\n",
       " 'PF03861': {'metadata': {'accession': 'PF03861',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005561',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ANTAR domain', 'short': 'ANTAR'},\n",
       "   'description': ['<p>ANTAR (AmiR and NasR transcription antitermination regulators) is an RNA-binding domain found in bacterial transcription antitermination regulatory proteins. The majority of the  domain consists of a coiled-coil. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011193': {'PMID': 11796212,\n",
       "     'ISBN': None,\n",
       "     'volume': '27',\n",
       "     'issue': '1',\n",
       "     'year': 2002,\n",
       "     'title': 'ANTAR: an RNA-binding domain in transcription antitermination regulatory proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3-5',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Shu CJ', 'Zhulin IB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(01)02036-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 135,\n",
       "    'interactions': 0,\n",
       "    'matches': 39104,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39087,\n",
       "    'proteomes': 4298,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34976, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 17148},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37853,\n",
       "    'alignment:full': 9251,\n",
       "    'alignment:seed': 613},\n",
       "   'cross_references': {}}},\n",
       " 'PF02671': {'metadata': {'accession': 'PF02671',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003822',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Paired amphipathic helix repeat', 'short': 'PAH'},\n",
       "   'description': ['<p>This family contains the paired amphipathic helix repeat. The family contains the yeast SIN3 gene Swiss:P22579 (also known as SDI1) that is a negative regulator of the yeast HO gene [PMID:2233725]. This repeat may be distantly related to the helix-loop-helix motif, which mediate protein-protein interactions.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008273': {'PMID': 2233725,\n",
       "     'ISBN': None,\n",
       "     'volume': '10',\n",
       "     'issue': '11',\n",
       "     'year': 1990,\n",
       "     'title': 'The Saccharomyces cerevisiae SIN3 gene, a negative regulator of HO, contains four paired amphipathic helix motifs.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5927-36',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Wang H',\n",
       "      'Clark I',\n",
       "      'Nicholson PR',\n",
       "      'Herskowitz I',\n",
       "      'Stillman DJ.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=2233725&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 221,\n",
       "    'interactions': 0,\n",
       "    'matches': 20197,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9343,\n",
       "    'proteomes': 2057,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4179, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 7407},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15808,\n",
       "    'alignment:full': 10284,\n",
       "    'alignment:seed': 348},\n",
       "   'cross_references': {}}},\n",
       " 'PF01352': {'metadata': {'accession': 'PF01352',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001909',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'KRAB box', 'short': 'KRAB'},\n",
       "   'description': ['<p>The KRAB domain (or Kruppel-associated box) is present in about a third of zinc finger proteins containing C2H2 fingers.  The KRAB domain is found to be involved in protein-protein interactions [PMID:8986806, PMID:8769649].  The KRAB domain is generally encoded by two exons.  The regions coded by the two exons are known as KRAB-A and KRAB-B. The A box plays an important role in repression by binding to corepressors, while the B box is thought to enhance this repression brought about by the A box. KRAB-containing proteins are thought to have critical functions in cell proliferation and differentiation, apoptosis and neoplastic transformation [PMID:14519192].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001925': {'PMID': 8769649,\n",
       "     'ISBN': None,\n",
       "     'volume': '10',\n",
       "     'issue': '16',\n",
       "     'year': 1996,\n",
       "     'title': 'KAP-1, a novel corepressor for the highly conserved KRAB repression domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2067-78',\n",
       "     'medline_journal': 'Genes Dev',\n",
       "     'ISO_journal': 'Genes Dev.',\n",
       "     'authors': ['Friedman JR',\n",
       "      'Fredericks WJ',\n",
       "      'Jensen DE',\n",
       "      'Speicher DW',\n",
       "      'Huang XP',\n",
       "      'Neilson EG',\n",
       "      'Rauscher FJ 3rd.'],\n",
       "     'DOI_URL': 'http://www.genesdev.org/cgi/content/abstract/10/16/2067'},\n",
       "    'PUB00004392': {'PMID': 1861988,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '14',\n",
       "     'year': 1991,\n",
       "     'title': 'Conserved KRAB protein domain identified upstream from the zinc finger region of Kox 8.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3996',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Thiesen HJ', 'Bellefroid E', 'Revelant O', 'Martial JA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/19.14.3996'},\n",
       "    'PUB00004882': {'PMID': 8986806,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '26',\n",
       "     'year': 1996,\n",
       "     'title': 'A novel member of the RING finger family, KRIP-1, associates with the KRAB-A transcriptional repressor domain of zinc finger proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15299-304',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Kim SS',\n",
       "      'Chen YM',\n",
       "      \"O'Leary E\",\n",
       "      'Witzgall R',\n",
       "      'Vidal M',\n",
       "      'Bonventre JV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.26.15299'},\n",
       "    'PUB00011911': {'PMID': 14519192,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '10',\n",
       "     'year': 2003,\n",
       "     'title': 'KRAB-containing zinc-finger repressor proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '231',\n",
       "     'medline_journal': 'Genome Biol',\n",
       "     'ISO_journal': 'Genome Biol.',\n",
       "     'authors': ['Urrutia R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/gb-2003-4-10-231'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 3065,\n",
       "    'interactions': 0,\n",
       "    'matches': 67996,\n",
       "    'pathways': 0,\n",
       "    'proteins': 66297,\n",
       "    'proteomes': 503,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46519, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 1620},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51729,\n",
       "    'alignment:full': 39716,\n",
       "    'alignment:seed': 427},\n",
       "   'cross_references': {}}},\n",
       " 'PF07971': {'metadata': {'accession': 'PF07971',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012939',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolase family 92', 'short': 'Glyco_hydro_92'},\n",
       "   'description': ['<p>Members of this family are alpha-1,2-mannosidases, enzymes which remove alpha-1,2-linked mannose residues from Man(9)(GlcNAc)(2) by hydrolysis. They are critical for the maturation of N-linked oligosaccharides and ER-associated degradation [PMID:10026209].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016430': {'PMID': 10026209,\n",
       "     'ISBN': None,\n",
       "     'volume': '274',\n",
       "     'issue': '9',\n",
       "     'year': 1999,\n",
       "     'title': 'Oligosaccharide modification in the early secretory pathway directs the selection of a misfolded glycoprotein for degradation by the proteasome.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5861-7',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Liu Y', 'Choudhury P', 'Cabral CM', 'Sifers RN.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.274.9.5861'},\n",
       "    'PUB00044284': {'PMID': 16131397,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': None,\n",
       "     'year': 2005,\n",
       "     'title': 'GH97 is a new family of glycoside hydrolases, which is related to the alpha-galactosidase superfamily.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '112',\n",
       "     'medline_journal': 'BMC Genomics',\n",
       "     'ISO_journal': 'BMC Genomics',\n",
       "     'authors': ['Naumoff DG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2164-6-112'}},\n",
       "   'set_info': {'accession': 'CL0268', 'name': 'Pec_lyase-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 292,\n",
       "    'interactions': 0,\n",
       "    'matches': 30110,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29528,\n",
       "    'proteomes': 2511,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25271, 'rosettafold': 0},\n",
       "    'structures': 18,\n",
       "    'taxa': 9959},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28074,\n",
       "    'alignment:full': 7669,\n",
       "    'alignment:seed': 625},\n",
       "   'cross_references': {}}},\n",
       " 'PF02563': {'metadata': {'accession': 'PF02563',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003715',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polysaccharide biosynthesis/export protein',\n",
       "    'short': 'Poly_export'},\n",
       "   'description': ['<p>This is a family of periplasmic proteins involved in polysaccharide biosynthesis and/or export.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008202': {'PMID': 8759852,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '16',\n",
       "     'year': 1996,\n",
       "     'title': 'Organization of the Escherichia coli K-12 gene cluster responsible for production of the extracellular polysaccharide colanic acid.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4885-93',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Stevenson G', 'Andrianopoulos K', 'Hobbs M', 'Reeves PR.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8759852&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 199,\n",
       "    'interactions': 0,\n",
       "    'matches': 49648,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48820,\n",
       "    'proteomes': 4203,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41981, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 19021},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45679,\n",
       "    'alignment:full': 9230,\n",
       "    'alignment:seed': 541},\n",
       "   'cross_references': {}}},\n",
       " 'PF04366': {'metadata': {'accession': 'PF04366',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007461',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Las17-binding protein actin regulator', 'short': 'Ysc84'},\n",
       "   'description': ['<p>Ysc84 is a family of Las17-binding proteins found in metazoa. Together, Las17 and Ysc84 are essential for proper polymerisation of actin; Ysc84 is able to bind to and stabilise the actin dimer presented by Las17 and thereby promote polymerisation. An active actin cytoskeleton is necessary for adequate endocytosis.  (Pfam:PF00018), or a FYVE zinc finger (Pfam:PF01363).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 155,\n",
       "    'interactions': 0,\n",
       "    'matches': 15282,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15259,\n",
       "    'proteomes': 3078,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12720, 'rosettafold': 1},\n",
       "    'structures': 1,\n",
       "    'taxa': 11750},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13106,\n",
       "    'alignment:full': 5147,\n",
       "    'alignment:seed': 261},\n",
       "   'cross_references': {}}},\n",
       " 'PF04892': {'metadata': {'accession': 'PF04892',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006976',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'VanZ like family', 'short': 'VanZ'},\n",
       "   'description': ['<p>This family contains several examples of the VanZ protein, but also contains examples of phosphotransbutyrylases [PMID:7867956].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008695': {'PMID': 7867956,\n",
       "     'ISBN': None,\n",
       "     'volume': '154',\n",
       "     'issue': '1',\n",
       "     'year': 1995,\n",
       "     'title': 'The vanZ gene of Tn1546 from Enterococcus faecium BM4147 confers resistance to teicoplanin.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '87-92',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Arthur M',\n",
       "      'Depardieu F',\n",
       "      'Molinas C',\n",
       "      'Reynolds P',\n",
       "      'Courvalin P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(94)00851-I'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 117,\n",
       "    'interactions': 0,\n",
       "    'matches': 40639,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40363,\n",
       "    'proteomes': 5280,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35225, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21370},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38095,\n",
       "    'alignment:full': 8602,\n",
       "    'alignment:seed': 316},\n",
       "   'cross_references': {}}},\n",
       " 'PF13376': {'metadata': {'accession': 'PF13376',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacteriocin-protection, YdeI or OmpD-Associated',\n",
       "    'short': 'OmdA'},\n",
       "   'description': ['<p>This is a family of archaeal and bacterial proteins predicted to be periplasmic. YdeI is important for resistance to polymyxin B in broth and for bacterial survival in mice upon oral, but not intraperitoneal inoculation, suggesting a role for YdeI in the gastrointestinal tract of mice [PMID:17010160]. Production of the ydeI gene is regulated by the Rcs (regulator of capsule synthesis) phospho-relay system pathway independently of RcsA, and additionally transcription of the protein is regulated by the stationary-phase sigma factor, RpoS (sigma-S) [PMID:17010160]. YdeI confers protection against cationic AMPs (Antimicrobial peptides) or bacteriocins in conjunction with the general porin Omp, thus justifying its name of OmdA, for OmpD-Associated protein [PMID:19767429].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 23081,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22977,\n",
       "    'proteomes': 3485,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19364, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 14302},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20475,\n",
       "    'alignment:full': 5228,\n",
       "    'alignment:seed': 290},\n",
       "   'cross_references': {}}},\n",
       " 'PF01987': {'metadata': {'accession': 'PF01987',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002838',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Mitochondrial biogenesis AIM24', 'short': 'AIM24'},\n",
       "   'description': ['<p>In eukaryotes, this domain is involved in mitochondrial biogenesis [PMID:19300474]. Its function in prokaryotes in unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 78,\n",
       "    'interactions': 0,\n",
       "    'matches': 28755,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28717,\n",
       "    'proteomes': 4495,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24845, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 17898},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26146,\n",
       "    'alignment:full': 7524,\n",
       "    'alignment:seed': 460},\n",
       "   'cross_references': {}}},\n",
       " 'PF01825': {'metadata': {'accession': 'PF01825',\n",
       "   'entry_id': None,\n",
       "   'type': 'conserved_site',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000203',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GPCR proteolysis site, GPS, motif', 'short': 'GPS'},\n",
       "   'description': ['<p>The GPS motif is found in GPCRs, and is the site for auto-proteolysis, so is thus named, GPS [PMID:9920906, PMID:9830014, PMID:10469603, PMID:17525154]. The GPS motif is a conserved sequence of ~40 amino acids containing canonical cysteine and tryptophan residues, and is the most highly conserved part of the domain. In most, if not all, cell-adhesion GPCRs these undergo autoproteolysis in the GPS between a conserved aliphatic residue (usually a leucine) and a threonine, serine, or cysteine residue [PMID:12270923]. In higher eukaryotes this motif is found embedded in the C-terminal beta-stranded part of a GAIN domain - GPCR-Autoproteolysis INducing (GAIN). The GAIN-GPS domain adopts a fold in which the GPS motif, at the C-terminus, forms five beta-strands that are tightly integrated into the overall GAIN domain. The GPS motif, evolutionarily conserved from tetrahymena to mammals, is the only extracellular domain shared by all human cell-adhesion GPCRs and PKD proteins, and is the locus of multiple human disease mutations. The GAIN-GPS domain is both necessary and sufficient functionally for autoproteolysis, suggesting an autoproteolytic mechanism whereby the overall GAIN domain fine-tunes the chemical environment in the GPS to catalyse peptide bond hydrolysis [PMID:22333914]. In the cell-adhesion GPCRs and PKD proteins, the GPS motif is always located at the end of their long N-terminal extracellular regions, immediately before the first transmembrane helix of the respective protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005835': {'PMID': 9830014,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '49',\n",
       "     'year': 1998,\n",
       "     'title': 'alpha-Latrotoxin receptor CIRL/latrophilin 1 (CL1) defines an unusual family of ubiquitous G-protein-linked receptors. G-protein coupling not required for triggering exocytosis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '32715-24',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Sugita S', 'Ichtchenko K', 'Khvotchev M', 'Sudhof TC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.49.32715'},\n",
       "    'PUB00005840': {'PMID': 9920906,\n",
       "     'ISBN': None,\n",
       "     'volume': '274',\n",
       "     'issue': '6',\n",
       "     'year': 1999,\n",
       "     'title': 'Structural requirements for alpha-latrotoxin binding and alpha-latrotoxin-stimulated secretion. A study with calcium-independent receptor of alpha-latrotoxin (CIRL) deletion mutants.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3590-6',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Krasnoperov V',\n",
       "      'Bittner MA',\n",
       "      'Holz RW',\n",
       "      'Chepurny O',\n",
       "      'Petrenko AG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.274.6.3590'},\n",
       "    'PUB00008017': {'PMID': 10469603,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '16',\n",
       "     'year': 1999,\n",
       "     'title': 'A latrophilin/CL-1-like GPS domain in polycystin-1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': 'R585-8',\n",
       "     'medline_journal': 'Curr Biol',\n",
       "     'ISO_journal': 'Curr. Biol.',\n",
       "     'authors': ['Ponting CP', 'Hofmann K', 'Bork P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0960-9822(99)80379-0'}},\n",
       "   'set_info': {'accession': 'CL0661', 'name': 'Gain'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1617,\n",
       "    'interactions': 0,\n",
       "    'matches': 38491,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38201,\n",
       "    'proteomes': 968,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15025, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 4003},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25550,\n",
       "    'alignment:full': 17360,\n",
       "    'alignment:seed': 295},\n",
       "   'cross_references': {}}},\n",
       " 'PF00763': {'metadata': {'accession': 'PF00763',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020630',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetrahydrofolate dehydrogenase/cyclohydrolase, catalytic domain',\n",
       "    'short': 'THF_DHG_CYH'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020432': {'PMID': 9519408,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'The 3-D structure of a folate-dependent dehydrogenase/cyclohydrolase bifunctional enzyme at 1.5 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '173-82',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Allaire M', 'Li Y', 'MacKenzie RE', 'Cygler M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(98)00019-7'}},\n",
       "   'set_info': {'accession': 'CL0603', 'name': 'AA_dh_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 223,\n",
       "    'interactions': 0,\n",
       "    'matches': 61521,\n",
       "    'pathways': 0,\n",
       "    'proteins': 61496,\n",
       "    'proteomes': 10763,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 52103, 'rosettafold': 0},\n",
       "    'structures': 42,\n",
       "    'taxa': 44826},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54727,\n",
       "    'alignment:full': 14112,\n",
       "    'alignment:seed': 1140},\n",
       "   'cross_references': {}}},\n",
       " 'PF02592': {'metadata': {'accession': 'PF02592',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003744',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative vitamin uptake transporter', 'short': 'Vut_1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0315', 'name': 'Gx_transp'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 40,\n",
       "    'interactions': 0,\n",
       "    'matches': 23043,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22681,\n",
       "    'proteomes': 4032,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19410, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 18329},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21842,\n",
       "    'alignment:full': 4371,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF06429': {'metadata': {'accession': 'PF06429',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010930',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flagellar basal body rod FlgEFG protein C-terminal',\n",
       "    'short': 'Flg_bbr_C'},\n",
       "   'description': ['<p>This family consists of a number of C-terminal domains of unknown function. This domain seems to be specific to flagellar basal-body rod and flagellar hook proteins in which Pfam:PF00460 is often present at the extreme N terminus.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 83,\n",
       "    'interactions': 0,\n",
       "    'matches': 105727,\n",
       "    'pathways': 0,\n",
       "    'proteins': 105709,\n",
       "    'proteomes': 4262,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 92347, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 18488},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 101228,\n",
       "    'alignment:full': 20840,\n",
       "    'alignment:seed': 450},\n",
       "   'cross_references': {}}},\n",
       " 'PF00250': {'metadata': {'accession': 'PF00250',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001766',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Forkhead domain', 'short': 'Forkhead'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004151': {'PMID': 8332212,\n",
       "     'ISBN': None,\n",
       "     'volume': '364',\n",
       "     'issue': '6436',\n",
       "     'year': 1993,\n",
       "     'title': 'Co-crystal structure of the HNF-3/fork head DNA-recognition motif resembles histone H5.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '412-20',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Clark KL', 'Halay ED', 'Lai E', 'Burley SK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/364412a0'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 301,\n",
       "    'interactions': 0,\n",
       "    'matches': 50145,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49871,\n",
       "    'proteomes': 1817,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37425, 'rosettafold': 0},\n",
       "    'structures': 52,\n",
       "    'taxa': 7545},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37549,\n",
       "    'alignment:full': 21731,\n",
       "    'alignment:seed': 276},\n",
       "   'cross_references': {}}},\n",
       " 'PF13277': {'metadata': {'accession': 'PF13277',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005235',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YmdB-like protein', 'short': 'YmdB'},\n",
       "   'description': ['<p>This family of putative phosphoesterases contains the B. subtilis protein YmdB Swiss:O31775.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0163', 'name': 'Calcineurin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 22,\n",
       "    'interactions': 0,\n",
       "    'matches': 14831,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14823,\n",
       "    'proteomes': 2742,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12395, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 11874},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12957,\n",
       "    'alignment:full': 2567,\n",
       "    'alignment:seed': 301},\n",
       "   'cross_references': {}}},\n",
       " 'PF00684': {'metadata': {'accession': 'PF00684',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001305',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DnaJ central domain', 'short': 'DnaJ_CXXCXGXG'},\n",
       "   'description': ['<p>The central cysteine-rich (CR) domain of DnaJ proteins contains four repeats of the motif CXXCXGXG where X is any amino acid. The isolated cysteine rich domain folds in zinc dependent fashion. Each set of two repeats binds one unit of zinc.  Although this domain has been implicated in substrate binding, no evidence of specific interaction between the isolated DNAJ cysteine rich domain and various hydrophobic peptides has been found [PMID:10891270].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019436': {'PMID': 10891270,\n",
       "     'ISBN': None,\n",
       "     'volume': '300',\n",
       "     'issue': '4',\n",
       "     'year': 2000,\n",
       "     'title': 'Solution structure of the cysteine-rich domain of the Escherichia coli chaperone protein DnaJ.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '805-18',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Martinez-Yamout M',\n",
       "      'Legge GB',\n",
       "      'Zhang O',\n",
       "      'Wright PE',\n",
       "      'Dyson HJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.2000.3923'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 283,\n",
       "    'interactions': 0,\n",
       "    'matches': 69101,\n",
       "    'pathways': 0,\n",
       "    'proteins': 69071,\n",
       "    'proteomes': 10992,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 58672, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 46216},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61182,\n",
       "    'alignment:full': 17430,\n",
       "    'alignment:seed': 1237},\n",
       "   'cross_references': {}}},\n",
       " 'PF03073': {'metadata': {'accession': 'PF03073',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004307',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TspO/MBR family', 'short': 'TspO_MBR'},\n",
       "   'description': ['<p>Tryptophan-rich sensory protein (TspO) is an integral membrane protein that acts as a negative regulator of the expression of specific photosynthesis genes in response to oxygen/light [PMID:7673149]. It is involved in the efflux of porphyrin intermediates from the cell. This reduces the activity of coproporphyrinogen III oxidase, which is thought to lead to the accumulation of a putative repressor molecule that inhibits the expression of specific photosynthesis genes. Several conserved aromatic residues are necessary for TspO function: they are thought to be involved in binding porphyrin intermediates [PMID:10681549]. In [PMID:9144197], the rat mitochondrial peripheral benzodiazepine receptor (MBR) was shown to not only retain its structure within a bacterial outer membrane, but also to be able to functionally substitute for TspO in TspO- mutants, and to act in a similar manner to TspO in its in situ location: the outer mitochondrial membrane. The biological significance of MBR remains unclear, however. It is thought to be involved in a variety of cellular functions, including cholesterol transport in steroidogenic tissues.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007483': {'PMID': 7673149,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '36',\n",
       "     'year': 1995,\n",
       "     'title': 'A sensory transducer homologous to the mammalian peripheral-type benzodiazepine receptor regulates photosynthetic membrane complex formation in Rhodobacter sphaeroides 2.4.1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '21167-75',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Yeliseev AA', 'Kaplan S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.270.36.21167'},\n",
       "    'PUB00007484': {'PMID': 10681549,\n",
       "     'ISBN': None,\n",
       "     'volume': '275',\n",
       "     'issue': '8',\n",
       "     'year': 2000,\n",
       "     'title': 'TspO of rhodobacter sphaeroides. A structural and functional model for the mammalian peripheral benzodiazepine receptor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5657-67',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Yeliseev AA', 'Kaplan S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.275.8.5657'},\n",
       "    'PUB00007485': {'PMID': 9144197,\n",
       "     'ISBN': None,\n",
       "     'volume': '94',\n",
       "     'issue': '10',\n",
       "     'year': 1997,\n",
       "     'title': 'A mammalian mitochondrial drug receptor functions as a bacterial \"oxygen\" sensor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5101-6',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Yeliseev AA', 'Krueger KE', 'Kaplan S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.94.10.5101'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 19944,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19889,\n",
       "    'proteomes': 4799,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16724, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 20371},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17506,\n",
       "    'alignment:full': 4992,\n",
       "    'alignment:seed': 425},\n",
       "   'cross_references': {}}},\n",
       " 'PF02574': {'metadata': {'accession': 'PF02574',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003726',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Homocysteine S-methyltransferase',\n",
       "    'short': 'S-methyl_trans'},\n",
       "   'description': ['<p>This is a family of related homocysteine S-methyltransferases enzymes: 5-methyltetrahydrofolate--homocysteine S-methyltransferases  also known EC:2.1.1.13, [PMID:9013615]; Betaine--homocysteine S-methyltransferase  (vitamin B12 dependent), EC:2.1.1.5, [PMID:8798461]; and Homocysteine  S-methyltransferase, EC:2.1.1.10, [PMID:9882684].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008217': {'PMID': 9882684,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '2',\n",
       "     'year': 1999,\n",
       "     'title': 'S-methylmethionine metabolism in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '662-5',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Thanbichler M', 'Neuhierl B', 'Bock A.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=9882684'},\n",
       "    'PUB00008218': {'PMID': 9013615,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'Human methionine synthase. cDNA cloning, gene localization, and expression.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3628-34',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Chen LH',\n",
       "      'Liu ML',\n",
       "      'Hwang HY',\n",
       "      'Chen LS',\n",
       "      'Korenberg J',\n",
       "      'Shane B.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.6.3444'},\n",
       "    'PUB00008219': {'PMID': 8798461,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '37',\n",
       "     'year': 1996,\n",
       "     'title': 'Purification, kinetic properties, and cDNA cloning of mammalian betaine-homocysteine methyltransferase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '22831-8',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Garrow TA.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/full/271/37/22831'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 260,\n",
       "    'interactions': 0,\n",
       "    'matches': 63066,\n",
       "    'pathways': 0,\n",
       "    'proteins': 62637,\n",
       "    'proteomes': 8881,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 51870, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 35986},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 56747,\n",
       "    'alignment:full': 13090,\n",
       "    'alignment:seed': 664},\n",
       "   'cross_references': {}}},\n",
       " 'PF00397': {'metadata': {'accession': 'PF00397',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001202',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'WW domain', 'short': 'WW'},\n",
       "   'description': ['<p>The WW domain is a protein module with two highly conserved tryptophans that binds proline-rich peptide motifs in vitro.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001690': {'PMID': 7641887,\n",
       "     'ISBN': None,\n",
       "     'volume': '369',\n",
       "     'issue': '1',\n",
       "     'year': 1995,\n",
       "     'title': 'Characterization of a novel protein-binding module--the WW domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '67-71',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Sudol M', 'Chen HI', 'Bougeret C', 'Einbond A', 'Bork P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0014-5793(95)00550-S'},\n",
       "    'PUB00004240': {'PMID': 8757138,\n",
       "     'ISBN': None,\n",
       "     'volume': '382',\n",
       "     'issue': '6592',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure of the WW domain of a kinase-associated protein complexed with a proline-rich peptide.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '646-9',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Macias MJ',\n",
       "      'Hyvonen M',\n",
       "      'Baraldi E',\n",
       "      'Schultz J',\n",
       "      'Sudol M',\n",
       "      'Saraste M',\n",
       "      'Oschkinat H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/382646a0'},\n",
       "    'PUB00004872': {'PMID': 7644498,\n",
       "     'ISBN': None,\n",
       "     'volume': '92',\n",
       "     'issue': '17',\n",
       "     'year': 1995,\n",
       "     'title': 'The WW domain of Yes-associated protein binds a proline-rich ligand that differs from the consensus established for Src homology 3-binding modules.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7819-23',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Chen HI', 'Sudol M.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=7644498&action=stream&blobtype=pdf'},\n",
       "    'PUB00005431': {'PMID': 7846762,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '12',\n",
       "     'year': 1994,\n",
       "     'title': 'The WW domain: a signalling site in dystrophin?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '531-3',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Bork P', 'Sudol M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0968-0004(94)90053-1'},\n",
       "    'PUB00020554': {'PMID': 11870222,\n",
       "     'ISBN': None,\n",
       "     'volume': '115',\n",
       "     'issue': 'Pt 5',\n",
       "     'year': 2002,\n",
       "     'title': 'Substrate proteolysis is inhibited by dominant-negative Nedd4 and Rsp5 mutants harboring alterations in WW domain 1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1041-8',\n",
       "     'medline_journal': 'J Cell Sci',\n",
       "     'ISO_journal': 'J. Cell. Sci.',\n",
       "     'authors': ['Shcherbik N', 'Kumar S', 'Haines DS.'],\n",
       "     'DOI_URL': 'http://jcs.biologists.org/cgi/content/abstract/115/5/1041'},\n",
       "    'PUB00020555': {'PMID': 11323714,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '5',\n",
       "     'year': 2001,\n",
       "     'title': 'Solution structure of a Nedd4 WW domain-ENaC peptide complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '407-12',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Kanelis V', 'Rotin D', 'Forman-Kay JD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/87562'},\n",
       "    'PUB00020556': {'PMID': 10932238,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '8',\n",
       "     'year': 2000,\n",
       "     'title': 'Converging on proline: the mechanism of WW domain peptide recognition.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '611-3',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Zarrinpar A', 'Lim WA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/77891'},\n",
       "    'PUB00020558': {'PMID': 10744724,\n",
       "     'ISBN': None,\n",
       "     'volume': '275',\n",
       "     'issue': '14',\n",
       "     'year': 2000,\n",
       "     'title': 'A novel pro-Arg motif recognized by WW domains.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10359-69',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Bedford MT', 'Sarbassova D', 'Xu J', 'Leder P', 'Yaffe MB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.275.14.10359'},\n",
       "    'PUB00020559': {'PMID': 10037602,\n",
       "     'ISBN': None,\n",
       "     'volume': '283',\n",
       "     'issue': '5406',\n",
       "     'year': 1999,\n",
       "     'title': 'Function of WW domains as phosphoserine- or phosphothreonine-binding modules.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1325-8',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Lu PJ', 'Zhou XZ', 'Shen M', 'Lu KP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.283.5406.1325'},\n",
       "    'PUB00020561': {'PMID': 9407065,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '52',\n",
       "     'year': 1997,\n",
       "     'title': 'The WW domain of neural protein FE65 interacts with proline-rich motifs in Mena, the mammalian homolog of Drosophila enabled.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '32869-77',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Ermekova KS',\n",
       "      'Zambrano N',\n",
       "      'Linn H',\n",
       "      'Minopoli G',\n",
       "      'Gertler F',\n",
       "      'Russo T',\n",
       "      'Sudol M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.52.32869'},\n",
       "    'PUB00020562': {'PMID': 8797792,\n",
       "     'ISBN': None,\n",
       "     'volume': '384',\n",
       "     'issue': '1',\n",
       "     'year': 1996,\n",
       "     'title': 'Towards prediction of cognate complexes between the WW domain and proline-rich ligands.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1-8',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Einbond A', 'Sudol M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0014-5793(96)00263-3'}},\n",
       "   'set_info': {'accession': 'CL0680', 'name': 'WW'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2004,\n",
       "    'interactions': 0,\n",
       "    'matches': 128937,\n",
       "    'pathways': 0,\n",
       "    'proteins': 79723,\n",
       "    'proteomes': 2166,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 51004, 'rosettafold': 0},\n",
       "    'structures': 206,\n",
       "    'taxa': 8251},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 93077,\n",
       "    'alignment:full': 59696,\n",
       "    'alignment:seed': 448},\n",
       "   'cross_references': {}}},\n",
       " 'PF04205': {'metadata': {'accession': 'PF04205',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007329',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FMN-binding domain', 'short': 'FMN_bind'},\n",
       "   'description': ['<p>This conserved region includes the FMN-binding site of the NqrC protein [PMID:11248234] as well as the NosR and NirI regulatory proteins. This domain is post-translationally flavinylated that may facilitate electron transfer, and thus, resembles multiheme cytochromes [PMID:34032212].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010090': {'PMID': 11248234,\n",
       "     'ISBN': None,\n",
       "     'volume': '492',\n",
       "     'issue': '1-2',\n",
       "     'year': 2001,\n",
       "     'title': 'Expression and mutagenesis of the NqrC subunit of the NQR respiratory Na(+) pump from Vibrio cholerae with covalently attached FMN.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '45-9',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Barquera B', 'Hase CC', 'Gennis RB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(01)02224-4'},\n",
       "    'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 245,\n",
       "    'interactions': 0,\n",
       "    'matches': 42605,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39581,\n",
       "    'proteomes': 4083,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33614, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 18229},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38742,\n",
       "    'alignment:full': 8318,\n",
       "    'alignment:seed': 835},\n",
       "   'cross_references': {}}},\n",
       " 'PF04777': {'metadata': {'accession': 'PF04777',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017905',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Erv1 / Alr family', 'short': 'Evr1_Alr'},\n",
       "   'description': ['<p>Biogenesis of Fe/S clusters involves a number of essential mitochondrial  proteins. Erv1p of Saccharomyces cerevisiae mitochondria is  required for the maturation of Fe/S proteins in the cytosol. The ALR  (augmenter of liver regeneration) represents a mammalian orthologue of yeast Erv1p. Both Erv1p and full-length ALR are located in the  mitochondrial intermembrane an d it thought to operate downstream of  the mitochondrial ABC transporter [PMID:11493598].  </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008628': {'PMID': 11493598,\n",
       "     'ISBN': None,\n",
       "     'volume': '2',\n",
       "     'issue': '8',\n",
       "     'year': 2001,\n",
       "     'title': 'An essential function of the mitochondrial sulfhydryl oxidase Erv1p/ALR in the maturation of cytosolic Fe/S proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '715-20',\n",
       "     'medline_journal': 'EMBO Rep',\n",
       "     'ISO_journal': 'EMBO Rep.',\n",
       "     'authors': ['Lange H',\n",
       "      'Lisowsky T',\n",
       "      'Gerber J',\n",
       "      'Muhlenhoff U',\n",
       "      'Kispal G',\n",
       "      'Lill R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/embo-reports/kve161'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 159,\n",
       "    'interactions': 0,\n",
       "    'matches': 9516,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9484,\n",
       "    'proteomes': 2244,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7490, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 8350},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7808,\n",
       "    'alignment:full': 4215,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF01193': {'metadata': {'accession': 'PF01193',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011263',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RNA polymerase Rpb3/Rpb11 dimerisation domain',\n",
       "    'short': 'RNA_pol_L'},\n",
       "   'description': ['<p>The two eukaryotic subunits Rpb3 and Rpb11 dimerise to from a platform onto which the other subunits of the RNA polymerase assemble (D/L in archaea). The prokaryotic equivalent of the Rpb3/Rpb11 platform is the alpha-alpha dimer.  The dimerisation domain of the alpha subunit/Rpb3 is interrupted by an insert domain (Pfam:PF01000).  Some of the alpha subunits also contain iron-sulphur binding domains (Pfam:PF00037).  Rpb11 is found as a continuous domain.  Members of this family include: alpha subunit from eubacteria, alpha subunits from chloroplasts, Rpb3 subunits from eukaryotes, Rpb11 subunits from eukaryotes, RpoD subunits from archaeal spp, and RpoL subunits from archaeal spp.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001064': {'PMID': 7613089,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '2',\n",
       "     'year': 1995,\n",
       "     'title': 'The Escherichia coli RNA polymerase alpha subunit: structure and function.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '197-203',\n",
       "     'medline_journal': 'Curr Opin Genet Dev',\n",
       "     'ISO_journal': 'Curr. Opin. Genet. Dev.',\n",
       "     'authors': ['Ebright RH', 'Busby S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0959-437X(95)80008-5'},\n",
       "    'PUB00004908': {'PMID': 9050843,\n",
       "     'ISBN': None,\n",
       "     'volume': '94',\n",
       "     'issue': '5',\n",
       "     'year': 1997,\n",
       "     'title': 'The two alpha subunits of Escherichia coli RNA polymerase are asymmetrically arranged and contact different halves of the DNA upstream element.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1709-14',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Murakami K',\n",
       "      'Kimura M',\n",
       "      'Owens JT',\n",
       "      'Meares CF',\n",
       "      'Ishihama A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.94.5.1709'},\n",
       "    'PUB00005231': {'PMID': 9657722,\n",
       "     'ISBN': None,\n",
       "     'volume': '281',\n",
       "     'issue': '5374',\n",
       "     'year': 1998,\n",
       "     'title': 'Structure of the Escherichia coli RNA polymerase alpha subunit amino-terminal domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '262-6',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Zhang G', 'Darst SA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.281.5374.262'}},\n",
       "   'set_info': {'accession': 'CL0509', 'name': 'RBP11-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 211,\n",
       "    'interactions': 0,\n",
       "    'matches': 70325,\n",
       "    'pathways': 0,\n",
       "    'proteins': 70258,\n",
       "    'proteomes': 11128,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 59932, 'rosettafold': 0},\n",
       "    'structures': 760,\n",
       "    'taxa': 65697},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61487,\n",
       "    'alignment:full': 12371,\n",
       "    'alignment:seed': 380},\n",
       "   'cross_references': {}}},\n",
       " 'PF00312': {'metadata': {'accession': 'PF00312',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000589',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S15', 'short': 'Ribosomal_S15'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005302': {'PMID': 9562554,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '4',\n",
       "     'year': 1998,\n",
       "     'title': 'Conformational variability of the N-terminal helix in the structure of ribosomal protein S15.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '429-38',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Clemons WM Jr', 'Davies C', 'White SW', 'Ramakrishnan V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(98)00045-8'}},\n",
       "   'set_info': {'accession': 'CL0600', 'name': 'S15_NS1'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 163,\n",
       "    'interactions': 0,\n",
       "    'matches': 64257,\n",
       "    'pathways': 0,\n",
       "    'proteins': 64238,\n",
       "    'proteomes': 11074,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 55097, 'rosettafold': 0},\n",
       "    'structures': 1240,\n",
       "    'taxa': 63986},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 55889,\n",
       "    'alignment:full': 12015,\n",
       "    'alignment:seed': 605},\n",
       "   'cross_references': {}}},\n",
       " 'PF14804': {'metadata': {'accession': 'PF14804',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032782',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Jag N-terminus', 'short': 'Jag_N'},\n",
       "   'description': ['<p>This domain is found at the N-terminus of proteins containing Pfam:PF13083 and Pfam:PF01424, including the jag proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 29,\n",
       "    'interactions': 0,\n",
       "    'matches': 12292,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12288,\n",
       "    'proteomes': 1988,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10564, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 8739},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11172,\n",
       "    'alignment:full': 2062,\n",
       "    'alignment:seed': 310},\n",
       "   'cross_references': {}}},\n",
       " 'PF00080': {'metadata': {'accession': 'PF00080',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001424',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Copper/zinc superoxide dismutase (SODC)',\n",
       "    'short': 'Sod_Cu'},\n",
       "   'description': [\"<p>superoxide dismutases (SODs) catalyse the conversion of superoxide radicals to hydrogen peroxide and molecular oxygen.  Three evolutionarily distinct families of SODs are known, of which the copper/zinc-binding  family is one. Defects in the human SOD1 gene cause familial amyotrophic lateral sclerosis (Lou Gehrig's disease). Structure is an eight-stranded beta sandwich, similar to the immunoglobulin fold.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 231,\n",
       "    'interactions': 0,\n",
       "    'matches': 28029,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27598,\n",
       "    'proteomes': 4923,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23693, 'rosettafold': 0},\n",
       "    'structures': 250,\n",
       "    'taxa': 21826},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25320,\n",
       "    'alignment:full': 8329,\n",
       "    'alignment:seed': 398},\n",
       "   'cross_references': {}}},\n",
       " 'PF03748': {'metadata': {'accession': 'PF03748',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005503',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flagellar basal body-associated protein FliL',\n",
       "    'short': 'FliL'},\n",
       "   'description': ['<p>This FliL protein controls the rotational direction of the flagella during chemotaxis [PMID:3519573]. FliL is a cytoplasmic membrane protein associated with the basal body [PMID:10439416].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008414': {'PMID': 3519573,\n",
       "     'ISBN': None,\n",
       "     'volume': '166',\n",
       "     'issue': '3',\n",
       "     'year': 1986,\n",
       "     'title': 'Sequence of the flaA (cheC) locus of Escherichia coli and discovery of a new gene.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1007-12',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Kuo SC', 'Koshland DE Jr.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=3519573&action=stream&blobtype=pdf'},\n",
       "    'PUB00008415': {'PMID': 10439416,\n",
       "     'ISBN': None,\n",
       "     'volume': '145 ( Pt 7)',\n",
       "     'issue': None,\n",
       "     'year': 1999,\n",
       "     'title': 'FliL is a membrane-associated component of the flagellar basal body of Salmonella.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1769-75',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Schoenhals GJ', 'Macnab RM.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 27,\n",
       "    'interactions': 0,\n",
       "    'matches': 22270,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22270,\n",
       "    'proteomes': 3777,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19455, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 16297},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21220,\n",
       "    'alignment:full': 4366,\n",
       "    'alignment:seed': 441},\n",
       "   'cross_references': {}}},\n",
       " 'PF02615': {'metadata': {'accession': 'PF02615',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003767',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Malate/L-lactate dehydrogenase', 'short': 'Ldh_2'},\n",
       "   'description': ['<p>This family consists of bacterial and archaeal Malate/L-lactate dehydrogenase. L-lactate dehydrogenase, EC:1.1.1.27, catalyses  the reaction (S)-lactate + NAD(+) <=> pyruvate + NADH. Malate dehydrogenase, EC:1.1.1.37 and EC:1.1.1.82, catalyses the reactions: (S)-malate + NAD(+) <=> oxaloacetate  + NADH, and (S)-malate + NADP(+) <=> oxaloacetate + NADPH respectively.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008807': {'PMID': 2110059,\n",
       "     'ISBN': None,\n",
       "     'volume': '188',\n",
       "     'issue': '3',\n",
       "     'year': 1990,\n",
       "     'title': 'Properties and primary structure of the L-malate dehydrogenase from the extremely thermophilic archaebacterium Methanothermus fervidus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '623-32',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Honka E', 'Fabry S', 'Niermann T', 'Palm P', 'Hensel R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1432-1033.1990.tb15443.x'},\n",
       "    'PUB00019783': {'PMID': 8405966,\n",
       "     'ISBN': None,\n",
       "     'volume': '112',\n",
       "     'issue': '2',\n",
       "     'year': 1993,\n",
       "     'title': 'The Alcaligenes eutrophus ldh structural gene encodes a novel type of lactate dehydrogenase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '229-35',\n",
       "     'medline_journal': 'FEMS Microbiol Lett',\n",
       "     'ISO_journal': 'FEMS Microbiol. Lett.',\n",
       "     'authors': ['Jendrossek D', 'Kratzin HD', 'Steinbuchel A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1097(93)90169-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 93,\n",
       "    'interactions': 0,\n",
       "    'matches': 30871,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30414,\n",
       "    'proteomes': 3615,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25578, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 16529},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28289,\n",
       "    'alignment:full': 5421,\n",
       "    'alignment:seed': 322},\n",
       "   'cross_references': {}}},\n",
       " 'PF17871': {'metadata': {'accession': 'PF17871',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041546',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AAA lid domain', 'short': 'AAA_lid_9'},\n",
       "   'description': ['<p>This entry represents the alpha helical AAA+ lid domain that is found to the C-terminus of AAA domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0671', 'name': 'AAA_lid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 345,\n",
       "    'interactions': 0,\n",
       "    'matches': 114968,\n",
       "    'pathways': 0,\n",
       "    'proteins': 114932,\n",
       "    'proteomes': 9932,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 99318, 'rosettafold': 0},\n",
       "    'structures': 77,\n",
       "    'taxa': 42480},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 107683,\n",
       "    'alignment:full': 22768,\n",
       "    'alignment:seed': 176},\n",
       "   'cross_references': {}}},\n",
       " 'PF00842': {'metadata': {'accession': 'PF00842',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011079',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Alanine racemase, C-terminal domain',\n",
       "    'short': 'Ala_racemase_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000440': {'PMID': 9063881,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'Determination of the structure of alanine racemase from Bacillus stearothermophilus at 1.9-A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1329-42',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Shaw JP', 'Petsko GA', 'Ringe D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi961856c'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 75,\n",
       "    'interactions': 0,\n",
       "    'matches': 53113,\n",
       "    'pathways': 0,\n",
       "    'proteins': 53088,\n",
       "    'proteomes': 8278,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45935, 'rosettafold': 0},\n",
       "    'structures': 66,\n",
       "    'taxa': 34904},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49846,\n",
       "    'alignment:full': 9001,\n",
       "    'alignment:seed': 881},\n",
       "   'cross_references': {}}},\n",
       " 'PF10385': {'metadata': {'accession': 'PF10385',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019462',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RNA polymerase beta subunit external 1 domain',\n",
       "    'short': 'RNA_pol_Rpb2_45'},\n",
       "   'description': ['<p>RNA polymerases catalyse the DNA-dependent polymerisation of RNA. Prokaryotes contain a single RNA polymerase compared with three in eukaryotes (not including mitochondrial or chloroplast polymerases). This domain in prokaryotes spans the gap between domains 4 and 5 of the yeast protein. It is also known as the external 1 region of the polymerase and is bound in association with the external 2 region [PMID:11313498].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008731': {'PMID': 11313498,\n",
       "     'ISBN': None,\n",
       "     'volume': '292',\n",
       "     'issue': '5523',\n",
       "     'year': 2001,\n",
       "     'title': 'Structural basis of transcription: RNA polymerase II at 2.8 angstrom resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1863-76',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Cramer P', 'Bushnell DA', 'Kornberg RD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.1059493'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 100,\n",
       "    'interactions': 0,\n",
       "    'matches': 56933,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56932,\n",
       "    'proteomes': 8521,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33977, 'rosettafold': 0},\n",
       "    'structures': 429,\n",
       "    'taxa': 43497},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 53277,\n",
       "    'alignment:full': 7898,\n",
       "    'alignment:seed': 621},\n",
       "   'cross_references': {}}},\n",
       " 'PF04402': {'metadata': {'accession': 'PF04402',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007497',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF541)', 'short': 'SIMPL'},\n",
       "   'description': ['<p>Members of this family have so far been found in bacteria and mouse SwissProt or TrEMBL entries.  However possible family members have also been identified in translated rat (Genbank:AW144450) and human (Genbank:AI478629) ESTs. A mouse family member has been named SIMPL (signalling molecule that associates with mouse pelle-like kinase).  SIMPL appears to facilitate and/or regulate complex formation between IRAK/mPLK (IL-1 receptor-associated kinase) and IKK (inhibitor of kappa-B kinase) containing complexes, and thus regulate NF-kappa-B activity [PMID:11096118]. Separate experiments demonstrate that a mouse family member (named LaXp180) binds the Listeria monocytogenes surface protein ActA, which is a virulence factor that induces actin polymerisation. It may also bind stathmin, a protein involved in signal transduction and in the regulation of microtubule dynamics [PMID:11207567]. In bacteria its function is unknown, but it is thought to be located in the periplasm or outer membrane.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00054591': {'PMID': 21128561,\n",
       "     'ISBN': None,\n",
       "     'volume': '65',\n",
       "     'issue': '5',\n",
       "     'year': 2010,\n",
       "     'title': 'Spuriously high HbA1c due to the presence of haemoglobin Raleigh: a case report and review of the literature.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '336-40',\n",
       "     'medline_journal': 'Acta Clin Belg',\n",
       "     'ISO_journal': 'Acta Clin Belg',\n",
       "     'authors': ['Vandewiele A', 'Genbrugge K', 'Delanghe J.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 44,\n",
       "    'interactions': 0,\n",
       "    'matches': 34874,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34761,\n",
       "    'proteomes': 6148,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 30017, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 27396},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32023,\n",
       "    'alignment:full': 6670,\n",
       "    'alignment:seed': 576},\n",
       "   'cross_references': {}}},\n",
       " 'PF00571': {'metadata': {'accession': 'PF00571',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000644',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CBS domain', 'short': 'CBS'},\n",
       "   'description': ['<p>CBS domains are small intracellular modules that pair together to form a stable globular domain [PMID:10200156]. This family represents a single CBS domain. Pairs of these domains have been termed a Bateman domain [PMID:14722609]. CBS domains have been shown to bind ligands with an adenosyl group such as AMP, ATP and S-AdoMet [PMID:14722619].  CBS domains are found attached to a wide range of other protein domains suggesting that CBS domains may play a regulatory role making proteins sensitive to adenosyl carrying ligands. The region containing the CBS domains in Cystathionine-beta synthase is involved in regulation by S-AdoMet [PMID:11524006]. CBS domain pairs from AMPK bind AMP or ATP [PMID:14722619]. The CBS domains from IMPDH and the chloride channel CLC2 bind ATP [PMID:14722619].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000457': {'PMID': 10200156,\n",
       "     'ISBN': None,\n",
       "     'volume': '38',\n",
       "     'issue': '15',\n",
       "     'year': 1999,\n",
       "     'title': \"Characteristics and crystal structure of bacterial inosine-5'-monophosphate dehydrogenase.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '4691-700',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Zhang R',\n",
       "      'Evans G',\n",
       "      'Rotella FJ',\n",
       "      'Westbrook EM',\n",
       "      'Beno D',\n",
       "      'Huberman E',\n",
       "      'Joachimiak A',\n",
       "      'Collart FR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi982858v'},\n",
       "    'PUB00003445': {'PMID': 9106071,\n",
       "     'ISBN': None,\n",
       "     'volume': '75',\n",
       "     'issue': '3',\n",
       "     'year': 1997,\n",
       "     'title': 'CBS domains in CIC chloride channels implicated in myotonia and nephrolithiasis (kidney stones).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '160-3',\n",
       "     'medline_journal': 'J Mol Med',\n",
       "     'ISO_journal': 'J. Mol. Med.',\n",
       "     'authors': ['Ponting CP.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00005461': {'PMID': 9020585,\n",
       "     'ISBN': None,\n",
       "     'volume': '22',\n",
       "     'issue': '1',\n",
       "     'year': 1997,\n",
       "     'title': 'The structure of a domain common to archaebacteria and the homocystinuria disease protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '12-3',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(96)30046-7'},\n",
       "    'PUB00014845': {'PMID': 14722619,\n",
       "     'ISBN': None,\n",
       "     'volume': '113',\n",
       "     'issue': '2',\n",
       "     'year': 2004,\n",
       "     'title': 'CBS domains form energy-sensing modules whose binding of adenosine ligands is disrupted by disease mutations.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '274-84',\n",
       "     'medline_journal': 'J Clin Invest',\n",
       "     'ISO_journal': 'J. Clin. Invest.',\n",
       "     'authors': ['Scott JW',\n",
       "      'Hawley SA',\n",
       "      'Green KA',\n",
       "      'Anis M',\n",
       "      'Stewart G',\n",
       "      'Scullion GA',\n",
       "      'Norman DG',\n",
       "      'Hardie DG.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=14722619&action=stream&blobtype=pdf'},\n",
       "    'PUB00019266': {'PMID': 11524006,\n",
       "     'ISBN': None,\n",
       "     'volume': '40',\n",
       "     'issue': '35',\n",
       "     'year': 2001,\n",
       "     'title': 'Regulation of human cystathionine beta-synthase by S-adenosyl-L-methionine: evidence for two catalytically active conformations involving an autoinhibitory domain in the C-terminal region.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10625-33',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Janosik M',\n",
       "      'Kery V',\n",
       "      'Gaustadnes M',\n",
       "      'Maclean KN',\n",
       "      'Kraus JP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi010711p'},\n",
       "    'PUB00019267': {'PMID': 14722609,\n",
       "     'ISBN': None,\n",
       "     'volume': '113',\n",
       "     'issue': '2',\n",
       "     'year': 2004,\n",
       "     'title': 'Bateman domains and adenosine derivatives form a binding contract.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '182-4',\n",
       "     'medline_journal': 'J Clin Invest',\n",
       "     'ISO_journal': 'J. Clin. Invest.',\n",
       "     'authors': ['Kemp BE.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=14722609&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2724,\n",
       "    'interactions': 0,\n",
       "    'matches': 825779,\n",
       "    'pathways': 0,\n",
       "    'proteins': 457238,\n",
       "    'proteomes': 11260,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 387862, 'rosettafold': 0},\n",
       "    'structures': 266,\n",
       "    'taxa': 47537},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 735709,\n",
       "    'alignment:full': 185434,\n",
       "    'alignment:seed': 840},\n",
       "   'cross_references': {}}},\n",
       " 'PF01642': {'metadata': {'accession': 'PF01642',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006099',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Methylmalonyl-CoA mutase', 'short': 'MM_CoA_mutase'},\n",
       "   'description': ['<p>The enzyme methylmalonyl-CoA mutase is a member of a class of enzymes that uses coenzyme B12 (adenosylcobalamin) as a cofactor. The enzyme induces the formation of an adenosyl radical from the cofactor. This radical then initiates a free-radical rearrangement of its substrate, succinyl-CoA, to methylmalonyl-CoA [PMID:8805541].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005279': {'PMID': 8805541,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'How coenzyme B12 radicals are generated: the crystal structure of methylmalonyl-coenzyme A mutase at 2 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '339-50',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Mancia F',\n",
       "      'Keep NH',\n",
       "      'Nakagawa A',\n",
       "      'Leadlay PF',\n",
       "      'McSweeney S',\n",
       "      'Rasmussen B',\n",
       "      'Bosecke P',\n",
       "      'Diat O',\n",
       "      'Evans PR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(96)00037-8'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 84,\n",
       "    'interactions': 0,\n",
       "    'matches': 56363,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55809,\n",
       "    'proteomes': 5650,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46156, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 22181},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48972,\n",
       "    'alignment:full': 11040,\n",
       "    'alignment:seed': 271},\n",
       "   'cross_references': {}}},\n",
       " 'PF02633': {'metadata': {'accession': 'PF02633',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003785',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Creatinine amidohydrolase', 'short': 'Creatininase'},\n",
       "   'description': ['<p>Creatinine amidohydrolase (EC:3.5.2.10), or creatininase, catalyses  the hydrolysis of creatinine to creatine [PMID:7670196]. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008253': {'PMID': 7670196,\n",
       "     'ISBN': None,\n",
       "     'volume': '59',\n",
       "     'issue': '7',\n",
       "     'year': 1995,\n",
       "     'title': 'Cloning of the creatinine amidohydrolase gene from Pseudomonas sp. PS-7.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1331-2',\n",
       "     'medline_journal': 'Biosci Biotechnol Biochem',\n",
       "     'ISO_journal': 'Biosci. Biotechnol. Biochem.',\n",
       "     'authors': ['Yamamoto K', 'Oka M', 'Kikuchi T', 'Emi S.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 27957,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27859,\n",
       "    'proteomes': 3379,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23414, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 15702},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24247,\n",
       "    'alignment:full': 4769,\n",
       "    'alignment:seed': 381},\n",
       "   'cross_references': {}}},\n",
       " 'PF10396': {'metadata': {'accession': 'PF10396',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018948',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GTP-binding protein TrmE N-terminus', 'short': 'TrmE_N'},\n",
       "   'description': [\"<p>This family represents the shorter, B, chain of the homo-dimeric structure which is a guanine nucleotide-binding protein that binds and hydrolyses GTP.  TrmE is homologous to the tetrahydrofolate-binding domain of N,N-dimethylglycine oxidase and indeed binds formyl-tetrahydrofolate. TrmE actively participates in the formylation reaction of uridine and regulates the ensuing hydrogenation reaction of a Schiff's base intermediate.  This B chain is the N-terminal portion of the protein consisting of five beta-strands and three alpha helices and is necessary for mediating dimer formation within the protein [PMID:15616586].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00038393': {'PMID': 15616586,\n",
       "     'ISBN': None,\n",
       "     'volume': '24',\n",
       "     'issue': '1',\n",
       "     'year': 2005,\n",
       "     'title': 'The structure of the TrmE GTP-binding protein and its implications for tRNA modification.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '23-33',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Scrima A', 'Vetter IR', 'Armengod ME', 'Wittinghofer A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/sj.emboj.7600507'}},\n",
       "   'set_info': {'accession': 'CL0289', 'name': 'FBD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 110,\n",
       "    'interactions': 0,\n",
       "    'matches': 39343,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39337,\n",
       "    'proteomes': 8296,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33511, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 34706},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35929,\n",
       "    'alignment:full': 7887,\n",
       "    'alignment:seed': 843},\n",
       "   'cross_references': {}}},\n",
       " 'PF00309': {'metadata': {'accession': 'PF00309',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000394',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sigma-54 factor, Activator interacting domain (AID)',\n",
       "    'short': 'Sigma54_AID'},\n",
       "   'description': ['<p>The sigma-54 holoenzyme is an enhancer dependent  form of the RNA polymerase.  The AID is necessary for activator interaction [PMID:11544185].  In addition, the AID also inhibits  transcription initiation in the sigma-54 holoenzyme prior to  interaction with the activator [PMID:11544185].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020341': {'PMID': 11544185,\n",
       "     'ISBN': None,\n",
       "     'volume': '15',\n",
       "     'issue': '17',\n",
       "     'year': 2001,\n",
       "     'title': 'Binding of transcriptional activators to sigma 54 in the presence of the transition state analog ADP-aluminum fluoride: insights into activator mechanochemical action.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2282-94',\n",
       "     'medline_journal': 'Genes Dev',\n",
       "     'ISO_journal': 'Genes Dev.',\n",
       "     'authors': ['Chaney M',\n",
       "      'Grande R',\n",
       "      'Wigneshweraraj SR',\n",
       "      'Cannon W',\n",
       "      'Casaz P',\n",
       "      'Gallegos MT',\n",
       "      'Schumacher J',\n",
       "      'Jones S',\n",
       "      'Elderkin S',\n",
       "      'Dago AE',\n",
       "      'Morett E',\n",
       "      'Buck M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1101/gad.205501'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 41,\n",
       "    'interactions': 0,\n",
       "    'matches': 30030,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30030,\n",
       "    'proteomes': 5045,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25745, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 22287},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27918,\n",
       "    'alignment:full': 5014,\n",
       "    'alignment:seed': 286},\n",
       "   'cross_references': {}}},\n",
       " 'PF05226': {'metadata': {'accession': 'PF05226',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007890',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CHASE2 domain', 'short': 'CHASE2'},\n",
       "   'description': ['<p>CHASE2 is an extracellular sensory domain, which is present in various classes of transmembrane receptors that are parts of signal transduction pathways in bacteria. Specifically, CHASE2 domains are found in histidine kinases, adenylate cyclases, serine/threonine kinases and predicted diguanylate cyclases/phosphodiesterases. Environmental factors that are recognised by CHASE2 domains are not known at this time [PMID:12486065].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010429': {'PMID': 12486065,\n",
       "     'ISBN': None,\n",
       "     'volume': '185',\n",
       "     'issue': '1',\n",
       "     'year': 2003,\n",
       "     'title': 'Common extracellular sensory domains in transmembrane receptors for diverse signal transduction pathways in bacteria and archaea.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '285-94',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Zhulin IB', 'Nikolskaya AN', 'Galperin MY.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.185.1.285-294.2003'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 197,\n",
       "    'interactions': 0,\n",
       "    'matches': 24772,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24666,\n",
       "    'proteomes': 2052,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19039, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 10506},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19754,\n",
       "    'alignment:full': 3786,\n",
       "    'alignment:seed': 396},\n",
       "   'cross_references': {}}},\n",
       " 'PF01765': {'metadata': {'accession': 'PF01765',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023584',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosome recycling factor', 'short': 'RRF'},\n",
       "   'description': ['<p>The ribosome recycling factor (RRF / ribosome release factor) dissociates the ribosome from the mRNA after termination of translation, and is  essential bacterial growth [PMID:8183897]. Thus ribosomes are \"recycled\" and ready for another round of protein synthesis.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004841': {'PMID': 8183897,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '10',\n",
       "     'year': 1994,\n",
       "     'title': 'Ribosome recycling factor (ribosome releasing factor) is essential for bacterial growth.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4249-53',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Janosi L', 'Shimizu I', 'Kaji A.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8183897&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 86,\n",
       "    'interactions': 0,\n",
       "    'matches': 48203,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48144,\n",
       "    'proteomes': 10683,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41168, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 44568},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43143,\n",
       "    'alignment:full': 10112,\n",
       "    'alignment:seed': 780},\n",
       "   'cross_references': {}}},\n",
       " 'PF10397': {'metadata': {'accession': 'PF10397',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019468',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Adenylosuccinate lyase C-terminus', 'short': 'ADSL_C'},\n",
       "   'description': ['<p>This is the C-terminal seven alpha helices of the structure whose full length represents the enzyme adenylosuccinate lyase. This sequence lies C-terminal to the conserved motif necessary for beta-elimination reactions [PMID:9274883], Adenylosuccinate lyase catalyses two steps in the synthesis of purine nucleotides: the conversion of succinylaminoimidazole-carboxamide ribotide into aminoimidazole-carboxamide ribotide, the eighth step of the de novo pathway, and the formation of adenosine monophosphate (AMP) from adenylosuccinate, the second step in the conversion of inosine monophosphate into AMP [PMID:17485188].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044138': {'PMID': 9274883,\n",
       "     'ISBN': None,\n",
       "     'volume': '88',\n",
       "     'issue': '1-2',\n",
       "     'year': 1997,\n",
       "     'title': 'Characterisation of the gene encoding adenylosuccinate lyase of Plasmodium falciparum.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '237-41',\n",
       "     'medline_journal': 'Mol Biochem Parasitol',\n",
       "     'ISO_journal': 'Mol. Biochem. Parasitol.',\n",
       "     'authors': ['Marshall VM', 'Coppel RL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0166-6851(97)00054-6'},\n",
       "    'PUB00044139': {'PMID': 17485188,\n",
       "     'ISBN': None,\n",
       "     'volume': '29',\n",
       "     'issue': '9',\n",
       "     'year': 2007,\n",
       "     'title': 'Adenylosuccinate lyase deficiency: the first identified polish patient.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '600-2',\n",
       "     'medline_journal': 'Brain Dev',\n",
       "     'ISO_journal': 'Brain Dev.',\n",
       "     'authors': ['Jurkiewicz E', 'Mierzewska H', 'Kusmierska K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.braindev.2007.03.005'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 38268,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38267,\n",
       "    'proteomes': 7234,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32824, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 30571},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34953,\n",
       "    'alignment:full': 7790,\n",
       "    'alignment:seed': 683},\n",
       "   'cross_references': {}}},\n",
       " 'PF14278': {'metadata': {'accession': 'PF14278',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR039532',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transcriptional regulator C-terminal region',\n",
       "    'short': 'TetR_C_8'},\n",
       "   'description': ['<p>This domain is a tetracycline repressor, domain 2, or C-terminus.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0174', 'name': 'TetR_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 19,\n",
       "    'interactions': 0,\n",
       "    'matches': 19782,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19780,\n",
       "    'proteomes': 1700,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17934, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 6945},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19987,\n",
       "    'alignment:full': 4036,\n",
       "    'alignment:seed': 311},\n",
       "   'cross_references': {}}},\n",
       " 'PF00902': {'metadata': {'accession': 'PF00902',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002033',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sec-independent protein translocase protein (TatC)',\n",
       "    'short': 'TatC'},\n",
       "   'description': ['<p>The bacterial Tat system has a remarkable ability to transport folded proteins even enzyme complexes across the cytoplasmic membrane. It is structurally and mechanistically similar to the Delta pH-driven thylakoidal protein import pathway. A functional Tat system or Delta pH-dependent pathway requires three integral membrane proteins: TatA/Tha4, TatB/Hcf106 and TatC/cpTatC. The TatC protein is essential for the function of both pathways. It might be involved in twin-arginine signal peptide recognition, protein translocation and proton translocation. Sequence analysis predicts that TatC contains six transmembrane helices (TMHs), and experimental data confirmed that N- and C-termini of TatC or cpTatC are exposed to the cytoplasmic or stromal face of the membrane. The cytoplasmic N-terminus and the first cytoplasmic loop region of the Escherichia coli TatC protein are essential for protein export. At least two TatC molecules co-exist within each Tat translocon [PMID:12163163].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011123': {'PMID': 12163163,\n",
       "     'ISBN': None,\n",
       "     'volume': '525',\n",
       "     'issue': '1-3',\n",
       "     'year': 2002,\n",
       "     'title': 'Topology determination and functional analysis of the Escherichia coli TatC protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '65-70',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Gouffi K', 'Santini CL', 'Wu LF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(02)03069-7'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 64,\n",
       "    'interactions': 0,\n",
       "    'matches': 45001,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44516,\n",
       "    'proteomes': 7582,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38093, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 35107},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40801,\n",
       "    'alignment:full': 7956,\n",
       "    'alignment:seed': 633},\n",
       "   'cross_references': {}}},\n",
       " 'PF07804': {'metadata': {'accession': 'PF07804',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012893',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HipA-like C-terminal domain', 'short': 'HipA_C'},\n",
       "   'description': ['<p>The members of this family are similar to a region close to the C-terminus of the HipA protein expressed by various bacterial species (for example Swiss:P23874). This protein is known to be involved in high-frequency persistence to the lethal effects of inhibition of either DNA or peptidoglycan synthesis [PMID:1715862]. When expressed alone, it is toxic to bacterial  cells [PMID:1715862], but it is usually tightly associated with  HipB [PMID:8021189], and the HipA-HipB complex may be involved in autoregulation of the hip operon. The hip proteins may be involved in cell division control and may interact with  cell division genes or their products [PMID:8021189].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016369': {'PMID': 8021189,\n",
       "     'ISBN': None,\n",
       "     'volume': '176',\n",
       "     'issue': '13',\n",
       "     'year': 1994,\n",
       "     'title': 'Autoregulation of hip, an operon that affects lethality due to inhibition of peptidoglycan or DNA synthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4081-91',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Black DS', 'Irwin B', 'Moyed HS.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8021189'},\n",
       "    'PUB00016440': {'PMID': 1715862,\n",
       "     'ISBN': None,\n",
       "     'volume': '173',\n",
       "     'issue': '18',\n",
       "     'year': 1991,\n",
       "     'title': 'Structure and organization of hip, an operon that affects lethality due to inhibition of peptidoglycan or DNA synthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5732-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Black DS', 'Kelly AJ', 'Mardis MJ', 'Moyed HS.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=1715862'}},\n",
       "   'set_info': {'accession': 'CL0016', 'name': 'PKinase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 41,\n",
       "    'interactions': 0,\n",
       "    'matches': 27784,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27780,\n",
       "    'proteomes': 2968,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24429, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 14667},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26594,\n",
       "    'alignment:full': 4971,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF01625': {'metadata': {'accession': 'PF01625',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002569',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peptide methionine sulfoxide reductase', 'short': 'PMSR'},\n",
       "   'description': ['<p>This enzyme repairs damaged proteins.  Methionine sulfoxide in proteins is reduced to methionine.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004885': {'PMID': 8700890,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '5',\n",
       "     'year': 1996,\n",
       "     'title': 'Cloning the expression of a mammalian gene involved in the reduction of methionine sulfoxide residues in proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2095-9',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Moskovitz J', 'Weissbach H', 'Brot N.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.5.2095'},\n",
       "    'PUB00004900': {'PMID': 8755589,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '15',\n",
       "     'year': 1996,\n",
       "     'title': 'Peptide methionine sulfoxide reductase contributes to the maintenance of adhesins in three major pathogens.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7985-90',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Wizemann TM',\n",
       "      'Moskovitz J',\n",
       "      'Pearce BJ',\n",
       "      'Cundell D',\n",
       "      'Arvidson CG',\n",
       "      'So M',\n",
       "      'Weissbach H',\n",
       "      'Brot N',\n",
       "      'Masure HR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.15.7985'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 230,\n",
       "    'interactions': 0,\n",
       "    'matches': 67815,\n",
       "    'pathways': 0,\n",
       "    'proteins': 67614,\n",
       "    'proteomes': 9861,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 57886, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 40887},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61546,\n",
       "    'alignment:full': 14014,\n",
       "    'alignment:seed': 590},\n",
       "   'cross_references': {}}},\n",
       " 'PF06022': {'metadata': {'accession': 'PF06022',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006477',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plasmodium variant antigen protein Cir/Yir/Bir',\n",
       "    'short': 'Cir_Bir_Yir'},\n",
       "   'description': ['<p>This family consists of several Cir, Yir and Bir proteins from the Plasmodium species P.chabaudi, P.yoelii and P.berghei.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0411', 'name': 'Vir'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11,\n",
       "    'interactions': 0,\n",
       "    'matches': 6001,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5863,\n",
       "    'proteomes': 6,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 5780, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 74},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6026,\n",
       "    'alignment:full': 1000,\n",
       "    'alignment:seed': 492},\n",
       "   'cross_references': {}}},\n",
       " 'PF05184': {'metadata': {'accession': 'PF05184',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007856',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Saposin-like type B, region 1', 'short': 'SapB_1'},\n",
       "   'description': ['<p>Saposin B is a small non-enzymatic glycoprotein required for the breakdown of cerebroside sulphates (sulphatides) in lysosomes. Saposin B contains three intramolecular disulphide bridges, exists as a dimer and is remarkably heat, protease, and pH stable. The crystal structure of human saposin B reveals an unusual shell-like dimer consisting of a monolayer of alpha-helices enclosing a large hydrophobic cavity [PMID:7610480, PMID:12518053]. It is one of the most studied members of the saposin protein family and it is involved in the hydrolysis of glycolipids and glycerolipids. SapB is unique in the saposin family in that it facilitates degradation by interacting with the substrate, not the enzymes [PMID:26616259].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005742': {'PMID': 7610480,\n",
       "     'ISBN': None,\n",
       "     'volume': '20',\n",
       "     'issue': '5',\n",
       "     'year': 1995,\n",
       "     'title': 'Swaposins: circular permutations within genes encoding saposin homologues.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '179-80',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Ponting CP', 'Russell RB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)89003-9'}},\n",
       "   'set_info': {'accession': 'CL0707', 'name': 'Saposin_like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 440,\n",
       "    'interactions': 0,\n",
       "    'matches': 13186,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6704,\n",
       "    'proteomes': 1100,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 5157, 'rosettafold': 0},\n",
       "    'structures': 24,\n",
       "    'taxa': 4532},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9411,\n",
       "    'alignment:full': 5405,\n",
       "    'alignment:seed': 417},\n",
       "   'cross_references': {}}},\n",
       " 'PF02446': {'metadata': {'accession': 'PF02446',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003385',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '4-alpha-glucanotransferase', 'short': 'Glyco_hydro_77'},\n",
       "   'description': ['<p>These enzymes EC:2.4.1.25 transfer a segment of a (1,4)-alpha-D-glucan to a new 4-position in an acceptor, which may be glucose or (1,4)-alpha-D-glucan [PMID:7678257].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009514': {'PMID': 7678257,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '2',\n",
       "     'year': 1993,\n",
       "     'title': 'Disproportionating enzyme (4-alpha-glucanotransferase; EC 2.4.1.25) of potato. Purification, molecular cloning, and potential role in starch metabolism.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1391-6',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Takaha T', 'Yanase M', 'Okada S', 'Smith SM.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/268/2/1391.pdf'}},\n",
       "   'set_info': {'accession': 'CL0058', 'name': 'Glyco_hydro_tim'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 105,\n",
       "    'interactions': 0,\n",
       "    'matches': 30650,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29269,\n",
       "    'proteomes': 4587,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24389, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 20972},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28015,\n",
       "    'alignment:full': 5507,\n",
       "    'alignment:seed': 387},\n",
       "   'cross_references': {}}},\n",
       " 'PF08376': {'metadata': {'accession': 'PF08376',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013587',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nitrate and nitrite sensing', 'short': 'NIT'},\n",
       "   'description': ['<p>The nitrate- and nitrite sensing domain (NIT) is found in receptor components of signal transducing pathways in bacteria which control gene expression, cellular motility and enzyme activity in response to nitrate and nitrite concentrations.  The NIT domain is predicted to be all alpha-helical in structure [PMID:12633990].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00013947': {'PMID': 12633990,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '3',\n",
       "     'year': 2003,\n",
       "     'title': 'The NIT domain: a predicted nitrate-responsive module in bacterial sensory receptors.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '121-4',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Shu CJ', 'Ulrich LE', 'Zhulin IB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(03)00032-X'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 229,\n",
       "    'interactions': 0,\n",
       "    'matches': 16772,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16745,\n",
       "    'proteomes': 1773,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14089, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 7802},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15744,\n",
       "    'alignment:full': 4542,\n",
       "    'alignment:seed': 316},\n",
       "   'cross_references': {}}},\n",
       " 'PF04055': {'metadata': {'accession': 'PF04055',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007197',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Radical SAM superfamily', 'short': 'Radical_SAM'},\n",
       "   'description': ['<p>Radical SAM proteins catalyse diverse reactions, including unusual methylations, isomerisation, sulphur insertion, ring formation, anaerobic oxidation and protein radical formation.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010539': {'PMID': 11222759,\n",
       "     'ISBN': None,\n",
       "     'volume': '29',\n",
       "     'issue': '5',\n",
       "     'year': 2001,\n",
       "     'title': 'Radical SAM, a novel protein superfamily linking unresolved steps in familiar biosynthetic pathways with radical mechanisms: functional characterization using new analysis and information visualization methods.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1097-106',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Sofia HJ',\n",
       "      'Chen G',\n",
       "      'Hetzler BG',\n",
       "      'Reyes-Spindola JF',\n",
       "      'Miller NE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/29.5.1097'},\n",
       "    'PUB00045088': {'PMID': 17335281,\n",
       "     'ISBN': None,\n",
       "     'volume': '129',\n",
       "     'issue': '12',\n",
       "     'year': 2007,\n",
       "     'title': 'Anaerobic sulfatase-maturating enzymes: radical SAM enzymes able to catalyze in vitro sulfatase post-translational modification.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3462-3',\n",
       "     'medline_journal': 'J Am Chem Soc',\n",
       "     'ISO_journal': 'J. Am. Chem. Soc.',\n",
       "     'authors': ['Benjdia A',\n",
       "      'Leprince J',\n",
       "      'Guillot A',\n",
       "      'Vaudry H',\n",
       "      'Rabot S',\n",
       "      'Berteau O.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/ja067175e'},\n",
       "    'PUB00045089': {'PMID': 16766528,\n",
       "     'ISBN': None,\n",
       "     'volume': '281',\n",
       "     'issue': '32',\n",
       "     'year': 2006,\n",
       "     'title': 'A new type of bacterial sulfatase reveals a novel maturation pathway in prokaryotes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '22464-70',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Berteau O', 'Guillot A', 'Benjdia A', 'Rabot S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M602504200'}},\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2041,\n",
       "    'interactions': 0,\n",
       "    'matches': 748225,\n",
       "    'pathways': 0,\n",
       "    'proteins': 743603,\n",
       "    'proteomes': 11282,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 630362, 'rosettafold': 0},\n",
       "    'structures': 108,\n",
       "    'taxa': 49677},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 665595,\n",
       "    'alignment:full': 134611,\n",
       "    'alignment:seed': 491},\n",
       "   'cross_references': {}}},\n",
       " 'PF04051': {'metadata': {'accession': 'PF04051',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007194',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transport protein particle (TRAPP) component',\n",
       "    'short': 'TRAPP'},\n",
       "   'description': ['<p>TRAPP plays a key role in the targeting and/or fusion of ER-to-Golgi transport vesicles with their acceptor compartment. TRAPP is a large multimeric protein that contains at least 10 subunits.  This family contains many TRAPP family proteins.  The Bet3 subunit is one of the better characterised TRAPP proteins and has a dimeric structure [PMID:15608655] with hydrophobic channels.  The channel entrances are located on a putative membrane-interacting surface that is distinctively flat, wide and decorated with positively charged residues.  Bet3 is proposed to localise TRAPP to the Golgi [PMID:15608655].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020479': {'PMID': 9564032,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'TRAPP, a highly conserved novel complex on the cis-Golgi that mediates vesicle docking and fusion.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2494-503',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Sacher M',\n",
       "      'Jiang Y',\n",
       "      'Barrowman J',\n",
       "      'Scarpa A',\n",
       "      'Burston J',\n",
       "      'Zhang L',\n",
       "      'Schieltz D',\n",
       "      'Yates JR 3rd',\n",
       "      'Abeliovich H',\n",
       "      'Ferro-Novick S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/17.9.2494'},\n",
       "    'PUB00020480': {'PMID': 15608655,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '1',\n",
       "     'year': 2005,\n",
       "     'title': 'Crystal structure of bet3 reveals a novel mechanism for Golgi localization of tethering factor TRAPP.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '38-45',\n",
       "     'medline_journal': 'Nat Struct Mol Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Mol. Biol.',\n",
       "     'authors': ['Kim YG',\n",
       "      'Sohn EJ',\n",
       "      'Seo J',\n",
       "      'Lee KJ',\n",
       "      'Lee HS',\n",
       "      'Hwang I',\n",
       "      'Whiteway M',\n",
       "      'Sacher M',\n",
       "      'Oh BH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsmb871'}},\n",
       "   'set_info': {'accession': 'CL0210', 'name': 'HNOX-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 174,\n",
       "    'interactions': 0,\n",
       "    'matches': 11477,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11424,\n",
       "    'proteomes': 2172,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 9517, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 7905},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9530,\n",
       "    'alignment:full': 5831,\n",
       "    'alignment:seed': 300},\n",
       "   'cross_references': {}}},\n",
       " 'PF08497': {'metadata': {'accession': 'PF08497',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013704',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Radical SAM N-terminal', 'short': 'Radical_SAM_N'},\n",
       "   'description': ['<p>This domain tends to occur to the N-terminus of the Pfam:PF04055 domain in hypothetical bacterial proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 22,\n",
       "    'interactions': 0,\n",
       "    'matches': 13700,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13687,\n",
       "    'proteomes': 2237,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11729, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11161},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12811,\n",
       "    'alignment:full': 2143,\n",
       "    'alignment:seed': 272},\n",
       "   'cross_references': {}}},\n",
       " 'PF04898': {'metadata': {'accession': 'PF04898',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006982',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutamate synthase central domain',\n",
       "    'short': 'Glu_syn_central'},\n",
       "   'description': ['<p>The central domain of glutamate synthase connects the amino terminal amidotransferase domain with the FMN-binding domain and has an alpha / beta overall topology [PMID:11967268].  This domain appears to be a rudimentary form of the FMN-binding TIM barrel according to SCOP.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008698': {'PMID': 11967268,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '27',\n",
       "     'year': 2002,\n",
       "     'title': 'Structural studies on the synchronization of catalytic centers in glutamate synthase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '24579-83',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['van den Heuvel RH',\n",
       "      'Ferrari D',\n",
       "      'Bossi RT',\n",
       "      'Ravasio S',\n",
       "      'Curti B',\n",
       "      'Vanoni MA',\n",
       "      'Florencio FJ',\n",
       "      'Mattevi A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M202541200'}},\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 144,\n",
       "    'interactions': 0,\n",
       "    'matches': 39145,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39051,\n",
       "    'proteomes': 8050,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 3948, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 33500},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37037,\n",
       "    'alignment:full': 8501,\n",
       "    'alignment:seed': 482},\n",
       "   'cross_references': {}}},\n",
       " 'PF00624': {'metadata': {'accession': 'PF00624',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001389',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flocculin repeat', 'short': 'Flocculin'},\n",
       "   'description': ['<p>This short repeat is rich in serine and threonine residues.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005655': {'PMID': 7502576,\n",
       "     'ISBN': None,\n",
       "     'volume': '11',\n",
       "     'issue': '11',\n",
       "     'year': 1995,\n",
       "     'title': 'Review: the dominant flocculation genes of Saccharomyces cerevisiae constitute a new subtelomeric gene family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1001-13',\n",
       "     'medline_journal': 'Yeast',\n",
       "     'ISO_journal': 'Yeast',\n",
       "     'authors': ['Teunissen AW', 'Steensma HY.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/yea.320111102'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 109,\n",
       "    'interactions': 0,\n",
       "    'matches': 2864,\n",
       "    'pathways': 0,\n",
       "    'proteins': 354,\n",
       "    'proteomes': 25,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 235, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 114},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 3285,\n",
       "    'alignment:full': 1663,\n",
       "    'alignment:seed': 384},\n",
       "   'cross_references': {}}},\n",
       " 'PF05598': {'metadata': {'accession': 'PF05598',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008490',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transposase domain (DUF772)', 'short': 'DUF772'},\n",
       "   'description': ['<p>This presumed domain is found at the N-terminus of many proteins found in transposons.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 149,\n",
       "    'interactions': 0,\n",
       "    'matches': 38828,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38820,\n",
       "    'proteomes': 3166,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34374, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 16034},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37446,\n",
       "    'alignment:full': 7854,\n",
       "    'alignment:seed': 316},\n",
       "   'cross_references': {}}},\n",
       " 'PF01168': {'metadata': {'accession': 'PF01168',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001608',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Alanine racemase, N-terminal domain',\n",
       "    'short': 'Ala_racemase_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000440': {'PMID': 9063881,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'Determination of the structure of alanine racemase from Bacillus stearothermophilus at 1.9-A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1329-42',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Shaw JP', 'Petsko GA', 'Ringe D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi961856c'}},\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 274,\n",
       "    'interactions': 0,\n",
       "    'matches': 133579,\n",
       "    'pathways': 0,\n",
       "    'proteins': 133487,\n",
       "    'proteomes': 10603,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 115026, 'rosettafold': 0},\n",
       "    'structures': 105,\n",
       "    'taxa': 43697},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 123974,\n",
       "    'alignment:full': 25691,\n",
       "    'alignment:seed': 268},\n",
       "   'cross_references': {}}},\n",
       " 'PF02517': {'metadata': {'accession': 'PF02517',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003675',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Type II CAAX prenyl endopeptidase Rce1-like',\n",
       "    'short': 'Rce1-like'},\n",
       "   'description': ['<p>This family (also known as the ABI (abortive infection) family) contains putative IMPs and has homologues in all three domains of life, including Rce1 from S. cerevisiae [PMID:20154137]. Rce1 is a type II CAAX prenyl protease that processes all farnesylated and geranylgeranylated CAAX proteins. It is an integral membrane endoprotease localized to the endoplasmic reticulum that mediates the cleavage of the carboxyl-terminal three amino acids from CaaX proteins. It is involved in processing the Ras family of small GTPases, the gamma-subunit of heterotrimeric GTPases, nuclear lamins, and protein kinases and phosphatases [PMID:29424242]. Three residues of S. cerevisiae Rce1 -E156, H194 and H248- are critical for catalysis [PMID:16361710]. The structure of Rce1 from the archaea Methanococcus (MmRce1) suggests that this group of proteins represents a novel IMP (intramembrane protease) family, the glutamate IMPs [PMID:24291792]. There is a conserved sequence motif EExxxR.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019085': {'PMID': 11343912,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '5',\n",
       "     'year': 2001,\n",
       "     'title': 'Type II CAAX prenyl endopeptidases belong to a novel superfamily of putative membrane-bound metalloproteases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '275-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Pei J', 'Grishin NV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(01)01813-8'}},\n",
       "   'set_info': {'accession': 'CL0472', 'name': 'Peptidase_U'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 338,\n",
       "    'interactions': 0,\n",
       "    'matches': 115768,\n",
       "    'pathways': 0,\n",
       "    'proteins': 115713,\n",
       "    'proteomes': 8906,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 99246, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 37479},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 105367,\n",
       "    'alignment:full': 23313,\n",
       "    'alignment:seed': 683},\n",
       "   'cross_references': {}}},\n",
       " 'PF01196': {'metadata': {'accession': 'PF01196',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000456',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L17', 'short': 'Ribosomal_L17'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 46376,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46360,\n",
       "    'proteomes': 10638,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39782, 'rosettafold': 0},\n",
       "    'structures': 835,\n",
       "    'taxa': 44626},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41519,\n",
       "    'alignment:full': 9938,\n",
       "    'alignment:seed': 560},\n",
       "   'cross_references': {}}},\n",
       " 'PF13356': {'metadata': {'accession': 'PF13356',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025166',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Arm DNA-binding domain', 'short': 'Arm-DNA-bind_3'},\n",
       "   'description': ['<p>This DNA-binding domain is found at the N-terminus of a wide variety of phage integrase proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0081', 'name': 'MBD-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 101,\n",
       "    'interactions': 0,\n",
       "    'matches': 44337,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44335,\n",
       "    'proteomes': 2718,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39561, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 12851},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46465,\n",
       "    'alignment:full': 6872,\n",
       "    'alignment:seed': 451},\n",
       "   'cross_references': {}}},\n",
       " 'PF04003': {'metadata': {'accession': 'PF04003',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007148',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dip2/Utp12 Family', 'short': 'Utp12'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of proteins containing WD40 repeats.  These proteins are part of the U3 ribonucleoprotein the yeast protein is called Utp12 or DIP2 Swiss:Q12220 [PMID:12068309].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008496': {'PMID': 12068309,\n",
       "     'ISBN': None,\n",
       "     'volume': '417',\n",
       "     'issue': '6892',\n",
       "     'year': 2002,\n",
       "     'title': 'A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '967-70',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Dragon F',\n",
       "      'Gallagher JE',\n",
       "      'Compagnone-Post PA',\n",
       "      'Mitchell BM',\n",
       "      'Porwancher KA',\n",
       "      'Wehner KA',\n",
       "      'Wormsley S',\n",
       "      'Settlage RE',\n",
       "      'Shabanowitz J',\n",
       "      'Osheim Y',\n",
       "      'Beyer AL',\n",
       "      'Hunt DF',\n",
       "      'Baserga SJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nature00769'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 193,\n",
       "    'interactions': 0,\n",
       "    'matches': 10520,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10511,\n",
       "    'proteomes': 2123,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8484, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 7681},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8514,\n",
       "    'alignment:full': 5249,\n",
       "    'alignment:seed': 375},\n",
       "   'cross_references': {}}},\n",
       " 'PF09365': {'metadata': {'accession': 'PF09365',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012808',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Conserved hypothetical protein (DUF2461)',\n",
       "    'short': 'DUF2461'},\n",
       "   'description': ['<p>Members of this family are widely (though sparsely) distributed bacterial proteins, about 230 residues in length. All members have a motif RxxRDxRFxxx[DN]KxxY. The function of this protein family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 12622,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12518,\n",
       "    'proteomes': 2875,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10501, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11144},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10972,\n",
       "    'alignment:full': 2969,\n",
       "    'alignment:seed': 252},\n",
       "   'cross_references': {}}},\n",
       " 'PF08669': {'metadata': {'accession': 'PF08669',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013977',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycine cleavage T-protein C-terminal barrel domain',\n",
       "    'short': 'GCV_T_C'},\n",
       "   'description': ['<p>This is a family of glycine cleavage T-proteins, part of the glycine  cleavage multienzyme complex (GCV) found in bacteria and the mitochondria of eukaryotes. GCV catalyses the catabolism of glycine in eukaryotes. The T-protein is an aminomethyl transferase. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001870': {'PMID': 9047339,\n",
       "     'ISBN': None,\n",
       "     'volume': '186',\n",
       "     'issue': '1',\n",
       "     'year': 1997,\n",
       "     'title': 'Cloning, and molecular characterization of the GCV1 gene encoding the glycine cleavage T-protein from Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13-20',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['McNeil JB',\n",
       "      'Zhang F',\n",
       "      'Taylor BV',\n",
       "      'Sinclair DA',\n",
       "      'Pearlman RE',\n",
       "      'Bognar AL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0378-1119(96)00670-1'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 254,\n",
       "    'interactions': 0,\n",
       "    'matches': 83587,\n",
       "    'pathways': 0,\n",
       "    'proteins': 83577,\n",
       "    'proteomes': 8995,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 69660, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 36174},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 75237,\n",
       "    'alignment:full': 17109,\n",
       "    'alignment:seed': 260},\n",
       "   'cross_references': {}}},\n",
       " 'PF13335': {'metadata': {'accession': 'PF13335',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025158',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Magnesium chelatase, subunit ChlI C-terminal',\n",
       "    'short': 'Mg_chelatase_C'},\n",
       "   'description': ['<p>This is a family of the C-terminal of putative bacterial magnesium chelatase subunit ChlI proteins. Most members have the associated Pfam:PF01078.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0671', 'name': 'AAA_lid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 39,\n",
       "    'interactions': 0,\n",
       "    'matches': 38905,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38905,\n",
       "    'proteomes': 7285,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33223, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 31309},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35271,\n",
       "    'alignment:full': 6843,\n",
       "    'alignment:seed': 598},\n",
       "   'cross_references': {}}},\n",
       " 'PF13280': {'metadata': {'accession': 'PF13280',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026881',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'WYL domain', 'short': 'WYL'},\n",
       "   'description': ['<p>WYL is a Sm-like SH3 beta-barrel fold containing domain. It is a member of the WYL-like superfamily, named for three  conserved amino acids found in a subset of the superfamily.  However, these residues are not strongly conserved throughout  the family. Rather, the conservation pattern includes four basic  residues and a position often occupied by a cysteine [PMID:24817877], which  are predicted to line a ligand-binding groove typical of the  Sm-like SH3 beta-barrels.  A WYL domain protein (sll7009) is a negative regulator of the  I-D CRISPR-Cas system in Synechocystis sp [PMID:23535141].  It is predicted  to be a ligand-sensing domain that could bind negatively charged  ligands, such as nucleotides or nucleic acid fragments, to  regulate CRISPR-Cas and other defense systems such as the  abortive infection AbiG system.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0654', 'name': 'WYL'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 159,\n",
       "    'interactions': 0,\n",
       "    'matches': 82303,\n",
       "    'pathways': 0,\n",
       "    'proteins': 81545,\n",
       "    'proteomes': 6553,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 71509, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 27405},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 77638,\n",
       "    'alignment:full': 18434,\n",
       "    'alignment:seed': 384},\n",
       "   'cross_references': {}}},\n",
       " 'PF00189': {'metadata': {'accession': 'PF00189',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001351',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S3, C-terminal domain',\n",
       "    'short': 'Ribosomal_S3_C'},\n",
       "   'description': ['<p>This family contains a central domain Pfam:PF00013, hence the amino and carboxyl terminal domains are stored separately. This is a minimal carboxyl-terminal domain.      Some are much longer.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005188': {'PMID': 8036511,\n",
       "     'ISBN': None,\n",
       "     'volume': '265',\n",
       "     'issue': '5172',\n",
       "     'year': 1994,\n",
       "     'title': 'Conserved structures and diversity of functions of RNA-binding proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '615-21',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Burd CG', 'Dreyfuss G.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/265/5172/615'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 139,\n",
       "    'interactions': 0,\n",
       "    'matches': 67681,\n",
       "    'pathways': 0,\n",
       "    'proteins': 67672,\n",
       "    'proteomes': 11014,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 58250, 'rosettafold': 0},\n",
       "    'structures': 1110,\n",
       "    'taxa': 69092},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 59602,\n",
       "    'alignment:full': 10741,\n",
       "    'alignment:seed': 231},\n",
       "   'cross_references': {}}},\n",
       " 'PF01957': {'metadata': {'accession': 'PF01957',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002810',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NfeD-like C-terminal, partner-binding', 'short': 'NfeD'},\n",
       "   'description': [\"<p>NfeD-like proteins are widely distributed throughout prokaryotes and are frequently associated with genes encoding stomatin-like proteins (slipins). There appear to be three major groups: an ancestral group with only an N-terminal serine protease domain and this C-terminal beta sheet-rich domain which is structurally very similar to the OB-fold domain, associated with its neighbouring slipin cluster; a second major group with an additional middle, membrane-spanning domain, associated in some species with eoslipin and in others with yqfA; a final 'artificial' group which unites truncated forms lacking the protease region and associated with their ancestral gene partner, either yqfA or eoslipin. This NefD, C-terminal, domain appears to be the major one for relating to the associated protein. NfeD homologues are clearly reliant on their conserved gene neighbour which is assumed to be necessary for function, either through direct physical interaction or by functioning in the same pathway, possibly involve with lipid-rafts [PMID:20012272].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011883': {'PMID': 10830257,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '6',\n",
       "     'year': 2000,\n",
       "     'title': 'Sinorhizobium meliloti nfe (nodulation formation efficiency) genes exhibit temporal and spatial expression patterns similar to those of genes involved in symbiotic nitrogen fixation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '583-91',\n",
       "     'medline_journal': 'Mol Plant Microbe Interact',\n",
       "     'ISO_journal': 'Mol. Plant Microbe Interact.',\n",
       "     'authors': ['Garcia-Rodriguez FM', 'Toro N.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 41194,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41181,\n",
       "    'proteomes': 6444,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 35034, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 27566},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37077,\n",
       "    'alignment:full': 7215,\n",
       "    'alignment:seed': 698},\n",
       "   'cross_references': {}}},\n",
       " 'PF00006': {'metadata': {'accession': 'PF00006',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000194',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase alpha/beta family, nucleotide-binding domain',\n",
       "    'short': 'ATP-synt_ab'},\n",
       "   'description': ['<p>This entry includes the ATP synthase alpha and beta subunits, the ATP synthase associated with flagella and the termination factor Rho.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004187': {'PMID': 8065448,\n",
       "     'ISBN': None,\n",
       "     'volume': '370',\n",
       "     'issue': '6491',\n",
       "     'year': 1994,\n",
       "     'title': 'Structure at 2.8 A resolution of F1-ATPase from bovine heart mitochondria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '621-8',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Abrahams JP', 'Leslie AG', 'Lutter R', 'Walker JE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/370621a0'},\n",
       "    'PUB00011716': {'PMID': 9261073,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'The crystal structure of the nucleotide-free alpha 3 beta 3 subcomplex of F1-ATPase from the thermophilic Bacillus PS3 is a symmetric trimer.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '825-36',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Shirakihara Y',\n",
       "      'Leslie AG',\n",
       "      'Abrahams JP',\n",
       "      'Walker JE',\n",
       "      'Ueda T',\n",
       "      'Sekimoto Y',\n",
       "      'Kambara M',\n",
       "      'Saika K',\n",
       "      'Kagawa Y',\n",
       "      'Yoshida M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00236-0'}},\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 550,\n",
       "    'interactions': 0,\n",
       "    'matches': 249245,\n",
       "    'pathways': 0,\n",
       "    'proteins': 248472,\n",
       "    'proteomes': 11258,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 213293, 'rosettafold': 0},\n",
       "    'structures': 490,\n",
       "    'taxa': 85743},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 224269,\n",
       "    'alignment:full': 42395,\n",
       "    'alignment:seed': 409},\n",
       "   'cross_references': {}}},\n",
       " 'PF00346': {'metadata': {'accession': 'PF00346',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001135',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Respiratory-chain NADH dehydrogenase, 49 Kd subunit',\n",
       "    'short': 'Complex1_49kDa'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 172,\n",
       "    'interactions': 0,\n",
       "    'matches': 78446,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65798,\n",
       "    'proteomes': 7862,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 55758, 'rosettafold': 0},\n",
       "    'structures': 191,\n",
       "    'taxa': 50750},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 69186,\n",
       "    'alignment:full': 11706,\n",
       "    'alignment:seed': 250},\n",
       "   'cross_references': {}}},\n",
       " 'PF04066': {'metadata': {'accession': 'PF04066',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007208',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Multiple resistance and pH regulation protein F (MrpF / PhaF)',\n",
       "    'short': 'MrpF_PhaF'},\n",
       "   'description': ['<p>Members of the PhaF / MrpF family are predicted to be an integral membrane proteins with three transmembrane regions, involved in regulation of pH. PhaF is part of a potassium efflux system involved in pH regulation.                          It is also involved in symbiosis in Rhizobium meliloti [PMID:9680201].  MrpF is part of a Na+/H+ antiporter complex, also involved in pH homeostasis.    MrpF is thought to be an efflux system for Na+ and cholate [PMID:10198001]. The Mrp system in Bacilli may also have primary energisation capacities [PMID:11356194].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009818': {'PMID': 10198001,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '8',\n",
       "     'year': 1999,\n",
       "     'title': 'mrp, a multigene, multifunctional locus in Bacillus subtilis with roles in resistance to cholate and to Na+ and in pH homeostasis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2394-402',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Ito M', 'Guffanti AA', 'Oudega B', 'Krulwich TA.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=10198001'},\n",
       "    'PUB00009819': {'PMID': 11356194,\n",
       "     'ISBN': None,\n",
       "     'volume': '496',\n",
       "     'issue': '2-3',\n",
       "     'year': 2001,\n",
       "     'title': 'Mrp-dependent Na(+)/H(+) antiporters of Bacillus exhibit characteristics that are unanticipated for completely secondary active transporters.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '117-20',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Ito M', 'Guffanti AA', 'Krulwich TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(01)02417-6'},\n",
       "    'PUB00010603': {'PMID': 9680201,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'The pha gene cluster of Rhizobium meliloti involved in pH adaptation and symbiosis encodes a novel type of K+ efflux system.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1091-101',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Putnoky P',\n",
       "      'Kereszt A',\n",
       "      'Nakamura T',\n",
       "      'Endre G',\n",
       "      'Grosskopf E',\n",
       "      'Kiss P',\n",
       "      'Kondorosi A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1998.00868.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 20,\n",
       "    'interactions': 0,\n",
       "    'matches': 19053,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19051,\n",
       "    'proteomes': 3832,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16758, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 15981},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17738,\n",
       "    'alignment:full': 4369,\n",
       "    'alignment:seed': 299},\n",
       "   'cross_references': {}}},\n",
       " 'PF02367': {'metadata': {'accession': 'PF02367',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003442',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Threonylcarbamoyl adenosine biosynthesis protein TsaE',\n",
       "    'short': 'TsaE'},\n",
       "   'description': ['<p>This family of proteins is involved in the synthesis of threonylcarbamoyl adenosine (t(6)A) [1-2].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 78,\n",
       "    'interactions': 0,\n",
       "    'matches': 44834,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44746,\n",
       "    'proteomes': 8653,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 38298, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 36853},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40558,\n",
       "    'alignment:full': 7977,\n",
       "    'alignment:seed': 622},\n",
       "   'cross_references': {}}},\n",
       " 'PF02502': {'metadata': {'accession': 'PF02502',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003500',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribose/Galactose Isomerase', 'short': 'LacAB_rpiB'},\n",
       "   'description': ['<p>This family of proteins contains the sugar isomerase enzymes ribose  5-phosphate isomerase B (rpiB), galactose isomerase subunit A (LacA) and galactose isomerase subunit B (LacB). </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002286': {'PMID': 8576032,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '4',\n",
       "     'year': 1996,\n",
       "     'title': 'Ribose catabolism of Escherichia coli: characterization of the rpiB gene encoding ribose phosphate isomerase B and of the rpiR gene, which is involved in regulation of rpiB expression.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1003-11',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Sorensen KI', 'Hove-Jensen B.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8576032'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 106,\n",
       "    'interactions': 0,\n",
       "    'matches': 42795,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42660,\n",
       "    'proteomes': 6724,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36629, 'rosettafold': 0},\n",
       "    'structures': 33,\n",
       "    'taxa': 28528},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38817,\n",
       "    'alignment:full': 8198,\n",
       "    'alignment:seed': 683},\n",
       "   'cross_references': {}}},\n",
       " 'PF07927': {'metadata': {'accession': 'PF07927',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012933',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HicA toxin of bacterial toxin-antitoxin,',\n",
       "    'short': 'HicA_toxin'},\n",
       "   'description': ['<p>HicA_toxin is a bacterial family of toxins that act as mRNA interferases. The antitoxin that neutralises this is family HicB, Pfam:PF15919 [PMID:16895922, PMID:19060138, PMID:21927020].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 32,\n",
       "    'interactions': 0,\n",
       "    'matches': 30874,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30872,\n",
       "    'proteomes': 2315,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25167, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 15019},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25864,\n",
       "    'alignment:full': 3848,\n",
       "    'alignment:seed': 373},\n",
       "   'cross_references': {}}},\n",
       " 'PF04675': {'metadata': {'accession': 'PF04675',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012308',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DNA ligase N terminus', 'short': 'DNA_ligase_A_N'},\n",
       "   'description': ['<p>This region is found in many but not all ATP-dependent DNA ligase enzymes (EC:6.5.1.1).  It is thought to be involved in DNA binding and in catalysis. In human DNA ligase I (Swiss:P18858), and in Saccharomyces cerevisiae (Swiss:P04819), this region was necessary for catalysis, and separated from the amino terminus by targeting elements. In vaccinia virus (Swiss:P16272) this region was not essential for catalysis, but deletion decreases the affinity for nicked DNA and decreased the rate of strand joining at a step subsequent to enzyme-adenylate formation [PMID:9016621].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016293': {'PMID': 9016621,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '4',\n",
       "     'year': 1997,\n",
       "     'title': 'Domain structure of vaccinia DNA ligase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '727-34',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Sekiguchi J', 'Shuman S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/25.4.727'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 306,\n",
       "    'interactions': 0,\n",
       "    'matches': 27392,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27255,\n",
       "    'proteomes': 5034,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22255, 'rosettafold': 0},\n",
       "    'structures': 42,\n",
       "    'taxa': 21149},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23600,\n",
       "    'alignment:full': 8486,\n",
       "    'alignment:seed': 406},\n",
       "   'cross_references': {}}},\n",
       " 'PF13368': {'metadata': {'accession': 'PF13368',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025589',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Topoisomerase C-terminal repeat',\n",
       "    'short': 'Toprim_C_rpt'},\n",
       "   'description': ['<p>This domain is repeated up to five times to form the C-terminal region of bacterial topoisomerase immediately downstream of the zinc-finger motif.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 121,\n",
       "    'interactions': 0,\n",
       "    'matches': 64763,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20433,\n",
       "    'proteomes': 4765,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16974, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 17984},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57580,\n",
       "    'alignment:full': 14232,\n",
       "    'alignment:seed': 303},\n",
       "   'cross_references': {}}},\n",
       " 'PF18759': {'metadata': {'accession': 'PF18759',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041078',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plavaka transposase', 'short': 'Plavaka'},\n",
       "   'description': ['<p>A transposase with an RNaseH catalytic domain that often has a histone binding BAM/BAH domain at the C-terminus and is sometimes associated with TET/JBP family of dioxygenases in fungi [PMID:24398522].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0219', 'name': 'RNase_H'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 117,\n",
       "    'interactions': 0,\n",
       "    'matches': 8986,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8440,\n",
       "    'proteomes': 178,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 6376, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 780},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7540,\n",
       "    'alignment:full': 6577,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF14841': {'metadata': {'accession': 'PF14841',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032779',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FliG middle domain', 'short': 'FliG_M'},\n",
       "   'description': ['<p>This is the middle domain of the flagellar rotor protein FliG [1-2].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0436', 'name': 'FliG'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 56,\n",
       "    'interactions': 0,\n",
       "    'matches': 21376,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21374,\n",
       "    'proteomes': 4173,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18468, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 17883},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19981,\n",
       "    'alignment:full': 4280,\n",
       "    'alignment:seed': 264},\n",
       "   'cross_references': {}}},\n",
       " 'PF02777': {'metadata': {'accession': 'PF02777',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019832',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Iron/manganese superoxide dismutases, C-terminal domain',\n",
       "    'short': 'Sod_Fe_C'},\n",
       "   'description': ['<p>superoxide dismutases (SODs) catalyse the conversion of superoxide radicals to hydrogen peroxide and molecular oxygen. Three evolutionarily distinct families of SODs are known, of which the Mn/Fe-binding  family is one. In humans, there is a cytoplasmic Cu/Zn SOD, and a mitochondrial Mn/Fe SOD. C-terminal domain is a mixed alpha/beta fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020354': {'PMID': 9878438,\n",
       "     'ISBN': None,\n",
       "     'volume': '285',\n",
       "     'issue': '2',\n",
       "     'year': 1999,\n",
       "     'title': 'Refined crystal structure of a superoxide dismutase from the hyperthermophilic archaeon Sulfolobus acidocaldarius at 2.2 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '689-702',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Knapp S',\n",
       "      'Kardinahl S',\n",
       "      'Hellgren N',\n",
       "      'Tibbelin G',\n",
       "      'Schafer G',\n",
       "      'Ladenstein R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1998.2344'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 157,\n",
       "    'interactions': 0,\n",
       "    'matches': 59177,\n",
       "    'pathways': 0,\n",
       "    'proteins': 58561,\n",
       "    'proteomes': 9444,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 51063, 'rosettafold': 0},\n",
       "    'structures': 184,\n",
       "    'taxa': 41055},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54544,\n",
       "    'alignment:full': 13350,\n",
       "    'alignment:seed': 829},\n",
       "   'cross_references': {}}},\n",
       " 'PF01923': {'metadata': {'accession': 'PF01923',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR016030',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cobalamin adenosyltransferase',\n",
       "    'short': 'Cob_adeno_trans'},\n",
       "   'description': ['<p>Cobalamin adenosyltransferase This family contains the gene products of PduO and EutT which are both cobalamin adenosyltransferases. PduO is a protein with ATP:cob(I)alamin adenosyltransferase activity. The main role of this protein is the conversion of inactive cobalamins to AdoCbl for 1,2-propanediol degradation [PMID:11160088].The EutT enzyme appears to be an adenosyl transferase, converting CNB12 to AdoB12 [PMID:10464203].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009955': {'PMID': 10464203,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '17',\n",
       "     'year': 1999,\n",
       "     'title': 'The 17-gene ethanolamine (eut) operon of Salmonella typhimurium encodes five homologues of carboxysome shell proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5317-29',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Kofoid E', 'Rappleye C', 'Stojiljkovic I', 'Roth J.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=10464203'},\n",
       "    'PUB00013593': {'PMID': 11160088,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '5',\n",
       "     'year': 2001,\n",
       "     'title': 'Functional genomic, biochemical, and genetic characterization of the Salmonella pduO gene, an ATP:cob(I)alamin adenosyltransferase gene.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1577-84',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Johnson CL',\n",
       "      'Pechonick E',\n",
       "      'Park SD',\n",
       "      'Havemann GD',\n",
       "      'Leal NA',\n",
       "      'Bobik TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.5.1577-1584.2001'}},\n",
       "   'set_info': {'accession': 'CL0601', 'name': 'Cob_adeno_trans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 63,\n",
       "    'interactions': 0,\n",
       "    'matches': 34837,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34797,\n",
       "    'proteomes': 6766,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 29707, 'rosettafold': 0},\n",
       "    'structures': 35,\n",
       "    'taxa': 28162},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31856,\n",
       "    'alignment:full': 6591,\n",
       "    'alignment:seed': 506},\n",
       "   'cross_references': {}}},\n",
       " 'PF00276': {'metadata': {'accession': 'PF00276',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013025',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L23', 'short': 'Ribosomal_L23'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 168,\n",
       "    'interactions': 0,\n",
       "    'matches': 64810,\n",
       "    'pathways': 0,\n",
       "    'proteins': 64801,\n",
       "    'proteomes': 11105,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 55575, 'rosettafold': 0},\n",
       "    'structures': 1158,\n",
       "    'taxa': 64843},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 56374,\n",
       "    'alignment:full': 12725,\n",
       "    'alignment:seed': 592},\n",
       "   'cross_references': {}}},\n",
       " 'PF01832': {'metadata': {'accession': 'PF01832',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002901',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase',\n",
       "    'short': 'Glucosaminidase'},\n",
       "   'description': ['<p>This family includes Mannosyl-glycoprotein endo-beta-N-acetylglucosaminidase EC:3.2.1.96. As well as the flageller protein J Swiss:P75942 that has been shown to hydrolyse peptidoglycan [PMID:10049388].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005848': {'PMID': 10049388,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '5',\n",
       "     'year': 1999,\n",
       "     'title': 'Peptidoglycan-hydrolyzing activity of the FlgJ protein, essential for flagellar rod formation in Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1555-61',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Nambu T', 'Minamino T', 'Macnab RM', 'Kutsukake K.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=10049388'}},\n",
       "   'set_info': {'accession': 'CL0037', 'name': 'Lysozyme'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 941,\n",
       "    'interactions': 0,\n",
       "    'matches': 33583,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33555,\n",
       "    'proteomes': 4013,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28854, 'rosettafold': 0},\n",
       "    'structures': 24,\n",
       "    'taxa': 18504},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33529,\n",
       "    'alignment:full': 5902,\n",
       "    'alignment:seed': 539},\n",
       "   'cross_references': {}}},\n",
       " 'PF01484': {'metadata': {'accession': 'PF01484',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002486',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nematode cuticle collagen N-terminal domain',\n",
       "    'short': 'Col_cuticle_N'},\n",
       "   'description': ['<p>The function of this domain is unknown. It is found in the N-terminal region of nematode cuticle collagens, see Pfam:PF01391. Cuticle is a tough elastic structure secreted by hypodermal cells and is primarily composed of collagen proteins [PMID:7828882].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001774': {'PMID': 2753356,\n",
       "     'ISBN': None,\n",
       "     'volume': '76',\n",
       "     'issue': '2',\n",
       "     'year': 1989,\n",
       "     'title': 'Sequence comparisons of developmentally regulated collagen genes of Caenorhabditis elegans.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '331-44',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Cox GN',\n",
       "      'Fields C',\n",
       "      'Kramer JM',\n",
       "      'Rosenzweig B',\n",
       "      'Hirsh D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(89)90173-X'},\n",
       "    'PUB00001853': {'PMID': 7828882,\n",
       "     'ISBN': None,\n",
       "     'volume': '151',\n",
       "     'issue': '1-2',\n",
       "     'year': 1994,\n",
       "     'title': 'Identification and analysis of a cuticular collagen-encoding gene from the plant-parasitic nematode Meloidogyne incognita.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '237-42',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Van der Eycken W',\n",
       "      'de Almeida Engler J',\n",
       "      'Van Montagu M',\n",
       "      'Gheysen G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(94)90663-7'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 184,\n",
       "    'interactions': 0,\n",
       "    'matches': 8922,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8765,\n",
       "    'proteomes': 72,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6677, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 345},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7296,\n",
       "    'alignment:full': 4681,\n",
       "    'alignment:seed': 243},\n",
       "   'cross_references': {}}},\n",
       " 'PF01043': {'metadata': {'accession': 'PF01043',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011130',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SecA preprotein cross-linking domain',\n",
       "    'short': 'SecA_PP_bind'},\n",
       "   'description': ['<p>The SecA ATPase is involved in the insertion and retraction of preproteins through the plasma membrane.  This domain has been found to cross-link to preproteins, thought to  indicate a role in  preprotein binding.  The pre-protein cross-linking domain is comprised of two sub domains that are inserted within the ATPase domain [PMID:12242434].  </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014329': {'PMID': 12242434,\n",
       "     'ISBN': None,\n",
       "     'volume': '297',\n",
       "     'issue': '5589',\n",
       "     'year': 2002,\n",
       "     'title': 'Nucleotide control of interdomain interactions in the conformational reaction cycle of SecA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2018-26',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Hunt JF',\n",
       "      'Weinkauf S',\n",
       "      'Henry L',\n",
       "      'Fak JJ',\n",
       "      'McNicholas P',\n",
       "      'Oliver DB',\n",
       "      'Deisenhofer J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.1074424'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 157,\n",
       "    'interactions': 0,\n",
       "    'matches': 53736,\n",
       "    'pathways': 0,\n",
       "    'proteins': 53719,\n",
       "    'proteomes': 8890,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45549, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 39108},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48984,\n",
       "    'alignment:full': 9700,\n",
       "    'alignment:seed': 541},\n",
       "   'cross_references': {}}},\n",
       " 'PF03186': {'metadata': {'accession': 'PF03186',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004485',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CobD/Cbib protein', 'short': 'CobD_Cbib'},\n",
       "   'description': ['<p>This family includes CobD proteins from a number of bacteria, in Salmonella this protein is called Cbib. Salmonella CobD is a different protein [PMID:7635831]. This protein is involved in cobalamin biosynthesis and is probably an enzyme responsible for the conversion of adenosylcobyric acid to adenosylcobinamide or adenosylcobinamide phosphate [PMID:7635831].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002275': {'PMID': 7635831,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '15',\n",
       "     'year': 1995,\n",
       "     'title': 'Identification and sequence analysis of genes involved in late steps in cobalamin (vitamin B12) synthesis in Rhodobacter capsulatus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4481-7',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Pollich M', 'Klug G.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/177/15/4481'},\n",
       "    'PUB00007899': {'PMID': 9446573,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '5',\n",
       "     'year': 1998,\n",
       "     'title': 'CobD, a novel enzyme with L-threonine-O-3-phosphate decarboxylase activity, is responsible for the synthesis of (R)-1-amino-2-propanol O-2-phosphate, a proposed new intermediate in cobalamin biosynthesis in Salmonella typhimurium LT2.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2684-91',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Brushaber KR', \"O'Toole GA\", 'Escalante-Semerena JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.5.2684'}},\n",
       "   'set_info': {'accession': 'CL0685', 'name': 'AmpE_CobD-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 23976,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23904,\n",
       "    'proteomes': 4827,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21069, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 20758},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22402,\n",
       "    'alignment:full': 4898,\n",
       "    'alignment:seed': 467},\n",
       "   'cross_references': {}}},\n",
       " 'PF05485': {'metadata': {'accession': 'PF05485',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006612',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'THAP domain', 'short': 'THAP'},\n",
       "   'description': ['<p>The THAP domain is a putative DNA-binding domain (DBD) and probably also binds a zinc ion. It features the conserved C2CH architecture (consensus sequence: Cys - 2-4 residues - Cys - 35-50 residues - Cys - 2 residues - His). Other universal features include the location of the domain at the N-termini of proteins, its size of about 90 residues, a C-terminal AVPTIF box and several other conserved residues. Orthologues of the human THAP domain have been identified in other vertebrates and probably worms and flies, but not in other eukaryotes or any prokaryotes [PMID:12575992].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016961': {'PMID': 12575992,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '2',\n",
       "     'year': 2003,\n",
       "     'title': 'The THAP domain: a novel protein motif with similarity to the DNA-binding domain of P element transposase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '66-9',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Roussigne M',\n",
       "      'Kossida S',\n",
       "      'Lavigne AC',\n",
       "      'Clouaire T',\n",
       "      'Ecochard V',\n",
       "      'Glories A',\n",
       "      'Amalric F',\n",
       "      'Girard JP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(02)00013-0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 860,\n",
       "    'interactions': 0,\n",
       "    'matches': 24343,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22505,\n",
       "    'proteomes': 884,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16526, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 3633},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19507,\n",
       "    'alignment:full': 11362,\n",
       "    'alignment:seed': 457},\n",
       "   'cross_references': {}}},\n",
       " 'PF02771': {'metadata': {'accession': 'PF02771',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013786',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acyl-CoA dehydrogenase, N-terminal domain',\n",
       "    'short': 'Acyl-CoA_dh_N'},\n",
       "   'description': ['<p>The N-terminal domain of Acyl-CoA dehydrogenase is an all-alpha  domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000414': {'PMID': 7857927,\n",
       "     'ISBN': None,\n",
       "     'volume': '34',\n",
       "     'issue': '7',\n",
       "     'year': 1995,\n",
       "     'title': 'Three-dimensional structure of butyryl-CoA dehydrogenase from Megasphaera elsdenii.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2163-71',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Djordjevic S', 'Pace CP', 'Stankovich MT', 'Kim JJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00007a009'},\n",
       "    'PUB00007933': {'PMID': 8356049,\n",
       "     'ISBN': None,\n",
       "     'volume': '90',\n",
       "     'issue': '16',\n",
       "     'year': 1993,\n",
       "     'title': 'Crystal structures of medium-chain acyl-CoA dehydrogenase from pig liver mitochondria with and without substrate.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7523-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Kim JJ', 'Wang M', 'Paschke R.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8356049&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0544', 'name': 'AcylCoA_ox_dh_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 951,\n",
       "    'interactions': 0,\n",
       "    'matches': 419054,\n",
       "    'pathways': 0,\n",
       "    'proteins': 414974,\n",
       "    'proteomes': 9359,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 355362, 'rosettafold': 0},\n",
       "    'structures': 120,\n",
       "    'taxa': 38415},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 389102,\n",
       "    'alignment:full': 89239,\n",
       "    'alignment:seed': 666},\n",
       "   'cross_references': {}}},\n",
       " 'PF01195': {'metadata': {'accession': 'PF01195',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001328',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peptidyl-tRNA hydrolase', 'short': 'Pept_tRNA_hydro'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001306': {'PMID': 9303320,\n",
       "     'ISBN': None,\n",
       "     'volume': '16',\n",
       "     'issue': '15',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure at 1.2 A resolution and active site mapping of Escherichia coli peptidyl-tRNA hydrolase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4760-9',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Schmitt E',\n",
       "      'Mechulam Y',\n",
       "      'Fromant M',\n",
       "      'Plateau P',\n",
       "      'Blanquet S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/16.15.4760'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 114,\n",
       "    'interactions': 0,\n",
       "    'matches': 50042,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49872,\n",
       "    'proteomes': 10249,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 42674, 'rosettafold': 0},\n",
       "    'structures': 76,\n",
       "    'taxa': 42366},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45158,\n",
       "    'alignment:full': 10245,\n",
       "    'alignment:seed': 836},\n",
       "   'cross_references': {}}},\n",
       " 'PF14226': {'metadata': {'accession': 'PF14226',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026992',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'non-haem dioxygenase in morphine synthesis N-terminal',\n",
       "    'short': 'DIOX_N'},\n",
       "   'description': ['<p>This is the highly conserved N-terminal region of proteins with 2-oxoglutarate/Fe(II)-dependent dioxygenase activity.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 653,\n",
       "    'interactions': 0,\n",
       "    'matches': 74006,\n",
       "    'pathways': 0,\n",
       "    'proteins': 73376,\n",
       "    'proteomes': 4248,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 63063, 'rosettafold': 0},\n",
       "    'structures': 126,\n",
       "    'taxa': 18614},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 65279,\n",
       "    'alignment:full': 34382,\n",
       "    'alignment:seed': 322},\n",
       "   'cross_references': {}}},\n",
       " 'PF06580': {'metadata': {'accession': 'PF06580',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010559',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Histidine kinase', 'short': 'His_kinase'},\n",
       "   'description': ['<p>This family represents a region within bacterial histidine kinase enzymes. Two-component signal transduction systems such as those mediated by histidine kinase are integral parts of bacterial cellular regulatory processes, and are used to regulate the expression of genes involved in virulence [PMID:12462127]. Members of this family often contain Pfam:PF02518 and/or Pfam:PF00672.</p>'],\n",
       "   'wikipedia': {'title': 'Two-component_regulatory_system',\n",
       "    'extract': '<p>In the field of molecular biology, a <b>two-component regulatory system</b> serves as a basic stimulus-response coupling mechanism to allow organisms to sense and respond to changes in many different environmental conditions. Two-component systems typically consist of a membrane-bound histidine kinase that senses a specific environmental stimulus and a corresponding response regulator that mediates the cellular response, mostly through differential expression of target genes. Although two-component signaling systems are found in all domains of life, they are most common by far in bacteria, particularly in Gram-negative and cyanobacteria; both histidine kinases and response regulators are among the largest gene families in bacteria. They are much less common in archaea and eukaryotes; although they do appear in yeasts, filamentous fungi, and slime molds, and are common in plants, two-component systems have been described as \"conspicuously absent\" from animals.</p>',\n",
       "    'thumbnail': '/9j/4AAQSkZJRgABAQEASABIAAD//gBGRmlsZSBzb3VyY2U6IGh0dHA6Ly9jb21tb25zLndpa2ltZWRpYS5vcmcvd2lraS9GaWxlOlBEQl8xam95X0VCSS5qcGf/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCADwAUADASIAAhEBAxEB/8QAHAABAAIDAQEBAAAAAAAAAAAAAAYHBAUIAwIB/8QAOhAAAQMDAwMDAwMCBgEDBQAAAQIDBAAFEQYSIRMxQQciURQyYRVxgSNCCBYkM1KRoRclcjRDYrHB/8QAGgEBAAMBAQEAAAAAAAAAAAAAAAMEBQIBBv/EADIRAAIBAgQDBgYDAQADAAAAAAABAgMRBCExQRJRYQUTInGR8IGhscHR4RQy8UIVIzP/2gAMAwEAAhEDEQA/AOqaUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApVZeqHqgNIX23WiDDEyWsCTL6mUpbj5I9p7FaiDjwMHPcVYFnucS8W1ifb3kvRnk7kqH/6I8EdiPFRqrBzcE80SyoVI01Va8L0Zm0pSpCIUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoDylspkRXmVkhLiCgkeARiuVtBTUenOsIX1jJYDkPDjaFZDjbiNzauPGQkn458iun77PbtlolzH1oQ20gkqc+1PjJ/HzVWep2lrRI9LIy7p9BBnwGWxDkywEPIbQsKDIV9wUUgJxzyT+9VMZQc4qrGVnE0uz66p3pTjdTsvfqVjr25ai1NqRsuS4txZQ0pTLcVpIShBwSBj3+APcTirp9No9yTpKw3kEyZbkFLE2OnCOtsJDawVY96ANpJxuB5PtTVM6guV6jwtPaf0lFdmRo7KpjCoxzJfKkFSlJIPuTyrjueO+BXzeIk1y620v3i5RFu2xD7ZlvOJQFKVtW3gEpHuSrKSPjI5rIw+IlRbrzTkmtevX6G5icLHERWHg1Fp5LXLpprqXjrnXqrHo2ddbfCzOYlpgiPL4CXCQTkoJBG054P/8Aa23p1q1Os9OC6pt8iB/VWyUOkKSspxlSFD7kZyMkDkHiqgZi23V+lmdFQ9QwbfOLX1EaOlBBXNSpRc38DKDnckJJOCVHO0Cvu6611dp2227Tb1pTp1USOiKZoT10PlKAN7KyAgA98cqHnFacsS6a72b8NlpnmZCwMaj/AI9NeO7zeWXl+LnQmRSua3PUnWNm0lBtLbalylhS2rzJPVW6wFEDCTwpWQU7zxgA4JOR0PZrgi52uNLQhTZdQCptXdtX9yT+Qcj+Kno4mFbKPJP1KeJwVXDK89Ltehm0pSrBUFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgMK92uJebVJt9wbLkV9O1aQopPfIII5BBAIPgiqM9YtOs2ddmE68yZyJkhaXXpjiVykICckoAA3JHbsAkkd810AaoH/ABN2x5+/aJltN5aU5IiulIAUdwQoJz+yVYH7/NUsfTjOjJvbM0ey606eIjGL1HpG/FtV8nW2Ou3Pfp0Rctm4Pe1LUZRG5BWQdoSTnAx3OTgYqOa8u1k1xEuVwjXJUKfFJXEe+jdQ3MATynaFKHvwnClbVDAOPFRybbDep1zYgyVuq3pYVBYXn6ltKxhJA+4BSUkJ84zzXhbrjbGoNytlx0/e5L7HU/1MIOKMf2FIDiARgJWM5/fvjnOw+LnOkqEEut18dtzcr4SnTqyxE5Pa1mr/AD2M1mW3qi4puQkpt1xjttqirSrI66AMKz/aeB/NSv1PdvOvtRQLJDRHbuNihKkPbckTJRQ2XmGiDjgds91Ajxmqz0rNtkNiY/cEtSIjkRSI5Kk4QsjBJCsZwQcZ7Z/FTOy6F1jPNuYtjkcQ5CxJZfbewbcnOA8SMZCtuUpSc5HgHNQYfvIOVKCun9N/jmS4lUWo1ZvhavZ9dvhkaLUuoZVyssZpC9jNtZU20kJUtQScApJOTjI4Hg8V0ppfUNkauRtbFwC5NwX9VFQpKsuoLackcY4KVZHiqMu1lkWa6XrQjExn6SRcG54lgjrOHaFJQ6U+N2fZgY4IwDVoegkmK9Z5saQmMm7wnlRiN2XiwAlQJBJITuUoccZHzVnBPuq7op3e9+nIpdpJVsKq7Vlslpnnd8sy2KUpW4fNClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKZortXOnqy5qnUPqJ/l1UjpwUuo+hhsqLSXso3BxxWfcchQ+Bt7Z5qDEV1QjxNXLOFw0sTPgTtu30LX9S9ds6IiwVrt789+Y4pDbba0oCQkAlSlHxkpHAPetdpD1MZvltnuz4Ldsmx/chlyQVNLSR7Sp3YAjKvacjjjvmqsvGmvUO720r1AI8eBZmXFIfukhO9YO0lIKclRO0AKUR271E7Qb6jUTVus1zhhu9Nm3KcdQ4Pa6PtUNp5yMZFZ08bWVaKtaL2evu5sU+zcPLDy8XFJataLP8F8elvqa9re6S4cqzt21bUdL7YTKLxUN2FA+xPbKf+6zo/qZa3ddSNMORJqH0yBGZkoSHG3V7cqBAOU4ORkjHB5FUHrHRdz01q+zWaHd213SdHRuQ2hTaMuOFvYlQO7bxySP4qQaVd/9LvUyQnW7T4a+jdMOYy2XG5KiUklJ75IyDnse/BzUkMTXTUZLR5vkR1MFhneVN6rwrm1v5dDpkEV+1R2mfUDUmpNeQZQcbtukWi4HmuiFdQ7FbElw8lZODhHx58znVfqLB0xb2Zk+23ItvOdJlGxCXXj3JS2pQVtA5JIGOPJrQhVjOHeL+vN5GXVwlWlUVJrxckTiop6kptYsTT9+THFvjSEyHHneCyEBSsoV3SokBAI5O7HmvW2atZuMBqfDiPSrc5jEiGpL4T8hSQd4I8jaSKq//ELebNLsrFtuD1+SJv8AVitsMbULebI27kLAJTlXI+duK7klwtvNfI5o0594ksmUlYNPzH7FIusRrqNF9aUMAK3EBJWdqiMK2JAyM5/FfNmdt/QlXKJqCfY7mlhEcQre2+VzDtKus85uCEoO4AgZ7Zx84tvdlKcTHhy3YyypQOUkKSrHgHyRxn8c+Kkd/Wza5kdmLHhpgxGSyuWy0QuRnBQtZycqHIPH79hXzaq93KUv+np7+h9W6Pexiv8Alb+36kYssZhb0N5XSfbD+zbIOxh3HYq3DhAwPkcnwasbTl+uGl7oxqW2LflwHeqh2CFqaZc44BwCncMJIOOBxUIZebfXK/8AbZ0l11HThBphWerkHgcZTtCu2a+rbcLhCRugSVR3X0I3MkENuIOCG1gdwe+e/P4pGdS6cbrP2jpwpuLpyzT1/P4N87177Mv+pdotrj0wyFMKcLmCoJSGwoAcgpJ3Y/Hirj9I7dbXJz2tW7hCQzLjfQrZCOl0ngpJcBJxySgcAfkEg1UzsmwxbTZZyYS13Ge5JjyWeqXEoSyQpRAHIV/UG0YxjJOTXx6f2JE/UkFExyRb4UyW4YBdQX20rzuDavcMEgDKh3OBXUG6VbvJK8na22vXoRVYRq4fuoNxgrpvV2XTra51224h1AW0tK0nspJyDX1UE9PtEv6bvuobpNfYekXJ1PTDO4JQ2nJ5B/uKlH+AKndb9KUpQTmrPkfK1oQhNxhK65mt1HdWLHZJ90l5LENhbygO6gBnA/J7fzVS+l901XqrWJ1Ct5X6OUrjyWVOlLCR3SlpHOVpIGVfBIJ8VZXqBYZWptKz7PCmtQjMbU0444x1coKSMAbhtOSDu5xjtVGw9T6iiaGt2nrYlFjlxli3rZaRtkFZHfK+xJ3EkD5Oap42qqU4Tm3ZbLd9TR7OoutTqQhFOT3ey6HSZPFU/wCs3qNfLDdm9PaOiR3bsqAue+5JTkoZ5SFMpJHUWnapZHPCex5A3vpbfbq/oWW1d25Mq82cuMOF775G1O9skngkpIBPyK5pUtj1CMF1iTrG666ewvY8lj6VKs7lIbUFpU02Oec8d8VPKspQUoblanhXGpKNT/n36F56S1G5Kj6JvVlvV1mN3SX+l3C3XF1LilKS24Vup49qkKQCduElJ7A4q5x2qr/S3Qdvt2mrZPgzJaHpLAkApcQvpKcwpSULKSdowkHnCtoKs1YNlmLm25DrwSHkrW05t7FaFlCiPwSnP81LSvwriIcQo8b4TPpSlSEApSlAKUpQClKUApSlAKUpQClKUApSlAKjWsdIw9SfRvqefh3KGvfHmRlbXUA8KRn4UCR8jORg1JaV40nqdRk4O8WVR60rXdvR1UqLKQwd8Z5SFqwXcLTua/8AlnPHynBqk9F3OHZb/b7hPaXNixpTKuouQpJjAK9zgAHvwFfafjjmumtSaS0vcFfW36Aw4hlZd/quKDYWrgq2A7So9s4yc1yJqGCxa9XX22RpsV21iasxynd/TTuylvB53IzsUD2KaysfTmpKtlkbnZlWnKEqGed+nvqW9J1JbvUXVsd2LbZdq1PbVOuWi5spElC2kEkJkNAZAORwAdpPCh2MdvjF9Xray2rWbzj0m6vMt/1FFQCFOBKkp4AT5OAPIPetNojWTunrA83aRFj6kelLRJmPJ35iBI2paVnAO8k/xz4rS3q6OzhCtN5ustxqN/qYhfX1ihS14Vhed2Djb9xxuJHPFeNRxKjGpe/Nf16XLVKMsM3Kkko7X/t1sW9rrXLlj1jFtui7NGmIgOpEh514NguFQHRaVztABKSAMZOMEivm+l3XlyiTrs+mwvpnJtTMCWkhTR273CSrG5eMEYGDlHPNVXIYgsRYkiAy8+oL+okMFJSmMpopSN+1GG2178IXkc/cQTXtPnTNQ3OA/qYOxnLs+3EjuSWzu6YXs6hCsZVgoz2zszgCrNdOrRk5+LfXr0y+epDQpxpVY92+Fq6bevt/Q6Q0nbzYw/G03CSqG3uPSdf6ZdUTgvLVtO5xez2jgBIByNwArT14u5ucXSr6JEi3T1yH37bPhvFJS0AlLiFjgpX7kD2qKTtJzjivvXF6Z0u1Dg2L6STLWGVzG3gp0IDaC2plxAUB7igH5GFcciqt1rqGRqKZFnXJ9AhtrLDDO3/SRElIAShH2jhI/Jxz2rzFYulTToU1msslpYhw+CnOaxNTNa5uzZh3iPFi3RCbPcJF0eU2Cp988gnBKfjO74z3715h9xcZoruEpMku79shz2gk4Ugggkk/vwQPyKkNiasb+oIdvhTUuWyStBmLaa/2E4xuWpIwlIPkYHNR7VjMWPrO7RrTKU9a0SVqaUWgFgE+4JHYpznb24xmsyMJOm6tstM/8NaUowmqeuW35N3YEXK/3eyR2RcDPb/1LEgvtN7el7wUhftO3bkZIz2zioS9JQZ9wfVD9zj7hLrrm5PuUTnaPaD+3HxW9uP0oW6YD8kWyKnqASUAulSgdwBB4GeMduTWXb7VHutuEpL0SJKhtgPNPqwHElQAWFYxxuwR3wO1Kc+KCpwjlnp7/IqR8XeSly1XwMCFL/TmTKjOLRJaKW1HHTcSVj2gkc8gq88+e4rfXI3C56bDH9WS3HYWT0GclpvOVLOO+M9zjwK1VtfN1mOIeSl5LbQbQo8LylW8JKvgYz58DxUh0xGnXrVUSxW98RjOZeQ9sPCmUoKygjzkpSOf3ryunWlDhbfTy+XpzJaclRpzlLJLf3mdGaJ1lpu5aeT+nXFTbNujoS+m4K6brKANoU4Vd+33ZIJ81+zvU7RsOC3MXqGC9HcQ4tKopVIO1GN5IbCiAMjJPyK5ClXK8fWhLOGFRZDExEJwBSHVN+5tSv8AkPcTjtg/I4kNz+mu+r5N209pWZHbkwQzdLcy2UpbmrKkpCCkf7alpQrj7iPzWnSxjcPFbi5Hz9bs6Maj4G3HmdO3vXMJHp9dNUaacjXhqJHU8Etu7RwMkL4ykgHJBAPHiuaNd64vWqTa7ncJCSwHuiwww2pDO5WASg87lDI8k4PirX9M4KbdbbJaWC1c40C1zE39xtvfHeLigtDBUoAOLBKxjnandnAUMxDVXq4jU+iHLbFsEK1RlBtTDrryXBHSkgpUlGwBKhgYI7Vxi2pQTnKy5czvs+LhUkqUOJrduzSMjUk7V2j4kPrqk/pkhpKZcOUttfWQU4dBSDuTlIIByPFY1u9CL+5AUuVqWzt2sNKWxLbQtSi3tJQpQISEgjBJ3HHPeojb71NuMOTcLg2q4OySFyeqVErIPkp5SPA7YHapXYtMaq1/Ck3ODcFW/Rz6lOxYUma662whHt6aW+xAKDjsAMeap4XxqUOHiUXkr5+uho4y9NwqcSg5LNpZdMtb/Uw9M6G1ZrK6R7xb0x4UBEVhqJcUSVJbQWGwztaSk70je2o9h3J84q9vR6Xcn9G/SX3oG622W/AkrYUFJWtCz78juSCCT3yTnByKqK5en+r3L/bomkzNYsaYrSFLNwLTUWQnPUUUhWTuyleQnndjjGBP/QSNKsUTUWlrq1GbuVtnfULUw71A628nKFk/PtUMHBwBkeTpUoOnVs1Z8/hsZOKmqlHKV7aK2aV9y2KUpV0yhSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUzWk1Pqe2abiF+6P9MYyEjGT8f9ngfP8ABqL27Usq9NLmQ7xCeti1obkogqDr0PcrAIyEkdwFEhR7lIGK9Vm+G6v8/QkVGbjx2y5ku1DCXOjMqjpS4/FfblIbUraFlBztJ8Z5wfBwa4217AvVsnpbvEKdGlRGXUureG9ThXJec66VDIWFdUEkE4UMHFdfz7RaoUYSEMKivNqSEyIw/rbioJGVd1ZJ5Csg+a9ksTJAehXRyItso3NvR1Kbd3A8K2HISRwcgnnxUVShGotciahiHRd1mcRocS5EfbaKmJy5KS3h1LaEsFvbswRk4UAcZ5yeCTkWbpb011DeNKM3KO5FNvQXZERh9O59IIyemUg5KiOxI+PzV2ML07Jvkm36pZ07Iv7O1Afcba3yG1AhHCuUrxwUZPgjggDQeozMvRPphLtliubLcdlkNxo6kn6oMBQ3hCgr3bUk87chIOTnmq0qPdK887K3w8uuhfWNc5KNNWba1zIFoOdpJj02udyuzUObe1yXYyYbpKiVIILaVN5xsCgFlR4/OcCpN6VIYv8AoC4wNV6dgTbNAwpuRsBXJdCTn2n/AO4BgbwR9wAxg1WU6fZHND296Lp5hb7jX0ab4ha46RK6ZdKFIST1lAY95ABJxx3ppLW1/wD8lSLCy+mLb4jrn1UqKMOkLcwRuPITuVjIGcZ5wDUcOOlKKj/89kt/MsVYRxEZJ/3b1bySXK33Jxq30feag/V2afGgSlyC643JeW8xGKjlLal8q28g7yDz42n2xrWukdUXCQ1Bd0obfAVc3pTLDb7bqC46N60heQCDswkbRj7fipH6S6niaZudxsd6fbZiSQ59Q3Ky4PrxIcYWFOgHIWGwSVZ8c/NhvPxb1oC66YXcWmr2mE5HXHKgpxhwhWxAHJXtICQoZJ254JqzPCxqJ8Ks8m7eWRUWKq0ZLj8S2bvpvbmVIl8af0rarPZ4Vui225lpiZceQslTqSveTwQkYHu7Ddio76saNt9g1eh+FKUYNyQ6+w+pwYV0ykLbTxgnKgQf/wAj+BWZYAyrQceTNRNkPiT01wsJW0XOnn7ODkDOSSf2qD32JPdctjyo8qVbEtKkJZjFbv0oUvatKtoKWlkISfPG0nORWdh8TOdN0qnia+Vuf6NXEUIUpKpDLpzvuj3t85hiRN/07QZejFlcdauor3EDelRACVJCc5xyePNYl4MZieoxng9GaUgNlCwVLcPOO/A45rOtq25lz+iTBZMZ8hDK55SgtAgrBUpPAOEYweMqyAM4r2sn0s1rofQpHWbBDaW9oQQrODnlKgefz/NeqHH/AOvl0z8l+OmWeR1Ftu+7PAykRGOrDjFtZBWsEgBPyc9v481mR3p1uuFqnKkKiSUpW4FNjY4N6SDlX/xUUj8fNeMaEZt56QCWYzSkZUohZZOfctXjgcgd/wDxW5tKLcm8FjUk6KqHKbLU0yHiUNs5SvqIIG4Pe07MeTzxxXFDDzS4UueettPwtiSvVja7ztbLS+vQ0uoYCGLG5IRaJqmHHWOjMU300Mp3HcQe6wr7cp9vB89pl6Iaot2mYWqI9/Ybd6kRtbK1klD5BUA3k5ySVg5+AfioZqdldpv09uJdFXW3xy2Y8lMhK1JYSg9NsqTkEpSoggDGc9q2NvlWWdYZXWtzZvkhAg9BAV/TSlxKkvKOPc6oqUnCcAAJ/nq6hLwWtFeefQrSj368Sd58uXtFzejWstQ3qXdIF4jyLraW2FvpmqQAW/AYJ4CypPIycjnJORitZerdDN6MuUfSejfo35rRQHJqw6lLR5PdSiP2GP34rLsU7UejGJOmYj8qCq4uNvsoACnQVgJIRknaSoEEYzkeKxNMX7RVjsb+nL/oZMmTHccjSni4kuKWhRHnlHPHtV+a6eIbXBJ2ktW1z2I/4ihN1FFNS0Sly3elzfXT1fv8KVcNPQn7UGbehphM+OlTiinpIPUCs7eSTg4/jNazSt11jKtcGFpAz7jZgz9Q44iNlvrrcWXWlqVwrCvGeQc45q0PSrU0G3+lFqffZZjstIUwXVvIDQCXFJQhS85UoJAGAConxzW50UyNGWOau8JZhi53N2REgspCSgLSCEAA4CjsUsgHCcnngmrMaE5yUlN5rK2xT/kxox4e7V0987vQqSRqz1Ku09myQYN1jT2HHDNEeKGVOpd5bcWpQwkDa4kKSQOBjmt/6PN3nSvqFItl+skqMb1HKkTHzuK3WtyincCQchSjyc8ZreeqGpNSW2xIvWmg4bgZrMdTTERUtCIxCxgq27SorKCSk4GEpBPcwLT2srhdPVnS/wDneTKjvMvlmPFEYsBDziChO5s8jcVYJOfHYVHOHdVYym25adCSEpVqE4xUVHV21vr6XOnqUpWkYYpSlAKUpQClKUApSlAKUpQClKUApSlAU/6qWa13n1H063qCaI0FNvkrZSsltt50KRwtwfbgHIz37ecHdaetaZ12jLRI02zDhx+l9PaVZUvK0KO7ttBUhGBz55JORqvVONAu9+Rap0WVNeVHaW04y6GxDUVOADJBClO4wEBKiemc4ABrV+kmj7Zp7VN5avSpYuE6OhtmHcWG0IdZSoKJThSkuEKx8EDHAzxBFtVfDFNc9/I0Xb+MvE7rbbz9r5Fo63dfi6XnzooSp+C0qYhCuyy2CsJ/nFc4ybhf39OPazZnXlc+KeoJypjMWBHcA5ZbQvK5GMlBPZRyBmukJllYjx1GBJNuA5IJ3sEf8VNqO3ae3G088EVx3Jism6Oi8TYqZER1cNaokQLJbaUUpXkFKVqVjGSewGfmo8ZPu0raeh32dT7y8V9L5Eitbzd2vke1OXJ+JA/Tl3FLk5RfXdJm1C3QpKztTyXEhKcH288mo5pbU0iDd7beOui3ltBK1LQtSQ0pWFcbgopPwkjzjJq4vTj0Z01qDROnrvd2LgzNeZD7jTU07Fq3EpXkZKSRg+0jHb5qR3j0et0aS1cNPIZ+oYjvRlRJK1bJLa8lKVOHcUFCiClQGMDaRg5BRqumrStv7+/3Jf5VGMnB5rQq/S+jrk5bo0efpy2pcklj6Sa3MUwmO+Ue4ONnIO4oSNoSMnJSTkEfVq9NZ0cTHV3xy3PNyg3ORIShKlbQlSeCVFxWSMKSgFYOMnNWLbTqabqGSxfGY9kV10TkRWcOqkO9NKArqEFIZRt3Ej3Z474zvZEN6Fco6dMXFqfdnmnHXErbaUhxLe3cpbn3BwlaUpJJ+45BA4vxpQazWW17r4vQryrzi9fv+vQqbW2m51iiRHWrZLQ48h12SpLHUwoqKkqcKc43cgnCcHgjkE764ayn69vrVn9PpK7bb8NpDsZIadWlJB3LI5Q2nco7QQT/ADip9pW7xrrf46mBJSREkpnNTX1ObAFN4GD7QfnHGCKhF+9TrQxqeKxZ/wBUk2qFILsiSy4kb8AoKW0kBTicY95V2BA75qti6cIzvOfDfX3t+CxRq1Kq4HT4mk7dL/XTI1WqrleYXrRLafjRIoQhDjDYUltqQypJClknGSpQWCTyMYycVErU7Kharjq0fqSBDu0wuplMTY22BuGQOmtQzhSOMKHcDnOANz66XWY/qCw3sJdbgT4rjLKXAgFKUqKjgAlX2qQST5NaLUlmtI0nAknUNsmXKWpKGoEfKilJGSSrxjGOwGfNZslKjipcMFnvf6Z/QvU1CrhYxnKzWVrZfH9mPfXoqb8+/e7sq+XC4xxK+piNJZbacHsQkp8AbVApwDgA9yKy74f1JNpXGfDs6cotOxWwEKbSMgc99uE/POK2F/aiatsQu819pq6wWW47rje55xbbTf3qCCRhS+MYyCSeR2in0MNVj/031ypS0hbn1AH9JYIwGyD5HfjwM5qPExXG6kMk18VZZp8r/PLmWcNxcCpyzt6fDmZlutstEBctyFIbtxcUlqSy2pKOojggqAwOQRz558VpswLnNcBmqjNoUVKfcJWmQoD7QByTn+OK2EDVMhy8bpv1DwUpQfajuGMy2k84bQn24zhRGMHzkmtdA0/OuuobXbbPHLjsx1RZG5O0EZ6hVzkJSOe3btnioXeb4Yt5L1+Hn9eh258Mbytb6dL9Nx7itr6pLiGipKQEkBXfJ7eMA/8Afit7ojWsbSOtW7nBjR5P1WWn2v7lJAPubJICVZ4z8E17a80dedJzIbd+iPOW1Z9k2AsFtayk4bUpQG05+RggcZ5qPafnfod8gyJsZf8AT3pStKAtDiFoUlSM9knCjg+P4ryDnQknJWl8/wBnNRwxEGoO8X712LP1Jf4esnrZfrYG7Xq6Eem7bnnUqZmNqO0KbK8IU4ndkZAOCe+EkRa0wXZeq7jfTCZnSLbNDsyLJc3iUlOQs85CtoTjcMjO0881iWy12mKu1StbfVOwZTw3QI6FZ+nVkBxTqccA4O1OSQD2rd3K3Wj9U1FfbKwxL0qVNiFEbQpDYUQlKw62khbbYKTgEJ3bhiuqsp1IKs5eLLT7ojhGnTn3EYvhd7Nr5XJ56HSbNd9a6pk2uy2q3RXEMvsRkNoLrZVuStQwOArAyB7QcfJJl+nZVpc9S7vZLa6iXEjxGphYIS4zCkblIUGj/YVJKcpHAIPkqqumtIM+p0OwXbSrLOlnExHI91XGbPTVhSQhohJTvI2qVz2CgFc4qz/T3Rdv0E+YMFa5Ds5nqPSXQAtSmyBgAcBP9TgeOeTmtWgqmS1WtzDxLpXk1k9FHlbK7ftkF1764WV/TjrGnGX5kiU79IsSWVsoSg5CyOyt3BA+D+1ahWt5/qPrPScFNst0MR7izLSVLDjwDZC3NqyBgFKT7QMnzUn1pp/0x0qzerperK7McclpflhsuOFLrywcD3BKfu3bQQcHIBqvdJrss/1IsbGlLCpDLF3EqLIcWovsxi0Q8hfJyjdlSdxJTnA71BX7yM4qcrJvRblnDRpSpSlTp3aTze2XzOp6UpWkYopSlAKUpQClKUApSlAKUpQClKUApSlAVN64Nz7HZZGobKltbq0tw3m1pJ2qWVNNPJI53ILyuPyk8beaZtN9tln1Bbrwy5eru40lS7qqcQrqtkEP5bUDkY2kZV/bycnA61usSPOt0iNMH+ncQUrOdpSP+QPgjuD4IBrieSm83QTdQXFm9qs9ySkKuCo5w6D/AGLUAlJBIbyeAcHGcVn4tODUo+f5Njs+SqQdOXl798jruws6dv1jg3mz26FIivth+KpcYI/bhQ9pz+K5x0y3f7M7N0ppy5RE4gB+8LnsoaS0s5Ssb3Bk7hjtnueAQa8/RK7XSHr22/p0ySixNNyHLmwFqMdDaUElak9kkHZg4Bycea01vESddE3DUhuExMgqXJLDyTIVncRtK+Mgq7HxkVHXxsZwhKV7t8yzhcDOE5xumkuV+ezOgfRi+KkaIRb4iIs5dnWq3LXCVgZQAUkpXjggg5B75+K2EaTqWG9MkS5Ei5SW+mswozbSG2Ud1hRPlQPtQFKUNoO47sVS3phqdqxeo1vbtMRyPbLmRAWxJlgqdcUcocPGEqBG3A8KwPFdMxYj7NueRvbTMeK3FLTnaHFdvyQOB+wrToVoTjeGa0MvF0nRqeJa5mpvky13CzwZq1unrN/URShYbUElIJUd3tAAIzuGBnHcgGuzr0QNRvMLtTjdzah9F2YENtBxKSVkhPcqAwACAASo48HPTcoljmyEQ7VDdvBIZYblTUtpiFLWdhCzwkLTnckYV1AQckiqfZuF4v1lnMXBwtSrS6FSGZDWHyokhYJ/BIJ+cg1V7UxNTCQXdLXd/Qudl4OliJtVPTzNrHvEXU94mtS51001GuCugDuRJZKSpPDqjtUkHbk8kEk5IAwYzbbyyt5r6eOhTzZ6LgbQFkAjafb88Vqbgm5So8R/6JxmGJI/1OFLaPByc44H/dT3RNksNw1PEhagiXGNJcBbTNZK2QtRAKEuHHHjaT33fkCseqquNnHvrKTTzzz98zcpyhhIylTb4Va6yv58/qabUIk6ivTkwtlSkBKEOrO32owEtgdzgdz8/vUcujTL8ZCbcuEp15BU4lG4KZO7GFYGN/GfPcVKvU2EvT+rp0KHFc/pMuOrSlYLQzktqT3KcgAqCifco44wKjwtjRhtuNTG+ttUtxEclIbXk5HPfPfyOeK7xlF0fHVln9Xq3ZfBfjQ9w1WNZKNJWX22JnqK5WCfapd1sUGbH1FdmhGfiOFOx5RRtekrKU7ckEfbt3LwSnvUPMVcFtcJUhRUlOCO6kII4Ge2ccH8g1uGrpEskB6HbnQ7do8gt9VkhSVJ2g/0/JzkePH81HbnBuNrchuyn25SpjylKaSvqFtW3J9+eR2znzXlSosVSbeTX29t+duZzRpRw0ko3a010v7+p7hVtjaTEdyI6L7HIbacSrKX9yuAR34HGB8VJtG9AO28aSjOMa2SFtKclSA02TgbnElR2rUdxShpPGApS8kACOwZc21asgXBlUfY0kpy6ApCVYOP3zkjI8VnRbm9eoDcDT8WINRRHPqY8mM5scWpCyopTnjJB9vbtUNCpKjO7WuvT8HuIpRqQavpp1tzW6N56i6n1L+lXKwal2OzXmQ1udwgx1ZQoD+n7VBQAIHk4ySOKjSLCq46dkPTHi3GbQNydmCp3J2pSQT+Dk4xzWfb4FwcQmddrZJkw+qfqXN+5e/urceecnnPNR+VdJMmBIgWqWlxkPLXHjuuAbRzgbjjnAx3rmpUq1lGTejybJIUqVG9la6zSyXWyN6zd4rLFqh3NSZjcJTiGUnCSW8diodyCa8y/GtV7dnNWp5cBTPTUtALi+ks/cs9k4OAMfA8mvrS2kWL7fZqbTAfuWl1REKXKUrpJgvFCS5h1eBvbVu7Z475zXzqNNktl8TAgTnLjLaSEvTcjouEpHDOO6R2z2yK5nhnTu2m170+55SxUatoppP6c78un6LZT6fwE6WgpZReVLJbUibEktpS9vUPclKV5xtVx84BPOTWy9Kb1+iM3GDrK5x0TmLqbTAekyFb30bG1pSnqEq7rB5+QM8CqCbm3yAXGY90mMwRIL8aKzLcSGiE/cEg+05z2+a2WpLREt0NiTEkQ70tx8ITLjurU/MddBWkFs5WkpKSn55GcHitunjIzhelFKyzz2yVt27GTUwNRvgrTybe3z5L9nTHqNadPO6Zu82/QYDjKGN7i5CQAoo5QFK484AOcjPFfmn7TYIEyInSECGy02ol+REQAhSCk+wrH387TjJxjPFbOBaZUlLDt9lGWUBLiI6m0pSheOSrHCyCTjsBxwTzWxh25qC5iF/Qi7cfTISA2k57pH9vnIHBznvWj4ddzB4nFcNzNpSleEYpSlAKUpQClKUApSlAKUpQClKUApSlAa3U0Jdx03dYLSEuOSYjrKUKVtCipBABPjv3rkOdfZ8u1zGv82qt0GOlaItpfkPAIaByG0lI2r2/anfnISOR2HYN7S+q0TBEbLrxaVtbSraV8faD4J7A+DXLrlmkWyZO059LMmWQpxFcZUy37XNywha1sqcbIGQtAUkgpJHBqhjclxPTc1uzVxXis3fL2/15ljenmkdJ6r9NIN0cZ+gdlw0s3J6BMWwHi0SFdbaQlR4ySoefioD6rW7RKbfZYujXpEecoBSZfUUmM4wVkdRxa0nflQ2pUn57kACvGEvUd29OtVRXjCgWy3usqmRrezgTSshCyVZwhKUtpO1AAPOT3rW2q5P6iud2YbbmzLgu0IiNfp0L6hKUt8JZcbHtShfbccbVpSoEYNQTrQqcMIxu2tbFqnh6lLiqTnaKel309dSY+k+nosX1DhxrrbXn57TSpSUT2QQxtHtkx3RhLicqCcbcglJB4Jro6q69KNK3a22iyzNWONLukOB9HGYQnmK0spUpK1kne57UJJGEgJwBySbFrQoU1CFkrGRi6ve1HK9yhPWHVOo9Oalg2qKr6SDLD76JDSP6jyt4yA53SEpUgHGDz3xiohZbsnTEs3q8W0OomBuRFlODkLbKs5J77gruTn2p5wav71LmWmz6Wl3q9Wdq6tQEhaGltIWQVKCcgqHtHI3H4HY9q5i0/f5f1jce53B23MNuB3ckgtEEKBKcgpwM4wRxntWdjak6OIjU4nbl8jb7NcK+GlScFfnz320+Jnh2bp1Ftv0TUUqAHFJiR22lFZKCSVBTZyAhB4yRjjgcCp3py8X2/wB7ZaN9s9wW+VqDUkKDTjqQkp9qAk7f6aTjnJTnyaruXb7RaYUJbtx6obmguwNxQlS0haQQclJwknn81jT4ssPxJNjjuNRpS0fTrVy4Fg4JQpJBSrdgA96qf+RqKUeB2W/X1v5beRfngKc4yckm9umXw/epMp2g7hcFStSamktvxWHZH6h0UKLiAkKSpWAPclSR/AI4AHFbtW26S2S7a0rUHQSmK4kl1QP9o75IB/bipdq3VV4bscq1agdmIVIcbf6LyQ31CcIO7CRuRlJOD575rARKUmBa3tOSnFXlnqtyEocyoNlJAJzwElBI/wDPevK9WmpWpK8eue2dvvfdHtGFThbqtKWyWXO1/TloeUO1W4t9G7OPQ5fTA9zIIyFYByDnG0E5+R+1YV2s0lmOFGVEkBHSlszYqlOBLS0qCmFpwMOHg47e0c4zWNNuES4wA1eWZ8aahJ+nkpySEj+wjPuQo+Rn9xW2sLMqxWwynUiVaZQLbjDquVkc7Tg5wOMEc/mue9apqVTNrbPf7fPO3Ik4VOdoZL4f6R+Y05c4+8dRpOUobQBlXOOCR2JBPP8A1UgixrVa7IxNYivPXuNJMfGxZ6WBnYcfcSeQPx+K+Vz4l9goYtsZqFIa2rVIZdKSAOOc9sEg5/FeCI1xsUZaoc3ruvp+ocS45z1MkKCiD93f/sec1CuKqpSk7K+e2v8Ah3aMZJJXuvPQwLPqibHdVEUuYyw44UvIQ4ThJweRnPkc4rITNt63ZamVCRuWcqIIKlDv3/PmsSPGU+87KCUsLyUvNocJHbkEnnIPOc/ivm92qXbXm5lxtk6KHU72H321sbiAPtJHuTyM/vXkqcJNqPy0Cqzgk5W6Xte1tjfRdW35L1hXcoUKdZ7a10m7O6kiMtO0pC1JzhayMkKUDgnOK1WnbMm/6tjRLPFclJfeD/00dO4QUlR3q3cBKQOBnAzj5FYrM1Uv/TTJMcIRypxvKgkHjOe5xnNTjSd1h6W1Lp2fGuU622GduZhy1Qkn6pIUlGH2wrIRuP3fIBTjnFqmqlZ+PS+z/JTrOlQi5U8nbfdryIszZ50rWDlu06l27YWtBPSLZjICj7nsk7QMck454xkgVcnprYNO6d1kVPhcu6NWdm4JcWlKhDU6shaEgD715Secq5ODhWKk0N6Nqq5ao0uouWHUdpkIfXcLThsu9VAKXhkHJUn2qQrdjAwo4Chk6C9M2tJJdP6mqdIkuNOS5DrOHHul/tp+4hKQcE8ZJ89sauGwsaed7vd+WxjYrHOtFxeS2Xna+ZNIt0bdlCM80/FkKBUhD6QOoB32kEg48jOa2ArVaigvz4LaIamkyGn2n0FzI+xYJAUOU5GU5weCeDQPXhob3osR5HlDDpCx+24AK/kpq/ZPQytTa0rEbuMZyEuV1QhlvPUK/aUEdwodwfxWXXJ4KUpQClKUApSlAKUpQClKUApSlAKUpQCo1rrSrWptPXCAy63BmSg3iYGd6klCspJAIKhgqGM/3GpLSuZRUk4y0Z1CcoSUouzRBfTLQR0bZbjb5txRd0zXi4tS4oaASUBJQU7lZHB7/JqV2a0W6yQkw7PAiwYqeQ1GaS2nPzgefzWfSkYRgrRWR7OpKo3Kbu2KUpXRwYN9tka9Weda56SqJMYXHdCTg7VAg4Pg896579RrDoqwITpyI5cZl1baSFIXIwiOlQyCvCRkkchI/c44z0lVZeovpYxqzVES+NXBMB1uOY8lJZ3pdAOUK4UnlOVDnOQR2xzTxtF1ab4FeS09s0OzcSqFZd5JqL11+dipNJ+nd8mW9m52hdsvER1xceRa5C0Dpo3e0qOSCCB2IChwRnxpNQWJVi1i/pm3TEMPQiHGUTJBQ2EFAc4WrsAcgHGSQKmvpw3H0v6sNR9N3Bd1gTx9HNdCUobWpIUoLQATkJIxkk53Kx4ztv8AEZOX+rWa2SbXb1RpDKn2LktOZLLjaxvQg+AQpHznJ47VmujSlQdTeOvK6/JsrEV4YqNO2U9NnZ/jkyr2pMy7Qnmr5apLjsV4L+oU+CpCNvKechSe5GDwc1kvI07eI7bFthSGZuz2K6gc3uYHGMcdia0F6lfqIMKO/FgSmI+0F4rImL35SNwBCV845wkgdxWZeLa/aP0aQ++/FuimfqAytBaIwoghQUAeCO/cfFZ0qTSU9E9LbeZqxqx4nDVrnv8Ak/bfYpU9q3GSHpSW1OJwy2Futjfj3AYwfI/c183i5R5sgwI0tw26M4pIS20pKiD+D2J8n/xWbdZcu1hFwtd6fnyrggmbFjw3IxZUMHIWo4cBOe2PxxXgia3aZEZE61lmdJYTK6clQWV5KhvRgnKSQePBBH4r2cJq8lmunXmeQnB2i/D5rN+tv8Na9b4dvEedHeZT9ShSG425ZdcCV4zgDCU5yNyj3Br2j228qUu6SugmAXSVbOFKIHYZwCfmvtd1fn3RYeiKGGFYDra2dpHIKeMYHJx55r8mzlTbG0z9dJS81J2iIY/AZKcqcQ4TjO7jaR+c11eb8NuV7+/M58K8V93a2XvYzdcxo0PSlj1HZZqnW7gpUKTCfby4y+jJUoKTxs+0AEZ9yTk5IGGL9fJMjT91vdwekwIkxtcVEl8B51CFpUvoJxnA6YBUo4zjk44tfTPp1C1noyxOWm9zIkOGypLFwbbRulOKdWVlSCTjYcJCsgn3eMVGNSeier7TJWxp96PeIC4hU5KdKWXgtOSWgnnhRxjH7KPHOq8LOC4oRWi08vszHWOpzlw1JvV67Z8/ImkzQem/U2+Sb3GTLssp+GlSmypG94L/ANt5bPICSAcEKBUR4KTnXac9D75ZNVQJLlys11tUNwOs/VNPJcSoHIPTSdpAV7gN2Cak+mPrrr6haYk2mwXWz2a02VcWU5cGFMFW/ZsYAVysoKASocd+fm3B2q9GjCdpSWZkzxNSn4YSyZHNJaVj6ffucwvvTbrdHg/NmPABTpSMJSEjhKEjhKfHye9SOlKsJJZIpttu7FKUr08NKLIpV+fnOyiqK70l/SBvA6qAQFqVn3cY4wOUg84GN1Slet3ApSleAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAVh3i2RbxaZltuDfVhy2VsPIzjchQIIz44PesylAc2SvSGXYNPSI1zvFuccdfLUNalLKnk4Vsb2YAC1YBJyQOfAzUs0Pf/p/Re9Naltkm6f5fbdjSI7iNyn2kpCgCVf8AFKgD8BOcdq9fXDQ9x1Bf9O3e1IedbZ3xJyWzuU20ohSXEoJ5IIUDjnkdwKw9C6rmDWT2i7xAb/RZKH2W1SWS2+ohOcOZ+/cgKySMk+ayYxVDFcKVoyXqzelOWKwXE3xSi/RIpO9XlmfDblz0BTSG+m2zFHtbbwcJTnJKUg+SSf3rozXFq0Ze/TyHfdYLF1iQoILdzY3IdO8JG9G08KUoJwDwCcHzXPjmoNPRR9Pp9mfCiKPWtrrziUOJTnncsHPA7AHPI3E1MdE6wjxdNTNLaktcs6SuCXWlXVtKlCOt3JIwUlOByoEZIIzg1XwM1SnKEt757FvtGjKvThUp7WyWuaRptP2+53MAWhAeitONx1vSNpLCHXNqFLI747kgY4P4q6ZUDQ14syNNX1FvDtuPSaccV09rhPC2XvkqPIByFZSQa97F6XaQsNn6qkvTA2yXDOfeKl7duSobcJxjntzUq0yqN+iRLVKS0mU3GS2/FWgJJOMK9nYgk9xwc1oYHB9xCTks3+zN7R7Q/lOPA3aPvm7/ACOf7Dc7VaLE5C1lZFM3uKy5FXcXQt5KVEgkOteEcAe0H2k4HJJ1mqdBy7DqBD8hhUPSlwksqMxTgcRAjn/cQpXz/wACRyCnJzmr/wBUenGnL/BXFcimClZBKoJDWf3Tjaf5FTEoQtBSpIKSMEEZBHxXNLBvhcKiTS095HtbtKPEqlG6b/td781e5RmndYsPepdgsvp27MudjQ0pm4I2BuHGZCcpU2kJTtUCO5+7dgZJyL0xWPBgQ4CFIgxWIyFHcpLLYQCfk4FZNXacXFZmZWnGcrxVvfwGKUpXZEKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUBiXiU7CtUyVHjOS3mGVuoYb+51QSSEj8ntVXXq3WO06Nf1TqO7P3N9hp4omR0JSpDzwKMNo/5ArwkLJ2+cY4tyqy9a4MiFYmb/AGhbjUqDPjypCQs9N1CFcFaOxwdvPfH7DEFeMOHjkr8OaLWFlJzVKLtxZfq/UpCwXuPaLJbILFotkppqS046xPaS+h5WQlJAIyhSQTggnvzkcVYv+IK7z06k09Y0QHlwpP8A9MQPZJkqVsDY+CkEd/8AmfAqaaL1nY/USQ/FdsjhkQUIfX9Uwh5pBJITtc593BOCAeMjtU8fisSFMqfZadUyvqtFaQotrwRuTnscEjI8E1ToYTioSpud4yNDE4908TGqqfDKOudygtWR3/TLTtqiyr/cLu/MlpdnWx18qjqipBK2mgRubQDtSOfdggjBIFyX+QxL07HuMVSVjexIjO47blpwR8ZCsfsTUd9QvSiz64uaLhPn3aHJSwI+YbyEpKQSeQpCufcRkYqc2+Exb7dFgxUbY0ZpDLSSc4SkAAZPfgCrtGM6c3f+uVihWqwqRi1/bO5kilBSpioKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQClKUApSlAKUpQCvh9luQytl9CHGnElK0LSFJUDwQQe4r7pQGo01pu0aYhOxLDBagxnXlPrbazhS1dzz+wGOwAAFbelKJW0PW23dilKUPBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgFKUoBSlKAUpSgP/Z'},\n",
       "   'literature': {'PUB00019704': {'PMID': 12462127,\n",
       "     'ISBN': None,\n",
       "     'volume': '2',\n",
       "     'issue': '3',\n",
       "     'year': 2002,\n",
       "     'title': 'Histidine kinase-mediated signal transduction systems of pathogenic microorganisms as targets for therapeutic intervention.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '235-46',\n",
       "     'medline_journal': 'Curr Drug Targets Infect Disord',\n",
       "     'ISO_journal': None,\n",
       "     'authors': ['Stephenson K', 'Hoch JA.'],\n",
       "     'DOI_URL': 'http://openurl.ingenta.com/content?genre=article&issn=1568-0053&volume=2&issue=3&spage=235'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 512,\n",
       "    'interactions': 0,\n",
       "    'matches': 92227,\n",
       "    'pathways': 0,\n",
       "    'proteins': 92217,\n",
       "    'proteomes': 4936,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 80464, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 20246},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 85238,\n",
       "    'alignment:full': 23604,\n",
       "    'alignment:seed': 1578},\n",
       "   'cross_references': {}}},\n",
       " 'PF01156': {'metadata': {'accession': 'PF01156',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001910',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Inosine-uridine preferring nucleoside hydrolase',\n",
       "    'short': 'IU_nuc_hydro'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000435': {'PMID': 8634238,\n",
       "     'ISBN': None,\n",
       "     'volume': '35',\n",
       "     'issue': '19',\n",
       "     'year': 1996,\n",
       "     'title': 'Three-dimensional structure of the inosine-uridine nucleoside N-ribohydrolase from Crithidia fasciculata.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5971-81',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Degano M',\n",
       "      'Gopaul DN',\n",
       "      'Scapin G',\n",
       "      'Schramm VL',\n",
       "      'Sacchettini JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi952999m'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 243,\n",
       "    'interactions': 0,\n",
       "    'matches': 47119,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46162,\n",
       "    'proteomes': 5674,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 40031, 'rosettafold': 0},\n",
       "    'structures': 48,\n",
       "    'taxa': 24309},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44728,\n",
       "    'alignment:full': 11393,\n",
       "    'alignment:seed': 664},\n",
       "   'cross_references': {}}},\n",
       " 'PF02514': {'metadata': {'accession': 'PF02514',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003672',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CobN/Magnesium Chelatase', 'short': 'CobN-Mg_chel'},\n",
       "   'description': ['<p>This family contains a domain common to the cobN protein and to magnesium protoporphyrin chelatase. CobN is implicated in the conversion of hydrogenobyrinic acid a,c-diamide to cobyrinic acid [PMID:1655697]. Magnesium protoporphyrin chelatase is involved in chlorophyll biosynthesis [PMID:8404842].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008140': {'PMID': 1655697,\n",
       "     'ISBN': None,\n",
       "     'volume': '173',\n",
       "     'issue': '19',\n",
       "     'year': 1991,\n",
       "     'title': 'Nucleotide sequence and genetic analysis of a 13.1-kilobase-pair Pseudomonas denitrificans DNA fragment containing five cob genes and identification of structural genes encoding Cob(I)alamin adenosyltransferase, cobyric acid synthase, and bifunctional cobinamide kinase-cobinamide phosphate guanylyltransferase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6074-87',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Crouzet J',\n",
       "      'Levy-Schil S',\n",
       "      'Cameron B',\n",
       "      'Cauchois L',\n",
       "      'Rigault S',\n",
       "      'Rouyez MC',\n",
       "      'Blanche F',\n",
       "      'Debussche L',\n",
       "      'Thibaut D.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=1655697'},\n",
       "    'PUB00019326': {'PMID': 8404842,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '10',\n",
       "     'year': 1993,\n",
       "     'title': 'Olive: a key gene required for chlorophyll biosynthesis in Antirrhinum majus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3711-9',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Hudson A', 'Carpenter R', 'Doyle S', 'Coen ES.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8404842&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 88,\n",
       "    'interactions': 0,\n",
       "    'matches': 22045,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18471,\n",
       "    'proteomes': 3300,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11499, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 14501},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20153,\n",
       "    'alignment:full': 4985,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF01820': {'metadata': {'accession': 'PF01820',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011127',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'D-ala D-ala ligase N-terminus',\n",
       "    'short': 'Dala_Dala_lig_N'},\n",
       "   'description': ['<p>This family represents the N-terminal region of the D-alanine--D-alanine ligase enzyme EC:6.3.2.4 which is thought to be involved in substrate binding [PMID:10908650]. D-Alanine is one of the central molecules of the cross-linking step of peptidoglycan assembly. There are three enzymes involved in the D-alanine branch of peptidoglycan biosynthesis: the pyridoxal phosphate-dependent D-alanine racemase (Alr), the ATP-dependent D-alanine:D-alanine ligase (Ddl), and the ATP-dependent D-alanine:D-alanine-adding enzyme (MurF) [PMID:12499203]. This domain is structurally related to the PreATP-grasp domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000444': {'PMID': 9054558,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '9',\n",
       "     'year': 1997,\n",
       "     'title': 'D-alanine:D-alanine ligase: phosphonate and phosphinate intermediates with wild type and the Y216F mutant.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2531-8',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Fan C', 'Park IS', 'Walsh CT', 'Knox JR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi962431t'},\n",
       "    'PUB00014287': {'PMID': 12499203,\n",
       "     'ISBN': None,\n",
       "     'volume': '47',\n",
       "     'issue': '1',\n",
       "     'year': 2003,\n",
       "     'title': 'Roles of Mycobacterium smegmatis D-alanine:D-alanine ligase and D-alanine racemase in the mechanisms of action of and resistance to the peptidoglycan inhibitor D-cycloserine.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '283-91',\n",
       "     'medline_journal': 'Antimicrob Agents Chemother',\n",
       "     'ISO_journal': 'Antimicrob. Agents Chemother.',\n",
       "     'authors': ['Feng Z', 'Barletta RG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/AAC.47.1.283-291.2003'},\n",
       "    'PUB00014326': {'PMID': 10908650,\n",
       "     'ISBN': None,\n",
       "     'volume': '97',\n",
       "     'issue': '16',\n",
       "     'year': 2000,\n",
       "     'title': 'The molecular basis of vancomycin resistance in clinically relevant Enterococci: crystal structure of D-alanyl-D-lactate ligase (VanA).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8921-5',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Roper DI', 'Huyton T', 'Vagin A', 'Dodson G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.150116497'}},\n",
       "   'set_info': {'accession': 'CL0483', 'name': 'PreATP-grasp'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 93,\n",
       "    'interactions': 0,\n",
       "    'matches': 53202,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52357,\n",
       "    'proteomes': 8437,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 45352, 'rosettafold': 0},\n",
       "    'structures': 56,\n",
       "    'taxa': 35911},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49517,\n",
       "    'alignment:full': 9608,\n",
       "    'alignment:seed': 508},\n",
       "   'cross_references': {}}},\n",
       " 'PF17963': {'metadata': {'accession': 'PF17963',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial Ig domain', 'short': 'Big_9'},\n",
       "   'description': ['<p>This entry represents a wide variety of bacterial Ig domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11888,\n",
       "    'interactions': 0,\n",
       "    'matches': 94889,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34818,\n",
       "    'proteomes': 3294,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15141, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 15378},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 83666,\n",
       "    'alignment:full': 21315,\n",
       "    'alignment:seed': 371},\n",
       "   'cross_references': {}}},\n",
       " 'PF01634': {'metadata': {'accession': 'PF01634',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013820',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP phosphoribosyltransferase', 'short': 'HisG'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0177', 'name': 'PBP'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 40492,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40294,\n",
       "    'proteomes': 8979,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 34647, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 36169},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36683,\n",
       "    'alignment:full': 8518,\n",
       "    'alignment:seed': 625},\n",
       "   'cross_references': {}}},\n",
       " 'PF02405': {'metadata': {'accession': 'PF02405',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR030802',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Permease MlaE', 'short': 'MlaE'},\n",
       "   'description': ['<p>MlaE is a permease which in E. coli is a component of the Mla pathway, an ABC transport system that functions to maintain the asymmetry of the outer membrane [PMID:19383799]. In Swiss:Q7DD59 it is involved in L-glutamate import into the cell [PMID:16495545]. In Swiss:Q8L4R0 it is involved in lipid transfer within the cell [PMID:16495545].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 54500,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54434,\n",
       "    'proteomes': 5564,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 47058, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 25209},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49936,\n",
       "    'alignment:full': 10360,\n",
       "    'alignment:seed': 618},\n",
       "   'cross_references': {}}},\n",
       " 'PF17912': {'metadata': {'accession': 'PF17912',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040582',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MalK OB fold domain', 'short': 'OB_MalK'},\n",
       "   'description': ['<p>This entry corresponds to one of two OB-fold domains found in the MalK transport protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 41,\n",
       "    'interactions': 0,\n",
       "    'matches': 55334,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55334,\n",
       "    'proteomes': 5143,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 49436, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 21984},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54747,\n",
       "    'alignment:full': 10520,\n",
       "    'alignment:seed': 262},\n",
       "   'cross_references': {}}},\n",
       " 'PF07264': {'metadata': {'accession': 'PF07264',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Etoposide-induced protein 2.4 (EI24)', 'short': 'EI24'},\n",
       "   'description': ['<p>This family contains a number of eukaryotic etoposide-induced 2.4 (EI24) proteins approximately 350 residues long as well as bacterial CysZ proteins (formerly known as DUF540). In cells treated with the cytotoxic drug etoposide, EI24 is induced by p53 [PMID:8649819]. It has been suggested to play an important role in negative cell growth control [PMID:10594026].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00013075': {'PMID': 8649819,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '6',\n",
       "     'year': 1996,\n",
       "     'title': 'Identification and cloning of EI24, a gene induced by p53 in etoposide-treated cells.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1181-7',\n",
       "     'medline_journal': 'Oncogene',\n",
       "     'ISO_journal': 'Oncogene',\n",
       "     'authors': ['Lehar SM',\n",
       "      'Nacht M',\n",
       "      'Jacks T',\n",
       "      'Vater CA',\n",
       "      'Chittenden T',\n",
       "      'Guild BC.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00013076': {'PMID': 10594026,\n",
       "     'ISBN': None,\n",
       "     'volume': '20',\n",
       "     'issue': '1',\n",
       "     'year': 2000,\n",
       "     'title': 'ei24, a p53 response gene involved in growth suppression and apoptosis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '233-41',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Gu Z', 'Flemington C', 'Chittenden T', 'Zambetti GP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/MCB.20.19.7109-7120.2000'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 89,\n",
       "    'interactions': 0,\n",
       "    'matches': 21222,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21163,\n",
       "    'proteomes': 5280,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18311, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 21176},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19669,\n",
       "    'alignment:full': 5739,\n",
       "    'alignment:seed': 262},\n",
       "   'cross_references': {}}},\n",
       " 'PF17852': {'metadata': {'accession': 'PF17852',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041466',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dynein heavy chain AAA lid domain',\n",
       "    'short': 'Dynein_AAA_lid'},\n",
       "   'description': ['<p>This entry corresponds to the extension domain of AAA domain 5 in the dynein heavy chain [PMID:22398446]. This domain is composed of 8 alpha helices [PMID:22398446].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1399,\n",
       "    'interactions': 0,\n",
       "    'matches': 21119,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21086,\n",
       "    'proteomes': 1861,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 1082, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 6888},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14485,\n",
       "    'alignment:full': 9073,\n",
       "    'alignment:seed': 230},\n",
       "   'cross_references': {}}},\n",
       " 'PF13408': {'metadata': {'accession': 'PF13408',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025827',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Recombinase zinc beta ribbon domain',\n",
       "    'short': 'Zn_ribbon_recom'},\n",
       "   'description': ['<p>This short bacterial protein contains a zinc ribbon domain that is likely to be DNA-binding. This domain is found in site specific recombinase proteins. This family appears most closely related to Pfam:PF04606.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0167', 'name': 'Zn_Beta_Ribbon'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 221,\n",
       "    'interactions': 0,\n",
       "    'matches': 36591,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36467,\n",
       "    'proteomes': 3080,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31593, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 14218},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34586,\n",
       "    'alignment:full': 8172,\n",
       "    'alignment:seed': 270},\n",
       "   'cross_references': {}}},\n",
       " 'PF16353': {'metadata': {'accession': 'PF16353',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032312',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Beta-galactosidase, domain 4', 'short': 'LacZ_4'},\n",
       "   'description': ['<p>This entry represents domain 4 found in beta-galactosidase [PMID:8008071] and it is organised in a jelly-roll type barrel (Rutkiewicz-Krotewicz M. et al. Crystals 2018, 8(1), 13, https://doi.org/10.3390/cryst8010013).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 245,\n",
       "    'interactions': 0,\n",
       "    'matches': 18817,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18813,\n",
       "    'proteomes': 2735,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15426, 'rosettafold': 0},\n",
       "    'structures': 92,\n",
       "    'taxa': 11079},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18413,\n",
       "    'alignment:full': 4160,\n",
       "    'alignment:seed': 456},\n",
       "   'cross_references': {}}},\n",
       " 'PF01250': {'metadata': {'accession': 'PF01250',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000529',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S6', 'short': 'Ribosomal_S6'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001244': {'PMID': 8137808,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '6',\n",
       "     'year': 1994,\n",
       "     'title': 'Crystal structure of the ribosomal protein S6 from Thermus thermophilus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1249-54',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Lindahl M',\n",
       "      'Svensson LA',\n",
       "      'Liljas A',\n",
       "      'Sedelnikova SE',\n",
       "      'Eliseikina IA',\n",
       "      'Fomenkova NP',\n",
       "      'Nevskaya N',\n",
       "      'Nikonov SV',\n",
       "      'Garber MB',\n",
       "      'Muranova TA.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8137808&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 85,\n",
       "    'interactions': 0,\n",
       "    'matches': 45937,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45932,\n",
       "    'proteomes': 10501,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39399, 'rosettafold': 0},\n",
       "    'structures': 907,\n",
       "    'taxa': 44465},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41131,\n",
       "    'alignment:full': 9884,\n",
       "    'alignment:seed': 719},\n",
       "   'cross_references': {}}},\n",
       " 'PF00343': {'metadata': {'accession': 'PF00343',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000811',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Carbohydrate phosphorylase', 'short': 'Phosphorylase'},\n",
       "   'description': ['<p>The members of this family catalyse the formation of  glucose 1-phosphate from one of the following polyglucoses;   glycogen, starch, glucan or maltodextrin.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004996': {'PMID': 1304390,\n",
       "     'ISBN': None,\n",
       "     'volume': '1',\n",
       "     'issue': '9',\n",
       "     'year': 1992,\n",
       "     'title': \"Control of phosphorylase b conformation by a modified cofactor: crystallographic studies on R-state glycogen phosphorylase reconstituted with pyridoxal 5'-diphosphate.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '1112-22',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Leonidas DD',\n",
       "      'Oikonomakos NG',\n",
       "      'Papageorgiou AC',\n",
       "      'Acharya KR',\n",
       "      'Barford D',\n",
       "      'Johnson LN.'],\n",
       "     'DOI_URL': 'http://www.proteinscience.org/cgi/content/abstract/1/9/1112'}},\n",
       "   'set_info': {'accession': 'CL0113', 'name': 'GT-B'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 162,\n",
       "    'interactions': 0,\n",
       "    'matches': 50308,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48014,\n",
       "    'proteomes': 7425,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39353, 'rosettafold': 0},\n",
       "    'structures': 266,\n",
       "    'taxa': 31864},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44150,\n",
       "    'alignment:full': 11375,\n",
       "    'alignment:seed': 485},\n",
       "   'cross_references': {}}},\n",
       " 'PF07811': {'metadata': {'accession': 'PF07811',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012495',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TadE-like protein', 'short': 'TadE'},\n",
       "   'description': ['<p>The members of this family are similar to a region of  the protein product of the bacterial tadE locus (Swiss:Q9S4A6). In various bacterial species, the tad locus is closely  linked to flp-like genes, which encode proteins required  for the production of pili involved in adherence to  surfaces [PMID:11553455]. It is thought that the tad loci encode  proteins that act to assemble or export an Flp pilus in  various bacteria [PMID:11553455]. All tad loci but TadA have putative transmembrane regions [PMID:11553455], and in fact the region in  question is this family has a high proportion of hydrophobic  amino acid residues.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016542': {'PMID': 11553455,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '9',\n",
       "     'year': 2001,\n",
       "     'title': 'Genes for tight adherence of Actinobacillus actinomycetemcomitans: from plaque to plague to pond scum.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '429-37',\n",
       "     'medline_journal': 'Trends Microbiol',\n",
       "     'ISO_journal': 'Trends Microbiol.',\n",
       "     'authors': ['Kachlany SC',\n",
       "      'Planet PJ',\n",
       "      'DeSalle R',\n",
       "      'Fine DH',\n",
       "      'Figurski DH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0966-842X(01)02161-8'}},\n",
       "   'set_info': {'accession': 'CL0496', 'name': 'Tad-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 36446,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36436,\n",
       "    'proteomes': 3603,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31651, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13879},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33615,\n",
       "    'alignment:full': 7773,\n",
       "    'alignment:seed': 349},\n",
       "   'cross_references': {}}},\n",
       " 'PF00557': {'metadata': {'accession': 'PF00557',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000994',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Metallopeptidase family M24', 'short': 'Peptidase_M24'},\n",
       "   'description': ['<p>This family contains metallopeptidases.  It also contains non-peptidase homologues such as the N terminal domain of Spt16 which is a histone H3-H4 binding module [PMID:18579787].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000379': {'PMID': 8471602,\n",
       "     'ISBN': None,\n",
       "     'volume': '32',\n",
       "     'issue': '15',\n",
       "     'year': 1993,\n",
       "     'title': 'Structure of the cobalt-dependent methionine aminopeptidase from Escherichia coli: a new type of proteolytic enzyme.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3907-12',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Roderick SL', 'Matthews BW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00066a009'},\n",
       "    'PUB00003579': {'PMID': 7674922,\n",
       "     'ISBN': None,\n",
       "     'volume': '248',\n",
       "     'issue': None,\n",
       "     'year': 1995,\n",
       "     'title': 'Evolutionary families of metallopeptidases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '183-228',\n",
       "     'medline_journal': 'Methods Enzymol',\n",
       "     'ISO_journal': 'Meth. Enzymol.',\n",
       "     'authors': ['Rawlings ND', 'Barrett AJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0076-6879(95)48015-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 837,\n",
       "    'interactions': 0,\n",
       "    'matches': 229173,\n",
       "    'pathways': 0,\n",
       "    'proteins': 227876,\n",
       "    'proteomes': 11318,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 194684, 'rosettafold': 0},\n",
       "    'structures': 306,\n",
       "    'taxa': 49058},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 207692,\n",
       "    'alignment:full': 52724,\n",
       "    'alignment:seed': 382},\n",
       "   'cross_references': {}}},\n",
       " 'PF12661': {'metadata': {'accession': 'PF12661',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013032',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Human growth factor-like EGF', 'short': 'hEGF'},\n",
       "   'description': [\"<p>hEGF, or human growth factor-like EGF, domains have six conserved residues disulfide-bonded into the characteristic 'ababcc' pattern. They are involved in growth and proliferation of cells, in proteins of the Notch/Delta pathway, neurogulin and selectins. hEGFs are also found in mosaic proteins with four-disulfide laminin EGFs such as aggrecan and perlecan.  The core fold of the EGF domain consists of two small beta-hairpins packed against each other. Two major structural variants have been identified based on the structural context of the C-terminal Cys residue of disulfide 'c' in the C-terminal hairpin: hEGFs and cEGFs. In hEGFs the C-terminal thiol resides in the beta-turn, resulting in shorter loop-lengths between the Cys residues of disulfide 'c', typically C[8-9]XC.  These shorter loop-lengths are also typical of the four-disulfide EGF domains, laminin ad integrin.  Tandem hEGF domains have six linking residues between terminal cysteines of adjacent domains. hEGF domains may or may not bind calcium in the linker region. hEGF domains with the consensus motif CXD4X[F,Y]XCXC are hydroxylated exclusively in the Asp residue.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0001', 'name': 'EGF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 13922,\n",
       "    'interactions': 0,\n",
       "    'matches': 80373,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37492,\n",
       "    'proteomes': 956,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14178, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 4975},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52052,\n",
       "    'alignment:full': 32688,\n",
       "    'alignment:seed': 335},\n",
       "   'cross_references': {}}},\n",
       " 'PF00936': {'metadata': {'accession': 'PF00936',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000249',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'BMC domain', 'short': 'BMC'},\n",
       "   'description': ['<p>Bacterial microcompartments are primitive organelles composed entirely of protein subunits. The prototypical bacterial microcompartment is the carboxysome, a protein shell for sequestering carbon fixation reactions. These proteins for hexameric structure [PMID:16081736].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00021036': {'PMID': 16081736,\n",
       "     'ISBN': None,\n",
       "     'volume': '309',\n",
       "     'issue': '5736',\n",
       "     'year': 2005,\n",
       "     'title': 'Protein structures forming the shell of primitive bacterial organelles.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '936-8',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Kerfeld CA',\n",
       "      'Sawaya MR',\n",
       "      'Tanaka S',\n",
       "      'Nguyen CV',\n",
       "      'Phillips M',\n",
       "      'Beeby M',\n",
       "      'Yeates TO.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.1113397'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 35723,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29455,\n",
       "    'proteomes': 1128,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25430, 'rosettafold': 0},\n",
       "    'structures': 99,\n",
       "    'taxa': 6526},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33777,\n",
       "    'alignment:full': 6224,\n",
       "    'alignment:seed': 380},\n",
       "   'cross_references': {}}},\n",
       " 'PF00709': {'metadata': {'accession': 'PF00709',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001114',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Adenylosuccinate synthetase', 'short': 'Adenylsucc_synt'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 154,\n",
       "    'interactions': 0,\n",
       "    'matches': 58952,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56883,\n",
       "    'proteomes': 10773,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 47664, 'rosettafold': 0},\n",
       "    'structures': 59,\n",
       "    'taxa': 44891},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52273,\n",
       "    'alignment:full': 12090,\n",
       "    'alignment:seed': 623},\n",
       "   'cross_references': {}}},\n",
       " 'PF08486': {'metadata': {'accession': 'PF08486',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013693',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Stage II sporulation protein', 'short': 'SpoIID'},\n",
       "   'description': ['<p>This domain is found in the stage II sporulation protein SpoIID. SpoIID is necessary for membrane migration as well as for some of the earlier steps in engulfment during bacterial endospore formation [PMID:12502745].  The domain is also found in amidase enhancer proteins.  Amidases, like SpoIID, are cell wall hydrolases [PMID:10961456].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020841': {'PMID': 12502745,\n",
       "     'ISBN': None,\n",
       "     'volume': '16',\n",
       "     'issue': '24',\n",
       "     'year': 2002,\n",
       "     'title': 'A cytoskeleton-like role for the bacterial cell wall during engulfment of the Bacillus subtilis forespore.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3253-64',\n",
       "     'medline_journal': 'Genes Dev',\n",
       "     'ISO_journal': 'Genes Dev.',\n",
       "     'authors': ['Abanes-De Mello A', 'Sun YL', 'Aung S', 'Pogliano K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1101/gad.1039902'},\n",
       "    'PUB00020927': {'PMID': 10961456,\n",
       "     'ISBN': None,\n",
       "     'volume': '151',\n",
       "     'issue': '6',\n",
       "     'year': 2000,\n",
       "     'title': 'Biological roles of two new murein hydrolases of Streptococcus pneumoniae representing examples of module shuffling.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '437-43',\n",
       "     'medline_journal': 'Res Microbiol',\n",
       "     'ISO_journal': 'Res. Microbiol.',\n",
       "     'authors': ['Lopez R',\n",
       "      'Gonzalez MP',\n",
       "      'Garcia E',\n",
       "      'Garcia JL',\n",
       "      'Garcia P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0923-2508(00)00172-8'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 341,\n",
       "    'interactions': 0,\n",
       "    'matches': 20170,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20158,\n",
       "    'proteomes': 2351,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16520, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 11543},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17694,\n",
       "    'alignment:full': 3352,\n",
       "    'alignment:seed': 465},\n",
       "   'cross_references': {}}},\n",
       " 'PF17517': {'metadata': {'accession': 'PF17517',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR035234',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'IgGFc binding protein', 'short': 'IgGFc_binding'},\n",
       "   'description': ['<p>This domain is found at the N terminal of Swiss:Q9Y6R7 and has been shown to confer IgG Fc binding activity [PMID:9182547]. It may play a role in immune protection and inflammation in the intestines of primates [PMID:9182547].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 869,\n",
       "    'interactions': 0,\n",
       "    'matches': 6538,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6255,\n",
       "    'proteomes': 601,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 3206, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 3031},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4667,\n",
       "    'alignment:full': 1763,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF00013': {'metadata': {'accession': 'PF00013',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004088',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'KH domain', 'short': 'KH_1'},\n",
       "   'description': ['<p>KH motifs bind RNA in vitro. Autoantibodies to Nova, a KH domain protein, cause paraneoplastic opsoclonus ataxia.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000930': {'PMID': 8612276,\n",
       "     'ISBN': None,\n",
       "     'volume': '85',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'Three-dimensional structure and stability of the KH domain: molecular insights into the fragile X syndrome.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '237-45',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Musco G',\n",
       "      'Stier G',\n",
       "      'Joseph C',\n",
       "      'Castiglione Morelli MA',\n",
       "      'Nilges M',\n",
       "      'Gibson TJ',\n",
       "      'Pastore A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)81100-9'},\n",
       "    'PUB00005188': {'PMID': 8036511,\n",
       "     'ISBN': None,\n",
       "     'volume': '265',\n",
       "     'issue': '5172',\n",
       "     'year': 1994,\n",
       "     'title': 'Conserved structures and diversity of functions of RNA-binding proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '615-21',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Burd CG', 'Dreyfuss G.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/265/5172/615'}},\n",
       "   'set_info': {'accession': 'CL0007', 'name': 'KH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1542,\n",
       "    'interactions': 0,\n",
       "    'matches': 274324,\n",
       "    'pathways': 0,\n",
       "    'proteins': 151726,\n",
       "    'proteomes': 11054,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 118803, 'rosettafold': 0},\n",
       "    'structures': 99,\n",
       "    'taxa': 47476},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 216351,\n",
       "    'alignment:full': 109086,\n",
       "    'alignment:seed': 783},\n",
       "   'cross_references': {}}},\n",
       " 'PF00366': {'metadata': {'accession': 'PF00366',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000266',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S17', 'short': 'Ribosomal_S17'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 138,\n",
       "    'interactions': 0,\n",
       "    'matches': 52855,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52841,\n",
       "    'proteomes': 11089,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 45547, 'rosettafold': 0},\n",
       "    'structures': 1095,\n",
       "    'taxa': 48464},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46848,\n",
       "    'alignment:full': 12173,\n",
       "    'alignment:seed': 531},\n",
       "   'cross_references': {}}},\n",
       " 'PF00475': {'metadata': {'accession': 'PF00475',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000807',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Imidazoleglycerol-phosphate dehydratase',\n",
       "    'short': 'IGPD'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 39661,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39594,\n",
       "    'proteomes': 8962,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 34221, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 36151},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36305,\n",
       "    'alignment:full': 8422,\n",
       "    'alignment:seed': 487},\n",
       "   'cross_references': {}}},\n",
       " 'PF01139': {'metadata': {'accession': 'PF01139',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001233',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'tRNA-splicing ligase RtcB', 'short': 'RtcB'},\n",
       "   'description': [\"<p>This family of RNA ligases (EC:6.5.1.3) join 2',3'-cyclic phosphate and 5'-OH ends. They catalyse the splicing of tRNA and may also participate in tRNA repair and recovery from stress-induced RNA damage [1-3].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 34331,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29238,\n",
       "    'proteomes': 4737,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 24601, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 22772},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30875,\n",
       "    'alignment:full': 6107,\n",
       "    'alignment:seed': 247},\n",
       "   'cross_references': {}}},\n",
       " 'PF02742': {'metadata': {'accession': 'PF02742',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001367',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Iron dependent repressor, metal binding and dimerisation domain',\n",
       "    'short': 'Fe_dep_repr_C'},\n",
       "   'description': ['<p>This family includes the Diphtheria toxin repressor [PMID:7568230]. It acts as an iron-binding repressor of diphtheria toxin gene expression and may serve as a global regulator of gene expression. DTXR comprises an N-terminal DNA-binding domain, an interface domain (which contains two metal-binding sites) and a third, very flexible C-terminal domain. The second domain is responsible for dimerization and metal binding. Binding of DTXR to Tox operator requires a divalent metal ion such as cobalt, ferric, manganese and nickel whereas zinc shows weak activation [PMID:7743135]. This domain can also bind Cd(II), Ca(II) and Cu(II) (Matilla et. al., FEMS Microbiology Reviews, fuab043, 45, 2021, 1. https://doi.org/10.1093/femsre/fuab043).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004874': {'PMID': 7568230,\n",
       "     'ISBN': None,\n",
       "     'volume': '92',\n",
       "     'issue': '21',\n",
       "     'year': 1995,\n",
       "     'title': 'Structures of the apo- and the metal ion-activated forms of the diphtheria tox repressor from Corynebacterium diphtheriae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9843-50',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Schiering N',\n",
       "      'Tao X',\n",
       "      'Zeng H',\n",
       "      'Murphy JR',\n",
       "      'Petsko GA',\n",
       "      'Ringe D.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=7568230'},\n",
       "    'PUB00005267': {'PMID': 7743135,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '1',\n",
       "     'year': 1995,\n",
       "     'title': 'Three-dimensional structure of the diphtheria toxin repressor in complex with divalent cation co-repressors.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '87-100',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Qiu X',\n",
       "      'Verlinde CL',\n",
       "      'Zhang S',\n",
       "      'Schmitt MP',\n",
       "      'Holmes RK',\n",
       "      'Hol WG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00137-X'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 73,\n",
       "    'interactions': 0,\n",
       "    'matches': 36488,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36483,\n",
       "    'proteomes': 5319,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31163, 'rosettafold': 0},\n",
       "    'structures': 66,\n",
       "    'taxa': 21826},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32429,\n",
       "    'alignment:full': 6983,\n",
       "    'alignment:seed': 379},\n",
       "   'cross_references': {}}},\n",
       " 'PF02787': {'metadata': {'accession': 'PF02787',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005480',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Carbamoyl-phosphate synthetase large chain, oligomerisation domain',\n",
       "    'short': 'CPSase_L_D3'},\n",
       "   'description': ['<p>Carbamoyl-phosphate synthase catalyses the ATP-dependent synthesis of carbamyl-phosphate from glutamine or ammonia and bicarbonate. The carbamoyl-phosphate synthase (CPS) enzyme in prokaryotes is a  heterodimer of a small and large chain.  </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007868': {'PMID': 10089390,\n",
       "     'ISBN': None,\n",
       "     'volume': '55',\n",
       "     'issue': 'Pt 1',\n",
       "     'year': 1999,\n",
       "     'title': 'The structure of carbamoyl phosphate synthetase determined to 2.1 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8-24',\n",
       "     'medline_journal': 'Acta Crystallogr D Biol Crystallogr',\n",
       "     'ISO_journal': 'Acta Crystallogr. D Biol. Crystallogr.',\n",
       "     'authors': ['Thoden JB',\n",
       "      'Raushel FM',\n",
       "      'Benning MM',\n",
       "      'Rayment I',\n",
       "      'Holden HM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1107/S0907444998006234'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 233,\n",
       "    'interactions': 0,\n",
       "    'matches': 67038,\n",
       "    'pathways': 0,\n",
       "    'proteins': 67034,\n",
       "    'proteomes': 10285,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 50869, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 62685},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61910,\n",
       "    'alignment:full': 11547,\n",
       "    'alignment:seed': 923},\n",
       "   'cross_references': {}}},\n",
       " 'PF01510': {'metadata': {'accession': 'PF01510',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002502',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'N-acetylmuramoyl-L-alanine amidase',\n",
       "    'short': 'Amidase_2'},\n",
       "   'description': ['<p>This family includes zinc amidases that have N-acetylmuramoyl-L-alanine amidase activity EC:3.5.1.28. This enzyme domain cleaves the amide bond between N-acetylmuramoyl and L-amino acids in bacterial cell walls (preferentially: D-lactyl-L-Ala).  The structure is known for the bacteriophage T7 structure and shows that two of the conserved histidines are zinc binding.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001806': {'PMID': 1677905,\n",
       "     'ISBN': None,\n",
       "     'volume': '102',\n",
       "     'issue': '1',\n",
       "     'year': 1991,\n",
       "     'title': 'Sequence analysis of a Staphylococcus aureus gene encoding a peptidoglycan hydrolase activity.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '105-9',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Wang X', 'Wilkinson BJ', 'Jayaswal RK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(91)90547-O'},\n",
       "    'PUB00003727': {'PMID': 3070348,\n",
       "     'ISBN': None,\n",
       "     'volume': '214',\n",
       "     'issue': '2',\n",
       "     'year': 1988,\n",
       "     'title': 'Cloning, sequencing and expression of a Bacillus bacteriolytic enzyme in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '241-8',\n",
       "     'medline_journal': 'Mol Gen Genet',\n",
       "     'ISO_journal': 'Mol. Gen. Genet.',\n",
       "     'authors': ['Potvin C',\n",
       "      'Leclerc D',\n",
       "      'Tremblay G',\n",
       "      'Asselin A',\n",
       "      'Bellemare G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/BF00337717'},\n",
       "    'PUB00004839': {'PMID': 8171031,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '9',\n",
       "     'year': 1994,\n",
       "     'title': 'The structure of bacteriophage T7 lysozyme, a zinc amidase and an inhibitor of T7 RNA polymerase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4034-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Cheng X', 'Zhang X', 'Pflugrath JW', 'Studier FW.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8171031&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 801,\n",
       "    'interactions': 0,\n",
       "    'matches': 67672,\n",
       "    'pathways': 0,\n",
       "    'proteins': 66994,\n",
       "    'proteomes': 7891,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 55426, 'rosettafold': 0},\n",
       "    'structures': 96,\n",
       "    'taxa': 34177},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 64129,\n",
       "    'alignment:full': 13872,\n",
       "    'alignment:seed': 523},\n",
       "   'cross_references': {}}},\n",
       " 'PF02618': {'metadata': {'accession': 'PF02618',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003770',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YceG-like family', 'short': 'YceG'},\n",
       "   'description': ['<p>This family of proteins is found in bacteria. Proteins in this family are typically between 332 and 389 amino acids in length. This family was previously incorrectly annotated and names as aminodeoxychorismate lyase. The structure of Swiss:P28306 was solved by X-ray crystallography.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016656': {'PMID': 11011151,\n",
       "     'ISBN': None,\n",
       "     'volume': '128',\n",
       "     'issue': '4',\n",
       "     'year': 2000,\n",
       "     'title': 'Stereochemistry of the transamination reaction catalyzed by aminodeoxychorismate lyase from Escherichia coli: close relationship between fold type and stereochemistry.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '679-86',\n",
       "     'medline_journal': 'J Biochem',\n",
       "     'ISO_journal': 'J. Biochem.',\n",
       "     'authors': ['Jhee KH',\n",
       "      'Yoshimura T',\n",
       "      'Miles EW',\n",
       "      'Takeda S',\n",
       "      'Miyahara I',\n",
       "      'Hirotsu K',\n",
       "      'Soda K',\n",
       "      'Kawata Y',\n",
       "      'Esaki N.'],\n",
       "     'DOI_URL': 'http://jb.oxfordjournals.org/cgi/content/abstract/128/4/679'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 56,\n",
       "    'interactions': 0,\n",
       "    'matches': 44867,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44732,\n",
       "    'proteomes': 7894,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38259, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 33676},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41163,\n",
       "    'alignment:full': 7719,\n",
       "    'alignment:seed': 761},\n",
       "   'cross_references': {}}},\n",
       " 'PF01379': {'metadata': {'accession': 'PF01379',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022417',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Porphobilinogen deaminase, dipyromethane cofactor binding domain',\n",
       "    'short': 'Porphobil_deam'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0177', 'name': 'PBP'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 126,\n",
       "    'interactions': 0,\n",
       "    'matches': 43551,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43007,\n",
       "    'proteomes': 9540,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 36863, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 38402},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39653,\n",
       "    'alignment:full': 9712,\n",
       "    'alignment:seed': 666},\n",
       "   'cross_references': {}}},\n",
       " 'PF02491': {'metadata': {'accession': 'PF02491',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003494',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SHS2 domain inserted in FTSA', 'short': 'SHS2_FTSA'},\n",
       "   'description': ['<p>FtsA is essential for bacterial cell division, and co-localises to  the septal ring with FtsZ. The SHS2 domain is inserted in to the RNAseH fold of FtsA [PMID:15281131], and is involved in protein-protein  interaction [PMID:9352931].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009447': {'PMID': 9352931,\n",
       "     'ISBN': None,\n",
       "     'volume': '179',\n",
       "     'issue': '21',\n",
       "     'year': 1997,\n",
       "     'title': 'Interactions between heterologous FtsA and FtsZ proteins at the FtsZ ring.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6788-97',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Ma X',\n",
       "      'Sun Q',\n",
       "      'Wang R',\n",
       "      'Singh G',\n",
       "      'Jonietz EL',\n",
       "      'Margolin W.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9352931&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0319', 'name': 'SHS2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 32839,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32837,\n",
       "    'proteomes': 5622,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28096, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 26601},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29893,\n",
       "    'alignment:full': 5252,\n",
       "    'alignment:seed': 284},\n",
       "   'cross_references': {}}},\n",
       " 'PF13472': {'metadata': {'accession': 'PF13472',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013830',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GDSL-like Lipase/Acylhydrolase family',\n",
       "    'short': 'Lipase_GDSL_2'},\n",
       "   'description': ['<p>This family of presumed lipases and related enzymes are similar to Pfam:PF00657.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0264', 'name': 'SGNH_hydrolase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1892,\n",
       "    'interactions': 0,\n",
       "    'matches': 162871,\n",
       "    'pathways': 0,\n",
       "    'proteins': 161478,\n",
       "    'proteomes': 9929,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 135352, 'rosettafold': 0},\n",
       "    'structures': 96,\n",
       "    'taxa': 39849},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 146819,\n",
       "    'alignment:full': 40181,\n",
       "    'alignment:seed': 1323},\n",
       "   'cross_references': {}}},\n",
       " 'PF00162': {'metadata': {'accession': 'PF00162',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001576',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoglycerate kinase', 'short': 'PGK'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 208,\n",
       "    'interactions': 0,\n",
       "    'matches': 60941,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59315,\n",
       "    'proteomes': 11012,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 50322, 'rosettafold': 0},\n",
       "    'structures': 61,\n",
       "    'taxa': 47502},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54184,\n",
       "    'alignment:full': 11839,\n",
       "    'alignment:seed': 741},\n",
       "   'cross_references': {}}},\n",
       " 'PF04427': {'metadata': {'accession': 'PF04427',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007109',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Brix domain', 'short': 'Brix'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00018304': {'PMID': 11406393,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '6',\n",
       "     'year': 2001,\n",
       "     'title': 'The Brix domain protein family -- a key to the ribosomal biogenesis pathway?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '345-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Eisenhaber F', 'Wechselberger C', 'Kreil G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(01)01851-5'},\n",
       "    'PUB00018305': {'PMID': 11246005,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '3',\n",
       "     'year': 2001,\n",
       "     'title': 'The archaeal homolog of the Imp4 protein, a eukaryotic U3 snoRNP component.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '143-4',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Mayer C', 'Suck D', 'Poch O.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)01779-5'},\n",
       "    'PUB00019231': {'PMID': 11864607,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '2',\n",
       "     'year': 2002,\n",
       "     'title': 'Ssf1p prevents premature processing of an early pre-60S ribosomal particle.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '341-51',\n",
       "     'medline_journal': 'Mol Cell',\n",
       "     'ISO_journal': 'Mol. Cell',\n",
       "     'authors': ['Fatica A', 'Cronshaw AD', 'Dlakic M', 'Tollervey D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1097-2765(02)00458-6'},\n",
       "    'PUB00019232': {'PMID': 11864606,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '2',\n",
       "     'year': 2002,\n",
       "     'title': 'The sigma(70)-like motif: a eukaryotic RNA binding domain unique to a superfamily of proteins required for ribosome biogenesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '329-39',\n",
       "     'medline_journal': 'Mol Cell',\n",
       "     'ISO_journal': 'Mol. Cell',\n",
       "     'authors': ['Wehner KA', 'Baserga SJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1097-2765(02)00438-0'}},\n",
       "   'set_info': {'accession': 'CL0458', 'name': 'IIaaRS-ABD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 233,\n",
       "    'interactions': 0,\n",
       "    'matches': 17762,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17707,\n",
       "    'proteomes': 2176,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14668, 'rosettafold': 0},\n",
       "    'structures': 77,\n",
       "    'taxa': 8372},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14776,\n",
       "    'alignment:full': 8855,\n",
       "    'alignment:seed': 478},\n",
       "   'cross_references': {}}},\n",
       " 'PF04547': {'metadata': {'accession': 'PF04547',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007632',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Calcium-activated chloride channel',\n",
       "    'short': 'Anoctamin'},\n",
       "   'description': ['<p>The family carries eight putative transmembrane domains, and, although it has no similarity to other known channel proteins, it is clearly a calcium-activated ionic channel. It is expressed in various secretory epithelia, the retina and sensory neurons, and mediates receptor-activated chloride currents in diverse physiological processes [PMID:18724360].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0416', 'name': 'Anoctamin-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 299,\n",
       "    'interactions': 0,\n",
       "    'matches': 24640,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23535,\n",
       "    'proteomes': 2038,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16727, 'rosettafold': 0},\n",
       "    'structures': 48,\n",
       "    'taxa': 7428},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17455,\n",
       "    'alignment:full': 10660,\n",
       "    'alignment:seed': 307},\n",
       "   'cross_references': {}}},\n",
       " 'PF13490': {'metadata': {'accession': 'PF13490',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027383',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative zinc-finger', 'short': 'zf-HC2'},\n",
       "   'description': ['<p>This is a putative zinc-finger found in some anti-sigma factor proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0645', 'name': 'Anti-sigma_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 356,\n",
       "    'interactions': 0,\n",
       "    'matches': 47538,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47431,\n",
       "    'proteomes': 3955,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 40158, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 16350},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42246,\n",
       "    'alignment:full': 9414,\n",
       "    'alignment:seed': 572},\n",
       "   'cross_references': {}}},\n",
       " 'PF07500': {'metadata': {'accession': 'PF07500',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003618',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transcription factor S-II (TFIIS), central domain',\n",
       "    'short': 'TFIIS_M'},\n",
       "   'description': ['<p>Transcription elongation by RNA polymerase II is regulated by  the general elongation factor TFIIS. This factor stimulates  RNA polymerase II to transcribe through regions of DNA that  promote the formation of stalled ternary complexes. TFIIS is composed of three structural domains, termed I, II, and III.  The two C-terminal  domains (II and III), this domain and Pfam:PF01096  are required for transcription activity [PMID:8855225].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014288': {'PMID': 8855225,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '20',\n",
       "     'year': 1996,\n",
       "     'title': 'Elongation factor TFIIS contains three structural domains: solution structure of domain II.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10604-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Morin PE', 'Awrey DE', 'Edwards AM', 'Arrowsmith CH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.20.10604'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 185,\n",
       "    'interactions': 0,\n",
       "    'matches': 13027,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13009,\n",
       "    'proteomes': 2182,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8672, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 8004},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10137,\n",
       "    'alignment:full': 6041,\n",
       "    'alignment:seed': 273},\n",
       "   'cross_references': {}}},\n",
       " 'PF03883': {'metadata': {'accession': 'PF03883',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005583',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peroxide stress protein YaaA', 'short': 'H2O2_YaaD'},\n",
       "   'description': ['<p>YaaA is a key element of the stress response to H2O2. It acts by reducing the level of intracellular iron levels after peroxide stress, thereby attenuating the Fenton reaction and the DNA damage that this would cause [PMID:21378183]. The molecular mechanism of action is not known.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 36,\n",
       "    'interactions': 0,\n",
       "    'matches': 22905,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22893,\n",
       "    'proteomes': 4724,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20141, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 19630},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22195,\n",
       "    'alignment:full': 4383,\n",
       "    'alignment:seed': 291},\n",
       "   'cross_references': {}}},\n",
       " 'PF07525': {'metadata': {'accession': 'PF07525',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001496',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SOCS box', 'short': 'SOCS_box'},\n",
       "   'description': ['<p>The SOCS box acts as a bridge between specific substrate- binding domains and more generic proteins that comprise a large family of E3 ubiquitin protein ligases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020353': {'PMID': 12076535,\n",
       "     'ISBN': None,\n",
       "     'volume': '27',\n",
       "     'issue': '5',\n",
       "     'year': 2002,\n",
       "     'title': 'The SOCS box: a tale of destruction and degradation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '235-41',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Kile BT',\n",
       "      'Schulman BA',\n",
       "      'Alexander WS',\n",
       "      'Nicola NA',\n",
       "      'Martin HM',\n",
       "      'Hilton DJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(02)02085-6'}},\n",
       "   'set_info': {'accession': 'CL0642', 'name': 'SOCS_box'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 481,\n",
       "    'interactions': 0,\n",
       "    'matches': 29610,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29590,\n",
       "    'proteomes': 913,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 22715, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 3763},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22121,\n",
       "    'alignment:full': 11995,\n",
       "    'alignment:seed': 365},\n",
       "   'cross_references': {}}},\n",
       " 'PF00499': {'metadata': {'accession': 'PF00499',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001457',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NADH-ubiquinone/plastoquinone oxidoreductase chain 6',\n",
       "    'short': 'Oxidored_q3'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 60,\n",
       "    'interactions': 0,\n",
       "    'matches': 69749,\n",
       "    'pathways': 0,\n",
       "    'proteins': 69734,\n",
       "    'proteomes': 6236,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 60166, 'rosettafold': 0},\n",
       "    'structures': 220,\n",
       "    'taxa': 69873},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 62042,\n",
       "    'alignment:full': 6979,\n",
       "    'alignment:seed': 661},\n",
       "   'cross_references': {}}},\n",
       " 'PF01925': {'metadata': {'accession': 'PF01925',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002781',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sulfite exporter TauE/SafE', 'short': 'TauE'},\n",
       "   'description': ['<p>This is a family of integral membrane proteins where the alignment appears to contain two duplicated modules of three transmembrane helices. The proteins are involved in the transport of anions across the cytoplasmic membrane [PMID:17768248] during taurine metabolism as an exporter of sulfoacetate [PMID:18506422]. This family used to be known as DUF81.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0292', 'name': 'LysE'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 209,\n",
       "    'interactions': 0,\n",
       "    'matches': 155222,\n",
       "    'pathways': 0,\n",
       "    'proteins': 151205,\n",
       "    'proteomes': 8548,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 130488, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 36627},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 144408,\n",
       "    'alignment:full': 32027,\n",
       "    'alignment:seed': 536},\n",
       "   'cross_references': {}}},\n",
       " 'PF13732': {'metadata': {'accession': 'PF13732',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025302',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4162)',\n",
       "    'short': 'DUF4162'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of bacterial ABC transporter proteins. The function is not known.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 24,\n",
       "    'interactions': 0,\n",
       "    'matches': 37781,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37778,\n",
       "    'proteomes': 4286,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32418, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 17956},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33770,\n",
       "    'alignment:full': 8114,\n",
       "    'alignment:seed': 241},\n",
       "   'cross_references': {}}},\n",
       " 'PF04015': {'metadata': {'accession': 'PF04015',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007160',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF362)', 'short': 'DUF362'},\n",
       "   'description': ['<p>Domain that is sometimes present in iron-sulphur proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0471', 'name': 'DUF362'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 142,\n",
       "    'interactions': 0,\n",
       "    'matches': 23521,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23270,\n",
       "    'proteomes': 1391,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18851, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 8056},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18760,\n",
       "    'alignment:full': 2720,\n",
       "    'alignment:seed': 378},\n",
       "   'cross_references': {}}},\n",
       " 'PF04519': {'metadata': {'accession': 'PF04519',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007607',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polymer-forming cytoskeletal', 'short': 'Bactofilin'},\n",
       "   'description': ['<p>This is a family of bactofilins, a functionally diverse class of cytoskeletal, polymer-forming, proteins that is widely conserved among bacteria. In the example species C. crescentus, two bactofilins assemble into a membrane-associated laminar structure that shows cell-cycle-dependent polar localisation and acts as a platform for the recruitment of a cell wall biosynthetic enzyme involved in polar morphogenesis. Bactofilins display distinct subcellular distributions and dynamics in different bacterial species, suggesting that they are versatile structural elements that have adopted a range of different cellular functions.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0268', 'name': 'Pec_lyase-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 115,\n",
       "    'interactions': 0,\n",
       "    'matches': 29161,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28954,\n",
       "    'proteomes': 3665,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 23902, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 16662},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25842,\n",
       "    'alignment:full': 4804,\n",
       "    'alignment:seed': 402},\n",
       "   'cross_references': {}}},\n",
       " 'PF06422': {'metadata': {'accession': 'PF06422',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010929',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CDR ABC transporter', 'short': 'PDR_CDR'},\n",
       "   'description': ['<p>Corresponds to a region of the PDR/CDR subgroup of ABC transporters comprising extracellular loop 3, transmembrane segment 6 and linker region.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014928': {'PMID': 12709320,\n",
       "     'ISBN': None,\n",
       "     'volume': '47',\n",
       "     'issue': '5',\n",
       "     'year': 2003,\n",
       "     'title': 'Functional similarities and differences between Candida albicans Cdr1p and Cdr2p transporters.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1543-54',\n",
       "     'medline_journal': 'Antimicrob Agents Chemother',\n",
       "     'ISO_journal': 'Antimicrob. Agents Chemother.',\n",
       "     'authors': ['Gauthier C',\n",
       "      'Weber S',\n",
       "      'Alarco AM',\n",
       "      'Alqawi O',\n",
       "      'Daoud R',\n",
       "      'Georges E',\n",
       "      'Raymond M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/AAC.47.5.1543-1554.2003'}},\n",
       "   'set_info': {'accession': 'CL0181', 'name': 'ABC-2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 305,\n",
       "    'interactions': 0,\n",
       "    'matches': 12822,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11599,\n",
       "    'proteomes': 882,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1337, 'rosettafold': 1},\n",
       "    'structures': 4,\n",
       "    'taxa': 2930},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10893,\n",
       "    'alignment:full': 6765,\n",
       "    'alignment:seed': 433},\n",
       "   'cross_references': {}}},\n",
       " 'PF13493': {'metadata': {'accession': 'PF13493',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025201',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4118)',\n",
       "    'short': 'DUF4118'},\n",
       "   'description': ['<p>This domain is found in a wide variety of bacterial signalling proteins. It is likely to be a transmembrane domain involved in ligand sensing.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1044,\n",
       "    'interactions': 0,\n",
       "    'matches': 24810,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24807,\n",
       "    'proteomes': 3511,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21390, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 16110},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23991,\n",
       "    'alignment:full': 4688,\n",
       "    'alignment:seed': 425},\n",
       "   'cross_references': {}}},\n",
       " 'PF00984': {'metadata': {'accession': 'PF00984',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014026',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UDP-glucose/GDP-mannose dehydrogenase family, central domain',\n",
       "    'short': 'UDPG_MGDP_dh'},\n",
       "   'description': ['<p>The UDP-glucose/GDP-mannose dehydrogenaseses are a small group of enzymes which possesses the ability to catalyse the NAD-dependent 2-fold oxidation of an alcohol to an acid without the release of an aldehyde  intermediate [PMID:9013585].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002546': {'PMID': 2470755,\n",
       "     'ISBN': None,\n",
       "     'volume': '264',\n",
       "     'issue': '16',\n",
       "     'year': 1989,\n",
       "     'title': 'Purification and characterization of guanosine diphospho-D-mannose dehydrogenase. A key enzyme in the biosynthesis of alginate by Pseudomonas aeruginosa.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9380-5',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Roychoudhury S',\n",
       "      'May TB',\n",
       "      'Gill JF',\n",
       "      'Singh SK',\n",
       "      'Feingold DS',\n",
       "      'Chakrabarty AM.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/264/16/9380'},\n",
       "    'PUB00003005': {'PMID': 9013585,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'Properties and kinetic analysis of UDP-glucose dehydrogenase from group A streptococci. Irreversible inhibition by UDP-chloroacetol.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3416-22',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Campbell RE', 'Sala RF', 'van de Rijn I', 'Tanner ME.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.6.3416'}},\n",
       "   'set_info': {'accession': 'CL0106', 'name': '6PGD_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 235,\n",
       "    'interactions': 0,\n",
       "    'matches': 93751,\n",
       "    'pathways': 0,\n",
       "    'proteins': 93725,\n",
       "    'proteomes': 9821,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 80858, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 41263},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 85569,\n",
       "    'alignment:full': 18371,\n",
       "    'alignment:seed': 586},\n",
       "   'cross_references': {}}},\n",
       " 'PF04032': {'metadata': {'accession': 'PF04032',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007175',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RNAse P Rpr2/Rpp21/SNM1 subunit domain', 'short': 'Rpr2'},\n",
       "   'description': ['<p>This family contains a ribonuclease P subunit of humans and yeast.  Other members of the family include the probable archaeal homologues. This family includes SNM1 [PMID:10523674].  It is a subunit of RNase MRP (mitochondrial RNA processing), a ribonucleoprotein endoribonuclease that has roles in both mitochondrial DNA replication and nuclear 5.8S rRNA processing.  SNM1 is an RNA binding protein that binds the MRP RNA specifically [PMID:10523674]. This subunit possibly binds the precursor tRNA [PMID:11497433].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010581': {'PMID': 11497433,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '8',\n",
       "     'year': 2001,\n",
       "     'title': 'Function and subnuclear distribution of Rpp21, a protein subunit of the human ribonucleoprotein ribonuclease P.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1153-64',\n",
       "     'medline_journal': 'RNA',\n",
       "     'ISO_journal': 'RNA',\n",
       "     'authors': ['Jarrous N',\n",
       "      'Reiner R',\n",
       "      'Wesolowski D',\n",
       "      'Mann H',\n",
       "      'Guerrier-Takada C',\n",
       "      'Altman S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1017/S1355838201010469'},\n",
       "    'PUB00017140': {'PMID': 10523674,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '11',\n",
       "     'year': 1999,\n",
       "     'title': 'Mutagenesis of SNM1, which encodes a protein component of the yeast RNase MRP, reveals a role for this ribonucleoprotein endoribonuclease in plasmid segregation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7857-69',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Cai T', 'Reilly TR', 'Cerio M', 'Schmitt ME.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=10523674'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 5964,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5964,\n",
       "    'proteomes': 1760,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5057, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 7619},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4887,\n",
       "    'alignment:full': 2099,\n",
       "    'alignment:seed': 271},\n",
       "   'cross_references': {}}},\n",
       " 'PF02016': {'metadata': {'accession': 'PF02016',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040449',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LD-carboxypeptidase N-terminal domain',\n",
       "    'short': 'Peptidase_S66'},\n",
       "   'description': ['<p>Muramoyl-tetrapeptide carboxypeptidase hydrolyses a peptide bond between a di-basic amino acid and the C-terminal D-alanine in the tetrapeptide moiety in peptidoglycan. This cleaves the bond between an L- and a D-amino acid. The function of this activity is in  murein recycling. This family also includes the microcin c7 self-immunity protein Swiss:Q47511. This family corresponds to  Merops family S66.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020063': {'PMID': 8522520,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '24',\n",
       "     'year': 1995,\n",
       "     'title': 'Structure and organization of plasmid genes required to produce the translation inhibitor microcin C7.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7131-40',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Gonzalez-Pastor JE',\n",
       "      'San Millan JL',\n",
       "      'Castilla MA',\n",
       "      'Moreno F.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/177/24/7131'},\n",
       "    'PUB00020064': {'PMID': 7559516,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '40',\n",
       "     'year': 1995,\n",
       "     'title': 'Chemical structure and translation inhibition studies of the antibiotic microcin C7.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '23520-32',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Guijarro JI',\n",
       "      'Gonzalez-Pastor JE',\n",
       "      'Baleux F',\n",
       "      'San Millan JL',\n",
       "      'Castilla MA',\n",
       "      'Rico M',\n",
       "      'Moreno F',\n",
       "      'Delepierre M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.270.40.23520'},\n",
       "    'PUB00020065': {'PMID': 10428950,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '15',\n",
       "     'year': 1999,\n",
       "     'title': 'A defect in cell wall recycling triggers autolysis during the stationary growth phase of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4108-17',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Templin MF', 'Ursinus A', 'Holtje JV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/18.15.4108'},\n",
       "    'PUB00021060': {'PMID': 16162494,\n",
       "     'ISBN': None,\n",
       "     'volume': '280',\n",
       "     'issue': '49',\n",
       "     'year': 2005,\n",
       "     'title': 'Pseudomonas aeruginosa LD-carboxypeptidase, a serine peptidase with a Ser-His-Glu triad and a nucleophilic elbow.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '40802-12',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Korza HJ', 'Bochtler M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M506328200'}},\n",
       "   'set_info': {'accession': 'CL0014', 'name': 'Glutaminase_I'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 53,\n",
       "    'interactions': 0,\n",
       "    'matches': 27644,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27638,\n",
       "    'proteomes': 4472,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 23993, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 19545},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26400,\n",
       "    'alignment:full': 5417,\n",
       "    'alignment:seed': 254},\n",
       "   'cross_references': {}}},\n",
       " 'PF13682': {'metadata': {'accession': 'PF13682',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025991',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Chemoreceptor zinc-binding domain', 'short': 'CZB'},\n",
       "   'description': [\"<p>The chemoreceptor zinc-binding domain (CZB) is found in bacterial signal transduction proteins - most frequently receptors involved in chemotaxis and motility, but also in c-di-GMP signalling and nitrate/nitrite-sensing. Originally discovered in the cytoplasmic chemoreceptor TlpD from Helicobacter pylori, it is often found C-terminal to the MCPsignal domain in cytoplasmic chemoreceptor proteins. The CZB domain contains a core sequence motif, Hxx[WFYL]x21-28Cx[LFMVI]Gx[WFLVI]x18-27HxxxH. The highly-conserved H-C-H-H residues of this motif are believed to coordinate zinc; mutating the latter two histidines of the motif to alanines abolishes Zn binding. This domain binds zinc with high affinity, with a Kd in the femtomolar range. This domain has been shown in E. coli to be a  zinc sensor that regulates the catalytic activity of Pfam:PF00990 [PMID:23769666]. This domain also binds the chemoattractant HOCl at a site very close to that of zinc. It has been shown that zinc participates in HOCl sensing by forming a redox 'Cys-Zn switch' that reacts towards HOCl (Matilla et. al.,FEMS Microbiology Reviews, fuab043, 45, 2021, 1. https://doi.org/10.1093/femsre/fuab043).</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 344,\n",
       "    'interactions': 0,\n",
       "    'matches': 8858,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8684,\n",
       "    'proteomes': 1064,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7494, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 5792},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7927,\n",
       "    'alignment:full': 1693,\n",
       "    'alignment:seed': 326},\n",
       "   'cross_references': {}}},\n",
       " 'PF00699': {'metadata': {'accession': 'PF00699',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002019',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Urease beta subunit', 'short': 'Urease_beta'},\n",
       "   'description': ['<p>This subunit is known as alpha in Heliobacter.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005206': {'PMID': 7754395,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '5213',\n",
       "     'year': 1995,\n",
       "     'title': 'The crystal structure of urease from Klebsiella aerogenes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '998-1004',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Jabri E', 'Carr MB', 'Hausinger RP', 'Karplus PA.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/268/5213/998'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 67,\n",
       "    'interactions': 0,\n",
       "    'matches': 17847,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17838,\n",
       "    'proteomes': 4140,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15706, 'rosettafold': 0},\n",
       "    'structures': 77,\n",
       "    'taxa': 17721},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17101,\n",
       "    'alignment:full': 4366,\n",
       "    'alignment:seed': 294},\n",
       "   'cross_references': {}}},\n",
       " 'PF01985': {'metadata': {'accession': 'PF01985',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001890',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CRS1 / YhbY (CRM) domain', 'short': 'CRS1_YhbY'},\n",
       "   'description': ['<p>Escherichia coli YhbY is associated with pre-50S ribosomal subunits, which implies a function in ribosome assembly.  GFP fused to a single-domain CRM protein from maize localises to the nucleolus, suggesting that an analogous activity may have been retained in plants [PMID:17105995].  A CRM domain containing protein in plant chloroplasts has been shown to function in group I and II intron splicing [PMID:18065687].            In vitro experiments with an isolated maize CRM domain have shown it to have RNA binding activity.  These and other results suggest that the CRM domain evolved in the context of ribosome function prior to the divergence of Archaea and Bacteria, that this function has been maintained in extant prokaryotes, and that the domain was recruited to serve as an RNA binding module during the evolution of plant genomes [PMID:17105995].  YhbY has a fold similar to that of the C-terminal domain of translation initiation factor 3 (IF3C), which binds to 16S rRNA in the 30S ribosome [PMID:12429100][PMID:12360533].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019352': {'PMID': 12429100,\n",
       "     'ISBN': None,\n",
       "     'volume': '10',\n",
       "     'issue': '11',\n",
       "     'year': 2002,\n",
       "     'title': 'Crystal structure of E. coli YhbY: a representative of a novel class of RNA binding proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1593-601',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Ostheimer GJ', 'Barkan A', 'Matthews BW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(02)00886-9'},\n",
       "    'PUB00019353': {'PMID': 12360533,\n",
       "     'ISBN': None,\n",
       "     'volume': '49',\n",
       "     'issue': '3',\n",
       "     'year': 2002,\n",
       "     'title': 'Structure of HI1333 (YhbY), a putative RNA-binding protein from Haemophilus influenzae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '423-6',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Willis MA',\n",
       "      'Krajewski W',\n",
       "      'Chalamasetty VR',\n",
       "      'Reddy P',\n",
       "      'Howard A',\n",
       "      'Herzberg O.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.10225'},\n",
       "    'PUB00019354': {'PMID': 11565746,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '9',\n",
       "     'year': 2001,\n",
       "     'title': 'CRS1 is a novel group II intron splicing factor that was derived from a domain of ancient origin.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1227-38',\n",
       "     'medline_journal': 'RNA',\n",
       "     'ISO_journal': 'RNA',\n",
       "     'authors': ['Till B',\n",
       "      'Schmitz-Linneweber C',\n",
       "      'Williams-Carrier R',\n",
       "      'Barkan A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1017/S1355838201010445'},\n",
       "    'PUB00043747': {'PMID': 17105995,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '1',\n",
       "     'year': 2007,\n",
       "     'title': 'The CRM domain: an RNA binding module derived from an ancient ribosome-associated protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '55-64',\n",
       "     'medline_journal': 'RNA',\n",
       "     'ISO_journal': 'RNA',\n",
       "     'authors': ['Barkan A',\n",
       "      'Klipcan L',\n",
       "      'Ostersetzer O',\n",
       "      'Kawamura T',\n",
       "      'Asakura Y',\n",
       "      'Watkins KP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1261/rna.139607'},\n",
       "    'PUB00044700': {'PMID': 18065687,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '12',\n",
       "     'year': 2007,\n",
       "     'title': 'A CRM domain protein functions dually in group I and group II intron splicing in land plant chloroplasts.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3864-75',\n",
       "     'medline_journal': 'Plant Cell',\n",
       "     'ISO_journal': 'Plant Cell',\n",
       "     'authors': ['Asakura Y', 'Barkan A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1105/tpc.107.055160'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 28105,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23329,\n",
       "    'proteomes': 3869,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20267, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 17526},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25566,\n",
       "    'alignment:full': 9552,\n",
       "    'alignment:seed': 481},\n",
       "   'cross_references': {}}},\n",
       " 'PF01155': {'metadata': {'accession': 'PF01155',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000688',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hydrogenase/urease nickel incorporation, metallochaperone, hypA',\n",
       "    'short': 'HypA'},\n",
       "   'description': ['<p>HypA is a metallochaperone that binds nickel to bring it safely to its target. The targets for Hypa are the nickel-containing enzymes [Ni,Fe]-hydrogenase and urease. The nickel coordinates with four nitrogens within the protein. The four conserved cysteines towards the C-terminus bind one zinc moiety probably to stabilise the protein fold [PMID:19621959].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 15126,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15126,\n",
       "    'proteomes': 2327,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12923, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 12344},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13445,\n",
       "    'alignment:full': 2488,\n",
       "    'alignment:seed': 405},\n",
       "   'cross_references': {}}},\n",
       " 'PF05173': {'metadata': {'accession': 'PF05173',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022663',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dihydrodipicolinate reductase, C-terminus',\n",
       "    'short': 'DapB_C'},\n",
       "   'description': ['<p>Dihydrodipicolinate reductase (DapB) reduces the alpha,beta-unsaturated cyclic imine, dihydro-dipicolinate.  This  reaction is the second committed step in the biosynthesis of L-lysine and its precursor meso-diaminopimelate, which are critical for both protein and cell wall biosynthesis. The C-terminal domain of DapB has been proposed to be the substrate- binding domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000417': {'PMID': 7893645,\n",
       "     'ISBN': None,\n",
       "     'volume': '34',\n",
       "     'issue': '11',\n",
       "     'year': 1995,\n",
       "     'title': 'Three-dimensional structure of Escherichia coli dihydrodipicolinate reductase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3502-12',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Scapin G', 'Blanchard JS', 'Sacchettini JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00011a003'},\n",
       "    'PUB00019387': {'PMID': 8873595,\n",
       "     'ISBN': None,\n",
       "     'volume': '35',\n",
       "     'issue': '41',\n",
       "     'year': 1996,\n",
       "     'title': 'Interaction of pyridine nucleotide substrates with Escherichia coli dihydrodipicolinate reductase: thermodynamic and structural analysis of binary complexes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13294-302',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Reddy SG', 'Scapin G', 'Blanchard JS.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi9615809'},\n",
       "    'PUB00019388': {'PMID': 9398235,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '49',\n",
       "     'year': 1997,\n",
       "     'title': 'Three-dimensional structure of Escherichia coli dihydrodipicolinate reductase in complex with NADH and the inhibitor 2,6-pyridinedicarboxylate.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15081-8',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Scapin G', 'Reddy SG', 'Zheng R', 'Blanchard JS.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi9719915'}},\n",
       "   'set_info': {'accession': 'CL0139', 'name': 'GADPH_aa-bio_dh'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 84,\n",
       "    'interactions': 0,\n",
       "    'matches': 45279,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45277,\n",
       "    'proteomes': 8656,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39089, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 36021},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41637,\n",
       "    'alignment:full': 8669,\n",
       "    'alignment:seed': 555},\n",
       "   'cross_references': {}}},\n",
       " 'PF03988': {'metadata': {'accession': 'PF03988',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007136',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Repeat of Unknown Function (DUF347)', 'short': 'DUF347'},\n",
       "   'description': ['<p>This repeat is found as four tandem repeats in a family of bacterial membrane proteins. Each repeat contains two transmembrane regions and a conserved tryptophan.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 10,\n",
       "    'interactions': 0,\n",
       "    'matches': 22355,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6029,\n",
       "    'proteomes': 912,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5266, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 5043},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22081,\n",
       "    'alignment:full': 4604,\n",
       "    'alignment:seed': 255},\n",
       "   'cross_references': {}}},\n",
       " 'PF02669': {'metadata': {'accession': 'PF02669',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003820',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'K+-transporting ATPase, c chain', 'short': 'KdpC'},\n",
       "   'description': ['<p>This family consists of K+-transporting ATPase, c chain, KdpC. KdpC forms strong interactions with the KdpA subunit, serving to assemble and stabilise the Kdp complex [PMID:9858692].  It has been suggested that KdpC could be one of the connecting links between the energy providing subunit KdpB and the K+-transporting subunit KdpA [PMID:9858692].  The K+ transport system actively transports K+ ions via ATP hydrolysis.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008269': {'PMID': 9858692,\n",
       "     'ISBN': None,\n",
       "     'volume': '1415',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'Assembly of the Kdp complex, the multi-subunit K+-transport ATPase of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '77-84',\n",
       "     'medline_journal': 'Biochim Biophys Acta',\n",
       "     'ISO_journal': 'Biochim. Biophys. Acta',\n",
       "     'authors': ['Gassel M', 'Siebers A', 'Epstein W', 'Altendorf K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0005-2736(98)00179-5'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 15,\n",
       "    'interactions': 0,\n",
       "    'matches': 17951,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17061,\n",
       "    'proteomes': 3273,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15266, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 15867},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17906,\n",
       "    'alignment:full': 3385,\n",
       "    'alignment:seed': 284},\n",
       "   'cross_references': {}}},\n",
       " 'PF14905': {'metadata': {'accession': 'PF14905',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041700',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Outer membrane protein beta-barrel family',\n",
       "    'short': 'OMP_b-brl_3'},\n",
       "   'description': ['<p>This family includes proteins annotated as TonB dependent receptors. But it is also likely to contain other membrane beta barrel proteins of other functions.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0193', 'name': 'MBB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 60,\n",
       "    'interactions': 0,\n",
       "    'matches': 33098,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32407,\n",
       "    'proteomes': 1268,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27981, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 5252},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30526,\n",
       "    'alignment:full': 7630,\n",
       "    'alignment:seed': 236},\n",
       "   'cross_references': {}}},\n",
       " 'PF00830': {'metadata': {'accession': 'PF00830',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026569',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal L28 family', 'short': 'Ribosomal_L28'},\n",
       "   'description': ['<p>The ribosomal 28 family includes L28 proteins from bacteria and chloroplasts.  The L24 protein from yeast Swiss:P36525 also contains a region of similarity to prokaryotic L28 proteins.  L24 from yeast is also found in the large ribosomal subunit</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 44468,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44465,\n",
       "    'proteomes': 10089,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38191, 'rosettafold': 0},\n",
       "    'structures': 755,\n",
       "    'taxa': 41689},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39867,\n",
       "    'alignment:full': 9946,\n",
       "    'alignment:seed': 415},\n",
       "   'cross_references': {}}},\n",
       " 'PF04408': {'metadata': {'accession': 'PF04408',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007502',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Helicase associated domain (HA2)', 'short': 'HA2'},\n",
       "   'description': ['<p>This presumed domain is about 90 amino acid residues in length. It is found is a diverse set of RNA helicases. Its function is unknown, however it seems likely to be involved in nucleic acid binding.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1144,\n",
       "    'interactions': 0,\n",
       "    'matches': 79185,\n",
       "    'pathways': 0,\n",
       "    'proteins': 79133,\n",
       "    'proteomes': 7205,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45917, 'rosettafold': 0},\n",
       "    'structures': 99,\n",
       "    'taxa': 27801},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 68255,\n",
       "    'alignment:full': 28891,\n",
       "    'alignment:seed': 361},\n",
       "   'cross_references': {}}},\n",
       " 'PF06441': {'metadata': {'accession': 'PF06441',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010497',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Epoxide hydrolase N terminus', 'short': 'EHN'},\n",
       "   'description': ['<p>This family represents the N-terminal region of the eukaryotic epoxide hydrolase protein. Epoxide hydrolases (EC:3.3.2.3) comprise a group of functionally related enzymes that catalyse the addition of water to oxirane compounds (epoxides), thereby usually generating vicinal trans-diols. EHs have been found in all types of living organisms, including mammals, invertebrates, plants, fungi and bacteria. In animals, the major interest in EH is directed towards their detoxification capacity for epoxides since they are important safeguards against the cytotoxic and genotoxic potential of oxirane derivatives that are often reactive electrophiles because of the high tension of the three-membered ring system and the strong polarization of the C--O bonds. This is of significant relevance because epoxides are frequent intermediary metabolites which arise during the biotransformation of foreign compounds [PMID:10548561]. This family is often found in conjunction with Pfam:PF00561.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012389': {'PMID': 10548561,\n",
       "     'ISBN': None,\n",
       "     'volume': '344 Pt 1',\n",
       "     'issue': None,\n",
       "     'year': 1999,\n",
       "     'title': 'Cloning and molecular characterization of a soluble epoxide hydrolase from Aspergillus niger that is related to mammalian microsomal epoxide hydrolase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '273-80',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Arand M',\n",
       "      'Hemmer H',\n",
       "      'Durk H',\n",
       "      'Baratti J',\n",
       "      'Archelas A',\n",
       "      'Furstoss R',\n",
       "      'Oesch F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1042/0264-6021:3440273'}},\n",
       "   'set_info': {'accession': 'CL0028', 'name': 'AB_hydrolase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 137,\n",
       "    'interactions': 0,\n",
       "    'matches': 21293,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21223,\n",
       "    'proteomes': 3105,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17940, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 12334},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18806,\n",
       "    'alignment:full': 6493,\n",
       "    'alignment:seed': 531},\n",
       "   'cross_references': {}}},\n",
       " 'PF13793': {'metadata': {'accession': 'PF13793',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029099',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'N-terminal domain of ribose phosphate pyrophosphokinase',\n",
       "    'short': 'Pribosyltran_N'},\n",
       "   'description': ['<p>This family is frequently found N-terminal to the Pribosyltran, Pfam:PF00156.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0533', 'name': 'PRTase-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 202,\n",
       "    'interactions': 0,\n",
       "    'matches': 66466,\n",
       "    'pathways': 0,\n",
       "    'proteins': 66323,\n",
       "    'proteomes': 11047,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 56002, 'rosettafold': 0},\n",
       "    'structures': 47,\n",
       "    'taxa': 45670},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 58507,\n",
       "    'alignment:full': 16729,\n",
       "    'alignment:seed': 303},\n",
       "   'cross_references': {}}},\n",
       " 'PF04390': {'metadata': {'accession': 'PF04390',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007485',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Lipopolysaccharide-assembly', 'short': 'LptE'},\n",
       "   'description': ['<p>LptE (formerly known as RplB) is involved in lipopolysaccharide-assembly on the outer membrane of Gram-negative organisms. The lipopolysaccharide component of the outer bacterial membrane is transported from its source of origin to the outer membrane by a set of proteins constituting a transport machinery that is made up of LptA, LptB, LptC, LptD, LptE. LptD appears to be anchored in the outer membrane, and LptE forms a complex with it. This part of the machinery complex is involved in the assembly of lipopolysaccharide in the outer leaflet of the outer membrane [PMID:18424520].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009978': {'PMID': 3316191,\n",
       "     'ISBN': None,\n",
       "     'volume': '169',\n",
       "     'issue': '12',\n",
       "     'year': 1987,\n",
       "     'title': 'Genes encoding two lipoproteins in the leuS-dacA region of the Escherichia coli chromosome.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5692-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Takase I',\n",
       "      'Ishino F',\n",
       "      'Wachi M',\n",
       "      'Kamata H',\n",
       "      'Doi M',\n",
       "      'Asoh S',\n",
       "      'Matsuzawa H',\n",
       "      'Ohta T',\n",
       "      'Matsuhashi M.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=3316191'}},\n",
       "   'set_info': {'accession': 'CL0342', 'name': 'TolB_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 21771,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21771,\n",
       "    'proteomes': 3934,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18625, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 17610},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19807,\n",
       "    'alignment:full': 3672,\n",
       "    'alignment:seed': 352},\n",
       "   'cross_references': {}}},\n",
       " 'PF02698': {'metadata': {'accession': 'PF02698',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003848',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DUF218 domain', 'short': 'DUF218'},\n",
       "   'description': ['<p>This large family of proteins contains several highly conserved charged amino acids, suggesting this may be an enzymatic domain (Bateman A pers. obs). The family includes   SanA Swiss:P33017 that is involved in Vancomycin resistance [PMID:8550448]. This protein may be involved in murein synthesis [PMID:9738879].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019464': {'PMID': 8550448,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '1',\n",
       "     'year': 1996,\n",
       "     'title': 'Amplification of a novel gene, sanA, abolishes a vancomycin-sensitive defect in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '94-102',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Rida S', 'Caillet J', 'Alix JH.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/178/1/94'},\n",
       "    'PUB00019465': {'PMID': 9738879,\n",
       "     'ISBN': None,\n",
       "     'volume': '259',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'The sfiX, rfe and metN genes of Salmonella typhimurium and their involvement in the His(c) pleiotropic response.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '46-53',\n",
       "     'medline_journal': 'Mol Gen Genet',\n",
       "     'ISO_journal': 'Mol. Gen. Genet.',\n",
       "     'authors': ['Mouslim C', 'Cano DA', 'Casadesus J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/s004380050787'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 165,\n",
       "    'interactions': 0,\n",
       "    'matches': 57087,\n",
       "    'pathways': 0,\n",
       "    'proteins': 57085,\n",
       "    'proteomes': 6649,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 49425, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 29335},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 53758,\n",
       "    'alignment:full': 10248,\n",
       "    'alignment:seed': 503},\n",
       "   'cross_references': {}}},\n",
       " 'PF00365': {'metadata': {'accession': 'PF00365',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000023',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphofructokinase', 'short': 'PFK'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0240', 'name': 'PFK'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 186,\n",
       "    'interactions': 0,\n",
       "    'matches': 77050,\n",
       "    'pathways': 0,\n",
       "    'proteins': 67574,\n",
       "    'proteomes': 8856,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 55524, 'rosettafold': 0},\n",
       "    'structures': 40,\n",
       "    'taxa': 36197},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 64789,\n",
       "    'alignment:full': 21644,\n",
       "    'alignment:seed': 706},\n",
       "   'cross_references': {}}},\n",
       " 'PF10996': {'metadata': {'accession': 'PF10996',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022712',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Beta-Casp domain', 'short': 'Beta-Casp'},\n",
       "   'description': [\"<p>The beta-CASP domain is found C terminal to the beta-lactamase domain in pre-mRNA 3'-end-processing endonuclease.  The active site of this enzyme is located at the interface of these two domains [PMID:17128255].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 244,\n",
       "    'interactions': 0,\n",
       "    'matches': 35138,\n",
       "    'pathways': 0,\n",
       "    'proteins': 35126,\n",
       "    'proteomes': 5309,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28940, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 24930},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29325,\n",
       "    'alignment:full': 8790,\n",
       "    'alignment:seed': 656},\n",
       "   'cross_references': {}}},\n",
       " 'PF12937': {'metadata': {'accession': 'PF12937',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001810',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'F-box-like', 'short': 'F-box-like'},\n",
       "   'description': ['<p>This is an F-box-like family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0271', 'name': 'F-box'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2183,\n",
       "    'interactions': 0,\n",
       "    'matches': 129476,\n",
       "    'pathways': 0,\n",
       "    'proteins': 129014,\n",
       "    'proteomes': 2166,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 100526, 'rosettafold': 0},\n",
       "    'structures': 39,\n",
       "    'taxa': 9327},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 101349,\n",
       "    'alignment:full': 53923,\n",
       "    'alignment:seed': 294},\n",
       "   'cross_references': {}}},\n",
       " 'PF03929': {'metadata': {'accession': 'PF03929',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005625',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PepSY-associated TM region', 'short': 'PepSY_TM'},\n",
       "   'description': ['<p>The PepSY_TM family is so named because it is an alignment of up to five transmembranes helices found in bacterial species some of which carry a nested PepSY domain, Pfam:PF03413.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0490', 'name': 'PepSY_TM-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 78,\n",
       "    'interactions': 0,\n",
       "    'matches': 41933,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41689,\n",
       "    'proteomes': 3823,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37667, 'rosettafold': 1},\n",
       "    'structures': 1,\n",
       "    'taxa': 16494},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42700,\n",
       "    'alignment:full': 7945,\n",
       "    'alignment:seed': 448},\n",
       "   'cross_references': {}}},\n",
       " 'PF03372': {'metadata': {'accession': 'PF03372',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005135',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Endonuclease/Exonuclease/phosphatase family',\n",
       "    'short': 'Exo_endo_phos'},\n",
       "   'description': ['<p>This large family of proteins includes magnesium dependent endonucleases and a large number of phosphatases involved in intracellular signalling [PMID:10838565]. This family includes: AP endonuclease proteins EC:4.2.99.18 e.g Swiss:P27695, DNase I proteins EC:3.1.21.1 e.g. Swiss:P24855, Synaptojanin an inositol-1,4,5-trisphosphate phosphatase EC:3.1.3.56 Swiss:O43426, Sphingomyelinase EC:3.1.4.12 Swiss:P11889 and Nocturnin Swiss:O35710.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003275': {'PMID': 1748997,\n",
       "     'ISBN': None,\n",
       "     'volume': '222',\n",
       "     'issue': '3',\n",
       "     'year': 1991,\n",
       "     'title': 'DNase I-induced DNA conformation. 2 A structure of a DNase I-octamer complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '645-67',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Lahm A', 'Suck D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(91)90502-W'},\n",
       "    'PUB00004207': {'PMID': 7885481,\n",
       "     'ISBN': None,\n",
       "     'volume': '374',\n",
       "     'issue': '6520',\n",
       "     'year': 1995,\n",
       "     'title': 'Structure and function of the multifunctional DNA-repair enzyme exonuclease III.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '381-6',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Mol CD',\n",
       "      'Kuo CF',\n",
       "      'Thayer MM',\n",
       "      'Cunningham RP',\n",
       "      'Tainer JA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/374381a0'},\n",
       "    'PUB00007746': {'PMID': 10838565,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '6',\n",
       "     'year': 2000,\n",
       "     'title': 'Functionally unrelated signalling proteins contain a fold similar to Mg2+-dependent endonucleases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '272-3',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Dlakic M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)01582-6'}},\n",
       "   'set_info': {'accession': 'CL0530', 'name': 'DNase_I-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1843,\n",
       "    'interactions': 0,\n",
       "    'matches': 215940,\n",
       "    'pathways': 0,\n",
       "    'proteins': 215576,\n",
       "    'proteomes': 10573,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 175287, 'rosettafold': 0},\n",
       "    'structures': 308,\n",
       "    'taxa': 43925},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 188321,\n",
       "    'alignment:full': 60750,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF00731': {'metadata': {'accession': 'PF00731',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000031',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AIR carboxylase', 'short': 'AIRC'},\n",
       "   'description': ['<p>Members of this family catalyse the decarboxylation of 1-(5-phosphoribosyl)-5-amino-4-imidazole-carboxylate (AIR). This family catalyse the sixth step of de novo purine biosynthesis. Some members of this family contain two copies of this domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016906': {'PMID': 10574791,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '11',\n",
       "     'year': 1999,\n",
       "     'title': 'Crystal structure of Escherichia coli PurE, an unusual mutase in the purine biosynthetic pathway.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1395-406',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Mathews II', 'Kappock TJ', 'Stubbe J', 'Ealick SE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(00)80029-5'}},\n",
       "   'set_info': {'accession': 'CL0224', 'name': 'DHQS'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 149,\n",
       "    'interactions': 0,\n",
       "    'matches': 57707,\n",
       "    'pathways': 0,\n",
       "    'proteins': 57567,\n",
       "    'proteomes': 10225,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 48994, 'rosettafold': 0},\n",
       "    'structures': 42,\n",
       "    'taxa': 42351},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51136,\n",
       "    'alignment:full': 11798,\n",
       "    'alignment:seed': 595},\n",
       "   'cross_references': {}}},\n",
       " 'PF18821': {'metadata': {'accession': 'PF18821',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040677',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Large polyvalent protein-associated domain 7',\n",
       "    'short': 'LPD7'},\n",
       "   'description': ['<p>This domain contains conserved aspartate and phenylalanine residues. It is widely present in polyvalent proteins and gene neighbourhoods of conjugative elements [PMID:28559295]. This domain is also known as PTox1 [PMID:25358815].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 47,\n",
       "    'interactions': 0,\n",
       "    'matches': 4124,\n",
       "    'pathways': 0,\n",
       "    'proteins': 4088,\n",
       "    'proteomes': 373,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 3498, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 2641},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4395,\n",
       "    'alignment:full': 529,\n",
       "    'alignment:seed': 174},\n",
       "   'cross_references': {}}},\n",
       " 'PF13850': {'metadata': {'accession': 'PF13850',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR039542',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Endoplasmic Reticulum-Golgi Intermediate Compartment (ERGIC)',\n",
       "    'short': 'ERGIC_N'},\n",
       "   'description': ['<p>This family is the N-terminal of ERGIC proteins [PMID:15308636], ER-Golgi intermediate compartment clusters, otherwise known as Ervs, and is associated with family COPIIcoated_ERV, Pfam:PF07970.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 148,\n",
       "    'interactions': 0,\n",
       "    'matches': 10214,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10199,\n",
       "    'proteomes': 2084,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8297, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 7602},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8141,\n",
       "    'alignment:full': 5106,\n",
       "    'alignment:seed': 283},\n",
       "   'cross_references': {}}},\n",
       " 'PF03334': {'metadata': {'accession': 'PF03334',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005133',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Na+/H+ antiporter subunit', 'short': 'PhaG_MnhG_YufB'},\n",
       "   'description': ['<p>This family includes PhaG from Rhizobium meliloti Swiss:Q9ZNG0, MnhG from Staphylococcus aureus Swiss:Q9ZNG0, YufB from Bacillus subtilis Swiss:O05227.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010583': {'PMID': 9852009,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '24',\n",
       "     'year': 1998,\n",
       "     'title': 'A putative multisubunit Na+/H+ antiporter from Staphylococcus aureus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6642-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Hiramatsu T',\n",
       "      'Kodama K',\n",
       "      'Kuroda T',\n",
       "      'Mizushima T',\n",
       "      'Tsuchiya T.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9852009'},\n",
       "    'PUB00010603': {'PMID': 9680201,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'The pha gene cluster of Rhizobium meliloti involved in pH adaptation and symbiosis encodes a novel type of K+ efflux system.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1091-101',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Putnoky P',\n",
       "      'Kereszt A',\n",
       "      'Nakamura T',\n",
       "      'Endre G',\n",
       "      'Grosskopf E',\n",
       "      'Kiss P',\n",
       "      'Kondorosi A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1998.00868.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 20089,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20089,\n",
       "    'proteomes': 3873,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17582, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 16741},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18634,\n",
       "    'alignment:full': 4447,\n",
       "    'alignment:seed': 408},\n",
       "   'cross_references': {}}},\n",
       " 'PF02308': {'metadata': {'accession': 'PF02308',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003416',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MgtC family', 'short': 'MgtC'},\n",
       "   'description': ['<p>The MgtC protein is found in an operon with the Mg2+ transporter protein MgtB.  The function of MgtC and its homologues is not known.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019030': {'PMID': 10550680,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '5',\n",
       "     'year': 1999,\n",
       "     'title': 'Magnesium transport in prokaryotes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '523-7',\n",
       "     'medline_journal': 'J Biol Inorg Chem',\n",
       "     'ISO_journal': 'J. Biol. Inorg. Chem.',\n",
       "     'authors': ['Moncrief MB', 'Maguire ME.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/s007750050374'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 33239,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33229,\n",
       "    'proteomes': 4826,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 29224, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 22303},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31715,\n",
       "    'alignment:full': 6130,\n",
       "    'alignment:seed': 416},\n",
       "   'cross_references': {}}},\n",
       " 'PF03313': {'metadata': {'accession': 'PF03313',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005130',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Serine dehydratase alpha chain', 'short': 'SDH_alpha'},\n",
       "   'description': ['<p>L-serine dehydratase (EC:4.2.1.13) is a found as a heterodimer of alpha and beta chain or as a fusion of the two chains in a single protein. This enzyme catalyses the deamination of serine to form pyruvate. This enzyme is part of the gluconeogenesis pathway.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 66,\n",
       "    'interactions': 0,\n",
       "    'matches': 40940,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40869,\n",
       "    'proteomes': 6364,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36524, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 25806},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40809,\n",
       "    'alignment:full': 7721,\n",
       "    'alignment:seed': 521},\n",
       "   'cross_references': {}}},\n",
       " 'PF00401': {'metadata': {'accession': 'PF00401',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020547',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase, Delta/Epsilon chain, long alpha-helix domain',\n",
       "    'short': 'ATP-synt_DE'},\n",
       "   'description': ['<p>Part of the ATP synthase CF(1). These subunits are part of the head unit of the ATP synthase. This subunit is called epsilon in bacteria and delta in  mitochondria.  In bacteria the delta (D) subunit is equivalent to the mitochondrial Oligomycin sensitive subunit, OSCP (Pfam:PF00213).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019175': {'PMID': 9331422,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '9',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure of the epsilon subunit of the proton-translocating ATP synthase from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1219-30',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Uhlin U', 'Cox GB', 'Guss JM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00272-4'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 30614,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30614,\n",
       "    'proteomes': 3383,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26539, 'rosettafold': 0},\n",
       "    'structures': 60,\n",
       "    'taxa': 34145},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27164,\n",
       "    'alignment:full': 3468,\n",
       "    'alignment:seed': 334},\n",
       "   'cross_references': {}}},\n",
       " 'PF01450': {'metadata': {'accession': 'PF01450',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000506',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acetohydroxy acid isomeroreductase, catalytic domain',\n",
       "    'short': 'IlvC'},\n",
       "   'description': ['<p>Acetohydroxy acid isomeroreductase catalyses the conversion of acetohydroxy acids into dihydroxy valerates. This reaction is the second in the synthetic pathway of the essential branched side chain amino acids valine and isoleucine.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001302': {'PMID': 9218783,\n",
       "     'ISBN': None,\n",
       "     'volume': '16',\n",
       "     'issue': '12',\n",
       "     'year': 1997,\n",
       "     'title': 'The crystal structure of plant acetohydroxy acid isomeroreductase complexed with NADPH, two magnesium ions and a herbicidal transition state analog determined at 1.65 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3405-15',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Biou V',\n",
       "      'Dumas R',\n",
       "      'Cohen-Addad C',\n",
       "      'Douce R',\n",
       "      'Job D',\n",
       "      'Pebay-Peyroula E.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/16.12.3405'}},\n",
       "   'set_info': {'accession': 'CL0106', 'name': '6PGD_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 75,\n",
       "    'interactions': 0,\n",
       "    'matches': 46821,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41470,\n",
       "    'proteomes': 8843,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 35970, 'rosettafold': 0},\n",
       "    'structures': 62,\n",
       "    'taxa': 36065},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43407,\n",
       "    'alignment:full': 9814,\n",
       "    'alignment:seed': 252},\n",
       "   'cross_references': {}}},\n",
       " 'PF03323': {'metadata': {'accession': 'PF03323',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004995',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacillus/Clostridium GerA spore germination protein',\n",
       "    'short': 'GerA'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019610': {'PMID': 10762253,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '9',\n",
       "     'year': 2000,\n",
       "     'title': 'Role of ger proteins in nutrient and nonnutrient triggering of spore germination in Bacillus subtilis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2513-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Paidhungat M', 'Setlow P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.9.2513-2519.2000'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 18,\n",
       "    'interactions': 0,\n",
       "    'matches': 19035,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18996,\n",
       "    'proteomes': 1192,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17110, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 3997},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19746,\n",
       "    'alignment:full': 5067,\n",
       "    'alignment:seed': 409},\n",
       "   'cross_references': {}}},\n",
       " 'PF03352': {'metadata': {'accession': 'PF03352',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005019',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Methyladenine glycosylase', 'short': 'Adenine_glyco'},\n",
       "   'description': ['<p>The DNA-3-methyladenine glycosylase I is constitutively expressed and is specific for the alkylated 3-methyladenine DNA. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019100': {'PMID': 8502545,\n",
       "     'ISBN': None,\n",
       "     'volume': '21',\n",
       "     'issue': '9',\n",
       "     'year': 1993,\n",
       "     'title': 'Excision of 3-methylguanine from alkylated DNA by 3-methyladenine DNA glycosylase I of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2045-9',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Bjelland S', 'Bjoras M', 'Seeberg E.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/21.9.2045'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 77,\n",
       "    'interactions': 0,\n",
       "    'matches': 36702,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36510,\n",
       "    'proteomes': 6026,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31831, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 25660},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34554,\n",
       "    'alignment:full': 7374,\n",
       "    'alignment:seed': 449},\n",
       "   'cross_references': {}}},\n",
       " 'PF00433': {'metadata': {'accession': 'PF00433',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017892',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein kinase C terminal domain', 'short': 'Pkinase_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 432,\n",
       "    'interactions': 0,\n",
       "    'matches': 54131,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54086,\n",
       "    'proteomes': 2067,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36160, 'rosettafold': 0},\n",
       "    'structures': 107,\n",
       "    'taxa': 7860},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38055,\n",
       "    'alignment:full': 24833,\n",
       "    'alignment:seed': 421},\n",
       "   'cross_references': {}}},\n",
       " 'PF01040': {'metadata': {'accession': 'PF01040',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000537',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UbiA prenyltransferase family', 'short': 'UbiA'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0613', 'name': 'Terp_synthase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 393,\n",
       "    'interactions': 0,\n",
       "    'matches': 135784,\n",
       "    'pathways': 0,\n",
       "    'proteins': 135589,\n",
       "    'proteomes': 10252,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 114916, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 43489},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 120907,\n",
       "    'alignment:full': 28982,\n",
       "    'alignment:seed': 878},\n",
       "   'cross_references': {}}},\n",
       " 'PF00438': {'metadata': {'accession': 'PF00438',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022628',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'S-adenosylmethionine synthetase, N-terminal domain',\n",
       "    'short': 'S-AdoMet_synt_N'},\n",
       "   'description': ['<p>The three domains of S-adenosylmethionine synthetase have the same alpha+beta fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020285': {'PMID': 8611562,\n",
       "     'ISBN': None,\n",
       "     'volume': '35',\n",
       "     'issue': '8',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure and function of S-adenosylmethionine synthetase: crystal structures of S-adenosylmethionine synthetase with ADP, BrADP, and PPi at 28 angstroms resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2586-96',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Takusagawa F', 'Kamitori S', 'Markham GD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi952604z'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 120,\n",
       "    'interactions': 0,\n",
       "    'matches': 54610,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54584,\n",
       "    'proteomes': 10732,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46168, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 44358},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48658,\n",
       "    'alignment:full': 12180,\n",
       "    'alignment:seed': 404},\n",
       "   'cross_references': {}}},\n",
       " 'PF01424': {'metadata': {'accession': 'PF01424',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001374',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'R3H domain', 'short': 'R3H'},\n",
       "   'description': ['<p>The name of the R3H domain comes from the characteristic spacing of the most conserved arginine and histidine residues.  The function of the domain is predicted to be binding ssDNA.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005485': {'PMID': 9787637,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'The R3H motif: a domain that binds single-stranded nucleic acids.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '329-30',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Grishin NV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(98)01258-4'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 396,\n",
       "    'interactions': 0,\n",
       "    'matches': 41836,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41824,\n",
       "    'proteomes': 6049,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32741, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 24654},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34374,\n",
       "    'alignment:full': 13268,\n",
       "    'alignment:seed': 635},\n",
       "   'cross_references': {}}},\n",
       " 'PF02542': {'metadata': {'accession': 'PF02542',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003526',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YgbB family', 'short': 'YgbB'},\n",
       "   'description': ['<p>The ygbB protein is a putative enzyme of deoxy-xylulose pathway (terpenoid biosynthesis) [PMID:10694574].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007261': {'PMID': 10694574,\n",
       "     'ISBN': None,\n",
       "     'volume': '97',\n",
       "     'issue': '6',\n",
       "     'year': 2000,\n",
       "     'title': 'Biosynthesis of terpenoids: YgbB protein converts 4-diphosphocytidyl-2C-methyl-D-erythritol 2-phosphate to 2C-methyl-D-erythritol 2,4-cyclodiphosphate.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2486-90',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Herz S',\n",
       "      'Wungsintaweekul J',\n",
       "      'Schuhr CA',\n",
       "      'Hecht S',\n",
       "      'Luttgen H',\n",
       "      'Sagner S',\n",
       "      'Fellermeier M',\n",
       "      'Eisenreich W',\n",
       "      'Zenk MH',\n",
       "      'Bacher A',\n",
       "      'Rohdich F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.040554697'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 58,\n",
       "    'interactions': 0,\n",
       "    'matches': 38233,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38214,\n",
       "    'proteomes': 7791,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32774, 'rosettafold': 0},\n",
       "    'structures': 71,\n",
       "    'taxa': 32123},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34829,\n",
       "    'alignment:full': 7230,\n",
       "    'alignment:seed': 613},\n",
       "   'cross_references': {}}},\n",
       " 'PF00815': {'metadata': {'accession': 'PF00815',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012131',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Histidinol dehydrogenase', 'short': 'Histidinol_dh'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0099', 'name': 'ALDH-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 49139,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48935,\n",
       "    'proteomes': 8981,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41901, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 36226},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45167,\n",
       "    'alignment:full': 10125,\n",
       "    'alignment:seed': 568},\n",
       "   'cross_references': {}}},\n",
       " 'PF01502': {'metadata': {'accession': 'PF01502',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002496',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoribosyl-AMP cyclohydrolase', 'short': 'PRA-CH'},\n",
       "   'description': ['<p>This enzyme catalyses the third step in the histidine biosynthetic pathway. It requires Zn ions for activity.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000456': {'PMID': 9931020,\n",
       "     'ISBN': None,\n",
       "     'volume': '38',\n",
       "     'issue': '5',\n",
       "     'year': 1999,\n",
       "     'title': \"N1-(5'-phosphoribosyl)adenosine-5'-monophosphate cyclohydrolase: purification and characterization of a unique metalloenzyme.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '1537-46',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': [\"D'Ordine RL\", 'Klem TJ', 'Davisson VJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi982475x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 40423,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40419,\n",
       "    'proteomes': 8942,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34923, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 36005},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37009,\n",
       "    'alignment:full': 8462,\n",
       "    'alignment:seed': 562},\n",
       "   'cross_references': {}}},\n",
       " 'PF02466': {'metadata': {'accession': 'PF02466',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tim17/Tim22/Tim23/Pmp24 family', 'short': 'Tim17'},\n",
       "   'description': ['<p>The pre-protein translocase of the mitochondrial outer membrane (Tom) allows the import of pre-proteins from the cytoplasm. Tom forms a complex with a number of proteins, including Tim17 [PMID:8893850, PMID:10366717, PMID:7721939]. Tim17, Tim22, and Tim23, are the central components of the widely conserved multi-subunit protein translocases, TIM23 and TIM22, which mediate protein transport across and into the inner mitochondrial membrane, respectively [PMID:27760563]. In addition, several Tim17 family proteins occupy the inner and outer membranes of plastids. This family also includes Pmp24 a peroxisomal protein. The involvement of this domain in the targeting of PMP24 remains to be proved. PMP24 was known as Pmp27 [PMID:7721939]. Family members are suggested to be exclusive to eukaryotes, where the distribution in the eukaryotic subgroups of the mitochondrial Tim17, Tim22 and Tim23 proteins, as well as the peroxisomal Tim17 family proteins, suggests that they all likely to be present in the last eukaryotic common ancestor (LECA) [PMID:27760563].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020437': {'PMID': 8893850,\n",
       "     'ISBN': None,\n",
       "     'volume': '262',\n",
       "     'issue': '4',\n",
       "     'year': 1996,\n",
       "     'title': 'The preprotein translocase of the inner mitochondrial membrane: evolutionary conservation of targeting and assembly of Tim17.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '389-95',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Bomer U',\n",
       "      'Rassow J',\n",
       "      'Zufall N',\n",
       "      'Pfanner N',\n",
       "      'Meijer M',\n",
       "      'Maarse AC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1996.0522'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 227,\n",
       "    'interactions': 0,\n",
       "    'matches': 18957,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18931,\n",
       "    'proteomes': 2158,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15818, 'rosettafold': 0},\n",
       "    'structures': 99,\n",
       "    'taxa': 7999},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15586,\n",
       "    'alignment:full': 9800,\n",
       "    'alignment:seed': 385},\n",
       "   'cross_references': {}}},\n",
       " 'PF18741': {'metadata': {'accession': 'PF18741',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'REase_MTES_1575', 'short': 'MTES_1575'},\n",
       "   'description': ['<p>Vsr REase Fold. Fused to HEPN (SWT1/Abi2 family), along with Transglutaminase and wHTH [PMID:23768067].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0236', 'name': 'PDDEXK'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 162,\n",
       "    'interactions': 0,\n",
       "    'matches': 9518,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9517,\n",
       "    'proteomes': 1914,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 2842, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 9452},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8993,\n",
       "    'alignment:full': 2112,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF16491': {'metadata': {'accession': 'PF16491',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032456',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CAAX prenyl protease N-terminal, five membrane helices',\n",
       "    'short': 'Peptidase_M48_N'},\n",
       "   'description': ['<p>The five N-terminal five transmembrane alpha-helices of peptidase_M48 family proteins including the CAAX prenyl proteases reside completely within the membrane of the endoplasmic reticulum.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 15154,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15068,\n",
       "    'proteomes': 4128,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12669, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 16412},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13235,\n",
       "    'alignment:full': 3873,\n",
       "    'alignment:seed': 285},\n",
       "   'cross_references': {}}},\n",
       " 'PF01867': {'metadata': {'accession': 'PF01867',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002729',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CRISPR associated protein Cas1', 'short': 'Cas_Cas1'},\n",
       "   'description': ['<p>Clustered regularly interspaced short palindromic repeats (CRISPRs) are a family of DNA direct repeats found in many prokaryotic genomes. This family of proteins corresponds to Cas1, a CRISPR-associated protein.  Cas1 may be involved in linking DNA segments to CRISPR [PMID:16079334].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 22941,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21978,\n",
       "    'proteomes': 3502,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18997, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 16672},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20435,\n",
       "    'alignment:full': 4827,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF01523': {'metadata': {'accession': 'PF01523',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002510',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PmbA/TldA metallopeptidase domain 1',\n",
       "    'short': 'PmbA_TldD'},\n",
       "   'description': ['<p>This entry represents a group of metalloproteases. The tertiary structure of the Escherichia coli TdlD/TdlE complex has been solved, and shows that the TdlD subunit is the active peptidase, binding a single zinc ion at an HEXXXH motif in which the glutamic acid is a substrate-binding residue and the two histidines are zinc ligands. The third zinc ligand is a cysteine, C-terminal to the HEXXXH motif. The TldE (also known as PmbA) by itself has no catalytic activity, does not bind zinc, and does not carry the HEXXXH motif [PMID:28943336]. TldD and TldE were originally identified as regulators of DNA gyrase. Later, they are shown to be metalloproteases involved in CcdA degradation [2-3].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003357': {'PMID': 8604133,\n",
       "     'ISBN': None,\n",
       "     'volume': '256',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Evidence for involvement of Escherichia coli genes pmbA, csrA and a previously unrecognized gene tldD, in the control of DNA gyrase by letD (ccdB) of sex factor F.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '483-502',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Murayama N',\n",
       "      'Shimizu H',\n",
       "      'Takiguchi S',\n",
       "      'Baba Y',\n",
       "      'Amino H',\n",
       "      'Horiuchi T',\n",
       "      'Sekimizu K',\n",
       "      'Miki T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1996.0103'},\n",
       "    'PUB00003799': {'PMID': 2082149,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '11',\n",
       "     'year': 1990,\n",
       "     'title': 'Molecular characterization of pmbA, an Escherichia coli chromosomal gene required for the production of the antibiotic peptide MccB17.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1921-32',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Rodriguez-Sainz MC', 'Hernandez-Chico C', 'Moreno F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.1990.tb02041.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 49214,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49209,\n",
       "    'proteomes': 4563,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41915, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 20912},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44685,\n",
       "    'alignment:full': 7932,\n",
       "    'alignment:seed': 479},\n",
       "   'cross_references': {}}},\n",
       " 'PF03975': {'metadata': {'accession': 'PF03975',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005659',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CheD chemotactic sensory transduction', 'short': 'CheD'},\n",
       "   'description': ['<p>This chemotaxis protein stimulates methylation of MCP proteins [PMID:8866475]. The chemotaxis machinery of Bacillus subtilis is similar to that of the well characterised system of Escherichia coli. However, B. subtilis contains several chemotaxis genes not found in the E. coli genome, such as CheC and CheD, indicating that the B. subtilis chemotactic system is more complex. CheD plays an important role in chemotactic sensory transduction for many organisms. CheD deamidates other B. subtilis chemoreceptors including McpB and McpC. Deamidation by CheD is required for B. subtilis chemoreceptors to effectively transduce signals to the CheA kinase [PMID:12011078]. The structure of a complex between the signal-terminating phosphatase, CheC, and the receptor-modifying deamidase, CheD, reveals how CheC mimics receptor substrates to inhibit CheD and how CheD stimulates CheC phosphatase activity. CheD resembles other cysteine deamidases from bacterial pathogens that inactivate host Rho-GTPases. Phospho-CheY, the intracellular signal and CheC target, stabilises the CheC-CheD complex and reduces availability of CheD [PMID:16469702]. A model is proposed whereby CheC acts as a CheY-P-induced regulator of CheD; CheY-P would cause CheC to sequester CheD from the chemoreceptors, inducing adaptation of the chemotaxis system [PMID:17908686].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019299': {'PMID': 8866475,\n",
       "     'ISBN': None,\n",
       "     'volume': '21',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'CheC and CheD interact to regulate methylation of Bacillus subtilis methyl-accepting chemotaxis proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '511-8',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Rosario MM', 'Ordal GW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.1996.tb02560.x'}},\n",
       "   'set_info': {'accession': 'CL0663', 'name': 'CNF1_YfiH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 20,\n",
       "    'interactions': 0,\n",
       "    'matches': 13579,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13578,\n",
       "    'proteomes': 2719,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11812, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 11640},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12281,\n",
       "    'alignment:full': 2999,\n",
       "    'alignment:seed': 278},\n",
       "   'cross_references': {}}},\n",
       " 'PF03625': {'metadata': {'accession': 'PF03625',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005180',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function DUF302', 'short': 'DUF302'},\n",
       "   'description': ['<p>Domain is found in an undescribed set of proteins [PMID:12625841]. Normally occurs uniquely within a sequence, but is found as a tandem repeat (Swiss:Q9X8B8). Shows interesting phylogenetic distribution with majority of examples in bacteria and archaea, but it is also found in some fungal proteins. The hypothetical protein TT1751 from Thermus thermophilus has a beta-alpha-beta(4)-alpha structural fold [PMID:15481054].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 16965,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16709,\n",
       "    'proteomes': 2563,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14231, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 12529},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14785,\n",
       "    'alignment:full': 3382,\n",
       "    'alignment:seed': 265},\n",
       "   'cross_references': {}}},\n",
       " 'PF07593': {'metadata': {'accession': 'PF07593',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011519',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ASPIC and UnbV', 'short': 'UnbV_ASPIC'},\n",
       "   'description': ['<p>This conserved sequence is found associated with Pfam:PF00515 in several paralogous proteins in Rhodopirellula baltica.  It is also found associated with Pfam:PF01839 in several eukaryotic integrin-like proteins (e.g. human ASPIC Swiss:Q9NQ78) and in several other bacterial proteins (e.g. Swiss:Q84HN1 [PMID:12536216]).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014744': {'PMID': 12536216,\n",
       "     'ISBN': None,\n",
       "     'volume': '21',\n",
       "     'issue': '2',\n",
       "     'year': 2003,\n",
       "     'title': 'A genomics-guided approach for discovering and expressing cryptic metabolic pathways.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '187-90',\n",
       "     'medline_journal': 'Nat Biotechnol',\n",
       "     'ISO_journal': 'Nat. Biotechnol.',\n",
       "     'authors': ['Zazopoulos E',\n",
       "      'Huang K',\n",
       "      'Staffa A',\n",
       "      'Liu W',\n",
       "      'Bachmann BO',\n",
       "      'Nonaka K',\n",
       "      'Ahlert J',\n",
       "      'Thorson JS',\n",
       "      'Shen B',\n",
       "      'Farnet CM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nbt784'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1120,\n",
       "    'interactions': 0,\n",
       "    'matches': 26981,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26456,\n",
       "    'proteomes': 1862,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19637, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7919},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21179,\n",
       "    'alignment:full': 4189,\n",
       "    'alignment:seed': 253},\n",
       "   'cross_references': {}}},\n",
       " 'PF03443': {'metadata': {'accession': 'PF03443',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005103',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Auxiliary Activity family 9 (formerly GH61)',\n",
       "    'short': 'AA9'},\n",
       "   'description': ['<p>Although weak endoglucanase activity has been demonstrated in several members of this family [1-3], they lack the clustered conserved catalytic acidic amino acids present in most glycoside hydrolases.  Many members of this family lack measurable cellulase activity on their own, but enhance the activity of other cellulolytic enzymes.  They are therefore unlikely to be true glycoside hydrolases [PMID:20230050]. The substrate-binding surface of this family is a flat Ig-like fold [PMID:24912171]. This family of enzymes were originally classified as glycoside hydrolases (GH61) and they have been reclassified as the Auxiliary Activity Family 9 (AA9) of CAZy.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 220,\n",
       "    'interactions': 0,\n",
       "    'matches': 12404,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12288,\n",
       "    'proteomes': 626,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 9894, 'rosettafold': 0},\n",
       "    'structures': 90,\n",
       "    'taxa': 1992},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10125,\n",
       "    'alignment:full': 6553,\n",
       "    'alignment:seed': 443},\n",
       "   'cross_references': {}}},\n",
       " 'PF10601': {'metadata': {'accession': 'PF10601',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006629',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LITAF-like zinc ribbon domain', 'short': 'zf-LITAF-like'},\n",
       "   'description': [\"<p>Members of this family display a conserved zinc ribbon structure [PMID:12527760] with the motif C-XX-C- separated from the more C-terminal HX-C(P)X-C-X4-G-R motif by a variable region of usually 25-30 (hydrophobic) residues. Although it belongs to one of the zinc finger's fold groups (zinc ribbon), this particular domain was first identified in LPS-induced tumour necrosis alpha factor (LITAF) which is produced in mammalian cells after being challenged with lipopolysaccharide (LPS)[PMID:17408970]. The hydrophobic region probably inserts into the membrane rather than traversing it. Such an insertion brings together the N- and C-terminal C-XX-C motifs to form a compact Zn2+-binding structure [PMID:11731489].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007973': {'PMID': 11731489,\n",
       "     'ISBN': None,\n",
       "     'volume': '11',\n",
       "     'issue': '12',\n",
       "     'year': 2001,\n",
       "     'title': 'Novel protein domains and repeats in Drosophila melanogaster: insights into structure, function, and evolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1996-2008',\n",
       "     'medline_journal': 'Genome Res',\n",
       "     'ISO_journal': 'Genome Res.',\n",
       "     'authors': ['Ponting CP', 'Mott R', 'Bork P', 'Copley RR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1101/gr.198701'},\n",
       "    'PUB00044560': {'PMID': 17408970,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '2',\n",
       "     'year': 2007,\n",
       "     'title': 'Molecular cloning and characterization of a putative lipopolysaccharide-induced TNF-alpha factor (LITAF) gene homologue from Zhikong scallop Chlamys farreri.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '419-29',\n",
       "     'medline_journal': 'Fish Shellfish Immunol',\n",
       "     'ISO_journal': 'Fish Shellfish Immunol.',\n",
       "     'authors': ['Yu Y',\n",
       "      'Qiu L',\n",
       "      'Song L',\n",
       "      'Zhao J',\n",
       "      'Ni D',\n",
       "      'Zhang Y',\n",
       "      'Xu W.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.fsi.2006.12.004'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 113,\n",
       "    'interactions': 0,\n",
       "    'matches': 8953,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8841,\n",
       "    'proteomes': 1573,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6757, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 6284},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6505,\n",
       "    'alignment:full': 3702,\n",
       "    'alignment:seed': 306},\n",
       "   'cross_references': {}}},\n",
       " 'PF09527': {'metadata': {'accession': 'PF09527',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032820',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative F0F1-ATPase subunit Ca2+/Mg2+ transporter',\n",
       "    'short': 'ATPase_gene1'},\n",
       "   'description': ['<p>This model represents a protein found encoded in F1F0-ATPase operons in several genomes, including Methanosarcina barkeri (archaeal) and Chlorobium tepidum (bacterial). It is a small protein (about 100 amino acids) with long hydrophobic stretches and is presumed to be a subunit of the enzyme. It carries two transmembrane helices and is a magnesium or calcium uniporter. The atp operon of alkaliphilic Bacillus pseudofirmus OF4, as in most prokaryotes, contains the eight structural genes for the F-ATPase (ATP synthase), which are preceded by an atpI gene that encodes a membrane protein with 2 TMSs. A tenth gene, atpZ, has been found in this operon, which is upstream of and overlapping with atpI [PMID:12917488].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 18,\n",
       "    'interactions': 0,\n",
       "    'matches': 19594,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19591,\n",
       "    'proteomes': 3650,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16409, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15739},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16829,\n",
       "    'alignment:full': 3647,\n",
       "    'alignment:seed': 326},\n",
       "   'cross_references': {}}},\n",
       " 'PF13556': {'metadata': {'accession': 'PF13556',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025736',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PucR C-terminal helix-turn-helix domain',\n",
       "    'short': 'HTH_30'},\n",
       "   'description': ['<p>This helix-turn-helix domain is often found at the C-terminus of PucR-like transcriptional regulators such as Swiss:O32138 and is likely to be DNA-binding.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 135,\n",
       "    'interactions': 0,\n",
       "    'matches': 68359,\n",
       "    'pathways': 0,\n",
       "    'proteins': 67568,\n",
       "    'proteomes': 3684,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 60736, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 15217},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 67400,\n",
       "    'alignment:full': 17577,\n",
       "    'alignment:seed': 248},\n",
       "   'cross_references': {}}},\n",
       " 'PF01981': {'metadata': {'accession': 'PF01981',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002833',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peptidyl-tRNA hydrolase PTH2', 'short': 'PTH2'},\n",
       "   'description': ['<p>Peptidyl-tRNA hydrolases are enzymes that release tRNAs from peptidyl-tRNA during translation.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020200': {'PMID': 12475929,\n",
       "     'ISBN': None,\n",
       "     'volume': '99',\n",
       "     'issue': '26',\n",
       "     'year': 2002,\n",
       "     'title': 'Orthologs of a novel archaeal and of the bacterial peptidyl-tRNA hydrolase are nonessential in yeast.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '16707-12',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Rosas-Sandoval G',\n",
       "      'Ambrogelly A',\n",
       "      'Rinehart J',\n",
       "      'Wei D',\n",
       "      'Cruz-Vera LR',\n",
       "      'Graham DE',\n",
       "      'Stetter KO',\n",
       "      'Guarneros G',\n",
       "      'Soll D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.222659199'}},\n",
       "   'set_info': {'accession': 'CL0305', 'name': 'PTH2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 119,\n",
       "    'interactions': 0,\n",
       "    'matches': 12474,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12412,\n",
       "    'proteomes': 3209,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10349, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 13053},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10448,\n",
       "    'alignment:full': 4242,\n",
       "    'alignment:seed': 274},\n",
       "   'cross_references': {}}},\n",
       " 'PF00126': {'metadata': {'accession': 'PF00126',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000847',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial regulatory helix-turn-helix protein, lysR family',\n",
       "    'short': 'HTH_1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005283': {'PMID': 9309218,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '8',\n",
       "     'year': 1997,\n",
       "     'title': 'The structure of the cofactor-binding fragment of the LysR family member, CysB: a familiar fold with a surprising subunit arrangement.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1017-32',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Tyrrell R',\n",
       "      'Verschueren KH',\n",
       "      'Dodson EJ',\n",
       "      'Murshudov GN',\n",
       "      'Addy C',\n",
       "      'Wilkinson AJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00254-2'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 386,\n",
       "    'interactions': 0,\n",
       "    'matches': 858446,\n",
       "    'pathways': 0,\n",
       "    'proteins': 855962,\n",
       "    'proteomes': 8531,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 779776, 'rosettafold': 0},\n",
       "    'structures': 63,\n",
       "    'taxa': 36415},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 888347,\n",
       "    'alignment:full': 148323,\n",
       "    'alignment:seed': 1186},\n",
       "   'cross_references': {}}},\n",
       " 'PF11915': {'metadata': {'accession': 'PF11915',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021840',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF3433)',\n",
       "    'short': 'DUF3433'},\n",
       "   'description': ['<p>This is a family of functionally uncharacterised proteins. The family is found in eukaryotes, and represents the conserved central region of the member proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 7537,\n",
       "    'pathways': 0,\n",
       "    'proteins': 4480,\n",
       "    'proteomes': 588,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 2965, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1696},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6406,\n",
       "    'alignment:full': 3738,\n",
       "    'alignment:seed': 360},\n",
       "   'cross_references': {}}},\n",
       " 'PF01904': {'metadata': {'accession': 'PF01904',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002763',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function DUF72', 'short': 'DUF72'},\n",
       "   'description': ['<p>The function of this family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 32924,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32891,\n",
       "    'proteomes': 4136,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28955, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 19121},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31383,\n",
       "    'alignment:full': 5307,\n",
       "    'alignment:seed': 491},\n",
       "   'cross_references': {}}},\n",
       " 'PF07662': {'metadata': {'accession': 'PF07662',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011657',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Na+ dependent nucleoside transporter C-terminus',\n",
       "    'short': 'Nucleos_tra2_C'},\n",
       "   'description': ['<p>This family consists of nucleoside transport proteins. Swiss:Q62773 is a purine-specific Na+-nucleoside cotransporter localised to the bile canalicular membrane [PMID:7775409]. Swiss:Q62674 is a a Na+-dependent nucleoside transporter selective for pyrimidine nucleosides and adenosine it also transports the anti-viral nucleoside analogues AZT and ddC [PMID:8027026]. This alignment covers the C-terminus of this family of transporters.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002855': {'PMID': 8027026,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '27',\n",
       "     'year': 1994,\n",
       "     'title': 'Cloning and functional expression of a complementary DNA encoding a mammalian nucleoside transport protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17757-60',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Huang QQ',\n",
       "      'Yao SY',\n",
       "      'Ritzel MW',\n",
       "      'Paterson AR',\n",
       "      'Cass CE',\n",
       "      'Young JD.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/27/17757'},\n",
       "    'PUB00002907': {'PMID': 7775409,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '23',\n",
       "     'year': 1995,\n",
       "     'title': 'Primary structure and functional expression of a cDNA encoding the bile canalicular, purine-specific Na(+)-nucleoside cotransporter.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13596-9',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Che M', 'Ortiz DF', 'Arias IM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.270.23.13596'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 24922,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24748,\n",
       "    'proteomes': 4060,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21157, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 17537},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23798,\n",
       "    'alignment:full': 5043,\n",
       "    'alignment:seed': 321},\n",
       "   'cross_references': {}}},\n",
       " 'PF00584': {'metadata': {'accession': 'PF00584',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001901',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SecE/Sec61-gamma subunits of protein translocation complex',\n",
       "    'short': 'SecE'},\n",
       "   'description': ['<p>SecE is part of the SecYEG complex in bacteria which  translocates proteins from the cytoplasm. In eukaryotes the complex, made from Sec61-gamma and Sec61-alpha  translocates protein from the cytoplasm to the ER.  Archaea have a similar complex.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000955': {'PMID': 9393849,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '5',\n",
       "     'year': 1997,\n",
       "     'title': 'Protein translocation in the three domains of life: variations on a theme.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '563-6',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Pohlschroder M', 'Prinz WA', 'Hartmann E', 'Beckwith J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80443-2'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 44757,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44757,\n",
       "    'proteomes': 10463,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38667, 'rosettafold': 0},\n",
       "    'structures': 56,\n",
       "    'taxa': 43867},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40054,\n",
       "    'alignment:full': 9768,\n",
       "    'alignment:seed': 695},\n",
       "   'cross_references': {}}},\n",
       " 'PF00453': {'metadata': {'accession': 'PF00453',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005813',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L20', 'short': 'Ribosomal_L20'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 76,\n",
       "    'interactions': 0,\n",
       "    'matches': 57035,\n",
       "    'pathways': 0,\n",
       "    'proteins': 57018,\n",
       "    'proteomes': 9820,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 48822, 'rosettafold': 0},\n",
       "    'structures': 811,\n",
       "    'taxa': 60093},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49944,\n",
       "    'alignment:full': 9027,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF02033': {'metadata': {'accession': 'PF02033',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000238',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosome-binding factor A', 'short': 'RBFA'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0494', 'name': 'DnaA_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 64,\n",
       "    'interactions': 0,\n",
       "    'matches': 43410,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43409,\n",
       "    'proteomes': 9436,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37158, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 39668},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39082,\n",
       "    'alignment:full': 8617,\n",
       "    'alignment:seed': 709},\n",
       "   'cross_references': {}}},\n",
       " 'PF09827': {'metadata': {'accession': 'PF09827',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019199',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CRISPR associated protein Cas2', 'short': 'CRISPR_Cas2'},\n",
       "   'description': ['<p>This entry represents members of the family of Cas2, one of the first four protein families found to be associated with prokaryotic genomes containing multiple CRISPR elements. CRISPR systems protect against invasive nucleic acid sequences, including phage. Cas2 proteins have been characterised as either endoribonuclease (for ssRNA) or endodeoxyribonuclease (for dsDNA), depending on the system to which the Cas2 belongs [PMID:17379808, PMID:18482976]. The cas genes usually are found near the palindromic repeats. The structural subunit of Cas2, belongs to the VapD family of interferases. The interferase catalytic site is intact in the majority of the Cas2 proteins but is disrupted in some, and is not required for spacer acquisition [PMID:30905284, PMID:24793649]. This entry also includes the endoribonuclease VapD [PMID:22241770].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00043287': {'PMID': 17379808,\n",
       "     'ISBN': None,\n",
       "     'volume': '315',\n",
       "     'issue': '5819',\n",
       "     'year': 2007,\n",
       "     'title': 'CRISPR provides acquired resistance against viruses in prokaryotes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1709-12',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Barrangou R',\n",
       "      'Fremaux C',\n",
       "      'Deveau H',\n",
       "      'Richards M',\n",
       "      'Boyaval P',\n",
       "      'Moineau S',\n",
       "      'Romero DA',\n",
       "      'Horvath P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.1138140'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 13,\n",
       "    'interactions': 0,\n",
       "    'matches': 18167,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18161,\n",
       "    'proteomes': 2800,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15774, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 14509},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16023,\n",
       "    'alignment:full': 3652,\n",
       "    'alignment:seed': 348},\n",
       "   'cross_references': {}}},\n",
       " 'PF13167': {'metadata': {'accession': 'PF13167',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025121',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GTP-binding GTPase N-terminal', 'short': 'GTP-bdg_N'},\n",
       "   'description': ['<p>This is the N-terminal region of GTP-binding HflX-like proteins. The full-length members bind and interact with the 50S ribosome and are GTPases, hydrolysing GTP/GDP/ATP/ADP. This N-terminal region is necessary for stability of the whole protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 46464,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46244,\n",
       "    'proteomes': 9135,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39326, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 37862},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42084,\n",
       "    'alignment:full': 8940,\n",
       "    'alignment:seed': 568},\n",
       "   'cross_references': {}}},\n",
       " 'PF02580': {'metadata': {'accession': 'PF02580',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003732',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'D-Tyr-tRNA(Tyr) deacylase', 'short': 'Tyr_Deacylase'},\n",
       "   'description': [\"<p>This family comprises of several D-Tyr-tRNA(Tyr) deacylase proteins.  Cell growth inhibition by several d-amino acids can be explained by an in vivo production of d-aminoacyl-tRNA molecules. Escherichia coli and yeast cells express an enzyme, d-Tyr-tRNA(Tyr) deacylase, capable of recycling such d-aminoacyl-tRNA molecules into free tRNA and d-amino acid. Accordingly, upon inactivation of the genes of the above deacylases, the toxicity of d-amino acids increases.  Orthologues of the deacylase are found in many cells [PMID:11568181].The D-aminoacyl-tRNA deacylase (DTD) enzyme is homodimeric with two active sites located at the dimeric interface. Each active site carries an invariant Gly-cisPro dipeptide motif in each monomer. The interaction between the dipeptide motifs from each monomer ensures substrate stereospecificity. This family also includes a subclass of DTDs which is present in Chordata and harbors a Gly-transPro motif. The cis to trans switch is the key to Animal DTDs (ATD) gaining of L-chiral selectivity. This 'gain of function' through relaxation of substrate chiral specificity underlies ATD's capability of correcting the error in tRNA selection [PMID:29410408].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00017393': {'PMID': 11568181,\n",
       "     'ISBN': None,\n",
       "     'volume': '276',\n",
       "     'issue': '50',\n",
       "     'year': 2001,\n",
       "     'title': 'Structure of crystalline D-Tyr-tRNA(Tyr) deacylase. A representative of a new class of tRNA-dependent hydrolases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '47285-90',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Ferri-Fioni ML',\n",
       "      'Schmitt E',\n",
       "      'Soutourina J',\n",
       "      'Plateau P',\n",
       "      'Mechulam Y',\n",
       "      'Blanquet S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M106550200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 129,\n",
       "    'interactions': 0,\n",
       "    'matches': 37849,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37771,\n",
       "    'proteomes': 8667,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32218, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 34575},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34070,\n",
       "    'alignment:full': 8414,\n",
       "    'alignment:seed': 799},\n",
       "   'cross_references': {}}},\n",
       " 'PF11929': {'metadata': {'accession': 'PF11929',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020683',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF3447)',\n",
       "    'short': 'DUF3447'},\n",
       "   'description': ['<p>This presumed domain is functionally uncharacterised. This domain is found in eukaryotes. This domain is about 80 amino acids in length.  This domain is found associated with Pfam:PF00023. This domain has a conserved SHN sequence motif. It seems likely that this region represents divergent Ankyrin repeats.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0465', 'name': 'Ank'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 246,\n",
       "    'interactions': 0,\n",
       "    'matches': 1348,\n",
       "    'pathways': 0,\n",
       "    'proteins': 1347,\n",
       "    'proteomes': 1,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1332, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 34},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 1342,\n",
       "    'alignment:full': 1340,\n",
       "    'alignment:seed': 575},\n",
       "   'cross_references': {}}},\n",
       " 'PF03796': {'metadata': {'accession': 'PF03796',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007694',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DnaB-like helicase C terminal domain', 'short': 'DnaB_C'},\n",
       "   'description': ['<p>The hexameric helicase DnaB unwinds the DNA duplex at the Escherichia coli chromosome replication fork. Although the mechanism by which DnaB both couples ATP hydrolysis to translocation along DNA and denatures the duplex is unknown, a change in the quaternary structure of the protein involving dimerisation of the N-terminal domain has been observed and may occur during the enzymatic cycle. This C-terminal domain contains an ATP-binding site and is therefore probably the site of ATP hydrolysis.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 219,\n",
       "    'interactions': 0,\n",
       "    'matches': 66941,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65024,\n",
       "    'proteomes': 10110,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 52104, 'rosettafold': 0},\n",
       "    'structures': 49,\n",
       "    'taxa': 44320},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61668,\n",
       "    'alignment:full': 11700,\n",
       "    'alignment:seed': 427},\n",
       "   'cross_references': {}}},\n",
       " 'PF00560': {'metadata': {'accession': 'PF00560',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001611',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Leucine Rich Repeat', 'short': 'LRR_1'},\n",
       "   'description': ['<p>CAUTION: This Pfam may not find all Leucine Rich Repeats in a protein.  Leucine Rich Repeats are short sequence motifs present in a number of proteins with diverse functions and cellular locations.  These repeats are usually involved in protein-protein interactions. Each Leucine Rich Repeat is composed of a beta-alpha unit. These units form elongated non-globular structures. Leucine Rich Repeats are often flanked by cysteine rich domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004163': {'PMID': 8264799,\n",
       "     'ISBN': None,\n",
       "     'volume': '366',\n",
       "     'issue': '6457',\n",
       "     'year': 1993,\n",
       "     'title': 'Crystal structure of porcine ribonuclease inhibitor, a protein with leucine-rich repeats.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '751-6',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Kobe B', 'Deisenhofer J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/366751a0'},\n",
       "    'PUB00005426': {'PMID': 7817399,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '10',\n",
       "     'year': 1994,\n",
       "     'title': 'The leucine-rich repeat: a versatile binding motif.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '415-21',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Kobe B', 'Deisenhofer J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0968-0004(94)90090-6'}},\n",
       "   'set_info': {'accession': 'CL0022', 'name': 'LRR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 6846,\n",
       "    'interactions': 0,\n",
       "    'matches': 97765,\n",
       "    'pathways': 0,\n",
       "    'proteins': 74351,\n",
       "    'proteomes': 1744,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 60283, 'rosettafold': 0},\n",
       "    'structures': 66,\n",
       "    'taxa': 9538},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 73985,\n",
       "    'alignment:full': 26691,\n",
       "    'alignment:seed': 2294},\n",
       "   'cross_references': {}}},\n",
       " 'PF10369': {'metadata': {'accession': 'PF10369',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019455',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Small subunit of acetolactate synthase',\n",
       "    'short': 'ALS_ss_C'},\n",
       "   'description': ['<p>ALS_ss_C is the C-terminal half of a family of proteins which are the small subunits of acetolactate synthase.  Acetolactate synthase is a tetrameric enzyme, containing probably two large and two small subunits, which catalyses the first step in branched-chain amino acid biosynthesis. This reaction is sensitive to certain herbicides [PMID:9197540].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044146': {'PMID': 9197540,\n",
       "     'ISBN': None,\n",
       "     'volume': '190',\n",
       "     'issue': '2',\n",
       "     'year': 1997,\n",
       "     'title': 'Identification of an acetolactate synthase small subunit gene in two eukaryotes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '245-9',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Duggleby RG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0378-1119(97)00002-4'}},\n",
       "   'set_info': {'accession': 'CL0070', 'name': 'ACT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 90,\n",
       "    'interactions': 0,\n",
       "    'matches': 37680,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36972,\n",
       "    'proteomes': 8460,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 32024, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 34458},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34224,\n",
       "    'alignment:full': 8911,\n",
       "    'alignment:seed': 592},\n",
       "   'cross_references': {}}},\n",
       " 'PF16192': {'metadata': {'accession': 'PF16192',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032421',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'C-terminal four TMM region of protein-O-mannosyltransferase',\n",
       "    'short': 'PMT_4TMC'},\n",
       "   'description': ['<p>PMT_4TMC is the C-terminal four membrane-pass region of protein-O-mannosyltransferases and similar enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 139,\n",
       "    'interactions': 0,\n",
       "    'matches': 15703,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15631,\n",
       "    'proteomes': 3778,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12983, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 13801},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13615,\n",
       "    'alignment:full': 5815,\n",
       "    'alignment:seed': 388},\n",
       "   'cross_references': {}}},\n",
       " 'PF13428': {'metadata': {'accession': 'PF13428',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetratricopeptide repeat', 'short': 'TPR_14'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2155,\n",
       "    'interactions': 0,\n",
       "    'matches': 10923,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10849,\n",
       "    'proteomes': 1939,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 8664, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 10090},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9118,\n",
       "    'alignment:full': 2498,\n",
       "    'alignment:seed': 389},\n",
       "   'cross_references': {}}},\n",
       " 'PF00472': {'metadata': {'accession': 'PF00472',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000352',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RF-1 domain', 'short': 'RF-1'},\n",
       "   'description': ['<p>This domain is found in peptide chain release factors such as RF-1 (Swiss:P07011) and RF-2 (Swiss:P07012), and a number of smaller proteins of unknown function such as Swiss:P40711. This domain contains the peptidyl-tRNA hydrolase activity. The domain contains a highly conserved motif GGQ, where the glutamine is thought to coordinate the water that mediates the hydrolysis.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007747': {'PMID': 10676813,\n",
       "     'ISBN': None,\n",
       "     'volume': '100',\n",
       "     'issue': '3',\n",
       "     'year': 2000,\n",
       "     'title': 'The crystal structure of human eukaryotic release factor eRF1--mechanism of stop codon recognition and peptidyl-tRNA hydrolysis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '311-21',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Song H',\n",
       "      'Mugnier P',\n",
       "      'Das AK',\n",
       "      'Webb HM',\n",
       "      'Evans DR',\n",
       "      'Tuite MF',\n",
       "      'Hemmings BA',\n",
       "      'Barford D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80667-4'}},\n",
       "   'set_info': {'accession': 'CL0337', 'name': 'RF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 295,\n",
       "    'interactions': 0,\n",
       "    'matches': 125380,\n",
       "    'pathways': 0,\n",
       "    'proteins': 125351,\n",
       "    'proteomes': 10921,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 106834, 'rosettafold': 0},\n",
       "    'structures': 131,\n",
       "    'taxa': 46028},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 112936,\n",
       "    'alignment:full': 26336,\n",
       "    'alignment:seed': 797},\n",
       "   'cross_references': {}}},\n",
       " 'PF00231': {'metadata': {'accession': 'PF00231',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000131',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase', 'short': 'ATP-synt'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004187': {'PMID': 8065448,\n",
       "     'ISBN': None,\n",
       "     'volume': '370',\n",
       "     'issue': '6491',\n",
       "     'year': 1994,\n",
       "     'title': 'Structure at 2.8 A resolution of F1-ATPase from bovine heart mitochondria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '621-8',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Abrahams JP', 'Leslie AG', 'Lutter R', 'Walker JE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/370621a0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 111,\n",
       "    'interactions': 0,\n",
       "    'matches': 50528,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50381,\n",
       "    'proteomes': 10400,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 43079, 'rosettafold': 0},\n",
       "    'structures': 280,\n",
       "    'taxa': 43219},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45327,\n",
       "    'alignment:full': 10861,\n",
       "    'alignment:seed': 851},\n",
       "   'cross_references': {}}},\n",
       " 'PF01546': {'metadata': {'accession': 'PF01546',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002933',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peptidase family M20/M25/M40', 'short': 'Peptidase_M20'},\n",
       "   'description': ['<p>This family includes a range of zinc metallopeptidases belonging to several families in the peptidase classification [PMID:7674922]. Family M20 are Glutamate carboxypeptidases.  Peptidase family M25 contains X-His dipeptidases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003579': {'PMID': 7674922,\n",
       "     'ISBN': None,\n",
       "     'volume': '248',\n",
       "     'issue': None,\n",
       "     'year': 1995,\n",
       "     'title': 'Evolutionary families of metallopeptidases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '183-228',\n",
       "     'medline_journal': 'Methods Enzymol',\n",
       "     'ISO_journal': 'Meth. Enzymol.',\n",
       "     'authors': ['Rawlings ND', 'Barrett AJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0076-6879(95)48015-3'}},\n",
       "   'set_info': {'accession': 'CL0035', 'name': 'Peptidase_MH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 745,\n",
       "    'interactions': 0,\n",
       "    'matches': 324066,\n",
       "    'pathways': 0,\n",
       "    'proteins': 323701,\n",
       "    'proteomes': 11003,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 279467, 'rosettafold': 0},\n",
       "    'structures': 80,\n",
       "    'taxa': 45837},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 302372,\n",
       "    'alignment:full': 70318,\n",
       "    'alignment:seed': 360},\n",
       "   'cross_references': {}}},\n",
       " 'PF02096': {'metadata': {'accession': 'PF02096',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR028055',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '60Kd inner membrane protein', 'short': '60KD_IMP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0376', 'name': 'Oxa1'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 198,\n",
       "    'interactions': 0,\n",
       "    'matches': 59566,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59475,\n",
       "    'proteomes': 10514,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 50912, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 43838},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54306,\n",
       "    'alignment:full': 12753,\n",
       "    'alignment:seed': 455},\n",
       "   'cross_references': {}}},\n",
       " 'PF04229': {'metadata': {'accession': 'PF04229',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007344',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GrpB protein', 'short': 'GrpB'},\n",
       "   'description': ['<p>This family has been suggested to belong to the nucleotidyltransferase superfamily [PMID:19833706].  It occurs at the C-terminus of dephospho-CoA kinase (CoaE) in a number of cases, where it plays a role in the proper folding of the enzyme [PMID:19876400].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0260', 'name': 'NTP_transf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 138,\n",
       "    'interactions': 0,\n",
       "    'matches': 14463,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14438,\n",
       "    'proteomes': 2505,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12733, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 11523},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14063,\n",
       "    'alignment:full': 3060,\n",
       "    'alignment:seed': 240},\n",
       "   'cross_references': {}}},\n",
       " 'PF01169': {'metadata': {'accession': 'PF01169',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001727',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterized protein family UPF0016',\n",
       "    'short': 'UPF0016'},\n",
       "   'description': ['<p>This family contains integral membrane proteins of unknown function. Most members of the family contain two copies of a region that contains an EXGD motif. Each of these regions contains three predicted transmembrane regions. It has been suggested that these proteins are calcium transporters [PMID:24955841].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0292', 'name': 'LysE'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 34584,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19367,\n",
       "    'proteomes': 4787,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16393, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 19647},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30787,\n",
       "    'alignment:full': 10303,\n",
       "    'alignment:seed': 614},\n",
       "   'cross_references': {}}},\n",
       " 'PF02201': {'metadata': {'accession': 'PF02201',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003121',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SWIB/MDM2 domain', 'short': 'SWIB'},\n",
       "   'description': ['<p>This family includes the SWIB domain and the MDM2 domain [PMID:12016060]. The p53-associated protein (MDM2) is an inhibitor of the p53 tumour suppressor gene binding the transactivation domain and down regulating the ability of p53 to activate transcription. This family contains the  p53 binding domain of MDM2 [PMID:8875929].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00013998': {'PMID': 8875929,\n",
       "     'ISBN': None,\n",
       "     'volume': '274',\n",
       "     'issue': '5289',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure of the MDM2 oncoprotein bound to the p53 tumor suppressor transactivation domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '948-53',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Kussie PH',\n",
       "      'Gorina S',\n",
       "      'Marechal V',\n",
       "      'Elenbaas B',\n",
       "      'Moreau J',\n",
       "      'Levine AJ',\n",
       "      'Pavletich NP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.274.5289.948'},\n",
       "    'PUB00014864': {'PMID': 12016060,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '4',\n",
       "     'year': 2002,\n",
       "     'title': 'The SWIB and the MDM2 domains are homologous and share a common fold.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '626-30',\n",
       "     'medline_journal': 'Bioinformatics',\n",
       "     'ISO_journal': 'Bioinformatics',\n",
       "     'authors': ['Bennett-Lovsey R', 'Hart SE', 'Shirai H', 'Mizuguchi K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/bioinformatics/18.4.626'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 251,\n",
       "    'interactions': 0,\n",
       "    'matches': 22299,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20758,\n",
       "    'proteomes': 2834,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16012, 'rosettafold': 0},\n",
       "    'structures': 178,\n",
       "    'taxa': 10838},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18097,\n",
       "    'alignment:full': 10046,\n",
       "    'alignment:seed': 333},\n",
       "   'cross_references': {}}},\n",
       " 'PF08148': {'metadata': {'accession': 'PF08148',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012961',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DSHCT (NUC185) domain', 'short': 'DSHCT'},\n",
       "   'description': ['<p>This C terminal domain is found in DOB1/SK12/helY-like DEAD box helicases [PMID:15112237].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016366': {'PMID': 15112237,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '5',\n",
       "     'year': 2004,\n",
       "     'title': 'Insights into the evolution of the nucleolus by an analysis of its protein domain repertoire.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '567-81',\n",
       "     'medline_journal': 'Bioessays',\n",
       "     'ISO_journal': 'Bioessays',\n",
       "     'authors': ['Staub E', 'Fiziev P', 'Rosenthal A', 'Hinzmann B.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/bies.20032'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 182,\n",
       "    'interactions': 0,\n",
       "    'matches': 17177,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17114,\n",
       "    'proteomes': 4106,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13154, 'rosettafold': 0},\n",
       "    'structures': 33,\n",
       "    'taxa': 15335},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14840,\n",
       "    'alignment:full': 6083,\n",
       "    'alignment:seed': 327},\n",
       "   'cross_references': {}}},\n",
       " 'PF18052': {'metadata': {'accession': 'PF18052',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041118',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Rx N-terminal domain', 'short': 'Rx_N'},\n",
       "   'description': ['<p>This entry represents the N-terminal domain found in many plant resistance proteins [PMID:24194517]. This domain has been predicted to be a coiled-coil, however the structure shows that it adopts a four helical bundle fold [PMID:24194517].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 843,\n",
       "    'interactions': 0,\n",
       "    'matches': 40071,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39366,\n",
       "    'proteomes': 196,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 30004, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 848},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37679,\n",
       "    'alignment:full': 27303,\n",
       "    'alignment:seed': 457},\n",
       "   'cross_references': {}}},\n",
       " 'PF00358': {'metadata': {'accession': 'PF00358',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001127',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'phosphoenolpyruvate-dependent sugar phosphotransferase system, EIIA 1',\n",
       "    'short': 'PTS_EIIA_1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0105', 'name': 'Hybrid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 63,\n",
       "    'interactions': 0,\n",
       "    'matches': 33398,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33383,\n",
       "    'proteomes': 2973,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30801, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 13005},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35791,\n",
       "    'alignment:full': 6197,\n",
       "    'alignment:seed': 606},\n",
       "   'cross_references': {}}},\n",
       " 'PF06026': {'metadata': {'accession': 'PF06026',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004788',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribose 5-phosphate isomerase A (phosphoriboisomerase A)',\n",
       "    'short': 'Rib_5-P_isom_A'},\n",
       "   'description': ['<p>This family consists of several ribose 5-phosphate isomerase A or phosphoriboisomerase A (EC:5.3.1.6) from bacteria, eukaryotes and archaea. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020248': {'PMID': 12211039,\n",
       "     'ISBN': None,\n",
       "     'volume': '48',\n",
       "     'issue': '4',\n",
       "     'year': 2002,\n",
       "     'title': 'Crystal structure of D-ribose-5-phosphate isomerase (RpiA) from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '737-40',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Rangarajan ES', 'Sivaraman J', 'Matte A', 'Cygler M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.10203'}},\n",
       "   'set_info': {'accession': 'CL0246', 'name': 'ISOCOT_Fold'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 100,\n",
       "    'interactions': 0,\n",
       "    'matches': 29178,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29092,\n",
       "    'proteomes': 6244,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25326, 'rosettafold': 0},\n",
       "    'structures': 31,\n",
       "    'taxa': 27617},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26810,\n",
       "    'alignment:full': 6591,\n",
       "    'alignment:seed': 468},\n",
       "   'cross_references': {}}},\n",
       " 'PF08439': {'metadata': {'accession': 'PF08439',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013647',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Oligopeptidase F', 'short': 'Peptidase_M3_N'},\n",
       "   'description': ['<p>This domain is found to the N-terminus of the Pfam:PF01432 domain in bacterial and archaeal proteins including Oligoendopeptidase F.  An example of this protein is Lactococcus lactis PepF [PMID:7798200].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00017380': {'PMID': 7798200,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '51',\n",
       "     'year': 1994,\n",
       "     'title': 'Biochemical and genetic characterization of PepF, an oligopeptidase from Lactococcus lactis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '32070-6',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Monnet V', 'Nardi M', 'Chopin A', 'Chopin MC', 'Gripon JC.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/51/32070'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 22340,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22339,\n",
       "    'proteomes': 3198,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19256, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 13814},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21048,\n",
       "    'alignment:full': 4248,\n",
       "    'alignment:seed': 412},\n",
       "   'cross_references': {}}},\n",
       " 'PF04087': {'metadata': {'accession': 'PF04087',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005240',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF389)', 'short': 'DUF389'},\n",
       "   'description': ['<p>Family of hypothetical bacterial proteins with  an undetermined function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 43,\n",
       "    'interactions': 0,\n",
       "    'matches': 13136,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13111,\n",
       "    'proteomes': 2573,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10998, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12010},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11195,\n",
       "    'alignment:full': 2793,\n",
       "    'alignment:seed': 278},\n",
       "   'cross_references': {}}},\n",
       " 'PF00100': {'metadata': {'accession': 'PF00100',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001507',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Zona pellucida-like domain', 'short': 'Zona_pellucida'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001633': {'PMID': 1313375,\n",
       "     'ISBN': None,\n",
       "     'volume': '300',\n",
       "     'issue': '3',\n",
       "     'year': 1992,\n",
       "     'title': 'A large domain common to sperm receptors (Zp2 and Zp3) and TGF-beta type III receptor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '237-40',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Bork P', 'Sander C.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0014-5793(92)80853-9'}},\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 792,\n",
       "    'interactions': 0,\n",
       "    'matches': 29441,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28853,\n",
       "    'proteomes': 902,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20515, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 5188},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22076,\n",
       "    'alignment:full': 12634,\n",
       "    'alignment:seed': 337},\n",
       "   'cross_references': {}}},\n",
       " 'PF03462': {'metadata': {'accession': 'PF03462',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005139',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PCRF domain', 'short': 'PCRF'},\n",
       "   'description': ['<p>This domain is found in peptide chain release factors.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 193,\n",
       "    'interactions': 0,\n",
       "    'matches': 95085,\n",
       "    'pathways': 0,\n",
       "    'proteins': 94936,\n",
       "    'proteomes': 10821,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 80834, 'rosettafold': 0},\n",
       "    'structures': 51,\n",
       "    'taxa': 45278},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 85430,\n",
       "    'alignment:full': 18927,\n",
       "    'alignment:seed': 868},\n",
       "   'cross_references': {}}},\n",
       " 'PF00221': {'metadata': {'accession': 'PF00221',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001106',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aromatic amino acid lyase', 'short': 'Lyase_aromatic'},\n",
       "   'description': ['<p>This family includes proteins with phenylalanine ammonia-lyase, EC:4.3.1.24, histidine ammonia-lyase, EC:4.3.1.3, and tyrosine aminomutase, EC:5.4.3.6, activities [1-3].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 146,\n",
       "    'interactions': 0,\n",
       "    'matches': 42968,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42079,\n",
       "    'proteomes': 5907,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36334, 'rosettafold': 0},\n",
       "    'structures': 47,\n",
       "    'taxa': 26123},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40395,\n",
       "    'alignment:full': 9345,\n",
       "    'alignment:seed': 419},\n",
       "   'cross_references': {}}},\n",
       " 'PF08378': {'metadata': {'accession': 'PF08378',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011528',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nuclease-related domain', 'short': 'NERD'},\n",
       "   'description': ['<p>The nuclease-related domain (NERD) is found in a range of bacterial as well as archaeal and plant proteins.  It has distant similarity to endonucleases (hence its name) and its predicted secondary structure is helix - sheet - sheet - sheet - sheet - weak sheet/long loop - helix - sheet - sheet. The majority of NERD-containing proteins are single-domain, but in several cases proteins containing NERD have additional domains which in 75% of cases are involved in DNA processing [PMID:15055202].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014771': {'PMID': 15055202,\n",
       "     'ISBN': None,\n",
       "     'volume': '29',\n",
       "     'issue': '3',\n",
       "     'year': 2004,\n",
       "     'title': 'NERD: a DNA processing-related domain present in the anthrax virulence plasmid, pXO1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '106-10',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Grynberg M', 'Godzik A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.tibs.2004.01.002'}},\n",
       "   'set_info': {'accession': 'CL0236', 'name': 'PDDEXK'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 159,\n",
       "    'interactions': 0,\n",
       "    'matches': 21892,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21889,\n",
       "    'proteomes': 3306,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17963, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15272},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20783,\n",
       "    'alignment:full': 5270,\n",
       "    'alignment:seed': 352},\n",
       "   'cross_references': {}}},\n",
       " 'PF16916': {'metadata': {'accession': 'PF16916',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027470',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dimerisation domain of Zinc Transporter',\n",
       "    'short': 'ZT_dimer'},\n",
       "   'description': ['<p>ZT_dimer is the dimerisation region of the whole molecule of zinc transporters since the full-length members form a homodimer during activity. The domain lies within the cytoplasm and exhibits an overall structural similarity with the copper metallochaperone Hah1 UniProtKB:O00244, exhibiting an open alpha-beta domain with two alpha helices (H1 and H2) aligned on one side and a three-stranded mixed beta-sheet (S1 to S3) on the other side. The N-terminal part of the members is the Cation_efflux family, Pfam:PF01545 [PMID:17717154].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 145,\n",
       "    'interactions': 0,\n",
       "    'matches': 50367,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47488,\n",
       "    'proteomes': 7709,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 40991, 'rosettafold': 0},\n",
       "    'structures': 64,\n",
       "    'taxa': 32707},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45528,\n",
       "    'alignment:full': 11206,\n",
       "    'alignment:seed': 362},\n",
       "   'cross_references': {}}},\n",
       " 'PF03120': {'metadata': {'accession': 'PF03120',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004150',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NAD-dependent DNA ligase OB-fold domain',\n",
       "    'short': 'DNA_ligase_OB'},\n",
       "   'description': ['<p>DNA ligases catalyse the crucial step of joining the breaks in duplex DNA during DNA replication, repair and recombination, utilising either ATP or NAD(+) as a cofactor [PMID:10698952]. This family is a small domain found after the adenylation domain Pfam:PF01653 in NAD dependent ligases [PMID:10698952]. OB-fold domains generally are involved in nucleic acid binding. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007386': {'PMID': 10698952,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '5',\n",
       "     'year': 2000,\n",
       "     'title': 'Crystal structure of NAD(+)-dependent DNA ligase: modular architecture and functional implications.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1119-29',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Lee JY',\n",
       "      'Chang C',\n",
       "      'Song HK',\n",
       "      'Moon J',\n",
       "      'Yang JK',\n",
       "      'Kim HK',\n",
       "      'Kwon ST',\n",
       "      'Suh SW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/19.5.1119'}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 171,\n",
       "    'interactions': 0,\n",
       "    'matches': 52581,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52580,\n",
       "    'proteomes': 8820,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 44755, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 37665},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49334,\n",
       "    'alignment:full': 8726,\n",
       "    'alignment:seed': 446},\n",
       "   'cross_references': {}}},\n",
       " 'PF06541': {'metadata': {'accession': 'PF06541',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010540',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative ABC-transporter type IV',\n",
       "    'short': 'ABC_trans_CmpB'},\n",
       "   'description': ['<p>CmpB is a family of membrane proteins that are likely to be part of a two-component type IV ABC-transporter system. Families can transport multiple drugs including ethidium and fluoroquinolones. UniProtKB:Q83XH0 is a member of TCDB family 3.A.1.121.4.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 10177,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9593,\n",
       "    'proteomes': 1594,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8360, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 6113},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9408,\n",
       "    'alignment:full': 2652,\n",
       "    'alignment:seed': 374},\n",
       "   'cross_references': {}}},\n",
       " 'PF02609': {'metadata': {'accession': 'PF02609',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003761',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Exonuclease VII small subunit', 'short': 'Exonuc_VII_S'},\n",
       "   'description': [\"<p>This family consist of exonuclease VII, small subunit EC:3.1.11.6 This enzyme catalyses exonucleolytic cleavage in either 5'->3' or 3'->5'  direction to yield 5'-phosphomononucleotides.  This exonuclease VII enzyme is composed of one large subunit and 4 small ones [PMID:6284744].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008230': {'PMID': 6284744,\n",
       "     'ISBN': None,\n",
       "     'volume': '257',\n",
       "     'issue': '15',\n",
       "     'year': 1982,\n",
       "     'title': 'Subunit structure of Escherichia coli exonuclease VII.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8799-805',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Vales LD', 'Rabin BA', 'Chase JW.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/257/15/8799'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 21,\n",
       "    'interactions': 0,\n",
       "    'matches': 37977,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37970,\n",
       "    'proteomes': 7869,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32642, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 32662},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34346,\n",
       "    'alignment:full': 7303,\n",
       "    'alignment:seed': 704},\n",
       "   'cross_references': {}}},\n",
       " 'PF01809': {'metadata': {'accession': 'PF01809',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002696',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative membrane protein insertion efficiency factor',\n",
       "    'short': 'YidD'},\n",
       "   'description': ['<p>This family consists of membrane insertion efficiency factor proteins. They contain three conserved cysteine residues. Family members such as YidD may be involved in insertion of integral membrane proteins into the membrane by assisting YidC (membrane protein insertase). Some members of the yidD family have been previously thought to posses alpha-hemolysin activity, however no sufficient evidence was found to corroborate this idea. Secondary structure prediction indicated the presence of three alpha-helices in YidD. None of the three alpha-helices appeared sufficiently hydrophobic to serve as a transmembrane, suggesting a cytoplasmic localization for YidD. However, a closer examination of the alpha-helical wheel projection of the predicted first alpha-helix in YidD suggested an amphipathic structure in its N-terminal region which might be involved in membrane targeting [PMID:21803992].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 46,\n",
       "    'interactions': 0,\n",
       "    'matches': 40274,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40273,\n",
       "    'proteomes': 7439,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34062, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 32638},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35660,\n",
       "    'alignment:full': 7158,\n",
       "    'alignment:seed': 649},\n",
       "   'cross_references': {}}},\n",
       " 'PF04117': {'metadata': {'accession': 'PF04117',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007248',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Mpv17 / PMP22 family', 'short': 'Mpv17_PMP22'},\n",
       "   'description': ['<p>The 22-kDa peroxisomal membrane protein (PMP22) is a major component of peroxisomal membranes. PMP22 seems to be involved in pore forming activity and may contribute to the unspecific permeability of the organelle membrane. PMP22 is synthesised on free cytosolic ribosomes and then directed to the peroxisome membrane by specific targeting information [PMID:11590176]. Mpv17 is a closely related peroxisomal protein. In mouse, the Mpv17 protein is involved in the development of early-onset glomerulosclerosis [PMID:11327696].  More recently a homolog of Mpv17 in S. cerevisiae has been been found to be an integral membrane protein of the inner mitochondrial membrane where it has been proposed to have a role in ethanol metabolism and tolerance during heat-shock [PMID:15189984]. Defects in MPV17 is associated with mitochondrial DNA depletion syndrome (MDDS) and Navajo neurohepatopathy (NNH) [PMID:16582910][PMID:16909392].  MDDS is a clinically heterogeneous group of disorders characterised by a reduction in mitochondrial DNA (mtDNA) copy number. Primary mtDNA depletion is inherited as an autosomal recessive trait and may affect single organs, typically muscle or liver, or multiple tissues.  Individuals with the hepatocerebral form of mitochondrial DNA depletion syndrome have early progressive liver failure and neurologic abnormalities, hypoglycemia, and increased lactate in body fluids.  NNH is an autosomal recessive disease that is prevalent among Navajo children in the South Western states of America.  The major clinical features are hepatopathy, peripheral neuropathy, corneal anesthesia and scarring, acral mutilation, cerebral leukoencephalopathy, failure to thrive, and recurrent metabolic acidosis with intercurrent infections. Infantile, childhood, and classic forms of NNH have been described. Mitochondrial DNA depletion was detected in the livers of patients, suggesting a primary defect in mtDNA maintenance [PMID:16909392].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009846': {'PMID': 11590176,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '1',\n",
       "     'year': 2002,\n",
       "     'title': 'Two different targeting signals direct human peroxisomal membrane protein 22 to peroxisomes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '774-84',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Brosius U', 'Dehmel T', 'Gartner J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M108155200'},\n",
       "    'PUB00009847': {'PMID': 11327696,\n",
       "     'ISBN': None,\n",
       "     'volume': '283',\n",
       "     'issue': '2',\n",
       "     'year': 2001,\n",
       "     'title': 'Cloning, mapping, genomic organization, and expression of mouse M-LP, a new member of the peroxisomal membrane protein Mpv17 domain family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '292-6',\n",
       "     'medline_journal': 'Biochem Biophys Res Commun',\n",
       "     'ISO_journal': 'Biochem. Biophys. Res. Commun.',\n",
       "     'authors': ['Iida R', 'Yasuda T', 'Tsubota E', 'Matsuki T', 'Kishi K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/bbrc.2001.4769'},\n",
       "    'PUB00015286': {'PMID': 15189984,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '3',\n",
       "     'year': 2004,\n",
       "     'title': 'SYM1 is the stress-induced Saccharomyces cerevisiae ortholog of the mammalian kidney disease gene Mpv17 and is required for ethanol metabolism and tolerance during heat shock.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '620-31',\n",
       "     'medline_journal': 'Eukaryot Cell',\n",
       "     'ISO_journal': 'Eukaryotic Cell',\n",
       "     'authors': ['Trott A', 'Morano KA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/EC.3.3.620-631.2004'},\n",
       "    'PUB00045033': {'PMID': 16582910,\n",
       "     'ISBN': None,\n",
       "     'volume': '38',\n",
       "     'issue': '5',\n",
       "     'year': 2006,\n",
       "     'title': 'MPV17 encodes an inner mitochondrial membrane protein and is mutated in infantile hepatic mitochondrial DNA depletion.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '570-5',\n",
       "     'medline_journal': 'Nat Genet',\n",
       "     'ISO_journal': 'Nat. Genet.',\n",
       "     'authors': ['Spinazzola A',\n",
       "      'Viscomi C',\n",
       "      'Fernandez-Vizarra E',\n",
       "      'Carrara F',\n",
       "      \"D'Adamo P\",\n",
       "      'Calvo S',\n",
       "      'Marsano RM',\n",
       "      'Donnini C',\n",
       "      'Weiher H',\n",
       "      'Strisciuglio P',\n",
       "      'Parini R',\n",
       "      'Sarzi E',\n",
       "      'Chan A',\n",
       "      'DiMauro S',\n",
       "      'Rotig A',\n",
       "      'Gasparini P',\n",
       "      'Ferrero I',\n",
       "      'Mootha VK',\n",
       "      'Tiranti V',\n",
       "      'Zeviani M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/ng1765'},\n",
       "    'PUB00045034': {'PMID': 16909392,\n",
       "     'ISBN': None,\n",
       "     'volume': '79',\n",
       "     'issue': '3',\n",
       "     'year': 2006,\n",
       "     'title': 'Navajo neurohepatopathy is caused by a mutation in the MPV17 gene.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '544-8',\n",
       "     'medline_journal': 'Am J Hum Genet',\n",
       "     'ISO_journal': 'Am. J. Hum. Genet.',\n",
       "     'authors': ['Karadimas CL',\n",
       "      'Vu TH',\n",
       "      'Holve SA',\n",
       "      'Chronopoulou P',\n",
       "      'Quinzii C',\n",
       "      'Johnsen SD',\n",
       "      'Kurth J',\n",
       "      'Eggers E',\n",
       "      'Palenzuela L',\n",
       "      'Tanji K',\n",
       "      'Bonilla E',\n",
       "      'De Vivo DC',\n",
       "      'DiMauro S',\n",
       "      'Hirano M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1086/506913'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 234,\n",
       "    'interactions': 0,\n",
       "    'matches': 16453,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16416,\n",
       "    'proteomes': 2080,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13733, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7767},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13671,\n",
       "    'alignment:full': 8332,\n",
       "    'alignment:seed': 502},\n",
       "   'cross_references': {}}},\n",
       " 'PF17921': {'metadata': {'accession': 'PF17921',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041588',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Integrase zinc binding domain',\n",
       "    'short': 'Integrase_H2C2'},\n",
       "   'description': ['<p>This zinc binding domain is found in a wide variety of integrase proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0361', 'name': 'C2H2-zf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4568,\n",
       "    'interactions': 0,\n",
       "    'matches': 91397,\n",
       "    'pathways': 0,\n",
       "    'proteins': 90620,\n",
       "    'proteomes': 1579,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 53252, 'rosettafold': 0},\n",
       "    'structures': 52,\n",
       "    'taxa': 6736},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 69648,\n",
       "    'alignment:full': 33149,\n",
       "    'alignment:seed': 475},\n",
       "   'cross_references': {}}},\n",
       " 'PF00238': {'metadata': {'accession': 'PF00238',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000218',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L14p/L23e', 'short': 'Ribosomal_L14'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 169,\n",
       "    'interactions': 0,\n",
       "    'matches': 65375,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65304,\n",
       "    'proteomes': 11105,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 56229, 'rosettafold': 0},\n",
       "    'structures': 1195,\n",
       "    'taxa': 66414},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57074,\n",
       "    'alignment:full': 12460,\n",
       "    'alignment:seed': 280},\n",
       "   'cross_references': {}}},\n",
       " 'PF01281': {'metadata': {'accession': 'PF01281',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020070',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L9, N-terminal domain',\n",
       "    'short': 'Ribosomal_L9_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001252': {'PMID': 8306963,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '1',\n",
       "     'year': 1994,\n",
       "     'title': 'Crystal structure of prokaryotic ribosomal protein L9: a bi-lobed RNA-binding protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '205-12',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Hoffman DW',\n",
       "      'Davies C',\n",
       "      'Gerchman SE',\n",
       "      'Kycia JH',\n",
       "      'Porter SJ',\n",
       "      'White SW',\n",
       "      'Ramakrishnan V.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8306963&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 107,\n",
       "    'interactions': 0,\n",
       "    'matches': 45239,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45237,\n",
       "    'proteomes': 9888,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38782, 'rosettafold': 0},\n",
       "    'structures': 681,\n",
       "    'taxa': 42233},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40679,\n",
       "    'alignment:full': 9418,\n",
       "    'alignment:seed': 527},\n",
       "   'cross_references': {}}},\n",
       " 'PF01774': {'metadata': {'accession': 'PF01774',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002669',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UreD urease accessory protein', 'short': 'UreD'},\n",
       "   'description': ['<p>UreD is a urease accessory protein. Urease Pfam:PF00449 hydrolyses  urea into ammonia and carbamic acid [PMID:8550495]. UreD is involved in  activation of the urease enzyme via the UreD-UreF-UreG-urease complex [PMID:9209019] and is required for urease nickel metallocenter assembly [PMID:7909161].  See also UreF Pfam:PF01730, UreG Pfam:PF01495. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002304': {'PMID': 8550495,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Organization of Ureaplasma urealyticum urease gene cluster and expression in a suppressor strain of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '647-55',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Neyrolles O',\n",
       "      'Ferris S',\n",
       "      'Behbahani N',\n",
       "      'Montagnier L',\n",
       "      'Blanchard A.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8550495&action=stream&blobtype=pdf'},\n",
       "    'PUB00002310': {'PMID': 9209019,\n",
       "     'ISBN': None,\n",
       "     'volume': '179',\n",
       "     'issue': '13',\n",
       "     'year': 1997,\n",
       "     'title': 'Characterization of UreG, identification of a UreD-UreF-UreG complex, and evidence suggesting that a nucleotide-binding site in UreG is required for in vivo metallocenter assembly of Klebsiella aerogenes urease.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4081-6',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Moncrief MB', 'Hausinger RP.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=9209019'},\n",
       "    'PUB00004836': {'PMID': 7909161,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '8',\n",
       "     'year': 1994,\n",
       "     'title': 'In vitro activation of urease apoprotein and role of UreD as a chaperone required for nickel metallocenter assembly.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3233-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Park IS', 'Carr MB', 'Hausinger RP.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=7909161'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 41,\n",
       "    'interactions': 0,\n",
       "    'matches': 17565,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17518,\n",
       "    'proteomes': 4054,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15484, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 17328},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17037,\n",
       "    'alignment:full': 4079,\n",
       "    'alignment:seed': 355},\n",
       "   'cross_references': {}}},\n",
       " 'PF08299': {'metadata': {'accession': 'PF08299',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013159',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial dnaA protein helix-turn-helix',\n",
       "    'short': 'Bac_DnaA_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 56,\n",
       "    'interactions': 0,\n",
       "    'matches': 48854,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48829,\n",
       "    'proteomes': 8549,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41513, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 37035},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44320,\n",
       "    'alignment:full': 8625,\n",
       "    'alignment:seed': 563},\n",
       "   'cross_references': {}}},\n",
       " 'PF02260': {'metadata': {'accession': 'PF02260',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003152',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FATC domain', 'short': 'FATC'},\n",
       "   'description': ['<p>The FATC domain is named after FRAP, ATM, TRRAP C-terminal [PMID:10782091]. The solution structure of the FATC domain suggests it plays a role in redox-dependent structural and cellular stability [PMID:15772072].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006515': {'PMID': 10782091,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '5',\n",
       "     'year': 2000,\n",
       "     'title': 'FAT: a novel domain in PIK-related kinases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '225-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Bosotti R', 'Isacchi A', 'Sonnhammer EL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)01563-2'},\n",
       "    'PUB00021047': {'PMID': 15772072,\n",
       "     'ISBN': None,\n",
       "     'volume': '280',\n",
       "     'issue': '21',\n",
       "     'year': 2005,\n",
       "     'title': 'The solution structure of the FATC domain of the protein kinase target of rapamycin suggests a role for redox-dependent structural and cellular stability.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '20558-64',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Dames SA',\n",
       "      'Mulet JM',\n",
       "      'Rathgeb-Szabo K',\n",
       "      'Hall MN',\n",
       "      'Grzesiek S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M501116200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 340,\n",
       "    'interactions': 0,\n",
       "    'matches': 14968,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14964,\n",
       "    'proteomes': 2135,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 1919, 'rosettafold': 0},\n",
       "    'structures': 98,\n",
       "    'taxa': 7860},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11796,\n",
       "    'alignment:full': 7433,\n",
       "    'alignment:seed': 289},\n",
       "   'cross_references': {}}},\n",
       " 'PF00468': {'metadata': {'accession': 'PF00468',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000271',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L34', 'short': 'Ribosomal_L34'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 37580,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37577,\n",
       "    'proteomes': 8919,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32074, 'rosettafold': 0},\n",
       "    'structures': 852,\n",
       "    'taxa': 38749},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33210,\n",
       "    'alignment:full': 8100,\n",
       "    'alignment:seed': 265},\n",
       "   'cross_references': {}}},\n",
       " 'PF00986': {'metadata': {'accession': 'PF00986',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002288',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DNA gyrase B subunit, carboxyl terminus',\n",
       "    'short': 'DNA_gyraseB_C'},\n",
       "   'description': ['<p>The amino terminus of eukaryotic and prokaryotic DNA topoisomerase II are similar, but they have a different carboxyl terminus. The amino-terminal portion of the DNA gyrase B  protein is thought to catalyse the ATP-dependent super-coiling of  DNA. See Pfam:PF00204. The carboxyl-terminal end supports the  complexation with the DNA gyrase A protein and the ATP-independent  relaxation. This family also contains Topoisomerase IV. This is a bacterial enzyme that is closely related to DNA gyrase, [PMID:7770916].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005437': {'PMID': 7770916,\n",
       "     'ISBN': None,\n",
       "     'volume': '20',\n",
       "     'issue': '4',\n",
       "     'year': 1995,\n",
       "     'title': 'The mechanisms of DNA topoisomerases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '156-60',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Roca J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)88993-8'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 117,\n",
       "    'interactions': 0,\n",
       "    'matches': 77525,\n",
       "    'pathways': 0,\n",
       "    'proteins': 77523,\n",
       "    'proteomes': 9194,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 66763, 'rosettafold': 0},\n",
       "    'structures': 75,\n",
       "    'taxa': 39998},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 71800,\n",
       "    'alignment:full': 14302,\n",
       "    'alignment:seed': 305},\n",
       "   'cross_references': {}}},\n",
       " 'PF00314': {'metadata': {'accession': 'PF00314',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001938',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Thaumatin family', 'short': 'Thaumatin'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0293', 'name': 'CDC'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 272,\n",
       "    'interactions': 0,\n",
       "    'matches': 13651,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13165,\n",
       "    'proteomes': 933,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11005, 'rosettafold': 0},\n",
       "    'structures': 182,\n",
       "    'taxa': 3831},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11331,\n",
       "    'alignment:full': 7308,\n",
       "    'alignment:seed': 261},\n",
       "   'cross_references': {}}},\n",
       " 'PF09118': {'metadata': {'accession': 'PF09118',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015202',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Galactose oxidase-like, Early set domain',\n",
       "    'short': 'GO-like_E_set'},\n",
       "   'description': [\"<p>E or 'early' set domains are associated with the catalytic domain of galactose oxidase at the C-terminal end. Galactose oxidase is an extracellular monomeric enzyme which catalyzes the stereospecific oxidation of a broad range of primary alcohol substrates, and possesses a unique mononuclear copper site essential for catalysing a two-electron transfer reaction during the oxidation of primary alcohols to corresponding aldehydes. The second redox active centre necessary for the reaction was found to be situated at a tyrosine residue. The C-terminal domain of galactose oxidase may be related to the immunoglobulin and/or fibronectin type III superfamilies. These domains are associated with different types of catalytic domains at either the N-terminal or C-terminal end, and may be involved in homodimeric/tetrameric/dodecameric interactions.  Members of this family adopt a secondary structure consisting of a bundle of seven, mostly antiparallel, beta-strands surrounding a hydrophobic core. The 7 strands are arranged in 2 sheets, in a Greek-key topology [PMID:11698678]. This domain is found in sugar-utilising enzymes, such as galactose oxidase or chitinase [2-6].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014151': {'PMID': 11698678,\n",
       "     'ISBN': None,\n",
       "     'volume': '98',\n",
       "     'issue': '23',\n",
       "     'year': 2001,\n",
       "     'title': 'Crystal structure of the precursor of galactose oxidase: an unusual self-processing enzyme.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '12932-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Firbank SJ',\n",
       "      'Rogers MS',\n",
       "      'Wilmot CM',\n",
       "      'Dooley DM',\n",
       "      'Halcrow MA',\n",
       "      'Knowles PF',\n",
       "      'McPherson MJ',\n",
       "      'Phillips SE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.231463798'}},\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 565,\n",
       "    'interactions': 0,\n",
       "    'matches': 10266,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10234,\n",
       "    'proteomes': 1551,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 8134, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 5761},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8587,\n",
       "    'alignment:full': 4476,\n",
       "    'alignment:seed': 347},\n",
       "   'cross_references': {}}},\n",
       " 'PF02417': {'metadata': {'accession': 'PF02417',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003370',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Chromate transporter', 'short': 'Chromate_transp'},\n",
       "   'description': ['<p>Members of this family probably act as chromate transporters [PMID:2152903, PMID:2180932].  Members of this family are found in both bacteria and archaebacteria.  The proteins are composed of one or two copies of this region. The alignment contains two conserved motifs, FGG and PGP.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009527': {'PMID': 2152903,\n",
       "     'ISBN': None,\n",
       "     'volume': '172',\n",
       "     'issue': '1',\n",
       "     'year': 1990,\n",
       "     'title': 'Cloning, nucleotide sequence, and expression of the chromate resistance determinant of Pseudomonas aeruginosa plasmid pUM505.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '287-91',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Cervantes C', 'Ohtake H', 'Chu L', 'Misra TK', 'Silver S.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=2152903&action=stream&blobtype=pdf'},\n",
       "    'PUB00009528': {'PMID': 2180932,\n",
       "     'ISBN': None,\n",
       "     'volume': '265',\n",
       "     'issue': '10',\n",
       "     'year': 1990,\n",
       "     'title': 'Nucleotide sequence and expression of a plasmid-encoded chromate resistance determinant from Alcaligenes eutrophus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5648-53',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Nies A', 'Nies DH', 'Silver S.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/265/10/5648'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 84,\n",
       "    'interactions': 0,\n",
       "    'matches': 63253,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43555,\n",
       "    'proteomes': 5309,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37635, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21803},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 58664,\n",
       "    'alignment:full': 13687,\n",
       "    'alignment:seed': 1321},\n",
       "   'cross_references': {}}},\n",
       " 'PF09995': {'metadata': {'accession': 'PF09995',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018713',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"ER-bound oxygenase mpaB/B'/Rubber oxygenase, catalytic domain\",\n",
       "    'short': 'MPAB_Lcp_cat'},\n",
       "   'description': [\"<p>This is the catalytic domain found in the endoplasmic reticulum (ER) -bound oxygenases mpaB' (MPAB2) and mpaB (MPAB) from Penicillium roqueforti and Penicillium brevicompactum and in the rubber oxygenase (Lcp) from Streptomyces sp., which contains highly conserved arginine and histidine residues. Structural analysis from Lcp revealed that Arg164, Thr168 and His198 are crucial active site residues [PMID:28733658]. The mpaB and mpaB' are part of the gene cluster that mediates the biosynthesis of mycophenolic acid (MPA) [PMID:31209052]. Lcp (Latex clearing proteins) is a rubber oxygenase that catalyses the extracellular cleavage of poly (cis-1,4-isoprene) [PMID:28733658, PMID:25819959]. This domain is also present in uncharacterised proteins from Mycobacterium sp. and hypothetical proteins, mainly from bacteria and fungi.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 104,\n",
       "    'interactions': 0,\n",
       "    'matches': 27032,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27005,\n",
       "    'proteomes': 3140,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 24130, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 12776},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26078,\n",
       "    'alignment:full': 6772,\n",
       "    'alignment:seed': 243},\n",
       "   'cross_references': {}}},\n",
       " 'PF00237': {'metadata': {'accession': 'PF00237',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001063',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L22p/L17e', 'short': 'Ribosomal_L22'},\n",
       "   'description': ['<p>This family includes L22 from prokaryotes and chloroplasts and L17 from eukaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005299': {'PMID': 9862810,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '12',\n",
       "     'year': 1998,\n",
       "     'title': 'The crystal structure of ribosomal protein L22 from Thermus thermophilus: insights into the mechanism of erythromycin resistance.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1577-86',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Unge J',\n",
       "      'berg A',\n",
       "      'Al-Kharadaghi S',\n",
       "      'Nikulin A',\n",
       "      'Nikonov S',\n",
       "      'Davydova N',\n",
       "      'Nevskaya N',\n",
       "      'Garber M',\n",
       "      'Liljas A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(98)00155-5'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 185,\n",
       "    'interactions': 0,\n",
       "    'matches': 65992,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65952,\n",
       "    'proteomes': 11119,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 56415, 'rosettafold': 0},\n",
       "    'structures': 1188,\n",
       "    'taxa': 64048},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57374,\n",
       "    'alignment:full': 12905,\n",
       "    'alignment:seed': 702},\n",
       "   'cross_references': {}}},\n",
       " 'PF13089': {'metadata': {'accession': 'PF13089',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025198',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polyphosphate kinase N-terminal domain',\n",
       "    'short': 'PP_kinase_N'},\n",
       "   'description': ['<p>Polyphosphate kinase (Ppk) catalyses the formation of polyphosphate from ATP, with chain lengths of up to a thousand or more orthophosphate molecules.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 34160,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34160,\n",
       "    'proteomes': 6757,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 29578, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 27781},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31671,\n",
       "    'alignment:full': 6599,\n",
       "    'alignment:seed': 539},\n",
       "   'cross_references': {}}},\n",
       " 'PF03619': {'metadata': {'accession': 'PF03619',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005178',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Organic solute transporter Ostalpha',\n",
       "    'short': 'Solute_trans_a'},\n",
       "   'description': ['<p>This family is a transmembrane organic solute transport protein. In vertebrates these proteins form a complex with Ostbeta, and function as bile transporters [PMID:15563450]. In plants they may transport brassinosteroid-like compounds and act as regulators of cell death [PMID:20830211].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0192', 'name': 'GPCR_A'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 164,\n",
       "    'interactions': 0,\n",
       "    'matches': 14572,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14235,\n",
       "    'proteomes': 2095,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11433, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7703},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11577,\n",
       "    'alignment:full': 7010,\n",
       "    'alignment:seed': 263},\n",
       "   'cross_references': {}}},\n",
       " 'PF16189': {'metadata': {'accession': 'PF16189',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Creatinase/Prolidase N-terminal domain',\n",
       "    'short': 'Creatinase_N_2'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0356', 'name': 'AMP_N-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 147,\n",
       "    'interactions': 0,\n",
       "    'matches': 20778,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20724,\n",
       "    'proteomes': 4828,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17433, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 19032},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18711,\n",
       "    'alignment:full': 5935,\n",
       "    'alignment:seed': 554},\n",
       "   'cross_references': {}}},\n",
       " 'PF04515': {'metadata': {'accession': 'PF04515',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007603',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plasma-membrane choline transporter',\n",
       "    'short': 'Choline_transpo'},\n",
       "   'description': ['<p>This family represents a high-affinity plasma-membrane choline transporter in C.elegans which is thought to be rate-limiting for ACh synthesis in cholinergic nerve terminals [PMID:15002745].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 186,\n",
       "    'interactions': 0,\n",
       "    'matches': 16987,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16565,\n",
       "    'proteomes': 2121,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12507, 'rosettafold': 1},\n",
       "    'structures': 1,\n",
       "    'taxa': 7822},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12458,\n",
       "    'alignment:full': 7794,\n",
       "    'alignment:seed': 265},\n",
       "   'cross_references': {}}},\n",
       " 'PF03483': {'metadata': {'accession': 'PF03483',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005146',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'B3/4 domain', 'short': 'B3_4'},\n",
       "   'description': ['<p>This domain is found in tRNA synthetase beta subunits as well as in some non tRNA synthetase proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007363': {'PMID': 10447505,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '8',\n",
       "     'year': 1999,\n",
       "     'title': 'Evolution of aminoacyl-tRNA synthetases--analysis of unique domain architectures and phylogenetic trees reveals a complex history of horizontal gene transfer events.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '689-710',\n",
       "     'medline_journal': 'Genome Res',\n",
       "     'ISO_journal': 'Genome Res.',\n",
       "     'authors': ['Wolf YI', 'Aravind L', 'Grishin NV', 'Koonin EV.'],\n",
       "     'DOI_URL': 'http://www.genome.org/cgi/content/abstract/9/8/689'}},\n",
       "   'set_info': {'accession': 'CL0383', 'name': 'PheT-TilS'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 193,\n",
       "    'interactions': 0,\n",
       "    'matches': 63126,\n",
       "    'pathways': 0,\n",
       "    'proteins': 63099,\n",
       "    'proteomes': 10816,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 53490, 'rosettafold': 0},\n",
       "    'structures': 40,\n",
       "    'taxa': 45428},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57231,\n",
       "    'alignment:full': 11977,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF03937': {'metadata': {'accession': 'PF03937',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005631',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flavinator of succinate dehydrogenase', 'short': 'Sdh5'},\n",
       "   'description': ['<p>This family includes the highly conserved mitochondrial and bacterial proteins Sdh5/SDHAF2/SdhE. Both yeast and human Sdh5/SDHAF2 interact with the catalytic subunit of the succinate dehydrogenase (SDH) complex, a component of both the electron transport chain and the tricarboxylic acid cycle. Sdh5 is required for SDH-dependent respiration and for Sdh1 flavination (incorporation of the flavin adenine dinucleotide cofactor). Mutational inactivation of Sdh5 confers tumor susceptibility in humans [PMID:19628817]. Bacterial homologues of Sdh5, termed SdhE, are functionally conserved being required for the flavinylation of SdhA and succinate dehydrogenase activity. Like Sdh5, SdhE interacts with SdhA. Furthermore, SdhE was characterised as a FAD co-factor chaperone that directly binds FAD to facilitate the flavinylation of SdhA. Phylogenetic analysis demonstrates that SdhE/Sdh5 proteins evolved only once in an ancestral alpha-proteobacteria prior to the evolution of the mitochondria and now remain in subsequent descendants including eukaryotic mitochondria and the alpha, beta and gamma proteobacteria [PMID:22474332].  This family was previously annotated in Pfam as being a divergent TPR repeat but structural evidence has indicated this is not true. The E. coli protein, YgfY also acts as the antitoxin to the membrane-bound toxin family Cpta, Pfam:PF13166, whose E. coli member YgfX, expressed from the same operon as YgfY [PMID:22239607].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 65,\n",
       "    'interactions': 0,\n",
       "    'matches': 17127,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17120,\n",
       "    'proteomes': 4856,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14871, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 19833},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15845,\n",
       "    'alignment:full': 4412,\n",
       "    'alignment:seed': 244},\n",
       "   'cross_references': {}}},\n",
       " 'PF13720': {'metadata': {'accession': 'PF13720',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029098',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Udp N-acetylglucosamine O-acyltransferase; Domain 2',\n",
       "    'short': 'Acetyltransf_11'},\n",
       "   'description': ['<p>This is domain 2, or the C-terminal domain, of Udp N-acetylglucosamine O-acyltransferase.  This enzyme is a zinc-dependent enzyme that catalyses the deacetylation of UDP-3-O-((R)-3-hydroxymyristoyl)-N-acetylglucosamine to form UDP-3-O-(R-hydroxymyristoyl)glucosamine and acetate.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 58,\n",
       "    'interactions': 0,\n",
       "    'matches': 28547,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28547,\n",
       "    'proteomes': 4786,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 24188, 'rosettafold': 0},\n",
       "    'structures': 46,\n",
       "    'taxa': 22328},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25595,\n",
       "    'alignment:full': 4884,\n",
       "    'alignment:seed': 465},\n",
       "   'cross_references': {}}},\n",
       " 'PF06201': {'metadata': {'accession': 'PF06201',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010400',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PITH domain', 'short': 'PITH'},\n",
       "   'description': ['<p>This family was formerly known as DUF1000. The full-length, Txnl1, protein which is a probable component of the 26S proteasome, uses its C-terminal, PITH, domain to associate specifically with the 26S proteasome. PITH derives from proteasome-interacting thioredoxin domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0202', 'name': 'GBD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 147,\n",
       "    'interactions': 0,\n",
       "    'matches': 7255,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7224,\n",
       "    'proteomes': 2105,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 6029, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 7712},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 5916,\n",
       "    'alignment:full': 3545,\n",
       "    'alignment:seed': 298},\n",
       "   'cross_references': {}}},\n",
       " 'PF16188': {'metadata': {'accession': 'PF16188',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032416',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'C-terminal region of peptidase_M24',\n",
       "    'short': 'Peptidase_M24_C'},\n",
       "   'description': ['<p>This is a short region at the C-terminus of a number of metallo-peptidases of the M24 family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 115,\n",
       "    'interactions': 0,\n",
       "    'matches': 20132,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20111,\n",
       "    'proteomes': 4822,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16952, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 18993},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18176,\n",
       "    'alignment:full': 5755,\n",
       "    'alignment:seed': 532},\n",
       "   'cross_references': {}}},\n",
       " 'PF05649': {'metadata': {'accession': 'PF05649',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008753',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peptidase family M13', 'short': 'Peptidase_M13_N'},\n",
       "   'description': ['<p>M13 peptidases are well-studied proteases found in a wide range of organisms including mammals and bacteria. In mammals they participate in processes such as cardiovascular development, blood-pressure  regulation, nervous control of respiration, and regulation of the function of neuropeptides in the central nervous system. In bacteria they may be used for digestion of milk. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003579': {'PMID': 7674922,\n",
       "     'ISBN': None,\n",
       "     'volume': '248',\n",
       "     'issue': None,\n",
       "     'year': 1995,\n",
       "     'title': 'Evolutionary families of metallopeptidases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '183-228',\n",
       "     'medline_journal': 'Methods Enzymol',\n",
       "     'ISO_journal': 'Meth. Enzymol.',\n",
       "     'authors': ['Rawlings ND', 'Barrett AJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0076-6879(95)48015-3'},\n",
       "    'PUB00011643': {'PMID': 11223883,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '3',\n",
       "     'year': 2001,\n",
       "     'title': 'The neprilysin (NEP) family of zinc metalloendopeptidases: genomics and function.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '261-9',\n",
       "     'medline_journal': 'Bioessays',\n",
       "     'ISO_journal': 'Bioessays',\n",
       "     'authors': ['Turner AJ', 'Isaac RE', 'Coates D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/1521-1878(200103)23:3<261::AID-BIES1036>3.0.CO;2-K'},\n",
       "    'PUB00020044': {'PMID': 12001226,\n",
       "     'ISBN': None,\n",
       "     'volume': '47',\n",
       "     'issue': '4',\n",
       "     'year': 2002,\n",
       "     'title': 'M13 endopeptidases: New conserved motifs correlated with structure, and simultaneous phylogenetic occurrence of PHEX and the bony fish.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '481-8',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Bianchetti L', 'Oudet C', 'Poch O.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.10075'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 322,\n",
       "    'interactions': 0,\n",
       "    'matches': 42417,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40854,\n",
       "    'proteomes': 4667,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33402, 'rosettafold': 0},\n",
       "    'structures': 24,\n",
       "    'taxa': 18546},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35454,\n",
       "    'alignment:full': 12661,\n",
       "    'alignment:seed': 726},\n",
       "   'cross_references': {}}},\n",
       " 'PF02773': {'metadata': {'accession': 'PF02773',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022630',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'S-adenosylmethionine synthetase, C-terminal domain',\n",
       "    'short': 'S-AdoMet_synt_C'},\n",
       "   'description': ['<p>The three domains of S-adenosylmethionine synthetase have the same alpha+beta fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020285': {'PMID': 8611562,\n",
       "     'ISBN': None,\n",
       "     'volume': '35',\n",
       "     'issue': '8',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure and function of S-adenosylmethionine synthetase: crystal structures of S-adenosylmethionine synthetase with ADP, BrADP, and PPi at 28 angstroms resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2586-96',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Takusagawa F', 'Kamitori S', 'Markham GD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi952604z'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 103,\n",
       "    'interactions': 0,\n",
       "    'matches': 55009,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54915,\n",
       "    'proteomes': 10732,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46440, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 44550},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49038,\n",
       "    'alignment:full': 12124,\n",
       "    'alignment:seed': 302},\n",
       "   'cross_references': {}}},\n",
       " 'PF16658': {'metadata': {'accession': 'PF16658',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032090',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Class II release factor RF3, C-terminal domain',\n",
       "    'short': 'RF3_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0437', 'name': 'EF-G_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 13,\n",
       "    'interactions': 0,\n",
       "    'matches': 27420,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27420,\n",
       "    'proteomes': 6103,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 23797, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 25214},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25671,\n",
       "    'alignment:full': 5681,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF01219': {'metadata': {'accession': 'PF01219',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000829',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Prokaryotic diacylglycerol kinase',\n",
       "    'short': 'DAGK_prokar'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 24905,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24905,\n",
       "    'proteomes': 4393,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21856, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 21620},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23547,\n",
       "    'alignment:full': 4396,\n",
       "    'alignment:seed': 463},\n",
       "   'cross_references': {}}},\n",
       " 'PF01084': {'metadata': {'accession': 'PF01084',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001648',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S18', 'short': 'Ribosomal_S18'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 60423,\n",
       "    'pathways': 0,\n",
       "    'proteins': 60421,\n",
       "    'proteomes': 10637,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 51604, 'rosettafold': 0},\n",
       "    'structures': 917,\n",
       "    'taxa': 62112},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52712,\n",
       "    'alignment:full': 11128,\n",
       "    'alignment:seed': 340},\n",
       "   'cross_references': {}}},\n",
       " 'PF02547': {'metadata': {'accession': 'PF02547',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003699',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Queuosine biosynthesis protein',\n",
       "    'short': 'Queuosine_synth'},\n",
       "   'description': ['<p>Queuosine (Q) biosynthesis protein, or S-adenosylmethionine:tRNA -ribosyltransferase-isomerase, is required for the synthesis of the queuosine precursor (oQ). It catalyses the transfer and isomerisation of the ribose moiety from AdoMet to the 7-aminomethyl group of 7-deazaguanine (preQ1-tRNA) to form epoxyqueuosine (oQ-tRNA).  Q is a hypermodified nucleoside usually found at the first position of the anticodon of asparagine, aspartate, histidine, and tyrosine tRNAs [PMID:8347586, PMID:12731872]. In Streptococcus gordonii , QueA has been shown to play a role in the regulation of arginine deiminase genes [PMID:18552185].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008181': {'PMID': 8347586,\n",
       "     'ISBN': None,\n",
       "     'volume': '32',\n",
       "     'issue': '30',\n",
       "     'year': 1993,\n",
       "     'title': 'A new function of S-adenosylmethionine: the ribosyl moiety of AdoMet is the precursor of the cyclopentenediol moiety of the tRNA wobble base queuine.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7811-7',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Slany RK', 'Bosl M', 'Crain PF', 'Kersten H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00081a028'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 44342,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44188,\n",
       "    'proteomes': 7114,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37417, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 30963},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39969,\n",
       "    'alignment:full': 7286,\n",
       "    'alignment:seed': 594},\n",
       "   'cross_references': {}}},\n",
       " 'PF12399': {'metadata': {'accession': 'PF12399',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032823',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Branched-chain amino acid ATP-binding cassette transporter',\n",
       "    'short': 'BCA_ABC_TP_C'},\n",
       "   'description': ['<p>This domain family is found in bacteria, archaea and eukaryotes, and is approximately 30 amino acids in length. The family is found in association with Pfam:PF00005. There is a conserved AYLG sequence motif. This family is the C terminal of an ATP dependent branched-chain amino acid transporter [PMID:2195019]. This domain is essential for LPS transport, through critical interactions with Walker A and switch helix domains [PMID:31431556].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 53,\n",
       "    'interactions': 0,\n",
       "    'matches': 108479,\n",
       "    'pathways': 0,\n",
       "    'proteins': 108476,\n",
       "    'proteomes': 5956,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 93599, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 26329},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 99903,\n",
       "    'alignment:full': 20683,\n",
       "    'alignment:seed': 507},\n",
       "   'cross_references': {}}},\n",
       " 'PF05025': {'metadata': {'accession': 'PF05025',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007721',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RbsD / FucU transport protein family',\n",
       "    'short': 'RbsD_FucU'},\n",
       "   'description': ['<p>The Escherichia coli high-affinity ribose-transport system consists of six proteins encoded by the rbs operon (rbsD, rbsA, rbsC, rbsB, rbsK and rbsR). RbsD was originally thought to be a high affinity ribose transport protein, but further analysis [PMID:16731978] shows that it is a D-ribose pyranase . It catalyzes the interconversion of beta-pyran and beta-furan forms of D-ribose. It also catalyzes the conversion between beta-allofuranose and beta-allopyranose. This family also includes FucU a component of the fucose operon and is a L-fucose mutarotase, involved in the anomeric conversion of L-fucose. It also exhibits a pyranase activity for D-ribose [PMID:16731978]. Both have been classified in the RbsD/FucU family of proteins. Members of this family are ubiquitous having been found in organisms from eubacteria to mammals.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010214': {'PMID': 11320319,\n",
       "     'ISBN': None,\n",
       "     'volume': '57',\n",
       "     'issue': 'Pt 5',\n",
       "     'year': 2001,\n",
       "     'title': 'Crystallization and preliminary X-ray crystallographic analysis of Escherichia coli RbsD, a component of the ribose-transport system with unknown biochemical function.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '728-30',\n",
       "     'medline_journal': 'Acta Crystallogr D Biol Crystallogr',\n",
       "     'ISO_journal': 'Acta Crystallogr. D Biol. Crystallogr.',\n",
       "     'authors': ['Kim MS', 'Oh H', 'Park C', 'Oh BH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1107/S0907444901003109'}},\n",
       "   'set_info': {'accession': 'CL0101', 'name': 'PELOTA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 27,\n",
       "    'interactions': 0,\n",
       "    'matches': 16524,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16513,\n",
       "    'proteomes': 2703,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14701, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 13227},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16451,\n",
       "    'alignment:full': 3148,\n",
       "    'alignment:seed': 246},\n",
       "   'cross_references': {}}},\n",
       " 'PF06912': {'metadata': {'accession': 'PF06912',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010699',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1275)',\n",
       "    'short': 'DUF1275'},\n",
       "   'description': ['<p>This family consists of several hypothetical bacterial proteins of around 200 residues in length. The function of this family is unknown although most members have 6 TM regions, and may be putative permeases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0142', 'name': 'Membrane_trans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 85,\n",
       "    'interactions': 0,\n",
       "    'matches': 22360,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22307,\n",
       "    'proteomes': 3797,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20125, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 16254},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21856,\n",
       "    'alignment:full': 5262,\n",
       "    'alignment:seed': 528},\n",
       "   'cross_references': {}}},\n",
       " 'PF01252': {'metadata': {'accession': 'PF01252',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001872',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Signal peptidase (SPase) II', 'short': 'Peptidase_A8'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 64,\n",
       "    'interactions': 0,\n",
       "    'matches': 50031,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50020,\n",
       "    'proteomes': 8542,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 42873, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 35700},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45374,\n",
       "    'alignment:full': 9288,\n",
       "    'alignment:seed': 684},\n",
       "   'cross_references': {}}},\n",
       " 'PF10442': {'metadata': {'accession': 'PF10442',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019494',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FIST C domain', 'short': 'FIST_C'},\n",
       "   'description': ['<p>The FIST C domain is a novel sensory domain, which is present in signal transduction proteins from Bacteria, Archaea and Eukarya. Chromosomal proximity of FIST-encoding genes to those coding for proteins involved in amino acid metabolism and transport suggest that FIST domains bind small ligands, such as amino acids [PMID:17855421].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044261': {'PMID': 17855421,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '19',\n",
       "     'year': 2007,\n",
       "     'title': 'FIST: a sensory domain for diverse signal transduction pathways in prokaryotes and ubiquitin signaling in eukaryotes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2518-21',\n",
       "     'medline_journal': 'Bioinformatics',\n",
       "     'ISO_journal': 'Bioinformatics',\n",
       "     'authors': ['Borziak K', 'Zhulin IB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/bioinformatics/btm384'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 157,\n",
       "    'interactions': 0,\n",
       "    'matches': 15772,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15763,\n",
       "    'proteomes': 2967,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12978, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13648},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13583,\n",
       "    'alignment:full': 3204,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF09298': {'metadata': {'accession': 'PF09298',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015377',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fumarylacetoacetase N-terminal',\n",
       "    'short': 'FAA_hydrolase_N'},\n",
       "   'description': ['<p>The N-terminal domain of fumarylacetoacetate hydrolase is functionally uncharacterised, and adopts a structure consisting of an SH3-like barrel [PMID:11154690].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00025833': {'PMID': 11154690,\n",
       "     'ISBN': None,\n",
       "     'volume': '276',\n",
       "     'issue': '18',\n",
       "     'year': 2001,\n",
       "     'title': 'Mechanistic inferences from the crystal structure of fumarylacetoacetate hydrolase with a bound phosphorus-based inhibitor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15284-91',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Bateman RL',\n",
       "      'Bhanumoorthy P',\n",
       "      'Witte JF',\n",
       "      'McClard RW',\n",
       "      'Grompe M',\n",
       "      'Timm DE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M007621200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 98,\n",
       "    'interactions': 0,\n",
       "    'matches': 14119,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14109,\n",
       "    'proteomes': 3904,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11976, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 14962},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12749,\n",
       "    'alignment:full': 4103,\n",
       "    'alignment:seed': 240},\n",
       "   'cross_references': {}}},\n",
       " 'PF00281': {'metadata': {'accession': 'PF00281',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031310',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L5', 'short': 'Ribosomal_L5'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0652', 'name': 'S24e_L23_L15e'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 124,\n",
       "    'interactions': 0,\n",
       "    'matches': 51219,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51215,\n",
       "    'proteomes': 11019,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 44109, 'rosettafold': 0},\n",
       "    'structures': 1070,\n",
       "    'taxa': 48913},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45673,\n",
       "    'alignment:full': 11001,\n",
       "    'alignment:seed': 490},\n",
       "   'cross_references': {}}},\n",
       " 'PF01253': {'metadata': {'accession': 'PF01253',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001950',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Translation initiation factor SUI1', 'short': 'SUI1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 202,\n",
       "    'interactions': 0,\n",
       "    'matches': 27817,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27761,\n",
       "    'proteomes': 4632,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23276, 'rosettafold': 0},\n",
       "    'structures': 33,\n",
       "    'taxa': 21207},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23826,\n",
       "    'alignment:full': 8542,\n",
       "    'alignment:seed': 337},\n",
       "   'cross_references': {}}},\n",
       " 'PF02965': {'metadata': {'accession': 'PF02965',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004223',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Vitamin B12 dependent methionine synthase, activation domain',\n",
       "    'short': 'Met_synt_B12'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016227': {'PMID': 8939751,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '11',\n",
       "     'year': 1996,\n",
       "     'title': 'The structure of the C-terminal domain of methionine synthase: presenting S-adenosylmethionine for reductive methylation of B12.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1263-75',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Dixon MM', 'Huang S', 'Matthews RG', 'Ludwig M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(96)00135-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 108,\n",
       "    'interactions': 0,\n",
       "    'matches': 32450,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32316,\n",
       "    'proteomes': 6352,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26696, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 26576},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29530,\n",
       "    'alignment:full': 6076,\n",
       "    'alignment:seed': 287},\n",
       "   'cross_references': {}}},\n",
       " 'PF02518': {'metadata': {'accession': 'PF02518',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003594',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Histidine kinase-, DNA gyrase B-, and HSP90-like ATPase',\n",
       "    'short': 'HATPase_c'},\n",
       "   'description': ['<p>This family represents the structurally related ATPase domains of histidine kinase, DNA gyrase B and HSP90.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0025', 'name': 'His_Kinase_A'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 39193,\n",
       "    'interactions': 0,\n",
       "    'matches': 1519005,\n",
       "    'pathways': 0,\n",
       "    'proteins': 1514289,\n",
       "    'proteomes': 11587,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1235780, 'rosettafold': 0},\n",
       "    'structures': 770,\n",
       "    'taxa': 56134},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 1393617,\n",
       "    'alignment:full': 300774,\n",
       "    'alignment:seed': 658},\n",
       "   'cross_references': {}}},\n",
       " 'PF05257': {'metadata': {'accession': 'PF05257',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007921',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CHAP domain', 'short': 'CHAP'},\n",
       "   'description': ['<p>This domain corresponds to an amidase function.  Many of these proteins are involved in cell wall metabolism of bacteria. This domain is found at the N-terminus of Swiss:P43675, where it functions as a glutathionylspermidine amidase EC:3.5.1.78 [PMID:7775463]. This domain is found to be the catalytic domain of PlyCA [PMID:16818874]. CHAP is the amidase domain of bifunctional Escherichia coli glutathionylspermidine synthetase/amidase, and it catalyses the hydrolysis of Gsp (glutathionylspermidine) into glutathione and spermidine [PMID:21226054].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008407': {'PMID': 7775463,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '23',\n",
       "     'year': 1995,\n",
       "     'title': 'Glutathionylspermidine metabolism in Escherichia coli. Purification, cloning, overproduction, and characterization of a bifunctional glutathionylspermidine synthetase/amidase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14031-41',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Bollinger JM Jr',\n",
       "      'Kwon DS',\n",
       "      'Huisman GW',\n",
       "      'Kolter R',\n",
       "      'Walsh CT.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.270.23.14031'},\n",
       "    'PUB00011839': {'PMID': 12765833,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '5',\n",
       "     'year': 2003,\n",
       "     'title': 'Amidase domains from bacterial and phage autolysins define a family of gamma-D,L-glutamate-specific amidohydrolases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '230-4',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Rigden DJ', 'Jedrzejas MJ', 'Galperin MY.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(03)00062-8'},\n",
       "    'PUB00011840': {'PMID': 12765834,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '5',\n",
       "     'year': 2003,\n",
       "     'title': 'The CHAP domain: a large family of amidases including GSP amidase and peptidoglycan hydrolases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '234-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Bateman A', 'Rawlings ND.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(03)00061-6'}},\n",
       "   'set_info': {'accession': 'CL0125', 'name': 'Peptidase_CA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 560,\n",
       "    'interactions': 0,\n",
       "    'matches': 23534,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23525,\n",
       "    'proteomes': 2741,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19561, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 12450},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22939,\n",
       "    'alignment:full': 4327,\n",
       "    'alignment:seed': 473},\n",
       "   'cross_references': {}}},\n",
       " 'PF13920': {'metadata': {'accession': 'PF13920',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Zinc finger, C3HC4 type (RING finger)',\n",
       "    'short': 'zf-C3HC4_3'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0229', 'name': 'RING'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2072,\n",
       "    'interactions': 0,\n",
       "    'matches': 87283,\n",
       "    'pathways': 0,\n",
       "    'proteins': 84200,\n",
       "    'proteomes': 2301,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 64285, 'rosettafold': 0},\n",
       "    'structures': 53,\n",
       "    'taxa': 9004},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 67038,\n",
       "    'alignment:full': 42345,\n",
       "    'alignment:seed': 337},\n",
       "   'cross_references': {}}},\n",
       " 'PF01988': {'metadata': {'accession': 'PF01988',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008217',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'VIT family', 'short': 'VIT1'},\n",
       "   'description': ['<p>This family includes the vacuolar Fe2+/Mn2+ uptake transporter Swiss:P47818, Ccc1 [PMID:7941738] and the vacuolar iron transporter VIT1 Swiss:Q9ZUA5.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 109,\n",
       "    'interactions': 0,\n",
       "    'matches': 30998,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28516,\n",
       "    'proteomes': 4961,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 24593, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 20728},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27661,\n",
       "    'alignment:full': 8275,\n",
       "    'alignment:seed': 474},\n",
       "   'cross_references': {}}},\n",
       " 'PF10410': {'metadata': {'accession': 'PF10410',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019475',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DnaB-helicase binding domain of primase',\n",
       "    'short': 'DnaB_bind'},\n",
       "   'description': ['<p>This domain is the C-terminal region three-helical domain of primase [PMID:10741967]. Primases synthesise short RNA strands on single-stranded DNA templates, thereby generating the hybrid duplexes required for the initiation of synthesis by DNA polymerases.  Primases are recruited to single-stranded DNA by helicases, and this domain is the region of the primase which binds DnaB-helicase [PMID:10873470]. It is associated with the Toprim domain (Pfam:PF01751) which is the central catalytic core.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007943': {'PMID': 10873470,\n",
       "     'ISBN': None,\n",
       "     'volume': '300',\n",
       "     'issue': '2',\n",
       "     'year': 2000,\n",
       "     'title': 'A TOPRIM domain in the crystal structure of the catalytic core of Escherichia coli primase confirms a structural link to DNA topoisomerases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '353-62',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Podobnik M', 'McInerney P', \"O'Donnell M\", 'Kuriyan J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.2000.3844'},\n",
       "    'PUB00028344': {'PMID': 10741967,\n",
       "     'ISBN': None,\n",
       "     'volume': '287',\n",
       "     'issue': '5462',\n",
       "     'year': 2000,\n",
       "     'title': 'Structure of the RNA polymerase domain of E. coli primase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2482-6',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Keck JL', 'Roche DD', 'Lynch AS', 'Berger JM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.287.5462.2482'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 76,\n",
       "    'interactions': 0,\n",
       "    'matches': 34264,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34264,\n",
       "    'proteomes': 6530,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 29368, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 28242},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31621,\n",
       "    'alignment:full': 6048,\n",
       "    'alignment:seed': 441},\n",
       "   'cross_references': {}}},\n",
       " 'PF09989': {'metadata': {'accession': 'PF09989',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018709',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CoA enzyme activase uncharacterised domain (DUF2229)',\n",
       "    'short': 'DUF2229'},\n",
       "   'description': ['<p>Members of this family include various bacterial hypothetical proteins, as well as CoA enzyme activases. The exact function of this domain has not, as yet, been defined.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0108', 'name': 'Actin_ATPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 20,\n",
       "    'interactions': 0,\n",
       "    'matches': 10231,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10223,\n",
       "    'proteomes': 1387,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4044, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 6363},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8906,\n",
       "    'alignment:full': 1845,\n",
       "    'alignment:seed': 282},\n",
       "   'cross_references': {}}},\n",
       " 'PF02632': {'metadata': {'accession': 'PF02632',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003784',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'BioY family', 'short': 'BioY'},\n",
       "   'description': ['<p>A number of bacterial genes are involved in bioconversion of pimelate into dethiobiotin [PMID:2110099]. BioY is a component of the BioMNY transport system involved in biotin uptake in prokaryotes [PMID:17301237].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008252': {'PMID': 2110099,\n",
       "     'ISBN': None,\n",
       "     'volume': '87',\n",
       "     'issue': '1',\n",
       "     'year': 1990,\n",
       "     'title': 'Cloning and characterization of the Bacillus sphaericus genes controlling the bioconversion of pimelate into dethiobiotin.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '63-70',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Gloeckler R',\n",
       "      'Ohsawa I',\n",
       "      'Speck D',\n",
       "      'Ledoux C',\n",
       "      'Bernard S',\n",
       "      'Zinsius M',\n",
       "      'Villeval D',\n",
       "      'Kisou T',\n",
       "      'Kamogawa K',\n",
       "      'Lemoine Y.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(90)90496-E'},\n",
       "    'PUB00043617': {'PMID': 17301237,\n",
       "     'ISBN': None,\n",
       "     'volume': '104',\n",
       "     'issue': '8',\n",
       "     'year': 2007,\n",
       "     'title': 'Biotin uptake in prokaryotes by solute transporters with an optional ATP-binding cassette-containing module.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2909-14',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Hebbeln P', 'Rodionov DA', 'Alfandega A', 'Eitinger T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.0609905104'},\n",
       "    'PUB00044924': {'PMID': 12368242,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '10',\n",
       "     'year': 2002,\n",
       "     'title': 'Conservation of the biotin regulon and the BirA regulatory signal in Eubacteria and Archaea.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1507-16',\n",
       "     'medline_journal': 'Genome Res',\n",
       "     'ISO_journal': 'Genome Res.',\n",
       "     'authors': ['Rodionov DA', 'Mironov AA', 'Gelfand MS.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1101/gr.314502'}},\n",
       "   'set_info': {'accession': 'CL0315', 'name': 'Gx_transp'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 26,\n",
       "    'interactions': 0,\n",
       "    'matches': 23435,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23374,\n",
       "    'proteomes': 4611,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20511, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 18723},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21856,\n",
       "    'alignment:full': 4982,\n",
       "    'alignment:seed': 637},\n",
       "   'cross_references': {}}},\n",
       " 'PF13478': {'metadata': {'accession': 'PF13478',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027051',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'XdhC Rossmann domain', 'short': 'XdhC_C'},\n",
       "   'description': ['<p>This entry is the rossmann domain found in the Xanthine dehydrogenase accessory protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 37617,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37607,\n",
       "    'proteomes': 4922,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 32371, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 20470},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35587,\n",
       "    'alignment:full': 7184,\n",
       "    'alignment:seed': 566},\n",
       "   'cross_references': {}}},\n",
       " 'PF13475': {'metadata': {'accession': 'PF13475',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025197',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4116)',\n",
       "    'short': 'DUF4116'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 266,\n",
       "    'interactions': 0,\n",
       "    'matches': 10818,\n",
       "    'pathways': 0,\n",
       "    'proteins': 2945,\n",
       "    'proteomes': 117,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 2643, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1134},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8241,\n",
       "    'alignment:full': 2850,\n",
       "    'alignment:seed': 568},\n",
       "   'cross_references': {}}},\n",
       " 'PF01741': {'metadata': {'accession': 'PF01741',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR037673',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Large-conductance mechanosensitive channel, MscL',\n",
       "    'short': 'MscL'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005232': {'PMID': 9856938,\n",
       "     'ISBN': None,\n",
       "     'volume': '282',\n",
       "     'issue': '5397',\n",
       "     'year': 1998,\n",
       "     'title': 'Structure of the MscL homolog from Mycobacterium tuberculosis: a gated mechanosensitive ion channel.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2220-6',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Chang G', 'Spencer RH', 'Lee AT', 'Barclay MT', 'Rees DC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.282.5397.2220'}},\n",
       "   'set_info': {'accession': 'CL0167', 'name': 'Zn_Beta_Ribbon'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 47,\n",
       "    'interactions': 0,\n",
       "    'matches': 32610,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32597,\n",
       "    'proteomes': 6935,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28096, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 28444},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29652,\n",
       "    'alignment:full': 6863,\n",
       "    'alignment:seed': 508},\n",
       "   'cross_references': {}}},\n",
       " 'PF13460': {'metadata': {'accession': 'PF13460',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR016040',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NAD(P)H-binding', 'short': 'NAD_binding_10'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 515,\n",
       "    'interactions': 0,\n",
       "    'matches': 133135,\n",
       "    'pathways': 0,\n",
       "    'proteins': 131949,\n",
       "    'proteomes': 8640,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 115281, 'rosettafold': 0},\n",
       "    'structures': 49,\n",
       "    'taxa': 35349},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 124288,\n",
       "    'alignment:full': 36336,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF04296': {'metadata': {'accession': 'PF04296',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007393',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF448)', 'short': 'YlxR'},\n",
       "   'description': ['<p>The YlxR family has been demonstrated to regulate metabolic gene expression [PMID:30355672].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11,\n",
       "    'interactions': 0,\n",
       "    'matches': 19392,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19392,\n",
       "    'proteomes': 4719,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16725, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 18333},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17636,\n",
       "    'alignment:full': 4327,\n",
       "    'alignment:seed': 514},\n",
       "   'cross_references': {}}},\n",
       " 'PF09371': {'metadata': {'accession': 'PF09371',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018974',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tex-like protein N-terminal domain', 'short': 'Tex_N'},\n",
       "   'description': ['<p>This presumed domain is found at the N-terminus of Swiss:Q45388. This protein defines a novel family of prokaryotic transcriptional accessory factors [PMID:8755871].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044514': {'PMID': 8755871,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '15',\n",
       "     'year': 1996,\n",
       "     'title': 'A new gene locus of Bordetella pertussis defines a novel family of prokaryotic transcriptional accessory proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4445-52',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Fuchs TM', 'Deppisch H', 'Scarlato V', 'Gross R.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8755871&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 29844,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29831,\n",
       "    'proteomes': 6753,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25765, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 27735},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28194,\n",
       "    'alignment:full': 6267,\n",
       "    'alignment:seed': 557},\n",
       "   'cross_references': {}}},\n",
       " 'PF04264': {'metadata': {'accession': 'PF04264',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007372',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YceI-like domain', 'short': 'YceI'},\n",
       "   'description': ['<p>E. coli YceI is a base-induced periplasmic protein [PMID:12107143]. The recent structure of a member of this family shows that it binds to poly-isoprenoid [PMID:15741337]. The structure consists of an extended, eight-stranded, antiparallel beta-barrel that resembles the lipocalin fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009923': {'PMID': 12107143,\n",
       "     'ISBN': None,\n",
       "     'volume': '184',\n",
       "     'issue': '15',\n",
       "     'year': 2002,\n",
       "     'title': 'pH-dependent expression of periplasmic proteins and amino acid catabolism in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4246-58',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Stancik LM',\n",
       "      'Stancik DM',\n",
       "      'Schmidt B',\n",
       "      'Barnhart DM',\n",
       "      'Yoncheva YN',\n",
       "      'Slonczewski JL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.184.15.4246-4258.2002'},\n",
       "    'PUB00032324': {'PMID': 15741337,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '4',\n",
       "     'year': 2005,\n",
       "     'title': 'Crystal structure of a novel polyisoprenoid-binding protein from Thermus thermophilus HB8.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1004-10',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Handa N',\n",
       "      'Terada T',\n",
       "      'Doi-Katayama Y',\n",
       "      'Hirota H',\n",
       "      'Tame JR',\n",
       "      'Park SY',\n",
       "      'Kuramitsu S',\n",
       "      'Shirouzu M',\n",
       "      'Yokoyama S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1110/ps.041183305'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 86,\n",
       "    'interactions': 0,\n",
       "    'matches': 59337,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59319,\n",
       "    'proteomes': 5409,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 51128, 'rosettafold': 0},\n",
       "    'structures': 24,\n",
       "    'taxa': 22119},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54397,\n",
       "    'alignment:full': 12135,\n",
       "    'alignment:seed': 729},\n",
       "   'cross_references': {}}},\n",
       " 'PF02843': {'metadata': {'accession': 'PF02843',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020560',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoribosylglycinamide synthetase, C domain',\n",
       "    'short': 'GARS_C'},\n",
       "   'description': [\"<p>Phosphoribosylglycinamide synthetase catalyses the second step in the de novo biosynthesis of purine. The reaction catalysed by Phosphoribosylglycinamide synthetase is the ATP- dependent addition of 5-phosphoribosylamine to glycine to form 5'phosphoribosylglycinamide. This domain is related to the C-terminal domain of biotin carboxylase/carbamoyl phosphate synthetase (see Pfam:PF02787).</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002527': {'PMID': 2687276,\n",
       "     'ISBN': None,\n",
       "     'volume': '264',\n",
       "     'issue': '35',\n",
       "     'year': 1989,\n",
       "     'title': 'Nucleotide sequence analysis of genes purH and purD involved in the de novo purine nucleotide biosynthesis of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '21239-46',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Aiba A', 'Mizobuchi K.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/264/35/21239.pdf'},\n",
       "    'PUB00014228': {'PMID': 9843369,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '45',\n",
       "     'year': 1998,\n",
       "     'title': 'X-ray crystal structure of glycinamide ribonucleotide synthetase from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15647-62',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Wang W', 'Kappock TJ', 'Stubbe J', 'Ealick SE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi981405n'}},\n",
       "   'set_info': {'accession': 'CL0105', 'name': 'Hybrid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 193,\n",
       "    'interactions': 0,\n",
       "    'matches': 48936,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48923,\n",
       "    'proteomes': 10217,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41485, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 42128},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44411,\n",
       "    'alignment:full': 9625,\n",
       "    'alignment:seed': 956},\n",
       "   'cross_references': {}}},\n",
       " 'PF17910': {'metadata': {'accession': 'PF17910',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041069',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FeoB cytosolic helical domain', 'short': 'FeoB_Cyto'},\n",
       "   'description': ['<p>FeoB is a G-protein coupled membrane protein essential for Fe(II) uptake in prokaryotes. In the structures, a canonical G-protein domain (G domain) is followed by a helical bundle domain (S-domain) which is represented by this entry.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 38,\n",
       "    'interactions': 0,\n",
       "    'matches': 18461,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18461,\n",
       "    'proteomes': 2304,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16115, 'rosettafold': 0},\n",
       "    'structures': 31,\n",
       "    'taxa': 11688},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17142,\n",
       "    'alignment:full': 3092,\n",
       "    'alignment:seed': 369},\n",
       "   'cross_references': {}}},\n",
       " 'PF03591': {'metadata': {'accession': 'PF03591',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011606',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AzlC protein', 'short': 'AzlC'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 20,\n",
       "    'interactions': 0,\n",
       "    'matches': 31494,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31489,\n",
       "    'proteomes': 5010,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 27956, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21257},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30871,\n",
       "    'alignment:full': 6005,\n",
       "    'alignment:seed': 576},\n",
       "   'cross_references': {}}},\n",
       " 'PF03453': {'metadata': {'accession': 'PF03453',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005110',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MoeA N-terminal region (domain I and II)',\n",
       "    'short': 'MoeA_N'},\n",
       "   'description': ['<p>This family contains two structural domains. One of these contains the conserved DGXA motif. This region is found in proteins involved in biosynthesis of molybdopterin cofactor however the exact molecular function of this region is uncertain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 175,\n",
       "    'interactions': 0,\n",
       "    'matches': 59801,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59748,\n",
       "    'proteomes': 8380,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 50851, 'rosettafold': 0},\n",
       "    'structures': 50,\n",
       "    'taxa': 34769},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54280,\n",
       "    'alignment:full': 12315,\n",
       "    'alignment:seed': 1111},\n",
       "   'cross_references': {}}},\n",
       " 'PF01813': {'metadata': {'accession': 'PF01813',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002699',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase subunit D', 'short': 'ATP-synt_D'},\n",
       "   'description': ['<p>This is a family of subunit D form various ATP synthases including V-type H+ transporting and Na+ dependent. Subunit D is suggested to be an integral part of the catalytic sector of the V-ATPase [PMID:7831318].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002959': {'PMID': 8702544,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '31',\n",
       "     'year': 1996,\n",
       "     'title': 'Subunit structure and organization of the genes of the A1A0 ATPase from the Archaeon Methanosarcina mazei Go1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '18843-52',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Wilms R',\n",
       "      'Freiberg C',\n",
       "      'Wegerle E',\n",
       "      'Meier I',\n",
       "      'Mayer F',\n",
       "      'Muller V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.31.18843'},\n",
       "    'PUB00004863': {'PMID': 7831318,\n",
       "     'ISBN': None,\n",
       "     'volume': '92',\n",
       "     'issue': '2',\n",
       "     'year': 1995,\n",
       "     'title': 'A bovine cDNA and a yeast gene (VMA8) encoding the subunit D of the vacuolar H(+)-ATPase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '497-501',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Nelson H', 'Mandiyan S', 'Nelson N.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=7831318&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 56,\n",
       "    'interactions': 0,\n",
       "    'matches': 14340,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14190,\n",
       "    'proteomes': 3573,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12018, 'rosettafold': 0},\n",
       "    'structures': 99,\n",
       "    'taxa': 15449},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12043,\n",
       "    'alignment:full': 3758,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF03938': {'metadata': {'accession': 'PF03938',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005632',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Outer membrane protein (OmpH-like)', 'short': 'OmpH'},\n",
       "   'description': ['<p>This family includes outer membrane proteins such as OmpH among others. Skp (OmpH) has been characterised as a molecular chaperone that interacts with unfolded proteins as they emerge in the periplasm from the Sec translocation machinery [PMID:15304217].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015159': {'PMID': 15304217,\n",
       "     'ISBN': None,\n",
       "     'volume': '15',\n",
       "     'issue': '3',\n",
       "     'year': 2004,\n",
       "     'title': 'Crystal structure of Skp, a prefoldin-like chaperone that protects soluble and membrane proteins from aggregation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '367-74',\n",
       "     'medline_journal': 'Mol Cell',\n",
       "     'ISO_journal': 'Mol. Cell',\n",
       "     'authors': ['Walton TA', 'Sousa MC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.molcel.2004.07.023'},\n",
       "    'PUB00019959': {'PMID': 2318304,\n",
       "     'ISBN': None,\n",
       "     'volume': '262',\n",
       "     'issue': '1',\n",
       "     'year': 1990,\n",
       "     'title': \"Bacterial 'histone-like protein I' (HLP-I) is an outer membrane constituent?\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '123-6',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Hirvas L', 'Coleman J', 'Koski P', 'Vaara M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0014-5793(90)80169-J'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 68,\n",
       "    'interactions': 0,\n",
       "    'matches': 33675,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33641,\n",
       "    'proteomes': 4166,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28162, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 18636},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29395,\n",
       "    'alignment:full': 6226,\n",
       "    'alignment:seed': 460},\n",
       "   'cross_references': {}}},\n",
       " 'PF09269': {'metadata': {'accession': 'PF09269',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015349',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1967)',\n",
       "    'short': 'DUF1967'},\n",
       "   'description': ['<p>Members of this family contain a four-stranded beta sheet and three alpha helices flanked by an additional beta strand. They are predominantly found in the bacterial GTP-binding protein Obg, and are still functionally uncharacterised [PMID:15019792].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00022779': {'PMID': 15019792,\n",
       "     'ISBN': None,\n",
       "     'volume': '337',\n",
       "     'issue': '3',\n",
       "     'year': 2004,\n",
       "     'title': 'Crystal structure of the GTP-binding protein Obg from Thermus thermophilus HB8.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '761-70',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Kukimoto-Niino M',\n",
       "      'Murayama K',\n",
       "      'Inoue M',\n",
       "      'Terada T',\n",
       "      'Tame JR',\n",
       "      'Kuramitsu S',\n",
       "      'Shirouzu M',\n",
       "      'Yokoyama S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.jmb.2004.01.047'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 16639,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16638,\n",
       "    'proteomes': 4057,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14360, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 14781},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15294,\n",
       "    'alignment:full': 3725,\n",
       "    'alignment:seed': 356},\n",
       "   'cross_references': {}}},\n",
       " 'PF08818': {'metadata': {'accession': 'PF08818',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014922',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DU1801)', 'short': 'DUF1801'},\n",
       "   'description': ['<p>This domain is found in the Intracellular iron chaperone frataxin YdhG [PMID:21744456] and the uncharacterised protein YdeI from Bacillus subtilis. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0631', 'name': 'YgaC_TfoX-N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 64,\n",
       "    'interactions': 0,\n",
       "    'matches': 33340,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33223,\n",
       "    'proteomes': 3816,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27746, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 15149},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29212,\n",
       "    'alignment:full': 7211,\n",
       "    'alignment:seed': 365},\n",
       "   'cross_references': {}}},\n",
       " 'PF04960': {'metadata': {'accession': 'PF04960',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015868',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutaminase', 'short': 'Glutaminase'},\n",
       "   'description': ['<p>This family of enzymes deaminates glutamine to glutamate EC:3.5.1.2.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0013', 'name': 'Beta-lactamase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 113,\n",
       "    'interactions': 0,\n",
       "    'matches': 22645,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22246,\n",
       "    'proteomes': 3807,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18716, 'rosettafold': 0},\n",
       "    'structures': 61,\n",
       "    'taxa': 18022},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21143,\n",
       "    'alignment:full': 5922,\n",
       "    'alignment:seed': 268},\n",
       "   'cross_references': {}}},\n",
       " 'PF00367': {'metadata': {'accession': 'PF00367',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018113',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'phosphotransferase system, EIIB', 'short': 'PTS_EIIB'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 70,\n",
       "    'interactions': 0,\n",
       "    'matches': 55171,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54102,\n",
       "    'proteomes': 2885,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 49963, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 12541},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 60858,\n",
       "    'alignment:full': 8986,\n",
       "    'alignment:seed': 712},\n",
       "   'cross_references': {}}},\n",
       " 'PF09347': {'metadata': {'accession': 'PF09347',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018959',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1989)',\n",
       "    'short': 'DUF1989'},\n",
       "   'description': ['<p>This family of proteins are functionally uncharacterised.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 17740,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17531,\n",
       "    'proteomes': 2491,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15537, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 10727},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17411,\n",
       "    'alignment:full': 4012,\n",
       "    'alignment:seed': 251},\n",
       "   'cross_references': {}}},\n",
       " 'PF02660': {'metadata': {'accession': 'PF02660',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003811',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycerol-3-phosphate acyltransferase',\n",
       "    'short': 'G3P_acyltransf'},\n",
       "   'description': ['<p>This family of enzymes catalyses the transfer of an acyl group from acyl-ACP to glycerol-3-phosphate to form lysophosphatidic acid [PMID:16949372]].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 37018,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37014,\n",
       "    'proteomes': 5735,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31437, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 25603},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33516,\n",
       "    'alignment:full': 6093,\n",
       "    'alignment:seed': 733},\n",
       "   'cross_references': {}}},\n",
       " 'PF16874': {'metadata': {'accession': 'PF16874',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031705',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolase family 36 C-terminal domain',\n",
       "    'short': 'Glyco_hydro_36C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of many family 36 glycoside hydrolases. It has a beta-sandwich structure with a Greek key motif [PMID:23012371].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0369', 'name': 'GHD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 13002,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12999,\n",
       "    'proteomes': 2448,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11536, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 9377},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12291,\n",
       "    'alignment:full': 3055,\n",
       "    'alignment:seed': 257},\n",
       "   'cross_references': {}}},\n",
       " 'PF02733': {'metadata': {'accession': 'PF02733',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004006',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dak1 domain', 'short': 'Dak1'},\n",
       "   'description': ['<p>This is the kinase domain of the dihydroxyacetone kinase family EC:2.7.1.29. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0245', 'name': 'EDD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 120,\n",
       "    'interactions': 0,\n",
       "    'matches': 24691,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24497,\n",
       "    'proteomes': 4630,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21750, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 19189},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24085,\n",
       "    'alignment:full': 6390,\n",
       "    'alignment:seed': 280},\n",
       "   'cross_references': {}}},\n",
       " 'PF02436': {'metadata': {'accession': 'PF02436',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003379',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Conserved carboxylase domain', 'short': 'PYC_OADA'},\n",
       "   'description': ['<p>This domain represents a conserved region in pyruvate carboxylase  (PYC), oxaloacetate decarboxylase alpha chain (OADA), and  transcarboxylase 5s subunit. The domain is found adjacent to the  HMGL-like domain (Pfam:PF00682) and often close to the biotin_lipoyl domain (Pfam:PF00364) of biotin requiring enzymes. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020215': {'PMID': 9478969,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'Purification, regulation, and molecular and biochemical characterization of pyruvate carboxylase from Methanobacterium thermoautotrophicum strain deltaH.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5155-66',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Mukhopadhyay B', 'Stoddard SF', 'Wolfe RS.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.9.5155'}},\n",
       "   'set_info': {'accession': 'CL0597', 'name': 'Post-HMGL'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 122,\n",
       "    'interactions': 0,\n",
       "    'matches': 34103,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34063,\n",
       "    'proteomes': 6334,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 29517, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 25668},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31539,\n",
       "    'alignment:full': 7037,\n",
       "    'alignment:seed': 373},\n",
       "   'cross_references': {}}},\n",
       " 'PF04255': {'metadata': {'accession': 'PF04255',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007367',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF433)', 'short': 'DUF433'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 70,\n",
       "    'interactions': 0,\n",
       "    'matches': 26039,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26027,\n",
       "    'proteomes': 1349,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19683, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 8305},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19446,\n",
       "    'alignment:full': 3330,\n",
       "    'alignment:seed': 248},\n",
       "   'cross_references': {}}},\n",
       " 'PF11975': {'metadata': {'accession': 'PF11975',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022616',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Family 4 glycosyl hydrolase C-terminal domain',\n",
       "    'short': 'Glyco_hydro_4C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0341', 'name': 'LDH_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 23261,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23247,\n",
       "    'proteomes': 2157,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20279, 'rosettafold': 0},\n",
       "    'structures': 18,\n",
       "    'taxa': 10240},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23095,\n",
       "    'alignment:full': 4063,\n",
       "    'alignment:seed': 288},\n",
       "   'cross_references': {}}},\n",
       " 'PF02873': {'metadata': {'accession': 'PF02873',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011601',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UDP-N-acetylenolpyruvoylglucosamine reductase, C-terminal domain',\n",
       "    'short': 'MurB_C'},\n",
       "   'description': ['<p>Members of this family are UDP-N-acetylenolpyruvoylglucosamine reductase enzymes EC:1.1.1.158. This enzyme is involved in the biosynthesis of peptidoglycan.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007935': {'PMID': 8805513,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '1',\n",
       "     'year': 1996,\n",
       "     'title': 'The structure of the substrate-free form of MurB, an essential enzyme for the synthesis of bacterial cell walls.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '47-54',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Benson TE', 'Walsh CT', 'Hogle JM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(96)00008-1'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 70,\n",
       "    'interactions': 0,\n",
       "    'matches': 47109,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47103,\n",
       "    'proteomes': 8562,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 40416, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 36659},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43277,\n",
       "    'alignment:full': 8120,\n",
       "    'alignment:seed': 767},\n",
       "   'cross_references': {}}},\n",
       " 'PF04149': {'metadata': {'accession': 'PF04149',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007278',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF397)', 'short': 'DUF397'},\n",
       "   'description': ['<p>The function of this family is unknown. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044962': {'PMID': 1569025,\n",
       "     'ISBN': None,\n",
       "     'volume': '174',\n",
       "     'issue': '9',\n",
       "     'year': 1992,\n",
       "     'title': 'abaA, a new pleiotropic regulatory locus for antibiotic production in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2958-67',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Fernandez-Moreno MA',\n",
       "      'Martin-Triana AJ',\n",
       "      'Martinez E',\n",
       "      'Niemi J',\n",
       "      'Kieser HM',\n",
       "      'Hopwood DA',\n",
       "      'Malpartida F.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 32,\n",
       "    'interactions': 0,\n",
       "    'matches': 43165,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39223,\n",
       "    'proteomes': 743,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34982, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 2327},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41013,\n",
       "    'alignment:full': 13786,\n",
       "    'alignment:seed': 585},\n",
       "   'cross_references': {}}},\n",
       " 'PF02021': {'metadata': {'accession': 'PF02021',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003509',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterised protein family UPF0102',\n",
       "    'short': 'UPF0102'},\n",
       "   'description': ['<p>The function of this family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0236', 'name': 'PDDEXK'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 38543,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38518,\n",
       "    'proteomes': 7502,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 32896, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 32187},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34553,\n",
       "    'alignment:full': 6890,\n",
       "    'alignment:seed': 780},\n",
       "   'cross_references': {}}},\n",
       " 'PF00348': {'metadata': {'accession': 'PF00348',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000092',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polyprenyl synthetase', 'short': 'polyprenyl_synt'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0613', 'name': 'Terp_synthase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 369,\n",
       "    'interactions': 0,\n",
       "    'matches': 122740,\n",
       "    'pathways': 0,\n",
       "    'proteins': 122159,\n",
       "    'proteomes': 10987,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 104255, 'rosettafold': 0},\n",
       "    'structures': 369,\n",
       "    'taxa': 47407},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 110104,\n",
       "    'alignment:full': 28182,\n",
       "    'alignment:seed': 415},\n",
       "   'cross_references': {}}},\n",
       " 'PF08340': {'metadata': {'accession': 'PF08340',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013551',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1732)',\n",
       "    'short': 'DUF1732'},\n",
       "   'description': ['<p>This domain of unknown function is often found at the C-terminus of bacterial proteins, many of which are hypothetical, including proteins of the YicC family which have Pfam:PF03755 at the N-terminus. These include a protein important in the stationary phase of growth, and required for growth at high temperature [PMID:1925027]. Structural modelling suggests this domain may bind nucleic acids [PMID:21348639].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 19,\n",
       "    'interactions': 0,\n",
       "    'matches': 27118,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27117,\n",
       "    'proteomes': 5237,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22942, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21933},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24644,\n",
       "    'alignment:full': 4923,\n",
       "    'alignment:seed': 366},\n",
       "   'cross_references': {}}},\n",
       " 'PF02410': {'metadata': {'accession': 'PF02410',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal silencing factor during starvation',\n",
       "    'short': 'RsfS'},\n",
       "   'description': ['<p>This family is expressed by almost all bacterial and eukaryotic genomes but not by archaea. Its function is to down-regulate protein synthesis under conditions of nutrient shortage, and it does this by binding to protein L14 of the large ribosomal subunit, thus acting as a ribosomal silencing factor (RsfS) by blocking the joining of the ribosomal subunits [PMID:22829778]. This family is structurally homologous to nucleotidyltransferases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0260', 'name': 'NTP_transf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 43728,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43721,\n",
       "    'proteomes': 10180,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37405, 'rosettafold': 0},\n",
       "    'structures': 39,\n",
       "    'taxa': 41204},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39305,\n",
       "    'alignment:full': 9344,\n",
       "    'alignment:seed': 786},\n",
       "   'cross_references': {}}},\n",
       " 'PF14008': {'metadata': {'accession': 'PF14008',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025733',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Iron/zinc purple acid phosphatase-like protein C',\n",
       "    'short': 'Metallophos_C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of Purple acid phosphatase proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 270,\n",
       "    'interactions': 0,\n",
       "    'matches': 11435,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11278,\n",
       "    'proteomes': 1218,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9477, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 5268},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9381,\n",
       "    'alignment:full': 6028,\n",
       "    'alignment:seed': 285},\n",
       "   'cross_references': {}}},\n",
       " 'PF03588': {'metadata': {'accession': 'PF03588',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004616',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Leucyl/phenylalanyl-tRNA protein transferase',\n",
       "    'short': 'Leu_Phe_trans'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0257', 'name': 'Acetyltrans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 22613,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22558,\n",
       "    'proteomes': 4434,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19188, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 19160},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20669,\n",
       "    'alignment:full': 4194,\n",
       "    'alignment:seed': 368},\n",
       "   'cross_references': {}}},\n",
       " 'PF16488': {'metadata': {'accession': 'PF16488',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032472',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Argonaute linker 2 domain', 'short': 'ArgoL2'},\n",
       "   'description': ['<p>ArgoL2 is the second linker domain in eukaryotic argonaute proteins. It starts with two alpha-helices aligned orthogonally to each other followed by a beta-strand involved in linking the two lobes, the PAZ lobe and the Piwi lobe of argonaute to each other. Linker 2 together with the N, PAZ and L1 domains form a compact global fold [PMID:16061186]. Numerous residues from Piwi, L1 and L2 linkers direct the path of the phosphate backbone of nucleotides 7-9, thus allowing DNA-slicing [PMID:23746446].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 291,\n",
       "    'interactions': 0,\n",
       "    'matches': 15021,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14946,\n",
       "    'proteomes': 1844,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11312, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 6774},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11698,\n",
       "    'alignment:full': 7839,\n",
       "    'alignment:seed': 335},\n",
       "   'cross_references': {}}},\n",
       " 'PF13449': {'metadata': {'accession': 'PF13449',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027372',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Esterase-like activity of phytase',\n",
       "    'short': 'Phytase-like'},\n",
       "   'description': ['<p>This is a repeated domain that carries several highly conserved Glu and Asp residues indicating the likelihood that the domain incorporates the enzymic activity of the PLC-like phospho-diesterase part of the proteins. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 315,\n",
       "    'interactions': 0,\n",
       "    'matches': 18988,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18539,\n",
       "    'proteomes': 3262,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15628, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13763},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17791,\n",
       "    'alignment:full': 4348,\n",
       "    'alignment:seed': 247},\n",
       "   'cross_references': {}}},\n",
       " 'PF02582': {'metadata': {'accession': 'PF02582',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003734',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RMND1/Sif2-Sif3/Mrx10, DUF155', 'short': 'DUF155'},\n",
       "   'description': ['<p>This entry represents a domain found in RMND1 from mammals, Sif2/Sif3 from fission yeasts and Rmd1/Rmd8/YDR282C (Mrx10) from budding yeasts. RMND1 and its yeast homologue, Mrx10, are mitochondrial proteins required for mitochondrial translation [PMID:23022098, PMID:23022099]. Rmd1 and Rmd8 are cytoplasmic proteins required for sporulation [PMID:12586695].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 9153,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9086,\n",
       "    'proteomes': 2565,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7656, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 10097},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7736,\n",
       "    'alignment:full': 4172,\n",
       "    'alignment:seed': 269},\n",
       "   'cross_references': {}}},\n",
       " 'PF01336': {'metadata': {'accession': 'PF01336',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004365',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'OB-fold nucleic acid binding domain',\n",
       "    'short': 'tRNA_anti-codon'},\n",
       "   'description': ['<p>This family contains OB-fold domains that bind to nucleic acids [PMID:10829230]. The family includes the anti-codon binding domain of lysyl,  aspartyl, and asparaginyl -tRNA synthetases (See Pfam:PF00152). Aminoacyl-tRNA synthetases catalyse the addition of an amino acid to the appropriate tRNA molecule EC:6.1.1.-. This family also includes part of RecG helicase involved in DNA repair. Replication factor A is a hetero-trimeric complex, that contains a subunit in this family [PMID:7760808, PMID:8990123]. This domain is also found at the C-terminus of bacterial DNA polymerase III alpha chain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005139': {'PMID': 2047877,\n",
       "     'ISBN': None,\n",
       "     'volume': '252',\n",
       "     'issue': '5013',\n",
       "     'year': 1991,\n",
       "     'title': 'Class II aminoacyl transfer RNA synthetases: crystal structure of yeast aspartyl-tRNA synthetase complexed with tRNA(Asp).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1682-9',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Ruff M',\n",
       "      'Krishnaswamy S',\n",
       "      'Boeglin M',\n",
       "      'Poterszman A',\n",
       "      'Mitschler A',\n",
       "      'Podjarny A',\n",
       "      'Rees B',\n",
       "      'Thierry JC',\n",
       "      'Moras D.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/252/5013/1682'},\n",
       "    'PUB00007673': {'PMID': 10829230,\n",
       "     'ISBN': None,\n",
       "     'volume': '54',\n",
       "     'issue': None,\n",
       "     'year': 2000,\n",
       "     'title': 'Protein fold recognition using sequence profiles and its application in structural genomics.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '245-75',\n",
       "     'medline_journal': 'Adv Protein Chem',\n",
       "     'ISO_journal': 'Adv. Protein Chem.',\n",
       "     'authors': ['Koonin EV', 'Wolf YI', 'Aravind L.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0065-3233(00)54008-X'},\n",
       "    'PUB00007674': {'PMID': 7760808,\n",
       "     'ISBN': None,\n",
       "     'volume': '15',\n",
       "     'issue': '6',\n",
       "     'year': 1995,\n",
       "     'title': 'Rpa4, a homolog of the 34-kilodalton subunit of the replication protein A complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3119-28',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Keshav KF', 'Chen C', 'Dutta A.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=7760808'},\n",
       "    'PUB00007675': {'PMID': 8990123,\n",
       "     'ISBN': None,\n",
       "     'volume': '385',\n",
       "     'issue': '6612',\n",
       "     'year': 1997,\n",
       "     'title': 'Structure of the single-stranded-DNA-binding domain of replication protein A bound to DNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '176-81',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Bochkarev A', 'Pfuetzner RA', 'Edwards AM', 'Frappier L.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/385176a0'}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 653,\n",
       "    'interactions': 0,\n",
       "    'matches': 184696,\n",
       "    'pathways': 0,\n",
       "    'proteins': 183140,\n",
       "    'proteomes': 11282,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 154402, 'rosettafold': 0},\n",
       "    'structures': 139,\n",
       "    'taxa': 48382},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 165873,\n",
       "    'alignment:full': 38516,\n",
       "    'alignment:seed': 540},\n",
       "   'cross_references': {}}},\n",
       " 'PF03091': {'metadata': {'accession': 'PF03091',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004323',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CutA1 divalent ion tolerance protein', 'short': 'CutA1'},\n",
       "   'description': ['<p>Several gene loci with a possible involvement in cellular tolerance to copper have been identified [PMID:7623666]. One such locus in eubacteria and  archaebacteria, cutA, is thought to be  involved in cellular tolerance to a wide variety of divalent cations other than copper. The cutA locus consists of two operons, of one and two genes. The CutA1 protein is a cytoplasmic protein, encoded by the single-gene operon and has been linked to divalent cation tolerance. It has no recognised structural motifs [PMID:9260936]. This family also contains putative proteins from eukaryotes (human and Drosophila).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007506': {'PMID': 7623666,\n",
       "     'ISBN': None,\n",
       "     'volume': '15',\n",
       "     'issue': '6',\n",
       "     'year': 1995,\n",
       "     'title': 'Molecular genetics of a chromosomal locus involved in copper tolerance in Escherichia coli K-12.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1127-37',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Fong ST', 'Camakaris J', 'Lee BT.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.1995.tb02286.x'},\n",
       "    'PUB00007507': {'PMID': 9260936,\n",
       "     'ISBN': None,\n",
       "     'volume': '179',\n",
       "     'issue': '16',\n",
       "     'year': 1997,\n",
       "     'title': 'A Salmonella typhimurium genetic locus which confers copper tolerance on copper-sensitive mutants of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4977-84',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Gupta SD', 'Wu HC', 'Rick PD.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9260936'}},\n",
       "   'set_info': {'accession': 'CL0089', 'name': 'GlnB-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 19391,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19345,\n",
       "    'proteomes': 3908,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16242, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 18894},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16640,\n",
       "    'alignment:full': 3845,\n",
       "    'alignment:seed': 412},\n",
       "   'cross_references': {}}},\n",
       " 'PF05949': {'metadata': {'accession': 'PF05949',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010273',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial protein of unknown function (DUF881)',\n",
       "    'short': 'DUF881'},\n",
       "   'description': ['<p>This family consists of a series of hypothetical bacterial proteins. One of the family members Swiss:Q45543 from Bacillus subtilis is thought to be involved in cell division and sporulation [PMID:2556375].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011928': {'PMID': 2556375,\n",
       "     'ISBN': None,\n",
       "     'volume': '171',\n",
       "     'issue': '12',\n",
       "     'year': 1989,\n",
       "     'title': 'Nucleotide sequence and insertional inactivation of a Bacillus subtilis gene that affects cell division, sporulation, and temperature sensitivity.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6821-34',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Beall B', 'Lutkenhaus J.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=2556375'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11,\n",
       "    'interactions': 0,\n",
       "    'matches': 13716,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13712,\n",
       "    'proteomes': 1606,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12159, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 5477},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12985,\n",
       "    'alignment:full': 3748,\n",
       "    'alignment:seed': 287},\n",
       "   'cross_references': {}}},\n",
       " 'PF01980': {'metadata': {'accession': 'PF01980',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023370',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'tRNA-methyltransferase O', 'short': 'TrmO'},\n",
       "   'description': ['<p>This family includes members such as TrmO (tRNA-methyltransferase O) also known as YaeB, which contains a single-sheeted beta-barrel structure. TrmO is an AdoMet-dependent methyltransferase responsible for m6t6A formation [PMID:25063302]. Its human homolog, is responsible for formation of m6t6A37 in cytoplasmic tRNASer. Lack of TrmO decreases attenuation activity of the thr operon, indicating that N6 methylation of m6t6A37 ensures efficient decoding of ACY codons [PMID:27913733]. In bacteria and eukaryotes, TrmO has a C-terminal domain containing the conserved DPRxxY motif. Where the Asp194 and Arg196 in this motif of E. coli TrmO are necessary for N6-methylation. However, no archaeal YaeB has a C-terminal domain containing the DPRxxY motif that is conserved in bacterial and mammalian TrmO homologs [PMID:25063302].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 98,\n",
       "    'interactions': 0,\n",
       "    'matches': 19374,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19329,\n",
       "    'proteomes': 3233,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16861, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 16871},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17918,\n",
       "    'alignment:full': 3626,\n",
       "    'alignment:seed': 357},\n",
       "   'cross_references': {}}},\n",
       " 'PF12781': {'metadata': {'accession': 'PF12781',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR035706',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP-binding dynein motor region', 'short': 'AAA_9'},\n",
       "   'description': [\"<p>This domain is found in human cytoplasmic dynein-2 proteins. Cytoplasmic dynein-2 (dynein-2) performs intraflagellar transport and is associated with human skeletal ciliopathies. Dyneins share a conserved motor domain that couples cycles of ATP hydrolysis with conformational changes to produce movement. Structural analysis reveal that the motor's ring consists of six AAA+ domains (ATPases associated with various cellular activities (AAA1-AAA6). This is the fifth AAA+ domain subdomain AAA5S. Structural analysis reveal that it is the coiled-coil buttress interface. The relative movement of AAA5S together with the stalk (AAA4S), is coupled to rearrangements in the AAA+ ring. Closure of the AAA1 site and the rigid body movement of AAA2-AAA4 force the AAA4/AAA5 interface to close and the AAA6L subdomain to rotate towards the ring centre. The AAA5S subdomain rotates as a unit together with AAA6L, and this movement pulls the buttress relative to the stalk [PMID:25470043].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1588,\n",
       "    'interactions': 0,\n",
       "    'matches': 25947,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25403,\n",
       "    'proteomes': 1920,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 3345, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 7586},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18213,\n",
       "    'alignment:full': 10710,\n",
       "    'alignment:seed': 379},\n",
       "   'cross_references': {}}},\n",
       " 'PF02445': {'metadata': {'accession': 'PF02445',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003473',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Quinolinate synthetase A protein', 'short': 'NadA'},\n",
       "   'description': ['<p>Quinolinate synthetase catalyses the second step of the de novo biosynthetic pathway of pyridine nucleotide formation. In particular, quinolinate synthetase is involved in the condensation of dihydroxyacetone phosphate and iminoaspartate to form quinolinic acid [PMID:10648170]. This synthesis requires two enzymes, a FAD-containing \"B protein\" and an \"A protein\".</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009471': {'PMID': 10648170,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '1',\n",
       "     'year': 2000,\n",
       "     'title': 'Cloning, overexpression, and purification of Escherichia coli quinolinate synthetase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '64-70',\n",
       "     'medline_journal': 'Protein Expr Purif',\n",
       "     'ISO_journal': 'Protein Expr. Purif.',\n",
       "     'authors': ['Ceciliani F',\n",
       "      'Caramori T',\n",
       "      'Ronchi S',\n",
       "      'Tedeschi G',\n",
       "      'Mortarino M',\n",
       "      'Galizzi A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/prep.1999.1153'},\n",
       "    'PUB00019910': {'PMID': 2841129,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '2',\n",
       "     'year': 1988,\n",
       "     'title': 'Molecular biology of pyridine nucleotide biosynthesis in Escherichia coli. Cloning and characterization of quinolinate synthesis genes nadA and nadB.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '221-8',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Flachmann R',\n",
       "      'Kunz N',\n",
       "      'Seifert J',\n",
       "      'Gutlich M',\n",
       "      'Wientjes FJ',\n",
       "      'Laufer A',\n",
       "      'Gassen HG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1432-1033.1988.tb14187.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 35901,\n",
       "    'pathways': 0,\n",
       "    'proteins': 35852,\n",
       "    'proteomes': 6587,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 30915, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 28898},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32936,\n",
       "    'alignment:full': 6175,\n",
       "    'alignment:seed': 517},\n",
       "   'cross_references': {}}},\n",
       " 'PF00095': {'metadata': {'accession': 'PF00095',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008197',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"WAP-type (Whey Acidic Protein) 'four-disulfide core'\",\n",
       "    'short': 'WAP'},\n",
       "   'description': ['<p>WAP belongs to the group of Elafin or elastase-specific inhibitors.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 964,\n",
       "    'interactions': 0,\n",
       "    'matches': 23144,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13879,\n",
       "    'proteomes': 912,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9165, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 3992},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16110,\n",
       "    'alignment:full': 9552,\n",
       "    'alignment:seed': 419},\n",
       "   'cross_references': {}}},\n",
       " 'PF00146': {'metadata': {'accession': 'PF00146',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001694',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NADH dehydrogenase', 'short': 'NADHdh'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003309': {'PMID': 7690854,\n",
       "     'ISBN': None,\n",
       "     'volume': '233',\n",
       "     'issue': '1',\n",
       "     'year': 1993,\n",
       "     'title': 'The gene locus of the proton-translocating NADH: ubiquinone oxidoreductase in Escherichia coli. Organization of the 14 genes and relationship between the derived proteins and subunits of mitochondrial complex I.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '109-22',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Weidner U',\n",
       "      'Geier S',\n",
       "      'Ptock A',\n",
       "      'Friedrich T',\n",
       "      'Leif H',\n",
       "      'Weiss H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1993.1488'}},\n",
       "   'set_info': {'accession': 'CL0425', 'name': 'ComplexI-N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 126,\n",
       "    'interactions': 0,\n",
       "    'matches': 114499,\n",
       "    'pathways': 0,\n",
       "    'proteins': 113883,\n",
       "    'proteomes': 6670,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 98582, 'rosettafold': 0},\n",
       "    'structures': 229,\n",
       "    'taxa': 97974},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 103731,\n",
       "    'alignment:full': 8869,\n",
       "    'alignment:seed': 411},\n",
       "   'cross_references': {}}},\n",
       " 'PF16325': {'metadata': {'accession': 'PF16325',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032525',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Peptidase family U32 C-terminal domain',\n",
       "    'short': 'Peptidase_U32_C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of many members of Peptidase family U32 (Pfam:PF01136).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 14,\n",
       "    'interactions': 0,\n",
       "    'matches': 13855,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13855,\n",
       "    'proteomes': 2570,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12226, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 12112},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13676,\n",
       "    'alignment:full': 2482,\n",
       "    'alignment:seed': 309},\n",
       "   'cross_references': {}}},\n",
       " 'PF04367': {'metadata': {'accession': 'PF04367',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007462',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF502)', 'short': 'DUF502'},\n",
       "   'description': ['<p>Predicted to be an integral membrane protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 17726,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17710,\n",
       "    'proteomes': 3145,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14664, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13736},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15182,\n",
       "    'alignment:full': 4003,\n",
       "    'alignment:seed': 276},\n",
       "   'cross_references': {}}},\n",
       " 'PF04234': {'metadata': {'accession': 'PF04234',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007348',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CopC domain', 'short': 'CopC'},\n",
       "   'description': ['<p>CopC is a bacterial blue copper protein that binds 1 atom of copper per  protein molecule.  Along with CopA, CopC mediates copper resistance by sequestration of copper in the periplasm [PMID:1924351].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009918': {'PMID': 1924351,\n",
       "     'ISBN': None,\n",
       "     'volume': '88',\n",
       "     'issue': '20',\n",
       "     'year': 1991,\n",
       "     'title': 'Copper resistance in Pseudomonas syringae mediated by periplasmic and outer membrane proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8915-9',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Cha JS', 'Cooksey DA.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=1924351'}},\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 58,\n",
       "    'interactions': 0,\n",
       "    'matches': 20431,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20387,\n",
       "    'proteomes': 2880,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18099, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 12294},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20073,\n",
       "    'alignment:full': 3973,\n",
       "    'alignment:seed': 304},\n",
       "   'cross_references': {}}},\n",
       " 'PF02375': {'metadata': {'accession': 'PF02375',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003349',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'jmjN domain', 'short': 'JmjN'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008126': {'PMID': 10838566,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '6',\n",
       "     'year': 2000,\n",
       "     'title': 'Evidence of domain swapping within the jumonji family of transcription factors.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '274-6',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Balciunas D', 'Ronne H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)01593-0'}},\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 271,\n",
       "    'interactions': 0,\n",
       "    'matches': 17590,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17579,\n",
       "    'proteomes': 2047,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7029, 'rosettafold': 0},\n",
       "    'structures': 467,\n",
       "    'taxa': 7347},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13036,\n",
       "    'alignment:full': 8894,\n",
       "    'alignment:seed': 281},\n",
       "   'cross_references': {}}},\n",
       " 'PF03719': {'metadata': {'accession': 'PF03719',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005324',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S5, C-terminal domain',\n",
       "    'short': 'Ribosomal_S5_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 180,\n",
       "    'interactions': 0,\n",
       "    'matches': 54512,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54494,\n",
       "    'proteomes': 11127,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46586, 'rosettafold': 0},\n",
       "    'structures': 1160,\n",
       "    'taxa': 48617},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48067,\n",
       "    'alignment:full': 12932,\n",
       "    'alignment:seed': 454},\n",
       "   'cross_references': {}}},\n",
       " 'PF17137': {'metadata': {'accession': 'PF17137',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR033403',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF5110)',\n",
       "    'short': 'DUF5110'},\n",
       "   'description': ['<p>This domain is likely to be a carbohydrate-binding domain of some description as it is found immediately C-terminal to the glycosyl-hydrolase family Glyco_hydro_31, Pfam:PF01055.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 304,\n",
       "    'interactions': 0,\n",
       "    'matches': 15930,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15927,\n",
       "    'proteomes': 2874,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13348, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 12240},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14765,\n",
       "    'alignment:full': 4145,\n",
       "    'alignment:seed': 302},\n",
       "   'cross_references': {}}},\n",
       " 'PF02646': {'metadata': {'accession': 'PF02646',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003798',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RmuC family', 'short': 'RmuC'},\n",
       "   'description': ['<p>This family contains several bacterial RmuC DNA recombination proteins. The function of the RMUC protein is unknown but it is suspected that it is either a structural protein that protects DNA against nuclease action, or is itself involved in DNA cleavage at the regions of DNA secondary structures [PMID:10886369]</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020261': {'PMID': 10886369,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '6',\n",
       "     'year': 2000,\n",
       "     'title': 'Genes involved in the determination of the rate of inversions at short inverted repeats.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '425-37',\n",
       "     'medline_journal': 'Genes Cells',\n",
       "     'ISO_journal': 'Genes Cells',\n",
       "     'authors': ['Slupska MM',\n",
       "      'Chiang JH',\n",
       "      'Luther WM',\n",
       "      'Stewart JL',\n",
       "      'Amii L',\n",
       "      'Conrad A',\n",
       "      'Miller JH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2443.2000.00341.x'}},\n",
       "   'set_info': {'accession': 'CL0236', 'name': 'PDDEXK'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 46,\n",
       "    'interactions': 0,\n",
       "    'matches': 32379,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32351,\n",
       "    'proteomes': 5588,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27487, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 24984},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29503,\n",
       "    'alignment:full': 5185,\n",
       "    'alignment:seed': 286},\n",
       "   'cross_references': {}}},\n",
       " 'PF04463': {'metadata': {'accession': 'PF04463',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007553',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '2-thiouracil desulfurase', 'short': '2-thiour_desulf'},\n",
       "   'description': ['<p>This family of proteins, predominantly found in Bacteria, are involved in the desulfuration of 2-thiouracil into uracil in the 2-thiouridine degradation pathway. It has been demonstrated that these proteins contain a Fe-S cluster required for their activity [PMID:29194984].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 16387,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16381,\n",
       "    'proteomes': 2111,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14534, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 11730},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16233,\n",
       "    'alignment:full': 2401,\n",
       "    'alignment:seed': 338},\n",
       "   'cross_references': {}}},\n",
       " 'PF04613': {'metadata': {'accession': 'PF04613',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020573',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase, LpxD',\n",
       "    'short': 'LpxD'},\n",
       "   'description': ['<p>UDP-3-O-[3-hydroxymyristoyl] glucosamine N-acyltransferase (EC 2.3.1.-) catalyses an early step in lipid A biosynthesis: UDP-3-O-(3-hydroxytetradecanoyl)glucosamine + (R)-3-hydroxytetradecanoyl- [acyl carrier protein] -> UDP-2,3-bis(3-hydroxytetradecanoyl)glucosamine + [acyl carrier protein] [PMID:8366125]. Members of this family also contain a hexapeptide repeat (Pfam:PF00132).  This family constitutes the non-repeating region of LPXD proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010088': {'PMID': 8366125,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '26',\n",
       "     'year': 1993,\n",
       "     'title': 'The firA gene of Escherichia coli encodes UDP-3-O-(R-3-hydroxymyristoyl)-glucosamine N-acyltransferase. The third step of endotoxin biosynthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '19866-74',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Kelly TM', 'Stachula SA', 'Raetz CR', 'Anderson MS.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/268/26/19866.pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 27468,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27467,\n",
       "    'proteomes': 4000,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23263, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 19707},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24778,\n",
       "    'alignment:full': 4347,\n",
       "    'alignment:seed': 438},\n",
       "   'cross_references': {}}},\n",
       " 'PF14693': {'metadata': {'accession': 'PF14693',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020057',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein TL5, C-terminal domain',\n",
       "    'short': 'Ribosomal_TL5_C'},\n",
       "   'description': ['<p>This family contains the C-terminal domain of ribosomal protein TL5. The N-terminal domain, which binds to 5S rRNA, is contained in family Ribosomal_L25p, Pfam:PF01386. Full length (N- and C-terminal domain) homologues of TL5 are also known as CTC proteins. TL5 or CTC are not found in Eukarya or Archaea. In some Bacteria, including E. coli, this ribosomal subunit occurs as a single domain protein (named Ribosomal subunit L25), where the only domain is homologous to TL5 N-terminal domain (hence included in family Pfam:PF01386). The function of the C-terminal domain of TLC is at present unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 37380,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37380,\n",
       "    'proteomes': 7445,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31739, 'rosettafold': 0},\n",
       "    'structures': 315,\n",
       "    'taxa': 31136},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33286,\n",
       "    'alignment:full': 7105,\n",
       "    'alignment:seed': 622},\n",
       "   'cross_references': {}}},\n",
       " 'PF10143': {'metadata': {'accession': 'PF10143',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004456',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '2,3-bisphosphoglycerate-independent phosphoglycerate mutase',\n",
       "    'short': 'PhosphMutase'},\n",
       "   'description': ['<p>This family represents 2,3-bisphosphoglycerate-independent phosphoglycerate mutase (iPGAM), a metalloenzyme found particularly in archaea and some eubacteria, which catalyses the interconversion of 2-phosphoglycerate and 3-phosphoglycerate in the reaction: [(2R)-2-phosphoglycerate = (2R)-3-phosphoglycerate] (EC 5.4.2.12) [PMID:17576516].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 43,\n",
       "    'interactions': 0,\n",
       "    'matches': 10393,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10381,\n",
       "    'proteomes': 1364,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8567, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 7011},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8395,\n",
       "    'alignment:full': 1497,\n",
       "    'alignment:seed': 262},\n",
       "   'cross_references': {}}},\n",
       " 'PF08487': {'metadata': {'accession': 'PF08487',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013694',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Vault protein inter-alpha-trypsin domain',\n",
       "    'short': 'VIT'},\n",
       "   'description': ['<p>Inter-alpha-trypsin inhibitors (ITIs) consist of one light chain and a variable set of heavy chains.  ITIs play a role in extracellular matrix (ECM) stabilisation and tumour metastasis as well as in plasma protease inhibition [PMID:14744536].  The vault protein inter-alpha-trypsin (VIT) domain described here is found to the N-terminus of a von Willebrand factor type A domain (Pfam:PF00092) in ITI heavy chains (ITIHs) and their precursors.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020822': {'PMID': 14744536,\n",
       "     'ISBN': None,\n",
       "     'volume': '204',\n",
       "     'issue': '1',\n",
       "     'year': 2004,\n",
       "     'title': 'ITIH5, a novel member of the inter-alpha-trypsin inhibitor heavy chain family is downregulated in breast cancer.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '69-77',\n",
       "     'medline_journal': 'Cancer Lett',\n",
       "     'ISO_journal': 'Cancer Lett.',\n",
       "     'authors': ['Himmelfarb M',\n",
       "      'Klopocki E',\n",
       "      'Grube S',\n",
       "      'Staub E',\n",
       "      'Klaman I',\n",
       "      'Hinzmann B',\n",
       "      'Kristiansen G',\n",
       "      'Rosenthal A',\n",
       "      'Durst M',\n",
       "      'Dahl E.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.canlet.2003.09.011'}},\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 469,\n",
       "    'interactions': 0,\n",
       "    'matches': 20472,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20358,\n",
       "    'proteomes': 2335,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14026, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 10382},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13938,\n",
       "    'alignment:full': 6310,\n",
       "    'alignment:seed': 288},\n",
       "   'cross_references': {}}},\n",
       " 'PF13634': {'metadata': {'accession': 'PF13634',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025574',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nucleoporin FG repeat region', 'short': 'Nucleoporin_FG'},\n",
       "   'description': ['<p>This family includes a number of FG repeats that are found in nucleoporin proteins. This family includes the yeast nucleoporins Nup116, Nup100, Nup49, Nup57 and Nup 145.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0647', 'name': 'FG_rpt'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 263,\n",
       "    'interactions': 0,\n",
       "    'matches': 14037,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5831,\n",
       "    'proteomes': 1227,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 3480, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 4717},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21818,\n",
       "    'alignment:full': 12130,\n",
       "    'alignment:seed': 299},\n",
       "   'cross_references': {}}},\n",
       " 'PF02594': {'metadata': {'accession': 'PF02594',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003746',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterised ACR, YggU family COG1872',\n",
       "    'short': 'DUF167'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 68,\n",
       "    'interactions': 0,\n",
       "    'matches': 20284,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20283,\n",
       "    'proteomes': 3991,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17027, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 19634},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17398,\n",
       "    'alignment:full': 3783,\n",
       "    'alignment:seed': 331},\n",
       "   'cross_references': {}}},\n",
       " 'PF04199': {'metadata': {'accession': 'PF04199',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007325',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative cyclase', 'short': 'Cyclase'},\n",
       "   'description': ['<p>Proteins in this family are thought to be cyclase enzymes.  They are found in proteins involved in antibiotic synthesis.  However they are also found in organisms that do not make antibiotics pointing to a wider role for these proteins. The proteins contain a conserved motif HXGTHXDXPXH that is likely to form part of the active site.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019376': {'PMID': 9573189,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'The Streptomyces peucetius dpsY and dnrX genes govern early and late steps of daunorubicin and doxorubicin biosynthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2379-86',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Lomovskaya N',\n",
       "      'Doi-Katayama Y',\n",
       "      'Filippini S',\n",
       "      'Nastro C',\n",
       "      'Fonstein L',\n",
       "      'Gallo M',\n",
       "      'Colombo AL',\n",
       "      'Hutchinson CR.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=9573189'}},\n",
       "   'set_info': {'accession': 'CL0364', 'name': 'Leu-IlvD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 189,\n",
       "    'interactions': 0,\n",
       "    'matches': 46283,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46009,\n",
       "    'proteomes': 5533,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 38863, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 23482},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40994,\n",
       "    'alignment:full': 10672,\n",
       "    'alignment:seed': 367},\n",
       "   'cross_references': {}}},\n",
       " 'PF09362': {'metadata': {'accession': 'PF09362',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018535',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1996)',\n",
       "    'short': 'DUF1996'},\n",
       "   'description': ['<p>This family of proteins are functionally uncharacterised.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 130,\n",
       "    'interactions': 0,\n",
       "    'matches': 10277,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10149,\n",
       "    'proteomes': 1230,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8664, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 4455},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9094,\n",
       "    'alignment:full': 4144,\n",
       "    'alignment:seed': 292},\n",
       "   'cross_references': {}}},\n",
       " 'PF00023': {'metadata': {'accession': 'PF00023',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002110',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ankyrin repeat', 'short': 'Ank'},\n",
       "   'description': [\"<p>Ankyrins are multifunctional adaptors that link specific proteins to the membrane-associated, spectrin- actin cytoskeleton. This repeat-domain is a 'membrane-binding' domain of up to 24 repeated units, and it mediates most of the protein's binding activities. Repeats 13-24 are especially active, with known sites of interaction for the Na/K ATPase, Cl/HCO(3) anion exchanger, voltage-gated sodium channel, clathrin heavy chain and L1 family cell adhesion molecules. The ANK repeats are found to form a contiguous spiral stack such that ion transporters like the anion exchanger associate in a large central cavity formed by the ANK repeat spiral, while clathrin and cell adhesion molecules associate with specific regions outside this cavity [PMID:12456646][PMID:14731966].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001704': {'PMID': 9013872,\n",
       "     'ISBN': None,\n",
       "     'volume': '401',\n",
       "     'issue': '2-3',\n",
       "     'year': 1997,\n",
       "     'title': 'NMR structural characterization of the CDK inhibitor p19INK4d.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '127-32',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Kalus W',\n",
       "      'Baumgartner R',\n",
       "      'Renner C',\n",
       "      'Noegel A',\n",
       "      'Chan FK',\n",
       "      'Winoto A',\n",
       "      'Holak TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(96)01465-2'},\n",
       "    'PUB00004066': {'PMID': 2141669,\n",
       "     'ISBN': None,\n",
       "     'volume': '345',\n",
       "     'issue': '6277',\n",
       "     'year': 1990,\n",
       "     'title': 'Hereditary spherocytosis associated with deletion of human erythrocyte ankyrin gene on chromosome 8.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '736-9',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Lux SE',\n",
       "      'Tse WT',\n",
       "      'Menninger JC',\n",
       "      'John KM',\n",
       "      'Harris P',\n",
       "      'Shalev O',\n",
       "      'Chilcote RR',\n",
       "      'Marchesi SL',\n",
       "      'Watkins PC',\n",
       "      'Bennett V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/345736a0'}},\n",
       "   'set_info': {'accession': 'CL0465', 'name': 'Ank'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 15070,\n",
       "    'interactions': 0,\n",
       "    'matches': 131595,\n",
       "    'pathways': 0,\n",
       "    'proteins': 108142,\n",
       "    'proteomes': 3622,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 67218, 'rosettafold': 0},\n",
       "    'structures': 109,\n",
       "    'taxa': 16916},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52743,\n",
       "    'alignment:full': 28509,\n",
       "    'alignment:seed': 1062},\n",
       "   'cross_references': {}}},\n",
       " 'PF06133': {'metadata': {'accession': 'PF06133',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010368',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Control of competence regulator ComK, YlbF/YmcA',\n",
       "    'short': 'Com_YlbF'},\n",
       "   'description': ['<p>YlbF Is a family of short Gram-positive and archaeal proteins that includes both YlbF and YmcA which may interact synergistically. The family is necessary for correct biofilm formation, as null mutants of ymcA and ylbF fail to form pellicles at air-liquid interfaces and grow on solid media as smooth, undifferentiated colonies. During development, YmcA, YlbF and YaaT, family PSPI, Pfam:PF04468, interact directly with one another forming a stable ternary complex, in vitro. All three proteins are required for competence, sporulation and the formation of biofilms. The YmcA-YlbF-YaaT complex affects the phosphotransfer between Spo0F and Spo0B, thus accelerating the production of Spo0A~P. The three processes of biofilm formation, mature spore formation and competence all require the active, phosphorylated form of Spo0A, as Spo0A-P [PMID:23490197, PMID:19202088].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11,\n",
       "    'interactions': 0,\n",
       "    'matches': 11312,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11309,\n",
       "    'proteomes': 1462,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10158, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 6058},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11162,\n",
       "    'alignment:full': 2632,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF11760': {'metadata': {'accession': 'PF11760',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021744',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cobalamin synthesis G N-terminal', 'short': 'CbiG_N'},\n",
       "   'description': ['<p>Members of this family are involved in cobalamin synthesis.  The gene encoded by Swiss:P72862 has been designated cbiH but in fact represents a fusion between cbiH and cbiG. As other multi-functional proteins involved in cobalamin biosynthesis catalyse adjacent steps in the pathway, including CysG, CobL (CbiET), CobIJ and CobA-HemD, it is therefore possible that CbiG catalyses a reaction step adjacent to CbiH. In the anaerobic pathway such a step could be the formation of a gamma lactone, which is thought to help to mediate the anaerobic ring contraction process [PMID:9742225]. Within the cobalamin synthesis pathway CbiG catalyses the both the opening of the lactone ring and the extrusion of the two-carbon fragment of cobalt-precorrin-5A from C-20 and its associated methyl group (deacylation) to give cobalt-precorrin-5B [PMID:16866557]. The N-terminal of the enzyme is conserved in this family, and the C-terminal and the mid-sections are conserved independently in other families, CbiG_C and CbiG_mid, although the distinct function of each region is unclear.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 10955,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10955,\n",
       "    'proteomes': 2220,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9694, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 11016},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10505,\n",
       "    'alignment:full': 2119,\n",
       "    'alignment:seed': 304},\n",
       "   'cross_references': {}}},\n",
       " 'PF12704': {'metadata': {'accession': 'PF12704',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025857',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MacB-like periplasmic core domain', 'short': 'MacB_PCD'},\n",
       "   'description': ['<p>This family represents the periplasmic core domain found in a variety of ABC transporters. The structure of this family has been solved for the MacB protein [PMID:19432486]. Some structural similarity was found to the periplasmic domain of the AcrB multidrug efflux transporter.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 214,\n",
       "    'interactions': 0,\n",
       "    'matches': 229454,\n",
       "    'pathways': 0,\n",
       "    'proteins': 201627,\n",
       "    'proteomes': 8329,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 170020, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 37211},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 203237,\n",
       "    'alignment:full': 39931,\n",
       "    'alignment:seed': 501},\n",
       "   'cross_references': {}}},\n",
       " 'PF03315': {'metadata': {'accession': 'PF03315',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005131',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Serine dehydratase beta chain', 'short': 'SDH_beta'},\n",
       "   'description': ['<p>L-serine dehydratase (EC:4.2.1.13) is a found as a heterodimer of alpha and beta chain or as a fusion of the two chains in a single protein. This enzyme catalyses the deamination of serine to form pyruvate. This enzyme is part of the gluconeogenesis pathway.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0697', 'name': 'SDH_beta'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 33984,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33156,\n",
       "    'proteomes': 6263,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 29745, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 25169},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34412,\n",
       "    'alignment:full': 6678,\n",
       "    'alignment:seed': 292},\n",
       "   'cross_references': {}}},\n",
       " 'PF00380': {'metadata': {'accession': 'PF00380',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000754',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S9/S16', 'short': 'Ribosomal_S9'},\n",
       "   'description': ['<p>This family includes small ribosomal subunit S9 from prokaryotes and S16 from eukaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 140,\n",
       "    'interactions': 0,\n",
       "    'matches': 54356,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54299,\n",
       "    'proteomes': 11126,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46450, 'rosettafold': 0},\n",
       "    'structures': 1194,\n",
       "    'taxa': 48900},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47876,\n",
       "    'alignment:full': 12364,\n",
       "    'alignment:seed': 621},\n",
       "   'cross_references': {}}},\n",
       " 'PF16859': {'metadata': {'accession': 'PF16859',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011075',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetracyclin repressor-like, C-terminal domain',\n",
       "    'short': 'TetR_C_11'},\n",
       "   'description': ['<p>This family of bacterial transcriptional repressors is characterised by the short approximately 50 amino acid stretch of residues constituting the helix-turn-helix DNA binding motif, around the YRFhY motif. The target proteins that are repressed are involved in the transcriptional control of multi-drug efflux pumps, pathways for the biosynthesis of antibiotics, response to osmotic stress and toxic chemicals, control of catabolic pathways, differentiation processes, and pathogenicity [PMID:15944459].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0174', 'name': 'TetR_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 30926,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30924,\n",
       "    'proteomes': 2498,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27690, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 10322},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30080,\n",
       "    'alignment:full': 7935,\n",
       "    'alignment:seed': 294},\n",
       "   'cross_references': {}}},\n",
       " 'PF16901': {'metadata': {'accession': 'PF16901',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031656',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'C-terminal domain of alpha-glycerophosphate oxidase',\n",
       "    'short': 'DAO_C'},\n",
       "   'description': ['<p>DAO_C is the C-terminal region of alpha-glycerophosphate oxidase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0667', 'name': 'Fer2_BFD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 96,\n",
       "    'interactions': 0,\n",
       "    'matches': 38140,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38130,\n",
       "    'proteomes': 7160,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33172, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 28815},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36234,\n",
       "    'alignment:full': 8790,\n",
       "    'alignment:seed': 303},\n",
       "   'cross_references': {}}},\n",
       " 'PF02511': {'metadata': {'accession': 'PF02511',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003669',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Thymidylate synthase complementing protein',\n",
       "    'short': 'Thy1'},\n",
       "   'description': ['<p>Thymidylate synthase complementing protein (Thy1) complements the thymidine growth requirement of the organisms in which it is found, but shows no homology to thymidylate synthase. The bacterial members of this family at least are flavin-dependent thymidylate synthases [PMID:12029065, PMID:15046578, PMID:17890305].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008137': {'PMID': 9665876,\n",
       "     'ISBN': None,\n",
       "     'volume': '281',\n",
       "     'issue': '5375',\n",
       "     'year': 1998,\n",
       "     'title': 'Complete genome sequence of Treponema pallidum, the syphilis spirochete.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '375-88',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Fraser CM',\n",
       "      'Norris SJ',\n",
       "      'Weinstock GM',\n",
       "      'White O',\n",
       "      'Sutton GG',\n",
       "      'Dodson R',\n",
       "      'Gwinn M',\n",
       "      'Hickey EK',\n",
       "      'Clayton R',\n",
       "      'Ketchum KA',\n",
       "      'Sodergren E',\n",
       "      'Hardham JM',\n",
       "      'McLeod MP',\n",
       "      'Salzberg S',\n",
       "      'Peterson J',\n",
       "      'Khalak H',\n",
       "      'Richardson D',\n",
       "      'Howell JK',\n",
       "      'Chidambaram M',\n",
       "      'Utterback T',\n",
       "      'McDonald L',\n",
       "      'Artiach P',\n",
       "      'Bowman C',\n",
       "      'Cotton MD',\n",
       "      'Fujii C',\n",
       "      'Garland S',\n",
       "      'Hatch B',\n",
       "      'Horst K',\n",
       "      'Roberts K',\n",
       "      'Sandusky M',\n",
       "      'Weidman J',\n",
       "      'Smith HO',\n",
       "      'Venter JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.281.5375.375'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 45,\n",
       "    'interactions': 0,\n",
       "    'matches': 24211,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20909,\n",
       "    'proteomes': 3614,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15659, 'rosettafold': 0},\n",
       "    'structures': 45,\n",
       "    'taxa': 17478},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20665,\n",
       "    'alignment:full': 3677,\n",
       "    'alignment:seed': 282},\n",
       "   'cross_references': {}}},\n",
       " 'PF01693': {'metadata': {'accession': 'PF01693',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011320',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Caulimovirus viroplasmin', 'short': 'Cauli_VI'},\n",
       "   'description': [\"<p>This family consists of various caulimovirus viroplasmin proteins.  The viroplasmin protein is encoded by gene VI and is the main component of viral inclusion bodies or viroplasms [PMID:2402462]. Inclusions are the site of viral assembly, DNA synthesis and accumulation [PMID:2402462]. Two domains exist within gene VI corresponding approximately to the 5' third and middle third of gene VI, these influence systemic infection in a light-dependent manner [PMID:8372449].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004381': {'PMID': 2402462,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '17',\n",
       "     'year': 1990,\n",
       "     'title': 'DNA sequence of gene VI of cauliflower mosaic virus strain PV147.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5297',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Volovitch M', 'Modjtahedi N', 'Chouikh Y', 'Yot P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/18.17.5297'},\n",
       "    'PUB00005595': {'PMID': 8372449,\n",
       "     'ISBN': None,\n",
       "     'volume': '196',\n",
       "     'issue': '2',\n",
       "     'year': 1993,\n",
       "     'title': 'Identification of domains within gene VI of cauliflower mosaic virus that influence systemic infection of Nicotiana bigelovii in a light-dependent manner.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '789-98',\n",
       "     'medline_journal': 'Virology',\n",
       "     'ISO_journal': 'Virology',\n",
       "     'authors': ['Wintermantel WM', 'Anderson EJ', 'Schoelz JE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0042-6822(83)90001-6'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 240,\n",
       "    'interactions': 0,\n",
       "    'matches': 13430,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12022,\n",
       "    'proteomes': 3031,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9866, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 11987},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11466,\n",
       "    'alignment:full': 5226,\n",
       "    'alignment:seed': 255},\n",
       "   'cross_references': {}}},\n",
       " 'PF01618': {'metadata': {'accession': 'PF01618',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002898',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MotA/TolQ/ExbB proton channel family',\n",
       "    'short': 'MotA_ExbB'},\n",
       "   'description': ['<p>This family groups together integral membrane proteins that appear to be involved translocation of proteins across a membrane. These proteins are probably proton channels.  MotA is an essential component of the flageller motor that uses a proton gradient to generate rotational motion in the flageller [PMID:10348868]. ExbB is part of the TonB-dependent transduction complex.  The TonB complex uses the proton gradient across the inner bacterial membrane to transport large molecules across the outer bacterial membrane.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002317': {'PMID': 9811664,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '22',\n",
       "     'year': 1998,\n",
       "     'title': 'Interactions in the TonB-dependent energy transduction complex: ExbB and ExbD form homomultimers.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6031-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Higgs PI', 'Myers PS', 'Postle K.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=9811664'},\n",
       "    'PUB00002320': {'PMID': 10348868,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '11',\n",
       "     'year': 1999,\n",
       "     'title': 'Function of proline residues of MotA in torque generation by the flagellar motor of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3542-51',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Braun TF',\n",
       "      'Poulson S',\n",
       "      'Gully JB',\n",
       "      'Empey JC',\n",
       "      'Van Way S',\n",
       "      'Putnam A',\n",
       "      'Blair DF.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=10348868'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 102,\n",
       "    'interactions': 0,\n",
       "    'matches': 114877,\n",
       "    'pathways': 0,\n",
       "    'proteins': 114863,\n",
       "    'proteomes': 6328,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 98050, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 27744},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 104991,\n",
       "    'alignment:full': 20016,\n",
       "    'alignment:seed': 440},\n",
       "   'cross_references': {}}},\n",
       " 'PF02625': {'metadata': {'accession': 'PF02625',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003777',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'XdhC and CoxI family', 'short': 'XdhC_CoxI'},\n",
       "   'description': ['<p>This domain is often found in association with an NAD-binding region, related to TrkA-N (Pfam:PF02254; personal obs:C. Yeats). XdhC is  believed to be involved in the attachment of molybdenum to Xanthine Dehydrogenase ([PMID:10217763]).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016133': {'PMID': 10217763,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '9',\n",
       "     'year': 1999,\n",
       "     'title': 'Role of XDHC in Molybdenum cofactor insertion into xanthine dehydrogenase of Rhodobacter capsulatus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2745-51',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Leimkuhler S', 'Klipp W.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=10217763&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 40775,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36409,\n",
       "    'proteomes': 4858,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31299, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 20195},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38597,\n",
       "    'alignment:full': 8094,\n",
       "    'alignment:seed': 450},\n",
       "   'cross_references': {}}},\n",
       " 'PF01416': {'metadata': {'accession': 'PF01416',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020097',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'tRNA pseudouridine synthase', 'short': 'PseudoU_synth_1'},\n",
       "   'description': ['<p>Involved in the formation of pseudouridine at the anticodon stem and loop of transfer-RNAs Pseudouridine is an isomer of uridine (5-(beta-D-ribofuranosyl) uracil, and id the most abundant modified nucleoside found in  all cellular RNAs. The TruA-like proteins also exhibit a conserved sequence with a strictly conserved aspartic acid, likely involved in catalysis.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000455': {'PMID': 9585540,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '20',\n",
       "     'year': 1998,\n",
       "     'title': 'Transfer RNA-pseudouridine synthetase Pus1 of Saccharomyces cerevisiae contains one atom of zinc essential for its native conformation and tRNA recognition.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7268-76',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Arluison V', 'Hountondji C', 'Robert B', 'Grosjean H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi972671o'}},\n",
       "   'set_info': {'accession': 'CL0649', 'name': 'PseudoU_synth'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 232,\n",
       "    'interactions': 0,\n",
       "    'matches': 94468,\n",
       "    'pathways': 0,\n",
       "    'proteins': 60829,\n",
       "    'proteomes': 10874,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 51956, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 44403},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 85480,\n",
       "    'alignment:full': 20779,\n",
       "    'alignment:seed': 262},\n",
       "   'cross_references': {}}},\n",
       " 'PF02256': {'metadata': {'accession': 'PF02256',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003149',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Iron hydrogenase small subunit', 'short': 'Fe_hyd_SSU'},\n",
       "   'description': ['<p>This family represents the small subunit of the Fe-only hydrogenases EC:1.18.99.1. The subunit is comprised of alternating random coil and alpha helical structures that encompasses the large subunit in a novel protein fold [PMID:10368269].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006430': {'PMID': 10368269,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Desulfovibrio desulfuricans iron hydrogenase: the structure shows unusual coordination to an active site Fe binuclear center.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13-23',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Nicolet Y',\n",
       "      'Piras C',\n",
       "      'Legrand P',\n",
       "      'Hatchikian CE',\n",
       "      'Fontecilla-Camps JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(99)80005-7'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 149,\n",
       "    'interactions': 0,\n",
       "    'matches': 10727,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10726,\n",
       "    'proteomes': 2032,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8849, 'rosettafold': 0},\n",
       "    'structures': 44,\n",
       "    'taxa': 8732},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8968,\n",
       "    'alignment:full': 2774,\n",
       "    'alignment:seed': 256},\n",
       "   'cross_references': {}}},\n",
       " 'PF03947': {'metadata': {'accession': 'PF03947',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022669',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal Proteins L2, C-terminal domain',\n",
       "    'short': 'Ribosomal_L2_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0107', 'name': 'KOW'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 202,\n",
       "    'interactions': 0,\n",
       "    'matches': 70563,\n",
       "    'pathways': 0,\n",
       "    'proteins': 70499,\n",
       "    'proteomes': 11079,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 60459, 'rosettafold': 0},\n",
       "    'structures': 1189,\n",
       "    'taxa': 67064},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61791,\n",
       "    'alignment:full': 12556,\n",
       "    'alignment:seed': 373},\n",
       "   'cross_references': {}}},\n",
       " 'PF01992': {'metadata': {'accession': 'PF01992',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002843',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase (C/AC39) subunit',\n",
       "    'short': 'vATP-synt_AC39'},\n",
       "   'description': ['<p>This family includes the AC39 subunit from vacuolar ATP synthase Swiss:P32366 [PMID:8509410], and the C subunit from archaebacterial ATP synthase [PMID:8702544].  The family also includes subunit C from the Sodium transporting ATP synthase from Enterococcus hirae Swiss:P43456 [PMID:8157629].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002959': {'PMID': 8702544,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '31',\n",
       "     'year': 1996,\n",
       "     'title': 'Subunit structure and organization of the genes of the A1A0 ATPase from the Archaeon Methanosarcina mazei Go1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '18843-52',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Wilms R',\n",
       "      'Freiberg C',\n",
       "      'Wegerle E',\n",
       "      'Meier I',\n",
       "      'Mayer F',\n",
       "      'Muller V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.31.18843'},\n",
       "    'PUB00005711': {'PMID': 8509410,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '17',\n",
       "     'year': 1993,\n",
       "     'title': 'The Saccharomyces cerevisiae VMA6 gene encodes the 36-kDa subunit of the vacuolar H(+)-ATPase membrane sector.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '12749-57',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Bauerle C', 'Ho MN', 'Lindorfer MA', 'Stevens TH.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/268/17/12749'},\n",
       "    'PUB00005718': {'PMID': 8157629,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '15',\n",
       "     'year': 1994,\n",
       "     'title': 'Sequencing and characterization of the ntp gene cluster for vacuolar-type Na(+)-translocating ATPase of Enterococcus hirae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '11037-44',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Takase K',\n",
       "      'Kakinuma S',\n",
       "      'Yamato I',\n",
       "      'Konishi K',\n",
       "      'Igarashi K',\n",
       "      'Kakinuma Y.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/15/11037'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 13228,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12925,\n",
       "    'proteomes': 3169,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10914, 'rosettafold': 0},\n",
       "    'structures': 67,\n",
       "    'taxa': 13745},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11073,\n",
       "    'alignment:full': 3559,\n",
       "    'alignment:seed': 307},\n",
       "   'cross_references': {}}},\n",
       " 'PF10017': {'metadata': {'accession': 'PF10017',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019257',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Histidine-specific methyltransferase, SAM-dependent',\n",
       "    'short': 'Methyltransf_33'},\n",
       "   'description': ['<p>The mycobacterial members of this family are expressed from part of the ergothioneine biosynthetic gene cluster. EGTD is the histidine methyltransferase that transfers three methyl groups to the alpha-amino moiety of histidine, in the first stage of the production of this histidine betaine derivative that carries a thiol group attached to the C2 atom of an imidazole ring [PMID:20420449].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 96,\n",
       "    'interactions': 0,\n",
       "    'matches': 15947,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15718,\n",
       "    'proteomes': 3590,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13262, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 14255},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13947,\n",
       "    'alignment:full': 4024,\n",
       "    'alignment:seed': 261},\n",
       "   'cross_references': {}}},\n",
       " 'PF00586': {'metadata': {'accession': 'PF00586',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR016188',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AIR synthase related protein, N-terminal domain',\n",
       "    'short': 'AIRS'},\n",
       "   'description': ['<p>This family includes Hydrogen expression/formation protein  HypE Swiss:P24193, AIR synthases Swiss:P08178 EC:6.3.3.1, FGAM synthase Swiss:P35852 EC:6.3.5.3 and selenide, water dikinase Swiss:P16456 EC:2.7.9.3. The N-terminal domain of AIR synthase forms the dimer interface of the protein, and is suggested as a putative ATP binding domain [PMID:10508786].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014643': {'PMID': 10508786,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '9',\n",
       "     'year': 1999,\n",
       "     'title': 'X-ray crystal structure of aminoimidazole ribonucleotide synthetase (PurM), from the Escherichia coli purine biosynthetic pathway at 2.5 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1155-66',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Li C', 'Kappock TJ', 'Stubbe J', 'Weaver TM', 'Ealick SE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(99)80182-8'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 380,\n",
       "    'interactions': 0,\n",
       "    'matches': 168075,\n",
       "    'pathways': 0,\n",
       "    'proteins': 145242,\n",
       "    'proteomes': 10748,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 120961, 'rosettafold': 0},\n",
       "    'structures': 50,\n",
       "    'taxa': 44929},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 151118,\n",
       "    'alignment:full': 31173,\n",
       "    'alignment:seed': 1106},\n",
       "   'cross_references': {}}},\n",
       " 'PF16212': {'metadata': {'accession': 'PF16212',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032630',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phospholipid-translocating P-type ATPase C-terminal',\n",
       "    'short': 'PhoLip_ATPase_C'},\n",
       "   'description': ['<p>PhoLip_ATPase_C is found at the C-terminus of a number of phospholipid-translocating ATPases.  It is found in higher eukaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0137', 'name': 'HAD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 476,\n",
       "    'interactions': 0,\n",
       "    'matches': 40069,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39460,\n",
       "    'proteomes': 2185,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21250, 'rosettafold': 0},\n",
       "    'structures': 50,\n",
       "    'taxa': 8029},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29754,\n",
       "    'alignment:full': 18992,\n",
       "    'alignment:seed': 560},\n",
       "   'cross_references': {}}},\n",
       " 'PF03561': {'metadata': {'accession': 'PF03561',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015908',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Allantoicase repeat', 'short': 'Allantoicase'},\n",
       "   'description': ['<p>This family is found in pairs in Allantoicases, forming the majority of the protein. These  proteins allow the use of purines as secondary nitrogen sources in nitrogen-limiting conditions through the reaction: allantoate + H(2)0 =  (-)-ureidoglycolate + urea.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0202', 'name': 'GBD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 16227,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8271,\n",
       "    'proteomes': 2222,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7081, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 9061},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15382,\n",
       "    'alignment:full': 4516,\n",
       "    'alignment:seed': 292},\n",
       "   'cross_references': {}}},\n",
       " 'PF18766': {'metadata': {'accession': 'PF18766',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040980',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SWI2/SNF2 ATPase', 'short': 'SWI2_SNF2'},\n",
       "   'description': ['<p>A SWi2/SNF2 ATPase found in polyvalent proteins [PMID:28559295].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 129,\n",
       "    'interactions': 0,\n",
       "    'matches': 29240,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29220,\n",
       "    'proteomes': 4205,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25593, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 19886},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27803,\n",
       "    'alignment:full': 5524,\n",
       "    'alignment:seed': 292},\n",
       "   'cross_references': {}}},\n",
       " 'PF06750': {'metadata': {'accession': 'PF06750',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010627',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial Peptidase A24 N-terminal domain',\n",
       "    'short': 'DiS_P_DiS'},\n",
       "   'description': [\"<p>This family is found at the N-terminus of the pre-pilin peptidases (Pfam:PF01478). It's function has not been specifically determined; however some of the family have been characterised as bifunctional ([PMID:8057924]), and this domain may contain the N-methylation activity (EC:2.1.1.-). It consists of an intracellular region between a pair of transmembrane. This region contains an invariant proline and two almost fully conserved disulphide bridges - hence the name DiS-P-DiS. The cysteines have been shown to be essential to the overall function of the enzyme in [PMID:8340405], but their role was incorrectly ascribed. </p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012818': {'PMID': 8057924,\n",
       "     'ISBN': None,\n",
       "     'volume': '235',\n",
       "     'issue': None,\n",
       "     'year': 1994,\n",
       "     'title': 'Posttranslational processing of type IV prepilin and homologs by PilD of Pseudomonas aeruginosa.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '527-40',\n",
       "     'medline_journal': 'Methods Enzymol',\n",
       "     'ISO_journal': 'Meth. Enzymol.',\n",
       "     'authors': ['Strom MS', 'Nunn DN', 'Lory S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0076-6879(94)35168-6'},\n",
       "    'PUB00012819': {'PMID': 8340405,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '21',\n",
       "     'year': 1993,\n",
       "     'title': 'Identification of active-site cysteines in the conserved domain of PilD, the bifunctional type IV pilin leader peptidase/N-methyltransferase of Pseudomonas aeruginosa.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15788-94',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Strom MS', 'Bergman P', 'Lory S.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/268/21/15788'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 33,\n",
       "    'interactions': 0,\n",
       "    'matches': 25827,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25825,\n",
       "    'proteomes': 3942,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22090, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 20129},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24042,\n",
       "    'alignment:full': 3759,\n",
       "    'alignment:seed': 391},\n",
       "   'cross_references': {}}},\n",
       " 'PF16320': {'metadata': {'accession': 'PF16320',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008932',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L7/L12 dimerisation domain',\n",
       "    'short': 'Ribosomal_L12_N'},\n",
       "   'description': ['<p>This is the N-terminal dimerisation domain of ribosomal protein L7/L12 [PMID:10637222].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 85,\n",
       "    'interactions': 0,\n",
       "    'matches': 44738,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44735,\n",
       "    'proteomes': 10336,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38679, 'rosettafold': 0},\n",
       "    'structures': 78,\n",
       "    'taxa': 44344},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40284,\n",
       "    'alignment:full': 9724,\n",
       "    'alignment:seed': 479},\n",
       "   'cross_references': {}}},\n",
       " 'PF17876': {'metadata': {'accession': 'PF17876',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040476',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cold shock domain', 'short': 'CSD2'},\n",
       "   'description': ['<p>Crystallographic structure analysis of E. coli wild-type RNase II revealed that the amino-terminal region starts with an alpha-helix followed by two consecutive five-stranded anti-parallel beta-barrels, identified as cold-shock domains (CSD1 and CSD2). This entry relates to CSD2 which lacks the typical sequence motifs RNPI and RNPII but contributes to RNA binding [PMID:16996291] [PMID:16957732].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 32465,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32454,\n",
       "    'proteomes': 5856,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28022, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 24875},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30964,\n",
       "    'alignment:full': 5752,\n",
       "    'alignment:seed': 546},\n",
       "   'cross_references': {}}},\n",
       " 'PF00271': {'metadata': {'accession': 'PF00271',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001650',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Helicase conserved C-terminal domain',\n",
       "    'short': 'Helicase_C'},\n",
       "   'description': ['<p>The Prosite family is restricted to DEAD/H helicases, whereas this domain family is found in a wide variety of helicases and helicase related proteins. It may be that this is not an autonomously folding unit, but an integral part of the helicase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 7856,\n",
       "    'interactions': 0,\n",
       "    'matches': 916492,\n",
       "    'pathways': 0,\n",
       "    'proteins': 913762,\n",
       "    'proteomes': 12771,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 663051, 'rosettafold': 0},\n",
       "    'structures': 598,\n",
       "    'taxa': 57431},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 807618,\n",
       "    'alignment:full': 272115,\n",
       "    'alignment:seed': 421},\n",
       "   'cross_references': {}}},\n",
       " 'PF02397': {'metadata': {'accession': 'PF02397',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003362',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial sugar transferase', 'short': 'Bac_transf'},\n",
       "   'description': ['<p>This Pfam family represents a conserved region from a number of  different bacterial sugar transferases, involved in diverse  biosynthesis pathways.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019196': {'PMID': 9423846,\n",
       "     'ISBN': None,\n",
       "     'volume': '66',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'Identification of a genetic locus essential for serotype b-specific antigen synthesis in Actinobacillus actinomycetemcomitans.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '107-14',\n",
       "     'medline_journal': 'Infect Immun',\n",
       "     'ISO_journal': 'Infect. Immun.',\n",
       "     'authors': ['Yoshida Y', 'Nakano Y', 'Yamashita Y', 'Koga T.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9423846'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 235,\n",
       "    'interactions': 0,\n",
       "    'matches': 89705,\n",
       "    'pathways': 0,\n",
       "    'proteins': 89572,\n",
       "    'proteomes': 7596,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 76665, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 32984},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 81108,\n",
       "    'alignment:full': 16445,\n",
       "    'alignment:seed': 1595},\n",
       "   'cross_references': {}}},\n",
       " 'PF12464': {'metadata': {'accession': 'PF12464',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024688',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Maltose acetyltransferase', 'short': 'Mac'},\n",
       "   'description': ['<p>This domain family is found in bacteria, archaea and eukaryotes, and is approximately 50 amino acids in length. The family is found in association with Pfam:PF00132. Mac uses acetyl-CoA as acetyl donor to acetylated cytoplasmic maltose.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 129,\n",
       "    'interactions': 0,\n",
       "    'matches': 20986,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20982,\n",
       "    'proteomes': 3569,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18890, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 15645},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20824,\n",
       "    'alignment:full': 4949,\n",
       "    'alignment:seed': 535},\n",
       "   'cross_references': {}}},\n",
       " 'PF12878': {'metadata': {'accession': 'PF12878',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024285',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SICA extracellular beta domain', 'short': 'SICA_beta'},\n",
       "   'description': ['<p>The SICA (schizont-infected cell agglutination) proteins of P. knowlesi, one of the variant antigen gene families, are associated with parasitic virulence. These proteins are comprised of multiple domains, with the extracellular domains occurring at different frequencies.  There can be between 1 and 10 copies of this cysteine-rich domain [PMID:18843368].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0474', 'name': 'SICA_like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 2799,\n",
       "    'pathways': 0,\n",
       "    'proteins': 614,\n",
       "    'proteomes': 1,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 247, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 2799,\n",
       "    'alignment:full': 761,\n",
       "    'alignment:seed': 66},\n",
       "   'cross_references': {}}},\n",
       " 'PF17136': {'metadata': {'accession': 'PF17136',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003256',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal proteins 50S L24/mitochondrial 39S L24',\n",
       "    'short': 'ribosomal_L24'},\n",
       "   'description': ['<p>This is the family of bacterial 50S ribosomal subunit proteins L24. It also carries some mitochondrial 39S L24 proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 87,\n",
       "    'interactions': 0,\n",
       "    'matches': 43674,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43670,\n",
       "    'proteomes': 9524,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37771, 'rosettafold': 0},\n",
       "    'structures': 805,\n",
       "    'taxa': 42175},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39387,\n",
       "    'alignment:full': 9045,\n",
       "    'alignment:seed': 431},\n",
       "   'cross_references': {}}},\n",
       " 'PF03914': {'metadata': {'accession': 'PF03914',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005612',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CBF/Mak21 family', 'short': 'CBF'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019257': {'PMID': 9786894,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '44',\n",
       "     'year': 1998,\n",
       "     'title': 'Mak21p of Saccharomyces cerevisiae, a homolog of human CAATT-binding protein, is essential for 60 S ribosomal subunit biogenesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '28912-20',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Edskes HK', 'Ohtake Y', 'Wickner RB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.44.28912'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 158,\n",
       "    'interactions': 0,\n",
       "    'matches': 10657,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10600,\n",
       "    'proteomes': 2154,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8527, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 7827},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8728,\n",
       "    'alignment:full': 5222,\n",
       "    'alignment:seed': 340},\n",
       "   'cross_references': {}}},\n",
       " 'PF13894': {'metadata': {'accession': 'PF13894',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'C2H2-type zinc finger', 'short': 'zf-C2H2_4'},\n",
       "   'description': ['<p>This family contains a number of divergent C2H2 type zinc fingers.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0361', 'name': 'C2H2-zf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4551,\n",
       "    'interactions': 0,\n",
       "    'matches': 32469,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30189,\n",
       "    'proteomes': 1348,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 22272, 'rosettafold': 1},\n",
       "    'structures': 1,\n",
       "    'taxa': 6426},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25242,\n",
       "    'alignment:full': 14929,\n",
       "    'alignment:seed': 313},\n",
       "   'cross_references': {}}},\n",
       " 'PF01556': {'metadata': {'accession': 'PF01556',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002939',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DnaJ C terminal domain', 'short': 'DnaJ_C'},\n",
       "   'description': ['<p>This family consists of the C terminal region of the DnaJ protein.  It is always found associated with Pfam:PF00226 and Pfam:PF00684.  DnaJ is a chaperone associated with the Hsp70 heat-shock system involved in protein folding and renaturation after stress. The two C-terminal domains CTDI and CTDII, both incorporated in this family are necessary for maintaining the J-domains in their specific relative positions [PMID:22011374]. Structural analysis of PDB:1nlt shows that PF00684 is nested within this DnaJ C-terminal region [PMID:14656432].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005482': {'PMID': 9644977,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'The J-domain family and the recruitment of chaperone power.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '222-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Kelley WL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(98)01215-8'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 568,\n",
       "    'interactions': 0,\n",
       "    'matches': 111582,\n",
       "    'pathways': 0,\n",
       "    'proteins': 111328,\n",
       "    'proteomes': 11148,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 94117, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 47535},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 97863,\n",
       "    'alignment:full': 28786,\n",
       "    'alignment:seed': 958},\n",
       "   'cross_references': {}}},\n",
       " 'PF02985': {'metadata': {'accession': 'PF02985',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000357',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HEAT repeat', 'short': 'HEAT'},\n",
       "   'description': ['<p>The HEAT repeat family is related to armadillo/beta-catenin-like  repeats (see Pfam:PF00514).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006179': {'PMID': 7550332,\n",
       "     'ISBN': None,\n",
       "     'volume': '11',\n",
       "     'issue': '2',\n",
       "     'year': 1995,\n",
       "     'title': \"HEAT repeats in the Huntington's disease protein.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '115-6',\n",
       "     'medline_journal': 'Nat Genet',\n",
       "     'ISO_journal': 'Nat. Genet.',\n",
       "     'authors': ['Andrade MA', 'Bork P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/ng1095-115'},\n",
       "    'PUB00006180': {'PMID': 9989501,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'The structure of the protein phosphatase 2A PR65/A subunit reveals the conformation of its 15 tandemly repeated HEAT motifs.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '99-110',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Groves MR',\n",
       "      'Hanlon N',\n",
       "      'Turowski P',\n",
       "      'Hemmings BA',\n",
       "      'Barford D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80963-0'}},\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1107,\n",
       "    'interactions': 0,\n",
       "    'matches': 26935,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22572,\n",
       "    'proteomes': 2372,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13735, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 11099},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18410,\n",
       "    'alignment:full': 11597,\n",
       "    'alignment:seed': 424},\n",
       "   'cross_references': {}}},\n",
       " 'PF09180': {'metadata': {'accession': 'PF09180',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR016061',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Prolyl-tRNA synthetase, C-terminal',\n",
       "    'short': 'ProRS-C_1'},\n",
       "   'description': ['<p>Members of this family are predominantly found in prokaryotic prolyl-tRNA synthetase. They contain a zinc binding site, and adopt a structure consisting of alpha helices and antiparallel beta sheets arranged in 2 layers, in a beta-alpha-beta-alpha-beta motif [PMID:12578991].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00022248': {'PMID': 12578991,\n",
       "     'ISBN': None,\n",
       "     'volume': '100',\n",
       "     'issue': '4',\n",
       "     'year': 2003,\n",
       "     'title': 'The structural basis of cysteine aminoacylation of tRNAPro by prolyl-tRNA synthetases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1673-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Kamtekar S',\n",
       "      'Kennedy WD',\n",
       "      'Wang J',\n",
       "      'Stathopoulos C',\n",
       "      'Soll D',\n",
       "      'Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.0437911100'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 174,\n",
       "    'interactions': 0,\n",
       "    'matches': 19017,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19013,\n",
       "    'proteomes': 4469,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14598, 'rosettafold': 0},\n",
       "    'structures': 76,\n",
       "    'taxa': 18120},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15818,\n",
       "    'alignment:full': 4954,\n",
       "    'alignment:seed': 370},\n",
       "   'cross_references': {}}},\n",
       " 'PF04430': {'metadata': {'accession': 'PF04430',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007523',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF498/DUF598)',\n",
       "    'short': 'DUF498'},\n",
       "   'description': ['<p>This is a large family of uncharacterised proteins found in all domains of life.  The structure shows a novel fold with three beta sheets.  A dimeric form is found in the crystal structure. It was suggested that the cleft in between the two monomers might bing nucleic acid [PMID:11746696].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019473': {'PMID': 11746696,\n",
       "     'ISBN': None,\n",
       "     'volume': '45',\n",
       "     'issue': '4',\n",
       "     'year': 2001,\n",
       "     'title': 'X-ray crystal structure of MTH938 from Methanobacterium thermoautotrophicum at 2.2 A resolution reveals a novel tertiary protein fold.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '486-8',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Das K',\n",
       "      'Xiao R',\n",
       "      'Wahlberg E',\n",
       "      'Hsu F',\n",
       "      'Arrowsmith CH',\n",
       "      'Montelione GT',\n",
       "      'Arnold E.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.1162'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 77,\n",
       "    'interactions': 0,\n",
       "    'matches': 13616,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13611,\n",
       "    'proteomes': 3965,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11559, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 15462},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11755,\n",
       "    'alignment:full': 3707,\n",
       "    'alignment:seed': 259},\n",
       "   'cross_references': {}}},\n",
       " 'PF03116': {'metadata': {'accession': 'PF03116',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004338',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NQR2, RnfD, RnfE family', 'short': 'NQR2_RnfD_RnfE'},\n",
       "   'description': ['<p>This family of bacterial proteins includes a sodium-translocating  NADH-ubiquinone oxidoreductase (i.e. a respiration linked sodium  pump). In Vibrio cholerae, it negatively regulates the expression of virulence factors through inhibiting  (by an unknown mechanism) the transcription of the transcriptional activator ToxT [PMID:10077658]. The family also includes proteins involved in nitrogen fixation, RnfD and RnfE. The similarity of these proteins to NADH-ubiquinone oxidoreductases was previously noted [PMID:9154934].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007527': {'PMID': 10077658,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '6',\n",
       "     'year': 1999,\n",
       "     'title': 'Effects of changes in membrane sodium flux on virulence gene expression in Vibrio cholerae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3183-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Hase CC', 'Mekalanos JJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.96.6.3183'},\n",
       "    'PUB00007528': {'PMID': 9154934,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '18',\n",
       "     'year': 1997,\n",
       "     'title': 'Membrane localization, topology, and mutual stabilization of the rnfABC gene products in Rhodobacter capsulatus and implications for a new family of energy-coupling NADH oxidoreductases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5509-21',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Kumagai H', 'Fujiwara T', 'Matsubara H', 'Saeki K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi970014q'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 67,\n",
       "    'interactions': 0,\n",
       "    'matches': 25029,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24790,\n",
       "    'proteomes': 3020,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20794, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 14328},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22414,\n",
       "    'alignment:full': 4282,\n",
       "    'alignment:seed': 440},\n",
       "   'cross_references': {}}},\n",
       " 'PF13378': {'metadata': {'accession': 'PF13378',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029065',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Enolase C-terminal domain-like', 'short': 'MR_MLE_C'},\n",
       "   'description': ['<p>This domain appears at the C-terminus of many of the proteins that carry the MR_MLE_N Pfam:PF02746 domain.  EC:4.2.1.40.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0256', 'name': 'Enolase_TIM'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 304,\n",
       "    'interactions': 0,\n",
       "    'matches': 140007,\n",
       "    'pathways': 0,\n",
       "    'proteins': 139875,\n",
       "    'proteomes': 7961,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 119000, 'rosettafold': 0},\n",
       "    'structures': 293,\n",
       "    'taxa': 33091},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 130073,\n",
       "    'alignment:full': 25875,\n",
       "    'alignment:seed': 334},\n",
       "   'cross_references': {}}},\n",
       " 'PF10415': {'metadata': {'accession': 'PF10415',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018951',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fumarase C C-terminus', 'short': 'FumaraseC_C'},\n",
       "   'description': [\"<p>Fumarase C catalyses the stereo-specific interconversion of fumarate to L-malate as part of the Kreb's cycle. The full-length protein forms a tetramer with visible globular shape. FumaraseC_C is the C-terminal 65 residues referred to as domain 3. The core of the molecule consists of a bundle of 20 alpha-helices from the five-helix bundle of domain 2. The projections from the core of the tetramer are generated from domains 1 and 3 of each subunit [PMID:8909293]. FumaraseC_C does not appear to be part of either the active site or the activation site but is helical in structure forming a little bundle.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00025036': {'PMID': 8909293,\n",
       "     'ISBN': None,\n",
       "     'volume': '35',\n",
       "     'issue': '44',\n",
       "     'year': 1996,\n",
       "     'title': 'Crystallographic studies of the catalytic and a second site in fumarase C from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13955-65',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Weaver T', 'Banaszak L.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi9614702'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 118,\n",
       "    'interactions': 0,\n",
       "    'matches': 52983,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52978,\n",
       "    'proteomes': 9205,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46429, 'rosettafold': 0},\n",
       "    'structures': 56,\n",
       "    'taxa': 37440},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50690,\n",
       "    'alignment:full': 10729,\n",
       "    'alignment:seed': 612},\n",
       "   'cross_references': {}}},\n",
       " 'PF02934': {'metadata': {'accession': 'PF02934',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006075',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GatB/GatE catalytic domain', 'short': 'GatB_N'},\n",
       "   'description': ['<p>This domain is found in the GatB and GatE proteins [PMID:16216574].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0286', 'name': 'GCS'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 129,\n",
       "    'interactions': 0,\n",
       "    'matches': 50358,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50045,\n",
       "    'proteomes': 9568,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42109, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 40342},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44138,\n",
       "    'alignment:full': 9382,\n",
       "    'alignment:seed': 442},\n",
       "   'cross_references': {}}},\n",
       " 'PF00381': {'metadata': {'accession': 'PF00381',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000032',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PTS HPr component phosphorylation site',\n",
       "    'short': 'PTS-HPr'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020203': {'PMID': 8226757,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '30',\n",
       "     'year': 1993,\n",
       "     'title': 'The 2.0-A resolution structure of Escherichia coli histidine-containing phosphocarrier protein HPr. A redetermination.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '22490-501',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Jia Z', 'Quail JW', 'Waygood EB', 'Delbaere LT.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/268/30/22490.pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 121,\n",
       "    'interactions': 0,\n",
       "    'matches': 54827,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54644,\n",
       "    'proteomes': 6739,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 48576, 'rosettafold': 0},\n",
       "    'structures': 60,\n",
       "    'taxa': 27700},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54220,\n",
       "    'alignment:full': 10932,\n",
       "    'alignment:seed': 939},\n",
       "   'cross_references': {}}},\n",
       " 'PF02545': {'metadata': {'accession': 'PF02545',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003697',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Maf-like protein', 'short': 'Maf'},\n",
       "   'description': ['<p>Maf is a putative inhibitor of septum formation [PMID:8387996] in eukaryotes,  bacteria, and archaea.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008179': {'PMID': 8387996,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '10',\n",
       "     'year': 1993,\n",
       "     'title': 'Amplification of the Bacillus subtilis maf gene results in arrested septum formation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3139-45',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Butler YX', 'Abhayawardhane Y', 'Stewart GC.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8387996&action=stream&blobtype=pdf'},\n",
       "    'PUB00019830': {'PMID': 1400224,\n",
       "     'ISBN': None,\n",
       "     'volume': '174',\n",
       "     'issue': '21',\n",
       "     'year': 1992,\n",
       "     'title': 'Identification of Bacillus subtilis genes for septum placement and shape determination.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6717-28',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Levin PA', 'Margolis PS', 'Setlow P', 'Losick R', 'Sun D.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=1400224'}},\n",
       "   'set_info': {'accession': 'CL0269', 'name': 'Maf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 159,\n",
       "    'interactions': 0,\n",
       "    'matches': 60108,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59898,\n",
       "    'proteomes': 9964,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 51172, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 40396},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54943,\n",
       "    'alignment:full': 12292,\n",
       "    'alignment:seed': 1096},\n",
       "   'cross_references': {}}},\n",
       " 'PF16199': {'metadata': {'accession': 'PF16199',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032432',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Radical_SAM C-terminal domain', 'short': 'Radical_SAM_C'},\n",
       "   'description': ['<p>This domain is found as a C-terminal extension to a subset of Radical_SAM domains. It is found in archaeal, bacterial, fungal, plant and human proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 78,\n",
       "    'interactions': 0,\n",
       "    'matches': 22459,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22456,\n",
       "    'proteomes': 4594,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19191, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 20210},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19912,\n",
       "    'alignment:full': 4962,\n",
       "    'alignment:seed': 501},\n",
       "   'cross_references': {}}},\n",
       " 'PF01132': {'metadata': {'accession': 'PF01132',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001059',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Elongation factor P (EF-P) OB domain', 'short': 'EFP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 53,\n",
       "    'interactions': 0,\n",
       "    'matches': 48037,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48035,\n",
       "    'proteomes': 9038,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41434, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 38025},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43671,\n",
       "    'alignment:full': 9414,\n",
       "    'alignment:seed': 685},\n",
       "   'cross_references': {}}},\n",
       " 'PF00020': {'metadata': {'accession': 'PF00020',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001368',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TNFR/NGFR cysteine-rich region', 'short': 'TNFR_c6'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0607', 'name': 'TNF_receptor'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 224,\n",
       "    'interactions': 0,\n",
       "    'matches': 30904,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16804,\n",
       "    'proteomes': 831,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12199, 'rosettafold': 0},\n",
       "    'structures': 92,\n",
       "    'taxa': 3306},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23405,\n",
       "    'alignment:full': 12940,\n",
       "    'alignment:seed': 489},\n",
       "   'cross_references': {}}},\n",
       " 'PF13927': {'metadata': {'accession': 'PF13927',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Immunoglobulin domain', 'short': 'Ig_3'},\n",
       "   'description': ['<p>This family contains immunoglobulin-like domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0011', 'name': 'Ig'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 13822,\n",
       "    'interactions': 0,\n",
       "    'matches': 328283,\n",
       "    'pathways': 0,\n",
       "    'proteins': 185048,\n",
       "    'proteomes': 1138,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 103530, 'rosettafold': 0},\n",
       "    'structures': 168,\n",
       "    'taxa': 6614},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 225048,\n",
       "    'alignment:full': 144080,\n",
       "    'alignment:seed': 794},\n",
       "   'cross_references': {}}},\n",
       " 'PF06421': {'metadata': {'accession': 'PF06421',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013842',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GTP-binding protein LepA C-terminus', 'short': 'LepA_C'},\n",
       "   'description': ['<p>This family consists of the C-terminal region of several pro- and eukaryotic GTP-binding LepA proteins [PMID:11489118].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007679': {'PMID': 11489118,\n",
       "     'ISBN': None,\n",
       "     'volume': '41',\n",
       "     'issue': '2',\n",
       "     'year': 2001,\n",
       "     'title': 'Evolution of a molecular switch: universal bacterial GTPases regulate ribosome function.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '289-97',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Caldon CE', 'Yoong P', 'March PE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.2001.02536.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 143,\n",
       "    'interactions': 0,\n",
       "    'matches': 48477,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48452,\n",
       "    'proteomes': 10547,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41258, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 43470},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43854,\n",
       "    'alignment:full': 10020,\n",
       "    'alignment:seed': 253},\n",
       "   'cross_references': {}}},\n",
       " 'PF13012': {'metadata': {'accession': 'PF13012',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024969',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Maintenance of mitochondrial structure and function',\n",
       "    'short': 'MitMem_reg'},\n",
       "   'description': ['<p>This is C-terminal to the Mov24 region of the yeast proteasomal subunit Rpn11 and seems likely to regulate the mitochondrial fission and tubulation processes, ie the outer mitochondrial membrane proteins. This function appears to be unrelated to the proteasome activity of the N-terminal region [PMID:18172023].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 182,\n",
       "    'interactions': 0,\n",
       "    'matches': 13382,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13361,\n",
       "    'proteomes': 2168,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11083, 'rosettafold': 0},\n",
       "    'structures': 118,\n",
       "    'taxa': 8264},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11042,\n",
       "    'alignment:full': 6786,\n",
       "    'alignment:seed': 257},\n",
       "   'cross_references': {}}},\n",
       " 'PF13774': {'metadata': {'accession': 'PF13774',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010908',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Regulated-SNARE-like domain', 'short': 'Longin'},\n",
       "   'description': ['<p>Longin is one of the approximately 26 components required for transporting proteins from the ER to the plasma membrane, via the Golgi apparatus. It is necessary for the steps of the transfer from the ER to the Golgi complex [PMID:16855025]. Longins are the only R-SNAREs that are common to all eukaryotes, and they are characterised by a conserved N-terminal domain with a profilin-like fold called a longin domain [PMID:15544955].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0431', 'name': 'PF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 189,\n",
       "    'interactions': 0,\n",
       "    'matches': 15860,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15828,\n",
       "    'proteomes': 2146,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13170, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 8011},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13043,\n",
       "    'alignment:full': 8032,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF00926': {'metadata': {'accession': 'PF00926',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000422',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '3,4-dihydroxy-2-butanone 4-phosphate synthase',\n",
       "    'short': 'DHBP_synthase'},\n",
       "   'description': ['<p>3,4-Dihydroxy-2-butanone 4-phosphate is biosynthesised from ribulose 5-phosphate and serves as the biosynthetic precursor for the xylene ring of riboflavin. Sometimes found as a bifunctional enzyme with Pfam:PF00925.</p>'],\n",
       "   'wikipedia': {'title': '3,4-dihydroxy-2-butanone-4-phosphate_synthase',\n",
       "    'extract': '<p>The enzyme <b>3,4-dihydroxy-2-butanone 4-phosphate synthase</b> (RibB) EC <span class=\"external text\">4.1.99.12</span> catalyses the conversion of D-ribulose 5-phosphate to formate and 3,4-dihydroxy-2-butanone 4-phosphate, the latter serving as the biosynthetic precursor for the xylene ring of riboflavin. In <i>Photobacterium leiognathi</i>, the riboflavin synthesis genes ribB, ribE, ribH and ribA all reside in the lux operon. RibB is sometimes found as a bifunctional enzyme with GTP cyclohydrolase II that catalyses the first committed step in the biosynthesis of riboflavin. No sequences with significant homology to DHBP synthase are found in the metazoa.</p>',\n",
       "    'thumbnail': '/9j/4AAQSkZJRgABAQEASABIAAD//gBGRmlsZSBzb3VyY2U6IGh0dHA6Ly9jb21tb25zLndpa2ltZWRpYS5vcmcvd2lraS9GaWxlOlBEQl8xZzU4X0VCSS5qcGf/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCADwAUADASIAAhEBAxEB/8QAHAABAAMBAQEBAQAAAAAAAAAAAAUGBwgEAwIB/8QAPxAAAQMDAwMCBAMHAwIFBQAAAQIDBAAFEQYSIQcxQRNRFCJhcRUygQgWI0JSYpEzcqFDsRckU4LBJTRj0fD/xAAaAQEAAwEBAQAAAAAAAAAAAAAAAwQFBgIB/8QAMhEAAQMCBAQFBAICAwEAAAAAAQACAwQREiExQQVRYXETgZGh8CKxweEU0RXxBiMyQv/aAAwDAQACEQMRAD8A6ppSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlBSiJSlCcURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURK/Li0tNqccUlCEgqUpRwAB3JNZd1D6zWfStwk2mBGkXW9MEJcZR/DZZUQDhbhHfBBwkK+uKjumfUNfUuDfNM6mhMwbg5GcTuik+m8wsbCU7iSFAKHGTnOfpURmZiwXzU4ppCzxLfSvzrPr7Y7QtbWm4jmoXGlD1XWXA0wkZ52uEHef8AaCPrWp6cvUPUNig3e2LK4cxoOtlQwQD3BHgg5BHuK46v2lbjpG6Ltd1jFBa4ac2n0pCB2Ug9iCO47jscVuH7O1+gM6Xn2t+bHZVFlLdaaddCVJaWAo8HuAoK5qnBWOfMY3iy0Krh7I6cSxm6025antttvcO1SlvJkSilKVJZUptClEhAWsDCSoghIPcipkqG7bkbsZxnmucdc9QpEqffhYrS7Itz0lkIu+xS20ehgpUjAxkObiCT+nmqjO6hami3h3UMW6//AFFxCW1bmkraU2k5DezHCck/lIOSTnNTSV8DCGi915ZwmaRmMWC6mhSH499mwZTqloeHxUZSvCOErR/7Tg/ZdTArN4uqHLxpe2ajlQXrXcrZLaauER9JSplLmwOJIPO0ocQ4PsnPatIFXSQ5oeN/ws2Rpac0qL1LdBaLS5JSj1ZCiGo7I7uuqOEJ/Un/ABmpSqipRvPUP0FHMSxR0vFJ8yXgoJJ8Ha2Dj29SvrACbnQL40Z5qZ01bDaLUiO66XpK1KekPE59R1RypX0Gew9sVK5qn9VpNyg6NfmWWeqDLjvsOhwIC8pDid6Sk/mBTnI9qpZ1Rc5j0PVbNzV8E3c4duct7TmWQw6EodWtPfd6q8pJ7BAA4Uc+X4g3xDzUjInS5/L8vdbIe1fhLqFLWhK0FSMb0g5KcjIz7cVHaouqLHp25XNe0iJHW6ArsSBwD9zgVx5Fu11tt1kX2Hc5ke6ukuvyGl4U6rOcLB4UM+CCPtVOprGU5AcNVZo+Hvqmuc02su16VnvSjWku99MI2o9XLiwVAuh1/wD0mihCykL5PGcf57e1XSzXWBere3OtM2PMiOD5XWHAtJ+mR2P07irIcCqTmFpN17qUpXpeUpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREoTjvSqxryRP8AgbfbrQ6uPKukxEMyUY3R29qluOJz/MEIUAfBINfHHCLr0xuM2U5LucGGsImTY0dRGQl11KDj9TX9i3GDLcU3FmRn3EjJS06lRA9yAai7VpayW9gNsW1hahyp59PrOuH+pTi8qUT3yTX6uulrTckoUqKmNJbO5qVFw080fBStPPnscg+QakGA80+nZYx+0Fof4eevV0AJLT+xqcjtsXwlDg+h4Sfrg+TjOOnmoF6d1lDuTFtk3NTKHEqYi5K9ik4J4B4HfnFbh1NmLjaZ/BtWFbzDsmOUS2U7fi20upKkEdkOYH2PcdiKmY/S/Tk23FN+s8V1+SoOvNMbmWkkDCUBKCMhI985OSe9Z0lBebxb2GXmen5+y2I64R0wjkF73Hll/ayrqnfXuqs+x2vQ1vnyn4XquTW5DCmfhlq2hIWpXyjgKPfzxmrhZ+nFugQrJpp+DAmXNwGZdrgplK1oaB/021KGU71YSCP5UrPerlCsdojS0taQu7VqcaSGXIcNbbjSgkY+Zk5woAfmGDxzmrHarY1b0urQpx+S+Qp6Q8crcIGBnAAAA7AYAqy6kjuXuNyfnz8KmK4hgbEbAafOi9rTSGGUNMIS22hISlCEgJSB4AHYVV7xpPTTTblxdt0aE7HHr/Fxmw240UjO5JA7/pzVrJzxWaXiJddYR9RuNPOLRbppiRLWl70WpHp7FL9ZQ5UV5UACdoG04zk1PG1rnAnZVY3EHI2X4iaau19sl1jyr5FZdvBDkxpcNLkhtG1KUAlKwAvYlPO3AUTwa05J4FYfrS526K1bGbTpd6zSyXCXpMRUZ5opAwW3UHaognn5lDH3r1QOrFyjBKLlbo8tKUgFbSy0s+5IIIz9OBVsUcj4w6MZeQVSo4jEJMLjn2Wzms51lDudmur0qz3P0hqSZHhLbUwCuO4UbC6hzPGEIyElJyoDBGan7VrrT1xtYmi5MRk8hbUlYQ6gjuCnP/bIPiq4zbv37gvP3C9hMpaQ/Et8OSAmDzltbgScqczjJPAOQkAjNVvDe0E6eV1ZgkjJDiclWuonTbUT0GKbZfLleozDex2FLeBcWfKgo43Z/pJ/z4rEfRGrXJ70izWZ2PESylp1mQtLPrpGDtSCeSFJCgTgZA5rTkvXO06XVqC3XOS6zFSpUu13FfrpJbUUuIQ6fnSoEKwSVDIAx5rRkLC0JUAcKAIyOaqVNCJJMbnHLbb0/paUfEpYIjEAD1tmuU9WXHX15glu7uXGfbniQ0Y0bY2tSVY2OJQOFhQIKVdlCq70ytcfWmrmbHJkrjMvNqcWpA+chsgqQM9lHkZ8c8eK6pDaLFqd6QVelb7tt3g8JRKSMbj7b0gD6lA96pXUDp5p+Pf0antFzd01qdbiloeiNl4SVqGFZYAJUognJSM5OTmqv+OLnB7ruP3srDeJWAjjbhBGo69Onuojr+03b7JpmwW5pDFrT6i/QTwP4QQlAx3IG8nnzg96yHTib9E1dZY+jZTsO4TpaGnVMHgtggqLifyqSE7icg9qs0fQWsr5e3vj3Q/OPJfmvqbdWjwoIWN236YwO3FXHpvaX9Eamv0ObEZuWoD8Oxb1NHCVocQpa1AnlKBtG448AeRmAU88lYHFpaOvb9dlaMkMNEYgQ51r+p39VvWaE4qr/uqi4ESNSyXLi/3DAUpEZo/2oBGf9ysn7dq+i9HWdQJZYkRXDwHI0p1pSfthVbmFg1d7ftc3Zo1KsgINKqWlplyhX+46dvEpU4x2W5cOYtAS46woqSUubQElaFJxkY3Ag4Bzm2ioWuxI9mA2SlKV6XlKUpREpSlESlKURKUpREpSlESlKURKUpRF/FHArJtTSNSa5m3ROkltW+Jp6QtMeSv/AFJ01CSlbSTnCGwFKQVeSeOxxrKuBVB6b3CBDm3vTLslCLxEuMl9cdw7VuNPOF1DiR/MClQBI7EGopAHWadCp4ThBe3UfD86rx6Kj2vWWn03O33HUdvnIWpiSyu6PKdhyEHCm1pWSkkHwpOD5FWvTlxliXIs94WhdyiIS4H0J2JktHIDgT4OQQodge3BrKNc6gT0162MXNts/g9/iNi5NJP/AFErKPWA/qSnZn3GfNXqbqKyXDVlgftt1hP+iZKJD7byShLewfKVZxysJx9jipKZgcXNAzA/Y/pSSB7wCdCMt7W/Cnddaba1Tpx+2uOBtwqS6y6RkIcScgke3g/QmqFbblOmfB2HWF8aitIQW30NKLT8hQJwh5ZIKQUgcoGF8/N764lSVpCkkKSRkEcgioPVumLZqe1vRLjHbLim1Iak7AXY6iOFoJ7EHB9uOalbIWtta6rXuMJNvK/t+wqn1M03ptWl0tBUK0TIo9a3PNJCFJcA4ACfmUk9jjPv3FZEOqdwgWz4K+LflwlhJUmSVFbZBBGHW1BYwR35ryWRc7ROuHrPqZ1Cn2j6K3uQiQwofI4nJ7EYyPBBHivZeOncy9WyfdokmIxZW9ykSHlHLnOMJQBnJPAzisr/ACtW2UxiP6Qc75+d9lrN4NReGJTIQ4jKxtfpbO5HwLXNMTH73Abd0jq9RQtG8xrrHEtbY/tUFIWR9SVfcVbdI2I2G3OsvTHJ0yS+uXKkrQEeq6vGSEjhKQAABzgAcnvXLFoErT1zabSHWmi4kFBJQptXA3pI7Htn3rY7N1EuVuAZuTQuDSeN+djoH37K/XH3rXpCOJQ+LT7ZEb377/dYvES/hk/gVWhzDhuOoGh9lYOsU7T1qsjE/UEIzpTRWmBGC1pLjhTynKew4BJPbFYfqC4sHRtku0O2iNOmuSZEhkvqUluKhRSjaFc88YV5wc96vvWDUNo1dpFpu3KkpusSQiQyy7HI390rQVD5R8qic5xkCszQpMq1xoSrWYJhNPsreW4FqlJWtBRx/KE4x35KjxWjTRSMaAbjNZz5YJXm9iLeeWeuul197eh2QZjrDaVNxoqpbm8hOG0FJJ/57VrXT7R97jzJskXBVrSptCEvRfQkB7CjuB3BWCCB7cnzVF0VLt1qjahevkV2XGl212OlptJWpwHugJHcq4x9quH7P9/I09eU3F34K1x5ykwjPc2vFKkhSgrccnGU8++6pKuSQNLQMslBSMjOAk5i517DPf4FpKdKwf3Uk2B1yS9FkodS8844C6tThKlrKsY3FSie2AfFRa7NeY0RxV81q6m3tJyt1qK1Gc2jyt05H3KQn6YqK1H1RhxUrZsLCp8jsHnMtsJ+v9S/sBj6ish1RebnfHPiL9PXIbbO5LZAbYa/2oHGfqcn61TipJnfU/IdVdl4lGw2Z9Tj5q2T+otjsqbhFsdscuKXyEuSpz7kgSAnOCQs9uTjkfapfpXplzUDC9QXuE1EhyAREjMqWlS09vUUc5CT/KkeOTnisissVF+ubERgKU24raVY7jz+la9P6sN6ddmW+Qw1L+DHpNuNj0juAHylABBA904HioRxCOWU01JcncjT1P3yHJWH0NTDEKiudYHRpOfoNegzPMKZ1jCl6et5eblGbCQrdFTIePxcZ7+UMrOS6Dj8iuSOMkdv301tM643q5a2vaG2ZN1abbhxUHd8PHSONx/qURkgdu32i9K3Z3VNv/EGCbnfZbZDb7aCI1tbWMbQ5jb6gH5tuVZ47DJ1K3RG4MCNFZz6bDaW0kjGQBjNeZHtLRY3IvmPQqdheyPC5uG+29tc+Wy8d7nOxkMR4YQqdKX6bKVdk45UtX9qRz98DzVT1badIWK1u3nWbgkbfzypjqlLUruEoSCAO3CUgAf5NT13Ko+rbFIVyw82/Ez/AEuKCVp/yG1/4FYH1Iua9c9XbfYJiybNEujUL0M8LwseoT9Tgp+2KpVbmsjaCL3/ANK7RxOebtNgBc217K96ZjXhy2SNaaejSYMctf8AkrLMeW6JMQZUSdxJaWsncgJ4GBkHccazaprVytkSdGJ9GS0h5G4YO1QBGfrzULrPVEDS0BK5CvUmv/w4UFobnpTvZKEIHJ5IyewzzXu0jbnLTpa0W94hT0WI0y4QcgqSkA4PtnNfI24DhbooZXmVuN2t8u36y9VLUpSp1WSlKURKUpREpSlESlKURKUpREPas16i9UI+mVuQ7TD/ABK5JyFFS9jLSvZSgCVKH9Kf1Ir+9d7prG16QL2iYoczn42Q188iO1x8zTf82ecnkpHIHlOV6cnRtZ2mPa0thy57B8MWhnePc+wHck1mcQqpYcLYhmVscKooajFJMcm/LnorBofr04/cHomtbczEa3DZMgJWpCAf/UQSVY/uGe44Het1gzI06I1JhSGpEZ1O5t1pYWlY9wRwa551b09tmnTaoSJqVXSS069JfcVhOAUBKUp8DJVz3OD9h7dDakjdNYNyauKnZbExaHYcaGNxU6PlWecJSCC3kk+PJqOKvcyXwZ/XyUs/DI5YP5FN6c87ZLfZEhmO0XJDqGmx3WtQSB+prK+pZ6YarMdvUN8trdxYyI8yHNSmSwc9gpOfP8qgR5xWKXES+pWu4itXXZUW2LdKlpDhQ3HZHOxsf1HhOe5JzVv6sWXTTcm1J0lbYTG1tQkrZPpHAwlAO4jkYPJGTnmrfjSSxGWJhI7FQs4dgmbFI4gnPLbz+dlATINk1tq59qJctX6tu0dr02XJSWY0X5EqKEKVtBOSCcYTu5JzzX7umuNSxw5BZtNqtEuNHTb0pcYL/pIbJOzCspzk8nHgcV6enOpWNK6ogqeLb0N5fpPuZO1oKwnen/b5PkZxVWvzrib7dHH1fxlTHS/k917yFH/NU6iteyICI2dvbn3+Z3WnT0DBIWyC7QBa60Nq2WR/QkRuX1GX+PuoVJ2id8G2lw/mQhkFOAMYAPfGRjIx5un3Ue8aau6IOoZz9wtRX6bq5Cy44xk8OJWfmKfJBzxyPr/Oh2p48SdqW3XFLb1q/DVzn23EhSf4ZCSCDwdwWBjzgVn8iNMuMV5UJLbqmWEtlAPO5IxtH1xjv3qUPnqmNfHnINhuN7+1v9Ku/wDj05fHUn/ruAHG2RPW3yy6f6jaMGpI3x1sUlu9x0AMrWo+m6kEn01+wOfzdwceOK58u+pH2Jj1paXJhllxCpkZz5VB9PICh528cjg5B5GK6o020qPp62MrcS6tuK0guJVuCiEAEg+azLrv0vOqoSr7p1tLepYiOUDgTmwP9NX94/lV/wC08EFOlUyzS0rqZpyPyywKKOnhrW1Thcj5dZ1G1VvbafuVpZuiY7gUQ4QlKuDhKjg5GecY5xUVY9VQZepvg79Fas1tkA7JMErU1HXngrQ4ThHPJSRjv27TelNO3WRpQW+baZsHLyPUlTmiwhta+xUFfNjsOAccV+LroGPEuUyDGuLcybHSlD5KAhG8jcUJGTwAU5J8+BXO0lRV8OBe0ENv2v8A2uqrKag4q7wnkOcBtnYfYZ9s1oEjpRPXzGu8NxtQyFKaUnP+Ca/tu6cwoGoIMG9yfj0TWHnC02CykFst4GQrJ5Xnx2FSPQUXePYbjAui3FRYUhLUQOHJbTsCigHykZGPbOOwFXDVVumvqgXGzhlVyt7qloaeUUpfbUNrjRVg7cjBBwcFIzxmurh4lJUMDiciPnuuOfwuGjncy2YuOmYWPy9GXmFcrnEtjK7rFgvJaS4HEJkBBQlSd6ThKvzYyCCcZIFeVvSeoZbgxYZ6VHjLwbSB+u+tHtd/kwJ+orrdrI/GjoW2mStqS06lj02gTu+ZP8qgeAe+K9t66i2u2W4ynIk8eo2VRRIa+GTKXt3BCFOYGSPfxzV4Vs7BYAHIfboq0nBonuuy/kqXD6eLZVHc1Ld4dsbdOEshxJcWfKQpWBn7BVZVqqxSDq28xFl5bEKU420HFfK02OUkntnaQcn3qPvl0ump9QPXfU6mvi3f4bTKVpU3Hbzw2jnGPc9ycmtI03Z4jdvUwUNie0kPFsOApU2cYcCfJTkAk52kjHBrBnnqOK1H8JzsA1O2XbfpmtplPTf8fpv55bjccgOR112y6LzXC5sOwrcu12lq0zW2EplTWUhpb6tuDtSngJPfcQFewHc+jRvTR7VUmLNu7ZbsIJUpJUQuUBjCR7IPOVdyAQO+a/WiFR9T6+atkWCbjbYgU5cZO7DLXB9NOf5yVDt5Ge4BroRKQhICQEpAwAOwFa0j6enYYKXPmefnv9lhsjqamQVFZlybnkNdDoOhzXPHVvqRcUTnrDpF42+DCPw6noowta0/LsRgfKkHgY54qraEjasRcHLh+82rnpjbim1wYMVcpCVf/nddPpDkcgbiOexFVq/F7T+rrhbZzio0yJKUQpZ2qUNxKFp+ihhQP1q+3LVvwXRvTFltbgZXP+IRKW2cHa24UqSfqonk+wPvXLMncHvdLtnbz0XbvpmCONkG5tfyuSeei+z3Vm+sJjNXtqBLegvKkJUw2pHqKQlQCVkEjOT3SMcDPGRUtA0/0v6kXhN9s+oJVvvclxL7jMKd6Lgf4JIStOd2R3TjnmsYLqGErSMYQSoAeOB/+q0LWDlka6b2WwS7WwbthqSh9plKNjS0BQUogck7tuO/G4+Myx1xmjc6QABug79efzReZ+HtY5oiuCcjb+uS3XSegbDpqc5Pisvyru4MOXGe8qRJWPqtXb9MVbh2rl7Qmpeps2dDtWlZ0SezFbBdZuZTtQ0CACXPznvjjJq3f+OzlsnT7XfLAH7jEyhLlrlJdYeX4ypQGwfYqI8jPFW4qqMsDtAsiehmbJgviK3PIqma26k6c0iVMTpfxFyAymBFHqPH2yOyB9VEVmszq3d77pm7QY9ochXR9kojPwni4W88EnIBBCckKGeccCqF0s0RDuWrosCaw+mE+06pakKUlRIQSlWfJzzzmoJOIsJDITclWYuESNDpKgWa3PK2mq2nSvWC33qSliZap1vKufUyl5CR7qKeR98Yq1adcbXqnU621oUlTsYjac5HoJwf1/8Aisgvkf8A8OUPQPSWuVIypiaR8rzY749lJ7KT9Qexr49GNfMW/VZslxwRd3EoYfJ5S6lOEtkf0kDA9j9+I6TiEolMEwzOX5Vir4dCIDUUxyt65jP9LoylKVrLn0pSlESlKURKUpRExUZbLDarVKmSrZbokWRMX6kh1loJU6r3UR39/uSfNSdK+WX25C5c6/3aTC6krRPYdZjfCstw3loIQ8MFSwlXYkKVyO9QFiiu3+I2YOX5anw2yyDkr7E4/wDn6DNdYagsds1Da3bde4LE6E7+Zp5O4Z9x5BHgjBHioDp/07sOhWX0WZt9xx5aj60pz1HEIJyG0nHCR/k+SayqnhYmkxg2vqtyk40aeHwy29hYftc0yrLITfn7JcYxj3NvJ9Fz+YDJyk9j3BBHcCpjVaLqrQMWdc5K5MmzzFxyl9wKeXGcShSSMncvavcPJwr2Bp1I1NHuHUq8SGQ2owZHwrLikgqQWk7FbT3HzFdRtwhyb1Bi3BbRkKdUYzexO5RIOAMDnkk4rLlk8KTw7EtGXuSFuxN8eJstwHnP2sV5rB0/1nqpEc2+0KiwJC+Z0l1tKGkE8qCQoqVgHOMCtt1T0Lst+v7tyTdrrAbkHdIjxi3ha8DKklSTtJIye4JOeKq2humk/QVhuWrpE9+Hd2mxJTBbdIZDSDuW28BwsrTkf2nBHNdCNq3tpUARuGcKGCK3aeCMNsW565rmKyqlLrh+WYy8lkqulmjNF6WvzrkWTcRMjhp0zpSip0BQUlAKdu0FYSTt54+mKpcjp5c4kViXpmHJbtiFNuOoLWxZ4Jc9NrO5SCcYKvmGTjPFav1btr0iwxbpDeU1Ls8lM1rjcknBTynscbgefaoHpJ1Gcvrrlr1I80m7qWTHWhsNodSAMpHP5wQo/UHjsa2acGGPxYxnv2WRUS4x4ch/9X+w/vQ6qpaS1rM028A3mVbVH54xV+Xnug/yn6dj9O9blYLzBv1tROtj4eYVweMKQod0qHcEe1U7qPoBu9tOz7MhDF2SNykflRJ+ivZXsr9D7jEtO6quejr8uRHQ4haF+lMhO/L6oB5SofyrHOFePqCQbErY6xviMydv8/Ky4XS0TvDkzbsV0hr2RMg6MvMi221d1mNxVlqGOS6cYxjuffA5OMDnFchWO+uy1NLZecfkvKHz7sqdWo9z7kmuw9LaktmqLaJ1okF1oEJWlSdqm1FIVtUPcAjtxWYay6LCdrlN/wBNSolvTJJXMZdQopS7kfxWwn+Y85BIBPOck1znEqR0zMO42XWcIrmU77nR26y/U1z1LaXZMJu+3OIts7imLJU2kKIGSMYz+tXzpFrTUGoW2rFddRpiXZtkKjuvQ0OmUkDJ3EkZWn24JHPOFVIdSNP6fsFvirva3bjd56gyl1X8MIQnBWtKE8cDAGSTlY5NV1GmIlxvdomaekj1FTmFpR/MghYUVD2wEk/pWZBPNQyNjcbk7a2v3WzLFT10JmaMPW2RI1+Fa1qS1We0dNrvF1HcFiCthxU2c6nK1OLPLm1I5VuI2pA/pArJuoWooWt9PaYYlh9i6sOoW6p5na04FtEKWn6btp7e48VCdYNWzNZa0ctYSprTtnlLb9Ld/wDcut5DjiseBtUlI8cnuRidtlgnQmnL7OtS37bKipeZMdHroKSgbdyRkp4+mBWrWVGKBxsXZ2y10zOh0y/sLPoKUMcJZTY6i/sPNfmwqt2inDMhyYsyYUei56rYDbAXwFlRGU4OMjuRkVolz6R2afZ2Wm5MuHcksltU2MvHqKUMKKmz8pSfYAccZrn6HbrnrC8x7JYcOPPkuKUSVNR2+ynV89gDj3JIA7108mXaunWkLVDuU+S7HjoTDaeeBccdUlBIHH0SceBwKj4a0Ph8JoJbffc9s7AdzuvPGXiGcS4rPAzty9uXLkvZpDTlq0Vptq3W4JajMguPPukBTq8fM64r3OP0AAGAAKqmp+o6TvjaeSFnkGWscD/Yk9/uePoazzWOu5Wo3FKkL+EtTastxs/4KyPzK+nYeMnmrPonQU67BuZe0O2+BkKTHUNr7w/uH/TT9PzHn8tdLDSw0zcdR6fP9LjZ6ueqeWU/mfn+1F2GI1ftUwWr3Dbu7Mh0+qiYErS4UpJyd3CinIO328VcdV9HNNXuzQINuQ5Y/gVOrjqgJSBlzG7elQO4ZAPgjHBFOpEexaXtUK5MBm3zo76FxAhJKVrSDncgHn5Cobu/OM84Ni0BrCHrC2SZEJSFKivqjuqbyUKI5CkEgHBB7EZHaq9aRUgS4ctFdoWGkGBr7u17fnzOqxO3/s8XsahQi6aihP2EKCnHGmVokujynacpTntu3HHtWZ6gvrk7UFwE1pbVyVJWl2Jg7mNh2+njwEgJSPoBXcCsY5OKze1aOsWqhfJeprXHnXBd1kJU86nDrSW1bGkoWMKSPTSg8HkqJ81iVFGySzW5LfpeIyRXe8307rBtJSLm1FvbFtjOrmXOAqCgtLwppKlAqV25OAQAPeolnTpt0cvIjrYab4zt27ecf962C/2tPTNqQ4ww5ITKVshzVchGRnYv2WMHB7KA8EEVlM3VM19iRYzKUsXd5Cdnpl1YcKh8yQnKj9QO+KwHtnDvAcLW2XSxyQOaahtrG2ZPy3bmrdFlrsOlYMsRdj10bUpmUnlCgkkKSPZQx29uRnnEbo7XUi26wsEBx9TrLtwbQlvG5Q3q2HHnHzE/pXRM3RFqmaHa0tISswWmEstupIDra0jh1JxwsHn25IIIJFf3RehNPaObV+CW9tuStO12W6fUfd/3OHnHHYYH0rVj4SGSh+LIW9VjS8bEkTmFtyb+m3svbq3TFt1XZnLbd21qZUQtDjatrjKx2WhXg/8ABBIOQSKidHdNdL6SUh6121Dk9I5nSv4sgnkZCz+Xg4wkAVcaVr4G3xWzWCJHhuC+XJKUpXpeEpSlESlKURKUpREpShoiZFeWfcYVuaS5cJkeK2o4Cn3Etgn7k1zfrzqjqDWC9TQNJhdusdoV6UmWwd8h9PqFBWORtRkDgHOOSQOBn/Rj8Hd17HOq4wucNavgQJf8RDbj24BZSrIwVDH0K8+KqvqmtdhCvR0LnMxk88uwJ/C2TqvZul2o1fGL1NarNfXyoInQnkLLqh/6zaThY+pwfZVfzpVO0TozT7c29avsUy5bi3liRubZIH/TSfmJI7qx5wPrkegYdvgdbGYrdtaTEj3l2O3EV8wbT6ym0gk8nZwefapa/wCnLbZ/2iPw6LbWEwFXCM+3Db/IfU9MkY8DcVHb2H2qBz2k+IWi4NvNW2QuA8EONnC/lyW3XHXektbpb07Zb3Elvy5TTLzPzIKmgresDIGchJTx71pwPHtXGv7QDbEHrC5NsUVy3y0Fp5xYSUFyQFf6yRgd8DkfmIJ811Jp3VcW86TF1DrEd1DSjIQ6rCWXEo3KCvoBzn25q+0uJLSNPzosmRrWhr2k2dewPTWy++sHlohwI7asCZOYjOAgELbKsuJOfBQFD9a5s6g2CVo7Uioc5Di2XAJEKcyotB3BGffa4g4yAexCvPEjYtQdUNZ3a2XS3trkxmn/AImMh5hDUZI5SFL7EfKo4yc+QOK3XWumHNY6ElWq5ojN3F1krZcQTsZkAHYsHBOM9/JBI81oNc6lIB31VLGKhpDRppcX+9wqf0z6uRL018DfEqjSI6UJXMcUnYrPAUsgAJyRjOAMnHGRn09SOlbOsJrt5tV3VEuDzaOFoS4w6AnCe2FDIxzk/ase0n0h1tbtawI98t+bJMKos+RDlJU2tgjKknsoAlKfH/NbvLXqLRtlkLXPtVys8NslD9xWuO+0gcJSsoSpLhHABwknzzzVaaYRPxx5BWGQioYGv/8AXLn26qA6CWa46Wd1HY7+0lqeJDcpvYrc28yUBAWhXkbkEYIBHFa7kVzBC6k671E/dtR2KBbmYtsYQ1JX6GUlG4qAClLyTyScAYGM+K+o656pc9Yw4VtfS00hxz1YjidueDylZH5sjB5I7c5ApS18ZdfPPor0XC5mttll1+Zr6ftMxbrbtU2++SHW3bK8wmGztOFR3ASohQ8hWchQ9sHHGYfp9qyRpRH4wm3x5vxSC236rqkFtKT8xBAI548ePqau+ldSu3LSE+8a+0/GuFouCFMSZ0B/4otoJwW3GsBTaU8fkzjg9+asmnulejZWmGU2+TLuUB9rdFlqlby2k9lIIAGfuD7HzVKopTJIJac2d1+FadPWMgiMFSCW6Zd+40WQXlUczJl2nQdnxcgTm7Ww8l1xxDiyVAqyNnJKsEA4PatZ6N6t/EZky2LY9GKoF+HtOUt44caJ8EEhQHkFRHArOdU9PL5plMlwRV3CCkFQkxGio7RzlaBlSTjv3H1r99N4cCHYLndISLzc9UOhOF2dnem2BPzIQpSyltSj3WjJODtwO5horsmPitw2ub7527AjfpzVuvMUlOMDsV7AdLe4y5ldHwrZAhypUqJBisSZJBfeaaShbxGcFZAyrue/vVI6n2J3V9wtVijKQwWCqe7IXyEpwWwkAckncT3Hb614NM9Y7TIhMJ1HDuVomlKEuqchuKYU4eCEKSD57A4P3qTi6tsl+1zbEWS5IfkMtyGJLOxbagkgFJwoDICkEceTW7DIGv5GxPtyXL1FHK5pa8G2/l1Xz0r0rtViucS4uy5M2XFUVthxKA0FEEBW3BORng54qOu/VlFisrzV6gBnVDUh2MLehRKDtPyvbj2aUkpI8nOAOCRol/vMOxWWZdLk56UOK2XXFYySPYDyonAA8kgVnbPTJrU1za1HqtbibhKAedghIKGvlAQ2d2eUpwDgDJyeanZI2VxMzvnkqzojFFaFuvzdYtLVd9a39Bedem3WYUtJXtJDKCoDhI4QhOcn/JJPNb/oKBCsGq77YIRQlqLCglpvIClJ2OJKyB3JKck+5qR0/oiHpTT9whaVX8LcJKXFJnyk+usLVnaVDjKU5ACeBge5JPG+qpF/0/rq7fitxl/j8SWQ9MyQteD8jo5HylGFJAwACAKmdK2f6W5BV42PpwXE3J1XequUmqvpOWzJ1HqtENxpcdmY2hexWSH/AEk+oD9vlH3BHg1FdQtQ3LSfT1h+GXLjc3ENxRODOG0KKOZCwMhI4JAzgqKR5rAdM6ru+h27jcbEy3LKY+6S1JK1IWkLBLiyD+bKj8x8qPvUMcJdG9/zmpZZgJWM6/fL8rqbVFkg6ksUy03NKjGko2koOFoPdK0nwoEAg+4qJ0ToHTujmybNCBmLTtdmvn1JDv3Wew/tGB9KxnS3UjXnUnUz9ssMm26fS1CW8AGPiCpaQAApSuwKj4HA9/NY0BqHXOudbWvTt+1Zc4CYzjokfCqTHcKk5KmyUj5lfKQM5AweKzTMy+IC+y2BSyhpYXWGts/X2XXOaVydN1VfxKmNwdU32Rb/AIhaoy33djnpdk7toB//ALsDmvvobqTedL9SLbE1BIuUmy3ppDZM1xavTWpZSl1sq/lzwccd/aoIuIMlkMYGm6sTcJkhh8Vx12XVVKClaCyUpSlESlKURKUpREpSlEShpQ0RcXaoj3Hpd1E1DAkepHtN59VKH0pyHIy3N42n3STtUO/HsRUJ6Tdg1AxeUsm5WB91tyUiKoKISFhZ+3b/AJINbn19iTWLym5XCCJunVRW2lurbC2o6wtWQsfy5KhhfHfGalug+ktPoti9QxLJbWJDjhbYcQ0FKQEcFQJyUkqJHGOAKxXML6ossR9iP7uujbMIqIS3B7ag/wClhFxuNxna9l690rEjrT+Kh5iAzuU84MfmKMd1bcq9lK4zVks9vk3rVN+1bqB7UMPUDbqZUKFGssgvJcBGxJSpO0pSAEbdwyOciutqYrQ/jX1KyxXWtZvz0/tcea0a1BqHWjV81Vb3Lc48y2WYbiNnpNpyAMHJPzbjz5PbFbRpfTVnvHSh+Q7CCpciM+VLayXErSVAbfOflH37HjivR1d0Zd9Q3WHOs7TLxZiqbUhbobJIVkAE++T9OPrXs0Hco2lNMx7dek3KIW1uLU7IguJab3KKtpcAKQBnGScVpxQtihvE4l5Iv77LNdVTT1GAts0AgWHbfX8dFB9PIkKYrTMnRzj0cpZakXp+K4REeUWhubUg5Sp0qPJSAU45PYVseK/DKm1tIWypKm1jclSDkEHnIqO1BfYFhioeuDxQXVhplpCStx5Z7IQgcqUfYVBLID9RUoGIhrQpTHNZJ+0w5Ib0DGDW74Zc9tMggcBO1e3P03bf1xVqi62c5Nz0zqW3tbsJcchh8Ee5DKlqH6ivnftVafm22RBm2+7XONIRscjtWWU8Fg+DhvGf1GKrSYZWll7EqeAPhkbLhuB5+6xu6Q/w79nu2TbQ8G4s8Nt3BAT+ZwOK+fcOx3AJPuMdsc1fQrrydLXSS+2E2WROLS3FseoguoivKSk4IV3UnkAgEA+MHS4XRmBqGCWX1aj03p71PVYtKbjvU4snl1xKt6UHHASCT5JB4r6XboMprSM2x6e1ZdGYbjpltRJSGnGi8BhO5QQFAdgSPvgkVE+mdjDjY2FleFc0At5m/wA6hRf7Lk1L/wC81seWhcdXw7qWVkEEqC0r4PfKUpz9qkekWo4+l4Wu7a6sKh2u9ONwW1uhO/eop2Aq4A3J3E9hvJrPYXUHTejOn37qq0rNb1chxSXUTGwoKk7theDiTuVynACRzjA45q+fs/6OftraXtT25CLlcGXpimpTf8UNqUhCUuA/RBVt7jfzzU1PC1rmtOdgbrxPIJMb3bkZdl9+rkrVkm12XT1wnW1j945aIyXLWHEqbGR8hUpXzJIUMqAB/tqH6rruGjNK2nTFqVEt7EnchuHbQrDaAocqcV8y1KJOSQOcnnNWLqlY7g3CbjaYjmRNsUqPfLdHAKlBsLUHGUjykKCVbRzg4HYCq7rHUNg6mSdOuW03CLqaG9tetTsNzdtyCpsrwEAgp4JPOecePby4SFrMibWO9stOS+wYRgJFwL+u1/ZUi6wRoyNHtUxsu3RsplOvocKF4cRn01E5yBj8pB7/AFwPW1ZY0zp3cNQWy4rhyIz6ZJhJ3JWFIWElxDhVkhJJwEAAecnBHr15pnX15h21pWjJ67uC6/PnfEsLTIWUoQkDargJQ2kAe5PGeT/bvJuzHTDT2jrRojUq71v9STJkW1QCFqWVLCF424UTySQAnvz2qtc92JrxZpvawzF+up655q0algazCcwc88t/nmtU6QMRdYaZt18vbs643KHIWlSZclS2UvoPDiWuEbsEEHGRnitWHHass6b9KYtj0zETe3py7ytRkSDFuL7Lbbij+VKULCeBgZxzio/VcvQNmuS4l91lqBuSk4Ww3eZbgbP1CCcGpGf9TbGw81myBs8hwXPQC9h0z0WyA58YrBtZ6Fd60Fy+QmotkZjtux4El5sqkT1JUU5eA/02twOByvknj8pl+k2onmdSSbP+9SNTWGWkuWqa66FSW1JwVsPdlbtpBBI5APbkCc1Tpe6ZVAsyC/YrlcWJE2OiR6KmEl0KkbT3LbgySkEHJVjIVgWIZA4Et1VeSANOB+68vU1h+ydFUW6Y+HpDbMSGt1IIDigpAJH+DWP6NlQmLfrRVweZbS5p2U2hDhA9QnHAz3OdvH1rS+t2l4tr0c29YoTjDSJKDKSy8UspaSlR3KQVbSQoJAIGeay3THS2+a3lB1iYbIzAVtU9IjqUslaQcJbONw247kA5q2HOFI87n9BZ4Yw1rAT9I37XKrPRfVzOi+oMGZc8pgS2lRXl4/KDj5vrhQGfvXs6Y3qHB63MTrql2BDmXN9xlTuR6a1lwJSon6O4Pt3rcY/RSJK03CsOorw5Mt0R5UhDMCG3DSpxWcqV+dRODjhQ/wC2JeH0Y0i1bHLdJRc59uWoLEaXcHVtoUBgFIyMHHGe+KxmwPsAugfVxXJB6eXwn9rObDD0zqW76jnQpKhBj3J1CAo4Ckk53J/tKivH0Aqual045qvq5pW2W5RkWxhptsLGcsstuFbgUfpu4+4FXvq9pS0aQskCbZPXiockfCLZL63EqSUKUCNxJyCj/k18v2ePWmalu0tnKoTUUMuL8eopYUkfcBKj9MjPcVlsifHW4LZHW3IrXfMyWh8cuuW6A21GXmt8Sc1/aAAdqV0a5NKUpREpSlESlKURKUpREpSlEXivdvj3a0TbdNZS/FlsLYdaUcBaVJIIz47964n0fc77pEbbdOmWyayotSGknjengpWhWUkg+4ruU1VtRaA0xqOaqZd7Qw9LUAC+kqbWoDtkpIz7c1VqoHygYDYhXqGqZTk+I24KxbRvXW4Qbu3D1kEzYj/ytyY7SUOoXjgFIOFA9vHNaxprqLG1HBXPtVg1E5ASso+IVGbCVEHCtqd+5WCCDgfSucmLVEOuhZpjZjtG6iE6nccobL4QQCf7fP1zWh9DLo3bOqertNQnt1ncdkPRUbipKVtO7cp58oIyfO0GqdLPI44Xnp5rRrqOFrfEYBpfpZbnZL1b73FVItklD7aVFCxgpU2od0rScFJHsQDUjVP1LFbtepbPfon8J5+Qi3TEp7SG3PlQVDypC9pB9twq3pzgZ71qtDgPqWG8NyLNCqlouVFg6OXNkPtRraHpMhta1BKGo5eWUcngAJxx4rmvV2o4uttcvXXUcW/KsbKVIhsQFNtutNjgK2rIOVcqUfGcc960LqG3OIk6GiyFLgWy2PXE+ig7y1na1v8ActlWcD82ATyKqAYujDFp6gaXbEppCG476Y6PUUwttpLS0vtdwlYGSAeM9+QTU4hI5z/pzbfufTVbFFC0EvJzOn3Hr8zXliWxi0QJOp+leopahAAXLhPp9OSwgnGXEfldR7nGPbtxr3R6TcL1Gtt2ZiSGIK2nDJlPP7jMXkpxjJJIUCdxxjGPNYom9Ku+v7c/o6yC13G477fIirWVxHi6lSVkYAOwJO8pxwUjFdU6G03H0jpO22OG6t5qG1s9VYwXFElSlY8ZUSceKqxUrZntkJP0m4scrqOvkLQGnU9r25G3I6KdpSlayyFWJ7Vyscx2Xa4RudveWXXYiFJS8ys/mW0VEBQPcoJHPIPOKrWo9WRmbvZb1FZmpMVTkabElR1RnPRdA+YFzalRStCPOMFRB99MpipGyC/1Bew4bhZ5b7zcL3qSdKsMOAuRHiNx1rdm72m9ylLGS2lW5WMZSMYyOeastmsbzEv8QvM1dxue0hK9nptMA90tN5O3PkklR98cVKQLfDt6HEQIkeKhxZdWlhtKAtZ7qOByeBz9K9VfXyYtBZC6+i/gH0r+0pUS8KtdSp8616Bv821bvjWITjjRSMlJA/MPsMn9K5xsWi9G2PTEC/dT7pKVLurYkx7bHcIWptXKVL2/Mong5yAM45INdYPNodZW26lK21pKVJUMgg9wa4r1B06uFnu2p0t322QrRZZnw7YuMpQeDS0hbQSAklXyqA4xkg+xqrUN/wDoLQo3Agxm47ar+y7hp2z6xtdy6bwplv8ARcQ+tFzfwCpG75BlSikKSSOTySOBXU92uiruzpuPbpLkaLeFFbj7KsLDaWivYlX8qicDI5ACsYPbljTszQsDR18N2gy7xfHU7YrrzPpxkEH5S2c7sg/MSoDIAHHOdU6aTZjemtH6fkuBd8iXNic00g7imE8hxZUo+MIU4CPBKR5FRU02B93Z3++yt1MAe0YRbD9ufqfutii6O0/GeS8i0RFvAhQceR6qwe+dy8nP171Oubtp2jKvAzjNfuqN1qvjlh6fXB9lOXZBRESrJAT6isEnH0zxV6WQhpe43sseKMyyBo1JVU1B1J1K4pcewWm1NLSSn4h6cXkLx/RhA855Pes/HVnW6ZEhiXNajSUDcEhhpaFpzjKVY8HAIPIyK9WhpsvWVx/C4dukMyWUpW5LCN0ZtB7KKvfg4R3OPbJE3rvQ9gtN80pam5E+Zdrxcktuhx8JPw+CXlhKANvA4I+vtXPxurKi73Gw7kfbVdS9vD6YiMC53yB9b6KvaDg3fq1qt6Nq+5zp1itgTIdaBS2hTpOEt/KBjI3E45xxkZrpS02uDZ4TcO1Q48OI3+VlhsISPc4Hn6159O2C16ctyYNjhNQ4oUVlKO6lHupRPKj25JJ4HtUpW1BEY22cbnmucqqjxn3aLN2CUpSp1WSlKURKUpREpSlESlKURKUpREpSlEWda76Rae1hdjc5L1xgTlgJddgupT6wAwNyVJUM4wMgA8VXdL9Lz071Mq62CNcNQyHGFNJXMmMx246SRkABGVLOO+AMGtnpjNeRGwOxYRdS+PJhwkm3JU5mNfb/AHe2u3m3RrXbre6ZPpIl/EOSHdpSjOEgJSncpXckkJ7Yq4imBSpHOuoybrEOsYuOkddWfXNvhqlwUsGFPbAyFNk/lV7AjGD23Dnvg5vPvOnrBOfv/TjXn4KuSCt20TIrilZ8NhO1SVnJIGe39WK61WhDiFIcSFIUCFJUMgj2IqAiaK0xEuXx8awWtqZkEOojIBSR5HHB+oqm6A4rg/rsr0dYAzC4Z6dCNrg8lTOjWi5kSM3qfVT7sm+zt8pthxAQiD62FObUdkuL4Kjx4SOxzqlKVYY0MFgqckhkcXFKUpXpeEpSlESlKURKUpREPaueuvugGb71BsMli4xbG9cYrsdUx7PpvPNAKbbXyACUlWFcnCSMHAroWoDW+kbPrSxLtOoIvrxSsOoKVFC2nBnC0KHIUMn9CQeCa8vbiFlJG/w3YgsFt/S/Sug0fjfUbUjF4dYTvatrYAQ4rnaNpJU5zjj5U++RVo/ZsYduTuqdUmDHgQbk+3HhR47YQ2221vKggAY25WBkdyk197d+zhomJcEyZL16ntJORGkyk+mfoShKVEfTNbFb4ca3wmIkGO1GisICG2WkhKEJHYADsKhZDZ1zsrEtQHMLQSSdyvRUHrXTkbVemZtnmLU2iSkbXUgEtrBBSoA+xANTlKnIBFiqjXFpDhqFgmj9I9Sun6LvGs0awXViWAptTkpbexwDAVtI7Y7p+g5957pb0+v7GqJOsuok9qbqFaCzFZZVubiIP5iDgDJHAAGAM8knjXaVGyFrLAbKaSpfISTqdUpSlSqBKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURKUpREpSlESlKURf/2Q=='},\n",
       "   'literature': {'PUB00003582': {'PMID': 9211332,\n",
       "     'ISBN': None,\n",
       "     'volume': '280',\n",
       "     'issue': None,\n",
       "     'year': 1997,\n",
       "     'title': 'Biosynthesis of riboflavin: 3,4-dihydroxy-2-butanone-4-phosphate synthase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '374-82',\n",
       "     'medline_journal': 'Methods Enzymol',\n",
       "     'ISO_journal': 'Meth. Enzymol.',\n",
       "     'authors': ['Richter G',\n",
       "      'Krieger C',\n",
       "      'Volk R',\n",
       "      'Kis K',\n",
       "      'Ritz H',\n",
       "      'Gotze E',\n",
       "      'Bacher A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0076-6879(97)80128-0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 48516,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48449,\n",
       "    'proteomes': 8863,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41967, 'rosettafold': 0},\n",
       "    'structures': 39,\n",
       "    'taxa': 36604},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44790,\n",
       "    'alignment:full': 9741,\n",
       "    'alignment:seed': 659},\n",
       "   'cross_references': {}}},\n",
       " 'PF09723': {'metadata': {'accession': 'PF09723',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013429',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Zinc ribbon domain', 'short': 'Zn-ribbon_8'},\n",
       "   'description': ['<p>This entry represents a region of about 41 amino acids found in a number of small proteins in a wide range of bacteria. The region usually begins with the initiator Met and contains two CxxC motifs separated by 17 amino acids. One protein in this entry has been noted as a putative regulatory protein, designated FmdB. Most proteins in this entry have a C-terminal region containing highly degenerate sequence.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0167', 'name': 'Zn_Beta_Ribbon'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 31912,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31912,\n",
       "    'proteomes': 3548,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26129, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15931},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27309,\n",
       "    'alignment:full': 4681,\n",
       "    'alignment:seed': 507},\n",
       "   'cross_references': {}}},\n",
       " 'PF13565': {'metadata': {'accession': 'PF13565',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Homeodomain-like domain', 'short': 'HTH_32'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 16408,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16405,\n",
       "    'proteomes': 2015,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14142, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 10920},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14701,\n",
       "    'alignment:full': 4419,\n",
       "    'alignment:seed': 306},\n",
       "   'cross_references': {}}},\n",
       " 'PF13517': {'metadata': {'accession': 'PF13517',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013517',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FG-GAP-like repeat', 'short': 'FG-GAP_3'},\n",
       "   'description': ['<p>This entry represents a repeat found in alpha integrins and related proteins in which form a 7-fold repeat that adopts a beta-propeller fold. This repeat contains a putative calcium-binding site. These repeats are found in multiple proteins from eukaryotes and bacteria and mediate diverse biological processes at both molecular and cellular levels, such as cell-cell interactions, host-pathogen recognition or innate immune responses.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 7699,\n",
       "    'interactions': 0,\n",
       "    'matches': 217655,\n",
       "    'pathways': 0,\n",
       "    'proteins': 90689,\n",
       "    'proteomes': 4673,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 62108, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 19750},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 204353,\n",
       "    'alignment:full': 45403,\n",
       "    'alignment:seed': 681},\n",
       "   'cross_references': {}}},\n",
       " 'PF09383': {'metadata': {'accession': 'PF09383',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018449',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NIL domain', 'short': 'NIL'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of ABC transporter proteins involved in D-methionine transport as well as a number of ferredoxin-like proteins. This domain is likely to act as a substrate binding domain. The domain has been named after a conserved sequence in some members of the family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0070', 'name': 'ACT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 38,\n",
       "    'interactions': 0,\n",
       "    'matches': 29490,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29399,\n",
       "    'proteomes': 4196,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26642, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 19973},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29725,\n",
       "    'alignment:full': 5025,\n",
       "    'alignment:seed': 392},\n",
       "   'cross_references': {}}},\n",
       " 'PF01220': {'metadata': {'accession': 'PF01220',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001874',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dehydroquinase class II', 'short': 'DHquinase_II'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 63,\n",
       "    'interactions': 0,\n",
       "    'matches': 38057,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38041,\n",
       "    'proteomes': 7565,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32764, 'rosettafold': 0},\n",
       "    'structures': 75,\n",
       "    'taxa': 30099},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34941,\n",
       "    'alignment:full': 7640,\n",
       "    'alignment:seed': 609},\n",
       "   'cross_references': {}}},\n",
       " 'PF07479': {'metadata': {'accession': 'PF07479',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006109',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NAD-dependent glycerol-3-phosphate dehydrogenase C-terminus',\n",
       "    'short': 'NAD_Gly3P_dh_C'},\n",
       "   'description': ['<p>NAD-dependent glycerol-3-phosphate dehydrogenase (GPDH) catalyses the interconversion of dihydroxyacetone phosphate and L-glycerol-3-phosphate. This family represents the C-terminal substrate-binding domain [PMID:10801498].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014289': {'PMID': 10801498,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '5',\n",
       "     'year': 2000,\n",
       "     'title': 'A potential target enzyme for trypanocidal drugs revealed by the crystal structure of NAD-dependent glycerol-3-phosphate dehydrogenase from Leishmania mexicana.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '541-52',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Suresh S',\n",
       "      'Turley S',\n",
       "      'Opperdoes FR',\n",
       "      'Michels PA',\n",
       "      'Hol WG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(00)00135-0'},\n",
       "    'PUB00019909': {'PMID': 12032156,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '31',\n",
       "     'year': 2002,\n",
       "     'title': 'Kinetic regulation of the mitochondrial glycerol-3-phosphate dehydrogenase by the external NADH dehydrogenase in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '27991-5',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Pahlman IL',\n",
       "      'Larsson C',\n",
       "      'Averet N',\n",
       "      'Bunoust O',\n",
       "      'Boubekeur S',\n",
       "      'Gustafsson L',\n",
       "      'Rigoulet M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M204079200'}},\n",
       "   'set_info': {'accession': 'CL0106', 'name': '6PGD_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 170,\n",
       "    'interactions': 0,\n",
       "    'matches': 52295,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52211,\n",
       "    'proteomes': 10434,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 44520, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 42418},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47301,\n",
       "    'alignment:full': 12024,\n",
       "    'alignment:seed': 756},\n",
       "   'cross_references': {}}},\n",
       " 'PF02820': {'metadata': {'accession': 'PF02820',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004092',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'mbt repeat', 'short': 'MBT'},\n",
       "   'description': ['<p>The function of this repeat is unknown, but is found in a number of nuclear proteins such as drosophila sex comb on midleg protein Swiss:Q9VHA0. The repeat is found in up to four copies as in Swiss:Q9UHJ3. The repeat contains a completely conserved glutamate at its amino terminus that may be important for function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019839': {'PMID': 9755199,\n",
       "     'ISBN': None,\n",
       "     'volume': '150',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Expression and properties of wild-type and mutant forms of the Drosophila sex comb on midleg (SCM) repressor protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '675-86',\n",
       "     'medline_journal': 'Genetics',\n",
       "     'ISO_journal': 'Genetics',\n",
       "     'authors': ['Bornemann D', 'Miller E', 'Simon J.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9755199&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0049', 'name': 'Tudor'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 239,\n",
       "    'interactions': 0,\n",
       "    'matches': 40547,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14598,\n",
       "    'proteomes': 935,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10261, 'rosettafold': 0},\n",
       "    'structures': 52,\n",
       "    'taxa': 3891},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28332,\n",
       "    'alignment:full': 17081,\n",
       "    'alignment:seed': 234},\n",
       "   'cross_references': {}}},\n",
       " 'PF13234': {'metadata': {'accession': 'PF13234',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025696',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'rRNA-processing arch domain', 'short': 'rRNA_proc-arch'},\n",
       "   'description': ['<p>Mtr4 is the essential RNA helicase, and is an exosome-activating cofactor. This arch domain is carried in Mtr4 and Ski2 (the cytosolic homologue of Mtr4). The arch domain is required for proper 5.8S rRNA processing, and appears to function independently of canonical helicase activity [PMID:20512111].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 143,\n",
       "    'interactions': 0,\n",
       "    'matches': 10432,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10265,\n",
       "    'proteomes': 2938,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7549, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 10409},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8904,\n",
       "    'alignment:full': 4798,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF00019': {'metadata': {'accession': 'PF00019',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001839',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transforming growth factor beta like domain',\n",
       "    'short': 'TGF_beta'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005153': {'PMID': 1631557,\n",
       "     'ISBN': None,\n",
       "     'volume': '257',\n",
       "     'issue': '5068',\n",
       "     'year': 1992,\n",
       "     'title': 'Crystal structure of transforming growth factor-beta 2: an unusual fold for the superfamily.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '369-73',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Daopin S', 'Piez KA', 'Ogawa Y', 'Davies DR.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/257/5068/369'},\n",
       "    'PUB00020430': {'PMID': 8570652,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'Three-dimensional structure of recombinant human osteogenic protein 1: structural paradigm for the transforming growth factor beta superfamily.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '878-83',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Griffith DL',\n",
       "      'Keck PC',\n",
       "      'Sampath TK',\n",
       "      'Rueger DC',\n",
       "      'Carlson WD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.2.878'},\n",
       "    'PUB00020431': {'PMID': 9187648,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'X-ray structure of glial cell-derived neurotrophic factor at 1.9 A resolution and implications for receptor binding.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '435-8',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Eigenbrot C', 'Gerber N.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0697-435'}},\n",
       "   'set_info': {'accession': 'CL0079', 'name': 'Cystine-knot'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 168,\n",
       "    'interactions': 0,\n",
       "    'matches': 33522,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33458,\n",
       "    'proteomes': 920,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26294, 'rosettafold': 0},\n",
       "    'structures': 136,\n",
       "    'taxa': 6370},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26057,\n",
       "    'alignment:full': 12309,\n",
       "    'alignment:seed': 263},\n",
       "   'cross_references': {}}},\n",
       " 'PF14791': {'metadata': {'accession': 'PF14791',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029398',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DNA polymerase beta thumb', 'short': 'DNA_pol_B_thumb'},\n",
       "   'description': ['<p>The catalytic region of DNA polymerase beta is split into three domains. An N-terminal fingers domain, a central palm domain and a C-terminal thumb domain. This entry represents the thumb domain [PMID:7516581].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 162,\n",
       "    'interactions': 0,\n",
       "    'matches': 16912,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16906,\n",
       "    'proteomes': 3201,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13689, 'rosettafold': 0},\n",
       "    'structures': 703,\n",
       "    'taxa': 14250},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14180,\n",
       "    'alignment:full': 4527,\n",
       "    'alignment:seed': 341},\n",
       "   'cross_references': {}}},\n",
       " 'PF02650': {'metadata': {'accession': 'PF02650',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023054',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'WhiA C-terminal HTH domain', 'short': 'HTH_WhiA'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of the sporulation regulator WhiA.  It is predicted to form a DNA-binding helix-turn-helix structure [PMID:17603302].  The WhiA protein also contains two N-terminal domains that are distant homologues of LAGLIDADG homing endonucleases [PMID:17603302].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044539': {'PMID': 17603302,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '13',\n",
       "     'year': 2007,\n",
       "     'title': 'Bacterial DUF199/COG1481 proteins including sporulation regulator WhiA are distant homologs of LAGLIDADG homing endonucleases that retained only DNA binding.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1666-70',\n",
       "     'medline_journal': 'Cell Cycle',\n",
       "     'ISO_journal': 'Cell Cycle',\n",
       "     'authors': ['Knizewski L', 'Ginalski K.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00044736': {'PMID': 10986251,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '19',\n",
       "     'year': 2000,\n",
       "     'title': 'WhiA, a protein of unknown function conserved among gram-positive bacteria, is essential for sporulation in Streptomyces coelicolor A3(2).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5470-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Ainsa JA',\n",
       "      'Ryding NJ',\n",
       "      'Hartley N',\n",
       "      'Findlay KC',\n",
       "      'Bruton CJ',\n",
       "      'Chater KF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.19.5470-5478.2000'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 13681,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13681,\n",
       "    'proteomes': 3645,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12127, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 12738},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12921,\n",
       "    'alignment:full': 3347,\n",
       "    'alignment:seed': 269},\n",
       "   'cross_references': {}}},\n",
       " 'PF13382': {'metadata': {'accession': 'PF13382',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026912',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Adenine deaminase C-terminal domain',\n",
       "    'short': 'Adenine_deam_C'},\n",
       "   'description': ['<p>This family represents a C-terminal region of the adenine deaminase enzyme.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 27,\n",
       "    'interactions': 0,\n",
       "    'matches': 15911,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15909,\n",
       "    'proteomes': 2699,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13860, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 11828},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15118,\n",
       "    'alignment:full': 3258,\n",
       "    'alignment:seed': 255},\n",
       "   'cross_references': {}}},\n",
       " 'PF07503': {'metadata': {'accession': 'PF07503',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011125',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HypF finger', 'short': 'zf-HYPF'},\n",
       "   'description': ['<p>The HypF family of proteins are involved in the maturation and  regulation of hydrogenase ([PMID:9492269]). In the N-terminus they appear to have two Zinc finger domains, as modelled by this family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014298': {'PMID': 9492269,\n",
       "     'ISBN': None,\n",
       "     'volume': '251',\n",
       "     'issue': '1-2',\n",
       "     'year': 1998,\n",
       "     'title': 'Rhodobacter capsulatus HypF is involved in regulation of hydrogenase synthesis through the HupUV proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '65-71',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Colbeau A',\n",
       "      'Elsen S',\n",
       "      'Tomiyama M',\n",
       "      'Zorin NA',\n",
       "      'Dimon B',\n",
       "      'Vignais PM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1432-1327.1998.2510065.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 40,\n",
       "    'interactions': 0,\n",
       "    'matches': 26898,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13646,\n",
       "    'proteomes': 2098,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11727, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 11213},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24802,\n",
       "    'alignment:full': 3976,\n",
       "    'alignment:seed': 484},\n",
       "   'cross_references': {}}},\n",
       " 'PF00687': {'metadata': {'accession': 'PF00687',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR028364',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L1p/L10e family',\n",
       "    'short': 'Ribosomal_L1'},\n",
       "   'description': ['<p>This family includes prokaryotic L1 and eukaryotic L10.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 252,\n",
       "    'interactions': 0,\n",
       "    'matches': 57763,\n",
       "    'pathways': 0,\n",
       "    'proteins': 57646,\n",
       "    'proteomes': 11111,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 49289, 'rosettafold': 0},\n",
       "    'structures': 371,\n",
       "    'taxa': 47563},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50944,\n",
       "    'alignment:full': 14564,\n",
       "    'alignment:seed': 541},\n",
       "   'cross_references': {}}},\n",
       " 'PF08522': {'metadata': {'accession': 'PF08522',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013728',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1735)',\n",
       "    'short': 'DUF1735'},\n",
       "   'description': ['<p>This domain of unknown function is found in a number of bacterial proteins including acylhydrolases. The structure of this domain has a beta-sandwich fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0594', 'name': 'DUF1735'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 49,\n",
       "    'interactions': 0,\n",
       "    'matches': 5963,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5363,\n",
       "    'proteomes': 366,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4943, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 1439},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 5696,\n",
       "    'alignment:full': 1086,\n",
       "    'alignment:seed': 173},\n",
       "   'cross_references': {}}},\n",
       " 'PF04810': {'metadata': {'accession': 'PF04810',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006895',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sec23/Sec24 zinc finger', 'short': 'zf-Sec23_Sec24'},\n",
       "   'description': ['<p>COPII-coated vesicles carry proteins from the endoplasmic reticulum to the Golgi complex. This vesicular transport can be reconstituted by using three cytosolic components containing five proteins: the small GTPase Sar1p, the Sec23p/24p complex, and the Sec13p/Sec31p complex. This domain is found to be zinc binding domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014931': {'PMID': 11535824,\n",
       "     'ISBN': None,\n",
       "     'volume': '98',\n",
       "     'issue': '19',\n",
       "     'year': 2001,\n",
       "     'title': 'Structure of the Sec23p/24p and Sec13p/31p complexes of COPII.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10704-9',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Lederkremer GZ',\n",
       "      'Cheng Y',\n",
       "      'Petre BM',\n",
       "      'Vogan E',\n",
       "      'Springer S',\n",
       "      'Schekman R',\n",
       "      'Walz T',\n",
       "      'Kirchhausen T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.191359398'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 245,\n",
       "    'interactions': 0,\n",
       "    'matches': 18207,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18197,\n",
       "    'proteomes': 2187,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13850, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 8023},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14176,\n",
       "    'alignment:full': 8970,\n",
       "    'alignment:seed': 300},\n",
       "   'cross_references': {}}},\n",
       " 'PF16653': {'metadata': {'accession': 'PF16653',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032095',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Saccharopine dehydrogenase C-terminal domain',\n",
       "    'short': 'Sacchrp_dh_C'},\n",
       "   'description': ['<p>This family comprises the C-terminal domain of saccharopine dehydrogenase. In some organisms this enzyme is found as a bifunctional polypeptide with lysine ketoglutarate reductase. The saccharopine dehydrogenase can also function as a saccharopine reductase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0139', 'name': 'GADPH_aa-bio_dh'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 141,\n",
       "    'interactions': 0,\n",
       "    'matches': 31326,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31293,\n",
       "    'proteomes': 5790,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26166, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 23936},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27569,\n",
       "    'alignment:full': 6979,\n",
       "    'alignment:seed': 333},\n",
       "   'cross_references': {}}},\n",
       " 'PF10518': {'metadata': {'accession': 'PF10518',\n",
       "   'entry_id': None,\n",
       "   'type': 'conserved_site',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019546',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TAT (twin-arginine translocation) pathway signal sequence',\n",
       "    'short': 'TAT_signal'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007954': {'PMID': 8939424,\n",
       "     'ISBN': None,\n",
       "     'volume': '22',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'A common export pathway for proteins binding complex redox cofactors?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '393-404',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Berks BC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1996.00114.x'}},\n",
       "   'set_info': {'accession': 'CL0300', 'name': 'TAT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 672,\n",
       "    'interactions': 0,\n",
       "    'matches': 32155,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32155,\n",
       "    'proteomes': 2968,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27310, 'rosettafold': 1},\n",
       "    'structures': 2,\n",
       "    'taxa': 15170},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28871,\n",
       "    'alignment:full': 5203,\n",
       "    'alignment:seed': 403},\n",
       "   'cross_references': {}}},\n",
       " 'PF07538': {'metadata': {'accession': 'PF07538',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006637',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Clostridial hydrophobic W', 'short': 'ChW'},\n",
       "   'description': ['<p>A novel extracellular macromolecular system has been proposed based on the proteins containing ChW repeats [PMID:11466286]. ChW stands for Clostridial hydrophobic with conserved W (tryptophan). This repeat was originally described in Clostridium acetobutylicum but is also found in other Gram-positive bacteria including Enterococcus faecalis, Streptococcus agalactiae and Streptomyces coelicolor.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014722': {'PMID': 11466286,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '16',\n",
       "     'year': 2001,\n",
       "     'title': 'Genome sequence and comparative analysis of the solvent-producing bacterium Clostridium acetobutylicum.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4823-38',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Nolling J',\n",
       "      'Breton G',\n",
       "      'Omelchenko MV',\n",
       "      'Makarova KS',\n",
       "      'Zeng Q',\n",
       "      'Gibson R',\n",
       "      'Lee HM',\n",
       "      'Dubois J',\n",
       "      'Qiu D',\n",
       "      'Hitti J',\n",
       "      'Wolf YI',\n",
       "      'Tatusov RL',\n",
       "      'Sabathe F',\n",
       "      'Doucette-Stamm L',\n",
       "      'Soucaille P',\n",
       "      'Daly MJ',\n",
       "      'Bennett GN',\n",
       "      'Koonin EV',\n",
       "      'Smith DR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.16.4823-4838.2001'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 427,\n",
       "    'interactions': 0,\n",
       "    'matches': 10410,\n",
       "    'pathways': 0,\n",
       "    'proteins': 2434,\n",
       "    'proteomes': 268,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 2058, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1541},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10281,\n",
       "    'alignment:full': 2238,\n",
       "    'alignment:seed': 395},\n",
       "   'cross_references': {}}},\n",
       " 'PF17755': {'metadata': {'accession': 'PF17755',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041552',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UvrA DNA-binding domain', 'short': 'UvrA_DNA-bind'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 107,\n",
       "    'interactions': 0,\n",
       "    'matches': 64242,\n",
       "    'pathways': 0,\n",
       "    'proteins': 63813,\n",
       "    'proteomes': 8677,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 52201, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 36832},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 59182,\n",
       "    'alignment:full': 11365,\n",
       "    'alignment:seed': 381},\n",
       "   'cross_references': {}}},\n",
       " 'PF01724': {'metadata': {'accession': 'PF01724',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002636',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function DUF29', 'short': 'DUF29'},\n",
       "   'description': ['<p>This family consists of various hypothetical proteins from cyanobacteria, none of which are functionally described. The aligned region is approximately 120-140 amino acids long corresponding to almost the entire length of the proteins in the family. Swiss:Q2RPE2, PDB:3fcn, is a small protein that has a novel all-alpha fold. The N-terminal helical hairpin is likely to function as a dimerisation module. This protein is a member of PFam family PF01724. The function of this protein is unknown. One protein sequence contains a fusion of this protein and a DnaB domain, suggesting a possible role in DNA helicase activity (hypothetical). Dali hits have low Z and high rmsd, suggesting probably only topological similarities (not functional relevance) (details derived from TOPSAN). The family has several highly conserved sequence motifs, including YD/ExD, DxxNVxEEIE, and CPY/F/W, as well as conserved tryptophans.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 10117,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10109,\n",
       "    'proteomes': 391,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8607, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 2723},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8561,\n",
       "    'alignment:full': 1903,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF00194': {'metadata': {'accession': 'PF00194',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001148',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Eukaryotic-type carbonic anhydrase',\n",
       "    'short': 'Carb_anhydrase'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 290,\n",
       "    'interactions': 0,\n",
       "    'matches': 38281,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36409,\n",
       "    'proteomes': 2729,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 27303, 'rosettafold': 0},\n",
       "    'structures': 1234,\n",
       "    'taxa': 12913},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29574,\n",
       "    'alignment:full': 14309,\n",
       "    'alignment:seed': 302},\n",
       "   'cross_references': {}}},\n",
       " 'PF00829': {'metadata': {'accession': 'PF00829',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR028909',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal prokaryotic L21 protein',\n",
       "    'short': 'Ribosomal_L21p'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 73,\n",
       "    'interactions': 0,\n",
       "    'matches': 46267,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46229,\n",
       "    'proteomes': 10567,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39600, 'rosettafold': 0},\n",
       "    'structures': 812,\n",
       "    'taxa': 45620},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41293,\n",
       "    'alignment:full': 9807,\n",
       "    'alignment:seed': 601},\n",
       "   'cross_references': {}}},\n",
       " 'PF08338': {'metadata': {'accession': 'PF08338',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013549',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1731)',\n",
       "    'short': 'DUF1731'},\n",
       "   'description': ['<p>This domain of unknown function appears towards the C-terminus of proteins of the NAD dependent epimerase/dehydratase family (Pfam:PF01370) in bacteria, eukaryotes and archaea.  Many of the proteins in which it is found are involved in cell-division inhibition.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 72,\n",
       "    'interactions': 0,\n",
       "    'matches': 28327,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28322,\n",
       "    'proteomes': 5388,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 24332, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 23308},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26479,\n",
       "    'alignment:full': 5717,\n",
       "    'alignment:seed': 473},\n",
       "   'cross_references': {}}},\n",
       " 'PF04815': {'metadata': {'accession': 'PF04815',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006900',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sec23/Sec24 helical domain', 'short': 'Sec23_helical'},\n",
       "   'description': ['<p>COPII-coated vesicles carry proteins from the endoplasmic reticulum to the Golgi complex. This vesicular transport can be reconstituted by using three cytosolic components containing five proteins: the small GTPase Sar1p, the Sec23p/24p complex, and the Sec13p/Sec31p complex. This domain is composed of five alpha helices.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014931': {'PMID': 11535824,\n",
       "     'ISBN': None,\n",
       "     'volume': '98',\n",
       "     'issue': '19',\n",
       "     'year': 2001,\n",
       "     'title': 'Structure of the Sec23p/24p and Sec13p/31p complexes of COPII.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10704-9',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Lederkremer GZ',\n",
       "      'Cheng Y',\n",
       "      'Petre BM',\n",
       "      'Vogan E',\n",
       "      'Springer S',\n",
       "      'Schekman R',\n",
       "      'Walz T',\n",
       "      'Kirchhausen T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.191359398'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 255,\n",
       "    'interactions': 0,\n",
       "    'matches': 18756,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18713,\n",
       "    'proteomes': 2184,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14298, 'rosettafold': 0},\n",
       "    'structures': 34,\n",
       "    'taxa': 7987},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14642,\n",
       "    'alignment:full': 9094,\n",
       "    'alignment:seed': 332},\n",
       "   'cross_references': {}}},\n",
       " 'PF04166': {'metadata': {'accession': 'PF04166',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005255',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pyridoxal phosphate biosynthetic protein PdxA',\n",
       "    'short': 'PdxA'},\n",
       "   'description': [\"<p>In Escherichia coli the coenzyme pyridoxal 5'-phosphate is synthesised  de novo by a pathway that is thought to involve the condensation of 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose, catalysed by  the enzymes PdxA and PdxJ, to form either pyridoxine (vitamin B6) or pyridoxine 5'-phosphate [PMID:10225425].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009884': {'PMID': 10225425,\n",
       "     'ISBN': None,\n",
       "     'volume': '449',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': \"Vitamin B6 biosynthesis: formation of pyridoxine 5'-phosphate from 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose-5-phosphate by PdxA and PdxJ protein.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '45-8',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Laber B',\n",
       "      'Maurer W',\n",
       "      'Scharf S',\n",
       "      'Stepusin K',\n",
       "      'Schmidt FS.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(99)00393-2'}},\n",
       "   'set_info': {'accession': 'CL0270', 'name': 'Iso_DH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 34913,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34872,\n",
       "    'proteomes': 5263,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 29964, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 23533},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32351,\n",
       "    'alignment:full': 5924,\n",
       "    'alignment:seed': 390},\n",
       "   'cross_references': {}}},\n",
       " 'PF04316': {'metadata': {'accession': 'PF04316',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031316',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Anti-sigma-28 factor, FlgM', 'short': 'FlgM'},\n",
       "   'description': ['<p>FlgM binds and inhibits the activity of the transcription factor sigma 28. Inhibition of sigma 28 prevents the expression of genes from flagellar transcriptional class 3, which include genes for the filament and chemotaxis.  Correctly assembled basal body-hook structures export FlgM, relieving inhibition of sigma 28 and allowing expression of class 3 genes. NMR studies show that free FlgM is mostly unfolded, which may facilitate its export. The C terminal half of FlgM adopts a tertiary structure when it binds to sigma 28.  All mutations in FlgM that prevent sigma 28 inhibition affect the C-terminal domain and is the region thought to constitute the binding domain. A minimal binding domain has been identified between Glu 64 and Arg 88 in Salmonella typhimurium (Swiss:P26477).  The N-terminal portion remains unstructured and may be necessary for recognition by the export machinery [PMID:9095196].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010593': {'PMID': 9095196,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '4',\n",
       "     'year': 1997,\n",
       "     'title': 'The C-terminal half of the anti-sigma factor, FlgM, becomes structured when bound to its target, sigma 28.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '285-91',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Daughdrill GW',\n",
       "      'Chadsey MS',\n",
       "      'Karlinsey JE',\n",
       "      'Hughes KT',\n",
       "      'Dahlquist FW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0497-285'},\n",
       "    'PUB00011947': {'PMID': 11985711,\n",
       "     'ISBN': None,\n",
       "     'volume': '43',\n",
       "     'issue': '2',\n",
       "     'year': 2002,\n",
       "     'title': 'Functional characterization of the antagonistic flagellar late regulators FliA and FlgM of Helicobacter pylori and their effects on the H. pylori transcriptome.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '307-22',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Josenhans C',\n",
       "      'Niehus E',\n",
       "      'Amersbach S',\n",
       "      'Horster A',\n",
       "      'Betz C',\n",
       "      'Drescher B',\n",
       "      'Hughes KT',\n",
       "      'Suerbaum S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.2002.02765.x'},\n",
       "    'PUB00019560': {'PMID': 15068809,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '1',\n",
       "     'year': 2004,\n",
       "     'title': 'Crystal structure of the flagellar sigma/anti-sigma complex sigma(28)/FlgM reveals an intact sigma factor in an inactive conformation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '127-38',\n",
       "     'medline_journal': 'Mol Cell',\n",
       "     'ISO_journal': 'Mol. Cell',\n",
       "     'authors': ['Sorenson MK', 'Ray SS', 'Darst SA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1097-2765(04)00150-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 12,\n",
       "    'interactions': 0,\n",
       "    'matches': 15009,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15002,\n",
       "    'proteomes': 2878,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13100, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 12819},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14269,\n",
       "    'alignment:full': 2867,\n",
       "    'alignment:seed': 254},\n",
       "   'cross_references': {}}},\n",
       " 'PF00298': {'metadata': {'accession': 'PF00298',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020783',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L11, RNA binding domain',\n",
       "    'short': 'Ribosomal_L11'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003382': {'PMID': 9398519,\n",
       "     'ISBN': None,\n",
       "     'volume': '274',\n",
       "     'issue': '1',\n",
       "     'year': 1997,\n",
       "     'title': 'The RNA binding domain of ribosomal protein L11: three-dimensional structure of the RNA-bound form of the protein and its interaction with 23 S rRNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '101-13',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Hinck AP',\n",
       "      'Markus MA',\n",
       "      'Huang S',\n",
       "      'Grzesiek S',\n",
       "      'Kustonovich I',\n",
       "      'Draper DE',\n",
       "      'Torchia DA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1997.1379'},\n",
       "    'PUB00003950': {'PMID': 8989327,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '1',\n",
       "     'year': 1997,\n",
       "     'title': 'High resolution solution structure of ribosomal protein L11-C76, a helical protein with a flexible loop that becomes structured upon binding to RNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '70-7',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Markus MA',\n",
       "      'Hinck AP',\n",
       "      'Huang S',\n",
       "      'Draper DE',\n",
       "      'Torchia DA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0197-70'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 163,\n",
       "    'interactions': 0,\n",
       "    'matches': 51822,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51815,\n",
       "    'proteomes': 11033,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 44412, 'rosettafold': 0},\n",
       "    'structures': 613,\n",
       "    'taxa': 47704},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45802,\n",
       "    'alignment:full': 12386,\n",
       "    'alignment:seed': 635},\n",
       "   'cross_references': {}}},\n",
       " 'PF03707': {'metadata': {'accession': 'PF03707',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005330',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial signalling protein N terminal repeat',\n",
       "    'short': 'MHYT'},\n",
       "   'description': ['<p>Found as an N terminal triplet tandem repeat in bacterial signalling proteins. Family includes CoxC (Swiss:Q9KX27) and CoxH (Swiss:Q9KX23) from P.carboxydovorans. Each repeat contains two transmembrane helices. Domain is also described as the MHYT domain [PMID:11728710].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016193': {'PMID': 11728710,\n",
       "     'ISBN': None,\n",
       "     'volume': '205',\n",
       "     'issue': '1',\n",
       "     'year': 2001,\n",
       "     'title': 'MHYT, a new integral membrane sensor domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17-23',\n",
       "     'medline_journal': 'FEMS Microbiol Lett',\n",
       "     'ISO_journal': 'FEMS Microbiol. Lett.',\n",
       "     'authors': ['Galperin MY',\n",
       "      'Gaidenko TA',\n",
       "      'Mulkidjanian AY',\n",
       "      'Nakano M',\n",
       "      'Price CW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1574-6968.2001.tb10919.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 554,\n",
       "    'interactions': 0,\n",
       "    'matches': 41361,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16214,\n",
       "    'proteomes': 2594,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14502, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11337},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42091,\n",
       "    'alignment:full': 8194,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF03755': {'metadata': {'accession': 'PF03755',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013527',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YicC-like family, N-terminal region', 'short': 'YicC_N'},\n",
       "   'description': ['<p>Family of bacterial proteins. Although poorly characterised, the members of this protein family have been demonstrated to play a role in stationary phase survival [PMID:1925027]. These proteins are  not essential during stationary phase [PMID:1925027].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020576': {'PMID': 1925027,\n",
       "     'ISBN': None,\n",
       "     'volume': '142',\n",
       "     'issue': '2-3',\n",
       "     'year': 1991,\n",
       "     'title': 'Three genes preceding pyrE on the Escherichia coli chromosome are essential for survival and normal cell morphology in stationary culture and at high temperature.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '283-8',\n",
       "     'medline_journal': 'Res Microbiol',\n",
       "     'ISO_journal': 'Res. Microbiol.',\n",
       "     'authors': ['Poulsen P', 'Jensen KF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0923-2508(91)90042-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 27164,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27156,\n",
       "    'proteomes': 5242,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22977, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21913},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24688,\n",
       "    'alignment:full': 4917,\n",
       "    'alignment:seed': 597},\n",
       "   'cross_references': {}}},\n",
       " 'PF01035': {'metadata': {'accession': 'PF01035',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014048',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '6-O-methylguanine DNA methyltransferase, DNA binding domain',\n",
       "    'short': 'DNA_binding_1'},\n",
       "   'description': ['<p>This is the C-terminal DNA-binding domain of 6-O-methylguanine-DNA methyltransferases [PMID:8156986]. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001246': {'PMID': 8156986,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '7',\n",
       "     'year': 1994,\n",
       "     'title': 'Crystal structure of a suicidal DNA repair protein: the Ada O6-methylguanine-DNA methyltransferase from E. coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1495-501',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Moore MH',\n",
       "      'Gulbis JM',\n",
       "      'Dodson EJ',\n",
       "      'Demple B',\n",
       "      'Moody PC.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8156986&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 247,\n",
       "    'interactions': 0,\n",
       "    'matches': 89679,\n",
       "    'pathways': 0,\n",
       "    'proteins': 89666,\n",
       "    'proteomes': 10033,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 77555, 'rosettafold': 0},\n",
       "    'structures': 47,\n",
       "    'taxa': 41845},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 83279,\n",
       "    'alignment:full': 17395,\n",
       "    'alignment:seed': 1380},\n",
       "   'cross_references': {}}},\n",
       " 'PF04860': {'metadata': {'accession': 'PF04860',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006944',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phage portal protein', 'short': 'Phage_portal'},\n",
       "   'description': ['<p>Bacteriophage portal proteins form a dodecamer and is located at a  five-fold vertex of the viral capsid.  The portal complex forms a channel through which the viral DNA is packaged into the capsid, and exits during  infection.  The portal protein is though to rotate during DNA packaging  [PMID:11839289]. Portal proteins from different phage show little sequence homology, so  this family does not represent all portal proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008675': {'PMID': 11839289,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '3',\n",
       "     'year': 2002,\n",
       "     'title': 'DNA packaging: a new class of molecular motors.',\n",
       "     'URL': None,\n",
       "     'raw_pages': 'R96-8',\n",
       "     'medline_journal': 'Curr Biol',\n",
       "     'ISO_journal': 'Curr. Biol.',\n",
       "     'authors': ['Moore SD', 'Prevelige PE Jr.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0960-9822(02)00670-X'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 72,\n",
       "    'interactions': 0,\n",
       "    'matches': 24328,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24315,\n",
       "    'proteomes': 3647,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18684, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 16979},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25330,\n",
       "    'alignment:full': 4485,\n",
       "    'alignment:seed': 244},\n",
       "   'cross_references': {}}},\n",
       " 'PF01313': {'metadata': {'accession': 'PF01313',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002191',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial export proteins, family 3',\n",
       "    'short': 'Bac_export_3'},\n",
       "   'description': ['<p>This family includes the following members;     FliQ, MopD, HrcS, Hrp, YopS and SpaQ All of these members export proteins, that do not possess signal  peptides, through the membrane.  Although the proteins that these exporters move may be different, the exporters are thought to function in similar ways [PMID:7814323].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002272': {'PMID': 7814323,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '2',\n",
       "     'year': 1995,\n",
       "     'title': 'Caulobacter FliQ and FliR membrane proteins, required for flagellar biogenesis and cell division, belong to a family of virulence factor export proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '343-56',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Zhuang WY', 'Shapiro L.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=7814323'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 13,\n",
       "    'interactions': 0,\n",
       "    'matches': 23771,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23771,\n",
       "    'proteomes': 4250,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21025, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 18238},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22695,\n",
       "    'alignment:full': 4677,\n",
       "    'alignment:seed': 410},\n",
       "   'cross_references': {}}},\n",
       " 'PF01088': {'metadata': {'accession': 'PF01088',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001578',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ubiquitin carboxyl-terminal hydrolase, family 1',\n",
       "    'short': 'Peptidase_C12'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001305': {'PMID': 9233788,\n",
       "     'ISBN': None,\n",
       "     'volume': '16',\n",
       "     'issue': '13',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure of a deubiquitinating enzyme (human UCH-L3) at 1.8 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3787-96',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Johnston SC',\n",
       "      'Larsen CN',\n",
       "      'Cook WJ',\n",
       "      'Wilkinson KD',\n",
       "      'Hill CP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/16.13.3787'}},\n",
       "   'set_info': {'accession': 'CL0125', 'name': 'Peptidase_CA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 169,\n",
       "    'interactions': 0,\n",
       "    'matches': 13300,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13069,\n",
       "    'proteomes': 2121,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10707, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 9147},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10937,\n",
       "    'alignment:full': 6366,\n",
       "    'alignment:seed': 349},\n",
       "   'cross_references': {}}},\n",
       " 'PF03478': {'metadata': {'accession': 'PF03478',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005174',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF295)', 'short': 'DUF295'},\n",
       "   'description': ['<p>This domain is found in plant proteins of unknown function. It can be found in association with F-box domain Pfam:PF00646.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 277,\n",
       "    'interactions': 0,\n",
       "    'matches': 12327,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11981,\n",
       "    'proteomes': 193,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10165, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 660},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10298,\n",
       "    'alignment:full': 7512,\n",
       "    'alignment:seed': 254},\n",
       "   'cross_references': {}}},\n",
       " 'PF04172': {'metadata': {'accession': 'PF04172',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007300',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LrgB-like family', 'short': 'LrgB'},\n",
       "   'description': ['<p>The two products of the lrgAB operon are potential membrane proteins, and LrgA and LrgB are both thought to control of murein hydrolase activity and penicillin tolerance [PMID:10714982].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009887': {'PMID': 10714982,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '7',\n",
       "     'year': 2000,\n",
       "     'title': 'The Staphylococcus aureus lrgAB operon modulates murein hydrolase activity and penicillin tolerance.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1794-801',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Groicher KH', 'Firek BA', 'Fujimoto DF', 'Bayles KW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.7.1794-1801.2000'}},\n",
       "   'set_info': {'accession': 'CL0064', 'name': 'CPA_AT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 32,\n",
       "    'interactions': 0,\n",
       "    'matches': 20088,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19880,\n",
       "    'proteomes': 3511,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17990, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15850},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20443,\n",
       "    'alignment:full': 4091,\n",
       "    'alignment:seed': 362},\n",
       "   'cross_references': {}}},\n",
       " 'PF05504': {'metadata': {'accession': 'PF05504',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR046953',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Spore germination B3/ GerAC like, C-terminal',\n",
       "    'short': 'Spore_GerAC'},\n",
       "   'description': [\"<p>The GerAC protein of the Bacillus subtilis spore is required for the  germination response to L-alanine. Members of this family are thought to be located in the inner spore membrane. Although the function of this family is unclear, they are likely to encode the components of the  germination apparatus that respond directly to this germinant,  mediating the spore's response [PMID:11418573].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011479': {'PMID': 11418573,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '14',\n",
       "     'year': 2001,\n",
       "     'title': 'Localization of GerAA and GerAC germination proteins in the Bacillus subtilis spore.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4317-22',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Hudson KD',\n",
       "      'Corfe BM',\n",
       "      'Kemp EH',\n",
       "      'Feavers IM',\n",
       "      'Coote PJ',\n",
       "      'Moir A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.14.4317-4322.2001'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 12,\n",
       "    'interactions': 0,\n",
       "    'matches': 15103,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15102,\n",
       "    'proteomes': 919,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13678, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 3047},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15788,\n",
       "    'alignment:full': 4096,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF17957': {'metadata': {'accession': 'PF17957',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial Ig domain', 'short': 'Big_7'},\n",
       "   'description': ['<p>This entry represents a bacterial ig-like domain that is found in glycosyl hydrolase enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 3772,\n",
       "    'interactions': 0,\n",
       "    'matches': 29037,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20068,\n",
       "    'proteomes': 1935,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13766, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 9810},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24334,\n",
       "    'alignment:full': 4724,\n",
       "    'alignment:seed': 232},\n",
       "   'cross_references': {}}},\n",
       " 'PF17919': {'metadata': {'accession': 'PF17919',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041577',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RNase H-like domain found in reverse transcriptase',\n",
       "    'short': 'RT_RNaseH_2'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0219', 'name': 'RNase_H'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 3589,\n",
       "    'interactions': 0,\n",
       "    'matches': 66145,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65573,\n",
       "    'proteomes': 1039,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42483, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 5103},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 53877,\n",
       "    'alignment:full': 26244,\n",
       "    'alignment:seed': 174},\n",
       "   'cross_references': {}}},\n",
       " 'PF00104': {'metadata': {'accession': 'PF00104',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000536',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ligand-binding domain of nuclear hormone receptor',\n",
       "    'short': 'Hormone_recep'},\n",
       "   'description': ['<p>This all helical domain is involved in binding the hormone in these receptors.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019715': {'PMID': 9600906,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '11',\n",
       "     'year': 1998,\n",
       "     'title': \"Crystallographic comparison of the estrogen and progesterone receptor's ligand binding domains.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '5998-6003',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Tanenbaum DM', 'Wang Y', 'Williams SP', 'Sigler PB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.95.11.5998'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 456,\n",
       "    'interactions': 0,\n",
       "    'matches': 76892,\n",
       "    'pathways': 0,\n",
       "    'proteins': 75937,\n",
       "    'proteomes': 916,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 56208, 'rosettafold': 0},\n",
       "    'structures': 1720,\n",
       "    'taxa': 6344},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 56469,\n",
       "    'alignment:full': 32199,\n",
       "    'alignment:seed': 315},\n",
       "   'cross_references': {}}},\n",
       " 'PF00233': {'metadata': {'accession': 'PF00233',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002073',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"3'5'-cyclic nucleotide phosphodiesterase\",\n",
       "    'short': 'PDEase_I'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0237', 'name': 'HD_PDEase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 323,\n",
       "    'interactions': 0,\n",
       "    'matches': 45355,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44744,\n",
       "    'proteomes': 1916,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31582, 'rosettafold': 0},\n",
       "    'structures': 678,\n",
       "    'taxa': 8528},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32234,\n",
       "    'alignment:full': 19591,\n",
       "    'alignment:seed': 247},\n",
       "   'cross_references': {}}},\n",
       " 'PF00140': {'metadata': {'accession': 'PF00140',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009042',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sigma-70 factor, region 1.2', 'short': 'Sigma70_r1_2'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000942': {'PMID': 8858155,\n",
       "     'ISBN': None,\n",
       "     'volume': '87',\n",
       "     'issue': '1',\n",
       "     'year': 1996,\n",
       "     'title': 'Crystal structure of a sigma 70 subunit fragment from E. coli RNA polymerase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '127-36',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Malhotra A', 'Severinova E', 'Darst SA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)81329-X'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 154,\n",
       "    'interactions': 0,\n",
       "    'matches': 86548,\n",
       "    'pathways': 0,\n",
       "    'proteins': 86546,\n",
       "    'proteomes': 8754,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 73754, 'rosettafold': 0},\n",
       "    'structures': 263,\n",
       "    'taxa': 38165},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 78150,\n",
       "    'alignment:full': 15588,\n",
       "    'alignment:seed': 402},\n",
       "   'cross_references': {}}},\n",
       " 'PF13649': {'metadata': {'accession': 'PF13649',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041698',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Methyltransferase domain', 'short': 'Methyltransf_25'},\n",
       "   'description': ['<p>This family appears to be a methyltransferase domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1929,\n",
       "    'interactions': 0,\n",
       "    'matches': 250527,\n",
       "    'pathways': 0,\n",
       "    'proteins': 250342,\n",
       "    'proteomes': 10364,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 211213, 'rosettafold': 0},\n",
       "    'structures': 154,\n",
       "    'taxa': 44987},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 226338,\n",
       "    'alignment:full': 57976,\n",
       "    'alignment:seed': 3938},\n",
       "   'cross_references': {}}},\n",
       " 'PF13692': {'metadata': {'accession': 'PF13692',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl transferases group 1',\n",
       "    'short': 'Glyco_trans_1_4'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0113', 'name': 'GT-B'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1189,\n",
       "    'interactions': 0,\n",
       "    'matches': 126065,\n",
       "    'pathways': 0,\n",
       "    'proteins': 125616,\n",
       "    'proteomes': 7942,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 103970, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 34226},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 111208,\n",
       "    'alignment:full': 25855,\n",
       "    'alignment:seed': 943},\n",
       "   'cross_references': {}}},\n",
       " 'PF07549': {'metadata': {'accession': 'PF07549',\n",
       "   'entry_id': None,\n",
       "   'type': 'conserved_site',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022646',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SecD/SecF GG Motif', 'short': 'Sec_GG'},\n",
       "   'description': ['<p>This family consists of various prokaryotic SecD and SecF protein export membrane proteins. This SecD and SecF proteins are part of the multimeric protein export complex comprising SecA, D, E, F, G, Y, and YajC [PMID:9694879]. SecD and SecF are required to maintain a proton motive force [PMID:8112309]. This alignment encompasses a -GG- motif typically found in N-terminal half of the SecD/SecF proteins .</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 74,\n",
       "    'interactions': 0,\n",
       "    'matches': 67502,\n",
       "    'pathways': 0,\n",
       "    'proteins': 60042,\n",
       "    'proteomes': 7362,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 51585, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 32529},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 62354,\n",
       "    'alignment:full': 11738,\n",
       "    'alignment:seed': 358},\n",
       "   'cross_references': {}}},\n",
       " 'PF00036': {'metadata': {'accession': 'PF00036',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002048',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'EF hand', 'short': 'EF-hand_1'},\n",
       "   'description': ['<p>The EF-hands can be divided into two classes: signalling proteins and buffering/transport proteins.  The first group is the largest and includes the most well-known members of the family such as calmodulin, troponin C and S100B. These proteins typically undergo a calcium-dependent conformational change which opens a target binding site.  The latter group is represented by calbindin D9k and do not undergo calcium dependent conformational changes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001594': {'PMID': 2401372,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '2',\n",
       "     'year': 1990,\n",
       "     'title': 'EF-hand motifs in inositol phospholipid-specific phospholipase C.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '454-6',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Bairoch A', 'Cox JA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0014-5793(90)81214-9'},\n",
       "    'PUB00002719': {'PMID': 1618836,\n",
       "     'ISBN': None,\n",
       "     'volume': '267',\n",
       "     'issue': '19',\n",
       "     'year': 1992,\n",
       "     'title': 'Comparison of terbium (III) luminescence enhancement in mutants of EF hand calcium binding proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13340-7',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Hogue CW', 'MacManus JP', 'Banville D', 'Szabo AG.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/267/19/13340.pdf'},\n",
       "    'PUB00003419': {'PMID': 1602495,\n",
       "     'ISBN': None,\n",
       "     'volume': '34',\n",
       "     'issue': '5',\n",
       "     'year': 1992,\n",
       "     'title': 'Evolution of EF-hand calcium-modulated proteins. II. Domains of several subfamilies have diverse evolutionary histories.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '416-48',\n",
       "     'medline_journal': 'J Mol Evol',\n",
       "     'ISO_journal': 'J. Mol. Evol.',\n",
       "     'authors': ['Nakayama S', 'Moncrief ND', 'Kretsinger RH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/BF00162998'},\n",
       "    'PUB00019499': {'PMID': 7743133,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '1',\n",
       "     'year': 1995,\n",
       "     'title': 'The evolving model of calmodulin structure, function and activation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7-11',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Finn BE', 'Forsen S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00130-7'}},\n",
       "   'set_info': {'accession': 'CL0220', 'name': 'EF_hand'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 880,\n",
       "    'interactions': 0,\n",
       "    'matches': 34535,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33674,\n",
       "    'proteomes': 2209,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26118, 'rosettafold': 0},\n",
       "    'structures': 141,\n",
       "    'taxa': 12117},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26421,\n",
       "    'alignment:full': 12785,\n",
       "    'alignment:seed': 564},\n",
       "   'cross_references': {}}},\n",
       " 'PF00344': {'metadata': {'accession': 'PF00344',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002208',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SecY', 'short': 'SecY'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0376', 'name': 'Oxa1'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 147,\n",
       "    'interactions': 0,\n",
       "    'matches': 59573,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59217,\n",
       "    'proteomes': 11109,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 50485, 'rosettafold': 0},\n",
       "    'structures': 61,\n",
       "    'taxa': 48831},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52911,\n",
       "    'alignment:full': 12351,\n",
       "    'alignment:seed': 746},\n",
       "   'cross_references': {}}},\n",
       " 'PF00646': {'metadata': {'accession': 'PF00646',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001810',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'F-box domain', 'short': 'F-box'},\n",
       "   'description': ['<p>This domain is approximately 50 amino acids long, and is usually found in the N-terminal half of a variety of proteins. Two motifs that are commonly found associated with the F-box domain are the leucine rich repeats (LRRs; Pfam:PF00560 and Pfam:PF07723) and the WD repeat (Pfam:PF00400). The F-box domain has a role in mediating protein-protein interactions in a variety of contexts, such as polyubiquitination, transcription elongation, centromere binding and translational repression [1-2].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000938': {'PMID': 8706131,\n",
       "     'ISBN': None,\n",
       "     'volume': '86',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'SKP1 connects cell cycle regulators to the ubiquitin proteolysis machinery through a novel motif, the F-box.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '263-74',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Bai C',\n",
       "      'Sen P',\n",
       "      'Hofmann K',\n",
       "      'Ma L',\n",
       "      'Goebl M',\n",
       "      'Harper JW',\n",
       "      'Elledge SJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80098-7'},\n",
       "    'PUB00000952': {'PMID': 9346238,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '2',\n",
       "     'year': 1997,\n",
       "     'title': 'F-box proteins are receptors that recruit phosphorylated substrates to the SCF ubiquitin-ligase complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '209-19',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Skowyra D',\n",
       "      'Craig KL',\n",
       "      'Tyers M',\n",
       "      'Elledge SJ',\n",
       "      'Harper JW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80403-1'}},\n",
       "   'set_info': {'accession': 'CL0271', 'name': 'F-box'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2157,\n",
       "    'interactions': 0,\n",
       "    'matches': 104812,\n",
       "    'pathways': 0,\n",
       "    'proteins': 102956,\n",
       "    'proteomes': 2150,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 84387, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 9087},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 87049,\n",
       "    'alignment:full': 65401,\n",
       "    'alignment:seed': 426},\n",
       "   'cross_references': {}}},\n",
       " 'PF02677': {'metadata': {'accession': 'PF02677',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003828',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Epoxyqueuosine reductase QueH', 'short': 'QueH'},\n",
       "   'description': ['<p>The reduction of epoxyqueuosine (oQ) is the last step in the synthesis of the tRNA modification queuosine (Q). members of this family were predicted to encode for an alternative epoxyqueuosine reductase. Furthermore, it has been suggested that family members are a non-orthologous replacement of queG, responsible for oQ to Q conversion. QueH contains conserved cysteines that could be involved in the coordination of a Fe/S center in a similar fashion to what has been identified in QueG. No cobalamin was identified associated with recombinant QueH protein, indicating that the reduction activity is independent from cobalamin [PMID:28128549].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0039', 'name': 'HUP'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 29,\n",
       "    'interactions': 0,\n",
       "    'matches': 10968,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10934,\n",
       "    'proteomes': 1685,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 9666, 'rosettafold': 1},\n",
       "    'structures': 2,\n",
       "    'taxa': 9059},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10320,\n",
       "    'alignment:full': 1624,\n",
       "    'alignment:seed': 258},\n",
       "   'cross_references': {}}},\n",
       " 'PF00329': {'metadata': {'accession': 'PF00329',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001268',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Respiratory-chain NADH dehydrogenase, 30 Kd subunit',\n",
       "    'short': 'Complex1_30kDa'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 85,\n",
       "    'interactions': 0,\n",
       "    'matches': 62414,\n",
       "    'pathways': 0,\n",
       "    'proteins': 62390,\n",
       "    'proteomes': 7839,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 53424, 'rosettafold': 0},\n",
       "    'structures': 223,\n",
       "    'taxa': 52069},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 55263,\n",
       "    'alignment:full': 8993,\n",
       "    'alignment:seed': 557},\n",
       "   'cross_references': {}}},\n",
       " 'PF04461': {'metadata': {'accession': 'PF04461',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007551',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF520)', 'short': 'DUF520'},\n",
       "   'description': ['<p>The structure of the DUF520 family of uncharacterised proteins shows that it has composed of two domains each of which has the same topology [PMID:12943362].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 14,\n",
       "    'interactions': 0,\n",
       "    'matches': 20389,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20381,\n",
       "    'proteomes': 4205,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17565, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 18326},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18898,\n",
       "    'alignment:full': 3839,\n",
       "    'alignment:seed': 248},\n",
       "   'cross_references': {}}},\n",
       " 'PF01722': {'metadata': {'accession': 'PF01722',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002634',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'BolA-like protein', 'short': 'BolA'},\n",
       "   'description': ['<p>This family consist of the morphoprotein BolA from E. coli and its various homologues. In E. coli over expression of this protein causes round morphology and may be involved in  switching the cell between elongation and septation systems during  cell division [PMID:10361282]. The expression of BolA is growth rate regulated and is induced during the transition into the the stationary phase [PMID:10361282]. BolA is also induced by stress during early stages of  growth [PMID:10361282] and may have a general role in stress response.  It has also been suggested that BolA can induce the transcription of penicillin binding proteins 6 and 5 [PMID:2684651, PMID:10361282].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001179': {'PMID': 2684651,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '12',\n",
       "     'year': 1989,\n",
       "     'title': 'Induction of a growth-phase-dependent promoter triggers transcription of bolA, an Escherichia coli morphogene.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3923-31',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Aldea M',\n",
       "      'Garrido T',\n",
       "      'Hernandez-Chico C',\n",
       "      'Vicente M',\n",
       "      'Kushner SR.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=2684651'},\n",
       "    'PUB00003882': {'PMID': 10361282,\n",
       "     'ISBN': None,\n",
       "     'volume': '32',\n",
       "     'issue': '4',\n",
       "     'year': 1999,\n",
       "     'title': 'The stationary-phase morphogene bolA from Escherichia coli is induced by stress during early stages of growth.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '789-98',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Santos JM', 'Freire P', 'Vicente M', 'Arraiano CM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1999.01397.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 143,\n",
       "    'interactions': 0,\n",
       "    'matches': 38995,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38961,\n",
       "    'proteomes': 5402,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33559, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 23167},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35588,\n",
       "    'alignment:full': 10044,\n",
       "    'alignment:seed': 351},\n",
       "   'cross_references': {}}},\n",
       " 'PF00490': {'metadata': {'accession': 'PF00490',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001731',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Delta-aminolevulinic acid dehydratase', 'short': 'ALAD'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 109,\n",
       "    'interactions': 0,\n",
       "    'matches': 42270,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42138,\n",
       "    'proteomes': 9493,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 36075, 'rosettafold': 0},\n",
       "    'structures': 42,\n",
       "    'taxa': 38336},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38528,\n",
       "    'alignment:full': 9123,\n",
       "    'alignment:seed': 592},\n",
       "   'cross_references': {}}},\n",
       " 'PF04241': {'metadata': {'accession': 'PF04241',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006696',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF423)', 'short': 'DUF423'},\n",
       "   'description': ['<p>This family of proteins with unknown function is a possible integral membrane protein from Caenorhabditis elegans. This family of proteins has GO references indicating the protein is involved in nematode larval development and is a positive regulator of growth rate.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 58,\n",
       "    'interactions': 0,\n",
       "    'matches': 19401,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19389,\n",
       "    'proteomes': 4411,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16713, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 19856},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17920,\n",
       "    'alignment:full': 4365,\n",
       "    'alignment:seed': 340},\n",
       "   'cross_references': {}}},\n",
       " 'PF18803': {'metadata': {'accession': 'PF18803',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041457',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CxC2 like cysteine cluster associated with KDZ transposases',\n",
       "    'short': 'CxC2'},\n",
       "   'description': ['<p>A predicted Zinc chelating domain present N-terminal to the KDZ transposase domain [PMID:24398522].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0687', 'name': 'Zn_chelatase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 28,\n",
       "    'interactions': 0,\n",
       "    'matches': 3336,\n",
       "    'pathways': 0,\n",
       "    'proteins': 3320,\n",
       "    'proteomes': 150,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 2385, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 595},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 2512,\n",
       "    'alignment:full': 2298,\n",
       "    'alignment:seed': 269},\n",
       "   'cross_references': {}}},\n",
       " 'PF13439': {'metadata': {'accession': 'PF13439',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR028098',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyltransferase Family 4', 'short': 'Glyco_transf_4'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0113', 'name': 'GT-B'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1135,\n",
       "    'interactions': 0,\n",
       "    'matches': 322511,\n",
       "    'pathways': 0,\n",
       "    'proteins': 321716,\n",
       "    'proteomes': 10424,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 271565, 'rosettafold': 0},\n",
       "    'structures': 35,\n",
       "    'taxa': 45386},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 285881,\n",
       "    'alignment:full': 57840,\n",
       "    'alignment:seed': 910},\n",
       "   'cross_references': {}}},\n",
       " 'PF04403': {'metadata': {'accession': 'PF04403',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007498',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Paraquat-inducible protein A', 'short': 'PqiA'},\n",
       "   'description': ['<p>Paraquat is a superoxide radical-generating agent.  The promoter for the  pqiA gene is also inducible by other known superoxide generators [PMID:7751275]. This is predicted to be a family of integral membrane proteins, possibly  located in the inner membrane. This family is related to NADH dehydrogenase subunit 2 (Pfam:PF00361).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009983': {'PMID': 7751275,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '10',\n",
       "     'year': 1995,\n",
       "     'title': 'Isolation of a novel paraquat-inducible (pqi) gene regulated by the soxRS locus in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2673-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Koh YS', 'Roe JH.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/177/10/2673'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 24134,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18088,\n",
       "    'proteomes': 1779,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16231, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 9097},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25101,\n",
       "    'alignment:full': 3728,\n",
       "    'alignment:seed': 232},\n",
       "   'cross_references': {}}},\n",
       " 'PF03993': {'metadata': {'accession': 'PF03993',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007139',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of Unknown Function (DUF349)', 'short': 'DUF349'},\n",
       "   'description': ['<p>This domain is found singly or as up to five tandem repeats in a small set of  bacterial proteins. There are two or three alpha-helices, and possibly a beta-strand.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 21,\n",
       "    'interactions': 0,\n",
       "    'matches': 39579,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11894,\n",
       "    'proteomes': 2903,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10287, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 10324},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36181,\n",
       "    'alignment:full': 9053,\n",
       "    'alignment:seed': 619},\n",
       "   'cross_references': {}}},\n",
       " 'PF00268': {'metadata': {'accession': 'PF00268',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000358',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribonucleotide reductase, small chain',\n",
       "    'short': 'Ribonuc_red_sm'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020250': {'PMID': 8331655,\n",
       "     'ISBN': None,\n",
       "     'volume': '232',\n",
       "     'issue': '1',\n",
       "     'year': 1993,\n",
       "     'title': 'Structure and function of the Escherichia coli ribonucleotide reductase protein R2.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '123-64',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Nordlund P', 'Eklund H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1993.1374'}},\n",
       "   'set_info': {'accession': 'CL0044', 'name': 'Ferritin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 118,\n",
       "    'interactions': 0,\n",
       "    'matches': 42835,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42676,\n",
       "    'proteomes': 9085,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 34270, 'rosettafold': 0},\n",
       "    'structures': 161,\n",
       "    'taxa': 37239},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40354,\n",
       "    'alignment:full': 10533,\n",
       "    'alignment:seed': 268},\n",
       "   'cross_references': {}}},\n",
       " 'PF01649': {'metadata': {'accession': 'PF01649',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002583',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S20', 'short': 'Ribosomal_S20p'},\n",
       "   'description': ['<p>Bacterial ribosomal protein S20 interacts with 16S rRNA [PMID:3373529].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003228': {'PMID': 3373529,\n",
       "     'ISBN': None,\n",
       "     'volume': '200',\n",
       "     'issue': '2',\n",
       "     'year': 1988,\n",
       "     'title': 'Interaction of proteins S16, S17 and S20 with 16 S ribosomal RNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '291-9',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Stern S', 'Changchien LM', 'Craven GR', 'Noller HF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(88)90241-0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 29,\n",
       "    'interactions': 0,\n",
       "    'matches': 42371,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42360,\n",
       "    'proteomes': 8861,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36357, 'rosettafold': 0},\n",
       "    'structures': 783,\n",
       "    'taxa': 38681},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37959,\n",
       "    'alignment:full': 8193,\n",
       "    'alignment:seed': 626},\n",
       "   'cross_references': {}}},\n",
       " 'PF02572': {'metadata': {'accession': 'PF02572',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003724',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP:corrinoid adenosyltransferase BtuR/CobO/CobP',\n",
       "    'short': 'CobA_CobO_BtuR'},\n",
       "   'description': ['<p>This family consists of the BtuR, CobO, CobP proteins all of which are Cob(I)alamin adenosyltransferase, EC:2.5.1.17, involved in cobalamin (vitamin B12) biosynthesis.  These enzymes catalyse the adenosylation reaction: ATP + cob(I)alamin + H2O <=> phosphate + diphosphate + adenosylcobalamin.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000553': {'PMID': 9742225,\n",
       "     'ISBN': None,\n",
       "     'volume': '335 ( Pt 1)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'Cobalamin (vitamin B12) biosynthesis: identification and characterization of a Bacillus megaterium cobI operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '159-66',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Raux E', 'Lanois A', 'Warren MJ', 'Rambach A', 'Thermes C.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=9742225&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 36,\n",
       "    'interactions': 0,\n",
       "    'matches': 25812,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25413,\n",
       "    'proteomes': 4445,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 22043, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 20493},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23957,\n",
       "    'alignment:full': 4857,\n",
       "    'alignment:seed': 431},\n",
       "   'cross_references': {}}},\n",
       " 'PF00576': {'metadata': {'accession': 'PF00576',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023416',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HIUase/Transthyretin family', 'short': 'Transthyretin'},\n",
       "   'description': ['<p>This family includes transthyretin that is a thyroid hormone-binding protein that transports thyroxine from the bloodstream to the brain.  However, most of the sequences listed in this family do not bind thyroid hormones. They are actually enzymes of the purine catabolism that catalyse the conversion of 5-hydroxyisourate (HIU) to OHCU [PMID:16462750, PMID:16098976].  HIU hydrolysis is the original function of the family and is conserved from bacteria to mammals; transthyretins arose by gene duplications in the vertebrate lineage [PMID:16952372]. HIUases are distinguished in the alignment from the conserved C-terminal YRGS sequence.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002809': {'PMID': 8428915,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '4',\n",
       "     'year': 1993,\n",
       "     'title': 'The x-ray crystal structure refinements of normal human transthyretin and the amyloidogenic Val-30-->Met variant to 1.7-A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2416-24',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Hamilton JA',\n",
       "      'Steinrauf LK',\n",
       "      'Braden BC',\n",
       "      'Liepnieks J',\n",
       "      'Benson MD',\n",
       "      'Holmgren G',\n",
       "      'Sandgren O',\n",
       "      'Steen L.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/268/4/2416'},\n",
       "    'PUB00034597': {'PMID': 16098976,\n",
       "     'ISBN': None,\n",
       "     'volume': '579',\n",
       "     'issue': '21',\n",
       "     'year': 2005,\n",
       "     'title': 'Transthyretin-related proteins function to facilitate the hydrolysis of 5-hydroxyisourate, the end product of the uricase reaction.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4769-74',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Lee Y',\n",
       "      'Lee DH',\n",
       "      'Kho CW',\n",
       "      'Lee AY',\n",
       "      'Jang M',\n",
       "      'Cho S',\n",
       "      'Lee CH',\n",
       "      'Lee JS',\n",
       "      'Myung PK',\n",
       "      'Park BC',\n",
       "      'Park SG.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00041243': {'PMID': 16952372,\n",
       "     'ISBN': None,\n",
       "     'volume': '363',\n",
       "     'issue': '1',\n",
       "     'year': 2006,\n",
       "     'title': 'Structure of zebra fish HIUase: insights into evolution of an enzyme to a hormone transporter.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1-9',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Zanotti G',\n",
       "      'Cendron L',\n",
       "      'Ramazzina I',\n",
       "      'Folli C',\n",
       "      'Percudani R',\n",
       "      'Berni R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.jmb.2006.07.079'},\n",
       "    'PUB00042841': {'PMID': 16462750,\n",
       "     'ISBN': None,\n",
       "     'volume': '2',\n",
       "     'issue': '3',\n",
       "     'year': 2006,\n",
       "     'title': 'Completing the uric acid degradation pathway through phylogenetic comparison of whole genomes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '144-8',\n",
       "     'medline_journal': 'Nat Chem Biol',\n",
       "     'ISO_journal': 'Nat. Chem. Biol.',\n",
       "     'authors': ['Ramazzina I',\n",
       "      'Folli C',\n",
       "      'Secchi A',\n",
       "      'Berni R',\n",
       "      'Percudani R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nchembio768'}},\n",
       "   'set_info': {'accession': 'CL0287', 'name': 'Transthyretin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 81,\n",
       "    'interactions': 0,\n",
       "    'matches': 19762,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19739,\n",
       "    'proteomes': 4759,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17254, 'rosettafold': 0},\n",
       "    'structures': 386,\n",
       "    'taxa': 19301},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18719,\n",
       "    'alignment:full': 4902,\n",
       "    'alignment:seed': 326},\n",
       "   'cross_references': {}}},\n",
       " 'PF00396': {'metadata': {'accession': 'PF00396',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000118',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Granulin', 'short': 'Granulin'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 106,\n",
       "    'interactions': 0,\n",
       "    'matches': 15528,\n",
       "    'pathways': 0,\n",
       "    'proteins': 4377,\n",
       "    'proteomes': 890,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 3355, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 3626},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9750,\n",
       "    'alignment:full': 6093,\n",
       "    'alignment:seed': 270},\n",
       "   'cross_references': {}}},\n",
       " 'PF04039': {'metadata': {'accession': 'PF04039',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007182',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain related to MnhB subunit of Na+/H+ antiporter',\n",
       "    'short': 'MnhB'},\n",
       "   'description': ['<p>Possible subunit of Na+/H+ antiporter [PMID:9852009], [PMID:11356194]. Predicted integral membrane protein, usually four transmembrane regions in this domain.  Often found in bacterial NADH dehydrogenase subunit.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009819': {'PMID': 11356194,\n",
       "     'ISBN': None,\n",
       "     'volume': '496',\n",
       "     'issue': '2-3',\n",
       "     'year': 2001,\n",
       "     'title': 'Mrp-dependent Na(+)/H(+) antiporters of Bacillus exhibit characteristics that are unanticipated for completely secondary active transporters.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '117-20',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Ito M', 'Guffanti AA', 'Krulwich TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(01)02417-6'},\n",
       "    'PUB00010583': {'PMID': 9852009,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '24',\n",
       "     'year': 1998,\n",
       "     'title': 'A putative multisubunit Na+/H+ antiporter from Staphylococcus aureus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6642-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Hiramatsu T',\n",
       "      'Kodama K',\n",
       "      'Kuroda T',\n",
       "      'Mizushima T',\n",
       "      'Tsuchiya T.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9852009'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 20826,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20810,\n",
       "    'proteomes': 3945,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18206, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 17025},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19441,\n",
       "    'alignment:full': 4525,\n",
       "    'alignment:seed': 374},\n",
       "   'cross_references': {}}},\n",
       " 'PF14815': {'metadata': {'accession': 'PF14815',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029119',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NUDIX domain', 'short': 'NUDIX_4'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0261', 'name': 'NUDIX'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 107,\n",
       "    'interactions': 0,\n",
       "    'matches': 35986,\n",
       "    'pathways': 0,\n",
       "    'proteins': 35986,\n",
       "    'proteomes': 6703,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30800, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 27713},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33629,\n",
       "    'alignment:full': 7364,\n",
       "    'alignment:seed': 383},\n",
       "   'cross_references': {}}},\n",
       " 'PF02690': {'metadata': {'accession': 'PF02690',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003841',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Na+/Pi-cotransporter', 'short': 'Na_Pi_cotrans'},\n",
       "   'description': ['<p>This is a family of mainly mammalian type II renal Na+/Pi-cotransporters with other related sequences from lower eukaryotes and bacteria some of  which are also Na+/Pi-cotransporters.  In the kidney the type II renal  Na+/Pi-cotransporters protein allows re-absorption of filtered Pi in the  proximal tubule [PMID:9826740].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019908': {'PMID': 9826740,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '24',\n",
       "     'year': 1998,\n",
       "     'title': 'Characterization of a murine type II sodium-phosphate cotransporter expressed in mammalian small intestine.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14564-9',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Hilfiker H',\n",
       "      'Hattenhauer O',\n",
       "      'Traebert M',\n",
       "      'Forster I',\n",
       "      'Murer H',\n",
       "      'Biber J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.95.24.14564'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 72,\n",
       "    'interactions': 0,\n",
       "    'matches': 53397,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28429,\n",
       "    'proteomes': 4372,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23929, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 19761},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48157,\n",
       "    'alignment:full': 11434,\n",
       "    'alignment:seed': 451},\n",
       "   'cross_references': {}}},\n",
       " 'PF00885': {'metadata': {'accession': 'PF00885',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002180',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '6,7-dimethyl-8-ribityllumazine synthase',\n",
       "    'short': 'DMRL_synthase'},\n",
       "   'description': ['<p>This family includes the beta chain of 6,7-dimethyl-8- ribityllumazine synthase EC:2.5.1.9, an enzyme involved in riboflavin biosynthesis. The family also includes a subfamily of distant archaebacterial proteins that may also have the same function for example Swiss:O28856. The family contains a number of different subsets including a family of proteins comprising archaeal lumazine and riboflavin synthases, type I lumazine synthases, and the eubacterial type II lumazine synthases [PMID:16923880]. It has been established that lumazine synthase catalyses the penultimate step in the biosynthesis of riboflavin in plants and microorganisms.  The type I lumazine synthases area active in pentameric or icosahedral quaternary assemblies, whereas the type II are decameric [PMID:17854827].  Brucella, a bacterial genus that causes brucellosis, and other Rhizobiales have an atypical riboflavin metabolic pathway. Brucella spp code for both a type-I and a type-II lumazine synthase, and it has been shown that at least one of these two has to be present in order for Brucella to be viable, showing that in the case of Brucella flavin metabolism is implicated in bacterial virulence [PMID:20195542].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 81,\n",
       "    'interactions': 0,\n",
       "    'matches': 44830,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44793,\n",
       "    'proteomes': 8963,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38887, 'rosettafold': 0},\n",
       "    'structures': 54,\n",
       "    'taxa': 37032},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40938,\n",
       "    'alignment:full': 9179,\n",
       "    'alignment:seed': 601},\n",
       "   'cross_references': {}}},\n",
       " 'PF02772': {'metadata': {'accession': 'PF02772',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022629',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'S-adenosylmethionine synthetase, central domain',\n",
       "    'short': 'S-AdoMet_synt_M'},\n",
       "   'description': ['<p>The three domains of S-adenosylmethionine synthetase have the same alpha+beta fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020285': {'PMID': 8611562,\n",
       "     'ISBN': None,\n",
       "     'volume': '35',\n",
       "     'issue': '8',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure and function of S-adenosylmethionine synthetase: crystal structures of S-adenosylmethionine synthetase with ADP, BrADP, and PPi at 28 angstroms resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2586-96',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Takusagawa F', 'Kamitori S', 'Markham GD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi952604z'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 120,\n",
       "    'interactions': 0,\n",
       "    'matches': 54718,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54666,\n",
       "    'proteomes': 10742,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46331, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 44568},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48834,\n",
       "    'alignment:full': 12123,\n",
       "    'alignment:seed': 685},\n",
       "   'cross_references': {}}},\n",
       " 'PF02583': {'metadata': {'accession': 'PF02583',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003735',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Metal-sensitive transcriptional repressor',\n",
       "    'short': 'Trns_repr_metal'},\n",
       "   'description': ['<p>This is a family of metal-sensitive repressors, involved in resistance to metal ions. Members of this family bind copper, nickel or cobalt ions via conserved cysteine and histidine residues. In the absence of metal ions, these proteins bind to promoter regions and repress transcription. When bound to metal ions they are unable to bind DNA, leading to transcriptional derepression [1-5].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 37404,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37382,\n",
       "    'proteomes': 5090,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33126, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 22828},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35828,\n",
       "    'alignment:full': 7975,\n",
       "    'alignment:seed': 575},\n",
       "   'cross_references': {}}},\n",
       " 'PF08711': {'metadata': {'accession': 'PF08711',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017923',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TFIIS helical bundle-like domain', 'short': 'Med26'},\n",
       "   'description': [\"<p>Mediator is a large complex of up to 33 proteins that is conserved from plants to fungi to humans - the number and representation of individual subunits varying with species {1-2]. It is arranged into four different sections, a core, a head, a tail and a kinase-activity part, and the number of subunits within each of these is what varies with species. Overall, Mediator regulates the transcriptional activity of RNA polymerase II but it would appear that each of the four different sections has a slightly different function [PMID:18050436]. Mediator exists in two major forms in human cells: a smaller form that interacts strongly with pol II and activates transcription, and a large form that does not interact strongly with pol II and does not directly activate transcription. Notably, the 'small' and 'large' Mediator complexes differ in their subunit composition: the Med26 subunit preferentially associates with the small, active complex, whereas cdk8, cyclin C, Med12 and Med13 associate with the large Mediator complex [PMID:18418385]. This family includesthe C terminal region of a number of eukaryotic hypothetical proteins which are homologous to the Saccharomyces cerevisiae protein IWS1. IWS1 is known to be an Pol II transcription elongation factor and interacts with Spt6 and Spt5 [PMID:12242279, PMID:12556496].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 312,\n",
       "    'interactions': 0,\n",
       "    'matches': 19420,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19356,\n",
       "    'proteomes': 2171,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14604, 'rosettafold': 0},\n",
       "    'structures': 46,\n",
       "    'taxa': 7885},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15287,\n",
       "    'alignment:full': 9427,\n",
       "    'alignment:seed': 416},\n",
       "   'cross_references': {}}},\n",
       " 'PF08529': {'metadata': {'accession': 'PF08529',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013735',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NusA N-terminal domain', 'short': 'NusA_N'},\n",
       "   'description': ['<p>This domain represents the RNA polymerase binding domain of NusA.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011739': {'PMID': 11430821,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '6',\n",
       "     'year': 2001,\n",
       "     'title': 'An extended RNA binding surface through arrayed S1 and KH domains in transcription factor NusA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1177-89',\n",
       "     'medline_journal': 'Mol Cell',\n",
       "     'ISO_journal': 'Mol. Cell',\n",
       "     'authors': ['Worbs M',\n",
       "      'Bourenkov GP',\n",
       "      'Bartunik HD',\n",
       "      'Huber R',\n",
       "      'Wahl MC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1097-2765(01)00262-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 100,\n",
       "    'interactions': 0,\n",
       "    'matches': 47138,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43725,\n",
       "    'proteomes': 8517,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37350, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 36636},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42642,\n",
       "    'alignment:full': 8539,\n",
       "    'alignment:seed': 490},\n",
       "   'cross_references': {}}},\n",
       " 'PF17676': {'metadata': {'accession': 'PF17676',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040921',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LD-carboxypeptidase C-terminal domain',\n",
       "    'short': 'Peptidase_S66C'},\n",
       "   'description': ['<p>Muramoyl-tetrapeptide carboxypeptidase hydrolyses a peptide bond between a di-basic amino acid and the C-terminal D-alanine in the tetrapeptide moiety in peptidoglycan. This cleaves the bond between an L- and a D-amino acid.                                The function of this activity is in murein recycling. This family also includes the microcin c7 self-immunity protein Swiss:Q47511. This family corresponds to Merops family S66.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0364', 'name': 'Leu-IlvD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 27852,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27844,\n",
       "    'proteomes': 4503,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24151, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 19604},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26552,\n",
       "    'alignment:full': 5474,\n",
       "    'alignment:seed': 240},\n",
       "   'cross_references': {}}},\n",
       " 'PF01367': {'metadata': {'accession': 'PF01367',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020045',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"5'-3' exonuclease, C-terminal SAM fold\",\n",
       "    'short': '5_3_exonuc'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019069': {'PMID': 8717047,\n",
       "     'ISBN': None,\n",
       "     'volume': '382',\n",
       "     'issue': '6588',\n",
       "     'year': 1996,\n",
       "     'title': 'Structure of Taq polymerase with DNA at the polymerase active site.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '278-81',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Eom SH', 'Wang J', 'Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/382278a0'},\n",
       "    'PUB00019070': {'PMID': 8657312,\n",
       "     'ISBN': None,\n",
       "     'volume': '382',\n",
       "     'issue': '6586',\n",
       "     'year': 1996,\n",
       "     'title': \"A helical arch allowing single-stranded DNA to thread through T5 5'-exonuclease.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '90-3',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Ceska TA', 'Sayers JR', 'Stier G', 'Suck D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/382090a0'}},\n",
       "   'set_info': {'accession': 'CL0464', 'name': '5_3_exonuc_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 127,\n",
       "    'interactions': 0,\n",
       "    'matches': 60916,\n",
       "    'pathways': 0,\n",
       "    'proteins': 60915,\n",
       "    'proteomes': 8947,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 51932, 'rosettafold': 0},\n",
       "    'structures': 23,\n",
       "    'taxa': 38313},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 56777,\n",
       "    'alignment:full': 10886,\n",
       "    'alignment:seed': 749},\n",
       "   'cross_references': {}}},\n",
       " 'PF08495': {'metadata': {'accession': 'PF08495',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013702',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FIST N domain', 'short': 'FIST'},\n",
       "   'description': ['<p>The FIST N domain is a novel sensory domain, which is present in signal transduction proteins from Bacteria, Archaea and Eukarya. Chromosomal proximity of FIST-encoding genes to those coding for proteins involved in amino acid metabolism and transport suggest that FIST domains bind small ligands, such as amino acids [PMID:17855421].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044261': {'PMID': 17855421,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '19',\n",
       "     'year': 2007,\n",
       "     'title': 'FIST: a sensory domain for diverse signal transduction pathways in prokaryotes and ubiquitin signaling in eukaryotes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2518-21',\n",
       "     'medline_journal': 'Bioinformatics',\n",
       "     'ISO_journal': 'Bioinformatics',\n",
       "     'authors': ['Borziak K', 'Zhulin IB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/bioinformatics/btm384'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 171,\n",
       "    'interactions': 0,\n",
       "    'matches': 15606,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15596,\n",
       "    'proteomes': 2526,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12873, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12183},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13559,\n",
       "    'alignment:full': 3073,\n",
       "    'alignment:seed': 298},\n",
       "   'cross_references': {}}},\n",
       " 'PF01106': {'metadata': {'accession': 'PF01106',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001075',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NifU-like domain', 'short': 'NifU'},\n",
       "   'description': ['<p>This is an alignment of the carboxy-terminal domain. This is the only common region between the NifU protein from nitrogen-fixing bacteria and rhodobacterial species. The biochemical function of NifU is unknown [PMID:8048161].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005420': {'PMID': 8048161,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '5',\n",
       "     'year': 1994,\n",
       "     'title': 'The modular structure of NifU proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '199-200',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Ouzounis C', 'Bork P', 'Sander C.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0968-0004(94)90021-3'}},\n",
       "   'set_info': {'accession': 'CL0232', 'name': 'NifU'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 164,\n",
       "    'interactions': 0,\n",
       "    'matches': 40746,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39926,\n",
       "    'proteomes': 7231,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 34030, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 31162},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36177,\n",
       "    'alignment:full': 9443,\n",
       "    'alignment:seed': 432},\n",
       "   'cross_references': {}}},\n",
       " 'PF13932': {'metadata': {'accession': 'PF13932',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026904',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'tRNA modifying enzyme MnmG/GidA C-terminal domain',\n",
       "    'short': 'GIDA_C'},\n",
       "   'description': ['<p>The GidA associated domain is a domain that has been identified at the C-terminus of protein GidA. It consists of several helices, the last three being rather short and forming small bundle.  GidA is an tRNA modification enzyme found in bacteria and mitochondrial. Based on mutational analysis this domain has been suggested to be implicated in binding of the D-stem of tRNA [PMID:19446527] and to be responsible for the interaction with protein MnmE [PMID:18565343]. Structures of GidA in complex with either tRNA or MnmE are missing. Reported to bind to Pfam family MnmE, Pfam:PF12631.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 125,\n",
       "    'interactions': 0,\n",
       "    'matches': 40434,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40004,\n",
       "    'proteomes': 8509,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33983, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 35322},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36882,\n",
       "    'alignment:full': 8084,\n",
       "    'alignment:seed': 838},\n",
       "   'cross_references': {}}},\n",
       " 'PF00887': {'metadata': {'accession': 'PF00887',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000582',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acyl CoA binding protein', 'short': 'ACBP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0632', 'name': 'FERM_M'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 335,\n",
       "    'interactions': 0,\n",
       "    'matches': 20382,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20309,\n",
       "    'proteomes': 3147,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16430, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 12017},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16479,\n",
       "    'alignment:full': 8768,\n",
       "    'alignment:seed': 453},\n",
       "   'cross_references': {}}},\n",
       " 'PF00199': {'metadata': {'accession': 'PF00199',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011614',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Catalase', 'short': 'Catalase'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 188,\n",
       "    'interactions': 0,\n",
       "    'matches': 51073,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50486,\n",
       "    'proteomes': 6564,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 44731, 'rosettafold': 0},\n",
       "    'structures': 136,\n",
       "    'taxa': 28941},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50168,\n",
       "    'alignment:full': 12367,\n",
       "    'alignment:seed': 341},\n",
       "   'cross_references': {}}},\n",
       " 'PF01357': {'metadata': {'accession': 'PF01357',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007117',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Expansin C-terminal domain', 'short': 'Expansin_C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of expansins, plant cell wall proteins involved in the non-enzymatic rearrangement of cell walls during cell growth. It contains the allergens lol PI, PII and PIII from Lolium perenne.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000312': {'PMID': 2605214,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '21',\n",
       "     'year': 1989,\n",
       "     'title': 'Complete primary structure of a Lolium perenne (perennial rye grass) pollen allergen, Lol p III: comparison with known Lol p I and II sequences.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8665-70',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Ansari AA', 'Shenbagamurthi P', 'Marsh DG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00447a058'}},\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 129,\n",
       "    'interactions': 0,\n",
       "    'matches': 13485,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13334,\n",
       "    'proteomes': 376,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11524, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 1784},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11694,\n",
       "    'alignment:full': 7677,\n",
       "    'alignment:seed': 274},\n",
       "   'cross_references': {}}},\n",
       " 'PF16863': {'metadata': {'accession': 'PF16863',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031727',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'N-terminal barrel of NtMGAM and CtMGAM, maltase-glucoamylase',\n",
       "    'short': 'NtCtMGAM_N'},\n",
       "   'description': ['<p>NtCtMGAM_N is a beta-barrel-like structure just N-terminal to the catalytic domain of maltase-glucoamylase in eukaryotes. It contributes to the architecture of the substrate-binding site, by donating a loop that comes into close contact with two regions in the catalytic domain thereby creating the site [PMID:18036614]. This family is frequently found at the N-terminus of Glycosyl hydrolase 31, Pfam:PF01055.to which it contributes as above.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 341,\n",
       "    'interactions': 0,\n",
       "    'matches': 11474,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9774,\n",
       "    'proteomes': 1835,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 6275, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 6518},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7717,\n",
       "    'alignment:full': 4963,\n",
       "    'alignment:seed': 309},\n",
       "   'cross_references': {}}},\n",
       " 'PF00988': {'metadata': {'accession': 'PF00988',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002474',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Carbamoyl-phosphate synthase small chain, CPSase domain',\n",
       "    'short': 'CPSase_sm_chain'},\n",
       "   'description': ['<p>The carbamoyl-phosphate synthase domain is in the amino terminus of  protein. Carbamoyl-phosphate synthase catalyses the ATP-dependent synthesis of carbamyl-phosphate from  glutamine or ammonia and bicarbonate.  This  important enzyme initiates both the urea cycle and the biosynthesis  of arginine and/or pyrimidines [PMID:1972379]. The carbamoyl-phosphate synthase (CPS) enzyme in prokaryotes is a  heterodimer of a small and large chain.  The small chain promotes the hydrolysis of glutamine to ammonia, which is used by the large chain to synthesise carbamoyl phosphate. See Pfam:PF00289. The small chain has a GATase domain in the carboxyl terminus. See Pfam:PF00117.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002551': {'PMID': 1972379,\n",
       "     'ISBN': None,\n",
       "     'volume': '265',\n",
       "     'issue': '18',\n",
       "     'year': 1990,\n",
       "     'title': 'Mammalian carbamyl phosphate synthetase (CPS). DNA sequence and evolution of the CPS domain of the Syrian hamster multifunctional protein CAD.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10395-402',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Simmer JP',\n",
       "      'Kelly RE',\n",
       "      'Rinker AG Jr',\n",
       "      'Scully JL',\n",
       "      'Evans DR.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/265/18/10395.pdf'}},\n",
       "   'set_info': {'accession': 'CL0364', 'name': 'Leu-IlvD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 174,\n",
       "    'interactions': 0,\n",
       "    'matches': 55989,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55966,\n",
       "    'proteomes': 10283,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 43081, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 48797},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50894,\n",
       "    'alignment:full': 11317,\n",
       "    'alignment:seed': 617},\n",
       "   'cross_references': {}}},\n",
       " 'PF02990': {'metadata': {'accession': 'PF02990',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004240',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Endomembrane protein 70', 'short': 'EMP70'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019510': {'PMID': 9332367,\n",
       "     'ISBN': None,\n",
       "     'volume': '197',\n",
       "     'issue': '1-2',\n",
       "     'year': 1997,\n",
       "     'title': 'Cloning of a human multispanning membrane protein cDNA: evidence for a new protein family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '195-204',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Chluba-de Tapia J', 'de Tapia M', 'Jaggin V', 'Eberle AN.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0378-1119(97)00263-1'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 248,\n",
       "    'interactions': 0,\n",
       "    'matches': 18913,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17833,\n",
       "    'proteomes': 2120,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14414, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 8436},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14936,\n",
       "    'alignment:full': 8940,\n",
       "    'alignment:seed': 301},\n",
       "   'cross_references': {}}},\n",
       " 'PF07743': {'metadata': {'accession': 'PF07743',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009073',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HSCB C-terminal oligomerisation domain',\n",
       "    'short': 'HSCB_C'},\n",
       "   'description': ['<p>This domain is the HSCB C-terminal oligomerisation domain and is found on co-chaperone proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 87,\n",
       "    'interactions': 0,\n",
       "    'matches': 11723,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11712,\n",
       "    'proteomes': 3260,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10137, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 14261},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11022,\n",
       "    'alignment:full': 2851,\n",
       "    'alignment:seed': 273},\n",
       "   'cross_references': {}}},\n",
       " 'PF02626': {'metadata': {'accession': 'PF02626',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003778',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Carboxyltransferase domain, subdomain A and B',\n",
       "    'short': 'CT_A_B'},\n",
       "   'description': ['<p>Urea carboxylase (UC) catalyses a two-step, ATP- and biotin-dependent carboxylation reaction of urea. It is composed of biotin carboxylase (BC), carboxyltransferase (CT), and biotin carboxyl carrier protein  (BCCP) domains. The CT domain of UC consists of four subdomains, named A, B, C and D. This domain covers the A and B subdomains of the CT domain. This domain covers the whole length of KipA (kinase A) from Bacillus subtilis [PMID:9334321]. It can also be found in S. cerevisiae urea  amidolyase Dur1,2, which is a multifunctional biotin-dependent enzyme with domains for urea carboxylase and allophanate (urea carboxylate) hydrolase activity[PMID:20884691]. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0475', 'name': 'Cyclophil-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 29653,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29525,\n",
       "    'proteomes': 5000,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26015, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 20414},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29681,\n",
       "    'alignment:full': 6055,\n",
       "    'alignment:seed': 534},\n",
       "   'cross_references': {}}},\n",
       " 'PF00288': {'metadata': {'accession': 'PF00288',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006204',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GHMP kinases N terminal domain',\n",
       "    'short': 'GHMP_kinases_N'},\n",
       "   'description': ['<p>This family includes homoserine kinases, galactokinases and mevalonate kinases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 501,\n",
       "    'interactions': 0,\n",
       "    'matches': 135402,\n",
       "    'pathways': 0,\n",
       "    'proteins': 135368,\n",
       "    'proteomes': 10680,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 115975, 'rosettafold': 0},\n",
       "    'structures': 117,\n",
       "    'taxa': 45141},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 122122,\n",
       "    'alignment:full': 29307,\n",
       "    'alignment:seed': 447},\n",
       "   'cross_references': {}}},\n",
       " 'PF02538': {'metadata': {'accession': 'PF02538',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003692',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hydantoinase B/oxoprolinase', 'short': 'Hydantoinase_B'},\n",
       "   'description': ['<p>This family includes N-methylhydaintoinase B which converts hydantoin to N-carbamyl-amino acids, and 5-oxoprolinase (Swiss:P97608) EC:3.5.2.9 which catalyses the formation of L-glutamate from  5-oxo-L-proline. These enzymes are part of the oxoprolinase family and are related to Pfam:PF01968.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019732': {'PMID': 1732229,\n",
       "     'ISBN': None,\n",
       "     'volume': '174',\n",
       "     'issue': '3',\n",
       "     'year': 1992,\n",
       "     'title': 'Cloning and sequencing of the genes involved in the conversion of 5-substituted hydantoins to the corresponding L-amino acids from the native plasmid of Pseudomonas sp. strain NS671.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '962-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Watabe K', 'Ishikawa T', 'Mukohara Y', 'Nakamura H.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=1732229'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 212,\n",
       "    'interactions': 0,\n",
       "    'matches': 36783,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36287,\n",
       "    'proteomes': 4816,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26529, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 19809},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31290,\n",
       "    'alignment:full': 8315,\n",
       "    'alignment:seed': 351},\n",
       "   'cross_references': {}}},\n",
       " 'PF18199': {'metadata': {'accession': 'PF18199',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041228',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dynein heavy chain C-terminal domain',\n",
       "    'short': 'Dynein_C'},\n",
       "   'description': ['<p>This family represents the C-terminal domain of dynein heavy chain. This domain is a complex structure comprising six alpha-helices and an incomplete six-stranded antiparallel beta-barrel. The shape of this domain is distinctively flat, spreading over the AAA1, AAA5 and AAA6 domain [PMID:22398446].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1340,\n",
       "    'interactions': 0,\n",
       "    'matches': 25686,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25027,\n",
       "    'proteomes': 1351,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4601, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 5474},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17757,\n",
       "    'alignment:full': 10431,\n",
       "    'alignment:seed': 366},\n",
       "   'cross_references': {}}},\n",
       " 'PF02805': {'metadata': {'accession': 'PF02805',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004026',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Metal binding domain of Ada', 'short': 'Ada_Zn_binding'},\n",
       "   'description': ['<p>The Escherichia coli Ada protein repairs O6-methylguanine residues and methyl phosphotriesters in DNA by direct transfer of the methyl group to a cysteine residue. This domain contains four conserved cysteines that form a zinc binding site [PMID:1581309, PMID:8500619]. One of these cysteines is a methyl group acceptor. The methylated domain can then specifically bind to the ada box on a DNA duplex [PMID:8500619].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007287': {'PMID': 1581309,\n",
       "     'ISBN': None,\n",
       "     'volume': '31',\n",
       "     'issue': '19',\n",
       "     'year': 1992,\n",
       "     'title': 'Zinc binding by the methylation signaling domain of the Escherichia coli Ada protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4541-7',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Myers LC',\n",
       "      'Terranova MP',\n",
       "      'Nash HM',\n",
       "      'Markus MA',\n",
       "      'Verdine GL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00134a002'},\n",
       "    'PUB00007288': {'PMID': 8500619,\n",
       "     'ISBN': None,\n",
       "     'volume': '323',\n",
       "     'issue': '3',\n",
       "     'year': 1993,\n",
       "     'title': 'Folding topology and DNA binding of the N-terminal fragment of Ada protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '252-6',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Sakashita H',\n",
       "      'Sakuma T',\n",
       "      'Ohkubo T',\n",
       "      'Kainosho M',\n",
       "      'Sakumi K',\n",
       "      'Sekiguchi M',\n",
       "      'Morikawa K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0014-5793(93)81351-Y'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 108,\n",
       "    'interactions': 0,\n",
       "    'matches': 23409,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23405,\n",
       "    'proteomes': 4296,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20593, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 18292},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22838,\n",
       "    'alignment:full': 4693,\n",
       "    'alignment:seed': 266},\n",
       "   'cross_references': {}}},\n",
       " 'PF11838': {'metadata': {'accession': 'PF11838',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024571',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ERAP1-like C-terminal domain', 'short': 'ERAP1_C'},\n",
       "   'description': ['<p>This large domain is composed of 16 alpha helices organized as 8  HEAT-like repeats. This domain forms a concave face that faces  towards the active site of the peptidase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 365,\n",
       "    'interactions': 0,\n",
       "    'matches': 39346,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37981,\n",
       "    'proteomes': 4547,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30127, 'rosettafold': 0},\n",
       "    'structures': 79,\n",
       "    'taxa': 17596},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32424,\n",
       "    'alignment:full': 13601,\n",
       "    'alignment:seed': 677},\n",
       "   'cross_references': {}}},\n",
       " 'PF02415': {'metadata': {'accession': 'PF02415',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003368',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Chlamydia polymorphic membrane protein (Chlamydia_PMP) repeat',\n",
       "    'short': 'Chlam_PMP'},\n",
       "   'description': ['<p>This family contains several Chlamydia polymorphic membrane proteins. Chlamydia pneumoniae is an obligate intracellular bacterium and a common human pathogen causing infection of the upper and lower respiratory tract. Common for the Pmps are the tetrapeptide GGA(I/V/L) motif repeated several times in the N-terminal part. The C-terminal half is characterised by conserved tryptophans and a carboxy-terminal phenylalanine. A signal peptide leader sequence is predicted in 20 C. pneumoniae Pmps, which indicates an outer membrane localisation. Pmp10 and Pmp11 contain a signal peptidase II cleavage site suggesting lipid modification. The C. pneumoniae pmp genes represent 17.5% of the chlamydia-specific coding capacity and they are all transcribed during chlamydial growth but the function of Pmps remains unknown [PMID:11583841]. This family shows some similarity to Pfam:PF05594 and hence is likely to also form a beta-helical structure (personal obs:C Yeats).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015051': {'PMID': 11583841,\n",
       "     'ISBN': None,\n",
       "     'volume': '203',\n",
       "     'issue': '2',\n",
       "     'year': 2001,\n",
       "     'title': 'Differential expression of Pmp10 in cell culture infected with Chlamydia pneumoniae CWL029.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '153-9',\n",
       "     'medline_journal': 'FEMS Microbiol Lett',\n",
       "     'ISO_journal': 'FEMS Microbiol. Lett.',\n",
       "     'authors': ['Pedersen AS', 'Christiansen G', 'Birkelund S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1574-6968.2001.tb10834.x'}},\n",
       "   'set_info': {'accession': 'CL0268', 'name': 'Pec_lyase-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 388,\n",
       "    'interactions': 0,\n",
       "    'matches': 4956,\n",
       "    'pathways': 0,\n",
       "    'proteins': 2304,\n",
       "    'proteomes': 203,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1536, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1672},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 5206,\n",
       "    'alignment:full': 712,\n",
       "    'alignment:seed': 524},\n",
       "   'cross_references': {}}},\n",
       " 'PF04371': {'metadata': {'accession': 'PF04371',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007466',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Porphyromonas-type peptidyl-arginine deiminase',\n",
       "    'short': 'PAD_porph'},\n",
       "   'description': ['<p>Peptidyl-arginine deiminase (PAD) enzymes catalyse the deimination of the guanidino group from carboxy-terminal arginine residues of various peptides to produce ammonia.  PAD from Porphyromonas gingivalis (PPAD) appears to be evolutionarily unrelated to mammalian PAD (Pfam:PF03068), which is a metalloenzyme.  PPAD is thought to belong to the same superfamily as aminotransferase and arginine deiminase, and to form an alpha/beta propeller structure. This family has previously been named PPADH (Porphyromonas peptidyl-arginine deiminase homologues) [PMID:11504612]. The predicted catalytic residues in PPAD (Swiss:Q9RQJ2) are Asp130, Asp187, His236, Asp238 and Cys351 [PMID:11504612]. These are absolutely conserved with the exception of Asp187 which is absent in two family members.  PPAD is also able to catalyse the deimination of free L-arginine, but has primarily peptidyl-arginine specificity.  It may have a FMN cofactor [PMID:10377098].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009967': {'PMID': 11504612,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '8',\n",
       "     'year': 2001,\n",
       "     'title': 'A novel superfamily of enzymes that catalyze the modification of guanidino groups.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '465-8',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Shirai H', 'Blundell TL', 'Mizuguchi K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(01)01906-5'},\n",
       "    'PUB00009968': {'PMID': 10377098,\n",
       "     'ISBN': None,\n",
       "     'volume': '67',\n",
       "     'issue': '7',\n",
       "     'year': 1999,\n",
       "     'title': 'Purification, characterization, and sequence analysis of a potential virulence factor from Porphyromonas gingivalis, peptidylarginine deiminase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3248-56',\n",
       "     'medline_journal': 'Infect Immun',\n",
       "     'ISO_journal': 'Infect. Immun.',\n",
       "     'authors': ['McGraw WT', 'Potempa J', 'Farley D', 'Travis J.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=10377098'}},\n",
       "   'set_info': {'accession': 'CL0197', 'name': 'GME'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 103,\n",
       "    'interactions': 0,\n",
       "    'matches': 23226,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23032,\n",
       "    'proteomes': 3601,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19935, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 16209},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21967,\n",
       "    'alignment:full': 4271,\n",
       "    'alignment:seed': 359},\n",
       "   'cross_references': {}}},\n",
       " 'PF00925': {'metadata': {'accession': 'PF00925',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032677',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GTP cyclohydrolase II', 'short': 'GTP_cyclohydro2'},\n",
       "   'description': ['<p>GTP cyclohydrolase II catalyses the first committed step in the biosynthesis of riboflavin.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002220': {'PMID': 8320220,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '13',\n",
       "     'year': 1993,\n",
       "     'title': 'Biosynthesis of riboflavin: cloning, sequencing, mapping, and expression of the gene coding for GTP cyclohydrolase II in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4045-51',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Richter G',\n",
       "      'Ritz H',\n",
       "      'Katzenmeier G',\n",
       "      'Volk R',\n",
       "      'Kohnle A',\n",
       "      'Lottspeich F',\n",
       "      'Allendorf D',\n",
       "      'Bacher A.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8320220'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 155,\n",
       "    'interactions': 0,\n",
       "    'matches': 59514,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59181,\n",
       "    'proteomes': 8716,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 51069, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 35783},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54978,\n",
       "    'alignment:full': 12635,\n",
       "    'alignment:seed': 586},\n",
       "   'cross_references': {}}},\n",
       " 'PF06803': {'metadata': {'accession': 'PF06803',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010652',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1232)',\n",
       "    'short': 'DUF1232'},\n",
       "   'description': ['<p>This family represents a conserved region of approximately 60 residues within a number of hypothetical bacterial and archaeal proteins of unknown function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 80,\n",
       "    'interactions': 0,\n",
       "    'matches': 22394,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22392,\n",
       "    'proteomes': 4202,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19124, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 19099},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20499,\n",
       "    'alignment:full': 4563,\n",
       "    'alignment:seed': 332},\n",
       "   'cross_references': {}}},\n",
       " 'PF00617': {'metadata': {'accession': 'PF00617',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001895',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RasGEF domain', 'short': 'RasGEF'},\n",
       "   'description': ['<p>Guanine nucleotide exchange factor for Ras-like small GTPases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000728': {'PMID': 7786285,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '5',\n",
       "     'year': 1995,\n",
       "     'title': 'Guanine nucleotide exchange factors: activators of the Ras superfamily of proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '395-404',\n",
       "     'medline_journal': 'Bioessays',\n",
       "     'ISO_journal': 'Bioessays',\n",
       "     'authors': ['Quilliam LA', 'Khosravi-Far R', 'Huff SY', 'Der CJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/bies.950170507'},\n",
       "    'PUB00004150': {'PMID': 8479541,\n",
       "     'ISBN': None,\n",
       "     'volume': '363',\n",
       "     'issue': '6424',\n",
       "     'year': 1993,\n",
       "     'title': 'Guanine-nucleotide-releasing factor hSos1 binds to Grb2 and links receptor tyrosine kinases to Ras signalling.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '85-8',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Li N',\n",
       "      'Batzer A',\n",
       "      'Daly R',\n",
       "      'Yajnik V',\n",
       "      'Skolnik E',\n",
       "      'Chardin P',\n",
       "      'Bar-Sagi D',\n",
       "      'Margolis B',\n",
       "      'Schlessinger J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/363085a0'},\n",
       "    'PUB00004162': {'PMID': 8259209,\n",
       "     'ISBN': None,\n",
       "     'volume': '366',\n",
       "     'issue': '6456',\n",
       "     'year': 1993,\n",
       "     'title': 'Proteins regulating Ras and its relatives.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '643-54',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Boguski MS', 'McCormick F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/366643a0'},\n",
       "    'PUB00005165': {'PMID': 8316835,\n",
       "     'ISBN': None,\n",
       "     'volume': '260',\n",
       "     'issue': '5116',\n",
       "     'year': 1993,\n",
       "     'title': 'The function of GRB2 in linking the insulin receptor to Ras signaling pathways.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1953-5',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Skolnik EY',\n",
       "      'Batzer A',\n",
       "      'Li N',\n",
       "      'Lee CH',\n",
       "      'Lowenstein E',\n",
       "      'Mohammadi M',\n",
       "      'Margolis B',\n",
       "      'Schlessinger J.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/260/5116/1953'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 746,\n",
       "    'interactions': 0,\n",
       "    'matches': 52647,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52257,\n",
       "    'proteomes': 1893,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28800, 'rosettafold': 0},\n",
       "    'structures': 94,\n",
       "    'taxa': 6887},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37111,\n",
       "    'alignment:full': 24240,\n",
       "    'alignment:seed': 619},\n",
       "   'cross_references': {}}},\n",
       " 'PF00353': {'metadata': {'accession': 'PF00353',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001343',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RTX calcium-binding nonapeptide repeat (4 copies)',\n",
       "    'short': 'HemolysinCabind'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001195': {'PMID': 2303029,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '2',\n",
       "     'year': 1990,\n",
       "     'title': 'The Rhizobium nodulation gene nodO encodes a Ca2(+)-binding protein that is exported without N-terminal cleavage and is homologous to haemolysin and related proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '349-54',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Economou A', 'Hamilton WD', 'Johnston AW', 'Downie JA.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=2303029&action=stream&blobtype=pdf'},\n",
       "    'PUB00001236': {'PMID': 8253063,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '9',\n",
       "     'year': 1993,\n",
       "     'title': 'Three-dimensional structure of the alkaline protease of Pseudomonas aeruginosa: a two-domain protein with a calcium binding parallel beta roll motif.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3357-64',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Baumann U', 'Wu S', 'Flaherty KM', 'McKay DB.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8253063&action=stream&blobtype=pdf'},\n",
       "    'PUB00002161': {'PMID': 1735728,\n",
       "     'ISBN': None,\n",
       "     'volume': '174',\n",
       "     'issue': '4',\n",
       "     'year': 1992,\n",
       "     'title': 'The subtilin gene of Bacillus subtilis ATCC 6633 is encoded in an operon that contains a homolog of the hemolysin B transport protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1417-22',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Chung YJ', 'Steen MT', 'Hansen JN.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=1735728'}},\n",
       "   'set_info': {'accession': 'CL0592', 'name': 'beta_Roll'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 12505,\n",
       "    'interactions': 0,\n",
       "    'matches': 322337,\n",
       "    'pathways': 0,\n",
       "    'proteins': 70077,\n",
       "    'proteomes': 2901,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 44055, 'rosettafold': 0},\n",
       "    'structures': 52,\n",
       "    'taxa': 13826},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 423907,\n",
       "    'alignment:full': 127693,\n",
       "    'alignment:seed': 252},\n",
       "   'cross_references': {}}},\n",
       " 'PF01944': {'metadata': {'accession': 'PF01944',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002798',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Stage II sporulation protein M', 'short': 'SpoIIM'},\n",
       "   'description': ['<p>SpoIIM is on e of four stage II sporulation proteins that is necessary for the forespore inside the mother-cell to be properly internalised through the breakdown of peptidoglycans trapped between the membranes of the septum separating the forespore and the mother-cell. The four proteins working in sequence are SpoIIB, Pfam:PF05036, SpoIIM, SpoIIP, Pfam:PF07454, and finally SpoIID, Pfam:PF08486. D, M and P are in a complex with each other and the complex assembles in a hierarchical manner such that M, which serves as a membrane anchor, recruits P to the septum and P, in turn, recruits D to the septum [PMID:17376078].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 18354,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18349,\n",
       "    'proteomes': 3646,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15522, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 14915},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16361,\n",
       "    'alignment:full': 3758,\n",
       "    'alignment:seed': 356},\n",
       "   'cross_references': {}}},\n",
       " 'PF03788': {'metadata': {'accession': 'PF03788',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005538',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LrgA family', 'short': 'LrgA'},\n",
       "   'description': ['<p>This family is uncharacterised. It contains the protein LrgA that has been hypothesised to export murein hydrolases [PMID:8824633].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008430': {'PMID': 8824633,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '19',\n",
       "     'year': 1996,\n",
       "     'title': 'Identification of LytSR-regulated genes from Staphylococcus aureus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5810-2',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Brunskill EW', 'Bayles KW.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/178/19/5810'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 8,\n",
       "    'interactions': 0,\n",
       "    'matches': 17523,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17523,\n",
       "    'proteomes': 2929,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15970, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13716},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18139,\n",
       "    'alignment:full': 3199,\n",
       "    'alignment:seed': 382},\n",
       "   'cross_references': {}}},\n",
       " 'PF02643': {'metadata': {'accession': 'PF02643',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003795',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterized ACR, COG1430', 'short': 'DUF192'},\n",
       "   'description': ['<p>Two structures have been solved for members of this large (>500 members) family of bacterial proteins present mostly in environmental bacteria and metagenomes (distant homologues are also present in several Plasmodium species).  TOPSAN analysis for pdb:3pjy shows that there is much similarity with the other solved structure, pdb:3m7a, solved for UniProt:Q2GA55 (Saro_0823), a homologue of Thermotoga maritima TM1668, UniProt:Q9X1Z6., The homologue in Caulobacter crescentus (CC1388), UniProt:Q9A8G6, is associated with CspD, a cold shock protein (CC1387), UniProt:Q9A8G7. However, the genomic context of UniProt:Q2GA55 is most conserved with a putative xylose isomerase, suggesting a possible role in extracellular sugar processing. Saro_0821, UniProt:Q2GA57, is annotated as an AMP-dependent synthetase and ligase. PDB:3m7a structure corresponds to the C-terminal (27-165) fragment of the YP_496102 (Saro_0823) protein and it is structurally unique, as the best hits from Dali have a Z-score of 3.8 (1nt0, 2j1t, 3kq4) and it is thus a likely candidate for a new fold. Interestingly, many of the top Dali hits are involved in sugar metabolism. There are no obvious active site-like cavities on the protein surface of 3m7a (http://www.topsan.org/Proteins/JCSG/).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 86,\n",
       "    'interactions': 0,\n",
       "    'matches': 22554,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22541,\n",
       "    'proteomes': 3614,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18745, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 17205},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19181,\n",
       "    'alignment:full': 3724,\n",
       "    'alignment:seed': 328},\n",
       "   'cross_references': {}}},\n",
       " 'PF02424': {'metadata': {'accession': 'PF02424',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024932',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ApbE family', 'short': 'ApbE'},\n",
       "   'description': ['<p>This prokaryotic family of lipoproteins are related to ApbE from Salmonella typhimurium. ApbE is involved in thiamine synthesis [PMID:9473043].  It acts as an FAD:protein FMN-transferase, catalysing the attachment of an FMN residue to a threonine residue of a protein via a phosphoester bond in such bacterial flavoproteins [PMID:23558683].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009525': {'PMID': 9473043,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '4',\n",
       "     'year': 1998,\n",
       "     'title': 'The apbE gene encodes a lipoprotein involved in thiamine synthesis in Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '885-91',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Beck BJ', 'Downs DM.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9473043&action=stream&blobtype=pdf'},\n",
       "    'PUB00019151': {'PMID': 10572132,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '23',\n",
       "     'year': 1999,\n",
       "     'title': 'A periplasmic location is essential for the role of the ApbE lipoprotein in thiamine synthesis in Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7285-90',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Beck BJ', 'Downs DM.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=10572132'},\n",
       "    'PUB00037979': {'PMID': 16779835,\n",
       "     'ISBN': None,\n",
       "     'volume': '64',\n",
       "     'issue': '4',\n",
       "     'year': 2006,\n",
       "     'title': 'Crystal structure of the ApbE protein (TM1553) from Thermotoga maritima at 1.58 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1083-90',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Han GW',\n",
       "      'Sri Krishna S',\n",
       "      'Schwarzenbacher R',\n",
       "      'McMullan D',\n",
       "      'Ginalski K',\n",
       "      'Elsliger MA',\n",
       "      'Brittain SM',\n",
       "      'Abdubek P',\n",
       "      'Agarwalla S',\n",
       "      'Ambing E',\n",
       "      'Astakhova T',\n",
       "      'Axelrod H',\n",
       "      'Canaves JM',\n",
       "      'Chiu HJ',\n",
       "      'DiDonato M',\n",
       "      'Grzechnik SK',\n",
       "      'Hale J',\n",
       "      'Hampton E',\n",
       "      'Haugen J',\n",
       "      'Jaroszewski L',\n",
       "      'Jin KK',\n",
       "      'Klock HE',\n",
       "      'Knuth MW',\n",
       "      'Koesema E',\n",
       "      'Kreusch A',\n",
       "      'Kuhn P',\n",
       "      'Miller MD',\n",
       "      'Morse AT',\n",
       "      'Moy K',\n",
       "      'Nigoghossian E',\n",
       "      'Oommachen S',\n",
       "      'Ouyang J',\n",
       "      'Paulsen J',\n",
       "      'Quijano K',\n",
       "      'Reyes R',\n",
       "      'Rife C',\n",
       "      'Spraggon G',\n",
       "      'Stevens RC',\n",
       "      'van den Bedem H',\n",
       "      'Velasquez J',\n",
       "      'Wang X',\n",
       "      'West B',\n",
       "      'White A',\n",
       "      'Wolf G',\n",
       "      'Xu Q',\n",
       "      'Hodgson KO',\n",
       "      'Wooley J',\n",
       "      'Deacon AM',\n",
       "      'Godzik A',\n",
       "      'Lesley SA',\n",
       "      'Wilson IA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.20950'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 89,\n",
       "    'interactions': 0,\n",
       "    'matches': 43878,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41517,\n",
       "    'proteomes': 5589,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35102, 'rosettafold': 0},\n",
       "    'structures': 23,\n",
       "    'taxa': 23835},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39921,\n",
       "    'alignment:full': 8156,\n",
       "    'alignment:seed': 683},\n",
       "   'cross_references': {}}},\n",
       " 'PF01641': {'metadata': {'accession': 'PF01641',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002579',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SelR domain', 'short': 'SelR'},\n",
       "   'description': ['<p>Methionine sulfoxide reduction is an important process, by which cells regulate biological processes and cope with oxidative stress. MsrA, a protein involved in the reduction of methionine sulfoxides in proteins, has been known for four decades and has been extensively characterised with respect to structure and function. However, recent studies revealed that MsrA is only specific for methionine-S-sulfoxides. Because oxidised methionines occur in a mixture of R and S isomers in vivo, it was unclear how stereo-specific MsrA could be responsible for the reduction of all protein methionine sulfoxides. It appears that a second methionine sulfoxide reductase, SelR , evolved that is specific for methionine-R-sulfoxides, the activity that is different but complementary to that of MsrA. Thus, these proteins, working together, could reduce both stereoisomers of methionine sulfoxide. This domain is found both in SelR proteins and fused with the peptide methionine sulfoxide reductase enzymatic domain Pfam:PF01625. The domain has two conserved cysteine and histidines. The domain binds both selenium and zinc [PMID:11929995]. The final cysteine is found to be replaced by the rare amino acid selenocysteine in some members of the family [PMID:10608886]. This family has methionine-R-sulfoxide reductase activity [PMID:11929995].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006463': {'PMID': 10608886,\n",
       "     'ISBN': None,\n",
       "     'volume': '274',\n",
       "     'issue': '53',\n",
       "     'year': 1999,\n",
       "     'title': 'Novel selenoproteins identified in silico and in vivo by using a conserved RNA structural motif.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '38147-54',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Lescure A', 'Gautheret D', 'Carbon P', 'Krol A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.274.53.38147'},\n",
       "    'PUB00020323': {'PMID': 11929995,\n",
       "     'ISBN': None,\n",
       "     'volume': '99',\n",
       "     'issue': '7',\n",
       "     'year': 2002,\n",
       "     'title': 'Selenoprotein R is a zinc-containing stereo-specific methionine sulfoxide reductase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4245-50',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Kryukov GV', 'Kumar RA', 'Koc A', 'Sun Z', 'Gladyshev VN.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.072603099'}},\n",
       "   'set_info': {'accession': 'CL0080', 'name': 'Beta-tent'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 200,\n",
       "    'interactions': 0,\n",
       "    'matches': 58468,\n",
       "    'pathways': 0,\n",
       "    'proteins': 58391,\n",
       "    'proteomes': 9643,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 49836, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 39931},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52441,\n",
       "    'alignment:full': 12736,\n",
       "    'alignment:seed': 805},\n",
       "   'cross_references': {}}},\n",
       " 'PF02817': {'metadata': {'accession': 'PF02817',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004167',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'e3 binding domain', 'short': 'E3_binding'},\n",
       "   'description': ['<p>This family represents a small domain of the E2 subunit of 2-oxo-acid dehydrogenases responsible for the binding of the E3 subunit.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019486': {'PMID': 8805537,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Protein-protein interactions in the pyruvate dehydrogenase multienzyme complex: dihydrolipoamide dehydrogenase complexed with the binding domain of dihydrolipoamide acetyltransferase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '277-86',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Mande SS', 'Sarfaty S', 'Allen MD', 'Perham RN', 'Hol WG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(96)00032-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 296,\n",
       "    'interactions': 0,\n",
       "    'matches': 99967,\n",
       "    'pathways': 0,\n",
       "    'proteins': 96751,\n",
       "    'proteomes': 9351,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 82643, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 37916},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 92444,\n",
       "    'alignment:full': 22592,\n",
       "    'alignment:seed': 808},\n",
       "   'cross_references': {}}},\n",
       " 'PF09719': {'metadata': {'accession': 'PF09719',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010181',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative redox-active protein (C_GCAxxG_C_C)',\n",
       "    'short': 'C_GCAxxG_C_C'},\n",
       "   'description': ['<p>This entry represents a putative redox-active protein of about 140 residues, with four perfectly conserved Cys residues. It includes a CGAXXG motif. Most members are found within one or two loci of transporter or oxidoreductase genes. A member from Geobacter sulfurreducens, located in a molybdenum transporter operon, has a TAT (twin-arginine translocation) signal sequence for Sec-independent transport across the plasma membrane, a hallmark of bound prosthetic groups such as FeS clusters.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0317', 'name': 'Multiheme_cytos'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 49,\n",
       "    'interactions': 0,\n",
       "    'matches': 9263,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9128,\n",
       "    'proteomes': 1174,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7695, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 5566},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7851,\n",
       "    'alignment:full': 1790,\n",
       "    'alignment:seed': 334},\n",
       "   'cross_references': {}}},\n",
       " 'PF02880': {'metadata': {'accession': 'PF02880',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005846',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoglucomutase/phosphomannomutase, alpha/beta/alpha domain III',\n",
       "    'short': 'PGM_PMM_III'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002760': {'PMID': 1532581,\n",
       "     'ISBN': None,\n",
       "     'volume': '267',\n",
       "     'issue': '9',\n",
       "     'year': 1992,\n",
       "     'title': 'The crystal structure of muscle phosphoglucomutase refined at 2.7-angstrom resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6322-37',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Dai JB', 'Liu Y', 'Ray WJ Jr', 'Konno M.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/267/9/6322.pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 342,\n",
       "    'interactions': 0,\n",
       "    'matches': 136812,\n",
       "    'pathways': 0,\n",
       "    'proteins': 136775,\n",
       "    'proteomes': 11082,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 116875, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 46857},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 124771,\n",
       "    'alignment:full': 27599,\n",
       "    'alignment:seed': 374},\n",
       "   'cross_references': {}}},\n",
       " 'PF02465': {'metadata': {'accession': 'PF02465',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003481',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flagellar hook-associated protein 2 N-terminus',\n",
       "    'short': 'FliD_N'},\n",
       "   'description': ['<p>The flagellar hook-associated protein 2 (HAP2 or FliD) forms the distal end of the flagella, and plays a role in mucin specific adhesion of the bacteria [PMID:9488388]. This alignment covers the N-terminal region of this family of proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008200': {'PMID': 8195064,\n",
       "     'ISBN': None,\n",
       "     'volume': '176',\n",
       "     'issue': '11',\n",
       "     'year': 1994,\n",
       "     'title': 'The Bacillus subtilis sigma D-dependent operon encoding the flagellar proteins FliD, FliS, and FliT.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3093-101',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Chen L', 'Helmann JD.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=8195064'},\n",
       "    'PUB00009465': {'PMID': 9488388,\n",
       "     'ISBN': None,\n",
       "     'volume': '66',\n",
       "     'issue': '3',\n",
       "     'year': 1998,\n",
       "     'title': 'The Pseudomonas aeruginosa flagellar cap protein, FliD, is responsible for mucin adhesion.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1000-7',\n",
       "     'medline_journal': 'Infect Immun',\n",
       "     'ISO_journal': 'Infect. Immun.',\n",
       "     'authors': ['Arora SK',\n",
       "      'Ritchings BW',\n",
       "      'Almira EC',\n",
       "      'Lory S',\n",
       "      'Ramphal R.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9488388&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 38,\n",
       "    'interactions': 0,\n",
       "    'matches': 18348,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18348,\n",
       "    'proteomes': 3249,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16038, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 14546},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18083,\n",
       "    'alignment:full': 3321,\n",
       "    'alignment:seed': 408},\n",
       "   'cross_references': {}}},\n",
       " 'PF07873': {'metadata': {'accession': 'PF07873',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022476',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YabP family', 'short': 'YabP'},\n",
       "   'description': ['<p>This family of proteins is involved in spore coat assembly during the process of sporulation [PMID:15231775].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016405': {'PMID': 15231775,\n",
       "     'ISBN': None,\n",
       "     'volume': '186',\n",
       "     'issue': '14',\n",
       "     'year': 2004,\n",
       "     'title': 'Dynamic patterns of subcellular protein localization during spore coat morphogenesis in Bacillus subtilis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4441-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['van Ooij C', 'Eichenberger P', 'Losick R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.186.14.4441-4448.2004'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1,\n",
       "    'interactions': 0,\n",
       "    'matches': 8033,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8033,\n",
       "    'proteomes': 1212,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6884, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 3949},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7387,\n",
       "    'alignment:full': 2243,\n",
       "    'alignment:seed': 376},\n",
       "   'cross_references': {}}},\n",
       " 'PF10590': {'metadata': {'accession': 'PF10590',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019576',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"Pyridoxine 5'-phosphate oxidase C-terminal dimerisation region\",\n",
       "    'short': 'PNP_phzG_C'},\n",
       "   'description': ['<p>This domain represents one of the two dimerisation regions of the protein, located at the edge of the dimer interface, at the C-terminus, being the last three beta strands, S6, S7, and S8 along with the last three residues to the end. In Swiss:P21159, S6 runs from residues 178-192, S7 from 200-206 and S8 from 211-215. the extended loop, of residues 167-177 may well be involved in the pocket formed between the two dimers that positions the FMN molecule [PMID:10903950].To date, the only time functional oxidase or phenazine biosynthesis activities have been experimentally demonstrated is when the sequences contain both Pfam:PF01243 and Pfam:PF10590. It is unknown the role performed by each domain in bringing about molecular functions of either oxidase or phenazine activity [PMID:26327315].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016343': {'PMID': 12686112,\n",
       "     'ISBN': None,\n",
       "     'volume': '1647',\n",
       "     'issue': '1-2',\n",
       "     'year': 2003,\n",
       "     'title': \"Structure and mechanism of Escherichia coli pyridoxine 5'-phosphate oxidase.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '76-82',\n",
       "     'medline_journal': 'Biochim Biophys Acta',\n",
       "     'ISO_journal': 'Biochim. Biophys. Acta',\n",
       "     'authors': ['di Salvo ML',\n",
       "      'Safo MK',\n",
       "      'Musayev FN',\n",
       "      'Bossa F',\n",
       "      'Schirch V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1570-9639(03)00060-8'},\n",
       "    'PUB00024258': {'PMID': 10903950,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '7',\n",
       "     'year': 2000,\n",
       "     'title': \"X-ray structure of Escherichia coli pyridoxine 5'-phosphate oxidase complexed with FMN at 1.8 A resolution.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '751-62',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Safo MK',\n",
       "      'Mathews I',\n",
       "      'Musayev FN',\n",
       "      'di Salvo ML',\n",
       "      'Thiel DJ',\n",
       "      'Abraham DJ',\n",
       "      'Schirch V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(00)00162-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 80,\n",
       "    'interactions': 0,\n",
       "    'matches': 30448,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30447,\n",
       "    'proteomes': 6930,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26104, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 27695},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28042,\n",
       "    'alignment:full': 7007,\n",
       "    'alignment:seed': 478},\n",
       "   'cross_references': {}}},\n",
       " 'PF00447': {'metadata': {'accession': 'PF00447',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000232',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HSF-type DNA-binding', 'short': 'HSF_DNA-bind'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 299,\n",
       "    'interactions': 0,\n",
       "    'matches': 24090,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23925,\n",
       "    'proteomes': 2084,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19601, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 7741},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19822,\n",
       "    'alignment:full': 11944,\n",
       "    'alignment:seed': 357},\n",
       "   'cross_references': {}}},\n",
       " 'PF02152': {'metadata': {'accession': 'PF02152',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006157',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dihydroneopterin aldolase', 'short': 'FolB'},\n",
       "   'description': ['<p>This enzyme EC:4.1.2.25 catalyses the conversion of 7,8-dihydroneopterin to 6-hydroxymethyl-7,8-dihydropterin in the biosynthetic pathway of tetrahydrofolate.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007944': {'PMID': 9651328,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '28',\n",
       "     'year': 1998,\n",
       "     'title': 'Biosynthesis of pteridines in Escherichia coli. Structural and mechanistic similarity of dihydroneopterin-triphosphate epimerase and dihydroneopterin aldolase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17418-24',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Haussmann C',\n",
       "      'Rohdich F',\n",
       "      'Schmidt E',\n",
       "      'Bacher A',\n",
       "      'Richter G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.28.17418'}},\n",
       "   'set_info': {'accession': 'CL0334', 'name': 'THBO-biosyn'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 106,\n",
       "    'interactions': 0,\n",
       "    'matches': 40986,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40471,\n",
       "    'proteomes': 8286,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 35226, 'rosettafold': 0},\n",
       "    'structures': 28,\n",
       "    'taxa': 32934},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38352,\n",
       "    'alignment:full': 8750,\n",
       "    'alignment:seed': 791},\n",
       "   'cross_references': {}}},\n",
       " 'PF00115': {'metadata': {'accession': 'PF00115',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000883',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cytochrome C and Quinol oxidase polypeptide I',\n",
       "    'short': 'COX1'},\n",
       "   'description': ['<p>Cytochrome c oxidase (E.C:7.1.1.9) is a key enzyme in aerobic metabolism. Proton pumping haem-copper oxidases represent the terminal, energy-transfer enzymes of respiratory chains in prokaryotes and eukaryotes. The CuB-haem a3 (or haem o) binuclear centre, associated with the largest subunit I of cytochrome c and ubiquinol oxidases (E.C:1.10.3.11), is directly involved in the coupling between dioxygen reduction and proton pumping [PMID:8638158, PMID:8013452, PMID:31489376, PMID:8049679]. Some terminal oxidases generate a transmembrane proton gradient across the plasma membrane (prokaryotes) or the mitochondrial inner membrane (eukaryotes). The enzyme complex consists of 3-4 subunits (prokaryotes) up to 13 polypeptides (mammals) of which only the catalytic subunit (equivalent to mammalian subunit I (COXI) is found in all haem-copper respiratory oxidases. The presence of a bimetallic centre (formed by a high-spin haem and copper B) as well as a low-spin haem, both ligated to six conserved histidine residues near the outer side of four transmembrane spans within CO I is common to all family members [PMID:8013452, PMID:31489376].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005218': {'PMID': 8638158,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '5265',\n",
       "     'year': 1996,\n",
       "     'title': 'The whole structure of the 13-subunit oxidized cytochrome c oxidase at 2.8 A.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1136-44',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Tsukihara T',\n",
       "      'Aoyama H',\n",
       "      'Yamashita E',\n",
       "      'Tomizaki T',\n",
       "      'Yamaguchi H',\n",
       "      'Shinzawa-Itoh K',\n",
       "      'Nakashima R',\n",
       "      'Yaono R',\n",
       "      'Yoshikawa S.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/272/5265/1136'}},\n",
       "   'set_info': {'accession': 'CL0714', 'name': 'COX1-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 174,\n",
       "    'interactions': 0,\n",
       "    'matches': 1799830,\n",
       "    'pathways': 0,\n",
       "    'proteins': 1798079,\n",
       "    'proteomes': 7364,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1548528, 'rosettafold': 0},\n",
       "    'structures': 193,\n",
       "    'taxa': 859486},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 1718008,\n",
       "    'alignment:full': 16920,\n",
       "    'alignment:seed': 619},\n",
       "   'cross_references': {}}},\n",
       " 'PF03862': {'metadata': {'accession': 'PF03862',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005562',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SpoVAC/SpoVAEB sporulation membrane protein',\n",
       "    'short': 'SpoVAC_SpoVAEB'},\n",
       "   'description': ['<p>Members of this family are all transcribed from the spoVA operon [PMID:11751839]. Bacillus and Clostridium are two well studied endospore forming bacteria. Spore formation provides a resistance mechanism in response to extreme or unfavourable environmental conditions such as heat, radiation, and chemical agents or nutrient deprivation.  The reverse process termed germination takes place where spores develop into growing cells in response to nutrient availability or stress reduction. Nutrient germinant receptors (GRs) and the SpoVA proteins are important players in the germination process. In B.subtilis the SpoVAC and SpoVAEB, belonging to this domain family, are predicted to be membrane proteins, with two to five membrane spanning. Biophysical and biochemical studies suggest that SpoVAC acts as a mechano-sensitive channel with properties that would allow the release of Ca-DPA (dipicolinic acid) and amino acids during germination of the spore. The release of Ca-DPA is a crucial event during spore germination. When expressed in E.coli SpoVAC provides protection against osmotic downshift. Furthermore, SpoVAC acts as channel that facilitates the efflux down the concentration gradient of osmolytes up to a mass of at least 600 Da [PMID:24666282]. Another conserved SpoVA protein in all spore-forming bacteria is SpoVAEb, which appears to be an integral membrane protein with no known function [PMID:27044622].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008450': {'PMID': 11751839,\n",
       "     'ISBN': None,\n",
       "     'volume': '184',\n",
       "     'issue': '2',\n",
       "     'year': 2002,\n",
       "     'title': 'The products of the spoVA operon are involved in dipicolinic acid uptake into developing spores of Bacillus subtilis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '584-7',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Tovar-Rojo F', 'Chander M', 'Setlow B', 'Setlow P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.184.2.584-587.2002'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 10,\n",
       "    'interactions': 0,\n",
       "    'matches': 9983,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9972,\n",
       "    'proteomes': 1217,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8689, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 4008},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9491,\n",
       "    'alignment:full': 2659,\n",
       "    'alignment:seed': 328},\n",
       "   'cross_references': {}}},\n",
       " 'PF03449': {'metadata': {'accession': 'PF03449',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022691',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transcription elongation factor, N-terminal',\n",
       "    'short': 'GreA_GreB_N'},\n",
       "   'description': ['<p>This domain adopts a long alpha-hairpin structure.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004205': {'PMID': 7854424,\n",
       "     'ISBN': None,\n",
       "     'volume': '373',\n",
       "     'issue': '6515',\n",
       "     'year': 1995,\n",
       "     'title': 'Crystal structure of the GreA transcript cleavage factor from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '636-40',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Stebbins CE',\n",
       "      'Borukhov S',\n",
       "      'Orlova M',\n",
       "      'Polyakov A',\n",
       "      'Goldfarb A',\n",
       "      'Darst SA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/373636a0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 60,\n",
       "    'interactions': 0,\n",
       "    'matches': 55732,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55728,\n",
       "    'proteomes': 8476,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 48061, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 35760},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50940,\n",
       "    'alignment:full': 10321,\n",
       "    'alignment:seed': 537},\n",
       "   'cross_references': {}}},\n",
       " 'PF03626': {'metadata': {'accession': 'PF03626',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005171',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Prokaryotic Cytochrome C oxidase subunit IV',\n",
       "    'short': 'COX4_pro'},\n",
       "   'description': ['<p>Cytochrome c oxidase (COX) is a multi-subunit enzyme complex that catalyses the final step of electron transfer through the respiratory chain on the mitochondrial inner membrane. This family is composed of cytochrome c oxidase subunit 4 from prokaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005218': {'PMID': 8638158,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '5265',\n",
       "     'year': 1996,\n",
       "     'title': 'The whole structure of the 13-subunit oxidized cytochrome c oxidase at 2.8 A.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1136-44',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Tsukihara T',\n",
       "      'Aoyama H',\n",
       "      'Yamashita E',\n",
       "      'Tomizaki T',\n",
       "      'Yamaguchi H',\n",
       "      'Shinzawa-Itoh K',\n",
       "      'Nakashima R',\n",
       "      'Yaono R',\n",
       "      'Yoshikawa S.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/272/5265/1136'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 26,\n",
       "    'interactions': 0,\n",
       "    'matches': 24471,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24471,\n",
       "    'proteomes': 3369,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21419, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 15980},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23348,\n",
       "    'alignment:full': 4163,\n",
       "    'alignment:seed': 331},\n",
       "   'cross_references': {}}},\n",
       " 'PF02401': {'metadata': {'accession': 'PF02401',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003451',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LytB protein', 'short': 'LYTB'},\n",
       "   'description': ['<p>The mevalonate-independent 2-C-methyl-D-erythritol 4-phosphate (MEP) pathway for isoprenoid biosynthesis is essential in many eubacteria, plants, and the malaria parasite. The LytB gene is involved in the trunk line of the MEP pathway.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007255': {'PMID': 11004185,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '20',\n",
       "     'year': 2000,\n",
       "     'title': 'Evidence of a role for LytB in the nonmevalonate pathway of isoprenoid biosynthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5841-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Cunningham FX Jr', 'Lafond TP', 'Gantt E.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.20.5841-5848.2000'},\n",
       "    'PUB00007256': {'PMID': 11818558,\n",
       "     'ISBN': None,\n",
       "     'volume': '99',\n",
       "     'issue': '3',\n",
       "     'year': 2002,\n",
       "     'title': 'Studies on the nonmevalonate terpene biosynthetic pathway: metabolic role of IspH (LytB) protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1158-63',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Rohdich F',\n",
       "      'Hecht S',\n",
       "      'Gartner K',\n",
       "      'Adam P',\n",
       "      'Krieger C',\n",
       "      'Amslinger S',\n",
       "      'Arigoni D',\n",
       "      'Bacher A',\n",
       "      'Eisenreich W.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.032658999'},\n",
       "    'PUB00019821': {'PMID': 8432714,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '4',\n",
       "     'year': 1993,\n",
       "     'title': 'Identification of the Escherichia coli lytB gene, which is involved in penicillin tolerance and control of the stringent response.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1203-5',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Gustafson CE', 'Kaul S', 'Ishiguro EE.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/175/4/1203'},\n",
       "    'PUB00019822': {'PMID': 11741609,\n",
       "     'ISBN': None,\n",
       "     'volume': '509',\n",
       "     'issue': '2',\n",
       "     'year': 2001,\n",
       "     'title': 'Identification of (E)-4-hydroxy-3-methyl-but-2-enyl pyrophosphate as a major activator for human gammadelta T cells in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '317-22',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Hintz M',\n",
       "      'Reichenberg A',\n",
       "      'Altincicek B',\n",
       "      'Bahr U',\n",
       "      'Gschwind RM',\n",
       "      'Kollas AK',\n",
       "      'Beck E',\n",
       "      'Wiesner J',\n",
       "      'Eberl M',\n",
       "      'Jomaa H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(01)03191-X'},\n",
       "    'PUB00019823': {'PMID': 11717301,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '24',\n",
       "     'year': 2001,\n",
       "     'title': 'The lytB gene of Escherichia coli is essential and specifies a product needed for isoprenoid biosynthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7403-7',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['McAteer S', 'Coulson A', 'McLennan N', 'Masters M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.24.7403-7407.2001'},\n",
       "    'PUB00019824': {'PMID': 11418107,\n",
       "     'ISBN': None,\n",
       "     'volume': '499',\n",
       "     'issue': '1-2',\n",
       "     'year': 2001,\n",
       "     'title': 'LytB, a novel gene of the 2-C-methyl-D-erythritol 4-phosphate pathway of isoprenoid biosynthesis in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '37-40',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Altincicek B',\n",
       "      'Kollas A',\n",
       "      'Eberl M',\n",
       "      'Wiesner J',\n",
       "      'Sanderbrand S',\n",
       "      'Hintz M',\n",
       "      'Beck E',\n",
       "      'Jomaa H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(01)02516-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 83,\n",
       "    'interactions': 0,\n",
       "    'matches': 41830,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41704,\n",
       "    'proteomes': 7796,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35593, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 32410},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38039,\n",
       "    'alignment:full': 7944,\n",
       "    'alignment:seed': 615},\n",
       "   'cross_references': {}}},\n",
       " 'PF14420': {'metadata': {'accession': 'PF14420',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025676',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Clr5 domain', 'short': 'Clr5'},\n",
       "   'description': ['<p>This domain is found at the N-terminus of the Clr5 protein which has been shown to be involved in silencing in fission yeast. This domain has been found to often be associated with proteins that contain ankyrin repeats and large regions of disordered sequence [PMID:21253571].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 251,\n",
       "    'interactions': 0,\n",
       "    'matches': 5764,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5759,\n",
       "    'proteomes': 487,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4624, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1435},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4774,\n",
       "    'alignment:full': 3068,\n",
       "    'alignment:seed': 248},\n",
       "   'cross_references': {}}},\n",
       " 'PF13802': {'metadata': {'accession': 'PF13802',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025887',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Galactose mutarotase-like', 'short': 'Gal_mutarotas_2'},\n",
       "   'description': ['<p>This family is found N-terminal to glycosyl-hydrolase domains, and appears to be similar to the galactose mutarotase superfamily.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 441,\n",
       "    'interactions': 0,\n",
       "    'matches': 32239,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31887,\n",
       "    'proteomes': 4756,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25470, 'rosettafold': 0},\n",
       "    'structures': 83,\n",
       "    'taxa': 19105},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27779,\n",
       "    'alignment:full': 10844,\n",
       "    'alignment:seed': 313},\n",
       "   'cross_references': {}}},\n",
       " 'PF09369': {'metadata': {'accession': 'PF09369',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018973',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MrfA Zn-binding domain', 'short': 'MZB'},\n",
       "   'description': ['<p>This is the C-terminal MrfA Zn+2-binding domain (MZB, also referred to as DUF1998) which contains a conserved four-cysteine signature motif.  These four Cys reside in a short coil between two alpha-helices and form a metal ion-binding site [PMID:33300032]. This domain is frequently found at the C-terminal of ndNTPases, however, it is also found encoded in a standalone gene, downstream of putative helicase domain-encoding genes associated with bacterial anti-phage defense system DISARM. MrfA (Mitomycin repair factor A, also known as YprA in Bacillus subtilis) is a DNA helicase that supports repair of mitomycin C-induced DNA damage. MrfA homologues are widely distributed in bacteria and are also present in archaea, fungi and plants. The MrfA-homologue in yeast, Hrq1, also reduces mitomycin C sensitivity. Hrq1 has high similarity to human RecQ4 and was therefore assigned to the RecQ-like helicase family. MrfA homologues appear to be missing in Enterobacteria, however, certain pathogenic Escherichia coli and Salmonella strains harbour Z5898-like helicases with this domain [PMID:33300032].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 103,\n",
       "    'interactions': 0,\n",
       "    'matches': 19046,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19046,\n",
       "    'proteomes': 4093,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12810, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 16626},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17024,\n",
       "    'alignment:full': 4668,\n",
       "    'alignment:seed': 381},\n",
       "   'cross_references': {}}},\n",
       " 'PF03747': {'metadata': {'accession': 'PF03747',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005502',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ADP-ribosylglycohydrolase', 'short': 'ADP_ribosyl_GH'},\n",
       "   'description': ['<p>This family includes enzymes that ADP-ribosylations, for example ADP-ribosylarginine hydrolase EC:3.2.2.19 cleaves ADP-ribose-L-arginine [PMID:8349667]. The family also includes  dinitrogenase reductase activating glycohydrolase [PMID:2506427]. Most surprisingly the family also includes jellyfish crystallins [PMID:2506427], these proteins appear to have lost the presumed active site residues.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008412': {'PMID': 8349667,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '24',\n",
       "     'year': 1993,\n",
       "     'title': 'Cloning and site-directed mutagenesis of human ADP-ribosylarginine hydrolase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17837-43',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Takada T', 'Iida K', 'Moss J.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/268/24/17837'},\n",
       "    'PUB00008413': {'PMID': 2506427,\n",
       "     'ISBN': None,\n",
       "     'volume': '218',\n",
       "     'issue': '2',\n",
       "     'year': 1989,\n",
       "     'title': 'Genes coding for the reversible ADP-ribosylation system of dinitrogenase reductase from Rhodospirillum rubrum.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '340-7',\n",
       "     'medline_journal': 'Mol Gen Genet',\n",
       "     'ISO_journal': 'Mol. Gen. Genet.',\n",
       "     'authors': ['Fitzmaurice WP',\n",
       "      'Saari LL',\n",
       "      'Lowery RG',\n",
       "      'Ludden PW',\n",
       "      'Roberts GP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/BF00331287'},\n",
       "    'PUB00015931': {'PMID': 8505315,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '16',\n",
       "     'year': 1993,\n",
       "     'title': 'J1-crystallins of the cubomedusan jellyfish lens constitute a novel family encoded in at least three intronless genes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '11894-901',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Piatigorsky J', 'Horwitz J', 'Norman BL.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/268/16/11894.pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 392,\n",
       "    'interactions': 0,\n",
       "    'matches': 47090,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46310,\n",
       "    'proteomes': 5275,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38889, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 23250},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39967,\n",
       "    'alignment:full': 10354,\n",
       "    'alignment:seed': 641},\n",
       "   'cross_references': {}}},\n",
       " 'PF03119': {'metadata': {'accession': 'PF03119',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004149',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NAD-dependent DNA ligase C4 zinc finger domain',\n",
       "    'short': 'DNA_ligase_ZBD'},\n",
       "   'description': ['<p>DNA ligases catalyse the crucial step of joining the breaks in duplex DNA during DNA replication, repair and recombination, utilising either ATP or NAD(+) as a cofactor [PMID:10698952]. This family is a small zinc binding motif that is presumably DNA binding [PMID:10698952]. IT is found only in NAD dependent DNA ligases [PMID:10698952].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007386': {'PMID': 10698952,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '5',\n",
       "     'year': 2000,\n",
       "     'title': 'Crystal structure of NAD(+)-dependent DNA ligase: modular architecture and functional implications.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1119-29',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Lee JY',\n",
       "      'Chang C',\n",
       "      'Song HK',\n",
       "      'Moon J',\n",
       "      'Yang JK',\n",
       "      'Kim HK',\n",
       "      'Kwon ST',\n",
       "      'Suh SW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/19.5.1119'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 91,\n",
       "    'interactions': 0,\n",
       "    'matches': 37552,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37552,\n",
       "    'proteomes': 7336,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32294, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 30957},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35031,\n",
       "    'alignment:full': 6735,\n",
       "    'alignment:seed': 419},\n",
       "   'cross_references': {}}},\n",
       " 'PF00376': {'metadata': {'accession': 'PF00376',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000551',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MerR family regulatory protein', 'short': 'MerR'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 155,\n",
       "    'interactions': 0,\n",
       "    'matches': 49683,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49555,\n",
       "    'proteomes': 4806,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 44121, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 21033},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48407,\n",
       "    'alignment:full': 11444,\n",
       "    'alignment:seed': 293},\n",
       "   'cross_references': {}}},\n",
       " 'PF03381': {'metadata': {'accession': 'PF03381',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005045',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LEM3 (ligand-effect modulator 3) family / CDC50 family',\n",
       "    'short': 'CDC50'},\n",
       "   'description': ['<p>Members of this family have been predicted to contain transmembrane helices. The family member LEM3 (Swiss:P42838) is a ligand-effect modulator, mutation of which increases glucocorticoid receptor activity in response to dexamethasone and also confers increased activity on other intracellular receptors including the progesterone, oestrogen and mineralocorticoid receptors. LEM3 is thought to affect a downstream step in the glucocorticoid receptor pathway. Factors that modulate ligand responsiveness are likely to contribute to the context-specific actions of the glucocorticoid receptor in mammalian cells [PMID:11063677]. The products of genes YNR048w (Swiss:P53740), YNL323w (Swiss:P42838) and YCR094w (Swiss:P25656) (CDC50) show redundancy of function and are involved in regulation of transcription via CDC39 [PMID:11180453]. CDC39 (also known as NOT1) is normally a negative regulator of transcription either by affecting the general RNA polymerase II machinery or by altering chromatin structure [PMID:8428577]. One function of CDC39 is to block activation of the mating response pathway in the absence of pheromone, and mutation causes arrest in G1 by activation of the pathway [PMID:2099190]. It may be that the cold-sensitive arrest in G1 noticed in CDC50 mutants [PMID:11180453] may be due to inactivation of CDC39.  The effects of LEM3 on glucocorticoid receptor activity may also be due to effects on transcription via CDC39.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019272': {'PMID': 11063677,\n",
       "     'ISBN': None,\n",
       "     'volume': '156',\n",
       "     'issue': '3',\n",
       "     'year': 2000,\n",
       "     'title': 'A genetic analysis of glucocorticoid receptor signaling. Identification and characterization of ligand-effect modulators in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '963-72',\n",
       "     'medline_journal': 'Genetics',\n",
       "     'ISO_journal': 'Genetics',\n",
       "     'authors': ['Sitcheran R', 'Emter R', 'Kralli A', 'Yamamoto KR.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=11063677'},\n",
       "    'PUB00019273': {'PMID': 11180453,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '3',\n",
       "     'year': 2001,\n",
       "     'title': 'The cloning and characterization of the CDC50 gene family in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '195-205',\n",
       "     'medline_journal': 'Yeast',\n",
       "     'ISO_journal': 'Yeast',\n",
       "     'authors': ['Radji M',\n",
       "      'Kim JM',\n",
       "      'Togan T',\n",
       "      'Yoshikawa H',\n",
       "      'Shirahige K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/1097-0061(200102)18:3<195::AID-YEA660>3.0.CO;2-L'},\n",
       "    'PUB00019274': {'PMID': 2099190,\n",
       "     'ISBN': None,\n",
       "     'volume': '1',\n",
       "     'issue': '5',\n",
       "     'year': 1990,\n",
       "     'title': 'CDC36 and CDC39 are negative elements in the signal transduction pathway of yeast.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '391-401',\n",
       "     'medline_journal': 'Cell Regul',\n",
       "     'ISO_journal': 'Cell Regul.',\n",
       "     'authors': ['Neiman AM', 'Chang F', 'Komachi K', 'Herskowitz I.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=2099190&action=stream&blobtype=pdf'},\n",
       "    'PUB00019275': {'PMID': 8428577,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '1',\n",
       "     'year': 1993,\n",
       "     'title': 'CDC39, an essential nuclear protein that negatively regulates transcription and differentially affects the constitutive and inducible HIS3 promoters.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '177-86',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Collart MA', 'Struhl K.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=8428577'},\n",
       "    'PUB00019276': {'PMID': 12589066,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '2',\n",
       "     'year': 2003,\n",
       "     'title': 'Cdc50p, a conserved endosomal membrane protein, controls polarized growth in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '730-47',\n",
       "     'medline_journal': 'Mol Biol Cell',\n",
       "     'ISO_journal': 'Mol. Biol. Cell',\n",
       "     'authors': ['Misu K',\n",
       "      'Fujimura-Kamada K',\n",
       "      'Ueda T',\n",
       "      'Nakano A',\n",
       "      'Katoh H',\n",
       "      'Tanaka K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1091/mbc.E02-06-0314'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 138,\n",
       "    'interactions': 0,\n",
       "    'matches': 7681,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7528,\n",
       "    'proteomes': 2120,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6201, 'rosettafold': 0},\n",
       "    'structures': 47,\n",
       "    'taxa': 7731},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6337,\n",
       "    'alignment:full': 3883,\n",
       "    'alignment:seed': 290},\n",
       "   'cross_references': {}}},\n",
       " 'PF00542': {'metadata': {'accession': 'PF00542',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013823',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L7/L12 C-terminal domain',\n",
       "    'short': 'Ribosomal_L12'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003217': {'PMID': 3309338,\n",
       "     'ISBN': None,\n",
       "     'volume': '195',\n",
       "     'issue': '3',\n",
       "     'year': 1987,\n",
       "     'title': 'Structure of the C-terminal domain of the ribosomal protein L7/L12 from Escherichia coli at 1.7 A.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '555-79',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Leijonmarck M', 'Liljas A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(87)90183-5'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 213,\n",
       "    'interactions': 0,\n",
       "    'matches': 48561,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48439,\n",
       "    'proteomes': 10609,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41691, 'rosettafold': 0},\n",
       "    'structures': 72,\n",
       "    'taxa': 45323},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43501,\n",
       "    'alignment:full': 10962,\n",
       "    'alignment:seed': 328},\n",
       "   'cross_references': {}}},\n",
       " 'PF04472': {'metadata': {'accession': 'PF04472',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007561',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cell division protein SepF', 'short': 'SepF'},\n",
       "   'description': ['<p>SepF accumulates at the cell division site in an FtsZ-dependent manner and is required for proper septum formation [PMID:16420366]. Mutants are viable but the formation of the septum is much slower and occurs with a very abnormal morphology. This family also includes archaeal related proteins of unknown function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11,\n",
       "    'interactions': 0,\n",
       "    'matches': 19607,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19604,\n",
       "    'proteomes': 3989,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16977, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 15559},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17786,\n",
       "    'alignment:full': 4394,\n",
       "    'alignment:seed': 299},\n",
       "   'cross_references': {}}},\n",
       " 'PF16124': {'metadata': {'accession': 'PF16124',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032284',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RecQ zinc-binding', 'short': 'RecQ_Zn_bind'},\n",
       "   'description': ['<p>This domain is the zinc-binding domain of ATP-dependent DNA helicase RecQ [1-2].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 433,\n",
       "    'interactions': 0,\n",
       "    'matches': 55157,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55145,\n",
       "    'proteomes': 8674,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 44576, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 34569},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49664,\n",
       "    'alignment:full': 14318,\n",
       "    'alignment:seed': 689},\n",
       "   'cross_references': {}}},\n",
       " 'PF05201': {'metadata': {'accession': 'PF05201',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015895',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutamyl-tRNAGlu reductase, N-terminal domain',\n",
       "    'short': 'GlutR_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019621': {'PMID': 3075378,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '4',\n",
       "     'year': 1988,\n",
       "     'title': 'tRNA(Glu) as a cofactor in delta-aminolevulinate biosynthesis: steps that regulate chlorophyll synthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '139-43',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Kannangara CG',\n",
       "      'Gough SP',\n",
       "      'Bruyant P',\n",
       "      'Hoober JK',\n",
       "      'Kahn A',\n",
       "      'von Wettstein D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0968-0004(88)90071-0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 74,\n",
       "    'interactions': 0,\n",
       "    'matches': 32692,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32681,\n",
       "    'proteomes': 6431,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28160, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 27698},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29986,\n",
       "    'alignment:full': 6292,\n",
       "    'alignment:seed': 521},\n",
       "   'cross_references': {}}},\n",
       " 'PF01255': {'metadata': {'accession': 'PF01255',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001441',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative undecaprenyl diphosphate synthase',\n",
       "    'short': 'Prenyltransf'},\n",
       "   'description': ['<p>Previously known as uncharacterized protein family UPF0015, a single member of this family Swiss:O82827 has been identified as an undecaprenyl diphosphate synthase [PMID:9677368].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003021': {'PMID': 9677368,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '31',\n",
       "     'year': 1998,\n",
       "     'title': 'Molecular cloning, expression, and purification of undecaprenyl diphosphate synthase. No sequence similarity between E- and Z-prenyl diphosphate synthases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '19476-81',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Shimizu N', 'Koyama T', 'Ogura K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.31.19476'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 152,\n",
       "    'interactions': 0,\n",
       "    'matches': 64127,\n",
       "    'pathways': 0,\n",
       "    'proteins': 63567,\n",
       "    'proteomes': 10956,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 54393, 'rosettafold': 0},\n",
       "    'structures': 91,\n",
       "    'taxa': 45668},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57336,\n",
       "    'alignment:full': 13804,\n",
       "    'alignment:seed': 1161},\n",
       "   'cross_references': {}}},\n",
       " 'PF03441': {'metadata': {'accession': 'PF03441',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005101',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FAD binding domain of DNA photolyase',\n",
       "    'short': 'FAD_binding_7'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003953': {'PMID': 9360600,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '11',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure of DNA photolyase from Anacystis nidulans.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '887-91',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Tamada T',\n",
       "      'Kitadokoro K',\n",
       "      'Higuchi Y',\n",
       "      'Inaka K',\n",
       "      'Yasui A',\n",
       "      'de Ruiter PE',\n",
       "      'Eker AP',\n",
       "      'Miki K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb1197-887'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 205,\n",
       "    'interactions': 0,\n",
       "    'matches': 47123,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46958,\n",
       "    'proteomes': 6808,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38698, 'rosettafold': 0},\n",
       "    'structures': 79,\n",
       "    'taxa': 29036},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41730,\n",
       "    'alignment:full': 11663,\n",
       "    'alignment:seed': 448},\n",
       "   'cross_references': {}}},\n",
       " 'PF15901': {'metadata': {'accession': 'PF15901',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031777',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sortilin, neurotensin receptor 3, C-terminal',\n",
       "    'short': 'Sortilin_C'},\n",
       "   'description': ['<p>Sortilin_C is the C-terminal cytoplasmic tail of sortilin, a Vps10p domain-containing family of proteins [PMID:9013611, PMID:9756851]. Most sortilin is expressed within intracellular compartments, where it chaperones diverse ligands, including proBDNF and acid hydrolases. The sortilin cytoplasmic tail is homologous to mannose 6-phosphate receptor and is required for the intracellular trafficking of cargo proteins via interactions with distinct adaptor molecules [PMID:10085125, PMID:11331584]. In addition to mediating lysosomal targeting of specific acid hydrolases, the sortilin cytoplasmic tail also directs trafficking of BDNF to the secretory pathway in neurons, where it can be released in response to depolarisation to modulate cell survival and synaptic plasticity [PMID:19122660].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 236,\n",
       "    'interactions': 0,\n",
       "    'matches': 10644,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9237,\n",
       "    'proteomes': 1713,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4416, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 6085},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7450,\n",
       "    'alignment:full': 5031,\n",
       "    'alignment:seed': 301},\n",
       "   'cross_references': {}}},\n",
       " 'PF04749': {'metadata': {'accession': 'PF04749',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006461',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PLAC8 family', 'short': 'PLAC8'},\n",
       "   'description': ['<p>This family includes Swiss:Q9NZF1, the Placenta-specific gene 8 protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 187,\n",
       "    'interactions': 0,\n",
       "    'matches': 14067,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13922,\n",
       "    'proteomes': 1523,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11637, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 5386},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11055,\n",
       "    'alignment:full': 7345,\n",
       "    'alignment:seed': 284},\n",
       "   'cross_references': {}}},\n",
       " 'PF01894': {'metadata': {'accession': 'PF01894',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001602',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterised protein family UPF0047',\n",
       "    'short': 'UPF0047'},\n",
       "   'description': ['<p>This family has no known function. The alignment contains a conserved aspartate and histidine that may be functionally important.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 80,\n",
       "    'interactions': 0,\n",
       "    'matches': 33916,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33865,\n",
       "    'proteomes': 5946,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28733, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 25860},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29493,\n",
       "    'alignment:full': 6361,\n",
       "    'alignment:seed': 618},\n",
       "   'cross_references': {}}},\n",
       " 'PF04085': {'metadata': {'accession': 'PF04085',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007221',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'rod shape-determining protein MreC', 'short': 'MreC'},\n",
       "   'description': ['<p>MreC (murein formation C) is involved in the rod shape  determination in E. coli, and more generally in cell shape determination of bacteria whether or not they are rod-shaped.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 28,\n",
       "    'interactions': 0,\n",
       "    'matches': 37591,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37577,\n",
       "    'proteomes': 6668,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31912, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 29674},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34048,\n",
       "    'alignment:full': 6215,\n",
       "    'alignment:seed': 608},\n",
       "   'cross_references': {}}},\n",
       " 'PF02730': {'metadata': {'accession': 'PF02730',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013983',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aldehyde ferredoxin oxidoreductase, N-terminal domain',\n",
       "    'short': 'AFOR_N'},\n",
       "   'description': ['<p>Aldehyde ferredoxin oxidoreductase (AOR) catalyses the reversible oxidation of aldehydes to their corresponding carboxylic acids with their accompanying reduction of the redox protein ferredoxin. This domain interacts with the tungsten cofactor [PMID:7878465].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005196': {'PMID': 7878465,\n",
       "     'ISBN': None,\n",
       "     'volume': '267',\n",
       "     'issue': '5203',\n",
       "     'year': 1995,\n",
       "     'title': 'Structure of a hyperthermophilic tungstopterin enzyme, aldehyde ferredoxin oxidoreductase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1463-9',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Chan MK', 'Mukund S', 'Kletzin A', 'Adams MW', 'Rees DC.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/267/5203/1463'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 20175,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20168,\n",
       "    'proteomes': 887,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16444, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 5640},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16268,\n",
       "    'alignment:full': 2088,\n",
       "    'alignment:seed': 441},\n",
       "   'cross_references': {}}},\n",
       " 'PF08742': {'metadata': {'accession': 'PF08742',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014853',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'C8 domain', 'short': 'C8'},\n",
       "   'description': ['<p>This domain contains 8 conserved cysteine residues, but this family only contains 7 of them to overlaps with other domains. It is found in disease-related proteins including von Willebrand factor, Alpha tectorin, Zonadhesin and Mucin.  It is often found on proteins containing Pfam:PF00094 and Pfam:PF01826.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2699,\n",
       "    'interactions': 0,\n",
       "    'matches': 41236,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16426,\n",
       "    'proteomes': 817,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6781, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 3465},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27351,\n",
       "    'alignment:full': 17008,\n",
       "    'alignment:seed': 686},\n",
       "   'cross_references': {}}},\n",
       " 'PF17757': {'metadata': {'accession': 'PF17757',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041471',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UvrB interaction domain', 'short': 'UvrB_inter'},\n",
       "   'description': ['<p>This domain is found in the UvrB protein where it interacts with the UvrA protein [PMID:19287003].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0664', 'name': 'UB2H'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 90989,\n",
       "    'pathways': 0,\n",
       "    'proteins': 90984,\n",
       "    'proteomes': 8843,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 77618, 'rosettafold': 0},\n",
       "    'structures': 33,\n",
       "    'taxa': 38216},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 83962,\n",
       "    'alignment:full': 15773,\n",
       "    'alignment:seed': 265},\n",
       "   'cross_references': {}}},\n",
       " 'PF01652': {'metadata': {'accession': 'PF01652',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001040',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Eukaryotic initiation factor 4E', 'short': 'IF4E'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0625', 'name': 'eIF4e'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 209,\n",
       "    'interactions': 0,\n",
       "    'matches': 14512,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14420,\n",
       "    'proteomes': 2201,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11723, 'rosettafold': 0},\n",
       "    'structures': 102,\n",
       "    'taxa': 8346},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11767,\n",
       "    'alignment:full': 6675,\n",
       "    'alignment:seed': 259},\n",
       "   'cross_references': {}}},\n",
       " 'PF11987': {'metadata': {'accession': 'PF11987',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023115',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Translation-initiation factor 2', 'short': 'IF-2'},\n",
       "   'description': ['<p>IF-2 is a translation initiator in each of the three main phylogenetic domains (Eukaryotes [PMID:17086204], Bacteria [PMID:10878130] and Archaea [PMID:16169924]). IF2 interacts with formylmethionine-tRNA, GTP, IF1, IF3 and both ribosomal subunits [PMID:10878130]. Through these interactions, IF2 promotes the binding of the initiator tRNA to the A site in the smaller ribosomal subunit and catalyses the hydrolysis of GTP following initiation-complex formation [PMID:10878130].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 253,\n",
       "    'interactions': 0,\n",
       "    'matches': 58285,\n",
       "    'pathways': 0,\n",
       "    'proteins': 58269,\n",
       "    'proteomes': 10983,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 48749, 'rosettafold': 0},\n",
       "    'structures': 62,\n",
       "    'taxa': 46857},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52025,\n",
       "    'alignment:full': 12278,\n",
       "    'alignment:seed': 479},\n",
       "   'cross_references': {}}},\n",
       " 'PF08331': {'metadata': {'accession': 'PF08331',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013542',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Epoxyqueuosine reductase QueG, DUF1730',\n",
       "    'short': 'QueG_DUF1730'},\n",
       "   'description': ['<p>This domain of unknown function occurs in Epoxyqueuosine reductase QueG, an iron-sulfur cluster-binding protein, together with the 4Fe-4S binding domain (Pfam:PF00037). QueG catalyses the conversion of epoxyqueuosine (oQ) to queuosine (Q), which is a hypermodified base found in the wobble positions of tRNA(Asp), tRNA(Asn), tRNA(His) and tRNA(Tyr) [PMID:21502530].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 44,\n",
       "    'interactions': 0,\n",
       "    'matches': 26497,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26496,\n",
       "    'proteomes': 4905,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22343, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 21369},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24405,\n",
       "    'alignment:full': 4541,\n",
       "    'alignment:seed': 353},\n",
       "   'cross_references': {}}},\n",
       " 'PF12344': {'metadata': {'accession': 'PF12344',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024759',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ultra-violet resistance protein B', 'short': 'UvrB'},\n",
       "   'description': ['<p>This domain family is found in bacteria, archaea and eukaryotes, and is approximately 40 amino acids in length. The family is found in association with Pfam:PF00271, Pfam:PF02151, Pfam:PF04851. There are two conserved sequence motifs: YAD and RRR. This family is the C terminal region of the UvrB protein which conveys mutational resistance against UV light to various different species.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 46305,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46305,\n",
       "    'proteomes': 8704,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39641, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 37254},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42388,\n",
       "    'alignment:full': 8043,\n",
       "    'alignment:seed': 623},\n",
       "   'cross_references': {}}},\n",
       " 'PF01715': {'metadata': {'accession': 'PF01715',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'IPP transferase', 'short': 'IPPT'},\n",
       "   'description': ['<p>This is a family of IPP transferases EC:2.5.1.8 also known as  tRNA delta(2)-isopentenylpyrophosphate transferase. These enzymes modify both cytoplasmic and mitochondrial tRNAs  at A(37) to give isopentenyl A(37) [PMID:8139535].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002313': {'PMID': 9294434,\n",
       "     'ISBN': None,\n",
       "     'volume': '179',\n",
       "     'issue': '18',\n",
       "     'year': 1997,\n",
       "     'title': 'The modified nucleoside 2-methylthio-N6-isopentenyladenosine in tRNA of Shigella flexneri is required for expression of virulence genes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5777-82',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Durand JM',\n",
       "      'Bjork GR',\n",
       "      'Kuwae A',\n",
       "      'Yoshikawa M',\n",
       "      'Sasakawa C.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9294434'},\n",
       "    'PUB00003669': {'PMID': 1850093,\n",
       "     'ISBN': None,\n",
       "     'volume': '11',\n",
       "     'issue': '5',\n",
       "     'year': 1991,\n",
       "     'title': 'MOD5 translation initiation sites determine N6-isopentenyladenosine modification of mitochondrial and cytoplasmic tRNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2382-90',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Gillman EC', 'Slusher LB', 'Martin NC', 'Hopper AK.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=1850093'},\n",
       "    'PUB00003694': {'PMID': 8139535,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '4',\n",
       "     'year': 1994,\n",
       "     'title': 'Subcellular locations of MOD5 proteins: mapping of sequences sufficient for targeting to mitochondria and demonstration that mitochondrial and nuclear isoforms commingle in the cytosol.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2298-306',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Boguta M',\n",
       "      'Hunter LA',\n",
       "      'Shen WC',\n",
       "      'Gillman EC',\n",
       "      'Martin NC',\n",
       "      'Hopper AK.'],\n",
       "     'DOI_URL': 'http://mcb.asm.org/cgi/content/abstract/14/4/2298'}},\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 207,\n",
       "    'interactions': 0,\n",
       "    'matches': 58229,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55924,\n",
       "    'proteomes': 10493,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 47484, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 43687},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 52126,\n",
       "    'alignment:full': 12880,\n",
       "    'alignment:seed': 995},\n",
       "   'cross_references': {}}},\n",
       " 'PF13620': {'metadata': {'accession': 'PF13620',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Carboxypeptidase regulatory-like domain',\n",
       "    'short': 'CarboxypepD_reg'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0287', 'name': 'Transthyretin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4860,\n",
       "    'interactions': 0,\n",
       "    'matches': 128884,\n",
       "    'pathways': 0,\n",
       "    'proteins': 106415,\n",
       "    'proteomes': 4931,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 80600, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 21426},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 107043,\n",
       "    'alignment:full': 24273,\n",
       "    'alignment:seed': 355},\n",
       "   'cross_references': {}}},\n",
       " 'PF16326': {'metadata': {'accession': 'PF16326',\n",
       "   'entry_id': None,\n",
       "   'type': 'unknown',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032524',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ABC transporter C-terminal domain',\n",
       "    'short': 'ABC_tran_CTD'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of ABC transporters. It has a coiled coil structure with an atypical 3(10)-helix in the alpha-hairpin region. It is involved in DNA_binding [PMID:22995754].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 44655,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44655,\n",
       "    'proteomes': 7030,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38603, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 29235},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42023,\n",
       "    'alignment:full': 8692,\n",
       "    'alignment:seed': 709},\n",
       "   'cross_references': {}}},\n",
       " 'PF14237': {'metadata': {'accession': 'PF14237',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025640',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GYF domain 2', 'short': 'GYF_2'},\n",
       "   'description': ['<p>This domain is found in bacteria, archaea and eukaryotes, and is approximately 50 amino acids in length. It contains an evolutionary conserved signature W-X-Y-X6-11-GPF-X4-M-X2-W-X3-GYF, the site of interaction with proline-rich peptides. Family members include RME-8 (Required for receptor-mediated endocytosis 8), a DNAJC13 protein. RME-8 was first identified as a protein that is required for endocytosis in Caenorhabditis elegans. It coordinates the activity of the WASH complex with the function of the retromer SNX dimer to control endosomal tubulation [PMID:24643499]. Family members found in Arabidopsis include Arabidopsis trithorax-related3 (Atxr3), also known as set domain group 2 (Sdg2). It is the major enzyme responsible for H3K4me3 in Arabidopsis and SDG2-dependent H3K4m3 is critical for regulating gene expression and plant development [PMID:20937886]. Another family member found in Arabidopsis is Tic56. It is an essential subunit of a 1-MDa protein complex at the inner chloroplast envelope membrane [PMID:28125316]. Furthermore, Tic56 is important for rRNA processing and chloroplast ribosome assembly [PMID:27733515].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0673', 'name': 'GYF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 305,\n",
       "    'interactions': 0,\n",
       "    'matches': 15082,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14738,\n",
       "    'proteomes': 2556,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10344, 'rosettafold': 1},\n",
       "    'structures': 2,\n",
       "    'taxa': 11280},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12653,\n",
       "    'alignment:full': 3806,\n",
       "    'alignment:seed': 298},\n",
       "   'cross_references': {}}},\n",
       " 'PF00759': {'metadata': {'accession': 'PF00759',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001701',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolase family 9', 'short': 'Glyco_hydro_9'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0059', 'name': '6_Hairpin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 551,\n",
       "    'interactions': 0,\n",
       "    'matches': 21730,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21150,\n",
       "    'proteomes': 1966,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17685, 'rosettafold': 0},\n",
       "    'structures': 48,\n",
       "    'taxa': 9091},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18993,\n",
       "    'alignment:full': 8198,\n",
       "    'alignment:seed': 431},\n",
       "   'cross_references': {}}},\n",
       " 'PF00333': {'metadata': {'accession': 'PF00333',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013810',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S5, N-terminal domain',\n",
       "    'short': 'Ribosomal_S5'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0196', 'name': 'DSRM'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 169,\n",
       "    'interactions': 0,\n",
       "    'matches': 55951,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55934,\n",
       "    'proteomes': 11117,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 47769, 'rosettafold': 0},\n",
       "    'structures': 1155,\n",
       "    'taxa': 50669},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49482,\n",
       "    'alignment:full': 12866,\n",
       "    'alignment:seed': 345},\n",
       "   'cross_references': {}}},\n",
       " 'PF05402': {'metadata': {'accession': 'PF05402',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008792',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Coenzyme PQQ synthesis protein D (PqqD)',\n",
       "    'short': 'PqqD'},\n",
       "   'description': ['<p>This family contains several bacterial coenzyme PQQ synthesis protein D (PqqD) sequences. This protein is required for coenzyme pyrrolo-quinoline-quinone (PQQ) biosynthesis [PMID:8002620, PMID:12437981].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010477': {'PMID': 12437981,\n",
       "     'ISBN': None,\n",
       "     'volume': '299',\n",
       "     'issue': '2',\n",
       "     'year': 2002,\n",
       "     'title': 'PqqC/D, which converts a biosynthetic intermediate to pyrroloquinoline quinone.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '268-72',\n",
       "     'medline_journal': 'Biochem Biophys Res Commun',\n",
       "     'ISO_journal': 'Biochem. Biophys. Res. Commun.',\n",
       "     'authors': ['Toyama H',\n",
       "      'Fukumoto H',\n",
       "      'Saeki M',\n",
       "      'Matsushita K',\n",
       "      'Adachi O',\n",
       "      'Lidstrom ME.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0006-291X(02)02603-7'},\n",
       "    'PUB00020172': {'PMID': 8002620,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '1',\n",
       "     'year': 1995,\n",
       "     'title': 'Transcriptional analysis of pqqD and study of the regulation of pyrroloquinoline quinone biosynthesis in Methylobacterium extorquens AM1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '206-11',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Ramamoorthi R', 'Lidstrom ME.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8002620&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 119,\n",
       "    'interactions': 0,\n",
       "    'matches': 20654,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20644,\n",
       "    'proteomes': 2937,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17551, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 14246},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18495,\n",
       "    'alignment:full': 3673,\n",
       "    'alignment:seed': 386},\n",
       "   'cross_references': {}}},\n",
       " 'PF06961': {'metadata': {'accession': 'PF06961',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010718',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1294)',\n",
       "    'short': 'DUF1294'},\n",
       "   'description': ['<p>This family includes a number of hypothetical bacterial and archaeal proteins of unknown function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 28,\n",
       "    'interactions': 0,\n",
       "    'matches': 12874,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12873,\n",
       "    'proteomes': 2428,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11374, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12164},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12589,\n",
       "    'alignment:full': 2334,\n",
       "    'alignment:seed': 310},\n",
       "   'cross_references': {}}},\n",
       " 'PF04468': {'metadata': {'accession': 'PF04468',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007557',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PSP1 C-terminal conserved region', 'short': 'PSP1'},\n",
       "   'description': ['<p>This region is present in both eukaryotes and eubacteria. The yeast PSP1 protein is involved in  suppressing mutations in the DNA polymerase alpha subunit in yeast [PMID:9529527].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010018': {'PMID': 9529527,\n",
       "     'ISBN': None,\n",
       "     'volume': '257',\n",
       "     'issue': '4',\n",
       "     'year': 1998,\n",
       "     'title': 'Suppressors of the temperature sensitivity of DNA polymerase alpha mutations in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '461-8',\n",
       "     'medline_journal': 'Mol Gen Genet',\n",
       "     'ISO_journal': 'Mol. Gen. Genet.',\n",
       "     'authors': ['Formosa T', 'Nittis T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/s004380050670'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 18040,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18039,\n",
       "    'proteomes': 4146,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15152, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15774},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15595,\n",
       "    'alignment:full': 4234,\n",
       "    'alignment:seed': 401},\n",
       "   'cross_references': {}}},\n",
       " 'PF01149': {'metadata': {'accession': 'PF01149',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012319',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Formamidopyrimidine-DNA glycosylase N-terminal domain',\n",
       "    'short': 'Fapy_DNA_glyco'},\n",
       "   'description': ['<p>Formamidopyrimidine-DNA glycosylase (Fpg) is a DNA repair enzyme that excises oxidised purines from damaged DNA. This family is the N-terminal domain contains eight beta-strands, forming a beta-sandwich with two alpha-helices parallel to its edges [PMID:11912217].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012853': {'PMID': 11912217,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '22',\n",
       "     'year': 2002,\n",
       "     'title': 'Structure of formamidopyrimidine-DNA glycosylase covalently complexed to DNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '19811-6',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Gilboa R',\n",
       "      'Zharkov DO',\n",
       "      'Golan G',\n",
       "      'Fernandes AS',\n",
       "      'Gerchman SE',\n",
       "      'Matz E',\n",
       "      'Kycia JH',\n",
       "      'Grollman AP',\n",
       "      'Shoham G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M202058200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 103,\n",
       "    'interactions': 0,\n",
       "    'matches': 55782,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55771,\n",
       "    'proteomes': 8139,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 47854, 'rosettafold': 0},\n",
       "    'structures': 112,\n",
       "    'taxa': 33934},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51571,\n",
       "    'alignment:full': 11647,\n",
       "    'alignment:seed': 675},\n",
       "   'cross_references': {}}},\n",
       " 'PF03613': {'metadata': {'accession': 'PF03613',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004704',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PTS system mannose/fructose/sorbose family IID component',\n",
       "    'short': 'EIID-AGA'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 15673,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15668,\n",
       "    'proteomes': 984,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14394, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 5794},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16450,\n",
       "    'alignment:full': 2379,\n",
       "    'alignment:seed': 186},\n",
       "   'cross_references': {}}},\n",
       " 'PF01773': {'metadata': {'accession': 'PF01773',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002668',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Na+ dependent nucleoside transporter N-terminus',\n",
       "    'short': 'Nucleos_tra2_N'},\n",
       "   'description': ['<p>This family consists of nucleoside transport proteins. Swiss:Q62773 is a purine-specific Na+-nucleoside cotransporter localised to the bile canalicular membrane [PMID:7775409]. Swiss:Q62674 is a a Na+-dependent nucleoside transporter selective for pyrimidine nucleosides and adenosine it also transports the anti-viral nucleoside analogues AZT and ddC [PMID:8027026]. This alignment covers the N terminus of this family</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002855': {'PMID': 8027026,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '27',\n",
       "     'year': 1994,\n",
       "     'title': 'Cloning and functional expression of a complementary DNA encoding a mammalian nucleoside transport protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17757-60',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Huang QQ',\n",
       "      'Yao SY',\n",
       "      'Ritzel MW',\n",
       "      'Paterson AR',\n",
       "      'Cass CE',\n",
       "      'Young JD.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/27/17757'},\n",
       "    'PUB00002907': {'PMID': 7775409,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '23',\n",
       "     'year': 1995,\n",
       "     'title': 'Primary structure and functional expression of a cDNA encoding the bile canalicular, purine-specific Na(+)-nucleoside cotransporter.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13596-9',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Che M', 'Ortiz DF', 'Arias IM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.270.23.13596'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 85,\n",
       "    'interactions': 0,\n",
       "    'matches': 24068,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24044,\n",
       "    'proteomes': 4013,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20609, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 17370},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23115,\n",
       "    'alignment:full': 4825,\n",
       "    'alignment:seed': 390},\n",
       "   'cross_references': {}}},\n",
       " 'PF17827': {'metadata': {'accession': 'PF17827',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040758',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PrmC N-terminal domain', 'short': 'PrmC_N'},\n",
       "   'description': ['<p>This entry corresponds to the N-terminal alpha helical domain of the HemK protein. HemK is a methyltransferase enzyme that carries out the methylation of the N5 nitrogen of the glutamine found in the conserved GGQ motif of class-1 release factors [PMID:12741815].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 82,\n",
       "    'interactions': 0,\n",
       "    'matches': 42809,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42706,\n",
       "    'proteomes': 8608,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36447, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 35390},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39132,\n",
       "    'alignment:full': 7898,\n",
       "    'alignment:seed': 470},\n",
       "   'cross_references': {}}},\n",
       " 'PF06094': {'metadata': {'accession': 'PF06094',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009288',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Gamma-glutamyl cyclotransferase, AIG2-like',\n",
       "    'short': 'GGACT'},\n",
       "   'description': ['<p>GGACT, gamma-glutamylamine cyclotransferase, is a ubiquitous enzyme found in bacteria, plants, and metazoans from Dictyostelium through to humans. It converts gamma-glutamylamines to free amines and 5-oxoproline.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007984': {'PMID': 8742710,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'Isolation of Arabidopsis genes that differentiate between resistance responses mediated by the RPS2 and RPM1 disease resistance genes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '241-9',\n",
       "     'medline_journal': 'Plant Cell',\n",
       "     'ISO_journal': 'Plant Cell',\n",
       "     'authors': ['Reuber TL', 'Ausubel FM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1105/tpc.8.2.241'},\n",
       "    'PUB00040892': {'PMID': 16754964,\n",
       "     'ISBN': None,\n",
       "     'volume': '62',\n",
       "     'issue': 'Pt 6',\n",
       "     'year': 2006,\n",
       "     'title': 'Solution structure of Arabidopsis thaliana protein At5g39720.1, a member of the AIG2-like protein family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '490-3',\n",
       "     'medline_journal': 'Acta Crystallogr Sect F Struct Biol Cryst Commun',\n",
       "     'ISO_journal': 'Acta Crystallogr. Sect. F Struct. Biol. Cryst. Commun.',\n",
       "     'authors': ['Lytle BL',\n",
       "      'Peterson FC',\n",
       "      'Tyler EM',\n",
       "      'Newman CL',\n",
       "      'Vinarov DA',\n",
       "      'Markley JL',\n",
       "      'Volkman BF.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=16754964&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0278', 'name': 'AIG2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 216,\n",
       "    'interactions': 0,\n",
       "    'matches': 24369,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24145,\n",
       "    'proteomes': 5080,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20403, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 21786},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21904,\n",
       "    'alignment:full': 6908,\n",
       "    'alignment:seed': 325},\n",
       "   'cross_references': {}}},\n",
       " 'PF01430': {'metadata': {'accession': 'PF01430',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000397',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hsp33 protein', 'short': 'HSP33'},\n",
       "   'description': ['<p>Hsp33 is a molecular chaperone, distinguished from all other known chaperones by its mode of functional regulation. Its activity is redox regulated. Hsp33 is a cytoplasmically localised protein with highly reactive cysteines that respond quickly to changes in the redox environment. Oxidising conditions like H2O2 cause disulfide bonds to form in Hsp33, a process that leads to the activation of its chaperone function [PMID:10025400].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000967': {'PMID': 10025400,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '3',\n",
       "     'year': 1999,\n",
       "     'title': 'Chaperone activity with a redox switch.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '341-52',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Jakob U', 'Muse W', 'Eser M', 'Bardwell JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80547-4'},\n",
       "    'PUB00019723': {'PMID': 11377197,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '5',\n",
       "     'year': 2001,\n",
       "     'title': 'The 2.2 A crystal structure of Hsp33: a heat shock protein with redox-regulated chaperone activity.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '367-75',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Vijayalakshmi J',\n",
       "      'Mukhergee MK',\n",
       "      'Graumann J',\n",
       "      'Jakob U',\n",
       "      'Saper MA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00597-4'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 24,\n",
       "    'interactions': 0,\n",
       "    'matches': 24359,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24314,\n",
       "    'proteomes': 5095,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21078, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 22022},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22955,\n",
       "    'alignment:full': 4849,\n",
       "    'alignment:seed': 513},\n",
       "   'cross_references': {}}},\n",
       " 'PF02603': {'metadata': {'accession': 'PF02603',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011126',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HPr Serine kinase N terminus', 'short': 'Hpr_kinase_N'},\n",
       "   'description': ['<p>This family represents the N-terminal region of Hpr Serine/threonine kinase PtsK.  This kinase is the sensor in a multicomponent phospho-relay system in control of carbon catabolic repression in bacteria [PMID:9570401].  This kinase in unusual in that it recognises the tertiary structure of its target and is a member of a novel family unrelated to any previously described protein phosphorylating enzymes [PMID:9570401]. X-ray analysis of the full-length crystalline enzyme from Staphylococcus xylosus at a resolution of 1.95 A shows the enzyme to consist of two clearly separated domains that are assembled in a hexameric structure resembling a three-bladed propeller. The blades are formed by two N-terminal domains each, and the compact central hub assembles the C-terminal kinase domains [PMID:11904409].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008233': {'PMID': 9570401,\n",
       "     'ISBN': None,\n",
       "     'volume': '27',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'A novel protein kinase that controls carbon catabolite repression in bacteria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1157-69',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Reizer J',\n",
       "      'Hoischen C',\n",
       "      'Titgemeyer F',\n",
       "      'Rivolta C',\n",
       "      'Rabus R',\n",
       "      'Stulke J',\n",
       "      'Karamata D',\n",
       "      'Saier MH Jr',\n",
       "      'Hillen W.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1998.00747.x'},\n",
       "    'PUB00014276': {'PMID': 11904409,\n",
       "     'ISBN': None,\n",
       "     'volume': '99',\n",
       "     'issue': '6',\n",
       "     'year': 2002,\n",
       "     'title': 'Structure of the full-length HPr kinase/phosphatase from Staphylococcus xylosus at 1.95 A resolution: Mimicking the product/substrate of the phospho transfer reactions.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3458-63',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Marquez JA',\n",
       "      'Hasenbein S',\n",
       "      'Koch B',\n",
       "      'Fieulaine S',\n",
       "      'Nessler S',\n",
       "      'Russell RB',\n",
       "      'Hengstenberg W',\n",
       "      'Scheffzek K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.052461499'}},\n",
       "   'set_info': {'accession': 'CL0365', 'name': 'MurF-HprK_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 12956,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12864,\n",
       "    'proteomes': 2644,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11075, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 11032},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11773,\n",
       "    'alignment:full': 2618,\n",
       "    'alignment:seed': 310},\n",
       "   'cross_references': {}}},\n",
       " 'PF00920': {'metadata': {'accession': 'PF00920',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000581',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dehydratase family', 'short': 'ILVD_EDD'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 152,\n",
       "    'interactions': 0,\n",
       "    'matches': 83364,\n",
       "    'pathways': 0,\n",
       "    'proteins': 82852,\n",
       "    'proteomes': 8948,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 71658, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 36674},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 77895,\n",
       "    'alignment:full': 17070,\n",
       "    'alignment:seed': 772},\n",
       "   'cross_references': {}}},\n",
       " 'PF14805': {'metadata': {'accession': 'PF14805',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023180',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetrahydrodipicolinate N-succinyltransferase N-terminal',\n",
       "    'short': 'THDPS_N_2'},\n",
       "   'description': ['<p>This is the N-terminal domain of 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate N-succinyltransferase [PMID:9671504].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 18,\n",
       "    'interactions': 0,\n",
       "    'matches': 17340,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17340,\n",
       "    'proteomes': 3750,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14823, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 15573},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15756,\n",
       "    'alignment:full': 3472,\n",
       "    'alignment:seed': 242},\n",
       "   'cross_references': {}}},\n",
       " 'PF04024': {'metadata': {'accession': 'PF04024',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007168',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PspC domain', 'short': 'PspC'},\n",
       "   'description': ['<p>This family includes Phage shock protein C (PspC) that is thought to be a transcriptional regulator. The presumed domain is 60 amino acid residues in length.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012784': {'PMID': 1712397,\n",
       "     'ISBN': None,\n",
       "     'volume': '220',\n",
       "     'issue': '1',\n",
       "     'year': 1991,\n",
       "     'title': 'Characterization and sequence of the Escherichia coli stress-induced psp operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '35-48',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Brissette JL', 'Weiner L', 'Ripmaster TL', 'Model P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(91)90379-K'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 50,\n",
       "    'interactions': 0,\n",
       "    'matches': 34397,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33519,\n",
       "    'proteomes': 4563,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 29295, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 18987},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31516,\n",
       "    'alignment:full': 7654,\n",
       "    'alignment:seed': 612},\n",
       "   'cross_references': {}}},\n",
       " 'PF03830': {'metadata': {'accession': 'PF03830',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004720',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PTS system sorbose subfamily IIB component',\n",
       "    'short': 'PTSIIB_sorb'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 16,\n",
       "    'interactions': 0,\n",
       "    'matches': 15943,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15940,\n",
       "    'proteomes': 990,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14735, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 5804},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16817,\n",
       "    'alignment:full': 2492,\n",
       "    'alignment:seed': 250},\n",
       "   'cross_references': {}}},\n",
       " 'PF00040': {'metadata': {'accession': 'PF00040',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000562',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fibronectin type II domain', 'short': 'fn2'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0602', 'name': 'Kringle'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 736,\n",
       "    'interactions': 0,\n",
       "    'matches': 24473,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15421,\n",
       "    'proteomes': 699,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4724, 'rosettafold': 0},\n",
       "    'structures': 63,\n",
       "    'taxa': 2763},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15739,\n",
       "    'alignment:full': 9529,\n",
       "    'alignment:seed': 256},\n",
       "   'cross_references': {}}},\n",
       " 'PF00303': {'metadata': {'accession': 'PF00303',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023451',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Thymidylate synthase', 'short': 'Thymidylat_synt'},\n",
       "   'description': ['<p>This is a family of proteins that are flavin-dependent thymidylate synthases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 104,\n",
       "    'interactions': 0,\n",
       "    'matches': 43762,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43439,\n",
       "    'proteomes': 9010,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35633, 'rosettafold': 0},\n",
       "    'structures': 329,\n",
       "    'taxa': 38270},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40179,\n",
       "    'alignment:full': 9030,\n",
       "    'alignment:seed': 432},\n",
       "   'cross_references': {}}},\n",
       " 'PF00755': {'metadata': {'accession': 'PF00755',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR039551',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Choline/Carnitine o-acyltransferase',\n",
       "    'short': 'Carn_acyltransf'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0149', 'name': 'CoA-acyltrans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 325,\n",
       "    'interactions': 0,\n",
       "    'matches': 22327,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20919,\n",
       "    'proteomes': 2045,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15450, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 8323},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16777,\n",
       "    'alignment:full': 10079,\n",
       "    'alignment:seed': 388},\n",
       "   'cross_references': {}}},\n",
       " 'PF00486': {'metadata': {'accession': 'PF00486',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001867',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transcriptional regulatory protein, C terminal',\n",
       "    'short': 'Trans_reg_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005284': {'PMID': 9016718,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '1',\n",
       "     'year': 1997,\n",
       "     'title': 'The DNA-binding domain of OmpR: crystal structures of a winged helix transcription factor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '109-24',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Martinez-Hackert E', 'Stock AM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00170-6'},\n",
       "    'PUB00020465': {'PMID': 9350875,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '6',\n",
       "     'year': 1997,\n",
       "     'title': 'A novel family of proteins that regulates antibiotic production in streptomycetes appears to contain an OmpR-like DNA-binding fold.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1181-4',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Wietzorrek A', 'Bibb M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1997.5421903.x'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1471,\n",
       "    'interactions': 0,\n",
       "    'matches': 520039,\n",
       "    'pathways': 0,\n",
       "    'proteins': 519937,\n",
       "    'proteomes': 8649,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 456024, 'rosettafold': 0},\n",
       "    'structures': 67,\n",
       "    'taxa': 37587},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 497911,\n",
       "    'alignment:full': 106662,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF00596': {'metadata': {'accession': 'PF00596',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001303',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Class II Aldolase and Adducin N-terminal domain',\n",
       "    'short': 'Aldolase_II'},\n",
       "   'description': ['<p>This family includes class II aldolases and adducins which have not been ascribed any enzymatic function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003307': {'PMID': 8515438,\n",
       "     'ISBN': None,\n",
       "     'volume': '231',\n",
       "     'issue': '3',\n",
       "     'year': 1993,\n",
       "     'title': 'The spatial structure of the class II L-fuculose-1-phosphate aldolase from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '549-53',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Dreyer MK', 'Schulz GE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1993.1307'},\n",
       "    'PUB00003364': {'PMID': 8676381,\n",
       "     'ISBN': None,\n",
       "     'volume': '259',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Catalytic mechanism of the metal-dependent fuculose aldolase from Escherichia coli as derived from the structure.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '458-66',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Dreyer MK', 'Schulz GE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1996.0332'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 333,\n",
       "    'interactions': 0,\n",
       "    'matches': 99549,\n",
       "    'pathways': 0,\n",
       "    'proteins': 98622,\n",
       "    'proteomes': 8821,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 84169, 'rosettafold': 0},\n",
       "    'structures': 51,\n",
       "    'taxa': 37982},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 91568,\n",
       "    'alignment:full': 21658,\n",
       "    'alignment:seed': 313},\n",
       "   'cross_references': {}}},\n",
       " 'PF16360': {'metadata': {'accession': 'PF16360',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032305',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GTP-binding GTPase Middle Region', 'short': 'GTP-bdg_M'},\n",
       "   'description': ['<p>This family locates between the N-terminal domain and MMR_HSR1 50S ribosome-binding GTPase of GTP-binding HflX-like proteins. The full-length members bind and interact with the 50S ribosome and are GTPases, hydrolysing GTP/GDP/ATP/ADP. This region is unknown for its function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 45719,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45718,\n",
       "    'proteomes': 9070,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38927, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 37657},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41535,\n",
       "    'alignment:full': 8830,\n",
       "    'alignment:seed': 428},\n",
       "   'cross_references': {}}},\n",
       " 'PF17862': {'metadata': {'accession': 'PF17862',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041569',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AAA+ lid domain', 'short': 'AAA_lid_3'},\n",
       "   'description': ['<p>This entry represents the alpha helical AAA+ lid domain that is found to the C-terminus of AAA domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0671', 'name': 'AAA_lid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1574,\n",
       "    'interactions': 0,\n",
       "    'matches': 186833,\n",
       "    'pathways': 0,\n",
       "    'proteins': 167155,\n",
       "    'proteomes': 11138,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 135063, 'rosettafold': 0},\n",
       "    'structures': 308,\n",
       "    'taxa': 49735},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 156471,\n",
       "    'alignment:full': 65108,\n",
       "    'alignment:seed': 582},\n",
       "   'cross_references': {}}},\n",
       " 'PF02929': {'metadata': {'accession': 'PF02929',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004199',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Beta galactosidase small chain', 'short': 'Bgal_small_N'},\n",
       "   'description': ['<p>This domain comprises the small chain of dimeric beta-galactosidases EC:3.2.1.23.  This domain is also found in single chain beta-galactosidase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 284,\n",
       "    'interactions': 0,\n",
       "    'matches': 21755,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21672,\n",
       "    'proteomes': 2920,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17908, 'rosettafold': 0},\n",
       "    'structures': 92,\n",
       "    'taxa': 11931},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21203,\n",
       "    'alignment:full': 4696,\n",
       "    'alignment:seed': 388},\n",
       "   'cross_references': {}}},\n",
       " 'PF18074': {'metadata': {'accession': 'PF18074',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041236',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Primosomal protein N C-terminal domain',\n",
       "    'short': 'PriA_C'},\n",
       "   'description': [\"<p>This is the C-terminal domain found in PriA DNA helicase, a multifunctional enzyme that mediates the process of restarting prematurely terminated DNA replication reactions in bacteria. The C-terminal domain (CTD) bears similarity to the S10 subunit which binds branched rRNA within the bacterial ribosome. The C-terminal domain is part of the helicase domain of PriA proteins. It acts together with the 3' DNA-binding domain to form a site for binding ssDNA-binding protein (SSB) [PMID:24379377].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 73,\n",
       "    'interactions': 0,\n",
       "    'matches': 37434,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37434,\n",
       "    'proteomes': 6562,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31696, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 28767},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34580,\n",
       "    'alignment:full': 6137,\n",
       "    'alignment:seed': 321},\n",
       "   'cross_references': {}}},\n",
       " 'PF04295': {'metadata': {'accession': 'PF04295',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007392',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'D-galactarate dehydratase / Altronate hydrolase, C terminus',\n",
       "    'short': 'GD_AH_C'},\n",
       "   'description': ['<p>Family members include the C termini of D-galactarate dehydratase  (EC:4.2.1.42) which is thought to catalyse the reaction  D-galactarate = 5-keto-4-deoxy-D-glucarate + H2O, [PMID:9772162] and altronate  hydrolase (altronic acid hydratase, EC:4.2.1.7), which catalyses D-altronate = 2-keto-2-deoxygluconate + H2O [PMID:9579062]. As purified, both enzymes are catalytically inactive in the absence of  added Fe2+, Mn2+, and beta-mercaptoethanol.  Synergistic activation of altronate hydrolase activity is seen in the presence of both iron and manganese ions, suggesting that the enzyme may have two ion binding sites. Mn2+ appears to be part of the enzyme active centre, but the function of  the single bound Fe2+ ion is unknown.  The hydratase has no Fe-S core [PMID:3038546].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009934': {'PMID': 3038546,\n",
       "     'ISBN': None,\n",
       "     'volume': '166',\n",
       "     'issue': '3',\n",
       "     'year': 1987,\n",
       "     'title': 'The role of iron in the activation of mannonic and altronic acid hydratases, two Fe-requiring hydro-lyases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '623-30',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Dreyer JL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1432-1033.1987.tb13559.x'},\n",
       "    'PUB00009935': {'PMID': 9579062,\n",
       "     'ISBN': None,\n",
       "     'volume': '144 ( Pt 4)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'A 35.7 kb DNA fragment from the Bacillus subtilis chromosome containing a putative 12.3 kb operon involved in hexuronate catabolism and a perfectly symmetrical hypothetical catabolite-responsive element.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '877-84',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Rivolta C',\n",
       "      'Soldo B',\n",
       "      'Lazarevic V',\n",
       "      'Joris B',\n",
       "      'Mauel C',\n",
       "      'Karamata D.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00009936': {'PMID': 9772162,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '41',\n",
       "     'year': 1998,\n",
       "     'title': 'Evolution of enzymatic activities in the enolase superfamily: characterization of the (D)-glucarate/galactarate catabolic pathway in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14369-75',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Hubbard BK',\n",
       "      'Koch M',\n",
       "      'Palmer DR',\n",
       "      'Babbitt PC',\n",
       "      'Gerlt JA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi981124f'},\n",
       "    'PUB00017194': {'PMID': 15146494,\n",
       "     'ISBN': None,\n",
       "     'volume': '55',\n",
       "     'issue': '4',\n",
       "     'year': 2004,\n",
       "     'title': 'The emergence of catalytic and structural diversity within the beta-clip fold.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '977-91',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Iyer LM', 'Aravind L.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.20076'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 22600,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22163,\n",
       "    'proteomes': 2830,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19231, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 13215},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21889,\n",
       "    'alignment:full': 4113,\n",
       "    'alignment:seed': 240},\n",
       "   'cross_references': {}}},\n",
       " 'PF12804': {'metadata': {'accession': 'PF12804',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025877',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MobA-like NTP transferase domain',\n",
       "    'short': 'NTP_transf_3'},\n",
       "   'description': ['<p>This family includes the MobA protein (Molybdopterin-guanine dinucleotide biosynthesis protein A). The family also includes a wide range of other NTP transferase domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0110', 'name': 'GT-A'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 391,\n",
       "    'interactions': 0,\n",
       "    'matches': 114763,\n",
       "    'pathways': 0,\n",
       "    'proteins': 114643,\n",
       "    'proteomes': 8422,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 97962, 'rosettafold': 0},\n",
       "    'structures': 68,\n",
       "    'taxa': 36958},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 104549,\n",
       "    'alignment:full': 20541,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF02586': {'metadata': {'accession': 'PF02586',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003738',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SOS response associated peptidase (SRAP)',\n",
       "    'short': 'SRAP'},\n",
       "   'description': ['<p>The SRAP family functions as a DNA-associated autoproteolytic switch that recruits diverse repair enzymes onto DNA damage. We propose that the human protein Q96FZ2:UniProtKB, the eukaryotic member of the SRAP family, which has been recently shown to bind specifically to DNA with 5-hydroxymethylcytosine, 5-formylcytosine and 5-carboxycytosine, is a sensor for these oxidized bases generated by the TET (tetrahedral aminopeptidase of the M42 family) enzymes from methylcytosine. Hence, its autoproteolytic activity might help it act as a switch that recruits DNA repair enzymes to remove these oxidized methylcytosine species as part of the DNA demethylation pathway downstream of the TET enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 92,\n",
       "    'interactions': 0,\n",
       "    'matches': 38141,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37928,\n",
       "    'proteomes': 7188,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32803, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 28139},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35926,\n",
       "    'alignment:full': 8345,\n",
       "    'alignment:seed': 404},\n",
       "   'cross_references': {}}},\n",
       " 'PF01055': {'metadata': {'accession': 'PF01055',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000322',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolases family 31',\n",
       "    'short': 'Glyco_hydro_31'},\n",
       "   'description': ['<p>Glycosyl hydrolases are key enzymes of carbohydrate metabolism. Family 31 comprises of enzymes that are, or similar to, alpha- galactosidases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000503': {'PMID': 1747104,\n",
       "     'ISBN': None,\n",
       "     'volume': '280 ( Pt 2)',\n",
       "     'issue': None,\n",
       "     'year': 1991,\n",
       "     'title': 'A classification of glycosyl hydrolases based on amino acid sequence similarities.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '309-16',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Henrissat B.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=1747104'},\n",
       "    'PUB00000571': {'PMID': 9649738,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Glycosidase families.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '153-6',\n",
       "     'medline_journal': 'Biochem Soc Trans',\n",
       "     'ISO_journal': 'Biochem. Soc. Trans.',\n",
       "     'authors': ['Henrissat B.'],\n",
       "     'DOI_URL': 'http://www.biochemsoctrans.org/bst/026/0153/0260153.pdf'}},\n",
       "   'set_info': {'accession': 'CL0058', 'name': 'Glyco_hydro_tim'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 943,\n",
       "    'interactions': 0,\n",
       "    'matches': 62686,\n",
       "    'pathways': 0,\n",
       "    'proteins': 58492,\n",
       "    'proteomes': 5631,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46832, 'rosettafold': 0},\n",
       "    'structures': 143,\n",
       "    'taxa': 22934},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 53439,\n",
       "    'alignment:full': 19731,\n",
       "    'alignment:seed': 659},\n",
       "   'cross_references': {}}},\n",
       " 'PF12806': {'metadata': {'accession': 'PF12806',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025878',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acetyl-CoA dehydrogenase C-terminal like',\n",
       "    'short': 'Acyl-CoA_dh_C'},\n",
       "   'description': ['<p>this domain would appear to be the very C-terminal region of many bacterial acetyl-CoA dehydrogenases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 34415,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34413,\n",
       "    'proteomes': 3601,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 30012, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 13984},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33442,\n",
       "    'alignment:full': 6442,\n",
       "    'alignment:seed': 411},\n",
       "   'cross_references': {}}},\n",
       " 'PF01312': {'metadata': {'accession': 'PF01312',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006135',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FlhB HrpN YscU SpaS Family', 'short': 'Bac_export_2'},\n",
       "   'description': ['<p>This family includes the following members:   FlhB, HrpN, YscU, SpaS, HrcU SsaU and YopU. All of these proteins export peptides using the type III secretion system.  The peptides exported are quite diverse.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002292': {'PMID': 8626302,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '6',\n",
       "     'year': 1996,\n",
       "     'title': 'Erwinia amylovora secretes harpin via a type III pathway and contains a homolog of yopN of Yersinia spp.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1720-30',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Bogdanove AJ', 'Wei ZM', 'Zhao L', 'Beer SV.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/178/6/1720'},\n",
       "    'PUB00003585': {'PMID': 9618447,\n",
       "     'ISBN': None,\n",
       "     'volume': '62',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Type III protein secretion systems in bacterial pathogens of animals and plants.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '379-433',\n",
       "     'medline_journal': 'Microbiol Mol Biol Rev',\n",
       "     'ISO_journal': 'Microbiol. Mol. Biol. Rev.',\n",
       "     'authors': ['Hueck CJ.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9618447'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 36,\n",
       "    'interactions': 0,\n",
       "    'matches': 37466,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37444,\n",
       "    'proteomes': 4263,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32980, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 18426},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36196,\n",
       "    'alignment:full': 6735,\n",
       "    'alignment:seed': 501},\n",
       "   'cross_references': {}}},\n",
       " 'PF17202': {'metadata': {'accession': 'PF17202',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR033463',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Single cache domain 3', 'short': 'sCache_3_3'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0165', 'name': 'Cache'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 149,\n",
       "    'interactions': 0,\n",
       "    'matches': 4947,\n",
       "    'pathways': 0,\n",
       "    'proteins': 4937,\n",
       "    'proteomes': 887,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4253, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 4306},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4559,\n",
       "    'alignment:full': 1196,\n",
       "    'alignment:seed': 269},\n",
       "   'cross_references': {}}},\n",
       " 'PF01311': {'metadata': {'accession': 'PF01311',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002010',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial export proteins, family 1',\n",
       "    'short': 'Bac_export_1'},\n",
       "   'description': ['<p>This family includes the following members;     FliR, MopE, SsaT, YopT, Hrp, HrcT and SpaR All of these members export proteins, that do not possess signal  peptides, through the membrane.  Although the proteins that these exporters move may be different, the exporters are thought to  function in similar ways [PMID:7814323].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002272': {'PMID': 7814323,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '2',\n",
       "     'year': 1995,\n",
       "     'title': 'Caulobacter FliQ and FliR membrane proteins, required for flagellar biogenesis and cell division, belong to a family of virulence factor export proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '343-56',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Zhuang WY', 'Shapiro L.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=7814323'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 26519,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26512,\n",
       "    'proteomes': 4247,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23370, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 18194},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25637,\n",
       "    'alignment:full': 4652,\n",
       "    'alignment:seed': 405},\n",
       "   'cross_references': {}}},\n",
       " 'PF03481': {'metadata': {'accession': 'PF03481',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005145',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Threonylcarbamoyl-AMP synthase, C-terminal domain',\n",
       "    'short': 'Sua5_C'},\n",
       "   'description': ['<p>This domain can be found in the C terminus of threonylcarbamoyl-AMP synthases, including Sua5 from Saccharomyces cerevisiae and YwlC from Bacillus subtilis. Threonylcarbamoyl-AMP synthase is required for the formation of a threonylcarbamoyl group on adenosine at position 37 (t6A37) in tRNAs that read codons beginning with adenine [PMID:19287007, PMID:23072323]. This domain adopts the Rossmann fold and may be involved in GTP and/or tRNA binding based on structural similarity with both GTP and tRNA binding proteins [PMID:18004774].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0113', 'name': 'GT-B'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 68,\n",
       "    'interactions': 0,\n",
       "    'matches': 21591,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21584,\n",
       "    'proteomes': 5364,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18111, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 20476},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19134,\n",
       "    'alignment:full': 4997,\n",
       "    'alignment:seed': 557},\n",
       "   'cross_references': {}}},\n",
       " 'PF01221': {'metadata': {'accession': 'PF01221',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001372',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dynein light chain type 1', 'short': 'Dynein_light'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003963': {'PMID': 9808041,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '11',\n",
       "     'year': 1998,\n",
       "     'title': 'Solution structure of a protein inhibitor of neuronal nitric oxide synthase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '965-9',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Tochio H', 'Ohki S', 'Zhang Q', 'Li M', 'Zhang M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/2940'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 181,\n",
       "    'interactions': 0,\n",
       "    'matches': 12596,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12344,\n",
       "    'proteomes': 2092,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10360, 'rosettafold': 0},\n",
       "    'structures': 55,\n",
       "    'taxa': 7698},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10286,\n",
       "    'alignment:full': 6398,\n",
       "    'alignment:seed': 234},\n",
       "   'cross_references': {}}},\n",
       " 'PF05198': {'metadata': {'accession': 'PF05198',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019814',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Translation initiation factor IF-3, N-terminal domain',\n",
       "    'short': 'IF3_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 100,\n",
       "    'interactions': 0,\n",
       "    'matches': 45115,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45099,\n",
       "    'proteomes': 9464,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38543, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 40511},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40431,\n",
       "    'alignment:full': 8982,\n",
       "    'alignment:seed': 445},\n",
       "   'cross_references': {}}},\n",
       " 'PF04377': {'metadata': {'accession': 'PF04377',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007472',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Arginine-tRNA-protein transferase, C terminus',\n",
       "    'short': 'ATE_C'},\n",
       "   'description': ['<p>This family represents the C terminal region of the enzyme  arginine-tRNA-protein transferase (EC 2.3.2.8), which catalyses the post-translational conjugation of arginine to the N terminus of a protein. In eukaryotes, this functions as part of the N-end rule pathway  of protein degradation by conjugating a destabilising amino acid to the amino terminal aspartate or glutamate of a protein, targeting the protein for ubiquitin-dependent proteolysis.  N terminal cysteine is sometimes  modified [PMID:9858543].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009970': {'PMID': 9858543,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Alternative splicing results in differential expression, activity, and localization of the two forms of arginyl-tRNA-protein transferase, a component of the N-end rule pathway.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '182-93',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Kwon YT', 'Kashina AS', 'Varshavsky A.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9858543'}},\n",
       "   'set_info': {'accession': 'CL0257', 'name': 'Acetyltrans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 75,\n",
       "    'interactions': 0,\n",
       "    'matches': 18066,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17950,\n",
       "    'proteomes': 4722,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14947, 'rosettafold': 1},\n",
       "    'structures': 5,\n",
       "    'taxa': 18550},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15956,\n",
       "    'alignment:full': 4954,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF10431': {'metadata': {'accession': 'PF10431',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019489',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'C-terminal, D2-small domain, of ClpB protein',\n",
       "    'short': 'ClpB_D2-small'},\n",
       "   'description': ['<p>This is the C-terminal domain of ClpB protein, referred to as the D2-small domain, and is a mixed alpha-beta structure. Compared with the D1-small domain (included in AAA, Pfam:PF00004) it lacks the long coiled-coil insertion, and instead of helix C4 contains a beta-strand (e3) that is part of a three stranded beta-pleated sheet. In Thermophilus the whole protein forms a hexamer with the D1-small and D2-small domains located on the outside of the hexamer, with the long coiled-coil being exposed on the surface. The D2-small domain is essential for oligomerisation, forming a tight interface with the D2-large domain of a neighbouring subunit and thereby providing enough binding energy to stabilise the functional assembly [PMID:14567920]. The domain is associated with two Clp_N, Pfam:PF02861, at the N-terminus as well as AAA, Pfam:PF00004 and AAA_2, Pfam:PF07724.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00030406': {'PMID': 14567920,\n",
       "     'ISBN': None,\n",
       "     'volume': '115',\n",
       "     'issue': '2',\n",
       "     'year': 2003,\n",
       "     'title': 'The structure of ClpB: a molecular chaperone that rescues proteins from an aggregated state.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '229-40',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Lee S',\n",
       "      'Sowa ME',\n",
       "      'Watanabe YH',\n",
       "      'Sigler PB',\n",
       "      'Chiu W',\n",
       "      'Yoshida M',\n",
       "      'Tsai FT.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(03)00807-9'}},\n",
       "   'set_info': {'accession': 'CL0671', 'name': 'AAA_lid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 475,\n",
       "    'interactions': 0,\n",
       "    'matches': 183497,\n",
       "    'pathways': 0,\n",
       "    'proteins': 183460,\n",
       "    'proteomes': 10920,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 157484, 'rosettafold': 0},\n",
       "    'structures': 133,\n",
       "    'taxa': 46578},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 169016,\n",
       "    'alignment:full': 37267,\n",
       "    'alignment:seed': 482},\n",
       "   'cross_references': {}}},\n",
       " 'PF16123': {'metadata': {'accession': 'PF16123',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032282',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hydroxyacylglutathione hydrolase C-terminus',\n",
       "    'short': 'HAGH_C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of hydroxyacylglutathione hydrolase enzymes. Substrate binding occurs at the interface between this domain and the catalytic domain (Pfam:PF00753) [1-3].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0381', 'name': 'Metallo-HOrase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 144,\n",
       "    'interactions': 0,\n",
       "    'matches': 23043,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23022,\n",
       "    'proteomes': 5217,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19517, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 21968},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20983,\n",
       "    'alignment:full': 5941,\n",
       "    'alignment:seed': 427},\n",
       "   'cross_references': {}}},\n",
       " 'PF04060': {'metadata': {'accession': 'PF04060',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007202',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative Fe-S cluster', 'short': 'FeS'},\n",
       "   'description': ['<p>This family includes a domain with four conserved cysteines that probably form an Fe-S redox cluster.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0344', 'name': '4Fe-4S'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 380,\n",
       "    'interactions': 0,\n",
       "    'matches': 26680,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26647,\n",
       "    'proteomes': 3079,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 22624, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 15283},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23590,\n",
       "    'alignment:full': 4422,\n",
       "    'alignment:seed': 353},\n",
       "   'cross_references': {}}},\n",
       " 'PF18345': {'metadata': {'accession': 'PF18345',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Zinc finger domain', 'short': 'zf_CCCH_4'},\n",
       "   'description': ['<p>This is a zinc finger domain found in Zinc finger CCCH-type with G patch domain-containing proteins such as ZIP. Functional studies indicate that ZIP specifically targets EGFR and represses its transcription, and that the zinc finger and the coiled-coil domains are central to that process [PMID:19644445].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0537', 'name': 'CCCH_zf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 174,\n",
       "    'interactions': 0,\n",
       "    'matches': 1998,\n",
       "    'pathways': 0,\n",
       "    'proteins': 1969,\n",
       "    'proteomes': 785,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1593, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 3688},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 1599,\n",
       "    'alignment:full': 1081,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF03489': {'metadata': {'accession': 'PF03489',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008138',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Saposin-like type B, region 2', 'short': 'SapB_2'},\n",
       "   'description': ['<p>Saposin B is a small non-enzymatic glycoprotein required for the breakdown of cerebroside sulphates (sulphatides) in lysosomes. Saposin B contains three intramolecular disulphide bridges, exists as a dimer and is remarkably heat, protease and pH stable. The crystal structure of human saposin B reveals an unusual shell-like dimer consisting of a monolayer of alpha-helices enclosing a large hydrophobic cavity [PMID:7610480, PMID:12518053]. It is one of the most studied members of the saposin protein family and it is involved in the hydrolysis of glycolipids and glycerolipids. SapB is unique in the saposin family in that it facilitates degradation by interacting with the substrate, not the enzymes [PMID:26616259].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005742': {'PMID': 7610480,\n",
       "     'ISBN': None,\n",
       "     'volume': '20',\n",
       "     'issue': '5',\n",
       "     'year': 1995,\n",
       "     'title': 'Swaposins: circular permutations within genes encoding saposin homologues.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '179-80',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Ponting CP', 'Russell RB.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)89003-9'}},\n",
       "   'set_info': {'accession': 'CL0707', 'name': 'Saposin_like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 463,\n",
       "    'interactions': 0,\n",
       "    'matches': 15908,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7949,\n",
       "    'proteomes': 1171,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 6076, 'rosettafold': 0},\n",
       "    'structures': 39,\n",
       "    'taxa': 4791},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11210,\n",
       "    'alignment:full': 6454,\n",
       "    'alignment:seed': 461},\n",
       "   'cross_references': {}}},\n",
       " 'PF03028': {'metadata': {'accession': 'PF03028',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004273',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dynein heavy chain region D6 P-loop domain',\n",
       "    'short': 'Dynein_heavy'},\n",
       "   'description': ['<p>This family represents the C-terminal region of dynein heavy chain.  The chain also contains ATPase activity and microtubule binding ability and acts as a motor for the movement of organelles and vesicles along microtubules. Dynein is also involved in cilia and flagella movement. The dynein subunit consists of at least two heavy chains and a number of intermediate and light chains [PMID:7866389]. The 380 kDa motor unit of dynein belongs to the AAA class of chaperone-like ATPases. The core of the 380 kDa motor unit contains a concatenated chain of six AAA modules, of which four correspond to the ATP binding sites with P-loop signatures described previously, and two are modules in which the P loop has been lost in evolution. This C-terminal domain carries the D6 region of the dynein motor where the P-loop has been lost in evolution but the general structure of a potential ATP binding site appears to be retained [PMID:11250194].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019481': {'PMID': 7866389,\n",
       "     'ISBN': None,\n",
       "     'volume': '41',\n",
       "     'issue': '6',\n",
       "     'year': 1994,\n",
       "     'title': 'Molecular characterization of a cytoplasmic dynein from Dictyostelium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '645-51',\n",
       "     'medline_journal': 'J Eukaryot Microbiol',\n",
       "     'ISO_journal': 'J. Eukaryot. Microbiol.',\n",
       "     'authors': ['Koonce MP',\n",
       "      'Grissom PM',\n",
       "      'Lyon M',\n",
       "      'Pope T',\n",
       "      'McIntosh JR.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1315,\n",
       "    'interactions': 0,\n",
       "    'matches': 24741,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24654,\n",
       "    'proteomes': 1919,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 3231, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 7129},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17220,\n",
       "    'alignment:full': 10272,\n",
       "    'alignment:seed': 355},\n",
       "   'cross_references': {}}},\n",
       " 'PF18076': {'metadata': {'accession': 'PF18076',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040707',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Formylglycinamide ribonucleotide amidotransferase N-terminal',\n",
       "    'short': 'FGAR-AT_N'},\n",
       "   'description': ['<p>This is the N-terminal domain found in Formylglycinamide ribonucleotide amidotransferase (FGAR-AT), also known as Phosphoribosylformylglycinamidine synthase (EC:6.3.5.3), PurL and formylglycinamidine ribonucleotide (FGAM) synthase. This enzyme catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide and glutamine to formylglycinamidine ribonucleotide, ADP, Pi, and glutamate in the fourth step of the purine biosynthetic pathway [PMID:24223728].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 77,\n",
       "    'interactions': 0,\n",
       "    'matches': 16778,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16776,\n",
       "    'proteomes': 3814,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4131, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 16615},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16057,\n",
       "    'alignment:full': 3787,\n",
       "    'alignment:seed': 412},\n",
       "   'cross_references': {}}},\n",
       " 'PF13288': {'metadata': {'accession': 'PF13288',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026877',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DXP reductoisomerase C-terminal domain',\n",
       "    'short': 'DXPR_C'},\n",
       "   'description': ['<p>This is the C-terminal domain of the 1-deoxy-D-xylulose-5-phosphate reductoisomerase enzyme. This domain forms a left handed super-helix.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 63,\n",
       "    'interactions': 0,\n",
       "    'matches': 38898,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38895,\n",
       "    'proteomes': 7604,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33312, 'rosettafold': 0},\n",
       "    'structures': 70,\n",
       "    'taxa': 31544},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35895,\n",
       "    'alignment:full': 7118,\n",
       "    'alignment:seed': 732},\n",
       "   'cross_references': {}}},\n",
       " 'PF00338': {'metadata': {'accession': 'PF00338',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027486',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S10p/S20e', 'short': 'Ribosomal_S10'},\n",
       "   'description': ['<p>This family includes small ribosomal subunit S10 from prokaryotes and S20 from eukaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 179,\n",
       "    'interactions': 0,\n",
       "    'matches': 52969,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52924,\n",
       "    'proteomes': 11106,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45662, 'rosettafold': 0},\n",
       "    'structures': 1187,\n",
       "    'taxa': 49585},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46835,\n",
       "    'alignment:full': 12938,\n",
       "    'alignment:seed': 343},\n",
       "   'cross_references': {}}},\n",
       " 'PF01804': {'metadata': {'accession': 'PF01804',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002692',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Penicillin amidase', 'short': 'Penicil_amidase'},\n",
       "   'description': ['<p>Penicillin amidase or penicillin acylase EC:3.5.1.11 catalyses the hydrolysis of benzylpenicillin to phenylacetic acid and 6-aminopenicillanic acid (6-APA) a key intermediate in the the synthesis of penicillins [PMID:9292993]. Also in the family is cephalosporin acylase Swiss:P07662 and  Swiss:P29958 aculeacin A acylase which are involved in the synthesis of related peptide antibiotics.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000144': {'PMID': 9292993,\n",
       "     'ISBN': None,\n",
       "     'volume': '63',\n",
       "     'issue': '9',\n",
       "     'year': 1997,\n",
       "     'title': 'Molecular cloning and analysis of the gene encoding the thermostable penicillin G acylase from Alcaligenes faecalis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3412-8',\n",
       "     'medline_journal': 'Appl Environ Microbiol',\n",
       "     'ISO_journal': 'Appl. Environ. Microbiol.',\n",
       "     'authors': ['Verhaert RM',\n",
       "      'Riemens AM',\n",
       "      'van der Laan JM',\n",
       "      'van Duin J',\n",
       "      'Quax WJ.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=9292993'},\n",
       "    'PUB00004199': {'PMID': 7816145,\n",
       "     'ISBN': None,\n",
       "     'volume': '373',\n",
       "     'issue': '6511',\n",
       "     'year': 1995,\n",
       "     'title': 'Penicillin acylase has a single-amino-acid catalytic centre.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '264-8',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Duggleby HJ',\n",
       "      'Tolley SP',\n",
       "      'Hill CP',\n",
       "      'Dodson EJ',\n",
       "      'Dodson G',\n",
       "      'Moody PC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/373264a0'}},\n",
       "   'set_info': {'accession': 'CL0052', 'name': 'NTN'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 86,\n",
       "    'interactions': 0,\n",
       "    'matches': 31943,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31619,\n",
       "    'proteomes': 3613,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25917, 'rosettafold': 0},\n",
       "    'structures': 77,\n",
       "    'taxa': 15177},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28917,\n",
       "    'alignment:full': 5513,\n",
       "    'alignment:seed': 288},\n",
       "   'cross_references': {}}},\n",
       " 'PF17941': {'metadata': {'accession': 'PF17941',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041108',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polyphosphate kinase C-terminal domain 1',\n",
       "    'short': 'PP_kinase_C_1'},\n",
       "   'description': ['<p>Polyphosphate kinase (Ppk) catalyses the formation of polyphosphate from ATP, with chain lengths of up to a thousand or more orthophosphate molecules. This C1-terminal domain has a structure similar to phospholipase D. It is one of two closely related carboxy-terminal domains (C1 and C2 domains). Both the C1 and C2 domains (residues 322-502 and 503-687, respectively) consist of a sevenstranded mixed beta-sheet flanked by five alpha-helices. However, the structural topology and relative orientations of the helices to the beta-sheet in these two domains are different. The C1 and C2 domains are highly conserved in the PPK family. Some of the residues previously shown to be crucial for the enzyme catalytic activity are located in these two domains [PMID:15947782].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0479', 'name': 'PLD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 60,\n",
       "    'interactions': 0,\n",
       "    'matches': 36493,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36490,\n",
       "    'proteomes': 6784,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31821, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 28040},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33877,\n",
       "    'alignment:full': 6629,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF04961': {'metadata': {'accession': 'PF04961',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007044',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Formiminotransferase-cyclodeaminase', 'short': 'FTCD_C'},\n",
       "   'description': ['<p>Members of this family are thought to be Formiminotransferase- cyclodeaminase enzymes EC:4.3.1.4. This domain is found in  the C-terminus of the bifunctional animal members of the family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019582': {'PMID': 7654689,\n",
       "     'ISBN': None,\n",
       "     'volume': '34',\n",
       "     'issue': '33',\n",
       "     'year': 1995,\n",
       "     'title': 'The two monofunctional domains of octameric formiminotransferase-cyclodeaminase exist as dimers.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10358-64',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Murley LL', 'MacKenzie RE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00033a006'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 11740,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11673,\n",
       "    'proteomes': 2119,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9546, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 9867},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9817,\n",
       "    'alignment:full': 1999,\n",
       "    'alignment:seed': 232},\n",
       "   'cross_references': {}}},\n",
       " 'PF01245': {'metadata': {'accession': 'PF01245',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001857',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L19', 'short': 'Ribosomal_L19'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0107', 'name': 'KOW'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 84,\n",
       "    'interactions': 0,\n",
       "    'matches': 46328,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46317,\n",
       "    'proteomes': 10656,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39614, 'rosettafold': 0},\n",
       "    'structures': 805,\n",
       "    'taxa': 45347},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41276,\n",
       "    'alignment:full': 9922,\n",
       "    'alignment:seed': 381},\n",
       "   'cross_references': {}}},\n",
       " 'PF07719': {'metadata': {'accession': 'PF07719',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013105',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetratricopeptide repeat', 'short': 'TPR_2'},\n",
       "   'description': ['<p>This Pfam entry includes outlying Tetratricopeptide-like repeats (TPR) that are not matched by Pfam:PF00515.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001313': {'PMID': 9482716,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '5',\n",
       "     'year': 1998,\n",
       "     'title': 'The structure of the tetratricopeptide repeats of protein phosphatase 5: implications for TPR-mediated protein-protein interactions.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1192-9',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Das AK', 'Cohen PW', 'Barford D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/17.5.1192'},\n",
       "    'PUB00005443': {'PMID': 7667876,\n",
       "     'ISBN': None,\n",
       "     'volume': '20',\n",
       "     'issue': '7',\n",
       "     'year': 1995,\n",
       "     'title': 'Tetratrico peptide repeat interactions: to TPR or not to TPR?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '257-9',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Lamb JR', 'Tugendreich S', 'Hieter P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)89037-4'}},\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 14809,\n",
       "    'interactions': 0,\n",
       "    'matches': 78421,\n",
       "    'pathways': 0,\n",
       "    'proteins': 75222,\n",
       "    'proteomes': 6253,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 59738, 'rosettafold': 0},\n",
       "    'structures': 31,\n",
       "    'taxa': 28873},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 62257,\n",
       "    'alignment:full': 17835,\n",
       "    'alignment:seed': 361},\n",
       "   'cross_references': {}}},\n",
       " 'PF14508': {'metadata': {'accession': 'PF14508',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029486',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl-hydrolase 97 N-terminal', 'short': 'GH97_N'},\n",
       "   'description': ['<p>This N-terminal domain of glycosyl-hydrolase-97 [PMID:16131397]contributes part of the active site pocket. It is also important for contact with the catalytic and C-terminal domains of the whole [PMID:18848471, PMID:18981178].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 135,\n",
       "    'interactions': 0,\n",
       "    'matches': 13029,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13018,\n",
       "    'proteomes': 1318,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11424, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 5755},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12032,\n",
       "    'alignment:full': 2525,\n",
       "    'alignment:seed': 236},\n",
       "   'cross_references': {}}},\n",
       " 'PF12775': {'metadata': {'accession': 'PF12775',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'P-loop containing dynein motor region', 'short': 'AAA_7'},\n",
       "   'description': [\"<p>This domain is found in human cytoplasmic dynein-2 proteins. Cytoplasmic dynein-2 (dynein-2) performs intraflagellar transport and is associated with human skeletal ciliopathies. Dyneins share a conserved motor domain that couples cycles of ATP hydrolysis with conformational changes to produce movement. Structural analysis reveal that the motor's ring consists of six AAA+ domains (ATPases associated with various cellular activities (AAA1-AAA6). This is the third nucleotide binding sites in the dynein motor. However, AAA3 has lost the catalytic residues necessary for ATP hydrolysis (the Walker B glutamate, the arginine finger, sensor-I and sensor-II motifs) [PMID:25470043].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1751,\n",
       "    'interactions': 0,\n",
       "    'matches': 24917,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24344,\n",
       "    'proteomes': 1929,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 1606, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 7213},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17104,\n",
       "    'alignment:full': 10575,\n",
       "    'alignment:seed': 296},\n",
       "   'cross_references': {}}},\n",
       " 'PF00318': {'metadata': {'accession': 'PF00318',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001865',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S2', 'short': 'Ribosomal_S2'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0067', 'name': 'SIS'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 213,\n",
       "    'interactions': 0,\n",
       "    'matches': 78466,\n",
       "    'pathways': 0,\n",
       "    'proteins': 70019,\n",
       "    'proteomes': 11097,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 59564, 'rosettafold': 0},\n",
       "    'structures': 1131,\n",
       "    'taxa': 66046},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 68244,\n",
       "    'alignment:full': 15891,\n",
       "    'alignment:seed': 367},\n",
       "   'cross_references': {}}},\n",
       " 'PF02325': {'metadata': {'accession': 'PF02325',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003425',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YGGT family', 'short': 'YGGT'},\n",
       "   'description': ['<p>This family includes subunit CCB3 of cofactor assembly complex C, involved in c-type cytochrome maturation in photosynthetic organisms [PMID:18593701]. This family also includes uncharacterised protein YggT, associated with bacteria outside the cyanobacteria.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019417': {'PMID': 9846742,\n",
       "     'ISBN': None,\n",
       "     'volume': '144 ( Pt 11)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'Unconventional organization of the division and cell wall gene cluster of Streptococcus pneumoniae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3069-78',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Massidda O', 'Anderluzzi D', 'Friedli L', 'Feger G.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 49,\n",
       "    'interactions': 0,\n",
       "    'matches': 41515,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33584,\n",
       "    'proteomes': 6589,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 29124, 'rosettafold': 1},\n",
       "    'structures': 2,\n",
       "    'taxa': 28719},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38570,\n",
       "    'alignment:full': 8107,\n",
       "    'alignment:seed': 669},\n",
       "   'cross_references': {}}},\n",
       " 'PF00831': {'metadata': {'accession': 'PF00831',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001854',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal L29 protein', 'short': 'Ribosomal_L29'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0346', 'name': 'Ribo_L29'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 87,\n",
       "    'interactions': 0,\n",
       "    'matches': 48326,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48316,\n",
       "    'proteomes': 10766,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41642, 'rosettafold': 0},\n",
       "    'structures': 1109,\n",
       "    'taxa': 47260},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42936,\n",
       "    'alignment:full': 10590,\n",
       "    'alignment:seed': 712},\n",
       "   'cross_references': {}}},\n",
       " 'PF02775': {'metadata': {'accession': 'PF02775',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011766',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Thiamine pyrophosphate enzyme, C-terminal TPP binding domain',\n",
       "    'short': 'TPP_enzyme_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015103': {'PMID': 8604141,\n",
       "     'ISBN': None,\n",
       "     'volume': '256',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Crystal structure of the thiamin diphosphate-dependent enzyme pyruvate decarboxylase from the yeast Saccharomyces cerevisiae at 2.3 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '590-600',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Arjunan P',\n",
       "      'Umland T',\n",
       "      'Dyda F',\n",
       "      'Swaminathan S',\n",
       "      'Furey W',\n",
       "      'Sax M',\n",
       "      'Farrenkopf B',\n",
       "      'Gao Y',\n",
       "      'Zhang D',\n",
       "      'Jordan F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1996.0111'}},\n",
       "   'set_info': {'accession': 'CL0254', 'name': 'THDP-binding'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 713,\n",
       "    'interactions': 0,\n",
       "    'matches': 257666,\n",
       "    'pathways': 0,\n",
       "    'proteins': 257582,\n",
       "    'proteomes': 10849,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 219107, 'rosettafold': 0},\n",
       "    'structures': 219,\n",
       "    'taxa': 46607},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 236127,\n",
       "    'alignment:full': 48448,\n",
       "    'alignment:seed': 323},\n",
       "   'cross_references': {}}},\n",
       " 'PF03124': {'metadata': {'accession': 'PF03124',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004342',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'EXS family', 'short': 'EXS'},\n",
       "   'description': [\"<p>We have named this region the EXS family after (ERD1, XPR1, and SYG1). This family includes C-terminus portions from the SYG1 G-protein associated signal transduction protein from Saccharomyces cerevisiae, and sequences that are thought to be murine leukaemia virus (MLV)  receptors (XPR1). N-terminus portions from these proteins are aligned in the SPX  Pfam:PF03105 family. The previously noted similarity between SYG1 and MLV receptors over their whole sequences [PMID:9990033] is thus borne out in Pfam:PF03105 and this family. While the N-termini aligned in Pfam:PF03105 are thought to be involved in signal transduction, the role of the C-terminus sequences aligned in this family is not known. This region of similarity contains several predicted transmembrane helices. This family also includes the ERD1 (ERD: ER retention  defective) yeast proteins Swiss:P16151.  ERD1 proteins are involved in the localisation of endogenous endoplasmic reticulum (ER) proteins. erd1 null mutants secrete such proteins even though they possess the C-terminal HDEL ER lumen localisation label sequence. In addition, null mutants also exhibit defects in the Golgi-dependent processing of several glycoproteins, which led to the suggestion that the sorting of luminal ER proteins actually occurs in the Golgi, with subsequent return of these proteins to the ER via  `salvage' vesicles [PMID:2178921].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007516': {'PMID': 9990033,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '4',\n",
       "     'year': 1999,\n",
       "     'title': 'A human cell-surface receptor for xenotropic and polytropic murine leukemia viruses: possible role in G protein-coupled signal transduction.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1385-90',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Battini JL', 'Rasko JE', 'Miller AD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.96.4.1385'},\n",
       "    'PUB00007533': {'PMID': 2178921,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '3',\n",
       "     'year': 1990,\n",
       "     'title': 'ERD1, a yeast gene required for the retention of luminal endoplasmic reticulum proteins, affects glycoprotein processing in the Golgi apparatus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '623-30',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Hardwick KG',\n",
       "      'Lewis MJ',\n",
       "      'Semenza J',\n",
       "      'Dean N',\n",
       "      'Pelham HR.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=2178921'}},\n",
       "   'set_info': {'accession': 'CL0182', 'name': 'IT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 136,\n",
       "    'interactions': 0,\n",
       "    'matches': 10104,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9543,\n",
       "    'proteomes': 2091,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7823, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7666},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8429,\n",
       "    'alignment:full': 5360,\n",
       "    'alignment:seed': 300},\n",
       "   'cross_references': {}}},\n",
       " 'PF08393': {'metadata': {'accession': 'PF08393',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013602',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dynein heavy chain, N-terminal region 2',\n",
       "    'short': 'DHC_N2'},\n",
       "   'description': ['<p>Dyneins are described as motor proteins of eukaryotic cells, as they can convert energy derived from the hydrolysis of ATP to force and movement along cytoskeletal polymers, such as microtubules. This region is found C-terminal to the dynein heavy chain N-terminal region 1 (Pfam:PF08385) in many members of this family. No functions seem to have been attributed specifically to this region.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1576,\n",
       "    'interactions': 0,\n",
       "    'matches': 26086,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25145,\n",
       "    'proteomes': 1930,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 2567, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 7162},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18129,\n",
       "    'alignment:full': 11107,\n",
       "    'alignment:seed': 523},\n",
       "   'cross_references': {}}},\n",
       " 'PF00174': {'metadata': {'accession': 'PF00174',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000572',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Oxidoreductase molybdopterin binding domain',\n",
       "    'short': 'Oxidored_molyb'},\n",
       "   'description': ['<p>This domain is found in a variety of oxidoreductases. This domain binds to a molybdopterin cofactor. Xanthine dehydrogenases, that also bind molybdopterin, have essentially no similarity.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007725': {'PMID': 9428520,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '7',\n",
       "     'year': 1997,\n",
       "     'title': 'Molecular basis of sulfite oxidase deficiency from the structure of sulfite oxidase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '973-83',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Kisker C',\n",
       "      'Schindelin H',\n",
       "      'Pacheco A',\n",
       "      'Wehbi WA',\n",
       "      'Garrett RM',\n",
       "      'Rajagopalan KV',\n",
       "      'Enemark JH',\n",
       "      'Rees DC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80488-2'}},\n",
       "   'set_info': {'accession': 'CL0072', 'name': 'Ubiquitin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 238,\n",
       "    'interactions': 0,\n",
       "    'matches': 63710,\n",
       "    'pathways': 0,\n",
       "    'proteins': 63599,\n",
       "    'proteomes': 6633,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 54584, 'rosettafold': 0},\n",
       "    'structures': 28,\n",
       "    'taxa': 27670},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 58052,\n",
       "    'alignment:full': 14151,\n",
       "    'alignment:seed': 536},\n",
       "   'cross_references': {}}},\n",
       " 'PF00241': {'metadata': {'accession': 'PF00241',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002108',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cofilin/tropomyosin-type actin-binding protein',\n",
       "    'short': 'Cofilin_ADF'},\n",
       "   'description': ['<p>Severs actin filaments and binds to actin monomers.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003942': {'PMID': 9145106,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '5',\n",
       "     'year': 1997,\n",
       "     'title': 'Structure determination of yeast cofilin.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '366-9',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Fedorov AA',\n",
       "      'Lappalainen P',\n",
       "      'Fedorov EV',\n",
       "      'Drubin DG',\n",
       "      'Almo SC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0597-366'},\n",
       "    'PUB00003943': {'PMID': 9145107,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '5',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure of the actin-binding protein actophorin from Acanthamoeba.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '369-73',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Leonard SA',\n",
       "      'Gittis AG',\n",
       "      'Petrella EC',\n",
       "      'Pollard TD',\n",
       "      'Lattman EE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0597-369'},\n",
       "    'PUB00004259': {'PMID': 9214506,\n",
       "     'ISBN': None,\n",
       "     'volume': '388',\n",
       "     'issue': '6637',\n",
       "     'year': 1997,\n",
       "     'title': 'Cofilin promotes rapid actin filament turnover in vivo.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '78-82',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Lappalainen P', 'Drubin DG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/40418'},\n",
       "    'PUB00004915': {'PMID': 9275236,\n",
       "     'ISBN': None,\n",
       "     'volume': '94',\n",
       "     'issue': '18',\n",
       "     'year': 1997,\n",
       "     'title': 'F-actin and G-actin binding are uncoupled by mutation of conserved tyrosine residues in maize actin depolymerizing factor (ZmADF).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9973-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Jiang CJ', 'Weeds AG', 'Khan S', 'Hussey PJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.94.18.9973'}},\n",
       "   'set_info': {'accession': 'CL0092', 'name': 'ADF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 339,\n",
       "    'interactions': 0,\n",
       "    'matches': 27362,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23279,\n",
       "    'proteomes': 2194,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18676, 'rosettafold': 0},\n",
       "    'structures': 67,\n",
       "    'taxa': 8346},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21786,\n",
       "    'alignment:full': 13440,\n",
       "    'alignment:seed': 440},\n",
       "   'cross_references': {}}},\n",
       " 'PF05033': {'metadata': {'accession': 'PF05033',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007728',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pre-SET motif', 'short': 'Pre-SET'},\n",
       "   'description': ['<p>This protein motif is a zinc binding motif [PMID:12389037].  It  contains 9 conserved cysteines that coordinate three zinc ions. It is thought that this region plays a structural role in stabilising SET domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010205': {'PMID': 12389037,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '11',\n",
       "     'year': 2002,\n",
       "     'title': 'Structure of the SET domain histone lysine methyltransferase Clr4.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '828-32',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Min J', 'Zhang X', 'Cheng X', 'Grewal SI', 'Xu RM.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 310,\n",
       "    'interactions': 0,\n",
       "    'matches': 15179,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15161,\n",
       "    'proteomes': 1827,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10136, 'rosettafold': 0},\n",
       "    'structures': 55,\n",
       "    'taxa': 6656},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11434,\n",
       "    'alignment:full': 7727,\n",
       "    'alignment:seed': 298},\n",
       "   'cross_references': {}}},\n",
       " 'PF14416': {'metadata': {'accession': 'PF14416',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025846',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PMR5 N terminal Domain', 'short': 'PMR5N'},\n",
       "   'description': ['<p>The plant family with PMR5, ESK1, TBL3 etc have a N-terminal   C rich predicted sugar binding domain followed by the PC-Esterase (acyl esterase) domain [PMID:20056006].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 206,\n",
       "    'interactions': 0,\n",
       "    'matches': 16066,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15832,\n",
       "    'proteomes': 203,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13462, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 780},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13608,\n",
       "    'alignment:full': 9539,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF16193': {'metadata': {'accession': 'PF16193',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032423',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AAA C-terminal domain', 'short': 'AAA_assoc_2'},\n",
       "   'description': ['<p>AAA_assoc_2 is found at the C-terminus of a relatively small set of AAA domains in proteins ranging from archaeal to fungi, plants and mammals.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0671', 'name': 'AAA_lid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 111,\n",
       "    'interactions': 0,\n",
       "    'matches': 45840,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45827,\n",
       "    'proteomes': 9665,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39408, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 38942},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42101,\n",
       "    'alignment:full': 9239,\n",
       "    'alignment:seed': 659},\n",
       "   'cross_references': {}}},\n",
       " 'PF03461': {'metadata': {'accession': 'PF03461',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005118',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TRCF domain', 'short': 'TRCF'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 74,\n",
       "    'interactions': 0,\n",
       "    'matches': 45037,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45033,\n",
       "    'proteomes': 8424,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38321, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 34478},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41930,\n",
       "    'alignment:full': 7923,\n",
       "    'alignment:seed': 514},\n",
       "   'cross_references': {}}},\n",
       " 'PF03990': {'metadata': {'accession': 'PF03990',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007137',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'G5-linked-Ubiquitin-like domain', 'short': 'DUF348'},\n",
       "   'description': ['<p>This domain normally occurs as tandem repeats; however it is found as a single copy in the S. cerevisiae DNA-binding nuclear protein YCR593 (Swiss:P25357). This protein is involved in sporulation part of the SET3C complex, which is required to repress early/middle sporulation genes during meiosis ([PMID:11711434]). The bacterial proteins are likely to be involved in a cell wall function as they are found in conjunction with the Pfam:PF07501 domain, which is involved in various cell surface processes. This domain is also present in the resuscitation-promoting factors RpfB from Mycobacterium tuberculosis and Rpf2 from Corynebacterium glutamicum. These are factors that stimulate resuscitation of dormant cells [PMID:19799629]. This domain has a beta grasp fold. Structural description of this domain revealed a structural conservation between these domains and ubiquitin, hence it is termed UBL-G5 [PMID:26549874].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'},\n",
       "    'PUB00019472': {'PMID': 11711434,\n",
       "     'ISBN': None,\n",
       "     'volume': '15',\n",
       "     'issue': '22',\n",
       "     'year': 2001,\n",
       "     'title': 'The S. cerevisiae SET3 complex includes two histone deacetylases, Hos2 and Hst1, and is a meiotic-specific repressor of the sporulation gene program.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2991-3004',\n",
       "     'medline_journal': 'Genes Dev',\n",
       "     'ISO_journal': 'Genes Dev.',\n",
       "     'authors': ['Pijnappel WW',\n",
       "      'Schaft D',\n",
       "      'Roguev A',\n",
       "      'Shevchenko A',\n",
       "      'Tekotte H',\n",
       "      'Wilm M',\n",
       "      'Rigaut G',\n",
       "      'Seraphin B',\n",
       "      'Aasland R',\n",
       "      'Stewart AF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1101/gad.207401'}},\n",
       "   'set_info': {'accession': 'CL0072', 'name': 'Ubiquitin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 20838,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8718,\n",
       "    'proteomes': 2090,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7529, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 7054},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19450,\n",
       "    'alignment:full': 5301,\n",
       "    'alignment:seed': 355},\n",
       "   'cross_references': {}}},\n",
       " 'PF16886': {'metadata': {'accession': 'PF16886',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031686',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATPsynthase alpha/beta subunit N-term extension',\n",
       "    'short': 'ATP-synt_ab_Xtn'},\n",
       "   'description': ['<p>ATP-synt_ab_Xtn is an extension of the alpha-beta catalytic subunit of VATA or V-type proton ATPase catalytic subunit at the N-terminal end. It is found from bacteria to humans, and was not modelled in family ATP-synt_ab, Pfam:PF00006.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 114,\n",
       "    'interactions': 0,\n",
       "    'matches': 14119,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14099,\n",
       "    'proteomes': 3333,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11732, 'rosettafold': 0},\n",
       "    'structures': 125,\n",
       "    'taxa': 14831},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11831,\n",
       "    'alignment:full': 3634,\n",
       "    'alignment:seed': 364},\n",
       "   'cross_references': {}}},\n",
       " 'PF16177': {'metadata': {'accession': 'PF16177',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032387',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acetyl-coenzyme A synthetase N-terminus',\n",
       "    'short': 'ACAS_N'},\n",
       "   'description': ['<p>This domain is found at the N-terminus of many acetyl-coenzyme A synthetase enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0378', 'name': 'ANL'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 176,\n",
       "    'interactions': 0,\n",
       "    'matches': 71781,\n",
       "    'pathways': 0,\n",
       "    'proteins': 71726,\n",
       "    'proteomes': 8289,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 60116, 'rosettafold': 0},\n",
       "    'structures': 35,\n",
       "    'taxa': 34033},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 64431,\n",
       "    'alignment:full': 16566,\n",
       "    'alignment:seed': 702},\n",
       "   'cross_references': {}}},\n",
       " 'PF03458': {'metadata': {'accession': 'PF03458',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005115',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycine transporter', 'short': 'Gly_transporter'},\n",
       "   'description': ['<p>This domain contains three transmembrane helices. Proteins containing this domain are important for glycine utilisation, being identified as glycine transporters. Some members of the family are also important for alanine utilisation. In these proteins this domain is found in pairs [PMID:29769716]. An archaeal member of this family which contains this domain is a TRIC-type potassium channel [PMID:28524849].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 52443,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26624,\n",
       "    'proteomes': 5615,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23988, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 23134},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51492,\n",
       "    'alignment:full': 11339,\n",
       "    'alignment:seed': 853},\n",
       "   'cross_references': {}}},\n",
       " 'PF00478': {'metadata': {'accession': 'PF00478',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001093',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'IMP dehydrogenase / GMP reductase domain',\n",
       "    'short': 'IMPDH'},\n",
       "   'description': ['<p>This family is involved in biosynthesis of guanosine nucleotide. Members of this family contain a TIM barrel  structure. In the inosine monophosphate dehydrogenases 2 CBS domains Pfam:PF00571 are inserted in the TIM barrel [PMID:10200156]. This family is a member of the common phosphate binding site TIM barrel family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000438': {'PMID': 9271497,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '35',\n",
       "     'year': 1997,\n",
       "     'title': \"Crystal structure of Tritrichomonas foetus inosine-5'-monophosphate dehydrogenase and the enzyme-product complex.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '10666-74',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Whitby FG',\n",
       "      'Luecke H',\n",
       "      'Kuhn P',\n",
       "      'Somoza JR',\n",
       "      'Huete-Perez JA',\n",
       "      'Phillips JD',\n",
       "      'Hill CP',\n",
       "      'Fletterick RJ',\n",
       "      'Wang CC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi9708850'},\n",
       "    'PUB00000457': {'PMID': 10200156,\n",
       "     'ISBN': None,\n",
       "     'volume': '38',\n",
       "     'issue': '15',\n",
       "     'year': 1999,\n",
       "     'title': \"Characteristics and crystal structure of bacterial inosine-5'-monophosphate dehydrogenase.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '4691-700',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Zhang R',\n",
       "      'Evans G',\n",
       "      'Rotella FJ',\n",
       "      'Westbrook EM',\n",
       "      'Beno D',\n",
       "      'Huberman E',\n",
       "      'Joachimiak A',\n",
       "      'Collart FR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi982858v'}},\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 199,\n",
       "    'interactions': 0,\n",
       "    'matches': 74982,\n",
       "    'pathways': 0,\n",
       "    'proteins': 74139,\n",
       "    'proteomes': 10777,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 63084, 'rosettafold': 0},\n",
       "    'structures': 146,\n",
       "    'taxa': 44252},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 67467,\n",
       "    'alignment:full': 16983,\n",
       "    'alignment:seed': 439},\n",
       "   'cross_references': {}}},\n",
       " 'PF00734': {'metadata': {'accession': 'PF00734',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000254',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fungal cellulose binding domain', 'short': 'CBM_1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0083', 'name': 'Omega_toxin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 519,\n",
       "    'interactions': 0,\n",
       "    'matches': 15511,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14273,\n",
       "    'proteomes': 636,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11410, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 2162},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12279,\n",
       "    'alignment:full': 7945,\n",
       "    'alignment:seed': 715},\n",
       "   'cross_references': {}}},\n",
       " 'PF02092': {'metadata': {'accession': 'PF02092',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015944',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycyl-tRNA synthetase beta subunit',\n",
       "    'short': 'tRNA_synt_2f'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 58,\n",
       "    'interactions': 0,\n",
       "    'matches': 27636,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27370,\n",
       "    'proteomes': 4890,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23554, 'rosettafold': 1},\n",
       "    'structures': 3,\n",
       "    'taxa': 22353},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26105,\n",
       "    'alignment:full': 4843,\n",
       "    'alignment:seed': 419},\n",
       "   'cross_references': {}}},\n",
       " 'PF12951': {'metadata': {'accession': 'PF12951',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013425',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Passenger-associated-transport-repeat', 'short': 'PATR'},\n",
       "   'description': ['<p>This Autotransporter-associated beta strand repeat model represents a core 32-residue region of a class of bacterial protein repeat found in one to 30 copies per protein. Most proteins with a copy of this repeat have domains associated with membrane autotransporters (Pfam:PF03797). The repeats occur with a periodicity of 60 to 100 residues. A pattern of sequence conservation is that every second residue is well-conserved across most of the domain.  These repeats as likely to have a beta-helical structure. This repeat plays a role in the efficient transport of autotransporter virulence factors to the bacterial surface during growth and infection. The repeat is always associated with the passenger domain of the autotransporter. For these reasons it has been coined the Passenger-associated Transport Repeat (PATR) [PMID:25869731]. The mechanism by which the PATR motif promotes transport is uncertain but it is likely that the conserved glycines (see HMM Logo) are required for flexibility of folding and that this folding drives secretion [PMID:25869731]. Autotransporters that contain PATR(s) associate with distinct virulence traits such as subtilisin (S8) type protease domains and polymorphic outer-membrane protein repeats, whilst SPATE (S6) type protease and lipase-like autotransporters do not tend to contain PATR motifs [PMID:25869731].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0268', 'name': 'Pec_lyase-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1249,\n",
       "    'interactions': 0,\n",
       "    'matches': 85942,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18827,\n",
       "    'proteomes': 1126,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11282, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 6572},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 85349,\n",
       "    'alignment:full': 11201,\n",
       "    'alignment:seed': 675},\n",
       "   'cross_references': {}}},\n",
       " 'PF04127': {'metadata': {'accession': 'PF04127',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007085',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DNA / pantothenate metabolism flavoprotein',\n",
       "    'short': 'DFP'},\n",
       "   'description': ['<p>The DNA/pantothenate metabolism flavoprotein (EC:4.1.1.36) affects synthesis of DNA, and pantothenate metabolism.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015610': {'PMID': 3123465,\n",
       "     'ISBN': None,\n",
       "     'volume': '170',\n",
       "     'issue': '2',\n",
       "     'year': 1988,\n",
       "     'title': 'beta-Alanine auxotrophy associated with dfp, a locus affecting DNA synthesis in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '872-6',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Spitzer ED', 'Jimenez-Billini HE', 'Weiss B.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=3123465&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 100,\n",
       "    'interactions': 0,\n",
       "    'matches': 50505,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48352,\n",
       "    'proteomes': 10077,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41482, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 41714},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46204,\n",
       "    'alignment:full': 10363,\n",
       "    'alignment:seed': 367},\n",
       "   'cross_references': {}}},\n",
       " 'PF13640': {'metadata': {'accession': 'PF13640',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR044862',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '2OG-Fe(II) oxygenase superfamily',\n",
       "    'short': '2OG-FeII_Oxy_3'},\n",
       "   'description': ['<p>This family contains members of the 2-oxoglutarate (2OG) and Fe(II)-dependent oxygenase superfamily [PMID:11276424].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 576,\n",
       "    'interactions': 0,\n",
       "    'matches': 57809,\n",
       "    'pathways': 0,\n",
       "    'proteins': 57639,\n",
       "    'proteomes': 5227,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46857, 'rosettafold': 0},\n",
       "    'structures': 70,\n",
       "    'taxa': 21994},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49341,\n",
       "    'alignment:full': 18586,\n",
       "    'alignment:seed': 308},\n",
       "   'cross_references': {}}},\n",
       " 'PF05195': {'metadata': {'accession': 'PF05195',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007865',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aminopeptidase P, N-terminal domain', 'short': 'AMP_N'},\n",
       "   'description': ['<p>This domain is structurally very similar [PMID:9520390] to the creatinase N-terminal domain (Pfam:PF01321). However, little or no sequence similarity exists between the two families.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010137': {'PMID': 9520390,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '7',\n",
       "     'year': 1998,\n",
       "     'title': 'Structure and mechanism of a proline-specific aminopeptidase from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3472-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Wilce MC',\n",
       "      'Bond CS',\n",
       "      'Dixon NE',\n",
       "      'Freeman HC',\n",
       "      'Guss JM',\n",
       "      'Lilley PE',\n",
       "      'Wilce JA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.95.7.3472'}},\n",
       "   'set_info': {'accession': 'CL0356', 'name': 'AMP_N-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 180,\n",
       "    'interactions': 0,\n",
       "    'matches': 33895,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33877,\n",
       "    'proteomes': 6181,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28356, 'rosettafold': 0},\n",
       "    'structures': 54,\n",
       "    'taxa': 25785},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30358,\n",
       "    'alignment:full': 8421,\n",
       "    'alignment:seed': 441},\n",
       "   'cross_references': {}}},\n",
       " 'PF01302': {'metadata': {'accession': 'PF01302',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000938',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CAP-Gly domain', 'short': 'CAP_GLY'},\n",
       "   'description': ['<p>Cytoskeleton-associated proteins (CAPs) are involved in the organisation of microtubules and transportation of vesicles and organelles along the cytoskeletal network. A conserved motif, CAP-Gly, has been identified in a number of CAPs, including CLIP-170 and dynactins. The crystal structure of Caenorhabditis elegans F53F4.3 protein Swiss:Q20728 CAP-Gly domain was recently solved [PMID:12221106]. The domain contains three beta-strands. The most conserved sequence, GKNDG, is located in two consecutive sharp turns on the surface, forming the entrance to a groove [PMID:12221106].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005410': {'PMID': 8480366,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '3',\n",
       "     'year': 1993,\n",
       "     'title': 'Sequence homologies between four cytoskeleton-associated proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '82-3',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Riehemann K', 'Sorg C.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0968-0004(93)90159-K'},\n",
       "    'PUB00015605': {'PMID': 12221106,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '50',\n",
       "     'year': 2002,\n",
       "     'title': 'Crystal structure of the cytoskeleton-associated protein glycine-rich (CAP-Gly) domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '48596-601',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Li S',\n",
       "      'Finley J',\n",
       "      'Liu ZJ',\n",
       "      'Qiu SH',\n",
       "      'Chen H',\n",
       "      'Luan CH',\n",
       "      'Carson M',\n",
       "      'Tsao J',\n",
       "      'Johnson D',\n",
       "      'Lin G',\n",
       "      'Zhao J',\n",
       "      'Thomas W',\n",
       "      'Nagy LA',\n",
       "      'Sha B',\n",
       "      'DeLucas LJ',\n",
       "      'Wang BC',\n",
       "      'Luo M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M208512200'}},\n",
       "   'set_info': {'accession': 'CL0010', 'name': 'SH3'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 559,\n",
       "    'interactions': 0,\n",
       "    'matches': 36926,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28166,\n",
       "    'proteomes': 2151,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15724, 'rosettafold': 0},\n",
       "    'structures': 39,\n",
       "    'taxa': 7898},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26851,\n",
       "    'alignment:full': 17056,\n",
       "    'alignment:seed': 593},\n",
       "   'cross_references': {}}},\n",
       " 'PF02738': {'metadata': {'accession': 'PF02738',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008274',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Molybdopterin cofactor-binding domain',\n",
       "    'short': 'MoCoBD_1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005209': {'PMID': 7502041,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '5239',\n",
       "     'year': 1995,\n",
       "     'title': 'Crystal structure of the xanthine oxidase-related aldehyde oxido-reductase from D. gigas.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1170-6',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Romao MJ',\n",
       "      'Archer M',\n",
       "      'Moura I',\n",
       "      'Moura JJ',\n",
       "      'LeGall J',\n",
       "      'Engh R',\n",
       "      'Schneider M',\n",
       "      'Hof P',\n",
       "      'Huber R.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/270/5239/1170'},\n",
       "    'PUB00019122': {'PMID': 10430865,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '16',\n",
       "     'year': 1999,\n",
       "     'title': 'Crystal structure and mechanism of CO dehydrogenase, a molybdo iron-sulfur flavoprotein containing S-selanylcysteine.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8884-9',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Dobbek H', 'Gremer L', 'Meyer O', 'Huber R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.96.16.8884'}},\n",
       "   'set_info': {'accession': 'CL0635', 'name': 'DmpA_ArgJ'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 514,\n",
       "    'interactions': 0,\n",
       "    'matches': 112046,\n",
       "    'pathways': 0,\n",
       "    'proteins': 111001,\n",
       "    'proteomes': 6787,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 88529, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 27683},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 101418,\n",
       "    'alignment:full': 22252,\n",
       "    'alignment:seed': 455},\n",
       "   'cross_references': {}}},\n",
       " 'PF03775': {'metadata': {'accession': 'PF03775',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005526',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Septum formation inhibitor MinC, C-terminal domain',\n",
       "    'short': 'MinC_C'},\n",
       "   'description': ['<p>In Escherichia coli Swiss:P06138 assembles into a Z ring at midcell while assembly at polar sites is prevented by the min system. MinC Swiss:P18196 a component of this system, is an inhibitor of FtsZ assembly that is positioned within the cell by interaction with MinDE. MinC is an oligomer, probably a dimer [PMID:10869074]. The C terminal half of MinC is the most conserved and interacts with MinD.  The N terminal half is thought interact with FtsZ.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008426': {'PMID': 10869074,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '14',\n",
       "     'year': 2000,\n",
       "     'title': 'Analysis of MinC reveals two independent domains involved in interaction with MinD and FtsZ.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3965-71',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Hu Z', 'Lutkenhaus J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.14.3965-3971.2000'},\n",
       "    'PUB00019861': {'PMID': 10611296,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '26',\n",
       "     'year': 1999,\n",
       "     'title': 'The MinC component of the division site selection system in Escherichia coli interacts with FtsZ to prevent polymerization.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14819-24',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Hu Z', 'Mukherjee A', 'Pichoff S', 'Lutkenhaus J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.96.26.14819'}},\n",
       "   'set_info': {'accession': 'CL0268', 'name': 'Pec_lyase-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 9,\n",
       "    'interactions': 0,\n",
       "    'matches': 16717,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16716,\n",
       "    'proteomes': 3331,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14969, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 15978},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16405,\n",
       "    'alignment:full': 3201,\n",
       "    'alignment:seed': 281},\n",
       "   'cross_references': {}}},\n",
       " 'PF00352': {'metadata': {'accession': 'PF00352',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000814',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transcription factor TFIID (or TATA-binding protein, TBP)',\n",
       "    'short': 'TBP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004068': {'PMID': 2374612,\n",
       "     'ISBN': None,\n",
       "     'volume': '346',\n",
       "     'issue': '6282',\n",
       "     'year': 1990,\n",
       "     'title': 'Highly conserved core domain and unique N terminus with presumptive regulatory motifs in a human TATA factor (TFIID).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '387-90',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Hoffman A',\n",
       "      'Sinn E',\n",
       "      'Yamamoto T',\n",
       "      'Wang J',\n",
       "      'Roy A',\n",
       "      'Horikoshi M',\n",
       "      'Roeder RG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/346387a0'},\n",
       "    'PUB00004069': {'PMID': 2197561,\n",
       "     'ISBN': None,\n",
       "     'volume': '346',\n",
       "     'issue': '6282',\n",
       "     'year': 1990,\n",
       "     'title': 'Arabidopsis thaliana contains two genes for TFIID.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '390-4',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Gasch A',\n",
       "      'Hoffmann A',\n",
       "      'Horikoshi M',\n",
       "      'Roeder RG',\n",
       "      'Chua NH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/346390a0'},\n",
       "    'PUB00004136': {'PMID': 1436073,\n",
       "     'ISBN': None,\n",
       "     'volume': '360',\n",
       "     'issue': '6399',\n",
       "     'year': 1992,\n",
       "     'title': 'Crystal structure of TFIID TATA-box binding protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '40-6',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Nikolov DB',\n",
       "      'Hu SH',\n",
       "      'Lin J',\n",
       "      'Gasch A',\n",
       "      'Hoffmann A',\n",
       "      'Horikoshi M',\n",
       "      'Chua NH',\n",
       "      'Roeder RG',\n",
       "      'Burley SK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/360040a0'},\n",
       "    'PUB00004231': {'PMID': 8598932,\n",
       "     'ISBN': None,\n",
       "     'volume': '380',\n",
       "     'issue': '6572',\n",
       "     'year': 1996,\n",
       "     'title': 'A histone octamer-like structure within TFIID.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '356-9',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Hoffmann A',\n",
       "      'Chiang CM',\n",
       "      'Oelgeschlager T',\n",
       "      'Xie X',\n",
       "      'Burley SK',\n",
       "      'Nakatani Y',\n",
       "      'Roeder RG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/380356a0'}},\n",
       "   'set_info': {'accession': 'CL0407', 'name': 'TBP-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 120,\n",
       "    'interactions': 0,\n",
       "    'matches': 24713,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13358,\n",
       "    'proteomes': 2503,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11079, 'rosettafold': 0},\n",
       "    'structures': 129,\n",
       "    'taxa': 10753},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19811,\n",
       "    'alignment:full': 7937,\n",
       "    'alignment:seed': 274},\n",
       "   'cross_references': {}}},\n",
       " 'PF09754': {'metadata': {'accession': 'PF09754',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019151',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PAC2 family', 'short': 'PAC2'},\n",
       "   'description': ['<p>This PAC2 (Proteasome assembly chaperone) family of proteins is found in bacteria, archaea and eukaryotes.  Proteins in this family are typically between 247 and 307 amino acids in length. These proteins function as a chaperone for the 26S proteasome. The 26S proteasome mediates ubiquitin-dependent proteolysis in eukaryotic cells. A number of studies including very recent ones have revealed that assembly of its 20S catalytic core particle is an ordered process that involves several conserved proteasome assembly chaperones (PACs). Two heterodimeric chaperones, PAC1-PAC2 and PAC3-PAC4, promote the assembly of rings composed of seven alpha subunits [PMID:18786393].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044298': {'PMID': 11854909,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '6',\n",
       "     'year': 2001,\n",
       "     'title': 'Expression of liver cancer associated gene HCCA3.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '821-5',\n",
       "     'medline_journal': 'World J Gastroenterol',\n",
       "     'ISO_journal': 'World J. Gastroenterol.',\n",
       "     'authors': ['Wang ZX', 'Hu GF', 'Wang HY', 'Wu MC.'],\n",
       "     'DOI_URL': 'http://www.wjgnet.com/abstract.asp?url=/1007-9327/7/821'}},\n",
       "   'set_info': {'accession': 'CL0408', 'name': 'PUP'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 23395,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23384,\n",
       "    'proteomes': 3493,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20013, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 13587},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20160,\n",
       "    'alignment:full': 4552,\n",
       "    'alignment:seed': 338},\n",
       "   'cross_references': {}}},\n",
       " 'PF00771': {'metadata': {'accession': 'PF00771',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001712',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FHIPEP family', 'short': 'FHIPEP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 52,\n",
       "    'interactions': 0,\n",
       "    'matches': 29418,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28903,\n",
       "    'proteomes': 4244,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25194, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 18397},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28557,\n",
       "    'alignment:full': 4856,\n",
       "    'alignment:seed': 375},\n",
       "   'cross_references': {}}},\n",
       " 'PF01805': {'metadata': {'accession': 'PF01805',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000061',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Surp module', 'short': 'Surp'},\n",
       "   'description': ['<p>This domain is also known as the SWAP domain. SWAP stands for Suppressor-of-White-APricot. It has been suggested that these domains may be RNA binding [PMID:8206918].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002852': {'PMID': 8206918,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '23',\n",
       "     'year': 1994,\n",
       "     'title': 'Conservation of regulated alternative splicing and identification of functional domains in vertebrate homologs to the Drosophila splicing regulator, suppressor-of-white-apricot.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '16170-9',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Denhez F', 'Lafyatis R.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/23/16170'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 281,\n",
       "    'interactions': 0,\n",
       "    'matches': 21578,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15176,\n",
       "    'proteomes': 2135,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11747, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 7793},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16796,\n",
       "    'alignment:full': 10240,\n",
       "    'alignment:seed': 312},\n",
       "   'cross_references': {}}},\n",
       " 'PF00393': {'metadata': {'accession': 'PF00393',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006114',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '6-phosphogluconate dehydrogenase, C-terminal domain',\n",
       "    'short': '6PGD'},\n",
       "   'description': ['<p>This family represents the C-terminal all-alpha domain of 6-phosphogluconate dehydrogenase.              The domain contains two structural repeats of 5 helices each.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0106', 'name': '6PGD_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 115,\n",
       "    'interactions': 0,\n",
       "    'matches': 45122,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43589,\n",
       "    'proteomes': 7479,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 38070, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 34882},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42581,\n",
       "    'alignment:full': 9912,\n",
       "    'alignment:seed': 328},\n",
       "   'cross_references': {}}},\n",
       " 'PF09397': {'metadata': {'accession': 'PF09397',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018541',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ftsk gamma domain', 'short': 'FtsK_gamma'},\n",
       "   'description': ['<p>This domain directs oriented DNA translocation and forms a winged helix structure [PMID:17057717].  Mutated proteins with substitutions in the FtsK gamma DNA-recognition helix are impaired in DNA binding [PMID:17057717].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00041884': {'PMID': 17057717,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '11',\n",
       "     'year': 2006,\n",
       "     'title': 'The FtsK gamma domain directs oriented DNA translocation by interacting with KOPS.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '965-72',\n",
       "     'medline_journal': 'Nat Struct Mol Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Mol. Biol.',\n",
       "     'authors': ['Sivanathan V',\n",
       "      'Allen MD',\n",
       "      'de Bekker C',\n",
       "      'Baker R',\n",
       "      'Arciszewska LK',\n",
       "      'Freund SM',\n",
       "      'Bycroft M',\n",
       "      'Lowe J',\n",
       "      'Sherratt DJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsmb1158'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 149,\n",
       "    'interactions': 0,\n",
       "    'matches': 51570,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51436,\n",
       "    'proteomes': 8157,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42444, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 34537},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48718,\n",
       "    'alignment:full': 8435,\n",
       "    'alignment:seed': 401},\n",
       "   'cross_references': {}}},\n",
       " 'PF00327': {'metadata': {'accession': 'PF00327',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR016082',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L30p/L7e', 'short': 'Ribosomal_L30'},\n",
       "   'description': ['<p>This family includes prokaryotic L30 and eukaryotic L7.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 115,\n",
       "    'interactions': 0,\n",
       "    'matches': 45986,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45970,\n",
       "    'proteomes': 10298,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39635, 'rosettafold': 0},\n",
       "    'structures': 1118,\n",
       "    'taxa': 41472},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40826,\n",
       "    'alignment:full': 12174,\n",
       "    'alignment:seed': 759},\n",
       "   'cross_references': {}}},\n",
       " 'PF17853': {'metadata': {'accession': 'PF17853',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041522',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GGDEF-like domain', 'short': 'GGDEF_2'},\n",
       "   'description': ['<p>This domain is distantly related to the GGDEF domain, suggesting these may by diguanylate cyclase enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0276', 'name': 'Nucleot_cyclase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 98,\n",
       "    'interactions': 0,\n",
       "    'matches': 48608,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48606,\n",
       "    'proteomes': 3265,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 43383, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13154},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47165,\n",
       "    'alignment:full': 13122,\n",
       "    'alignment:seed': 714},\n",
       "   'cross_references': {}}},\n",
       " 'PF02657': {'metadata': {'accession': 'PF02657',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003808',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fe-S metabolism associated domain', 'short': 'SufE'},\n",
       "   'description': ['<p>This family consists of the SufE-related proteins. These have been implicated in Fe-S metabolism and export [PMID:11251816]).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014310': {'PMID': 11251816,\n",
       "     'ISBN': None,\n",
       "     'volume': '39',\n",
       "     'issue': '4',\n",
       "     'year': 2001,\n",
       "     'title': 'SoxR-dependent response to oxidative stress and virulence of Erwinia chrysanthemi: the key role of SufC, an orphan ABC ATPase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '960-72',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Nachin L',\n",
       "      'El Hassouni M',\n",
       "      'Loiseau L',\n",
       "      'Expert D',\n",
       "      'Barras F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.2001.02288.x'}},\n",
       "   'set_info': {'accession': 'CL0233', 'name': 'SufE_NifU'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 60,\n",
       "    'interactions': 0,\n",
       "    'matches': 22206,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22204,\n",
       "    'proteomes': 4256,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19370, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 18515},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21061,\n",
       "    'alignment:full': 4540,\n",
       "    'alignment:seed': 319},\n",
       "   'cross_references': {}}},\n",
       " 'PF00662': {'metadata': {'accession': 'PF00662',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001516',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NADH-Ubiquinone oxidoreductase (complex I), chain 5 N-terminus',\n",
       "    'short': 'Proton_antipo_N'},\n",
       "   'description': ['<p>This sub-family represents an amino terminal extension of Pfam:PF00361. Only NADH-Ubiquinone chain 5 and eubacterial chain L are in this family. This sub-family is part of complex I which catalyses the transfer of two electrons from NADH to ubiquinone in a reaction that is associated with proton translocation across the membrane.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005074': {'PMID': 1470679,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '3',\n",
       "     'year': 1992,\n",
       "     'title': 'The NADH:ubiquinone oxidoreductase (complex I) of respiratory chains.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '253-324',\n",
       "     'medline_journal': 'Q Rev Biophys',\n",
       "     'ISO_journal': 'Q. Rev. Biophys.',\n",
       "     'authors': ['Walker JE.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 100,\n",
       "    'interactions': 0,\n",
       "    'matches': 120797,\n",
       "    'pathways': 0,\n",
       "    'proteins': 120774,\n",
       "    'proteomes': 7312,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 101028, 'rosettafold': 0},\n",
       "    'structures': 222,\n",
       "    'taxa': 96553},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 110841,\n",
       "    'alignment:full': 11473,\n",
       "    'alignment:seed': 411},\n",
       "   'cross_references': {}}},\n",
       " 'PF00400': {'metadata': {'accession': 'PF00400',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001680',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'WD domain, G-beta repeat', 'short': 'WD40'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004188': {'PMID': 8090199,\n",
       "     'ISBN': None,\n",
       "     'volume': '371',\n",
       "     'issue': '6495',\n",
       "     'year': 1994,\n",
       "     'title': 'The ancient regulatory-protein family of WD-repeat proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '297-300',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Neer EJ', 'Schmidt CJ', 'Nambudripad R', 'Smith TF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/371297a0'}},\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 15029,\n",
       "    'interactions': 0,\n",
       "    'matches': 2070137,\n",
       "    'pathways': 0,\n",
       "    'proteins': 600259,\n",
       "    'proteomes': 4662,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 428566, 'rosettafold': 0},\n",
       "    'structures': 1709,\n",
       "    'taxa': 23657},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 1592792,\n",
       "    'alignment:full': 938410,\n",
       "    'alignment:seed': 1465},\n",
       "   'cross_references': {}}},\n",
       " 'PF03150': {'metadata': {'accession': 'PF03150',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004852',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Di-haem cytochrome c peroxidase', 'short': 'CCP_MauG'},\n",
       "   'description': ['<p>This is a family of distinct cytochrome c peroxidases (CCPs) that contain two haem groups. Similar to other cytochrome c peroxidases, they reduce hydrogen peroxide to water using c-type haem as an oxidisable substrate. However, since they possess two, instead of one, haem prosthetic groups, bacterial CCPs reduce hydrogen peroxide without the need to generate semi-stable free radicals. The two haem groups have significantly different redox potentials. The high potential (+320 mV) haem feeds electrons from electron shuttle proteins to the low potential (-330 mV) haem, where peroxide is reduced (indeed, the low potential site is known as the peroxidatic site) [PMID:8591033]. The CCP protein itself is structured into two domains, each containing one c-type haem group, with a calcium-binding site at the domain interface. This family also includes MauG proteins, whose similarity to di-haem CCP was previously recognised [PMID:9202457].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007705': {'PMID': 8591033,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '11',\n",
       "     'year': 1995,\n",
       "     'title': 'Crystal structure of the di-haem cytochrome c peroxidase from Pseudomonas aeruginosa.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1225-33',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Fulop V', 'Ridout CJ', 'Greenwood C', 'Hajdu J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00258-1'},\n",
       "    'PUB00007706': {'PMID': 9202457,\n",
       "     'ISBN': None,\n",
       "     'volume': '143 ( Pt 6)',\n",
       "     'issue': None,\n",
       "     'year': 1997,\n",
       "     'title': \"Organization of methylamine utilization genes (mau) in 'Methylobacillus flagellatum ' KT and analysis of mau mutants.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '1827-35',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Gak ER', 'Tsygankov YD', 'Chistoserdov AY.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': {'accession': 'CL0318', 'name': 'Cytochrome-c'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 220,\n",
       "    'interactions': 0,\n",
       "    'matches': 29914,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29710,\n",
       "    'proteomes': 2793,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24384, 'rosettafold': 0},\n",
       "    'structures': 48,\n",
       "    'taxa': 13032},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26085,\n",
       "    'alignment:full': 5277,\n",
       "    'alignment:seed': 448},\n",
       "   'cross_references': {}}},\n",
       " 'PF02508': {'metadata': {'accession': 'PF02508',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003667',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Rnf-Nqr subunit, membrane protein', 'short': 'Rnf-Nqr'},\n",
       "   'description': ['<p>This is a family of integral membrane proteins including  Rhodobacter-specific nitrogen fixation (rnf) proteins RnfA and RnfE [PMID:9492268] and Na+-translocating NADH:ubiquinone oxidoreductase  (Na+-NQR) subunits NqrD and NqrE. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008135': {'PMID': 9492268,\n",
       "     'ISBN': None,\n",
       "     'volume': '251',\n",
       "     'issue': '1-2',\n",
       "     'year': 1998,\n",
       "     'title': 'Overexpression in Escherichia coli of the rnf genes from Rhodobacter capsulatus--characterization of two membrane-bound iron-sulfur proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '54-64',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Jouanneau Y',\n",
       "      'Jeong HS',\n",
       "      'Hugo N',\n",
       "      'Meyer C',\n",
       "      'Willison JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1432-1327.1998.2510054.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 41230,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41170,\n",
       "    'proteomes': 2749,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34905, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 12797},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36954,\n",
       "    'alignment:full': 7615,\n",
       "    'alignment:seed': 622},\n",
       "   'cross_references': {}}},\n",
       " 'PF01799': {'metadata': {'accession': 'PF01799',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002888',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '[2Fe-2S] binding domain', 'short': 'Fer2_2'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005209': {'PMID': 7502041,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '5239',\n",
       "     'year': 1995,\n",
       "     'title': 'Crystal structure of the xanthine oxidase-related aldehyde oxido-reductase from D. gigas.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1170-6',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Romao MJ',\n",
       "      'Archer M',\n",
       "      'Moura I',\n",
       "      'Moura JJ',\n",
       "      'LeGall J',\n",
       "      'Engh R',\n",
       "      'Schneider M',\n",
       "      'Hof P',\n",
       "      'Huber R.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/270/5239/1170'}},\n",
       "   'set_info': {'accession': 'CL0486', 'name': 'Fer2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 401,\n",
       "    'interactions': 0,\n",
       "    'matches': 95317,\n",
       "    'pathways': 0,\n",
       "    'proteins': 95205,\n",
       "    'proteomes': 6771,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 76240, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 28301},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 86534,\n",
       "    'alignment:full': 20368,\n",
       "    'alignment:seed': 1075},\n",
       "   'cross_references': {}}},\n",
       " 'PF10588': {'metadata': {'accession': 'PF10588',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019574',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NADH-ubiquinone oxidoreductase-G iron-sulfur binding region',\n",
       "    'short': 'NADH-G_4Fe-4S_3'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 328,\n",
       "    'interactions': 0,\n",
       "    'matches': 49230,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49208,\n",
       "    'proteomes': 7905,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41047, 'rosettafold': 0},\n",
       "    'structures': 177,\n",
       "    'taxa': 33427},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44510,\n",
       "    'alignment:full': 9553,\n",
       "    'alignment:seed': 399},\n",
       "   'cross_references': {}}},\n",
       " 'PF02537': {'metadata': {'accession': 'PF02537',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003691',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CrcB-like protein, Camphor Resistance (CrcB)',\n",
       "    'short': 'CRCB'},\n",
       "   'description': ['<p>CRCB is a family of bacterial integral membrane proteins with four TMs.. Over expression in E. coli also leads to camphor resistance [PMID:8844142, PMID:12904550].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008171': {'PMID': 8844142,\n",
       "     'ISBN': None,\n",
       "     'volume': '143',\n",
       "     'issue': '4',\n",
       "     'year': 1996,\n",
       "     'title': 'Overproduction of three genes leads to camphor resistance and chromosome condensation in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1521-32',\n",
       "     'medline_journal': 'Genetics',\n",
       "     'ISO_journal': 'Genetics',\n",
       "     'authors': ['Hu KH',\n",
       "      'Liu E',\n",
       "      'Dean K',\n",
       "      'Gingras M',\n",
       "      'DeGraff W',\n",
       "      'Trun NJ.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=8844142'}},\n",
       "   'set_info': {'accession': 'CL0184', 'name': 'DMT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 73,\n",
       "    'interactions': 0,\n",
       "    'matches': 49281,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47381,\n",
       "    'proteomes': 7959,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41631, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 32435},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46168,\n",
       "    'alignment:full': 11326,\n",
       "    'alignment:seed': 811},\n",
       "   'cross_references': {}}},\n",
       " 'PF02833': {'metadata': {'accession': 'PF02833',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004097',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DHHA2 domain', 'short': 'DHHA2'},\n",
       "   'description': ['<p>This domain is often found adjacent to the DHH domain Pfam:PF01368 and is called DHHA2 for DHH associated domain. This domain is diagnostic of DHH subfamily 2 members [PMID:9478130]. The domain is about 120 residues long and contains a conserved DXK motif at its amino terminus.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005478': {'PMID': 9478130,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'A novel family of predicted phosphoesterases includes Drosophila prune protein and bacterial RecJ exonuclease.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17-9',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Aravind L', 'Koonin EV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(97)01162-6'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 104,\n",
       "    'interactions': 0,\n",
       "    'matches': 14867,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14862,\n",
       "    'proteomes': 4120,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12592, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 16227},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13460,\n",
       "    'alignment:full': 4217,\n",
       "    'alignment:seed': 443},\n",
       "   'cross_references': {}}},\n",
       " 'PF14310': {'metadata': {'accession': 'PF14310',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026891',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fibronectin type III-like domain', 'short': 'Fn3-like'},\n",
       "   'description': ['<p>This domain has a fibronectin type III-like structure [PMID:20138890]. It is often found in association with Pfam:PF00933 and Pfam:PF01915. Its function is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 649,\n",
       "    'interactions': 0,\n",
       "    'matches': 68944,\n",
       "    'pathways': 0,\n",
       "    'proteins': 68882,\n",
       "    'proteomes': 5257,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 59688, 'rosettafold': 0},\n",
       "    'structures': 64,\n",
       "    'taxa': 21233},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 63364,\n",
       "    'alignment:full': 19544,\n",
       "    'alignment:seed': 589},\n",
       "   'cross_references': {}}},\n",
       " 'PF03840': {'metadata': {'accession': 'PF03840',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004692',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Preprotein translocase SecG subunit', 'short': 'SecG'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 20,\n",
       "    'interactions': 0,\n",
       "    'matches': 42064,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42064,\n",
       "    'proteomes': 8500,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36170, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 36689},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37967,\n",
       "    'alignment:full': 7808,\n",
       "    'alignment:seed': 633},\n",
       "   'cross_references': {}}},\n",
       " 'PF13660': {'metadata': {'accession': 'PF13660',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025286',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4147)',\n",
       "    'short': 'DUF4147'},\n",
       "   'description': ['<p>This domain is frequently found at the N-terminus of proteins carrying the glycerate kinase-like domain MOFRL, Pfam:PF05161.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 45,\n",
       "    'interactions': 0,\n",
       "    'matches': 15650,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15613,\n",
       "    'proteomes': 3022,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13056, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 14179},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13692,\n",
       "    'alignment:full': 2882,\n",
       "    'alignment:seed': 252},\n",
       "   'cross_references': {}}},\n",
       " 'PF03746': {'metadata': {'accession': 'PF03746',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005501',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LamB/YcsF family', 'short': 'LamB_YcsF'},\n",
       "   'description': ['<p>This family includes LamB. The lam locus of Aspergillus nidulans consists of two divergently transcribed genes, lamA and lamB, involved in the utilisation of lactams such as 2-pyrrolidinone. Both genes are under the control of the positive regulatory gene amdR and are subject to carbon and nitrogen metabolite repression [PMID:1729609]. The exact molecular function of the proteins in this family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008411': {'PMID': 1729609,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '1',\n",
       "     'year': 1992,\n",
       "     'title': 'Molecular characterization of the lam locus and sequences involved in regulation by the AmdR protein of Aspergillus nidulans.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '337-46',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Richardson IB', 'Katz ME', 'Hynes MJ.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=1729609&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0158', 'name': 'GH_CE'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 49,\n",
       "    'interactions': 0,\n",
       "    'matches': 23657,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23621,\n",
       "    'proteomes': 4564,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21048, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 18855},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23419,\n",
       "    'alignment:full': 4879,\n",
       "    'alignment:seed': 372},\n",
       "   'cross_references': {}}},\n",
       " 'PF12911': {'metadata': {'accession': 'PF12911',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025966',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'N-terminal TM domain of oligopeptide transport permease C',\n",
       "    'short': 'OppC_N'},\n",
       "   'description': ['<p>Oligopeptide permeases (Opp) have been identified in numerous gram-negative and -positive bacteria. These transport systems belong to the superfamily of highly conserved ATP-binding cassette transporters. Typically, Opp importers comprise a complex of five proteins. The oligopeptide-binding protein OppA is responsible for the capture of peptides from the external medium. Two integral highly hydrophobic membrane spanning proteins, OppB and OppC, form a channel through the membrane used for peptide translocation.  This N-terminal domain appears to be the first TM domain of the molecule [PMID:1738314].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 77,\n",
       "    'interactions': 0,\n",
       "    'matches': 107976,\n",
       "    'pathways': 0,\n",
       "    'proteins': 107976,\n",
       "    'proteomes': 6481,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 94468, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 28026},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 102868,\n",
       "    'alignment:full': 21570,\n",
       "    'alignment:seed': 559},\n",
       "   'cross_references': {}}},\n",
       " 'PF09994': {'metadata': {'accession': 'PF09994',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018712',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterized alpha/beta hydrolase domain (DUF2235)',\n",
       "    'short': 'DUF2235'},\n",
       "   'description': ['<p>This domain, found in various hypothetical bacterial proteins, has no known function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0028', 'name': 'AB_hydrolase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 246,\n",
       "    'interactions': 0,\n",
       "    'matches': 18956,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16544,\n",
       "    'proteomes': 2207,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13946, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 9748},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17790,\n",
       "    'alignment:full': 5802,\n",
       "    'alignment:seed': 306},\n",
       "   'cross_references': {}}},\n",
       " 'PF03479': {'metadata': {'accession': 'PF03479',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005175',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plants and Prokaryotes Conserved (PCC) domain',\n",
       "    'short': 'PCC'},\n",
       "   'description': ['<p>This is a Plants and Prokaryotes Conserved (PPC) domain found in proteins that contain AT-hook motifs Pfam:PF02178 which strongly suggests a DNA-binding function for the proteins as a whole. Proteins with PPC domains are found in Bacteria, Archaea and the plant kingdom [PMID:15604740, PMID:17295322].The PPC domain has a single alpha-helix packed against an antiparallel beta-sheet, which is formed by five beta-strands. There are three highly conserved histidine residues, eg at 117, 119 and 133 in Swiss:Q46QL5 which appear to form a zinc-binding site, and the domain has been observed to form homotrimers. The domain co-occurs with a thioredoxin-like domain in uncharacterized cyanobacterial proteins [PMID:17295322].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0615', 'name': 'ALDC'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 118,\n",
       "    'interactions': 0,\n",
       "    'matches': 20768,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20715,\n",
       "    'proteomes': 1855,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17628, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 10715},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17925,\n",
       "    'alignment:full': 7461,\n",
       "    'alignment:seed': 341},\n",
       "   'cross_references': {}}},\n",
       " 'PF07486': {'metadata': {'accession': 'PF07486',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011105',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cell Wall Hydrolase', 'short': 'Hydrolase_2'},\n",
       "   'description': ['<p>These enzymes have been implicated in cell wall hydrolysis, most extensively in Bacillus subtilis. For instance Swiss:P50739 is expressed during sporulation as an inactive form and then deposited on the cell outer cortex. During germination the the enzyme is activated and hydrolyses the cortex([PMID:10658652]). A similar role is carried out by the partially redundant Swiss:P42249 ([PMID:9515903]). It is not clear  whether these enzymes are amidases or peptidases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014269': {'PMID': 9515903,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'Regulation and characterization of a newly deduced cell wall hydrolase gene (cwlJ) which affects germination of Bacillus subtilis spores.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1375-80',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Ishikawa S', 'Yamane K', 'Sekiguchi J.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/180/6/1375'},\n",
       "    'PUB00014275': {'PMID': 10658652,\n",
       "     'ISBN': None,\n",
       "     'volume': '146 ( Pt 1)',\n",
       "     'issue': None,\n",
       "     'year': 2000,\n",
       "     'title': 'Complete spore-cortex hydrolysis during germination of Bacillus subtilis 168 requires SleB and YpeB.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '57-64',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Boland FM',\n",
       "      'Atrih A',\n",
       "      'Chirakkal H',\n",
       "      'Foster SJ',\n",
       "      'Moir A.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 124,\n",
       "    'interactions': 0,\n",
       "    'matches': 22294,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22291,\n",
       "    'proteomes': 2866,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19001, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 11134},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21282,\n",
       "    'alignment:full': 5520,\n",
       "    'alignment:seed': 381},\n",
       "   'cross_references': {}}},\n",
       " 'PF01702': {'metadata': {'accession': 'PF01702',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002616',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Queuine tRNA-ribosyltransferase', 'short': 'TGT'},\n",
       "   'description': ['<p>This is a family of queuine tRNA-ribosyltransferases EC:2.4.2.29, also known as tRNA-guanine transglycosylase and guanine insertion enzyme. Queuine tRNA-ribosyltransferase modifies tRNAs for asparagine,  aspartic acid, histidine and tyrosine with queuine.  It catalyses the exchange of guanine-34 at the wobble position with 7-aminomethyl-7-deazaguanine, and the addition of a cyclopentenediol  moiety to 7-aminomethyl-7-deazaguanine-34 tRNA; giving a hypermodified  base queuine in the wobble position [PMID:8654383, PMID:8323579]. The aligned region contains a zinc binding motif C-x-C-x2-C-x29-H,  and important tRNA and 7-aminomethyl-7deazaguanine binding residues [PMID:8654383].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001287': {'PMID': 8654383,\n",
       "     'ISBN': None,\n",
       "     'volume': '15',\n",
       "     'issue': '11',\n",
       "     'year': 1996,\n",
       "     'title': 'Crystal structure of tRNA-guanine transglycosylase: RNA modification by base exchange.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2850-7',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Romier C', 'Reuter K', 'Suck D', 'Ficner R.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8654383'},\n",
       "    'PUB00003305': {'PMID': 8323579,\n",
       "     'ISBN': None,\n",
       "     'volume': '231',\n",
       "     'issue': '2',\n",
       "     'year': 1993,\n",
       "     'title': 'tRNA-guanine transglycosylase from Escherichia coli. Overexpression, purification and quaternary structure.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '489-97',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Garcia GA', 'Koch KA', 'Chong S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1993.1296'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 169,\n",
       "    'interactions': 0,\n",
       "    'matches': 55899,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54878,\n",
       "    'proteomes': 9446,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 46363, 'rosettafold': 0},\n",
       "    'structures': 188,\n",
       "    'taxa': 41352},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49200,\n",
       "    'alignment:full': 11261,\n",
       "    'alignment:seed': 513},\n",
       "   'cross_references': {}}},\n",
       " 'PF07004': {'metadata': {'accession': 'PF07004',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010736',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sperm-tail PG-rich repeat', 'short': 'SHIPPO-rpt'},\n",
       "   'description': ['<p>This family represents a short conserved region carrying a PGP motif that is repeated in eukaryotic proteins of sperm-tails. Shippo orthologues from some species may include up to 40 Pro-Gly-Pro repeats.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 188,\n",
       "    'interactions': 0,\n",
       "    'matches': 31918,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7412,\n",
       "    'proteomes': 844,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5363, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 3791},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25192,\n",
       "    'alignment:full': 15384,\n",
       "    'alignment:seed': 323},\n",
       "   'cross_references': {}}},\n",
       " 'PF05552': {'metadata': {'accession': 'PF05552',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008910',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Conserved TM helix', 'short': 'TM_helix'},\n",
       "   'description': ['<p>This alignment represents a conserved transmembrane helix as well as some flanking sequence. It is often found in association with Pfam:PF00924.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 29649,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18867,\n",
       "    'proteomes': 2658,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16014, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 14717},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26614,\n",
       "    'alignment:full': 4743,\n",
       "    'alignment:seed': 266},\n",
       "   'cross_references': {}}},\n",
       " 'PF13396': {'metadata': {'accession': 'PF13396',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027379',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phospholipase_D-nuclease N-terminal', 'short': 'PLDc_N'},\n",
       "   'description': ['<p>This family is often found at the very N-terminus of proteins from the phospholipase_D-nuclease family, PLDc, Pfam:PF00614. However, a large number of members are full-length within this family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 39533,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39532,\n",
       "    'proteomes': 5794,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35000, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 23520},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38387,\n",
       "    'alignment:full': 8746,\n",
       "    'alignment:seed': 639},\n",
       "   'cross_references': {}}},\n",
       " 'PF01967': {'metadata': {'accession': 'PF01967',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002820',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MoaC family', 'short': 'MoaC'},\n",
       "   'description': ['<p>Members of this family are involved in molybdenum cofactor biosynthesis. However their molecular function is not known.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005858': {'PMID': 10411269,\n",
       "     'ISBN': None,\n",
       "     'volume': '145 ( Pt 6)',\n",
       "     'issue': None,\n",
       "     'year': 1999,\n",
       "     'title': 'Characterization of a molybdenum cofactor biosynthetic gene cluster in Rhodobacter capsulatus which is specific for the biogenesis of dimethylsulfoxide reductase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1421-9',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Solomon PS',\n",
       "      'Shaw AL',\n",
       "      'Lane I',\n",
       "      'Hanson GR',\n",
       "      'Palmer T',\n",
       "      'McEwan AG.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 117,\n",
       "    'interactions': 0,\n",
       "    'matches': 37676,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37613,\n",
       "    'proteomes': 8430,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32319, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 34943},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34232,\n",
       "    'alignment:full': 7736,\n",
       "    'alignment:seed': 646},\n",
       "   'cross_references': {}}},\n",
       " 'PF09363': {'metadata': {'accession': 'PF09363',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018969',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'XFP C-terminal domain', 'short': 'XFP_C'},\n",
       "   'description': ['<p>Bacterial enzyme splits fructose-6-P and/or xylulose-5-P with the aid of inorganic phosphate into either acetyl-P and erythrose-4-P and/or acetyl-P and glyeraldehyde-3-P EC:4.1.2.9, EC:4.1.2.22 [PMID:11292814].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008466': {'PMID': 11292814,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '9',\n",
       "     'year': 2001,\n",
       "     'title': 'Characterization of the D-xylulose 5-phosphate/D-fructose 6-phosphate phosphoketolase gene (xfp) from Bifidobacterium lactis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2929-36',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Meile L',\n",
       "      'Rohr LM',\n",
       "      'Geissmann TA',\n",
       "      'Herensperger M',\n",
       "      'Teuber M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.9.2929-2936.2001'}},\n",
       "   'set_info': {'accession': 'CL0591', 'name': 'TKC_like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 40,\n",
       "    'interactions': 0,\n",
       "    'matches': 12702,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12689,\n",
       "    'proteomes': 2944,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11062, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 12278},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11791,\n",
       "    'alignment:full': 3221,\n",
       "    'alignment:seed': 236},\n",
       "   'cross_references': {}}},\n",
       " 'PF12662': {'metadata': {'accession': 'PF12662',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026823',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Complement Clr-like EGF-like', 'short': 'cEGF'},\n",
       "   'description': [\"<p>cEGF, or complement Clr-like EGF, domains have six conserved cysteine residues disulfide-bonded into the characteristic pattern 'ababcc'. They are found in blood coagulation proteins such as fibrillin, Clr and Cls, thrombomodulin, and the LDL receptor. The core fold of the EGF domain consists of two small beta-hairpins packed against each other. Two major structural variants have been identified based on the structural context of the C-terminal cysteine residue of disulfide 'c' in the C-terminal hairpin: hEGFs and cEGFs. In cEGFs the C-terminal thiol resides on the C-terminal beta-sheet, resulting in long loop-lengths between the cysteine residues of disulfide 'c', typically C[10+]XC. These longer loop-lengths may have arisen by selective cysteine loss from a four-disulfide EGF template such as laminin or integrin.  Tandem cEGF domains have five linking residues between terminal cysteines of adjacent domains. cEGF domains may or may not bind calcium in the linker region. cEGF domains with the consensus motif CXN4X[F,Y]XCXC are hydroxylated exclusively on the asparagine residue.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0001', 'name': 'EGF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 7499,\n",
       "    'interactions': 0,\n",
       "    'matches': 38917,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27843,\n",
       "    'proteomes': 1089,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11911, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 4469},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26811,\n",
       "    'alignment:full': 17925,\n",
       "    'alignment:seed': 523},\n",
       "   'cross_references': {}}},\n",
       " 'PF04536': {'metadata': {'accession': 'PF04536',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007621',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TPM domain', 'short': 'TPM_phosphatase'},\n",
       "   'description': ['<p>This family was first named TPM domain after its founding proteins: TLP18.3, Psb32 and MOLO-1. In Arabidopsis, this domain is called the thylakoid acid phosphatase -TAP - domain and has a Rossmann-like fold [PMID:21908686]. In plants, the family resides in the thylakoid lumen attached to the outer membrane of the chloroplast/plastid. It is active in the photosystem II [PMID:17576201, PMID:21653280].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 74,\n",
       "    'interactions': 0,\n",
       "    'matches': 36468,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36276,\n",
       "    'proteomes': 4576,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31067, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 20640},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33703,\n",
       "    'alignment:full': 6670,\n",
       "    'alignment:seed': 539},\n",
       "   'cross_references': {}}},\n",
       " 'PF03636': {'metadata': {'accession': 'PF03636',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005196',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolase family 65, N-terminal domain',\n",
       "    'short': 'Glyco_hydro_65N'},\n",
       "   'description': ['<p>This family of glycosyl hydrolases contains vacuolar acid trehalase and maltose phosphorylase.Maltose phosphorylase (MP)  is a dimeric enzyme that catalyses the conversion of maltose and  inorganic phosphate into beta-D-glucose-1-phosphate and glucose.  This domain is believed to be essential for catalytic activity [PMID:11587643] although its precise function remains unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008368': {'PMID': 11587643,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '8',\n",
       "     'year': 2001,\n",
       "     'title': 'Crystal structure of maltose phosphorylase from Lactobacillus brevis: unexpected evolutionary relationship with glucoamylases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '689-97',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Egloff MP', 'Uppenberg J', 'Haalck L', 'van Tilbeurgh H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00626-8'}},\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 81,\n",
       "    'interactions': 0,\n",
       "    'matches': 16481,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16478,\n",
       "    'proteomes': 3175,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14362, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 12540},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15912,\n",
       "    'alignment:full': 3978,\n",
       "    'alignment:seed': 296},\n",
       "   'cross_references': {}}},\n",
       " 'PF13676': {'metadata': {'accession': 'PF13676',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000157',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TIR domain', 'short': 'TIR_2'},\n",
       "   'description': ['<p>This is a family of Toll-like receptors.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0173', 'name': 'STIR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2727,\n",
       "    'interactions': 0,\n",
       "    'matches': 44821,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43315,\n",
       "    'proteomes': 3793,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31783, 'rosettafold': 0},\n",
       "    'structures': 55,\n",
       "    'taxa': 18205},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31189,\n",
       "    'alignment:full': 9678,\n",
       "    'alignment:seed': 429},\n",
       "   'cross_references': {}}},\n",
       " 'PF02863': {'metadata': {'accession': 'PF02863',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020899',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Arginine repressor, C-terminal domain',\n",
       "    'short': 'Arg_repressor_C'},\n",
       "   'description': ['<p>This is the C-terminal domain of the arginine repressor, responsible for arginine binding and multimerization [PMID:9334747, PMID:11856827]. It binds mainly Arg, but also ornithine, Pro and Tyr (Matilla et. al., FEMS Microbiology Reviews, fuab043, 45, 2021, 1. https://doi.org/10.1093/femsre/fuab043).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003952': {'PMID': 9334747,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '10',\n",
       "     'year': 1997,\n",
       "     'title': 'Solution structure of the DNA-binding domain and model for the complex of multifunctional hexameric arginine repressor with DNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '819-26',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Sunnerhagen M', 'Nilges M', 'Otting G', 'Carey J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb1097-819'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 18,\n",
       "    'interactions': 0,\n",
       "    'matches': 19663,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19662,\n",
       "    'proteomes': 3832,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17566, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 15221},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18880,\n",
       "    'alignment:full': 3876,\n",
       "    'alignment:seed': 353},\n",
       "   'cross_references': {}}},\n",
       " 'PF06689': {'metadata': {'accession': 'PF06689',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010603',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ClpX C4-type zinc finger', 'short': 'zf-C4_ClpX'},\n",
       "   'description': ['<p>The ClpX heat shock protein of Escherichia coli is a member of the universally conserved Hsp100 family of proteins, and possesses a putative zinc finger motif of the C4 type. This presumed zinc binding domain is found at the N-terminus of the ClpX protein. ClpX is an ATPase which functions both as a substrate specificity component of the ClpXP protease and as a molecular chaperone. The molecular function of this domain is now known.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014898': {'PMID': 11278349,\n",
       "     'ISBN': None,\n",
       "     'volume': '276',\n",
       "     'issue': '22',\n",
       "     'year': 2001,\n",
       "     'title': 'Structure-function analysis of the zinc-binding region of the Clpx molecular chaperone.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '18843-8',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Banecki B',\n",
       "      'Wawrzynow A',\n",
       "      'Puzewicz J',\n",
       "      'Georgopoulos C',\n",
       "      'Zylicz M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M007507200'}},\n",
       "   'set_info': {'accession': 'CL0167', 'name': 'Zn_Beta_Ribbon'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 68,\n",
       "    'interactions': 0,\n",
       "    'matches': 41235,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41195,\n",
       "    'proteomes': 8302,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 35182, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 33855},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37477,\n",
       "    'alignment:full': 8113,\n",
       "    'alignment:seed': 300},\n",
       "   'cross_references': {}}},\n",
       " 'PF00681': {'metadata': {'accession': 'PF00681',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001101',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plectin repeat', 'short': 'Plectin'},\n",
       "   'description': ['<p>This family includes repeats from plectin, desmoplakin, envoplakin and bullous pemphigoid antigen.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 737,\n",
       "    'interactions': 0,\n",
       "    'matches': 97315,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10381,\n",
       "    'proteomes': 681,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 907, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 3225},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 64151,\n",
       "    'alignment:full': 44769,\n",
       "    'alignment:seed': 306},\n",
       "   'cross_references': {}}},\n",
       " 'PF14497': {'metadata': {'accession': 'PF14497',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004046',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutathione S-transferase, C-terminal domain',\n",
       "    'short': 'GST_C_3'},\n",
       "   'description': ['<p>This domain is closely related to Pfam:PF00043.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0497', 'name': 'GST_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 401,\n",
       "    'interactions': 0,\n",
       "    'matches': 33683,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33345,\n",
       "    'proteomes': 3562,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27423, 'rosettafold': 0},\n",
       "    'structures': 267,\n",
       "    'taxa': 15809},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27673,\n",
       "    'alignment:full': 10888,\n",
       "    'alignment:seed': 265},\n",
       "   'cross_references': {}}},\n",
       " 'PF06165': {'metadata': {'accession': 'PF06165',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010383',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyltransferase family 36',\n",
       "    'short': 'Glyco_transf_36'},\n",
       "   'description': ['<p>The glycosyltransferase family 36 includes cellobiose phosphorylase  (EC:2.4.1.20), cellodextrin phosphorylase (EC:2.4.1.49), chitobiose  phosphorylase (EC:2.4.1.-).  Many members of this family contain two copies of this domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 47,\n",
       "    'interactions': 0,\n",
       "    'matches': 17255,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12115,\n",
       "    'proteomes': 1756,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 5988, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 8686},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15770,\n",
       "    'alignment:full': 3237,\n",
       "    'alignment:seed': 318},\n",
       "   'cross_references': {}}},\n",
       " 'PF01226': {'metadata': {'accession': 'PF01226',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000292',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Formate/nitrite transporter', 'short': 'Form_Nir_trans'},\n",
       "   'description': ['<p>Proteins in this entry belong to the Formate-Nitrite Transporter (FNT) family and includes the nitrite transport protein NirC and formate channel FocA [PMID:22407320, PMID:]. They have a pentameric architecture with structural similarity to aquaporins and glyceroporins [PMID:22407320]. Proteins in this family transport the structurally related compounds, formate and nitrite.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0716', 'name': 'Aquaporin-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 75,\n",
       "    'interactions': 0,\n",
       "    'matches': 25051,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24995,\n",
       "    'proteomes': 3889,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 22441, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 17681},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25101,\n",
       "    'alignment:full': 5017,\n",
       "    'alignment:seed': 484},\n",
       "   'cross_references': {}}},\n",
       " 'PF13677': {'metadata': {'accession': 'PF13677',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025713',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Membrane MotB of proton-channel complex MotA/MotB',\n",
       "    'short': 'MotB_plug'},\n",
       "   'description': ['<p>This is the MotB member of the E.coli MotA/MotB proton-channel complex that forms the stator of the bacterial membrane flagellar motor. Key residues act as a plug to prevent premature proton flow. The plug is in the periplasm just C-terminal to the MotB TM, consisting of an amphipathic alpha helix flanked by Pro-52 and Pro-65, eg in Swiss:D3V2T1. In addition to the Pro residues, Ile-58, Tyr-61, and Phe 62 are also essential for plug function [PMID:17052729][PMID:14705929].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 30098,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30097,\n",
       "    'proteomes': 4133,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26158, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 17764},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28779,\n",
       "    'alignment:full': 5598,\n",
       "    'alignment:seed': 289},\n",
       "   'cross_references': {}}},\n",
       " 'PF08379': {'metadata': {'accession': 'PF08379',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013589',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial transglutaminase-like N-terminal region',\n",
       "    'short': 'Bact_transglu_N'},\n",
       "   'description': ['<p>This region is found towards the N-terminus of various archaeal and bacterial hypothetical proteins. Some of these are annotated as being transglutaminase-like proteins, and in fact contain a transglutaminase-like superfamily domain (Pfam:PF01841).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 29,\n",
       "    'interactions': 0,\n",
       "    'matches': 26352,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26352,\n",
       "    'proteomes': 3152,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22795, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13212},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24798,\n",
       "    'alignment:full': 5578,\n",
       "    'alignment:seed': 402},\n",
       "   'cross_references': {}}},\n",
       " 'PF03928': {'metadata': {'accession': 'PF03928',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005624',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Haem degrading protein HbpS-like', 'short': 'HbpS-like'},\n",
       "   'description': ['<p>This entry includes haem degrading protein HbpS from Streptomyces reticuli (swiss:Q9RIM2) and and GlcG from Escherichia coli [PMID:8606183]. HbpS is up-regulated in response to haemin- and peroxide-based oxidative stress. It interacts with the SenS/SenR two-component signal transduction system. Iron binds to surface-exposed lysine residues of an octomeric assembly of the protein [PMID:19244623]. The structure of GlcG is composed of an alpha-beta(2)-alpha(3)-beta(2)-alpha fold, similar to the Roadblock/LC7 domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0161', 'name': 'GAF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 101,\n",
       "    'interactions': 0,\n",
       "    'matches': 33164,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32831,\n",
       "    'proteomes': 4498,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 29015, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 19074},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32015,\n",
       "    'alignment:full': 6955,\n",
       "    'alignment:seed': 485},\n",
       "   'cross_references': {}}},\n",
       " 'PF12874': {'metadata': {'accession': 'PF12874',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Zinc-finger of C2H2 type', 'short': 'zf-met'},\n",
       "   'description': ['<p>This is a zinc-finger domain with the CxxCx(12)Hx(6)H motif, found in multiple copies in a wide range of proteins from plants to metazoans. Some member proteins, particularly those from plants, are annotated as being RNA-binding.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0361', 'name': 'C2H2-zf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 5735,\n",
       "    'interactions': 0,\n",
       "    'matches': 101702,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65945,\n",
       "    'proteomes': 2189,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46710, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 8954},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 77376,\n",
       "    'alignment:full': 46597,\n",
       "    'alignment:seed': 266},\n",
       "   'cross_references': {}}},\n",
       " 'PF01888': {'metadata': {'accession': 'PF01888',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002748',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CbiD', 'short': 'CbiD'},\n",
       "   'description': ['<p>CbiD is essential for cobalamin biosynthesis in both S. typhimurium and B. megaterium, no functional role has been ascribed to the protein. The CbiD protein has a putative S-AdoMet binding site. It is possible that CbiD might have the same role as CobF in undertaking the C-1 methylation and deacylation reactions required during the ring contraction process [PMID:9742225].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000553': {'PMID': 9742225,\n",
       "     'ISBN': None,\n",
       "     'volume': '335 ( Pt 1)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'Cobalamin (vitamin B12) biosynthesis: identification and characterization of a Bacillus megaterium cobI operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '159-66',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Raux E', 'Lanois A', 'Warren MJ', 'Rambach A', 'Thermes C.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=9742225&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 21,\n",
       "    'interactions': 0,\n",
       "    'matches': 11314,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11251,\n",
       "    'proteomes': 2358,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9916, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 11254},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10694,\n",
       "    'alignment:full': 2257,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF04092': {'metadata': {'accession': 'PF04092',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007226',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SRS domain', 'short': 'SAG'},\n",
       "   'description': ['<p>Toxoplasma gondii is a persistent protozoan parasite capable of infecting almost any warm-blooded vertebrate. The surface of Toxoplasma is coated with a family of developmentally regulated glycosylphosphatidylinositol (GPI)-linked proteins (SRSs), of which SAG1 is the prototypic member. SRS proteins mediate attachment to host cells and interface with the host immune response to regulate the virulence of the parasite. SAG1 is composed of two disulphide linked SRS domains. These have 6 cysteines that form 1-6,2-5 and 3-4 pairings. The structure of the immunodominant SAG1 antigen reveals a homodimeric configuration [PMID:12091874]. The SRS domain is found in a single copy in the SAG2 proteins. This family of surface antigens are found in other apicomplexans.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00018995': {'PMID': 11566296,\n",
       "     'ISBN': None,\n",
       "     'volume': '31',\n",
       "     'issue': '12',\n",
       "     'year': 2001,\n",
       "     'title': 'Surface antigens of Toxoplasma gondii: variations on a theme.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1285-92',\n",
       "     'medline_journal': 'Int J Parasitol',\n",
       "     'ISO_journal': 'Int. J. Parasitol.',\n",
       "     'authors': ['Lekutis C',\n",
       "      'Ferguson DJ',\n",
       "      'Grigg ME',\n",
       "      'Camps M',\n",
       "      'Boothroyd JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0020-7519(01)00261-2'},\n",
       "    'PUB00020297': {'PMID': 8206924,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '23',\n",
       "     'year': 1994,\n",
       "     'title': 'Similarities between the primary structures of two distinct major surface proteins of Toxoplasma gondii.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '16217-22',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Cesbron-Delauw MF',\n",
       "      'Tomavo S',\n",
       "      'Beauchamps P',\n",
       "      'Fourmaux MP',\n",
       "      'Camus D',\n",
       "      'Capron A',\n",
       "      'Dubremetz JF.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/23/16217'},\n",
       "    'PUB00020298': {'PMID': 12091874,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '8',\n",
       "     'year': 2002,\n",
       "     'title': 'Structure of the immunodominant surface antigen from the Toxoplasma gondii SRS superfamily.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '606-11',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['He XL', 'Grigg ME', 'Boothroyd JC', 'Garcia KC.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 10,\n",
       "    'interactions': 0,\n",
       "    'matches': 4179,\n",
       "    'pathways': 0,\n",
       "    'proteins': 2391,\n",
       "    'proteomes': 6,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 2348, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 79},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4114,\n",
       "    'alignment:full': 1234,\n",
       "    'alignment:seed': 89},\n",
       "   'cross_references': {}}},\n",
       " 'PF05192': {'metadata': {'accession': 'PF05192',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007696',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MutS domain III', 'short': 'MutS_III'},\n",
       "   'description': ['<p>This domain is found in proteins of the MutS family (DNA mismatch repair proteins) and is found associated with Pfam:PF00488, Pfam:PF05188, Pfam:PF01624 and Pfam:PF05190. The MutS family of proteins is named after the Salmonella typhimurium MutS protein involved in mismatch repair; other members of the family included the eukaryotic MSH 1,2,3, 4,5 and 6 proteins. These have various roles in DNA repair and recombination. Human MSH has been implicated in non-polyposis colorectal carcinoma (HNPCC) and is a mismatch binding protein [PMID:8036718]. The aligned region corresponds with domain III, which is central to the structure of Thermus aquaticus MutS as characterised in [PMID:11048710].   </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003761': {'PMID': 8510668,\n",
       "     'ISBN': None,\n",
       "     'volume': '239',\n",
       "     'issue': '1-2',\n",
       "     'year': 1993,\n",
       "     'title': 'The yeast gene MSH3 defines a new class of eukaryotic MutS homologues.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '97-108',\n",
       "     'medline_journal': 'Mol Gen Genet',\n",
       "     'ISO_journal': 'Mol. Gen. Genet.',\n",
       "     'authors': ['New L', 'Liu K', 'Crouse GF.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00010188': {'PMID': 8036718,\n",
       "     'ISBN': None,\n",
       "     'volume': '10',\n",
       "     'issue': '5',\n",
       "     'year': 1994,\n",
       "     'title': 'Colon cancer and DNA repair: have mismatches met their match?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '164-8',\n",
       "     'medline_journal': 'Trends Genet',\n",
       "     'ISO_journal': 'Trends Genet.',\n",
       "     'authors': ['Jiricny J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0168-9525(94)90093-0'},\n",
       "    'PUB00010189': {'PMID': 11048710,\n",
       "     'ISBN': None,\n",
       "     'volume': '407',\n",
       "     'issue': '6805',\n",
       "     'year': 2000,\n",
       "     'title': 'Crystal structures of mismatch repair protein MutS and its complex with a substrate DNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '703-10',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Obmolova G', 'Ban C', 'Hsieh P', 'Yang W.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/35037509'},\n",
       "    'PUB00019900': {'PMID': 9722634,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '18',\n",
       "     'year': 1998,\n",
       "     'title': 'Domain organization and functional analysis of Thermus thermophilus MutS protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4153-9',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Tachiki H',\n",
       "      'Kato R',\n",
       "      'Masui R',\n",
       "      'Hasegawa K',\n",
       "      'Itakura H',\n",
       "      'Fukuyama K',\n",
       "      'Kuramitsu S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/26.18.4153'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 414,\n",
       "    'interactions': 0,\n",
       "    'matches': 57731,\n",
       "    'pathways': 0,\n",
       "    'proteins': 57705,\n",
       "    'proteomes': 8824,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 47017, 'rosettafold': 0},\n",
       "    'structures': 42,\n",
       "    'taxa': 37391},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51135,\n",
       "    'alignment:full': 16151,\n",
       "    'alignment:seed': 1053},\n",
       "   'cross_references': {}}},\n",
       " 'PF13340': {'metadata': {'accession': 'PF13340',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025161',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative transposase of IS4/5 family (DUF4096)',\n",
       "    'short': 'DUF4096'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 152,\n",
       "    'interactions': 0,\n",
       "    'matches': 29532,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29528,\n",
       "    'proteomes': 2467,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26111, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11407},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28288,\n",
       "    'alignment:full': 7555,\n",
       "    'alignment:seed': 353},\n",
       "   'cross_references': {}}},\n",
       " 'PF01632': {'metadata': {'accession': 'PF01632',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021137',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L35', 'short': 'Ribosomal_L35p'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 33,\n",
       "    'interactions': 0,\n",
       "    'matches': 42217,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42216,\n",
       "    'proteomes': 9849,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36273, 'rosettafold': 0},\n",
       "    'structures': 765,\n",
       "    'taxa': 42234},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37744,\n",
       "    'alignment:full': 9086,\n",
       "    'alignment:seed': 538},\n",
       "   'cross_references': {}}},\n",
       " 'PF13244': {'metadata': {'accession': 'PF13244',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025383',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MBH, subunit D', 'short': 'MbhD'},\n",
       "   'description': ['<p>Hydrogen gas-evolving membrane-bound hydrogenase (MBH) is a respiratory complex homologous to the quinone-reducing Complex I. Like Complex I, MBH has peripheral and membrane arms. MBH is made of 14 subunits (MbhA-N). MbhJ, K, L, N and M form the  Membrane-anchored hydrogenase module. MbhJ, K, L, N are predicted to  be exposed to the cytoplasm and form the peripheral arm. The remaining 10 subunits are predicted to be integral membrane proteins forming the membrane arm, made of 44 transmembrane helices  (TMH) [PMID:33229520, PMID:32735215]. MbhA, B, C and F form the Sodium translocation module. MbhD, E, G and H form the Proton translocation module. MbhI is the linker between the hydrogenase module and the proton-translocating membrane module. It anchors the discontinuous TMH7 of MbhH via its middle lateral helix and the C-terminal  of TMH2, found in MbhE. MbhD and MbhE together are equivalent to Nqo10 of Complex I [PMID:29754813]. MbhD has three TM helices.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 21054,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21054,\n",
       "    'proteomes': 3945,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18366, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 17005},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19761,\n",
       "    'alignment:full': 4503,\n",
       "    'alignment:seed': 345},\n",
       "   'cross_references': {}}},\n",
       " 'PF01392': {'metadata': {'accession': 'PF01392',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020067',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fz domain', 'short': 'Fz'},\n",
       "   'description': ['<p>Also known as the CRD (cysteine rich domain), the C6 box in MuSK receptor. This domain of unknown function has been independently identified by several groups [PMID:9637908, PMID:9637909, PMID:9684897, PMID:9852758].  The domain contains 10 conserved cysteines.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001039': {'PMID': 9637908,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '12',\n",
       "     'year': 1998,\n",
       "     'title': 'The Frizzled CRD domain is conserved in diverse proteins including several receptor tyrosine kinases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': 'R405-6',\n",
       "     'medline_journal': 'Curr Biol',\n",
       "     'ISO_journal': 'Curr. Biol.',\n",
       "     'authors': ['Xu YK', 'Nusse R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0960-9822(98)70262-3'},\n",
       "    'PUB00005055': {'PMID': 9684897,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '7',\n",
       "     'year': 1998,\n",
       "     'title': 'Identification of a Frizzled-like cysteine rich domain in the extracellular region of developmental receptor tyrosine kinases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1632-5',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Saldanha J', 'Singh J', 'Mahadevan D.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9684897&action=stream&blobtype=pdf'},\n",
       "    'PUB00005486': {'PMID': 9852758,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '11',\n",
       "     'year': 1998,\n",
       "     'title': 'The frizzled motif: in how many different protein families does it occur?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '415-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Rehn M', 'Pihlajaniemi T', 'Hofmann K', 'Bucher P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(98)01290-0'},\n",
       "    'PUB00033661': {'PMID': 9637909,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '12',\n",
       "     'year': 1998,\n",
       "     'title': 'The Wnt receptor CRD domain is also found in MuSK and related orphan receptor tyrosine kinases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': 'R407',\n",
       "     'medline_journal': 'Curr Biol',\n",
       "     'ISO_journal': 'Curr. Biol.',\n",
       "     'authors': ['Masiakowski P', 'Yancopoulos GD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0960-9822(98)70263-5'}},\n",
       "   'set_info': {'accession': 'CL0644', 'name': 'Fz'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 546,\n",
       "    'interactions': 0,\n",
       "    'matches': 24501,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22525,\n",
       "    'proteomes': 946,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16539, 'rosettafold': 0},\n",
       "    'structures': 58,\n",
       "    'taxa': 4059},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17704,\n",
       "    'alignment:full': 9880,\n",
       "    'alignment:seed': 260},\n",
       "   'cross_references': {}}},\n",
       " 'PF00387': {'metadata': {'accession': 'PF00387',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001711',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphatidylinositol-specific phospholipase C, Y domain',\n",
       "    'short': 'PI-PLC-Y'},\n",
       "   'description': ['<p>This associates with Pfam:PF00388 to form a single structural unit.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0384', 'name': 'PLC'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 509,\n",
       "    'interactions': 0,\n",
       "    'matches': 30249,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30173,\n",
       "    'proteomes': 2045,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18403, 'rosettafold': 0},\n",
       "    'structures': 23,\n",
       "    'taxa': 7698},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21101,\n",
       "    'alignment:full': 13199,\n",
       "    'alignment:seed': 298},\n",
       "   'cross_references': {}}},\n",
       " 'PF13004': {'metadata': {'accession': 'PF13004',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024361',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative binding domain, N-terminal', 'short': 'BACON'},\n",
       "   'description': [\"<p>The BACON (Bacteroidetes-Associated Carbohydrate-binding Often N-terminal) domain is an all-beta domain found in diverse architectures, principally in combination with carbohydrate-active enzymes and proteases. These architectures suggest a carbohydrate-binding function which is also supported by the nature of BACON's few conserved amino-acids. The phyletic distribution of BACON and other data tentatively suggest that it may frequently function to bind mucin [PMID:20416301]. Further work with the characterised structure of a member of glycoside hydrolase family 5 enzyme, PDB:3ZMR, has found no evidence for carbohydrate-binding for this domain [PMID:24463512].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 760,\n",
       "    'interactions': 0,\n",
       "    'matches': 7492,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5328,\n",
       "    'proteomes': 290,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4386, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 1659},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6628,\n",
       "    'alignment:full': 1281,\n",
       "    'alignment:seed': 239},\n",
       "   'cross_references': {}}},\n",
       " 'PF01018': {'metadata': {'accession': 'PF01018',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006169',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GTP1/OBG', 'short': 'GTP1_OBG'},\n",
       "   'description': ['<p>The N-terminal domain of Swiss:P20964 has the OBG fold, which is formed by three glycine-rich regions inserted into a small 8-stranded beta-sandwich these regions form six left-handed collagen-like helices packed and H-bonded together.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 166,\n",
       "    'interactions': 0,\n",
       "    'matches': 51923,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50453,\n",
       "    'proteomes': 10654,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 42854, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 43453},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46540,\n",
       "    'alignment:full': 11484,\n",
       "    'alignment:seed': 606},\n",
       "   'cross_references': {}}},\n",
       " 'PF04145': {'metadata': {'accession': 'PF04145',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007274',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ctr copper transporter family', 'short': 'Ctr'},\n",
       "   'description': ['<p>The redox active metal copper is an essential cofactor in critical biological processes such as respiration, iron transport, oxidative stress protection, hormone production, and pigmentation. A widely conserved family of high-affinity copper transport proteins (Ctr proteins) mediates copper uptake at the plasma membrane.  A series of clustered methionine residues in the hydrophilic extracellular domain, and an MXXXM motif in the second transmembrane domain, are important for copper uptake. These methionine probably coordinate copper during the process of metal transport.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019362': {'PMID': 11983704,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '29',\n",
       "     'year': 2002,\n",
       "     'title': 'Biochemical and genetic analyses of yeast and human high affinity copper transporters suggest a conserved mechanism for copper uptake.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '26021-30',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Puig S', 'Lee J', 'Lau M', 'Thiele DJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M202547200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 157,\n",
       "    'interactions': 0,\n",
       "    'matches': 12490,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10967,\n",
       "    'proteomes': 2091,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9208, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 7578},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10282,\n",
       "    'alignment:full': 6336,\n",
       "    'alignment:seed': 411},\n",
       "   'cross_references': {}}},\n",
       " 'PF00886': {'metadata': {'accession': 'PF00886',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000307',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S16', 'short': 'Ribosomal_S16'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 67,\n",
       "    'interactions': 0,\n",
       "    'matches': 56325,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56320,\n",
       "    'proteomes': 10493,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 48150, 'rosettafold': 0},\n",
       "    'structures': 904,\n",
       "    'taxa': 58916},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49330,\n",
       "    'alignment:full': 9725,\n",
       "    'alignment:seed': 577},\n",
       "   'cross_references': {}}},\n",
       " 'PF13508': {'metadata': {'accession': 'PF13508',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000182',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acetyltransferase (GNAT) domain',\n",
       "    'short': 'Acetyltransf_7'},\n",
       "   'description': ['<p>This domain catalyses N-acetyltransferase reactions.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0257', 'name': 'Acetyltrans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 292,\n",
       "    'interactions': 0,\n",
       "    'matches': 48184,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48151,\n",
       "    'proteomes': 6073,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42298, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 27060},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47474,\n",
       "    'alignment:full': 12386,\n",
       "    'alignment:seed': 1800},\n",
       "   'cross_references': {}}},\n",
       " 'PF07075': {'metadata': {'accession': 'PF07075',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008302',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1343)',\n",
       "    'short': 'DUF1343'},\n",
       "   'description': ['<p>This family consists of several hypothetical bacterial proteins of around 400 residues in length. The function of this family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 15747,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15691,\n",
       "    'proteomes': 2535,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12785, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 10668},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13356,\n",
       "    'alignment:full': 2692,\n",
       "    'alignment:seed': 251},\n",
       "   'cross_references': {}}},\n",
       " 'PF01628': {'metadata': {'accession': 'PF01628',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021153',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HrcA protein C terminal domain', 'short': 'HrcA'},\n",
       "   'description': ['<p>HrcA is found to negatively regulate the transcription of heat shock genes [PMID:8576042, PMID:8606155]. HrcA contains an amino terminal helix-turn-helix domain, however this corresponds to the carboxy terminal domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002287': {'PMID': 8576042,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '4',\n",
       "     'year': 1996,\n",
       "     'title': 'hrcA, the first gene of the Bacillus subtilis dnaK operon encodes a negative regulator of class I heat shock genes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1088-93',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Schulz A', 'Schumann W.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8576042&action=stream&blobtype=pdf'},\n",
       "    'PUB00002293': {'PMID': 8606155,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '7',\n",
       "     'year': 1996,\n",
       "     'title': 'Identification of a Caulobacter crescentus operon encoding hrcA, involved in negatively regulating heat-inducible transcription, and the chaperone gene grpE.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1829-41',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Roberts RC',\n",
       "      'Toochinda C',\n",
       "      'Avedissian M',\n",
       "      'Baldini RL',\n",
       "      'Gomes SL',\n",
       "      'Shapiro L.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8606155&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0161', 'name': 'GAF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 29988,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29975,\n",
       "    'proteomes': 6328,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25500, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 25460},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26790,\n",
       "    'alignment:full': 5779,\n",
       "    'alignment:seed': 581},\n",
       "   'cross_references': {}}},\n",
       " 'PF13405': {'metadata': {'accession': 'PF13405',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002048',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'EF-hand domain', 'short': 'EF-hand_6'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0220', 'name': 'EF_hand'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 711,\n",
       "    'interactions': 0,\n",
       "    'matches': 17000,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16788,\n",
       "    'proteomes': 1961,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13381, 'rosettafold': 0},\n",
       "    'structures': 70,\n",
       "    'taxa': 8295},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13078,\n",
       "    'alignment:full': 7276,\n",
       "    'alignment:seed': 543},\n",
       "   'cross_references': {}}},\n",
       " 'PF03946': {'metadata': {'accession': 'PF03946',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020784',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L11, N-terminal domain',\n",
       "    'short': 'Ribosomal_L11_N'},\n",
       "   'description': ['<p>The N-terminal domain of Ribosomal protein L11 adopts an  alpha/beta fold and is followed by the RNA binding C-terminal domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 157,\n",
       "    'interactions': 0,\n",
       "    'matches': 51101,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51098,\n",
       "    'proteomes': 11005,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 43819, 'rosettafold': 0},\n",
       "    'structures': 597,\n",
       "    'taxa': 47585},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45201,\n",
       "    'alignment:full': 12238,\n",
       "    'alignment:seed': 250},\n",
       "   'cross_references': {}}},\n",
       " 'PF10557': {'metadata': {'accession': 'PF10557',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019559',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cullin protein neddylation domain',\n",
       "    'short': 'Cullin_Nedd8'},\n",
       "   'description': ['<p>This is the neddylation site of cullin proteins which are a family of structurally related proteins containing an evolutionarily conserved cullin domain. With the exception of APC2, each member of the cullin family is modified by Nedd8 and several cullins function in Ubiquitin-dependent proteolysis, a process in which the 26S proteasome recognises and subsequently degrades a target protein tagged with K48-linked poly-ubiquitin chains. Cullins are molecular scaffolds responsible for assembling the ROC1/Rbx1 RING-based E3 ubiquitin ligases, of which several play a direct role in tumorigenesis. Nedd8/Rub1 is a small ubiquitin-like protein, which was originally found to be conjugated to Cdc53, a cullin component of the SCF (Skp1-Cdc53/CUL1-F-box protein) E3 Ub ligase complex in Saccharomyces cerevisiae, and Nedd8 modification has now emerged as a regulatory pathway of fundamental importance for cell cycle control and for embryogenesis in metazoans. The only identified Nedd8 substrates are cullins. Neddylation results in covalent conjugation of a Nedd8 moiety onto a conserved cullin lysine residue [PMID:15021886].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044220': {'PMID': 15021886,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '11',\n",
       "     'year': 2004,\n",
       "     'title': 'Nedd8 on cullin: building an expressway to protein destruction.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1985-97',\n",
       "     'medline_journal': 'Oncogene',\n",
       "     'ISO_journal': 'Oncogene',\n",
       "     'authors': ['Pan ZQ', 'Kentsis A', 'Dias DC', 'Yamoah K', 'Wu K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/sj.onc.1207414'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 226,\n",
       "    'interactions': 0,\n",
       "    'matches': 19469,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19440,\n",
       "    'proteomes': 2124,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14458, 'rosettafold': 0},\n",
       "    'structures': 37,\n",
       "    'taxa': 7962},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14472,\n",
       "    'alignment:full': 8952,\n",
       "    'alignment:seed': 331},\n",
       "   'cross_references': {}}},\n",
       " 'PF14510': {'metadata': {'accession': 'PF14510',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029481',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ABC-transporter N-terminal', 'short': 'ABC_trans_N'},\n",
       "   'description': ['<p>This domain is found at the N-terminus of ABC-transporter proteins from fungi, plants to higher eukaryotes. It is predicted to be an intracellular domain [PMID:8294477, PMID:14711635, PMID:21034832].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 373,\n",
       "    'interactions': 0,\n",
       "    'matches': 14146,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14084,\n",
       "    'proteomes': 1012,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 1655, 'rosettafold': 1},\n",
       "    'structures': 4,\n",
       "    'taxa': 3275},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12209,\n",
       "    'alignment:full': 8102,\n",
       "    'alignment:seed': 363},\n",
       "   'cross_references': {}}},\n",
       " 'PF02277': {'metadata': {'accession': 'PF02277',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003200',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoribosyltransferase', 'short': 'DBI_PRT'},\n",
       "   'description': [\"<p>This family of proteins represent the nicotinate-nucleotide- dimethylbenzimidazole phosphoribosyltransferase (NN:DBI PRT) enzymes involved in dimethylbenzimidazole synthesis. This function is essential to de novo cobalamin (vitamin B12) production in bacteria. Nicotinate mononucleotide (NaMN):5,6-dimethylbenzimidazole (DMB) phosphoribosyltransferase (CobT) from Salmonella enterica plays a central role in the synthesis of alpha-ribazole-5'-phosphate, an intermediate for the lower ligand of cobalamin [PMID:12101181].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015329': {'PMID': 8501034,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '11',\n",
       "     'year': 1993,\n",
       "     'title': 'Characterization of the cobalamin (vitamin B12) biosynthetic genes of Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3303-16',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Roth JR',\n",
       "      'Lawrence JG',\n",
       "      'Rubenfield M',\n",
       "      'Kieffer-Higgins S',\n",
       "      'Church GM.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8501034'},\n",
       "    'PUB00015874': {'PMID': 12101181,\n",
       "     'ISBN': None,\n",
       "     'volume': '277',\n",
       "     'issue': '43',\n",
       "     'year': 2002,\n",
       "     'title': 'Capture of a labile substrate by expulsion of water molecules from the active site of nicotinate mononucleotide:5,6-dimethylbenzimidazole phosphoribosyltransferase (CobT) from Salmonella enterica.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '41120-7',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Cheong CG', 'Escalante-Semerena JC', 'Rayment I.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M203535200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 25160,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24969,\n",
       "    'proteomes': 4746,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 21695, 'rosettafold': 0},\n",
       "    'structures': 46,\n",
       "    'taxa': 20334},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24020,\n",
       "    'alignment:full': 5163,\n",
       "    'alignment:seed': 408},\n",
       "   'cross_references': {}}},\n",
       " 'PF02844': {'metadata': {'accession': 'PF02844',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020562',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoribosylglycinamide synthetase, N domain',\n",
       "    'short': 'GARS_N'},\n",
       "   'description': [\"<p>Phosphoribosylglycinamide synthetase catalyses the second step in the de novo biosynthesis of purine. The reaction catalysed by Phosphoribosylglycinamide synthetase is the ATP- dependent addition of 5-phosphoribosylamine to glycine to form 5'phosphoribosylglycinamide. This domain is related to the N-terminal domain of biotin carboxylase/carbamoyl phosphate synthetase (see Pfam:PF00289). This domain is structurally related to the PreATP-grasp domain.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002527': {'PMID': 2687276,\n",
       "     'ISBN': None,\n",
       "     'volume': '264',\n",
       "     'issue': '35',\n",
       "     'year': 1989,\n",
       "     'title': 'Nucleotide sequence analysis of genes purH and purD involved in the de novo purine nucleotide biosynthesis of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '21239-46',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Aiba A', 'Mizobuchi K.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/264/35/21239.pdf'},\n",
       "    'PUB00014228': {'PMID': 9843369,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '45',\n",
       "     'year': 1998,\n",
       "     'title': 'X-ray crystal structure of glycinamide ribonucleotide synthetase from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15647-62',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Wang W', 'Kappock TJ', 'Stubbe J', 'Ealick SE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi981405n'}},\n",
       "   'set_info': {'accession': 'CL0483', 'name': 'PreATP-grasp'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 181,\n",
       "    'interactions': 0,\n",
       "    'matches': 49162,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49148,\n",
       "    'proteomes': 10225,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41630, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 42118},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44485,\n",
       "    'alignment:full': 9648,\n",
       "    'alignment:seed': 860},\n",
       "   'cross_references': {}}},\n",
       " 'PF01384': {'metadata': {'accession': 'PF01384',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001204',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphate transporter family', 'short': 'PHO4'},\n",
       "   'description': ['<p>This family includes PHO-4 from Neurospora crassa which is a is a Na(+)-phosphate symporter [PMID:7732001]. This family also contains the leukaemia virus receptor Swiss:Q08344.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004860': {'PMID': 7732001,\n",
       "     'ISBN': None,\n",
       "     'volume': '92',\n",
       "     'issue': '9',\n",
       "     'year': 1995,\n",
       "     'title': 'Repressible cation-phosphate symporters in Neurospora crassa.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3884-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Versaw WK', 'Metzenberg RL.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=7732001'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 104,\n",
       "    'interactions': 0,\n",
       "    'matches': 70969,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50975,\n",
       "    'proteomes': 8091,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 43938, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 34010},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 65158,\n",
       "    'alignment:full': 15600,\n",
       "    'alignment:seed': 768},\n",
       "   'cross_references': {}}},\n",
       " 'PF04314': {'metadata': {'accession': 'PF04314',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007410',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Copper chaperone PCu(A)C', 'short': 'PCuAC'},\n",
       "   'description': ['<p>PCu(A)C is a periplasmic copper chaperone. Its role  may be to  capture and transfer copper to two other copper chaperones, PrrC and Cox11, which in turn deliver Cu(I) to cytochrome c oxidase [PMID:22248670]. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 45,\n",
       "    'interactions': 0,\n",
       "    'matches': 20082,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20027,\n",
       "    'proteomes': 3582,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17234, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 14094},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18824,\n",
       "    'alignment:full': 4175,\n",
       "    'alignment:seed': 303},\n",
       "   'cross_references': {}}},\n",
       " 'PF08459': {'metadata': {'accession': 'PF08459',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001162',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UvrC RNAse H endonuclease domain',\n",
       "    'short': 'UvrC_RNaseH_dom'},\n",
       "   'description': [\"<p>This domain is found in the C subunits of the bacterial and archaeal UvrABC system which catalyses nucleotide excision repair in a multi-step process.  UvrC catalyses the first incision on the fourth or fifth phosphodiester bond 3' and on the eighth phosphodiester bond 5' from the damage that is to be excised [PMID:17245438]. The domain described here represents the RNAse H endonuclease domain, located at the C-terminal, between the UvrBC and the (HhH)2 domains, nearby the N-terminal of the HhH. Despite the lack of sequence homology, the endonuclease domain has an RNase H-like fold, which is characteristic of enzymes with nuclease or polynucleotide transferase activities. RNase H-related enzymes typically contain a highly conserved carboxylate triad, usually DDE, in their catalytic centre. However, instead of a third carboxylate, UvrC of Thermotoga maritima was found to contain a highly conserved histidine (H488) on helix-4 in close proximity to two aspartates [PMID:17245438].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010573': {'PMID': 12034838,\n",
       "     'ISBN': None,\n",
       "     'volume': '30',\n",
       "     'issue': '11',\n",
       "     'year': 2002,\n",
       "     'title': 'The C-terminal region of Escherichia coli UvrC contributes to the flexibility of the UvrABC nucleotide excision repair system.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2492-500',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Verhoeven EE',\n",
       "      'van Kesteren M',\n",
       "      'Turner JJ',\n",
       "      'van der Marel GA',\n",
       "      'van Boom JH',\n",
       "      'Moolenaar GF',\n",
       "      'Goosen N.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/30.11.2492'}},\n",
       "   'set_info': {'accession': 'CL0219', 'name': 'RNase_H'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 108,\n",
       "    'interactions': 0,\n",
       "    'matches': 48507,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48402,\n",
       "    'proteomes': 8668,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41299, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 37538},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44362,\n",
       "    'alignment:full': 8039,\n",
       "    'alignment:seed': 665},\n",
       "   'cross_references': {}}},\n",
       " 'PF04376': {'metadata': {'accession': 'PF04376',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007471',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Arginine-tRNA-protein transferase, N terminus',\n",
       "    'short': 'ATE_N'},\n",
       "   'description': ['<p>This family represents the N terminal region of the enzyme  arginine-tRNA-protein transferase (EC 2.3.2.8), which catalyses the post-translational conjugation of arginine to the N terminus of a protein. In eukaryotes, this functions as part of the N-end rule pathway  of protein degradation by conjugating a de-stabilising amino acid to the amino terminal aspartate or glutamate of a protein, targeting the protein for ubiquitin-dependent proteolysis.  N terminal cysteine is sometimes  modified [PMID:9858543]. In S cerevisiae, Cys20, 23, 94 and/or 95 are thought to be important  for activity [PMID:7495814].  Of these, only Cys 94 appears to be completely conserved in this family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009970': {'PMID': 9858543,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Alternative splicing results in differential expression, activity, and localization of the two forms of arginyl-tRNA-protein transferase, a component of the N-end rule pathway.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '182-93',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Kwon YT', 'Kashina AS', 'Varshavsky A.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9858543'},\n",
       "    'PUB00009971': {'PMID': 7495814,\n",
       "     'ISBN': None,\n",
       "     'volume': '34',\n",
       "     'issue': '48',\n",
       "     'year': 1995,\n",
       "     'title': 'Binding of phenylarsenoxide to Arg-tRNA protein transferase is independent of vicinal thiols.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '15829-37',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Li J', 'Pickart CM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi00048a028'}},\n",
       "   'set_info': {'accession': 'CL0257', 'name': 'Acetyltrans'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 16655,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16642,\n",
       "    'proteomes': 4258,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13914, 'rosettafold': 1},\n",
       "    'structures': 5,\n",
       "    'taxa': 17198},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14811,\n",
       "    'alignment:full': 4555,\n",
       "    'alignment:seed': 258},\n",
       "   'cross_references': {}}},\n",
       " 'PF10035': {'metadata': {'accession': 'PF10035',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019264',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterized protein conserved in bacteria (DUF2179)',\n",
       "    'short': 'DUF2179'},\n",
       "   'description': ['<p>This domain, found in various hypothetical bacterial proteins, has no known function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0089', 'name': 'GlnB-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 21,\n",
       "    'interactions': 0,\n",
       "    'matches': 34111,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34111,\n",
       "    'proteomes': 2808,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30171, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 11583},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32940,\n",
       "    'alignment:full': 7517,\n",
       "    'alignment:seed': 843},\n",
       "   'cross_references': {}}},\n",
       " 'PF00203': {'metadata': {'accession': 'PF00203',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002222',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S19', 'short': 'Ribosomal_S19'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 132,\n",
       "    'interactions': 0,\n",
       "    'matches': 66505,\n",
       "    'pathways': 0,\n",
       "    'proteins': 66481,\n",
       "    'proteomes': 11007,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 57482, 'rosettafold': 0},\n",
       "    'structures': 1121,\n",
       "    'taxa': 68530},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 58252,\n",
       "    'alignment:full': 11633,\n",
       "    'alignment:seed': 299},\n",
       "   'cross_references': {}}},\n",
       " 'PF02020': {'metadata': {'accession': 'PF02020',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003307',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'eIF4-gamma/eIF5/eIF2-epsilon', 'short': 'W2'},\n",
       "   'description': ['<p>This domain of unknown function is found at the C-terminus of several translation initiation factors [PMID:8520487].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005748': {'PMID': 8520487,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '8',\n",
       "     'year': 1995,\n",
       "     'title': 'Multidomain organization of eukaryotic guanine nucleotide exchange translation initiation factor eIF-2B subunits revealed by analysis of conserved sequence motifs.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1608-17',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Koonin EV.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8520487&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 206,\n",
       "    'interactions': 0,\n",
       "    'matches': 17711,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17703,\n",
       "    'proteomes': 2104,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10249, 'rosettafold': 0},\n",
       "    'structures': 44,\n",
       "    'taxa': 7841},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12864,\n",
       "    'alignment:full': 8342,\n",
       "    'alignment:seed': 350},\n",
       "   'cross_references': {}}},\n",
       " 'PF07331': {'metadata': {'accession': 'PF07331',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009936',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tripartite tricarboxylate transporter TctB family',\n",
       "    'short': 'TctB'},\n",
       "   'description': ['<p>This family consists of several hypothetical bacterial proteins of around 150 residues in length. This family was formerly known as DUF1468.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 29988,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29986,\n",
       "    'proteomes': 2803,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25978, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12285},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27486,\n",
       "    'alignment:full': 7015,\n",
       "    'alignment:seed': 420},\n",
       "   'cross_references': {}}},\n",
       " 'PF00880': {'metadata': {'accession': 'PF00880',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000900',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nebulin repeat', 'short': 'Nebulin'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001250': {'PMID': 8168478,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '8',\n",
       "     'year': 1994,\n",
       "     'title': 'Nebulin, a helical actin binding protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1782-9',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Pfuhl M', 'Winder SJ', 'Pastore A.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8168478&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 259,\n",
       "    'interactions': 0,\n",
       "    'matches': 148875,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9280,\n",
       "    'proteomes': 852,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5003, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 3535},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 101218,\n",
       "    'alignment:full': 77167,\n",
       "    'alignment:seed': 582},\n",
       "   'cross_references': {}}},\n",
       " 'PF08345': {'metadata': {'accession': 'PF08345',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013556',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flagellar M-ring protein C-terminal',\n",
       "    'short': 'YscJ_FliF_C'},\n",
       "   'description': ['<p>This domain is found in bacterial flagellar M-ring (FliF) proteins together with the YscJ/FliF domain (Pfam:PF01514).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 39,\n",
       "    'interactions': 0,\n",
       "    'matches': 21843,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21795,\n",
       "    'proteomes': 4054,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19006, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 17549},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21058,\n",
       "    'alignment:full': 4051,\n",
       "    'alignment:seed': 354},\n",
       "   'cross_references': {}}},\n",
       " 'PF00908': {'metadata': {'accession': 'PF00908',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000888',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'dTDP-4-dehydrorhamnose 3,5-epimerase',\n",
       "    'short': 'dTDP_sugar_isom'},\n",
       "   'description': ['<p>This family catalyse the isomerisation of dTDP-4-dehydro-6-deoxy -D-glucose with dTDP-4-dehydro-6-deoxy-L-mannose. The EC number of this enzyme is 5.1.3.13.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 58,\n",
       "    'interactions': 0,\n",
       "    'matches': 42486,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42473,\n",
       "    'proteomes': 6621,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 36651, 'rosettafold': 0},\n",
       "    'structures': 45,\n",
       "    'taxa': 28923},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39142,\n",
       "    'alignment:full': 7886,\n",
       "    'alignment:seed': 685},\n",
       "   'cross_references': {}}},\n",
       " 'PF03705': {'metadata': {'accession': 'PF03705',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022641',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CheR methyltransferase, all-alpha domain',\n",
       "    'short': 'CheR_N'},\n",
       "   'description': ['<p>CheR proteins are part of the chemotaxis signaling mechanism in bacteria. CheR methylates the chemotaxis receptor at specific glutamate residues. CheR is an S-adenosylmethionine- dependent methyltransferase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005291': {'PMID': 9115443,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '4',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure of the chemotaxis receptor methyltransferase CheR suggests a conserved structural motif for binding S-adenosylmethionine.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '545-58',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Djordjevic S', 'Stock AM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00210-4'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 874,\n",
       "    'interactions': 0,\n",
       "    'matches': 38252,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38252,\n",
       "    'proteomes': 4554,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31265, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 20056},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35885,\n",
       "    'alignment:full': 8044,\n",
       "    'alignment:seed': 656},\n",
       "   'cross_references': {}}},\n",
       " 'PF00888': {'metadata': {'accession': 'PF00888',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001373',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cullin family', 'short': 'Cullin'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 420,\n",
       "    'interactions': 0,\n",
       "    'matches': 30033,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28417,\n",
       "    'proteomes': 2172,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21114, 'rosettafold': 0},\n",
       "    'structures': 65,\n",
       "    'taxa': 8376},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22713,\n",
       "    'alignment:full': 14015,\n",
       "    'alignment:seed': 341},\n",
       "   'cross_references': {}}},\n",
       " 'PF06130': {'metadata': {'accession': 'PF06130',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008300',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphate propanoyltransferase', 'short': 'PTAC'},\n",
       "   'description': ['<p>This family includes phosphotransacylases (PTACs) required for the degradation of 1,2-propanediol (1,2-PD) [PMID:17158662].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 12703,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6462,\n",
       "    'proteomes': 1044,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5531, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 4974},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11645,\n",
       "    'alignment:full': 2588,\n",
       "    'alignment:seed': 274},\n",
       "   'cross_references': {}}},\n",
       " 'PF09851': {'metadata': {'accession': 'PF09851',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018649',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Short C-terminal domain', 'short': 'SHOCT'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0660', 'name': 'SHOCT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 278,\n",
       "    'interactions': 0,\n",
       "    'matches': 26435,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26139,\n",
       "    'proteomes': 3454,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 22413, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 16560},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23701,\n",
       "    'alignment:full': 5782,\n",
       "    'alignment:seed': 557},\n",
       "   'cross_references': {}}},\n",
       " 'PF01940': {'metadata': {'accession': 'PF01940',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002794',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Integral membrane protein DUF92', 'short': 'DUF92'},\n",
       "   'description': ['<p>Members of this family have several predicted transmembrane helices.  One member of the family has been characterised as protein PGR (AtPGR). PGR is suggested to be a potential glucose-responsive regulator in carbohydrate metabolism in plants. This entry also includes protein VTE6, which is a Pphytyl-phosphate kinase catalysing the conversion of phytyl-monophosphate to phytyl-diphosphate [PMID:26452599].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 77,\n",
       "    'interactions': 0,\n",
       "    'matches': 9033,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8951,\n",
       "    'proteomes': 2738,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7482, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11581},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7539,\n",
       "    'alignment:full': 2739,\n",
       "    'alignment:seed': 284},\n",
       "   'cross_references': {}}},\n",
       " 'PF01161': {'metadata': {'accession': 'PF01161',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008914',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphatidylethanolamine-binding protein',\n",
       "    'short': 'PBP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005297': {'PMID': 9782050,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '10',\n",
       "     'year': 1998,\n",
       "     'title': 'Function from structure? The crystal structure of human phosphatidylethanolamine-binding protein suggests a role in membrane signal transduction.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1245-54',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Banfield MJ', 'Barker JJ', 'Perry AC', 'Brady RL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(98)00125-7'},\n",
       "    'PUB00005298': {'PMID': 9782057,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '10',\n",
       "     'year': 1998,\n",
       "     'title': 'Crystal structure of the phosphatidylethanolamine-binding protein from bovine brain: a novel structural class of phospholipid-binding proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1255-65',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Serre L',\n",
       "      'Vallee B',\n",
       "      'Bureaud N',\n",
       "      'Schoentgen F',\n",
       "      'Zelwer C.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(98)00126-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 296,\n",
       "    'interactions': 0,\n",
       "    'matches': 41376,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41261,\n",
       "    'proteomes': 6043,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35677, 'rosettafold': 0},\n",
       "    'structures': 103,\n",
       "    'taxa': 26954},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37470,\n",
       "    'alignment:full': 11812,\n",
       "    'alignment:seed': 271},\n",
       "   'cross_references': {}}},\n",
       " 'PF00995': {'metadata': {'accession': 'PF00995',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001619',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sec1 family', 'short': 'Sec1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003457': {'PMID': 8769846,\n",
       "     'ISBN': None,\n",
       "     'volume': '66',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'The Sec1 family: a novel family of proteins involved in synaptic transmission and general secretion.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '889-97',\n",
       "     'medline_journal': 'J Neurochem',\n",
       "     'ISO_journal': 'J. Neurochem.',\n",
       "     'authors': ['Halachmi N', 'Lev Z.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 327,\n",
       "    'interactions': 0,\n",
       "    'matches': 25590,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24108,\n",
       "    'proteomes': 2182,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18808, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 8108},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19827,\n",
       "    'alignment:full': 12294,\n",
       "    'alignment:seed': 585},\n",
       "   'cross_references': {}}},\n",
       " 'PF01890': {'metadata': {'accession': 'PF01890',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002750',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cobalamin synthesis G C-terminus', 'short': 'CbiG_C'},\n",
       "   'description': ['<p>Members of this family are involved in cobalamin synthesis.  The gene encoded by Swiss:P72862 has been designated cbiH but in fact represents a fusion between cbiH and cbiG. As other multi-functional proteins involved in cobalamin biosynthesis catalyse adjacent steps in the pathway, including CysG, CobL (CbiET), CobIJ and CobA-HemD, it is therefore possible that CbiG catalyses a reaction step adjacent to CbiH. In the anaerobic pathway such a step could be the formation of a gamma lactone, which is thought to help to mediate the anaerobic ring contraction process [PMID:9742225]. Within the cobalamin synthesis pathway CbiG catalyses the both the opening of the lactone ring and the extrusion of the two-carbon fragment of cobalt-precorrin-5A from C-20 and its associated methyl group (deacylation) to give cobalt-precorrin-5B [PMID:16936030]. This family is the C-terminal region, and the mid- and N-termival parts are conserved independently in other families.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000553': {'PMID': 9742225,\n",
       "     'ISBN': None,\n",
       "     'volume': '335 ( Pt 1)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'Cobalamin (vitamin B12) biosynthesis: identification and characterization of a Bacillus megaterium cobI operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '159-66',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Raux E', 'Lanois A', 'Warren MJ', 'Rambach A', 'Thermes C.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=9742225&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 14196,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14174,\n",
       "    'proteomes': 3117,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12644, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 14209},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13655,\n",
       "    'alignment:full': 2939,\n",
       "    'alignment:seed': 419},\n",
       "   'cross_references': {}}},\n",
       " 'PF07494': {'metadata': {'accession': 'PF07494',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011110',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Two component regulator propeller', 'short': 'Reg_prop'},\n",
       "   'description': ['<p>A large group of two component regulator proteins appear to have the same N-terminal structure of  14 tandem repeats. These repeats show homology to Pfam:PF01011 and Pfam:PF00400 indicating that they are likely to form a beta-propeller. This family has been built with artificially high cut-offs in order to avoid overlaps with other beta-propeller  families. The fourteen repeats are likely to form two propellers; it is not clear if these structures are likely to recruit other proteins or interact with DNA.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1488,\n",
       "    'interactions': 0,\n",
       "    'matches': 133509,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43997,\n",
       "    'proteomes': 1706,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20140, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 7988},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 115454,\n",
       "    'alignment:full': 24168,\n",
       "    'alignment:seed': 659},\n",
       "   'cross_references': {}}},\n",
       " 'PF01458': {'metadata': {'accession': 'PF01458',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000825',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SUF system FeS cluster assembly, SufBD',\n",
       "    'short': 'SUFBD'},\n",
       "   'description': ['<p>Iron-sulphur (FeS) clusters are important cofactors for numerous proteins involved in electron transfer, in redox and non-redox catalysis, in gene regulation, and as sensors of oxygen and iron. These functions depend on the various FeS cluster prosthetic groups, the most common being [2Fe-2S] and [4Fe-4S] [PMID:16221578]. FeS cluster assembly is a complex process involving the mobilisation of Fe and S atoms from storage sources, their assembly into [Fe-S] form, their transport to specific cellular locations, and their transfer to recipient apoproteins. So far, three FeS assembly machineries have been identified, which are capable of synthesising all types of [Fe-S] clusters: ISC (iron-sulphur cluster), SUF (sulphur assimilation), and NIF (nitrogen fixation) systems. The SUF system is an alternative pathway to the ISC system that operates under iron starvation and oxidative stress. It is found in eubacteria, archaea and eukaryotes (plastids). The SUF system is encoded by the suf operon (sufABCDSE), and the six encoded proteins are arranged into two complexes (SufSE and SufBCD) and one protein (SufA). SufS is a pyridoxal-phosphate (PLP) protein displaying cysteine desulphurase activity. SufE acts as a scaffold protein that accepts S from SufS and donates it to SufA [PMID:17350000]. SufC is an ATPase with an unorthodox ATP-binding cassette (ABC)-like component. SufA is homologous to IscA [PMID:15278785], acting as a scaffold protein in which Fe and S atoms are assembled into [FeS] cluster forms, which can then easily be transferred to apoproteins targets. This entry represents SufB and SufD proteins, which are homologous, and form part of the SufBCD complex in the SUF system [PMID:26472926]. SufB accepts sulfur transferred from SufE [PMID:17350958], whereas SufD may play a role in iron acquisition [PMID:20857974].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 103,\n",
       "    'interactions': 0,\n",
       "    'matches': 71583,\n",
       "    'pathways': 0,\n",
       "    'proteins': 71485,\n",
       "    'proteomes': 7478,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 60981, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 32120},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 64885,\n",
       "    'alignment:full': 13545,\n",
       "    'alignment:seed': 805},\n",
       "   'cross_references': {}}},\n",
       " 'PF17940': {'metadata': {'accession': 'PF17940',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041583',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetracyclin repressor-like, C-terminal domain',\n",
       "    'short': 'TetR_C_31'},\n",
       "   'description': ['<p>This is the C-terminal domain found in putative transcriptional regulator, TetR family proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0174', 'name': 'TetR_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 9,\n",
       "    'interactions': 0,\n",
       "    'matches': 8091,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8091,\n",
       "    'proteomes': 1090,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7234, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 3690},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7713,\n",
       "    'alignment:full': 2312,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF01730': {'metadata': {'accession': 'PF01730',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002639',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UreF', 'short': 'UreF'},\n",
       "   'description': ['<p>This family consists of the Urease accessory protein  UreF. The urease enzyme (urea amidohydrolase) hydrolyses urea into ammonia and carbamic acid [PMID:8550495].  UreF is proposed to modulate the activation process of urease by eliminating the binding of nickel irons to  noncarbamylated protein [PMID:8808930].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002303': {'PMID': 8808930,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '18',\n",
       "     'year': 1996,\n",
       "     'title': 'Purification and activation properties of UreD-UreF-urease apoprotein complexes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5417-21',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Moncrief MB', 'Hausinger RP.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8808930'},\n",
       "    'PUB00002304': {'PMID': 8550495,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Organization of Ureaplasma urealyticum urease gene cluster and expression in a suppressor strain of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '647-55',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Neyrolles O',\n",
       "      'Ferris S',\n",
       "      'Behbahani N',\n",
       "      'Montagnier L',\n",
       "      'Blanchard A.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8550495&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 36,\n",
       "    'interactions': 0,\n",
       "    'matches': 17380,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17370,\n",
       "    'proteomes': 4050,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15360, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 17333},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16806,\n",
       "    'alignment:full': 3928,\n",
       "    'alignment:seed': 289},\n",
       "   'cross_references': {}}},\n",
       " 'PF00479': {'metadata': {'accession': 'PF00479',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022674',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glucose-6-phosphate dehydrogenase, NAD binding domain',\n",
       "    'short': 'G6PD_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019591': {'PMID': 9485426,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'On the mechanism of the reaction catalyzed by glucose 6-phosphate dehydrogenase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2759-67',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Cosgrove MS',\n",
       "      'Naylor C',\n",
       "      'Paludan S',\n",
       "      'Adams MJ',\n",
       "      'Levy HR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi972069y'}},\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 150,\n",
       "    'interactions': 0,\n",
       "    'matches': 47289,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47096,\n",
       "    'proteomes': 8041,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 40647, 'rosettafold': 0},\n",
       "    'structures': 44,\n",
       "    'taxa': 34881},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43994,\n",
       "    'alignment:full': 11139,\n",
       "    'alignment:seed': 471},\n",
       "   'cross_references': {}}},\n",
       " 'PF13545': {'metadata': {'accession': 'PF13545',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012318',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Crp-like helix-turn-helix domain', 'short': 'HTH_Crp_2'},\n",
       "   'description': ['<p>This family represents a crp-like helix-turn-helix domain that is likely to bind DNA.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 188,\n",
       "    'interactions': 0,\n",
       "    'matches': 120561,\n",
       "    'pathways': 0,\n",
       "    'proteins': 120552,\n",
       "    'proteomes': 7732,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 104142, 'rosettafold': 0},\n",
       "    'structures': 130,\n",
       "    'taxa': 32275},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 110050,\n",
       "    'alignment:full': 24882,\n",
       "    'alignment:seed': 786},\n",
       "   'cross_references': {}}},\n",
       " 'PF03609': {'metadata': {'accession': 'PF03609',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004700',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PTS system sorbose-specific iic component',\n",
       "    'short': 'EII-Sor'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 15,\n",
       "    'interactions': 0,\n",
       "    'matches': 15246,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15241,\n",
       "    'proteomes': 960,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14049, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 5685},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16027,\n",
       "    'alignment:full': 2320,\n",
       "    'alignment:seed': 194},\n",
       "   'cross_references': {}}},\n",
       " 'PF16344': {'metadata': {'accession': 'PF16344',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032508',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4974)',\n",
       "    'short': 'DUF4974'},\n",
       "   'description': ['<p>This family consists of uncharacterized proteins around 340 residues in length and is mainly found in various Bacteroides and  Parabacterodies species. The function of this protein is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 29761,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29760,\n",
       "    'proteomes': 749,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 27251, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 3189},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28382,\n",
       "    'alignment:full': 8022,\n",
       "    'alignment:seed': 521},\n",
       "   'cross_references': {}}},\n",
       " 'PF02548': {'metadata': {'accession': 'PF02548',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003700',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ketopantoate hydroxymethyltransferase',\n",
       "    'short': 'Pantoate_transf'},\n",
       "   'description': ['<p>Ketopantoate hydroxymethyltransferase (EC:2.1.2.11) is the first enzyme in the pantothenate biosynthesis pathway.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00017389': {'PMID': 8096212,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '7',\n",
       "     'year': 1993,\n",
       "     'title': 'Cloning and sequencing of the Escherichia coli panB gene, which encodes ketopantoate hydroxymethyltransferase, and overexpression of the enzyme.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2125-30',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Jones CE', 'Brook JM', 'Buck D', 'Abell C', 'Smith AG.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=8096212'}},\n",
       "   'set_info': {'accession': 'CL0151', 'name': 'PK_TIM'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 104,\n",
       "    'interactions': 0,\n",
       "    'matches': 42723,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42593,\n",
       "    'proteomes': 8125,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 36562, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 33685},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39080,\n",
       "    'alignment:full': 8254,\n",
       "    'alignment:seed': 643},\n",
       "   'cross_references': {}}},\n",
       " 'PF02601': {'metadata': {'accession': 'PF02601',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020579',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Exonuclease VII, large subunit', 'short': 'Exonuc_VII_L'},\n",
       "   'description': [\"<p>This family consist of exonuclease VII, large subunit EC:3.1.11.6 This enzyme catalyses exonucleolytic cleavage in either 5'->3' or 3'->5'  direction to yield 5'-phosphomononucleotides.  This exonuclease VII enzyme is composed of one large subunit and 4 small ones [PMID:6284744].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008230': {'PMID': 6284744,\n",
       "     'ISBN': None,\n",
       "     'volume': '257',\n",
       "     'issue': '15',\n",
       "     'year': 1982,\n",
       "     'title': 'Subunit structure of Escherichia coli exonuclease VII.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8799-805',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Vales LD', 'Rabin BA', 'Chase JW.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/257/15/8799'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 51777,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43233,\n",
       "    'proteomes': 7832,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36927, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 32840},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47288,\n",
       "    'alignment:full': 9033,\n",
       "    'alignment:seed': 418},\n",
       "   'cross_references': {}}},\n",
       " 'PF14714': {'metadata': {'accession': 'PF14714',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032859',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'KH-domain-like of EngA bacterial GTPase enzymes, C-terminal',\n",
       "    'short': 'KH_dom-like'},\n",
       "   'description': ['<p>The KH-like domain at the C-terminus of the EngA subfamily of essential bacterial GTPases has a unique domain structure position.  The two adjacent GTPase domains (GD1 and GD2), two domains of family MMR_HSR1, Pfam:PF01926, pack at either side of the C-terminal domain.  This C-terminal domain resembles a KH domain but is missing the distinctive RNA recognition elements. Conserved motifs of the nucleotide binding site of GD1 are integral parts of the GD1-KH domain interface, suggesting the interactions between these two domains are directly influenced by the GTP/GDP cycling of the protein. In contrast, the GD2-KH domain interface is distal to the GDP binding site of GD2. This family has not been added to the KH clan since SCOP classifies it separately due to its missing the key KH motif/fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 67,\n",
       "    'interactions': 0,\n",
       "    'matches': 44018,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43998,\n",
       "    'proteomes': 8988,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37533, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 36868},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40025,\n",
       "    'alignment:full': 8585,\n",
       "    'alignment:seed': 533},\n",
       "   'cross_references': {}}},\n",
       " 'PF00349': {'metadata': {'accession': 'PF00349',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022672',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hexokinase', 'short': 'Hexokinase_1'},\n",
       "   'description': ['<p>Hexokinase (EC:2.7.1.1) contains two structurally similar domains represented by this family and Pfam:PF03727.  Some members of the family have two copies of each of these domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003193': {'PMID': 5133118,\n",
       "     'ISBN': None,\n",
       "     'volume': '61',\n",
       "     'issue': '3',\n",
       "     'year': 1971,\n",
       "     'title': 'Structure of yeast hexokinase-B. I. Preliminary x-ray studies and subunit structure.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '695-700',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(71)90073-8'},\n",
       "    'PUB00003196': {'PMID': 7001032,\n",
       "     'ISBN': None,\n",
       "     'volume': '140',\n",
       "     'issue': '2',\n",
       "     'year': 1980,\n",
       "     'title': 'Structure of a complex between yeast hexokinase A and glucose. II. Detailed comparisons of conformation and active site configuration with the native hexokinase B monomer and dimer.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '211-30',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Bennett WS Jr', 'Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(80)90103-5'}},\n",
       "   'set_info': {'accession': 'CL0108', 'name': 'Actin_ATPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 212,\n",
       "    'interactions': 0,\n",
       "    'matches': 23019,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17818,\n",
       "    'proteomes': 2228,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14046, 'rosettafold': 0},\n",
       "    'structures': 82,\n",
       "    'taxa': 8595},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17276,\n",
       "    'alignment:full': 10452,\n",
       "    'alignment:seed': 378},\n",
       "   'cross_references': {}}},\n",
       " 'PF13597': {'metadata': {'accession': 'PF13597',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012833',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Anaerobic ribonucleoside-triphosphate reductase',\n",
       "    'short': 'NRDD'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0339', 'name': 'PFL-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 25610,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25247,\n",
       "    'proteomes': 3172,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21190, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 16392},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23581,\n",
       "    'alignment:full': 4217,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF01977': {'metadata': {'accession': 'PF01977',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002830',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '3-octaprenyl-4-hydroxybenzoate carboxy-lyase',\n",
       "    'short': 'UbiD'},\n",
       "   'description': ['<p>This family has been characterised as 3-octaprenyl-4- hydroxybenzoate carboxy-lyase enzymes [PMID:782527]. This enzyme catalyses the third reaction in ubiquinone biosynthesis. For optimal activity the carboxy-lase was shown to require Mn2+ [PMID:782527].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020511': {'PMID': 782527,\n",
       "     'ISBN': None,\n",
       "     'volume': '436',\n",
       "     'issue': '4',\n",
       "     'year': 1976,\n",
       "     'title': 'Membrane-associated reactions in ubiquinone biosynthesis in Escherichia coli. 3-Octaprenyl-4-hydroxybenzoate carboxy-lyase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '800-10',\n",
       "     'medline_journal': 'Biochim Biophys Acta',\n",
       "     'ISO_journal': 'Biochim. Biophys. Acta',\n",
       "     'authors': ['Leppik RA', 'Young IG', 'Gibson F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0005-2736(76)90407-7'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 35491,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34117,\n",
       "    'proteomes': 4152,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28609, 'rosettafold': 0},\n",
       "    'structures': 73,\n",
       "    'taxa': 19804},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32045,\n",
       "    'alignment:full': 5157,\n",
       "    'alignment:seed': 397},\n",
       "   'cross_references': {}}},\n",
       " 'PF02729': {'metadata': {'accession': 'PF02729',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006132',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aspartate/ornithine carbamoyltransferase, carbamoyl-P binding domain',\n",
       "    'short': 'OTCace_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007942': {'PMID': 10318893,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '10',\n",
       "     'year': 1999,\n",
       "     'title': 'Assessment of the allosteric mechanism of aspartate transcarbamoylase based on the crystalline structure of the unregulated catalytic subunit.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5388-93',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Beernink PT', 'Endrizzi JA', 'Alber T', 'Schachman HK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.96.10.5388'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 246,\n",
       "    'interactions': 0,\n",
       "    'matches': 103822,\n",
       "    'pathways': 0,\n",
       "    'proteins': 103774,\n",
       "    'proteomes': 10721,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 86960, 'rosettafold': 0},\n",
       "    'structures': 179,\n",
       "    'taxa': 44932},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 94558,\n",
       "    'alignment:full': 20404,\n",
       "    'alignment:seed': 1247},\n",
       "   'cross_references': {}}},\n",
       " 'PF00119': {'metadata': {'accession': 'PF00119',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000568',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase A chain', 'short': 'ATP-synt_A'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 86,\n",
       "    'interactions': 0,\n",
       "    'matches': 98988,\n",
       "    'pathways': 0,\n",
       "    'proteins': 98891,\n",
       "    'proteomes': 8848,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 85597, 'rosettafold': 0},\n",
       "    'structures': 186,\n",
       "    'taxa': 98738},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 89422,\n",
       "    'alignment:full': 9171,\n",
       "    'alignment:seed': 565},\n",
       "   'cross_references': {}}},\n",
       " 'PF01264': {'metadata': {'accession': 'PF01264',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000453',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Chorismate synthase', 'short': 'Chorismate_synt'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 110,\n",
       "    'interactions': 0,\n",
       "    'matches': 48888,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48627,\n",
       "    'proteomes': 9639,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41593, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 39285},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44459,\n",
       "    'alignment:full': 9362,\n",
       "    'alignment:seed': 664},\n",
       "   'cross_references': {}}},\n",
       " 'PF00300': {'metadata': {'accession': 'PF00300',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013078',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Histidine phosphatase superfamily (branch 1)',\n",
       "    'short': 'His_Phos_1'},\n",
       "   'description': [\"<p>The histidine phosphatase superfamily is so named because catalysis centres on a conserved His residue that is transiently phosphorylated during the catalytic cycle.  Other conserved residues contribute to a 'phosphate pocket' and interact with the phospho group of substrate before, during and after its transfer to the His residue. Structure and sequence analyses show that different families contribute different additional residues to the 'phosphate pocket' and, more surprisingly, differ in the position, in sequence and in three dimensions, of a catalytically essential acidic residue.  The superfamily may be divided into two main branches.  The larger branch 1 contains a wide variety of catalytic functions, the best known being fructose 2,6-bisphosphatase (found in a bifunctional protein with 2-phosphofructokinase) and cofactor-dependent phosphoglycerate mutase.  The latter is an unusual example of a mutase activity in the superfamily: the vast majority of members appear to be phosphatases.  The bacterial regulatory protein phosphatase SixA is also in branch 1 and has a minimal, and possible ancestral-like structure, lacking the large domain insertions that contribute to binding of small molecules in branch 1 members.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020083': {'PMID': 9507137,\n",
       "     'ISBN': None,\n",
       "     'volume': '780',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Localization of mGluR1a-like immunoreactivity and mGluR5-like immunoreactivity in identified populations of striatal neurons.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '210-7',\n",
       "     'medline_journal': 'Brain Res',\n",
       "     'ISO_journal': 'Brain Res.',\n",
       "     'authors': ['Tallaksen-Greene SJ', 'Kaatz KW', 'Romano C', 'Albin RL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0006-8993(97)01141-4'}},\n",
       "   'set_info': {'accession': 'CL0071', 'name': 'His_phosphatase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1076,\n",
       "    'interactions': 0,\n",
       "    'matches': 258473,\n",
       "    'pathways': 0,\n",
       "    'proteins': 237901,\n",
       "    'proteomes': 10709,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 203540, 'rosettafold': 0},\n",
       "    'structures': 177,\n",
       "    'taxa': 45248},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 235672,\n",
       "    'alignment:full': 65753,\n",
       "    'alignment:seed': 330},\n",
       "   'cross_references': {}}},\n",
       " 'PF01668': {'metadata': {'accession': 'PF01668',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000037',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SmpB protein', 'short': 'SmpB'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 24,\n",
       "    'interactions': 0,\n",
       "    'matches': 43060,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43035,\n",
       "    'proteomes': 8718,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36912, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 37184},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38862,\n",
       "    'alignment:full': 8139,\n",
       "    'alignment:seed': 460},\n",
       "   'cross_references': {}}},\n",
       " 'PF00673': {'metadata': {'accession': 'PF00673',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031309',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ribosomal L5P family C-terminus',\n",
       "    'short': 'Ribosomal_L5_C'},\n",
       "   'description': ['<p>This region is found associated with Pfam:PF00281.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 161,\n",
       "    'interactions': 0,\n",
       "    'matches': 52987,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52979,\n",
       "    'proteomes': 11084,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45577, 'rosettafold': 0},\n",
       "    'structures': 1076,\n",
       "    'taxa': 49649},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47139,\n",
       "    'alignment:full': 11560,\n",
       "    'alignment:seed': 386},\n",
       "   'cross_references': {}}},\n",
       " 'PF00828': {'metadata': {'accession': 'PF00828',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021131',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal proteins 50S-L15, 50S-L18e, 60S-L27A',\n",
       "    'short': 'Ribosomal_L27A'},\n",
       "   'description': ['<p>This family includes higher eukaryotic ribosomal 60S L27A, archaeal 50S L18e, prokaryotic 50S L15, fungal mitochondrial L10, plant L27A, mitochondrial L15 and chloroplast L18-3 proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0588', 'name': 'Ribos_L15p_L18e'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 149,\n",
       "    'interactions': 0,\n",
       "    'matches': 55398,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55370,\n",
       "    'proteomes': 11145,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 47407, 'rosettafold': 0},\n",
       "    'structures': 1206,\n",
       "    'taxa': 47724},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48883,\n",
       "    'alignment:full': 12601,\n",
       "    'alignment:seed': 465},\n",
       "   'cross_references': {}}},\n",
       " 'PF00330': {'metadata': {'accession': 'PF00330',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001030',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aconitase family (aconitate hydratase)',\n",
       "    'short': 'Aconitase'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 386,\n",
       "    'interactions': 0,\n",
       "    'matches': 130434,\n",
       "    'pathways': 0,\n",
       "    'proteins': 127344,\n",
       "    'proteomes': 10522,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 108825, 'rosettafold': 0},\n",
       "    'structures': 24,\n",
       "    'taxa': 43511},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 119613,\n",
       "    'alignment:full': 27560,\n",
       "    'alignment:seed': 579},\n",
       "   'cross_references': {}}},\n",
       " 'PF07664': {'metadata': {'accession': 'PF07664',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011640',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ferrous iron transport protein B C terminus',\n",
       "    'short': 'FeoB_C'},\n",
       "   'description': ['<p>Escherichia coli has an iron(II) transport system (feo) which may make an important contribution to the iron supply of the cell under anaerobic conditions [PMID:8407793]. FeoB has been identified as part of this transport system. FeoB is a large 700-800 amino acid integral membrane protein. The N-terminus has been previously erroneously described as being ATP-binding [PMID:8407793]. Recent work shows that it is similar to eukaryotic G-proteins and that it is a GTPase [PMID:12446835]. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009526': {'PMID': 8407793,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '19',\n",
       "     'year': 1993,\n",
       "     'title': 'Characterization of the ferrous iron uptake system of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6212-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Kammler M', 'Schon C', 'Hantke K.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=8407793'},\n",
       "    'PUB00015189': {'PMID': 12446835,\n",
       "     'ISBN': None,\n",
       "     'volume': '99',\n",
       "     'issue': '25',\n",
       "     'year': 2002,\n",
       "     'title': 'The membrane protein FeoB contains an intramolecular G protein essential for Fe(II) uptake in bacteria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '16243-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Marlovits TC',\n",
       "      'Haase W',\n",
       "      'Herrmann C',\n",
       "      'Aller SG',\n",
       "      'Unger VM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.242338299'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 33770,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33770,\n",
       "    'proteomes': 4689,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28956, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21297},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30604,\n",
       "    'alignment:full': 5890,\n",
       "    'alignment:seed': 757},\n",
       "   'cross_references': {}}},\n",
       " 'PF01887': {'metadata': {'accession': 'PF01887',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR046469',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SAM hydroxide adenosyltransferase N-terminal domain',\n",
       "    'short': 'SAM_HAT_N'},\n",
       "   'description': ['<p>This is a family of proteins, previously known as DUF62, found in archaebacteria and bacteria. The structure of proteins in this family is similar to that of a bacterial fluorinating enzyme [PMID:17910070]. S-adenosyl-l-methionine hydroxide adenosyltransferases utilises a rigorously conserved amino acid side chain triad (Asp-Arg-His) which may have a role in activating water to hydroxide ion [PMID:18675376]. This family used to be known as DUF62.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 28,\n",
       "    'interactions': 0,\n",
       "    'matches': 13700,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13658,\n",
       "    'proteomes': 2038,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11469, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 10359},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11750,\n",
       "    'alignment:full': 1940,\n",
       "    'alignment:seed': 291},\n",
       "   'cross_references': {}}},\n",
       " 'PF02233': {'metadata': {'accession': 'PF02233',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR034300',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NAD(P) transhydrogenase beta subunit', 'short': 'PNTB'},\n",
       "   'description': ['<p>This family corresponds to the beta subunit of NADP transhydrogenase in prokaryotes, and either the protein N- or C terminal in eukaryotes.  The domain is often found in conjunction with Pfam:PF01262. Pyridine  nucleotide transhydrogenase catalyses the reduction of NAD+ to NADPH. A complete loss of activity occurs upon  mutation of Gly314 in  E. coli [PMID:1633824].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020146': {'PMID': 1633824,\n",
       "     'ISBN': None,\n",
       "     'volume': '207',\n",
       "     'issue': '2',\n",
       "     'year': 1992,\n",
       "     'title': 'A mutation at Gly314 of the beta subunit of the Escherichia coli pyridine nucleotide transhydrogenase abolishes activity and affects the NADP(H)-induced conformational change.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '733-9',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Ahmad S', 'Glavas NA', 'Bragg PD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1432-1033.1992.tb17103.x'}},\n",
       "   'set_info': {'accession': 'CL0085', 'name': 'FAD_DHS'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 29193,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28822,\n",
       "    'proteomes': 5640,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 23895, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 23907},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26011,\n",
       "    'alignment:full': 5960,\n",
       "    'alignment:seed': 266},\n",
       "   'cross_references': {}}},\n",
       " 'PF02700': {'metadata': {'accession': 'PF02700',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003850',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphoribosylformylglycinamidine (FGAM) synthase',\n",
       "    'short': 'PurS'},\n",
       "   'description': ['<p>This family forms a component of the de novo purine biosynthesis pathway. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020211': {'PMID': 10071207,\n",
       "     'ISBN': None,\n",
       "     'volume': '261',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Isolation and characterization of a purC(orf)QLF operon from Lactococcus [correction of Lactobacillus] lactis MG1614.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '31-41',\n",
       "     'medline_journal': 'Mol Gen Genet',\n",
       "     'ISO_journal': 'Mol. Gen. Genet.',\n",
       "     'authors': ['Peltonen T', 'Mantsala P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/s004380050938'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 40,\n",
       "    'interactions': 0,\n",
       "    'matches': 21403,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21081,\n",
       "    'proteomes': 4748,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18092, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 19099},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18986,\n",
       "    'alignment:full': 4283,\n",
       "    'alignment:seed': 297},\n",
       "   'cross_references': {}}},\n",
       " 'PF09285': {'metadata': {'accession': 'PF09285',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015365',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Elongation factor P, C-terminal',\n",
       "    'short': 'Elong-fact-P_C'},\n",
       "   'description': ['<p>Members of this family of nucleic acid binding domains are predominantly found in elongation factor P, where they adopt an OB-fold, with five beta-strands forming a beta-barrel in a Greek-key topology [PMID:15210970].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015919': {'PMID': 15210970,\n",
       "     'ISBN': None,\n",
       "     'volume': '101',\n",
       "     'issue': '26',\n",
       "     'year': 2004,\n",
       "     'title': 'Crystal structure of elongation factor P from Thermus thermophilus HB8.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9595-600',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Hanawa-Suetsugu K',\n",
       "      'Sekine S',\n",
       "      'Sakai H',\n",
       "      'Hori-Takemoto C',\n",
       "      'Terada T',\n",
       "      'Unzai S',\n",
       "      'Tame JR',\n",
       "      'Kuramitsu S',\n",
       "      'Shirouzu M',\n",
       "      'Yokoyama S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.0308667101'}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 48742,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48739,\n",
       "    'proteomes': 9037,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42049, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 38622},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44267,\n",
       "    'alignment:full': 9379,\n",
       "    'alignment:seed': 372},\n",
       "   'cross_references': {}}},\n",
       " 'PF02194': {'metadata': {'accession': 'PF02194',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003114',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PXA domain', 'short': 'PXA'},\n",
       "   'description': ['<p>This domain is associated with PX domains Pfam:PF00787.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 196,\n",
       "    'interactions': 0,\n",
       "    'matches': 12113,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12047,\n",
       "    'proteomes': 1983,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8816, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 6974},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9106,\n",
       "    'alignment:full': 5913,\n",
       "    'alignment:seed': 280},\n",
       "   'cross_references': {}}},\n",
       " 'PF01176': {'metadata': {'accession': 'PF01176',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006196',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Translation initiation factor 1A / IF-1',\n",
       "    'short': 'eIF-1a'},\n",
       "   'description': ['<p>This family includes both the eukaryotic translation factor eIF-1A and the bacterial translation initiation factor IF-1.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016925': {'PMID': 9419357,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'Universally conserved translation initiation factors.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '224-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Kyrpides NC', 'Woese CR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.95.1.224'}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 112,\n",
       "    'interactions': 0,\n",
       "    'matches': 60707,\n",
       "    'pathways': 0,\n",
       "    'proteins': 60697,\n",
       "    'proteomes': 11098,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 52096, 'rosettafold': 0},\n",
       "    'structures': 70,\n",
       "    'taxa': 58125},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 53049,\n",
       "    'alignment:full': 12645,\n",
       "    'alignment:seed': 341},\n",
       "   'cross_references': {}}},\n",
       " 'PF03880': {'metadata': {'accession': 'PF03880',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005580',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DbpA RNA binding domain', 'short': 'DbpA'},\n",
       "   'description': ['<p>This RNA binding domain is found at the C-terminus of a number of DEAD helicase proteins [PMID:10481020].  It is sufficient to confer specificity  for hairpin 92 of 23S rRNA, which is part of the ribosomal A-site. However, several members of this family lack specificity for 23S rRNA. These can proteins can generally be distinguished by a basic region that extends beyond this domain [Karl Kossen, unpublished data].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008462': {'PMID': 10481020,\n",
       "     'ISBN': None,\n",
       "     'volume': '27',\n",
       "     'issue': '19',\n",
       "     'year': 1999,\n",
       "     'title': 'Cloning and biochemical characterization of Bacillus subtilis YxiN, a DEAD protein specifically activated by 23S rRNA: delineation of a novel sub-family of bacterial DEAD proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3811-20',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Kossen K', 'Uhlenbeck OC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/27.19.3811'},\n",
       "    'PUB00019391': {'PMID': 12460566,\n",
       "     'ISBN': None,\n",
       "     'volume': '324',\n",
       "     'issue': '4',\n",
       "     'year': 2002,\n",
       "     'title': 'The carboxy-terminal domain of the DExDH protein YxiN is sufficient to confer specificity for 23S rRNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '625-36',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Kossen K', 'Karginov FV', 'Uhlenbeck OC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0022-2836(02)01140-3'}},\n",
       "   'set_info': {'accession': 'CL0221', 'name': 'RRM'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 34691,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34552,\n",
       "    'proteomes': 5688,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30363, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 23181},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33233,\n",
       "    'alignment:full': 6889,\n",
       "    'alignment:seed': 505},\n",
       "   'cross_references': {}}},\n",
       " 'PF10436': {'metadata': {'accession': 'PF10436',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018955',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Mitochondrial branched-chain alpha-ketoacid dehydrogenase kinase',\n",
       "    'short': 'BCDHK_Adom3'},\n",
       "   'description': ['<p>Catabolism and synthesis of leucine, isoleucine and valine are finely balanced, allowing the body to make the most of dietary input but removing excesses to prevent toxic build-up of their corresponding keto-acids. This is the butyryl-CoA dehydrogenase, subunit A domain 3, a largely alpha-helical bundle of the enzyme BCDHK. This enzyme is the regulator of the dehydrogenase complex that breaks branched-chain amino-acids down, by phosphorylating and thereby inactivating it when synthesis is required. The domain is associated with family HATPase_c Pfam:PF02518 which is towards the C-terminal.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044923': {'PMID': 1377677,\n",
       "     'ISBN': None,\n",
       "     'volume': '267',\n",
       "     'issue': '19',\n",
       "     'year': 1992,\n",
       "     'title': 'Branched-chain alpha-ketoacid dehydrogenase kinase. Molecular cloning, expression, and sequence similarity with histidine protein kinases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13127-30',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Popov KM',\n",
       "      'Zhao Y',\n",
       "      'Shimomura Y',\n",
       "      'Kuntz MJ',\n",
       "      'Harris RA.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 141,\n",
       "    'interactions': 0,\n",
       "    'matches': 12320,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12157,\n",
       "    'proteomes': 2070,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9347, 'rosettafold': 0},\n",
       "    'structures': 67,\n",
       "    'taxa': 7554},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9278,\n",
       "    'alignment:full': 5937,\n",
       "    'alignment:seed': 268},\n",
       "   'cross_references': {}}},\n",
       " 'PF02953': {'metadata': {'accession': 'PF02953',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004217',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tim10/DDP family zinc finger', 'short': 'zf-Tim10_DDP'},\n",
       "   'description': ['<p>Putative zinc binding domain with four conserved cysteine residues.  This domain is found in the human disease protein Swiss:O60220.  Members of this family such as Tim9 and Tim10 are involved in mitochondrial protein import [PMID:11101512].       Members of this family seem to be localised to the mitochondrial intermembrane space [PMID:8663351].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007429': {'PMID': 11101512,\n",
       "     'ISBN': None,\n",
       "     'volume': '19',\n",
       "     'issue': '23',\n",
       "     'year': 2000,\n",
       "     'title': 'The role of the TIM8-13 complex in the import of Tim23 into mitochondria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6392-400',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Paschen SA',\n",
       "      'Rothbauer U',\n",
       "      'Kaldi K',\n",
       "      'Bauer MF',\n",
       "      'Neupert W',\n",
       "      'Brunner M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/19.23.6392'},\n",
       "    'PUB00007430': {'PMID': 8663351,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '29',\n",
       "     'year': 1996,\n",
       "     'title': 'Mrs5p, an essential protein of the mitochondrial intermembrane space, affects protein import into yeast mitochondria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17219-25',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Jarosch E',\n",
       "      'Tuller G',\n",
       "      'Daum G',\n",
       "      'Waldherr M',\n",
       "      'Voskova A',\n",
       "      'Schweyen RJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.29.17219'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 166,\n",
       "    'interactions': 0,\n",
       "    'matches': 13625,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13611,\n",
       "    'proteomes': 2129,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11433, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 7756},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11271,\n",
       "    'alignment:full': 7064,\n",
       "    'alignment:seed': 406},\n",
       "   'cross_references': {}}},\n",
       " 'PF16486': {'metadata': {'accession': 'PF16486',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032474',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'N-terminal domain of argonaute', 'short': 'ArgoN'},\n",
       "   'description': ['<p>ArgoN is the N-terminal domain of argonaute proteins in eukaryotes. ArgoN is composed of an antiparallel four-stranded beta sheet core that has two alpha helices positioned along one face of the sheet and an extended beta strand towards its N-terminus. The core fold of the N domain most closely resembles the catalytic domain of replication-initiator protein Rep. The N domain is linked to the PAZ domain via linker 1 region, and together these three regions are designated the PAZ-containing lobe of argonaute.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 360,\n",
       "    'interactions': 0,\n",
       "    'matches': 16665,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16594,\n",
       "    'proteomes': 1919,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12559, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 7046},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12906,\n",
       "    'alignment:full': 8918,\n",
       "    'alignment:seed': 271},\n",
       "   'cross_references': {}}},\n",
       " 'PF02811': {'metadata': {'accession': 'PF02811',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004013',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PHP domain', 'short': 'PHP'},\n",
       "   'description': ['<p>The PHP (Polymerase and Histidinol Phosphatase) domain is a putative phosphoesterase domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019430': {'PMID': 9685491,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '16',\n",
       "     'year': 1998,\n",
       "     'title': 'Phosphoesterase domains associated with DNA polymerases of diverse origins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3746-52',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Aravind L', 'Koonin EV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/26.16.3746'}},\n",
       "   'set_info': {'accession': 'CL0034', 'name': 'Amidohydrolase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 484,\n",
       "    'interactions': 0,\n",
       "    'matches': 150346,\n",
       "    'pathways': 0,\n",
       "    'proteins': 150307,\n",
       "    'proteomes': 10211,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 118581, 'rosettafold': 0},\n",
       "    'structures': 40,\n",
       "    'taxa': 43509},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 136822,\n",
       "    'alignment:full': 27454,\n",
       "    'alignment:seed': 478},\n",
       "   'cross_references': {}}},\n",
       " 'PF05697': {'metadata': {'accession': 'PF05697',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008881',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial trigger factor protein (TF)',\n",
       "    'short': 'Trigger_N'},\n",
       "   'description': ['<p>In the E. coli cytosol, a fraction of the newly synthesised proteins requires the activity of molecular chaperones for folding to the native state. The major chaperones implicated in this folding process are the ribosome-associated Trigger Factor (TF), and the DnaK and GroEL chaperones with their respective co-chaperones. Trigger Factor is an ATP-independent chaperone and displays chaperone and peptidyl-prolyl-cis-trans-isomerase (PPIase) activities in vitro. It is composed of at least three domains, an N-terminal domain which mediates association with the large ribosomal subunit, a central substrate binding and PPIase domain with homology to FKBP proteins, and a C-terminal domain of unknown function. The positioning of TF at the peptide exit channel, together with its ability to interact with nascent chains as short as 57 residues renders TF a prime candidate for being the first chaperone that binds to the nascent polypeptide chains [PMID:12603737]. This family represents the N-terminal region of the protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011507': {'PMID': 12603737,\n",
       "     'ISBN': None,\n",
       "     'volume': '47',\n",
       "     'issue': '5',\n",
       "     'year': 2003,\n",
       "     'title': 'Trigger Factor and DnaK possess overlapping substrate pools and binding specificities.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1317-28',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Deuerling E',\n",
       "      'Patzelt H',\n",
       "      'Vorderwulbecke S',\n",
       "      'Rauch T',\n",
       "      'Kramer G',\n",
       "      'Schaffitzel E',\n",
       "      'Mogk A',\n",
       "      'Schulze-Specking A',\n",
       "      'Langen H',\n",
       "      'Bukau B.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.2003.03370.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 63,\n",
       "    'interactions': 0,\n",
       "    'matches': 46266,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46258,\n",
       "    'proteomes': 8802,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39531, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 37574},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41990,\n",
       "    'alignment:full': 8558,\n",
       "    'alignment:seed': 796},\n",
       "   'cross_references': {}}},\n",
       " 'PF03055': {'metadata': {'accession': 'PF03055',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004294',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Retinal pigment epithelial membrane protein',\n",
       "    'short': 'RPE65'},\n",
       "   'description': ['<p>This family represents a retinal pigment epithelial membrane receptor which is abundantly expressed in retinal pigment epithelium, and binds plasma retinal binding protein.  The  family also includes the sequence related neoxanthin cleavage enzyme in plants and lignostilbene-alpha,beta-dioxygenase in bacteria.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020273': {'PMID': 7633413,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '4',\n",
       "     'year': 1995,\n",
       "     'title': 'Molecular characterization of the human gene encoding an abundant 61 kDa protein specific to the retinal pigment epithelium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '641-9',\n",
       "     'medline_journal': 'Hum Mol Genet',\n",
       "     'ISO_journal': 'Hum. Mol. Genet.',\n",
       "     'authors': ['Nicoletti A',\n",
       "      'Wong DJ',\n",
       "      'Kawase K',\n",
       "      'Gibson LH',\n",
       "      'Yang-Feng TL',\n",
       "      'Richards JE',\n",
       "      'Thompson DA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/hmg/4.4.641'}},\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 205,\n",
       "    'interactions': 0,\n",
       "    'matches': 27010,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26164,\n",
       "    'proteomes': 3180,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21026, 'rosettafold': 0},\n",
       "    'structures': 59,\n",
       "    'taxa': 13772},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22287,\n",
       "    'alignment:full': 9447,\n",
       "    'alignment:seed': 494},\n",
       "   'cross_references': {}}},\n",
       " 'PF01899': {'metadata': {'accession': 'PF01899',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002758',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Na+/H+ ion antiporter subunit', 'short': 'MNHE'},\n",
       "   'description': ['<p>Subunit of a Na+/H+ Prokaryotic antiporter complex ([PMID:9852009],[PMID:9680201]).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010583': {'PMID': 9852009,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '24',\n",
       "     'year': 1998,\n",
       "     'title': 'A putative multisubunit Na+/H+ antiporter from Staphylococcus aureus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6642-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Hiramatsu T',\n",
       "      'Kodama K',\n",
       "      'Kuroda T',\n",
       "      'Mizushima T',\n",
       "      'Tsuchiya T.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9852009'},\n",
       "    'PUB00010603': {'PMID': 9680201,\n",
       "     'ISBN': None,\n",
       "     'volume': '28',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'The pha gene cluster of Rhizobium meliloti involved in pH adaptation and symbiosis encodes a novel type of K+ efflux system.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1091-101',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Putnoky P',\n",
       "      'Kereszt A',\n",
       "      'Nakamura T',\n",
       "      'Endre G',\n",
       "      'Grosskopf E',\n",
       "      'Kiss P',\n",
       "      'Kondorosi A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1998.00868.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 20122,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20122,\n",
       "    'proteomes': 3836,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17638, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 16524},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18709,\n",
       "    'alignment:full': 4412,\n",
       "    'alignment:seed': 329},\n",
       "   'cross_references': {}}},\n",
       " 'PF06719': {'metadata': {'accession': 'PF06719',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009594',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AraC-type transcriptional regulator N-terminus',\n",
       "    'short': 'AraC_N'},\n",
       "   'description': [\"<p>This family represents the N-terminus of bacterial ARAC-type transcriptional regulators. In E. coli, these regulate the L-arabinose operon through sensing the presence of arabinose, and when the sugar is present, transmitting this information from the arabinose-binding domains to the protein's DNA-binding domains [PMID:12683999]. This family might represent the N-terminal arm of the protein, which binds to the C-terminal DNA binding domains to hold them in a state where the protein prefers to loop and remain non-activating [PMID:9600837]. All family members contain the Pfam:PF00165 domain.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012804': {'PMID': 12683999,\n",
       "     'ISBN': None,\n",
       "     'volume': '328',\n",
       "     'issue': '1',\n",
       "     'year': 2003,\n",
       "     'title': 'Mutational analysis of residue roles in AraC function.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '85-93',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Ross JJ', 'Gryczynski U', 'Schleif R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0022-2836(03)00262-6'},\n",
       "    'PUB00012805': {'PMID': 9600837,\n",
       "     'ISBN': None,\n",
       "     'volume': '278',\n",
       "     'issue': '3',\n",
       "     'year': 1998,\n",
       "     'title': 'Arm-domain interactions in AraC.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '539-48',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Saviola B', 'Seabold R', 'Schleif RF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1998.1712'}},\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 14,\n",
       "    'interactions': 0,\n",
       "    'matches': 18706,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18701,\n",
       "    'proteomes': 2184,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17000, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 11230},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19609,\n",
       "    'alignment:full': 3222,\n",
       "    'alignment:seed': 248},\n",
       "   'cross_references': {}}},\n",
       " 'PF03740': {'metadata': {'accession': 'PF03740',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004569',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pyridoxal phosphate biosynthesis protein PdxJ',\n",
       "    'short': 'PdxJ'},\n",
       "   'description': [\"<p>Members of this family belong to the PdxJ family that catalyses the condensation of 1-deoxy-d-xylulose-5-phosphate (DXP) and 1-amino-3-oxo-4-(phosphohydroxy)propan-2-one to form pyridoxine 5'-phosphate (PNP). This reaction is involved in de novo synthesis of pyridoxine (vitamin B6) and pyridoxal phosphate [PMID:10225425].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009884': {'PMID': 10225425,\n",
       "     'ISBN': None,\n",
       "     'volume': '449',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': \"Vitamin B6 biosynthesis: formation of pyridoxine 5'-phosphate from 4-(phosphohydroxy)-L-threonine and 1-deoxy-D-xylulose-5-phosphate by PdxA and PdxJ protein.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '45-8',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Laber B',\n",
       "      'Maurer W',\n",
       "      'Scharf S',\n",
       "      'Stepusin K',\n",
       "      'Schmidt FS.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(99)00393-2'}},\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 36,\n",
       "    'interactions': 0,\n",
       "    'matches': 25623,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25609,\n",
       "    'proteomes': 4479,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21723, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 20416},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23253,\n",
       "    'alignment:full': 4181,\n",
       "    'alignment:seed': 335},\n",
       "   'cross_references': {}}},\n",
       " 'PF01687': {'metadata': {'accession': 'PF01687',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015865',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Riboflavin kinase', 'short': 'Flavokinase'},\n",
       "   'description': ['<p>This family represents the C-terminal region of the bifunctional riboflavin biosynthesis protein known as RibC in Bacillus subtilis.  The RibC protein from Bacillus subtilis has both flavokinase and flavin adenine dinucleotide synthetase (FAD-synthetase) activities. RibC plays an essential role in the flavin metabolism [PMID:9473052].  This domain is thought to have kinase activity [PMID:15468322].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012656': {'PMID': 9473052,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '4',\n",
       "     'year': 1998,\n",
       "     'title': 'Regulation of riboflavin biosynthesis in Bacillus subtilis is affected by the activity of the flavokinase/flavin adenine dinucleotide synthetase encoded by ribC.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '950-5',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Mack M', 'van Loon AP', 'Hohmann HP.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/180/4/950'},\n",
       "    'PUB00019535': {'PMID': 15468322,\n",
       "     'ISBN': None,\n",
       "     'volume': '58',\n",
       "     'issue': '1',\n",
       "     'year': 2005,\n",
       "     'title': 'Crystal structure of flavin binding to FAD synthetase of Thermotoga maritima.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '246-8',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Wang W', 'Kim R', 'Yokota H', 'Kim SH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/prot.20207'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 178,\n",
       "    'interactions': 0,\n",
       "    'matches': 48456,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48419,\n",
       "    'proteomes': 10395,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41335, 'rosettafold': 0},\n",
       "    'structures': 26,\n",
       "    'taxa': 42110},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43978,\n",
       "    'alignment:full': 9974,\n",
       "    'alignment:seed': 1005},\n",
       "   'cross_references': {}}},\n",
       " 'PF01179': {'metadata': {'accession': 'PF01179',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015798',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Copper amine oxidase, enzyme domain',\n",
       "    'short': 'Cu_amine_oxid'},\n",
       "   'description': ['<p>Copper amine oxidases are a ubiquitous and novel group of quinoenzymes that catalyse the oxidative deamination of primary amines to the corresponding aldehydes, with concomitant reduction of molecular oxygen to hydrogen peroxide. The enzymes are dimers of identical 70-90 kDa subunits, each of which contains a single copper ion and a covalently bound cofactor formed by the post-translational modification of a tyrosine side chain to 2,4,5-trihydroxyphenylalanine quinone (TPQ). This family corresponds to the catalytic domain of the enzyme. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005251': {'PMID': 8591028,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '11',\n",
       "     'year': 1995,\n",
       "     'title': 'Crystal structure of a quinoenzyme: copper amine oxidase of Escherichia coli at 2 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1171-84',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Parsons MR',\n",
       "      'Convery MA',\n",
       "      'Wilmot CM',\n",
       "      'Yadav KD',\n",
       "      'Blakeley V',\n",
       "      'Corner AS',\n",
       "      'Phillips SE',\n",
       "      'McPherson MJ',\n",
       "      'Knowles PF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00253-2'}},\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 228,\n",
       "    'interactions': 0,\n",
       "    'matches': 17268,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16595,\n",
       "    'proteomes': 2554,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13613, 'rosettafold': 0},\n",
       "    'structures': 145,\n",
       "    'taxa': 9966},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14986,\n",
       "    'alignment:full': 7217,\n",
       "    'alignment:seed': 299},\n",
       "   'cross_references': {}}},\n",
       " 'PF14667': {'metadata': {'accession': 'PF14667',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029303',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polysaccharide biosynthesis C-terminal domain',\n",
       "    'short': 'Polysacc_synt_C'},\n",
       "   'description': ['<p>This family represents the C-terminal integral membrane region of polysaccharide biosynthesis proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0222', 'name': 'MviN_MATE'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 160,\n",
       "    'interactions': 0,\n",
       "    'matches': 66767,\n",
       "    'pathways': 0,\n",
       "    'proteins': 66761,\n",
       "    'proteomes': 5709,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 57392, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 26058},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 60925,\n",
       "    'alignment:full': 12144,\n",
       "    'alignment:seed': 317},\n",
       "   'cross_references': {}}},\n",
       " 'PF03799': {'metadata': {'accession': 'PF03799',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005548',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cell division protein FtsQ/DivIB, C-terminal',\n",
       "    'short': 'FtsQ_DivIB_C'},\n",
       "   'description': ['<p>FtsQ is an essential cell division protein. It may link together the upstream cell division proteins, which are predominantly cytoplasmic, with the downstream cell division proteins, which are predominantly periplasmic [PMID:17185541]. FtsQ may control the correct divisome assembly [PMID:19233928]. DivIB is a cell division protein from Gram-positive bacteria, probably homologous to Escherichia coli FtsQ. DivIB interacts with FtsL, DivIC and PBP-2B [PMID:16936019, PMID:20870765]. DivIB plays an essential role in division at high temperatures, maybe by protecting FtsL from degradation or by promoting formation of the FtsL-DivIC complex [PMID:10792716]. It is also required for efficient sporulation at all temperatures [PMID:16936026]. FtsQ and DivIB have a short N-terminal cytoplasmic domain and a larger C-terminal periplasmic domain [PMID:19233928, PMID:20870765]. This entry represents the C-terminal region.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008438': {'PMID': 9864306,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Metabolic alarms and cell division in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9-14',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Joseleau-Petit D', 'Vinella D', \"D'Ari R.\"],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9864306&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 27578,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27576,\n",
       "    'proteomes': 5844,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23909, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 24573},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25824,\n",
       "    'alignment:full': 5343,\n",
       "    'alignment:seed': 318},\n",
       "   'cross_references': {}}},\n",
       " 'PF00677': {'metadata': {'accession': 'PF00677',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026017',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Lumazine binding domain', 'short': 'Lum_binding'},\n",
       "   'description': ['<p>This domain binds to derivatives of lumazine  in some proteins. Some proteins have lost the residues involved in binding lumazine.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0076', 'name': 'FAD_Lum_binding'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 76,\n",
       "    'interactions': 0,\n",
       "    'matches': 83858,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42693,\n",
       "    'proteomes': 8786,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 36953, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 36131},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 77415,\n",
       "    'alignment:full': 16593,\n",
       "    'alignment:seed': 1598},\n",
       "   'cross_references': {}}},\n",
       " 'PF16355': {'metadata': {'accession': 'PF16355',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032311',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4982)',\n",
       "    'short': 'DUF4982'},\n",
       "   'description': ['<p>This family is found in the C-terminal of uncharacterized proteins and beta-galactosidases around 680 residues in length from various Bacteroides species. The function of this protein is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 323,\n",
       "    'interactions': 0,\n",
       "    'matches': 13563,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13534,\n",
       "    'proteomes': 1514,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11199, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 6219},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12805,\n",
       "    'alignment:full': 3297,\n",
       "    'alignment:seed': 259},\n",
       "   'cross_references': {}}},\n",
       " 'PF08310': {'metadata': {'accession': 'PF08310',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013207',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LGFP repeat', 'short': 'LGFP'},\n",
       "   'description': ['<p>This 54 amino acid repeat is found in many hypothetical proteins. Several hypothetical proteins from C.glutamicum and C.efficiens along with PS1 protein contain this repeat region. The N-terminus region of PS1 contains an esterase domain which transfers corynomycolic acid. The C-terminus region consists of 4 tandem LGFP repeats. It is hypothesised that the PS1 proteins in Corynebacterium, when associated with the cell wall, may be anchored via the LGFP tandem repeats that may be important for maintaining cell wall integrity [Adindla et al. Comparative and Functional Genomics 2004; 5:2-16].  Deletion of Swiss:Q01377 protein results in a 10-fold increase in the cell volume of the organism and infers the corresponding proteins involvement in the cell shape formation [PMID:12740729]. The secondary structure of each repeat is predicted to comprise two beta-strands and one alpha-helix [Adindla et al. 2004].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00017162': {'PMID': 12740729,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '1',\n",
       "     'year': 2003,\n",
       "     'title': 'Identification and functional analysis of six mycolyltransferase genes of Corynebacterium glutamicum ATCC 13032: the genes cop1, cmt1, and cmt2 can replace each other in the synthesis of trehalose dicorynomycolate, a component of the mycolic acid layer of the cell envelope.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '33-44',\n",
       "     'medline_journal': 'Arch Microbiol',\n",
       "     'ISO_journal': 'Arch. Microbiol.',\n",
       "     'authors': ['Brand S', 'Niehaus K', 'Puhler A', 'Kalinowski J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/s00203-003-0556-1'}},\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 389,\n",
       "    'interactions': 0,\n",
       "    'matches': 17447,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6342,\n",
       "    'proteomes': 819,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 5538, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 3470},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17014,\n",
       "    'alignment:full': 4847,\n",
       "    'alignment:seed': 336},\n",
       "   'cross_references': {}}},\n",
       " 'PF01590': {'metadata': {'accession': 'PF01590',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003018',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GAF domain', 'short': 'GAF'},\n",
       "   'description': [\"<p>This domain is present in cGMP-specific phosphodiesterases, adenylyl and guanylyl cyclases, phytochromes, FhlA and NifA. Adenylyl and guanylyl cyclases catalyse ATP and GTP to the second messengers cAMP and cGMP, respectively, these products up-regulating catalytic activity by binding to the regulatory GAF domain(s). The opposite hydrolysis reaction is catalysed by phosphodiesterase. cGMP-dependent 3',5'-cyclic phosphodiesterase catalyses the conversion of guanosine 3',5'-cyclic phosphate to guanosine 5'-phosphate. Here too, cGMP regulates catalytic activity by GAF-domain binding. Phytochromes are regulatory photoreceptors in plants and bacteria which exist in two thermally-stable states that are reversibly inter-convertible by light: the Pr state absorbs maximally in the red region of the spectrum, while the Pfr state absorbs maximally in the far-red region. This domain is also found in FhlA (formate hydrogen lyase transcriptional activator) and NifA, a transcriptional activator which is required for activation of most Nif operons which are directly involved in nitrogen fixation. NifA interacts with sigma-54. This domain can bind biliverdine and phycocyanobilin (Matilla et al., FEMS Microbiology Reviews, fuab043, 45, 2021, 1. https://doi.org/10.1093/femsre/fuab043).</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005474': {'PMID': 9433123,\n",
       "     'ISBN': None,\n",
       "     'volume': '22',\n",
       "     'issue': '12',\n",
       "     'year': 1997,\n",
       "     'title': 'The GAF domain: an evolutionary link between diverse phototransducing proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '458-9',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Aravind L', 'Ponting CP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(97)01148-1'}},\n",
       "   'set_info': {'accession': 'CL0161', 'name': 'GAF'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11091,\n",
       "    'interactions': 0,\n",
       "    'matches': 185287,\n",
       "    'pathways': 0,\n",
       "    'proteins': 166960,\n",
       "    'proteomes': 8048,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 127646, 'rosettafold': 0},\n",
       "    'structures': 151,\n",
       "    'taxa': 42537},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 161482,\n",
       "    'alignment:full': 40137,\n",
       "    'alignment:seed': 442},\n",
       "   'cross_references': {}}},\n",
       " 'PF04168': {'metadata': {'accession': 'PF04168',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007296',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'A predicted alpha-helical domain with a conserved ER motif.',\n",
       "    'short': 'Alpha-E'},\n",
       "   'description': ['<p>An uncharacterized alpha helical domain containing a highly  conserved ER motif and typically found as a tandem duplication.  Contextual analysis suggests that it functions in a distinct peptide  synthesis/modification system comprising of a transglutaminase,  a peptidase of the NTN-hydrolase superfamily, an active and inactive  circularly permuted ATP-grasp domains and a transglutaminase fused  N-terminal to a circularly permuted COOH-NH2 ligase domain [PMID:20023723].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 32,\n",
       "    'interactions': 0,\n",
       "    'matches': 21566,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21517,\n",
       "    'proteomes': 3317,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18425, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13898},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19981,\n",
       "    'alignment:full': 4529,\n",
       "    'alignment:seed': 335},\n",
       "   'cross_references': {}}},\n",
       " 'PF00476': {'metadata': {'accession': 'PF00476',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001098',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DNA polymerase family A', 'short': 'DNA_pol_A'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 317,\n",
       "    'interactions': 0,\n",
       "    'matches': 73082,\n",
       "    'pathways': 0,\n",
       "    'proteins': 72201,\n",
       "    'proteomes': 12008,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 52714, 'rosettafold': 0},\n",
       "    'structures': 251,\n",
       "    'taxa': 49343},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 66462,\n",
       "    'alignment:full': 14825,\n",
       "    'alignment:seed': 640},\n",
       "   'cross_references': {}}},\n",
       " 'PF04325': {'metadata': {'accession': 'PF04325',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007420',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF465)', 'short': 'DUF465'},\n",
       "   'description': ['<p>Family members are found in small bacterial proteins, and also in the heavy chains of eukaryotic myosin and kinesin, C terminal of the motor domain (Myosin Pfam:PF00063, Kinesin Pfam:PF00225). Members of this family may form coiled coil structures.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 22,\n",
       "    'interactions': 0,\n",
       "    'matches': 18919,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18919,\n",
       "    'proteomes': 2595,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16395, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 11419},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17494,\n",
       "    'alignment:full': 4211,\n",
       "    'alignment:seed': 250},\n",
       "   'cross_references': {}}},\n",
       " 'PF02887': {'metadata': {'accession': 'PF02887',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015795',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pyruvate kinase, alpha/beta domain', 'short': 'PK_C'},\n",
       "   'description': ['<p>As well as being found in pyruvate kinase this family is found as an isolated domain in some bacterial proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020130': {'PMID': 9308890,\n",
       "     'ISBN': None,\n",
       "     'volume': '345',\n",
       "     'issue': '2',\n",
       "     'year': 1997,\n",
       "     'title': 'Ligand-induced domain movement in pyruvate kinase: structure of the enzyme from rabbit muscle with Mg2+, K+, and L-phospholactate at 2.7 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '199-206',\n",
       "     'medline_journal': 'Arch Biochem Biophys',\n",
       "     'ISO_journal': 'Arch. Biochem. Biophys.',\n",
       "     'authors': ['Larsen TM',\n",
       "      'Benning MM',\n",
       "      'Wesenberg GE',\n",
       "      'Rayment I',\n",
       "      'Reed GH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/abbi.1997.0257'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 196,\n",
       "    'interactions': 0,\n",
       "    'matches': 60770,\n",
       "    'pathways': 0,\n",
       "    'proteins': 60621,\n",
       "    'proteomes': 10427,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 51504, 'rosettafold': 0},\n",
       "    'structures': 160,\n",
       "    'taxa': 42522},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 54573,\n",
       "    'alignment:full': 14885,\n",
       "    'alignment:seed': 599},\n",
       "   'cross_references': {}}},\n",
       " 'PF07501': {'metadata': {'accession': 'PF07501',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011098',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'G5 domain', 'short': 'G5'},\n",
       "   'description': ['<p>This domain is found in a wide range of extracellular proteins. It is found tandemly repeated in up to 8 copies.  It is found in the N-terminus of peptidases belonging to the M26 family which cleave human IgA.  The domain is also found in proteins involved in metabolism of bacterial cell walls suggesting this domain may have an adhesive function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0593', 'name': 'G5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 764,\n",
       "    'interactions': 0,\n",
       "    'matches': 26324,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19992,\n",
       "    'proteomes': 2662,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14899, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 9368},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25896,\n",
       "    'alignment:full': 4813,\n",
       "    'alignment:seed': 291},\n",
       "   'cross_references': {}}},\n",
       " 'PF00507': {'metadata': {'accession': 'PF00507',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000440',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NADH-ubiquinone/plastoquinone oxidoreductase, chain 3',\n",
       "    'short': 'Oxidored_q4'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 64,\n",
       "    'interactions': 0,\n",
       "    'matches': 79867,\n",
       "    'pathways': 0,\n",
       "    'proteins': 79696,\n",
       "    'proteomes': 6205,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 69696, 'rosettafold': 0},\n",
       "    'structures': 220,\n",
       "    'taxa': 86343},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 71270,\n",
       "    'alignment:full': 7148,\n",
       "    'alignment:seed': 366},\n",
       "   'cross_references': {}}},\n",
       " 'PF01709': {'metadata': {'accession': 'PF01709',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002876',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transcriptional regulator', 'short': 'Transcrip_reg'},\n",
       "   'description': ['<p>This is a family of transcriptional regulators. In mammals, it activates the transcription of mitochondrially-encoded COX1 [PMID:19503089]. In bacteria, it negatively regulates the quorum-sensing response regulator by binding to its promoter region [PMID:18641136].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00027019': {'PMID': 12060744,\n",
       "     'ISBN': None,\n",
       "     'volume': '99',\n",
       "     'issue': '12',\n",
       "     'year': 2002,\n",
       "     'title': 'Crystal structure of conserved hypothetical protein Aq1575 from Aquifex aeolicus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7980-5',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Shin DH', 'Yokota H', 'Kim R', 'Kim SH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.132241399'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 52990,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52912,\n",
       "    'proteomes': 10181,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45109, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 43304},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48299,\n",
       "    'alignment:full': 9984,\n",
       "    'alignment:seed': 695},\n",
       "   'cross_references': {}}},\n",
       " 'PF00253': {'metadata': {'accession': 'PF00253',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001209',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S14p/S29e', 'short': 'Ribosomal_S14'},\n",
       "   'description': ['<p>This family includes both ribosomal S14 from prokaryotes and S29 from eukaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 115,\n",
       "    'interactions': 0,\n",
       "    'matches': 65457,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65449,\n",
       "    'proteomes': 10713,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 56358, 'rosettafold': 0},\n",
       "    'structures': 1020,\n",
       "    'taxa': 64317},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57482,\n",
       "    'alignment:full': 12097,\n",
       "    'alignment:seed': 336},\n",
       "   'cross_references': {}}},\n",
       " 'PF08516': {'metadata': {'accession': 'PF08516',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006586',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ADAM cysteine-rich', 'short': 'ADAM_CR'},\n",
       "   'description': [\"<p>ADAMs are membrane-anchored proteases that proteolytically modify cell surface and extracellular matrix (ECM) in order to alter cell behaviour.  It has been shown that the cysteine-rich domain of ADAM13 regulates the protein's metalloprotease activity [PMID:12460986].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020872': {'PMID': 12460986,\n",
       "     'ISBN': None,\n",
       "     'volume': '159',\n",
       "     'issue': '5',\n",
       "     'year': 2002,\n",
       "     'title': 'The cysteine-rich domain regulates ADAM protease function in vivo.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '893-902',\n",
       "     'medline_journal': 'J Cell Biol',\n",
       "     'ISO_journal': 'J. Cell Biol.',\n",
       "     'authors': ['Smith KM',\n",
       "      'Gaultier A',\n",
       "      'Cousin H',\n",
       "      'Alfandari D',\n",
       "      'White JM',\n",
       "      'DeSimone DW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1083/jcb.200206023'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 203,\n",
       "    'interactions': 0,\n",
       "    'matches': 20137,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20062,\n",
       "    'proteomes': 908,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13196, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 3754},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13984,\n",
       "    'alignment:full': 8308,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF07735': {'metadata': {'accession': 'PF07735',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012885',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'F-box associated', 'short': 'FBA_2'},\n",
       "   'description': ['<p>Most of these proteins contain Pfam:PF00646 at the N terminus, suggesting that they are effectors linked with ubiquitination.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 82,\n",
       "    'interactions': 0,\n",
       "    'matches': 5056,\n",
       "    'pathways': 0,\n",
       "    'proteins': 4739,\n",
       "    'proteomes': 9,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4598, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 55},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 5047,\n",
       "    'alignment:full': 3126,\n",
       "    'alignment:seed': 366},\n",
       "   'cross_references': {}}},\n",
       " 'PF03948': {'metadata': {'accession': 'PF03948',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020069',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L9, C-terminal domain',\n",
       "    'short': 'Ribosomal_L9_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001252': {'PMID': 8306963,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '1',\n",
       "     'year': 1994,\n",
       "     'title': 'Crystal structure of prokaryotic ribosomal protein L9: a bi-lobed RNA-binding protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '205-12',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Hoffman DW',\n",
       "      'Davies C',\n",
       "      'Gerchman SE',\n",
       "      'Kycia JH',\n",
       "      'Porter SJ',\n",
       "      'White SW',\n",
       "      'Ramakrishnan V.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8306963&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 43344,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43343,\n",
       "    'proteomes': 8981,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37215, 'rosettafold': 0},\n",
       "    'structures': 583,\n",
       "    'taxa': 38534},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39124,\n",
       "    'alignment:full': 8421,\n",
       "    'alignment:seed': 819},\n",
       "   'cross_references': {}}},\n",
       " 'PF03134': {'metadata': {'accession': 'PF03134',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004345',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TB2/DP1, HVA22 family', 'short': 'TB2_DP1_HVA22'},\n",
       "   'description': ['<p>This family includes members from a wide variety of eukaryotes. It includes the TB2/DP1 (deleted in polyposis) protein  (e.g. Swiss:Q00765), which in humans is deleted in severe forms of familial adenomatous polyposis, an autosomal dominant oncological inherited disease. The family also includes the plant protein of known similarity to TB2/DP1, the HVA22 abscisic acid-induced protein (e.g. Swiss:Q07764), which is thought to be a regulatory protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020410': {'PMID': 10805953,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '4',\n",
       "     'year': 2000,\n",
       "     'title': 'Familial adenomatous polyposis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '314-23',\n",
       "     'medline_journal': 'Semin Surg Oncol',\n",
       "     'ISO_journal': None,\n",
       "     'authors': ['Lal G', 'Gallinger S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/(SICI)1098-2388(200006)18:4<314::AID-SSU6>3.0.CO;2-9'},\n",
       "    'PUB00020411': {'PMID': 8226892,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '31',\n",
       "     'year': 1993,\n",
       "     'title': 'Hormone response complex in a novel abscisic acid and cycloheximide-inducible barley gene.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '23652-60',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Shen Q', 'Uknes SJ', 'Ho TH.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/268/31/23652'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 265,\n",
       "    'interactions': 0,\n",
       "    'matches': 17474,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17383,\n",
       "    'proteomes': 2136,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14040, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7918},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13960,\n",
       "    'alignment:full': 8645,\n",
       "    'alignment:seed': 313},\n",
       "   'cross_references': {}}},\n",
       " 'PF00707': {'metadata': {'accession': 'PF00707',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019815',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Translation initiation factor IF-3, C-terminal domain',\n",
       "    'short': 'IF3_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 45070,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45065,\n",
       "    'proteomes': 9273,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38651, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 39883},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40642,\n",
       "    'alignment:full': 8883,\n",
       "    'alignment:seed': 446},\n",
       "   'cross_references': {}}},\n",
       " 'PF01969': {'metadata': {'accession': 'PF01969',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002822',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nickel insertion protein', 'short': 'Ni_insertion'},\n",
       "   'description': ['<p>Members of this family may be involved in the activation of nickel-pincer cofactor-dependent enzymes. LarC from Lactobacillus plantarum is involved, together with LarB and LarE, in the synthesis of the enzyme-bound cofactor of lactate racemase (LarA). Larc C binds Ni2+, and functions in nickel delivery to pyridinium-3,5-bisthiocarboxylic acid mononucleotide (P2TMN), to form the mature cofactor [PMID:24710389, PMID:27114550].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 30,\n",
       "    'interactions': 0,\n",
       "    'matches': 12877,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12771,\n",
       "    'proteomes': 1954,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10656, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 9677},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10886,\n",
       "    'alignment:full': 2219,\n",
       "    'alignment:seed': 269},\n",
       "   'cross_references': {}}},\n",
       " 'PF04298': {'metadata': {'accession': 'PF04298',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007395',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative neutral zinc metallopeptidase',\n",
       "    'short': 'Zn_peptidase_2'},\n",
       "   'description': ['<p>Zinc metallopeptidase zinc binding regions have been predicted in some family members by a pattern match (Prosite:PS00142), of the characteristic HEXXH motif.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0126', 'name': 'Peptidase_MA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 11,\n",
       "    'interactions': 0,\n",
       "    'matches': 12313,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12299,\n",
       "    'proteomes': 2661,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10257, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 10623},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10776,\n",
       "    'alignment:full': 2618,\n",
       "    'alignment:seed': 300},\n",
       "   'cross_references': {}}},\n",
       " 'PF03140': {'metadata': {'accession': 'PF03140',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004158',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plant protein of unknown function', 'short': 'DUF247'},\n",
       "   'description': ['<p>The function of the plant proteins constituting this family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 203,\n",
       "    'interactions': 0,\n",
       "    'matches': 15802,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14113,\n",
       "    'proteomes': 204,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12071, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 751},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13346,\n",
       "    'alignment:full': 9205,\n",
       "    'alignment:seed': 272},\n",
       "   'cross_references': {}}},\n",
       " 'PF01016': {'metadata': {'accession': 'PF01016',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001684',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal L27 protein', 'short': 'Ribosomal_L27'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 94,\n",
       "    'interactions': 0,\n",
       "    'matches': 45334,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45303,\n",
       "    'proteomes': 10618,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38852, 'rosettafold': 0},\n",
       "    'structures': 827,\n",
       "    'taxa': 44880},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40374,\n",
       "    'alignment:full': 9904,\n",
       "    'alignment:seed': 345},\n",
       "   'cross_references': {}}},\n",
       " 'PF04248': {'metadata': {'accession': 'PF04248',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007361',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF427)',\n",
       "    'short': 'NTP_transf_9'},\n",
       "   'description': ['<p>This domain contains a beta-tent fold [PMID:25569776].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0080', 'name': 'Beta-tent'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 81,\n",
       "    'interactions': 0,\n",
       "    'matches': 21875,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19257,\n",
       "    'proteomes': 3491,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16583, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 13645},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19861,\n",
       "    'alignment:full': 5528,\n",
       "    'alignment:seed': 398},\n",
       "   'cross_references': {}}},\n",
       " 'PF14031': {'metadata': {'accession': 'PF14031',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026956',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative serine dehydratase domain',\n",
       "    'short': 'D-ser_dehydrat'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of yeast D-serine dehydratase [PMID:17937657]. Structures have been solved for two bacterial members of this family. The yeast protein has been shown to be a zinc dependant enzyme.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 72,\n",
       "    'interactions': 0,\n",
       "    'matches': 20907,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20904,\n",
       "    'proteomes': 3895,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17883, 'rosettafold': 0},\n",
       "    'structures': 18,\n",
       "    'taxa': 15599},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19260,\n",
       "    'alignment:full': 4567,\n",
       "    'alignment:seed': 346},\n",
       "   'cross_references': {}}},\n",
       " 'PF18758': {'metadata': {'accession': 'PF18758',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040521',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Kyakuja-Dileera-Zisupton transposase', 'short': 'KDZ'},\n",
       "   'description': ['<p>A transposase family with an RNaseH catalytic domain, often fused to DNA binding domains such as SAP or cysteine cluster domains. KDZ transposases are widely present in fungi, metazoa, chlorophytes and haotpohytes. Fungal versions are often associated with a TET/JBP family of dioxygenases [PMID:24398522, PMID:21873630].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0219', 'name': 'RNase_H'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 78,\n",
       "    'interactions': 0,\n",
       "    'matches': 8403,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8187,\n",
       "    'proteomes': 247,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 6408, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1006},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6845,\n",
       "    'alignment:full': 5815,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF13742': {'metadata': {'accession': 'PF13742',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025824',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'OB-fold nucleic acid binding domain',\n",
       "    'short': 'tRNA_anti_2'},\n",
       "   'description': ['<p>This family contains OB-fold domains that bind to nucleic acids.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 41353,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41351,\n",
       "    'proteomes': 7733,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 35366, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 32413},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38154,\n",
       "    'alignment:full': 7105,\n",
       "    'alignment:seed': 634},\n",
       "   'cross_references': {}}},\n",
       " 'PF01052': {'metadata': {'accession': 'PF01052',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001543',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Type III flagellar switch regulator (C-ring) FliN C-term',\n",
       "    'short': 'FliMN_C'},\n",
       "   'description': ['<p>This family includes the C-terminal region of flagellar motor switch proteins FliN and FliM. It is associated with family FliM, Pfam:PF02154 and family FliN_N Pfam:PF16973.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003825': {'PMID': 1447979,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '18',\n",
       "     'year': 1992,\n",
       "     'title': 'Identification and characterization of FliY, a novel component of the Bacillus subtilis flagellar switch complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2715-23',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Bischoff DS', 'Ordal GW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.1992.tb01448.x'},\n",
       "    'PUB00003876': {'PMID': 8885278,\n",
       "     'ISBN': None,\n",
       "     'volume': '21',\n",
       "     'issue': '5',\n",
       "     'year': 1996,\n",
       "     'title': 'Salmonella spp. are cytotoxic for cultured macrophages.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1101-15',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Chen LM', 'Kaniga K', 'Galan JE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1365-2958.1996.471410.x'},\n",
       "    'PUB00043937': {'PMID': 15805535,\n",
       "     'ISBN': None,\n",
       "     'volume': '187',\n",
       "     'issue': '8',\n",
       "     'year': 2005,\n",
       "     'title': 'Crystal structure of the flagellar rotor protein FliN from Thermotoga maritima.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2890-902',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Brown PN', 'Mathews MA', 'Joss LA', 'Hill CP', 'Blair DF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.187.8.2890-2902.2005'},\n",
       "    'PUB00045137': {'PMID': 16547037,\n",
       "     'ISBN': None,\n",
       "     'volume': '188',\n",
       "     'issue': '7',\n",
       "     'year': 2006,\n",
       "     'title': 'Organization of FliN subunits in the flagellar motor of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2502-11',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Paul K', 'Blair DF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.188.7.2502-2511.2006'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 50062,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49926,\n",
       "    'proteomes': 4312,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 43807, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 18531},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47888,\n",
       "    'alignment:full': 9268,\n",
       "    'alignment:seed': 633},\n",
       "   'cross_references': {}}},\n",
       " 'PF13207': {'metadata': {'accession': 'PF13207',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AAA domain', 'short': 'AAA_17'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 6722,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6719,\n",
       "    'proteomes': 1065,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 5852, 'rosettafold': 0},\n",
       "    'structures': 14,\n",
       "    'taxa': 6627},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 5672,\n",
       "    'alignment:full': 1189,\n",
       "    'alignment:seed': 264},\n",
       "   'cross_references': {}}},\n",
       " 'PF03808': {'metadata': {'accession': 'PF03808',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004629',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl transferase WecG/TagA/CpsF family',\n",
       "    'short': 'Glyco_tran_WecG'},\n",
       "   'description': ['<p>The WecG member of this family, believed to be UDP-N-acetyl-D-mannosaminuronic acid transferase, plays a role in enterobacterial common antigen (eca) synthesis in Escherichia coli. Another family member, the Bacillus subtilis TagA protein, is involved in the biosynthesis of the cell wall polymer poly(glycerol phosphate). The third family member, CpsF, CMP-N-acetylneuraminic acid synthetase has a role in the capsular polysaccharide biosynthesis pathway. Also included in this group is Xanthomonas campestris pv. campestris GumM, a glycosyltransferase participating in the biosynthesis of the exopolysaccharide xanthan [PMID:8830246, PMID:11673418, PMID:12618464, PMID:18156271, PMID:16953575, PMID:3275612, PMID:9537354].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 88,\n",
       "    'interactions': 0,\n",
       "    'matches': 24217,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24164,\n",
       "    'proteomes': 3776,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20628, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 18445},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22111,\n",
       "    'alignment:full': 4423,\n",
       "    'alignment:seed': 537},\n",
       "   'cross_references': {}}},\n",
       " 'PF01329': {'metadata': {'accession': 'PF01329',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001533',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pterin 4 alpha carbinolamine dehydratase',\n",
       "    'short': 'Pterin_4a'},\n",
       "   'description': ['<p>Pterin 4 alpha carbinolamine dehydratase is also known as DCoH (dimerisation cofactor of hepatocyte nuclear factor 1-alpha).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005041': {'PMID': 8897596,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '10',\n",
       "     'year': 1996,\n",
       "     'title': 'High-resolution structures of the bifunctional enzyme and transcriptional coactivator DCoH and its complex with a product analogue.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1963-72',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Cronk JD', 'Endrizzi JA', 'Alber T.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8897596'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 111,\n",
       "    'interactions': 0,\n",
       "    'matches': 32871,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32858,\n",
       "    'proteomes': 7172,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 27646, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 29052},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28739,\n",
       "    'alignment:full': 7668,\n",
       "    'alignment:seed': 481},\n",
       "   'cross_references': {}}},\n",
       " 'PF02673': {'metadata': {'accession': 'PF02673',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003824',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacitracin resistance protein BacA', 'short': 'BacA'},\n",
       "   'description': ['<p>Bacitracin resistance protein (BacA) is a putative undecaprenol kinase. BacA confers resistance to bacitracin, probably by phosphorylation of undecaprenol [PMID:8389741]. More recent studies show that BacA has undecaprenyl pyrophosphate phosphatase activity. Undecaprenyl phosphate is a key lipid intermediate involved in the synthesis of various bacterial cell wall polymers. Bacitracin, a mixture of related cyclic polypeptide antibiotics, is used to treat surface tissue infections. Its primary mode of action is the inhibition of bacterial cell wall synthesis through sequestration of the essential carrier lipid undecaprenyl pyrophosphate, C55-PP, resulting in the loss of cell integrity and lysis [PMID:15138271, PMID:15778224]. The characteristic phosphatase sequence-motif in this family is likely to be the PGxSRSGG, compared with the PSGH of the PAP family of phosphatases [PMID:15778224].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008274': {'PMID': 8389741,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '12',\n",
       "     'year': 1993,\n",
       "     'title': 'Amplification of the bacA gene confers bacitracin resistance to Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3784-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Cain BD', 'Norton PJ', 'Eubanks W', 'Nick HS', 'Allen CM.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8389741&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0292', 'name': 'LysE'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 49715,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49693,\n",
       "    'proteomes': 8311,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42607, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 35097},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45642,\n",
       "    'alignment:full': 8870,\n",
       "    'alignment:seed': 815},\n",
       "   'cross_references': {}}},\n",
       " 'PF00593': {'metadata': {'accession': 'PF00593',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000531',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TonB dependent receptor', 'short': 'TonB_dep_Rec'},\n",
       "   'description': ['<p>This model now only covers the conserved part of the barrel structure.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006673': {'PMID': 9886293,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Crystal structure of the outer membrane active transporter FepA from Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '56-63',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Buchanan SK',\n",
       "      'Smith BS',\n",
       "      'Venkatramani L',\n",
       "      'Xia D',\n",
       "      'Esser L',\n",
       "      'Palnitkar M',\n",
       "      'Chakraborty R',\n",
       "      'van der Helm D',\n",
       "      'Deisenhofer J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/4931'}},\n",
       "   'set_info': {'accession': 'CL0193', 'name': 'MBB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 470,\n",
       "    'interactions': 0,\n",
       "    'matches': 468186,\n",
       "    'pathways': 0,\n",
       "    'proteins': 467819,\n",
       "    'proteomes': 4555,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 411747, 'rosettafold': 0},\n",
       "    'structures': 123,\n",
       "    'taxa': 21749},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 453374,\n",
       "    'alignment:full': 86128,\n",
       "    'alignment:seed': 657},\n",
       "   'cross_references': {}}},\n",
       " 'PF00611': {'metadata': {'accession': 'PF00611',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001060',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fes/CIP4, and EFC/F-BAR homology domain', 'short': 'FCH'},\n",
       "   'description': ['<p>Alignment extended from [PMID:9210375]. Highly alpha-helical.  The cytosolic endocytic adaptor proteins in fungi carry this domain at the N-terminus; several of these have been referred to as muniscin proteins [PMID:19713939]. These N-terminal BAR, N-BAR, and EFC/F-BAR domains are found in proteins that regulate membrane trafficking events by inducing membrane tubulation. The domain dimerises into a curved structure that binds to liposomes and either senses or induces the curvature of the membrane bilayer to cause biophysical changes to the shape of the bilayer; it also thereby recruits other trafficking factors, such as the GTPase dynamin. Most EFC/F-BAR domain-family members localise to actin-rich structures [PMID:16938488].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001037': {'PMID': 9210375,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '7',\n",
       "     'year': 1997,\n",
       "     'title': 'A Cdc42 target protein with homology to the non-kinase domain of FER has a potential role in regulating the actin cytoskeleton.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '479-87',\n",
       "     'medline_journal': 'Curr Biol',\n",
       "     'ISO_journal': 'Curr. Biol.',\n",
       "     'authors': ['Aspenstrom P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0960-9822(06)00219-3'}},\n",
       "   'set_info': {'accession': 'CL0145', 'name': 'Golgi-transport'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 290,\n",
       "    'interactions': 0,\n",
       "    'matches': 34523,\n",
       "    'pathways': 0,\n",
       "    'proteins': 34520,\n",
       "    'proteomes': 1811,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24558, 'rosettafold': 0},\n",
       "    'structures': 32,\n",
       "    'taxa': 6436},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24712,\n",
       "    'alignment:full': 16135,\n",
       "    'alignment:seed': 520},\n",
       "   'cross_references': {}}},\n",
       " 'PF07907': {'metadata': {'accession': 'PF07907',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012507',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YibE/F-like protein', 'short': 'YibE_F'},\n",
       "   'description': ['<p>The sequences featured in this family are similar to two proteins expressed by Lactococcus lactis, YibE (Swiss:Q9CHC5) and YibF (Swiss:Q9CHC4). Most of the members of this family are annotated as being putative  membrane proteins, and in fact the sequences contain  a high proportion of hydrophobic residues.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 10,\n",
       "    'interactions': 0,\n",
       "    'matches': 10166,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10121,\n",
       "    'proteomes': 1911,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8940, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7604},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9706,\n",
       "    'alignment:full': 2336,\n",
       "    'alignment:seed': 268},\n",
       "   'cross_references': {}}},\n",
       " 'PF01798': {'metadata': {'accession': 'PF01798',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002687',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'snoRNA binding domain, fibrillarin', 'short': 'Nop'},\n",
       "   'description': [\"<p>This family consists of various Pre RNA processing ribonucleoproteins. The function of the aligned region is unknown however it may be a common RNA or snoRNA or Nop1p binding domain. Nop5p (Nop58p) Swiss:Q12499 from yeast is the protein component of a ribonucleoprotein required for pre-18s rRNA processing and is suggested to function with Nop1p in a snoRNA complex [PMID:9632712]. Nop56p Swiss:O00567 and Nop5p interact with Nop1p and are required for ribosome biogenesis [PMID:9372940]. Prp31p Swiss:p49704 is required for pre-mRNA splicing in S. cerevisiae [PMID:8604353]. Fibrillarin, or Nop, is the catalytic subunit responsible for the methyl transfer reaction of the site-specific 2'-O-methylation of ribosomal and spliceosomal RNA [PMID:20864039].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003017': {'PMID': 9632712,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '26',\n",
       "     'year': 1998,\n",
       "     'title': 'Nop5p is a small nucleolar ribonucleoprotein component required for pre-18 S rRNA processing in yeast.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '16453-63',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Wu P',\n",
       "      'Brockenbrough JS',\n",
       "      'Metcalfe AC',\n",
       "      'Chen S',\n",
       "      'Aris JP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.26.16453'},\n",
       "    'PUB00003706': {'PMID': 9372940,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '12',\n",
       "     'year': 1997,\n",
       "     'title': 'Nucleolar KKE/D repeat proteins Nop56p and Nop58p interact with Nop1p and are required for ribosome biogenesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7088-98',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Gautier T', 'Berges T', 'Tollervey D', 'Hurt E.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=9372940'},\n",
       "    'PUB00004465': {'PMID': 8604353,\n",
       "     'ISBN': None,\n",
       "     'volume': '24',\n",
       "     'issue': '6',\n",
       "     'year': 1996,\n",
       "     'title': 'The PRP31 gene encodes a novel protein required for pre-mRNA splicing in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1164-70',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Weidenhammer EM',\n",
       "      'Singh M',\n",
       "      'Ruiz-Noriega M',\n",
       "      'Woolford JL Jr.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/24.6.1164'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 200,\n",
       "    'interactions': 0,\n",
       "    'matches': 15289,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15017,\n",
       "    'proteomes': 2469,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12412, 'rosettafold': 0},\n",
       "    'structures': 67,\n",
       "    'taxa': 10211},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12346,\n",
       "    'alignment:full': 6362,\n",
       "    'alignment:seed': 304},\n",
       "   'cross_references': {}}},\n",
       " 'PF01970': {'metadata': {'accession': 'PF01970',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002823',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tripartite tricarboxylate transporter TctA family',\n",
       "    'short': 'TctA'},\n",
       "   'description': ['<p>This family, formerly known as DUF112, is a family of bacterial and archaeal tripartite tricarboxylate transporters of the extracytoplasmic solute binding receptor-dependent transporter group of families, distinct from the ABC and TRAP-T families [PMID:14499931]. TctA is part of the tripartite TctABC system which, as characterised in S. typhimurium [PMID:6141166], is a secondary carrier that depends for activity on the extracytoplasmic tricarboxylate-binding receptor TctC as well as two integral membrane proteins, TctA and TctB. complete three-component systems are found only in bacteria. TctA is a large transmembrane protein with up to 12 predicted membrane spanning regions in bacteria and up to 11 such in archaea, with the N-terminal within the cytoplasm. TctA is thought to be a permease, and in most other bacteria functions without TctB and TctC molecules [PMID:14499931].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 44,\n",
       "    'interactions': 0,\n",
       "    'matches': 37125,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37025,\n",
       "    'proteomes': 3108,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31514, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 14251},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33293,\n",
       "    'alignment:full': 8029,\n",
       "    'alignment:seed': 362},\n",
       "   'cross_references': {}}},\n",
       " 'PF13975': {'metadata': {'accession': 'PF13975',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'gag-polyprotein putative aspartyl protease',\n",
       "    'short': 'gag-asp_proteas'},\n",
       "   'description': ['<p>This family of putative aspartyl proteases is found pre-dominantly in retroviral proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0129', 'name': 'Peptidase_AA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 724,\n",
       "    'interactions': 0,\n",
       "    'matches': 24457,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24268,\n",
       "    'proteomes': 2890,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 18560, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 13293},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19944,\n",
       "    'alignment:full': 6741,\n",
       "    'alignment:seed': 261},\n",
       "   'cross_references': {}}},\n",
       " 'PF17764': {'metadata': {'accession': 'PF17764',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041222',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': \"3'DNA-binding domain (3'BD)\", 'short': 'PriA_3primeBD'},\n",
       "   'description': [\"<p>This domain represents the N-terminal DNA-binding domain found in the PriA protein. The 3'BD, which has been shown to bind the 3' end of the leading-strand arm of replication fork structures.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 92,\n",
       "    'interactions': 0,\n",
       "    'matches': 43835,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43821,\n",
       "    'proteomes': 8198,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37139, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 34221},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 40306,\n",
       "    'alignment:full': 7566,\n",
       "    'alignment:seed': 329},\n",
       "   'cross_references': {}}},\n",
       " 'PF08712': {'metadata': {'accession': 'PF08712',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014824',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Scaffold protein Nfu/NifU N terminal', 'short': 'Nfu_N'},\n",
       "   'description': ['<p>This domain is found at the N terminus of NifU and NifU related proteins, and in the human Nfu protein.  Both of these proteins are thought to be involved in the the assembly of iron-sulphur clusters [PMID:12886008][PMID:14993221].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00035424': {'PMID': 12886008,\n",
       "     'ISBN': None,\n",
       "     'volume': '100',\n",
       "     'issue': '17',\n",
       "     'year': 2003,\n",
       "     'title': 'Subcellular compartmentalization of human Nfu, an iron-sulfur cluster scaffold protein, and its ability to assemble a [4Fe-4S] cluster.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9762-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Tong WH', 'Jameson GN', 'Huynh BH', 'Rouault TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.1732541100'},\n",
       "    'PUB00035425': {'PMID': 14993221,\n",
       "     'ISBN': None,\n",
       "     'volume': '279',\n",
       "     'issue': '19',\n",
       "     'year': 2004,\n",
       "     'title': 'Iron-sulfur cluster assembly: NifU-directed activation of the nitrogenase Fe protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '19705-11',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Dos Santos PC',\n",
       "      'Smith AD',\n",
       "      'Frazzon J',\n",
       "      'Cash VL',\n",
       "      'Johnson MK',\n",
       "      'Dean DR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M400278200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 92,\n",
       "    'interactions': 0,\n",
       "    'matches': 16840,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15465,\n",
       "    'proteomes': 4655,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12953, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 17192},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14841,\n",
       "    'alignment:full': 4857,\n",
       "    'alignment:seed': 266},\n",
       "   'cross_references': {}}},\n",
       " 'PF02812': {'metadata': {'accession': 'PF02812',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006097',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glu/Leu/Phe/Val dehydrogenase, dimerisation domain',\n",
       "    'short': 'ELFV_dehydrog_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005264': {'PMID': 8591046,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '7',\n",
       "     'year': 1995,\n",
       "     'title': 'A role for quaternary structure in the substrate specificity of leucine dehydrogenase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '693-705',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Baker PJ',\n",
       "      'Turnbull AP',\n",
       "      'Sedelnikova SE',\n",
       "      'Stillman TJ',\n",
       "      'Rice DW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(01)00204-0'}},\n",
       "   'set_info': {'accession': 'CL0603', 'name': 'AA_dh_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 164,\n",
       "    'interactions': 0,\n",
       "    'matches': 68767,\n",
       "    'pathways': 0,\n",
       "    'proteins': 68535,\n",
       "    'proteomes': 9182,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 58320, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 38510},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 61822,\n",
       "    'alignment:full': 14717,\n",
       "    'alignment:seed': 343},\n",
       "   'cross_references': {}}},\n",
       " 'PF16220': {'metadata': {'accession': 'PF16220',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032623',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4880)',\n",
       "    'short': 'DUF4880'},\n",
       "   'description': ['<p>This domain can be found on the N-terminal of uncharacterized  proteins from various Rhodopseudomonas and Pseudomonas species, often, but not always followed by the ron siderophore sensor  protein family (FecR, PF04773). The function of this domain is  unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 25002,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25002,\n",
       "    'proteomes': 974,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22874, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 5446},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27771,\n",
       "    'alignment:full': 3102,\n",
       "    'alignment:seed': 253},\n",
       "   'cross_references': {}}},\n",
       " 'PF01116': {'metadata': {'accession': 'PF01116',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000771',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fructose-bisphosphate aldolase class-II',\n",
       "    'short': 'F_bP_aldolase'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003935': {'PMID': 8836102,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '10',\n",
       "     'year': 1996,\n",
       "     'title': 'Novel active site in Escherichia coli fructose 1,6-bisphosphate aldolase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '856-62',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Blom NS', 'Tetreault S', 'Coulombe R', 'Sygusch J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb1096-856'}},\n",
       "   'set_info': {'accession': 'CL0036', 'name': 'TIM_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 159,\n",
       "    'interactions': 0,\n",
       "    'matches': 56681,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56464,\n",
       "    'proteomes': 8348,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 49019, 'rosettafold': 0},\n",
       "    'structures': 59,\n",
       "    'taxa': 35455},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 53400,\n",
       "    'alignment:full': 12245,\n",
       "    'alignment:seed': 688},\n",
       "   'cross_references': {}}},\n",
       " 'PF11799': {'metadata': {'accession': 'PF11799',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017961',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'impB/mucB/samB family C-terminal domain',\n",
       "    'short': 'IMS_C'},\n",
       "   'description': ['<p>These proteins are involved in UV protection (Swiss).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 367,\n",
       "    'interactions': 0,\n",
       "    'matches': 81377,\n",
       "    'pathways': 0,\n",
       "    'proteins': 81367,\n",
       "    'proteomes': 10125,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 70026, 'rosettafold': 0},\n",
       "    'structures': 468,\n",
       "    'taxa': 40648},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 76275,\n",
       "    'alignment:full': 18492,\n",
       "    'alignment:seed': 694},\n",
       "   'cross_references': {}}},\n",
       " 'PF05164': {'metadata': {'accession': 'PF05164',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007838',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cell division protein ZapA', 'short': 'ZapA'},\n",
       "   'description': ['<p>ZapA is a cell division protein which interacts with FtsZ.  FtsZ is part of a mid-cell cytokinetic structure termed the Z-ring that recruits a hierarchy of fission related proteins early in the bacterial cell cycle.  The interaction of FtsZ with ZapA drives its polymerisation and promotes FtsZ filament bundling thereby contributing to the spatio-temporal tuning of the Z-ring [PMID:15288790][PMID:12368265].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00021077': {'PMID': 15288790,\n",
       "     'ISBN': None,\n",
       "     'volume': '341',\n",
       "     'issue': '3',\n",
       "     'year': 2004,\n",
       "     'title': 'The crystal structure of ZapA and its modulation of FtsZ polymerisation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '839-52',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Low HH', 'Moncrieffe MC', 'Lowe J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.jmb.2004.05.031'},\n",
       "    'PUB00021078': {'PMID': 12368265,\n",
       "     'ISBN': None,\n",
       "     'volume': '16',\n",
       "     'issue': '19',\n",
       "     'year': 2002,\n",
       "     'title': 'A widely conserved bacterial cell division protein that promotes assembly of the tubulin-like protein FtsZ.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2544-56',\n",
       "     'medline_journal': 'Genes Dev',\n",
       "     'ISO_journal': 'Genes Dev.',\n",
       "     'authors': ['Gueiros-Filho FJ', 'Losick R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1101/gad.1014102'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 26865,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26865,\n",
       "    'proteomes': 5494,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23158, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 23067},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24851,\n",
       "    'alignment:full': 5184,\n",
       "    'alignment:seed': 372},\n",
       "   'cross_references': {}}},\n",
       " 'PF02503': {'metadata': {'accession': 'PF02503',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024953',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Polyphosphate kinase middle domain',\n",
       "    'short': 'PP_kinase'},\n",
       "   'description': ['<p>Polyphosphate kinase (Ppk) catalyses the formation of polyphosphate from ATP, with chain lengths of up to a thousand or more orthophosphate molecules.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020166': {'PMID': 9501429,\n",
       "     'ISBN': None,\n",
       "     'volume': '64',\n",
       "     'issue': '3',\n",
       "     'year': 1998,\n",
       "     'title': 'Transcription of ppk from Acinetobacter sp. strain ADP1, encoding a putative polyphosphate kinase, is induced by phosphate starvation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '896-901',\n",
       "     'medline_journal': 'Appl Environ Microbiol',\n",
       "     'ISO_journal': 'Appl. Environ. Microbiol.',\n",
       "     'authors': ['Geissdorfer W', 'Ratajczak A', 'Hillen W.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9501429&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 60,\n",
       "    'interactions': 0,\n",
       "    'matches': 36556,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36543,\n",
       "    'proteomes': 6774,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31845, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 28005},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33907,\n",
       "    'alignment:full': 6639,\n",
       "    'alignment:seed': 564},\n",
       "   'cross_references': {}}},\n",
       " 'PF01725': {'metadata': {'accession': 'PF01725',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002637',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ham1 family', 'short': 'Ham1p_like'},\n",
       "   'description': ['<p>This family consists of the HAM1 protein Swiss:P47119 and hypothetical archaeal bacterial and C. elegans proteins. HAM1 controls 6-N-hydroxylaminopurine (HAP) sensitivity and mutagenesis in S. cerevisiae Swiss:P47119 [PMID:8789257]. The HAM1 protein protects the cell from HAP, either on the level of deoxynucleoside triphosphate or the DNA level by a yet unidentified set of reactions [PMID:8789257].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005658': {'PMID': 8789257,\n",
       "     'ISBN': None,\n",
       "     'volume': '12',\n",
       "     'issue': '1',\n",
       "     'year': 1996,\n",
       "     'title': 'HAM1, the gene controlling 6-N-hydroxylaminopurine sensitivity and mutagenesis in the yeast Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17-29',\n",
       "     'medline_journal': 'Yeast',\n",
       "     'ISO_journal': 'Yeast',\n",
       "     'authors': ['Noskov VN',\n",
       "      'Staak K',\n",
       "      'Shcherbakova PV',\n",
       "      'Kozmin SG',\n",
       "      'Negishi K',\n",
       "      'Ono BC',\n",
       "      'Hayatsu H',\n",
       "      'Pavlov YI.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/(SICI)1097-0061(199601)12:1<17::AID-YEA875>3.3.CO;2-9'}},\n",
       "   'set_info': {'accession': 'CL0269', 'name': 'Maf'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 203,\n",
       "    'interactions': 0,\n",
       "    'matches': 53056,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52935,\n",
       "    'proteomes': 10774,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 44872, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 44432},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47551,\n",
       "    'alignment:full': 10408,\n",
       "    'alignment:seed': 1061},\n",
       "   'cross_references': {}}},\n",
       " 'PF01218': {'metadata': {'accession': 'PF01218',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001260',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Coproporphyrinogen III oxidase',\n",
       "    'short': 'Coprogen_oxidas'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 63,\n",
       "    'interactions': 0,\n",
       "    'matches': 23111,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22904,\n",
       "    'proteomes': 5669,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19463, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 23380},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21227,\n",
       "    'alignment:full': 5522,\n",
       "    'alignment:seed': 239},\n",
       "   'cross_references': {}}},\n",
       " 'PF04002': {'metadata': {'accession': 'PF04002',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025657',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RadC-like JAB domain', 'short': 'RadC'},\n",
       "   'description': [\"<p>A family of proteins present widely across the bacteria. This family was named initially with reference to the E. coli radC102 mutation which suggested that RadC was involved in repair of DNA lesions [PMID:10224240]. However the relevant mutation has subsequently been shown to be in recG, where radC is in fact an allele of recG [PMID:11053371]. In addition, a personal communication from Claverys, J-P, et al, indicates a total failure of all attempts to characterise a radiation-related function for RadC in Streptococcus pneumoniae, suggesting that it is not involved in repair of DNA lesions, in recombination during transformation, in gene conversion, nor in mismatch repair. Computational analysis, however, provides a possible function. The RadC-like family belong to the JAB superfamily  of metalloproteins [PMID:21890906]. The domain shows fusions to an N-terminal  Helix-hairpin-Helix (HhH) domain in most instances. Other domain combinations include fusions to the anti-restriction module ArdC, the DinG/RAD3-like superfamily II helicases and the DNAG-like primase. In some bacteria, closely related DinG/Rad3- like superfamily II helicases are fused to a 3'-5' exonuclease in the  same position as the RadC-like JAB domain. These conserved domain  associations lead to the hypothesis that the RadC-like JAB domains  might function as a nuclease [PMID:21890906].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009802': {'PMID': 10224240,\n",
       "     'ISBN': None,\n",
       "     'volume': '152',\n",
       "     'issue': '1',\n",
       "     'year': 1999,\n",
       "     'title': 'Tandem repeat recombination induced by replication fork defects in Escherichia coli requires a novel factor, RadC.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5-13',\n",
       "     'medline_journal': 'Genetics',\n",
       "     'ISO_journal': 'Genetics',\n",
       "     'authors': ['Saveson CJ', 'Lovett ST.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=10224240&action=stream&blobtype=pdf'},\n",
       "    'PUB00044698': {'PMID': 11053371,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '22',\n",
       "     'year': 2000,\n",
       "     'title': 'radC102 of Escherichia coli is an allele of recG.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6287-91',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Lombardo MJ', 'Rosenberg SM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.22.6287-6291.2000'}},\n",
       "   'set_info': {'accession': 'CL0366', 'name': 'JAB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 153,\n",
       "    'interactions': 0,\n",
       "    'matches': 47300,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47277,\n",
       "    'proteomes': 6364,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 40593, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 28036},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44396,\n",
       "    'alignment:full': 8219,\n",
       "    'alignment:seed': 790},\n",
       "   'cross_references': {}}},\n",
       " 'PF02244': {'metadata': {'accession': 'PF02244',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003146',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Carboxypeptidase activation peptide',\n",
       "    'short': 'Propep_M14'},\n",
       "   'description': ['<p>Carboxypeptidases are found in abundance in pancreatic secretions.  The pro-segment moiety (activation peptide) accounts for up to a  quarter of the total length of the peptidase, and is responsible  for modulation of folding and activity of the pro-enzyme.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020189': {'PMID': 9524066,\n",
       "     'ISBN': None,\n",
       "     'volume': '379',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Comparative analysis of the sequences and three-dimensional models of human procarboxypeptidases A1, A2 and B.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '149-55',\n",
       "     'medline_journal': 'Biol Chem',\n",
       "     'ISO_journal': 'Biol. Chem.',\n",
       "     'authors': ['Aloy P',\n",
       "      'Catasus L',\n",
       "      'Villegas V',\n",
       "      'Reverter D',\n",
       "      'Vendrell J',\n",
       "      'Aviles FX.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': {'accession': 'CL0570', 'name': 'PPP-I'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 142,\n",
       "    'interactions': 0,\n",
       "    'matches': 10056,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9829,\n",
       "    'proteomes': 1070,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 7625, 'rosettafold': 0},\n",
       "    'structures': 23,\n",
       "    'taxa': 4410},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7632,\n",
       "    'alignment:full': 3953,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF04413': {'metadata': {'accession': 'PF04413',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007507',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '3-Deoxy-D-manno-octulosonic-acid transferase (kdotransferase)',\n",
       "    'short': 'Glycos_transf_N'},\n",
       "   'description': ['<p>Members of this family transfer activated sugars to a variety of substrates, including glycogen, fructose-6-phosphate and lipopolysaccharides.  Members of the family transfer UDP, ADP, GDP or CMP linked sugars.  The Glycos_transf_N region is flanked at the N-terminus by a signal peptide and at the C-terminus by Glycos_transf_1 (Pfam:PF00534).  The eukaryotic  glycogen synthases may be distant members of this bacterial  family [PMID:10952982].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009991': {'PMID': 10952982,\n",
       "     'ISBN': None,\n",
       "     'volume': '275',\n",
       "     'issue': '45',\n",
       "     'year': 2000,\n",
       "     'title': '3-Deoxy-D-manno-oct-2-ulosonic acid (Kdo) transferase (WaaA) and kdo kinase (KdkA) of Haemophilus influenzae are both required to complement a waaA knockout mutation of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '34954-62',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Brabetz W', 'Muller-Loennies S', 'Brade H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M005204200'}},\n",
       "   'set_info': {'accession': 'CL0113', 'name': 'GT-B'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 52,\n",
       "    'interactions': 0,\n",
       "    'matches': 27538,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27528,\n",
       "    'proteomes': 4393,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 23298, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 20214},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 25160,\n",
       "    'alignment:full': 4669,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF06686': {'metadata': {'accession': 'PF06686',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025664',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Stage III sporulation protein AC/AD protein family',\n",
       "    'short': 'SpoIIIAC'},\n",
       "   'description': ['<p>This family consists of several bacterial stage III sporulation protein AC (SpoIIIAC) and SpoIIIAD sequences. The exact function of this family is unknown.  SpoIIIAD is the an uncharacterised protein which is part of the spoIIIA operon that acts at sporulation stage III as part of a cascade of events leading to endospore formation. The operon is regulated by sigmaG [PMID:8969508].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020362': {'PMID': 8969508,\n",
       "     'ISBN': None,\n",
       "     'volume': '142 ( Pt 11)',\n",
       "     'issue': None,\n",
       "     'year': 1996,\n",
       "     'title': 'Systematic sequencing of the 283 kb 210 degrees-232 degrees region of the Bacillus subtilis genome containing the skin element and many sporulation genes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3103-11',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Mizuno M',\n",
       "      'Masuda S',\n",
       "      'Takemaru K',\n",
       "      'Hosono S',\n",
       "      'Sato T',\n",
       "      'Takeuchi M',\n",
       "      'Kobayashi Y.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4,\n",
       "    'interactions': 0,\n",
       "    'matches': 10677,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7500,\n",
       "    'proteomes': 1127,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6417, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 3686},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9839,\n",
       "    'alignment:full': 3023,\n",
       "    'alignment:seed': 408},\n",
       "   'cross_references': {}}},\n",
       " 'PF17042': {'metadata': {'accession': 'PF17042',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031475',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nucleotide-binding C-terminal domain', 'short': 'NBD_C'},\n",
       "   'description': [\"<p>This is the C-terminal domain found in proteins in a range of Proteobacteria as well as the Gram-positive Oceanobacillus iheyensis. Structural analysis of the whole protein indicates the N- and C-termini act together to produce a surface into which a threonate-ADP complex is bound, demonstrating that a sugar binding site is on the N-terminal domain, and a nucleotide binding site is in the C-terminal domain [PMID:27402745]. There is a critical motif, DDXTG, at approximately residues 22-25. Proteins containing this domain have been predicted as kinases. Some members are associated with PdxA2 by physical clustering and gene fusion with PdxA2. Some members that are fused with PdxA2 have been shown to be involved in L-4-hydroxythreonine (4HT) phosphorylation, part of the alternative pathway to make PLP (pyridoxal 5'-phosphate) out of a toxic metabolite, 4HT. However, 4HT phosphorylation might not be the main function of this group of proteins. Moreover, some members that are not associated with pdxA2, and even one that is associated with pdxA2, have lost 4HT kinase activity [PMID:27294475]. Functional analysis demonstrate that family members include D-Threonate kinases (DtnK), D-Erythronate kinases (DenK) and 3-Oxo-tetronate kinases (OtnK) [PMID:27402745].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 75,\n",
       "    'interactions': 0,\n",
       "    'matches': 17381,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17364,\n",
       "    'proteomes': 2740,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15169, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 12656},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16952,\n",
       "    'alignment:full': 3772,\n",
       "    'alignment:seed': 246},\n",
       "   'cross_references': {}}},\n",
       " 'PF10589': {'metadata': {'accession': 'PF10589',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019575',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'NADH-ubiquinone oxidoreductase-F iron-sulfur binding region',\n",
       "    'short': 'NADH_4Fe-4S'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 258,\n",
       "    'interactions': 0,\n",
       "    'matches': 56583,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56538,\n",
       "    'proteomes': 7870,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 47074, 'rosettafold': 0},\n",
       "    'structures': 240,\n",
       "    'taxa': 32690},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49262,\n",
       "    'alignment:full': 10991,\n",
       "    'alignment:seed': 607},\n",
       "   'cross_references': {}}},\n",
       " 'PF01369': {'metadata': {'accession': 'PF01369',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000904',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sec7 domain', 'short': 'Sec7'},\n",
       "   'description': ['<p>The Sec7 domain is a guanine-nucleotide-exchange-factor (GEF) for the Pfam:PF00025 family [PMID:8945478].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004247': {'PMID': 8945478,\n",
       "     'ISBN': None,\n",
       "     'volume': '384',\n",
       "     'issue': '6608',\n",
       "     'year': 1996,\n",
       "     'title': 'A human exchange factor for ARF contains Sec7- and pleckstrin-homology domains.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '481-4',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Chardin P',\n",
       "      'Paris S',\n",
       "      'Antonny B',\n",
       "      'Robineau S',\n",
       "      'Beraud-Dufour S',\n",
       "      'Jackson CL',\n",
       "      'Chabre M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/384481a0'},\n",
       "    'PUB00004276': {'PMID': 9510256,\n",
       "     'ISBN': None,\n",
       "     'volume': '392',\n",
       "     'issue': '6671',\n",
       "     'year': 1998,\n",
       "     'title': 'Structure of the Sec7 domain of the Arf exchange factor ARNO.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '101-5',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Cherfils J',\n",
       "      'Menetrey J',\n",
       "      'Mathieu M',\n",
       "      'Le Bras G',\n",
       "      'Robineau S',\n",
       "      'Beraud-Dufour S',\n",
       "      'Antonny B',\n",
       "      'Chardin P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/32210'},\n",
       "    'PUB00005838': {'PMID': 9868368,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '12',\n",
       "     'year': 1998,\n",
       "     'title': 'Triple association of CDC25-, Dbl- and Sec7-related domains in mammalian guanine-nucleotide-exchange factors.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '472-3',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Abergel C', 'Chavrier P', 'Claverie JM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(98)01283-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 416,\n",
       "    'interactions': 0,\n",
       "    'matches': 35597,\n",
       "    'pathways': 0,\n",
       "    'proteins': 35263,\n",
       "    'proteomes': 2188,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15055, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 8237},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26175,\n",
       "    'alignment:full': 16719,\n",
       "    'alignment:seed': 370},\n",
       "   'cross_references': {}}},\n",
       " 'PF00200': {'metadata': {'accession': 'PF00200',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001762',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Disintegrin', 'short': 'Disintegrin'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 293,\n",
       "    'interactions': 0,\n",
       "    'matches': 26907,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26836,\n",
       "    'proteomes': 1618,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18214, 'rosettafold': 0},\n",
       "    'structures': 45,\n",
       "    'taxa': 6055},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19035,\n",
       "    'alignment:full': 11098,\n",
       "    'alignment:seed': 512},\n",
       "   'cross_references': {}}},\n",
       " 'PF06415': {'metadata': {'accession': 'PF06415',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011258',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'BPG-independent PGAM N-terminus (iPGM_N)',\n",
       "    'short': 'iPGM_N'},\n",
       "   'description': ['<p>This family represents the N-terminal region of the 2,3-bisphosphoglycerate-independent phosphoglycerate mutase (or phosphoglyceromutase or BPG-independent PGAM) protein (EC:5.4.2.1). The family is found in conjunction with Pfam:PF01676 (located in the C-terminal region of the protein).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014365': {'PMID': 10764795,\n",
       "     'ISBN': None,\n",
       "     'volume': '275',\n",
       "     'issue': '30',\n",
       "     'year': 2000,\n",
       "     'title': 'Mechanism of catalysis of the cofactor-independent phosphoglycerate mutase from Bacillus stearothermophilus. Crystal structure of the complex with 2-phosphoglycerate.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '23146-53',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Jedrzejas MJ', 'Chander M', 'Setlow P', 'Krishnasamy G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M002544200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 88,\n",
       "    'interactions': 0,\n",
       "    'matches': 30977,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30767,\n",
       "    'proteomes': 6004,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26201, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 26683},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28051,\n",
       "    'alignment:full': 6007,\n",
       "    'alignment:seed': 568},\n",
       "   'cross_references': {}}},\n",
       " 'PF00572': {'metadata': {'accession': 'PF00572',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005822',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L13', 'short': 'Ribosomal_L13'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 145,\n",
       "    'interactions': 0,\n",
       "    'matches': 53905,\n",
       "    'pathways': 0,\n",
       "    'proteins': 53812,\n",
       "    'proteomes': 10950,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45931, 'rosettafold': 0},\n",
       "    'structures': 1139,\n",
       "    'taxa': 47865},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47460,\n",
       "    'alignment:full': 12444,\n",
       "    'alignment:seed': 53},\n",
       "   'cross_references': {}}},\n",
       " 'PF05635': {'metadata': {'accession': 'PF05635',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012657',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '23S rRNA-intervening sequence protein',\n",
       "    'short': '23S_rRNA_IVP'},\n",
       "   'description': ['<p>This family consists of bacterial proteins encoded within an intervening sequence present within some 23S rRNA genes [1-3]. It folds into an anti-parallel four-helix bundle and forms homopentamers [PMID:16948161].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011556': {'PMID': 8341711,\n",
       "     'ISBN': None,\n",
       "     'volume': '90',\n",
       "     'issue': '14',\n",
       "     'year': 1993,\n",
       "     'title': 'Intervening sequence with conserved open reading frame in eubacterial 23S rRNA genes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6864-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Ralph D', 'McClelland M.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8341711&action=stream&blobtype=pdf'},\n",
       "    'PUB00020253': {'PMID': 7751314,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '10',\n",
       "     'year': 1995,\n",
       "     'title': 'Characterization of the 23S and 5S rRNA genes of Coxiella burnetii and identification of an intervening sequence within the 23S rRNA gene.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2946-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Afseth G', 'Mo YY', 'Mallavia LP.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/177/10/2946'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 80,\n",
       "    'interactions': 0,\n",
       "    'matches': 39521,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39471,\n",
       "    'proteomes': 1743,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 31828, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 11135},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31938,\n",
       "    'alignment:full': 5143,\n",
       "    'alignment:seed': 346},\n",
       "   'cross_references': {}}},\n",
       " 'PF05681': {'metadata': {'accession': 'PF05681',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004646',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fumarate hydratase (Fumerase)', 'short': 'Fumerase'},\n",
       "   'description': ['<p>This family consists of several bacterial fumarate hydratase proteins FumA and FumB. Fumarase, or fumarate hydratase (EC 4.2.1.2), is a component of the citric acid cycle. In facultative anaerobes such as Escherichia coli, fumarase also engages in the reductive pathway from oxaloacetate to succinate during anaerobic growth. Three fumarases, FumA, FumB, and FumC, have been reported in E. coli. fumA and fumB genes are homologous and encode products of identical sizes which form thermolabile dimers of Mr 120,000. FumA and FumB are class I enzymes and are members of the iron-dependent hydrolases, which include aconitase and malate hydratase. The active FumA contains a 4Fe-4S centre, and it can be inactivated upon oxidation to give a 3Fe-4S centre [PMID:11133938].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011340': {'PMID': 11133938,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '2',\n",
       "     'year': 2001,\n",
       "     'title': 'Oxygen- and growth rate-dependent regulation of Escherichia coli fumarase (FumA, FumB, and FumC) activity.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '461-7',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Tseng CP', 'Yu CC', 'Lin HH', 'Chang CY', 'Kuo JT.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.2.461-467.2001'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 32,\n",
       "    'interactions': 0,\n",
       "    'matches': 30749,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30722,\n",
       "    'proteomes': 5023,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26899, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 22103},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29169,\n",
       "    'alignment:full': 5163,\n",
       "    'alignment:seed': 346},\n",
       "   'cross_references': {}}},\n",
       " 'PF00285': {'metadata': {'accession': 'PF00285',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002020',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Citrate synthase, C-terminal domain',\n",
       "    'short': 'Citrate_synt'},\n",
       "   'description': ['<p>This is the long, C-terminal part of the enzyme.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 230,\n",
       "    'interactions': 0,\n",
       "    'matches': 90573,\n",
       "    'pathways': 0,\n",
       "    'proteins': 89209,\n",
       "    'proteomes': 9901,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 77036, 'rosettafold': 0},\n",
       "    'structures': 98,\n",
       "    'taxa': 42440},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 82887,\n",
       "    'alignment:full': 19007,\n",
       "    'alignment:seed': 719},\n",
       "   'cross_references': {}}},\n",
       " 'PF13445': {'metadata': {'accession': 'PF13445',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027370',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RING-type zinc-finger', 'short': 'zf-RING_UBOX'},\n",
       "   'description': ['<p>This zinc-finger is a typical RING-type of plant ubiquitin ligases [PMID:15644464].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0229', 'name': 'RING'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1365,\n",
       "    'interactions': 0,\n",
       "    'matches': 51545,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51224,\n",
       "    'proteomes': 2158,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37084, 'rosettafold': 1},\n",
       "    'structures': 15,\n",
       "    'taxa': 8131},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37636,\n",
       "    'alignment:seed': 484},\n",
       "   'cross_references': {}}},\n",
       " 'PF02391': {'metadata': {'accession': 'PF02391',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003448',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MoaE protein', 'short': 'MoaE'},\n",
       "   'description': ['<p>This family contains the MoaE protein that is involved in biosynthesis of molybdopterin [PMID:8514782]. Molybdopterin, the universal component of the pterin molybdenum cofactors, contains a dithiolene group serving to bind Mo. Addition of the dithiolene sulfurs to a molybdopterin precursor requires the activity of the converting factor. Converting factor contains the MoaE and MoaD proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008129': {'PMID': 8514782,\n",
       "     'ISBN': None,\n",
       "     'volume': '268',\n",
       "     'issue': '18',\n",
       "     'year': 1993,\n",
       "     'title': 'The biosynthesis of molybdopterin in Escherichia coli. Purification and characterization of the converting factor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13499-505',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Pitterle DM', 'Rajagopalan KV.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/268/18/13499'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 108,\n",
       "    'interactions': 0,\n",
       "    'matches': 35508,\n",
       "    'pathways': 0,\n",
       "    'proteins': 35474,\n",
       "    'proteomes': 7894,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 30611, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 32484},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 32672,\n",
       "    'alignment:full': 7161,\n",
       "    'alignment:seed': 479},\n",
       "   'cross_references': {}}},\n",
       " 'PF02482': {'metadata': {'accession': 'PF02482',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003489',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sigma 54 modulation protein / S30EA ribosomal protein',\n",
       "    'short': 'Ribosomal_S30AE'},\n",
       "   'description': ['<p>This Pfam family contains the sigma-54 modulation protein family and the S30AE family of ribosomal proteins which includes the light- repressed protein (lrtA) (Swiss:P47908) [PMID:8063707].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009462': {'PMID': 8063707,\n",
       "     'ISBN': None,\n",
       "     'volume': '269',\n",
       "     'issue': '33',\n",
       "     'year': 1994,\n",
       "     'title': 'A light-repressed transcript found in Synechococcus PCC 7002 is similar to a chloroplast-specific small subunit ribosomal protein and to a transcription modulator protein associated with sigma 54.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '20905-12',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Tan X', 'Varughese M', 'Widger WR.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/content/abstract/269/33/20905'}},\n",
       "   'set_info': {'accession': 'CL0219', 'name': 'RNase_H'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 54,\n",
       "    'interactions': 0,\n",
       "    'matches': 49279,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49276,\n",
       "    'proteomes': 8601,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42365, 'rosettafold': 0},\n",
       "    'structures': 47,\n",
       "    'taxa': 36197},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44888,\n",
       "    'alignment:full': 9683,\n",
       "    'alignment:seed': 756},\n",
       "   'cross_references': {}}},\n",
       " 'PF00719': {'metadata': {'accession': 'PF00719',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008162',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Inorganic pyrophosphatase', 'short': 'Pyrophosphatase'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 168,\n",
       "    'interactions': 0,\n",
       "    'matches': 39476,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39260,\n",
       "    'proteomes': 8344,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33571, 'rosettafold': 0},\n",
       "    'structures': 102,\n",
       "    'taxa': 34330},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35238,\n",
       "    'alignment:full': 10593,\n",
       "    'alignment:seed': 498},\n",
       "   'cross_references': {}}},\n",
       " 'PF02885': {'metadata': {'accession': 'PF02885',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017459',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl transferase family, helical bundle domain',\n",
       "    'short': 'Glycos_trans_3N'},\n",
       "   'description': ['<p>This family includes anthranilate phosphoribosyltransferase (TrpD), thymidine phosphorylase.  All these proteins can transfer a phosphorylated ribose substrate.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002573': {'PMID': 2199449,\n",
       "     'ISBN': None,\n",
       "     'volume': '265',\n",
       "     'issue': '23',\n",
       "     'year': 1990,\n",
       "     'title': 'Three-dimensional structure of thymidine phosphorylase from Escherichia coli at 2.8 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14016-22',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Walter MR',\n",
       "      'Cook WJ',\n",
       "      'Cole LB',\n",
       "      'Short SA',\n",
       "      'Koszalka GW',\n",
       "      'Krenitsky TA',\n",
       "      'Ealick SE.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/265/23/14016.pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 121,\n",
       "    'interactions': 0,\n",
       "    'matches': 72731,\n",
       "    'pathways': 0,\n",
       "    'proteins': 72725,\n",
       "    'proteomes': 9614,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 63006, 'rosettafold': 0},\n",
       "    'structures': 88,\n",
       "    'taxa': 39533},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 67627,\n",
       "    'alignment:full': 14262,\n",
       "    'alignment:seed': 1191},\n",
       "   'cross_references': {}}},\n",
       " 'PF02383': {'metadata': {'accession': 'PF02383',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002013',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SacI homology domain', 'short': 'Syja_N'},\n",
       "   'description': [\"<p>This Pfam family represents a protein domain which shows homology to the yeast protein SacI Swiss:P32368. The SacI homology domain is most notably found at the amino terminal of the inositol  5'-phosphatase synaptojanin.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020392': {'PMID': 9388224,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '49',\n",
       "     'year': 1997,\n",
       "     'title': 'Synaptojanin 2, a novel synaptojanin isoform with a distinct targeting domain and expression pattern.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '30817-21',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Nemoto Y', 'Arribas M', 'Haffner C', 'DeCamilli P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.49.30817'}},\n",
       "   'set_info': {'accession': 'CL0031', 'name': 'Phosphatase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 276,\n",
       "    'interactions': 0,\n",
       "    'matches': 21039,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20568,\n",
       "    'proteomes': 2177,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13321, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 7939},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16558,\n",
       "    'alignment:full': 10414,\n",
       "    'alignment:seed': 450},\n",
       "   'cross_references': {}}},\n",
       " 'PF01871': {'metadata': {'accession': 'PF01871',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002733',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AMMECR1', 'short': 'AMMECR1'},\n",
       "   'description': ['<p>This family consists of several AMMECR1 as well as several uncharacterised proteins. The contiguous gene deletion syndrome AMME is characterised by Alport syndrome, midface hypoplasia, mental retardation and elliptocytosis and is caused by a deletion in Xq22.3, comprising several genes including COL4A5, FACL4 and AMMECR1 [PMID:10828604]. This family contains sequences from several eukaryotic species as well as archaebacteria and it has been suggested that the AMMECR1 protein may have a basic cellular function, potentially in either the transcription, replication, repair or translation machinery [PMID:10049589].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006433': {'PMID': 10049589,\n",
       "     'ISBN': None,\n",
       "     'volume': '55',\n",
       "     'issue': '3',\n",
       "     'year': 1999,\n",
       "     'title': 'Identification and characterization of a highly conserved protein absent in the Alport syndrome (A), mental retardation (M), midface hypoplasia (M), and elliptocytosis (E) contiguous gene deletion syndrome (AMME).',\n",
       "     'URL': None,\n",
       "     'raw_pages': '335-40',\n",
       "     'medline_journal': 'Genomics',\n",
       "     'ISO_journal': 'Genomics',\n",
       "     'authors': ['Vitelli F',\n",
       "      'Piccini M',\n",
       "      'Caroli F',\n",
       "      'Franco B',\n",
       "      'Malandrini A',\n",
       "      'Pober B',\n",
       "      'Jonsson J',\n",
       "      'Sorrentino V',\n",
       "      'Renieri A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/geno.1998.5666'},\n",
       "    'PUB00015590': {'PMID': 10828604,\n",
       "     'ISBN': None,\n",
       "     'volume': '88',\n",
       "     'issue': '3-4',\n",
       "     'year': 2000,\n",
       "     'title': 'Identification and characterization of mouse orthologs of the AMMECR1 and FACL4 genes deleted in AMME syndrome: orthology of Xq22.3 and MmuXF1-F3.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '259-63',\n",
       "     'medline_journal': 'Cytogenet Cell Genet',\n",
       "     'ISO_journal': 'Cytogenet. Cell Genet.',\n",
       "     'authors': ['Vitelli F',\n",
       "      'Meloni I',\n",
       "      'Fineschi S',\n",
       "      'Favara F',\n",
       "      'Tiziana Storlazzi C',\n",
       "      'Rocchi M',\n",
       "      'Renieri A.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 77,\n",
       "    'interactions': 0,\n",
       "    'matches': 15761,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15711,\n",
       "    'proteomes': 3474,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12996, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 15653},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12650,\n",
       "    'alignment:full': 3391,\n",
       "    'alignment:seed': 330},\n",
       "   'cross_references': {}}},\n",
       " 'PF02590': {'metadata': {'accession': 'PF02590',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003742',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Predicted SPOUT methyltransferase',\n",
       "    'short': 'SPOUT_MTase'},\n",
       "   'description': ['<p>This family of proteins are predicted to be SPOUT methyltransferases [PMID:17338813].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044735': {'PMID': 17338813,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': None,\n",
       "     'year': 2007,\n",
       "     'title': 'Structural and evolutionary bioinformatics of the SPOUT superfamily of methyltransferases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '73',\n",
       "     'medline_journal': 'BMC Bioinformatics',\n",
       "     'ISO_journal': 'BMC Bioinformatics',\n",
       "     'authors': ['Tkaczuk KL', 'Dunin-Horkawicz S', 'Purta E', 'Bujnicki JM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2105-8-73'}},\n",
       "   'set_info': {'accession': 'CL0098', 'name': 'SPOUT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 33,\n",
       "    'interactions': 0,\n",
       "    'matches': 31131,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31119,\n",
       "    'proteomes': 6404,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26843, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 26915},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28668,\n",
       "    'alignment:full': 6122,\n",
       "    'alignment:seed': 559},\n",
       "   'cross_references': {}}},\n",
       " 'PF05336': {'metadata': {'accession': 'PF05336',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008000',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'L-rhamnose mutarotase', 'short': 'rhaM'},\n",
       "   'description': ['<p>This family contains L-rhamnose mutarotase which is a glycosyl  hydrolase that converts the monosaccharide L-rhamnopyranose from  the alpha to the beta stereoisomer.  In Escherichia coli this  enzyme is the product of the rhaM gene (also known as yiiL).  The tertiary structure has been solved, in complex with L-rhamnose,  and the catalytic mechanism determined.  His22 is the proton donor.  The enzyme naturally exists as a dimer.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0032', 'name': 'Dim_A_B_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 47,\n",
       "    'interactions': 0,\n",
       "    'matches': 15870,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15769,\n",
       "    'proteomes': 3014,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13978, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 12473},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15204,\n",
       "    'alignment:full': 3798,\n",
       "    'alignment:seed': 245},\n",
       "   'cross_references': {}}},\n",
       " 'PF12396': {'metadata': {'accession': 'PF12396',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022124',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF3659)',\n",
       "    'short': 'DUF3659'},\n",
       "   'description': ['<p>This domain family is found in bacteria and eukaryotes, and is approximately 70 amino acids in length.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 47,\n",
       "    'interactions': 0,\n",
       "    'matches': 9209,\n",
       "    'pathways': 0,\n",
       "    'proteins': 1343,\n",
       "    'proteomes': 446,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 761, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 1478},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7922,\n",
       "    'alignment:full': 4438,\n",
       "    'alignment:seed': 279},\n",
       "   'cross_references': {}}},\n",
       " 'PF14490': {'metadata': {'accession': 'PF14490',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029493',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Helix-hairpin-helix containing domain', 'short': 'HHH_4'},\n",
       "   'description': ['<p>This presumed domain contains at least one helix-hairpin-helix motif. This domain is often found in RecD helicases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0198', 'name': 'HHH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 15527,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15485,\n",
       "    'proteomes': 2824,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13204, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 11926},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14444,\n",
       "    'alignment:full': 3275,\n",
       "    'alignment:seed': 336},\n",
       "   'cross_references': {}}},\n",
       " 'PF16321': {'metadata': {'accession': 'PF16321',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032528',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sigma 54 modulation/S30EA ribosomal protein C terminus',\n",
       "    'short': 'Ribosom_S30AE_C'},\n",
       "   'description': ['<p>This domain often occurs at the C-terminus of proteins containing Pfam:PF02482.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 22767,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22365,\n",
       "    'proteomes': 5576,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19295, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 21384},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20701,\n",
       "    'alignment:full': 5436,\n",
       "    'alignment:seed': 314},\n",
       "   'cross_references': {}}},\n",
       " 'PF07261': {'metadata': {'accession': 'PF07261',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006343',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Replication initiation and membrane attachment',\n",
       "    'short': 'DnaB_2'},\n",
       "   'description': ['<p>This family consists of several bacterial replication initiation and membrane attachment (DnaB) proteins, as well as DnaD which is a component of the PriA primosome.  The PriA primosome functions to recruit the replication fork helicase onto the DNA [PMID:11679082]. The DnaB protein is essential for both replication initiation and membrane attachment of the origin region of the chromosome and plasmid pUB110 in Bacillus subtilis. It is known that there are two different classes (DnaBI and DnaBII) in the DnaB mutants; DnaBI is essential for both chromosome and pUB110 replication, whereas DnaBII is necessary only for chromosome replication [PMID:3027697]. DnaD has been merged into this family. This family also includes Ftn6, a cyanobacterial-specific divisome component possibly playing a role at the interface between DNA replication and cell division [PMID:19698108]. Ftn6 possesses a conserved domain localised within the N-terminus of the proteins. This domain, named FND, exhibits sequence and structure similarities with the DnaD-like domains Pfam:PF04271 now merged into Pfam:PF07261.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00013071': {'PMID': 3027697,\n",
       "     'ISBN': None,\n",
       "     'volume': '84',\n",
       "     'issue': '3',\n",
       "     'year': 1987,\n",
       "     'title': 'Nucleotide sequence of Bacillus subtilis dnaB: a gene essential for DNA replication initiation and membrane attachment.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '653-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Hoshino T',\n",
       "      'McKenzie T',\n",
       "      'Schmidt S',\n",
       "      'Tanaka T',\n",
       "      'Sueoka N.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=3027697&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 39,\n",
       "    'interactions': 0,\n",
       "    'matches': 19253,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16518,\n",
       "    'proteomes': 2137,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14338, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 7510},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19233,\n",
       "    'alignment:full': 3933,\n",
       "    'alignment:seed': 259},\n",
       "   'cross_references': {}}},\n",
       " 'PF00003': {'metadata': {'accession': 'PF00003',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017978',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '7 transmembrane sweet-taste receptor of 3 GCPR',\n",
       "    'short': '7tm_3'},\n",
       "   'description': ['<p>This is a domain of seven transmembrane regions that forms the C-terminus of some subclass 3 G-coupled-protein receptors. It is often associated with a downstream cysteine-rich linker domain, NCD3G Pfam:PF07562, which is the human sweet-taste receptor, and the N-terminal domain, ANF_receptor Pfam:PF01094. The seven TM regions assemble in such a way as to produce a docking pocket into which such molecules as cyclamate and lactisole have been found to bind and consequently confer the taste of sweetness [PMID:16076846].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044748': {'PMID': 16076846,\n",
       "     'ISBN': None,\n",
       "     'volume': '280',\n",
       "     'issue': '40',\n",
       "     'year': 2005,\n",
       "     'title': 'Identification of the cyclamate interaction site within the transmembrane domain of the human sweet taste receptor subunit T1R3.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '34296-305',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Jiang P',\n",
       "      'Cui M',\n",
       "      'Zhao B',\n",
       "      'Snyder LA',\n",
       "      'Benard LM',\n",
       "      'Osman R',\n",
       "      'Max M',\n",
       "      'Margolskee RF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M505255200'}},\n",
       "   'set_info': {'accession': 'CL0192', 'name': 'GPCR_A'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 406,\n",
       "    'interactions': 0,\n",
       "    'matches': 44711,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44389,\n",
       "    'proteomes': 1029,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30160, 'rosettafold': 0},\n",
       "    'structures': 68,\n",
       "    'taxa': 5012},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31871,\n",
       "    'alignment:full': 20570,\n",
       "    'alignment:seed': 616},\n",
       "   'cross_references': {}}},\n",
       " 'PF04973': {'metadata': {'accession': 'PF04973',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006419',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nicotinamide mononucleotide transporter',\n",
       "    'short': 'NMN_transporter'},\n",
       "   'description': ['<p>Members of this family are integral membrane proteins that are involved in transport of nicotinamide mononucleotide [PMID:2198247, PMID:2546921].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008725': {'PMID': 2198247,\n",
       "     'ISBN': None,\n",
       "     'volume': '172',\n",
       "     'issue': '8',\n",
       "     'year': 1990,\n",
       "     'title': 'Regulation of NAD metabolism in Salmonella typhimurium: molecular sequence analysis of the bifunctional nadR regulator and the nadA-pnuC operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4187-96',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Foster JW',\n",
       "      'Park YK',\n",
       "      'Penfound T',\n",
       "      'Fenger T',\n",
       "      'Spector MP.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=2198247'},\n",
       "    'PUB00008726': {'PMID': 2546921,\n",
       "     'ISBN': None,\n",
       "     'volume': '171',\n",
       "     'issue': '8',\n",
       "     'year': 1989,\n",
       "     'title': 'Genetic characterization of the pnuC gene, which encodes a component of the nicotinamide mononucleotide transport system in Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4402-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Zhu N', 'Olivera BM', 'Roth JR.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=2546921'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 18,\n",
       "    'interactions': 0,\n",
       "    'matches': 21253,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21242,\n",
       "    'proteomes': 3341,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18477, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 15721},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20621,\n",
       "    'alignment:full': 3612,\n",
       "    'alignment:seed': 288},\n",
       "   'cross_references': {}}},\n",
       " 'PF04122': {'metadata': {'accession': 'PF04122',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007253',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ell wall binding domain 2 (CWB2)',\n",
       "    'short': 'CW_binding_2'},\n",
       "   'description': ['<p>This domain is found in 1 to 3 tandem copies in a wide variety of bacterial cell surface proteins. It has been show the three tandem repeats of the CWB2 domain are essential for correct anchoring to the cell wall [PMID:25649385]. It was shown that in SlpA and Cwp2 that these domains were essential for the binding of PSII an anionic teichoic acid-like component of the cell wall [PMID:25649385]. The structure of the Cwp8 and Cwp6 proteins shows that this domain forms a trimeric arrangement with each domain adopting a structure with some similarity to the Toprim fold [PMID:28132783]. A groove containing many conserved residues was predicted to be the site of the PSII molecule [PMID:28132783].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009852': {'PMID': 1356138,\n",
       "     'ISBN': None,\n",
       "     'volume': '138',\n",
       "     'issue': '6',\n",
       "     'year': 1992,\n",
       "     'title': 'Molecular cloning and sequencing of the upstream region of the major Bacillus subtilis autolysin gene: a modifier protein exhibiting sequence homology to the major autolysin and the spoIID product.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1067-76',\n",
       "     'medline_journal': 'J Gen Microbiol',\n",
       "     'ISO_journal': 'J. Gen. Microbiol.',\n",
       "     'authors': ['Kuroda A', 'Rashid MH', 'Sekiguchi J.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00009853': {'PMID': 11254569,\n",
       "     'ISBN': None,\n",
       "     'volume': '69',\n",
       "     'issue': '4',\n",
       "     'year': 2001,\n",
       "     'title': 'Characterization of a cell surface protein of Clostridium difficile with adhesive properties.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2144-53',\n",
       "     'medline_journal': 'Infect Immun',\n",
       "     'ISO_journal': 'Infect. Immun.',\n",
       "     'authors': ['Waligora AJ',\n",
       "      'Hennequin C',\n",
       "      'Mullany P',\n",
       "      'Bourlioux P',\n",
       "      'Collignon A',\n",
       "      'Karjalainen T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/IAI.69.4.2144-2153.2001'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1010,\n",
       "    'interactions': 0,\n",
       "    'matches': 22916,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8407,\n",
       "    'proteomes': 331,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6732, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 1636},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22471,\n",
       "    'alignment:full': 6468,\n",
       "    'alignment:seed': 958},\n",
       "   'cross_references': {}}},\n",
       " 'PF05235': {'metadata': {'accession': 'PF05235',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007899',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CHAD domain', 'short': 'CHAD'},\n",
       "   'description': ['<p>The CHAD domain is an alpha-helical domain functionally associated with the Pfam:PF01928 domains. It has conserved histidines that may chelate metals [PMID:12456267].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010433': {'PMID': 12456267,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '1',\n",
       "     'year': 2002,\n",
       "     'title': 'The catalytic domains of thiamine triphosphatase and CyaB-like adenylyl cyclase define a novel superfamily of domains that bind organic phosphates.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '33',\n",
       "     'medline_journal': 'BMC Genomics',\n",
       "     'ISO_journal': 'BMC Genomics',\n",
       "     'authors': ['Iyer LM', 'Aravind L.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=12456267&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 19633,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19572,\n",
       "    'proteomes': 3319,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16739, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 15003},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18038,\n",
       "    'alignment:full': 3627,\n",
       "    'alignment:seed': 303},\n",
       "   'cross_references': {}}},\n",
       " 'PF00515': {'metadata': {'accession': 'PF00515',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001440',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetratricopeptide repeat', 'short': 'TPR_1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001313': {'PMID': 9482716,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '5',\n",
       "     'year': 1998,\n",
       "     'title': 'The structure of the tetratricopeptide repeats of protein phosphatase 5: implications for TPR-mediated protein-protein interactions.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1192-9',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Das AK', 'Cohen PW', 'Barford D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/17.5.1192'},\n",
       "    'PUB00005443': {'PMID': 7667876,\n",
       "     'ISBN': None,\n",
       "     'volume': '20',\n",
       "     'issue': '7',\n",
       "     'year': 1995,\n",
       "     'title': 'Tetratrico peptide repeat interactions: to TPR or not to TPR?',\n",
       "     'URL': None,\n",
       "     'raw_pages': '257-9',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Lamb JR', 'Tugendreich S', 'Hieter P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)89037-4'}},\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23294,\n",
       "    'interactions': 0,\n",
       "    'matches': 157287,\n",
       "    'pathways': 0,\n",
       "    'proteins': 118802,\n",
       "    'proteomes': 5818,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 94705, 'rosettafold': 0},\n",
       "    'structures': 64,\n",
       "    'taxa': 45330},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 119653,\n",
       "    'alignment:full': 36418,\n",
       "    'alignment:seed': 550},\n",
       "   'cross_references': {}}},\n",
       " 'PF00813': {'metadata': {'accession': 'PF00813',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005838',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FliP family', 'short': 'FliP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 20,\n",
       "    'interactions': 0,\n",
       "    'matches': 25810,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25778,\n",
       "    'proteomes': 4260,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22643, 'rosettafold': 0},\n",
       "    'structures': 21,\n",
       "    'taxa': 18257},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24857,\n",
       "    'alignment:full': 4683,\n",
       "    'alignment:seed': 356},\n",
       "   'cross_references': {}}},\n",
       " 'PF13570': {'metadata': {'accession': 'PF13570',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002372',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PQQ-like domain', 'short': 'PQQ_3'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0186', 'name': 'Beta_propeller'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1478,\n",
       "    'interactions': 0,\n",
       "    'matches': 19134,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14947,\n",
       "    'proteomes': 1949,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 11561, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 9481},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9788,\n",
       "    'alignment:full': 2387,\n",
       "    'alignment:seed': 622},\n",
       "   'cross_references': {}}},\n",
       " 'PF08766': {'metadata': {'accession': 'PF08766',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014876',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DEK C terminal domain', 'short': 'DEK_C'},\n",
       "   'description': ['<p>DEK is a chromatin associated protein that is linked with cancers and autoimmune disease.  This domain is found at the C terminal of DEK and is of clinical importance since it can reverse the characteristic abnormal DNA-mutagen sensitivity in fibroblasts from ataxia-telangiectasia (A-T) patients [PMID:7504406].        The structure of this domain shows it to be homologous to the E2F/DP transcription factor family [PMID:15238633].  This domain is also found in chitin synthase proteins like Swiss:Q8TF96, and in protein phosphastases such as Swiss:Q6NN85.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00030081': {'PMID': 15238633,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '8',\n",
       "     'year': 2004,\n",
       "     'title': 'Solution NMR structure of the C-terminal domain of the human protein DEK.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2252-9',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Devany M', 'Kotharu NP', 'Matsuo H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1110/ps.04797104'},\n",
       "    'PUB00035379': {'PMID': 7504406,\n",
       "     'ISBN': None,\n",
       "     'volume': '53',\n",
       "     'issue': '6',\n",
       "     'year': 1993,\n",
       "     'title': 'Expression cloning of multiple human cDNAs that complement the phenotypic defects of ataxia-telangiectasia group D fibroblasts.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1206-16',\n",
       "     'medline_journal': 'Am J Hum Genet',\n",
       "     'ISO_journal': 'Am. J. Hum. Genet.',\n",
       "     'authors': ['Meyn MS', 'Lu-Kuo JM', 'Herzing LB.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=7504406&action=stream&blobtype=pdf'},\n",
       "    'PUB00044944': {'PMID': 16254365,\n",
       "     'ISBN': None,\n",
       "     'volume': '79',\n",
       "     'issue': '22',\n",
       "     'year': 2005,\n",
       "     'title': 'The human DEK proto-oncogene is a senescence inhibitor and an upregulated target of high-risk human papillomavirus E7.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14309-17',\n",
       "     'medline_journal': 'J Virol',\n",
       "     'ISO_journal': 'J. Virol.',\n",
       "     'authors': ['Wise-Draper TM',\n",
       "      'Allen HV',\n",
       "      'Thobe MN',\n",
       "      'Jones EE',\n",
       "      'Habash KB',\n",
       "      'Munger K',\n",
       "      'Wells SI.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JVI.79.22.14309-14317.2005'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 232,\n",
       "    'interactions': 0,\n",
       "    'matches': 12727,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12716,\n",
       "    'proteomes': 2022,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7590, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 7531},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10115,\n",
       "    'alignment:full': 6371,\n",
       "    'alignment:seed': 239},\n",
       "   'cross_references': {}}},\n",
       " 'PF07963': {'metadata': {'accession': 'PF07963',\n",
       "   'entry_id': None,\n",
       "   'type': 'conserved_site',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012902',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Prokaryotic N-terminal methylation motif',\n",
       "    'short': 'N_methyl'},\n",
       "   'description': ['<p>This short motif directs methylation of the conserved phenylalanine residue.  It is most often found at the N-terminus of pilins and other proteins involved in secretion, see Pfam:PF00114, Pfam:PF05946, Pfam:PF02501 and Pfam:PF07596.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 497,\n",
       "    'interactions': 0,\n",
       "    'matches': 230380,\n",
       "    'pathways': 0,\n",
       "    'proteins': 230315,\n",
       "    'proteomes': 4666,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 192758, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 24520},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 204416,\n",
       "    'alignment:full': 35577,\n",
       "    'alignment:seed': 384},\n",
       "   'cross_references': {}}},\n",
       " 'PF04333': {'metadata': {'accession': 'PF04333',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007428',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MlaA lipoprotein', 'short': 'MlaA'},\n",
       "   'description': ['<p>MlaA is a component of the Mla pathway, an ABC transport system that functions to maintain the asymmetry of the outer membrane [PMID:19383799]. MlaA is required for the intercellular spreading of Shigella flexneri. It is attached to the outer membrane by a lipid anchor [PMID:8145644].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009953': {'PMID': 8145644,\n",
       "     'ISBN': None,\n",
       "     'volume': '11',\n",
       "     'issue': '1',\n",
       "     'year': 1994,\n",
       "     'title': 'Identification and characterization of a chromosomal virulence gene, vacJ, required for intercellular spreading of Shigella flexneri.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '31-41',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Suzuki T',\n",
       "      'Murai T',\n",
       "      'Fukuda I',\n",
       "      'Tobe T',\n",
       "      'Yoshikawa M',\n",
       "      'Sasakawa C.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.1994.tb00287.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 18,\n",
       "    'interactions': 0,\n",
       "    'matches': 16968,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16964,\n",
       "    'proteomes': 2489,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14714, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 11826},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16409,\n",
       "    'alignment:full': 2671,\n",
       "    'alignment:seed': 247},\n",
       "   'cross_references': {}}},\n",
       " 'PF12697': {'metadata': {'accession': 'PF12697',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000073',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Alpha/beta hydrolase family', 'short': 'Abhydrolase_6'},\n",
       "   'description': ['<p>This family contains alpha/beta hydrolase enzymes of diverse specificity.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0028', 'name': 'AB_hydrolase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1250,\n",
       "    'interactions': 0,\n",
       "    'matches': 297301,\n",
       "    'pathways': 0,\n",
       "    'proteins': 297029,\n",
       "    'proteomes': 9461,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 255350, 'rosettafold': 0},\n",
       "    'structures': 192,\n",
       "    'taxa': 39667},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 272967,\n",
       "    'alignment:full': 77902,\n",
       "    'alignment:seed': 391},\n",
       "   'cross_references': {}}},\n",
       " 'PF10728': {'metadata': {'accession': 'PF10728',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018931',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF2520)',\n",
       "    'short': 'DUF2520'},\n",
       "   'description': ['<p>This presumed domain is found C-terminal to a Rossmann-like domain suggesting that these proteins are oxidoreductases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0106', 'name': '6PGD_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 14237,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14235,\n",
       "    'proteomes': 3213,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12222, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 12310},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12856,\n",
       "    'alignment:full': 2950,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF05524': {'metadata': {'accession': 'PF05524',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008731',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PEP-utilising enzyme, N-terminal',\n",
       "    'short': 'PEP-utilisers_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 90,\n",
       "    'interactions': 0,\n",
       "    'matches': 44843,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44842,\n",
       "    'proteomes': 6482,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 39379, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 26955},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44302,\n",
       "    'alignment:full': 7879,\n",
       "    'alignment:seed': 729},\n",
       "   'cross_references': {}}},\n",
       " 'PF02699': {'metadata': {'accession': 'PF02699',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003849',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Preprotein translocase subunit', 'short': 'YajC'},\n",
       "   'description': ['<p>See [PMID:11051763].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020572': {'PMID': 11051763,\n",
       "     'ISBN': None,\n",
       "     'volume': '66',\n",
       "     'issue': None,\n",
       "     'year': 2001,\n",
       "     'title': 'Protein traffic in bacteria: multiple routes from the ribosome to and across the membrane.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '107-57',\n",
       "     'medline_journal': 'Prog Nucleic Acid Res Mol Biol',\n",
       "     'ISO_journal': 'Prog. Nucleic Acid Res. Mol. Biol.',\n",
       "     'authors': ['Muller M', 'Koch HG', 'Beck K', 'Schafer U.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0079-6603(00)66028-2'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 37890,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37889,\n",
       "    'proteomes': 8001,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32744, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 32529},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34585,\n",
       "    'alignment:full': 7442,\n",
       "    'alignment:seed': 680},\n",
       "   'cross_references': {}}},\n",
       " 'PF16875': {'metadata': {'accession': 'PF16875',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031704',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolase family 36 N-terminal domain',\n",
       "    'short': 'Glyco_hydro_36N'},\n",
       "   'description': ['<p>This domain is found at the N-terminus of many family 36 glycoside hydrolases. It has a beta-supersandwich fold [PMID:23012371].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 53,\n",
       "    'interactions': 0,\n",
       "    'matches': 14441,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14410,\n",
       "    'proteomes': 2680,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12849, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 10155},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13695,\n",
       "    'alignment:full': 3515,\n",
       "    'alignment:seed': 226},\n",
       "   'cross_references': {}}},\n",
       " 'PF02622': {'metadata': {'accession': 'PF02622',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003774',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterized ACR, COG1678', 'short': 'DUF179'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 90,\n",
       "    'interactions': 0,\n",
       "    'matches': 27033,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26984,\n",
       "    'proteomes': 5434,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23149, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 22339},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24712,\n",
       "    'alignment:full': 5766,\n",
       "    'alignment:seed': 391},\n",
       "   'cross_references': {}}},\n",
       " 'PF05191': {'metadata': {'accession': 'PF05191',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007862',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Adenylate kinase, active site lid', 'short': 'ADK_lid'},\n",
       "   'description': ['<p>Comparisons of adenylate kinases have revealed  a particular divergence in the active site lid. In some organisms, particularly the Gram-positive bacteria, residues in the lid domain have been mutated to cysteines and these cysteine residues are responsible for the binding of a zinc ion. The bound zinc ion in the lid domain, is clearly  structurally homologous to Zinc-finger domains. However, it is unclear whether the adenylate kinase lid is a novel zinc-finger DNA/RNA binding domain, or that the lid bound zinc serves a purely  structural function [PMID:9715904].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010133': {'PMID': 9715904,\n",
       "     'ISBN': None,\n",
       "     'volume': '32',\n",
       "     'issue': '3',\n",
       "     'year': 1998,\n",
       "     'title': 'Crystal structures of Bacillus stearothermophilus adenylate kinase with bound Ap5A, Mg2+ Ap5A, and Mn2+ Ap5A reveal an intermediate lid position and six coordinate octahedral geometry for bound Mg2+ and Mn2+.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '276-88',\n",
       "     'medline_journal': 'Proteins',\n",
       "     'ISO_journal': 'Proteins',\n",
       "     'authors': ['Berry MB', 'Phillips GN Jr.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/(SICI)1097-0134(19980815)32:3<276::AID-PROT3>3.0.CO;2-G'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 120,\n",
       "    'interactions': 0,\n",
       "    'matches': 38288,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38254,\n",
       "    'proteomes': 7441,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32968, 'rosettafold': 0},\n",
       "    'structures': 66,\n",
       "    'taxa': 32905},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34125,\n",
       "    'alignment:full': 9049,\n",
       "    'alignment:seed': 446},\n",
       "   'cross_references': {}}},\n",
       " 'PF02595': {'metadata': {'accession': 'PF02595',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004381',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycerate kinase family', 'short': 'Gly_kinase'},\n",
       "   'description': ['<p>This is family of Glycerate kinases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009609': {'PMID': 10601204,\n",
       "     'ISBN': None,\n",
       "     'volume': '181',\n",
       "     'issue': '24',\n",
       "     'year': 1999,\n",
       "     'title': 'Genetic analysis of a chromosomal region containing genes required for assimilation of allantoin nitrogen and linked glyoxylate metabolism in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7479-84',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Cusa E', 'Obradors N', 'Baldoma L', 'Badia J', 'Aguilar J.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/181/24/7479'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 54,\n",
       "    'interactions': 0,\n",
       "    'matches': 29675,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28023,\n",
       "    'proteomes': 4351,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25008, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 18830},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29493,\n",
       "    'alignment:full': 5124,\n",
       "    'alignment:seed': 334},\n",
       "   'cross_references': {}}},\n",
       " 'PF04000': {'metadata': {'accession': 'PF04000',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007146',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sas10/Utp3/C1D family', 'short': 'Sas10_Utp3'},\n",
       "   'description': [\"<p>This family contains Utp3 and LCP5 which are components of the U3 ribonucleoprotein complex [PMID:12068309][PMID:10690410]. It also includes the human C1D protein and Saccharomyces cerevisiae YHR081W (rrp47), an exosome-associated protein required for the 3' processing of stable RNAs [PMID:12972615], and Sas10 which has been identified as a regulator of chromatin silencing [PMID:9611201]. This family also includes the human protein Neuroguidin an initiation factor 4E (eIF4E) binding protein [PMID:16705177].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008496': {'PMID': 12068309,\n",
       "     'ISBN': None,\n",
       "     'volume': '417',\n",
       "     'issue': '6892',\n",
       "     'year': 2002,\n",
       "     'title': 'A large nucleolar U3 ribonucleoprotein required for 18S ribosomal RNA biogenesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '967-70',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Dragon F',\n",
       "      'Gallagher JE',\n",
       "      'Compagnone-Post PA',\n",
       "      'Mitchell BM',\n",
       "      'Porwancher KA',\n",
       "      'Wehner KA',\n",
       "      'Wormsley S',\n",
       "      'Settlage RE',\n",
       "      'Shabanowitz J',\n",
       "      'Osheim Y',\n",
       "      'Beyer AL',\n",
       "      'Hunt DF',\n",
       "      'Baserga SJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nature00769'},\n",
       "    'PUB00009814': {'PMID': 9611201,\n",
       "     'ISBN': None,\n",
       "     'volume': '149',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Sir- and silencer-independent disruption of silencing in Saccharomyces by Sas10p.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '903-14',\n",
       "     'medline_journal': 'Genetics',\n",
       "     'ISO_journal': 'Genetics',\n",
       "     'authors': ['Kamakaka RT', 'Rine J.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=9611201&action=stream&blobtype=pdf'},\n",
       "    'PUB00010095': {'PMID': 10690410,\n",
       "     'ISBN': None,\n",
       "     'volume': '33',\n",
       "     'issue': None,\n",
       "     'year': 1999,\n",
       "     'title': 'Ribosome synthesis in Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '261-311',\n",
       "     'medline_journal': 'Annu Rev Genet',\n",
       "     'ISO_journal': 'Annu. Rev. Genet.',\n",
       "     'authors': ['Venema J', 'Tollervey D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1146/annurev.genet.33.1.261'},\n",
       "    'PUB00014272': {'PMID': 12972615,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '19',\n",
       "     'year': 2003,\n",
       "     'title': \"Rrp47p is an exosome-associated protein required for the 3' processing of stable RNAs.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '6982-92',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Mitchell P',\n",
       "      'Petfalski E',\n",
       "      'Houalla R',\n",
       "      'Podtelejnikov A',\n",
       "      'Mann M',\n",
       "      'Tollervey D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/MCB.23.19.6982-6992.2003'},\n",
       "    'PUB00045124': {'PMID': 9814757,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '11',\n",
       "     'year': 1998,\n",
       "     'title': 'Synthetic lethal interactions with conditional poly(A) polymerase alleles identify LCP5, a gene involved in 18S rRNA maturation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1357-72',\n",
       "     'medline_journal': 'RNA',\n",
       "     'ISO_journal': 'RNA',\n",
       "     'authors': ['Wiederkehr T', 'Pretot RF', 'Minvielle-Sebastia L.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1017/S1355838298980955'},\n",
       "    'PUB00045125': {'PMID': 16705177,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '11',\n",
       "     'year': 2006,\n",
       "     'title': 'Translational control by neuroguidin, a eukaryotic initiation factor 4E and CPEB binding protein.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4277-87',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Jung MY', 'Lorenz L', 'Richter JD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/MCB.02470-05'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 95,\n",
       "    'interactions': 0,\n",
       "    'matches': 7823,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7821,\n",
       "    'proteomes': 2070,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6547, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 7488},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6509,\n",
       "    'alignment:full': 3898,\n",
       "    'alignment:seed': 310},\n",
       "   'cross_references': {}}},\n",
       " 'PF01266': {'metadata': {'accession': 'PF01266',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006076',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FAD dependent oxidoreductase', 'short': 'DAO'},\n",
       "   'description': ['<p>This family includes various FAD dependent oxidoreductases: Glycerol-3-phosphate dehydrogenase EC:1.1.99.5, Sarcosine oxidase beta subunit EC:1.5.3.1, D-alanine oxidase EC:1.4.99.1, D-aspartate oxidase EC:1.4.3.1.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000445': {'PMID': 9153426,\n",
       "     'ISBN': None,\n",
       "     'volume': '36',\n",
       "     'issue': '19',\n",
       "     'year': 1997,\n",
       "     'title': 'Active site plasticity in D-amino acid oxidase: a crystallographic analysis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5853-60',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Todone F',\n",
       "      'Vanoni MA',\n",
       "      'Mozzarelli A',\n",
       "      'Bolognesi M',\n",
       "      'Coda A',\n",
       "      'Curti B',\n",
       "      'Mattevi A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi9630570'}},\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1014,\n",
       "    'interactions': 0,\n",
       "    'matches': 335144,\n",
       "    'pathways': 0,\n",
       "    'proteins': 333590,\n",
       "    'proteomes': 10326,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 286765, 'rosettafold': 0},\n",
       "    'structures': 144,\n",
       "    'taxa': 42964},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 315164,\n",
       "    'alignment:full': 73495,\n",
       "    'alignment:seed': 521},\n",
       "   'cross_references': {}}},\n",
       " 'PF01192': {'metadata': {'accession': 'PF01192',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006110',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RNA polymerase Rpb6', 'short': 'RNA_pol_Rpb6'},\n",
       "   'description': ['<p>Rpb6 is an essential subunit in the eukaryotic polymerases Pol I, II and III. This family also contains the bacterial equivalent to Rpb6, the omega subunit.  Rpb6 and omega are structurally conserved and both function in polymerase assembly [PMID:11158566].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007936': {'PMID': 11158566,\n",
       "     'ISBN': None,\n",
       "     'volume': '98',\n",
       "     'issue': '3',\n",
       "     'year': 2001,\n",
       "     'title': 'Bacterial RNA polymerase subunit omega and eukaryotic RNA polymerase subunit RPB6 are sequence, structural, and functional homologs and promote RNA polymerase assembly.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '892-7',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Minakhin L',\n",
       "      'Bhagat S',\n",
       "      'Brunning A',\n",
       "      'Campbell EA',\n",
       "      'Darst SA',\n",
       "      'Ebright RH',\n",
       "      'Severinov K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.98.3.892'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 65,\n",
       "    'interactions': 0,\n",
       "    'matches': 42183,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42183,\n",
       "    'proteomes': 10473,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36363, 'rosettafold': 0},\n",
       "    'structures': 766,\n",
       "    'taxa': 42778},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37791,\n",
       "    'alignment:full': 9637,\n",
       "    'alignment:seed': 342},\n",
       "   'cross_references': {}}},\n",
       " 'PF14698': {'metadata': {'accession': 'PF14698',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029419',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Argininosuccinate lyase C-terminal', 'short': 'ASL_C2'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of argininosuccinate lyase [1-2].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 89,\n",
       "    'interactions': 0,\n",
       "    'matches': 42333,\n",
       "    'pathways': 0,\n",
       "    'proteins': 42330,\n",
       "    'proteomes': 8672,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36306, 'rosettafold': 0},\n",
       "    'structures': 28,\n",
       "    'taxa': 35182},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39142,\n",
       "    'alignment:full': 8884,\n",
       "    'alignment:seed': 600},\n",
       "   'cross_references': {}}},\n",
       " 'PF07332': {'metadata': {'accession': 'PF07332',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009937',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative Actinobacterial Holin-X, holin superfamily III',\n",
       "    'short': 'Phage_holin_3_6'},\n",
       "   'description': ['<p>Phage_holin_3_6 is a family of small hydrophobic proteins with two or three transmembrane domains of the Hol-X family. Holin proteins are produced by double-stranded DNA bacteriophages that use an endolysin-holin strategy to achieve lysis of their hosts. The endolysins are peptidoglycan-degrading enzymes that are usually accumulated in the cytosol until access to the cell wall substrate is provided by the holin membrane lesion.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 16,\n",
       "    'interactions': 0,\n",
       "    'matches': 20663,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20643,\n",
       "    'proteomes': 3726,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18310, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15048},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19752,\n",
       "    'alignment:full': 4747,\n",
       "    'alignment:seed': 266},\n",
       "   'cross_references': {}}},\n",
       " 'PF01259': {'metadata': {'accession': 'PF01259',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR028923',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SAICAR synthetase', 'short': 'SAICAR_synt'},\n",
       "   'description': ['<p>Also known as Phosphoribosylaminoimidazole-succinocarboxamide  synthase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 131,\n",
       "    'interactions': 0,\n",
       "    'matches': 50562,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50290,\n",
       "    'proteomes': 10405,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 42726, 'rosettafold': 0},\n",
       "    'structures': 51,\n",
       "    'taxa': 42652},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45266,\n",
       "    'alignment:full': 10318,\n",
       "    'alignment:seed': 584},\n",
       "   'cross_references': {}}},\n",
       " 'PF05658': {'metadata': {'accession': 'PF05658',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008640',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YadA head domain repeat (2 copies)',\n",
       "    'short': 'YadA_head'},\n",
       "   'description': ['<p>This entry represents two copies of a fourteen residue repeat that makes up the head domain of bacterial haemagglutinins and invasins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4378,\n",
       "    'interactions': 0,\n",
       "    'matches': 126135,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12731,\n",
       "    'proteomes': 767,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7640, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 5224},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 135857,\n",
       "    'alignment:full': 15905,\n",
       "    'alignment:seed': 337},\n",
       "   'cross_references': {}}},\n",
       " 'PF04299': {'metadata': {'accession': 'PF04299',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007396',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Putative FMN-binding domain', 'short': 'FMN_bind_2'},\n",
       "   'description': ['<p>In Bacillus subtilis, family member Swiss:P21341 (PAI 2/ORF-2) was found to be essential for growth [PMID:2108124]. The SUPERFAMILY database finds that this domain is related to FMN-binding domains, suggesting this protein is also FMN-binding.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009944': {'PMID': 2108124,\n",
       "     'ISBN': None,\n",
       "     'volume': '172',\n",
       "     'issue': '4',\n",
       "     'year': 1990,\n",
       "     'title': 'A novel Bacillus subtilis gene involved in negative control of sporulation and degradative-enzyme production.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1783-90',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Honjo M',\n",
       "      'Nakayama A',\n",
       "      'Fukazawa K',\n",
       "      'Kawamura K',\n",
       "      'Ando K',\n",
       "      'Hori M',\n",
       "      'Furutani Y.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=2108124'}},\n",
       "   'set_info': {'accession': 'CL0336', 'name': 'FMN-binding'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 15194,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15179,\n",
       "    'proteomes': 3324,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13264, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 13513},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14493,\n",
       "    'alignment:full': 3488,\n",
       "    'alignment:seed': 263},\n",
       "   'cross_references': {}}},\n",
       " 'PF03098': {'metadata': {'accession': 'PF03098',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019791',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Animal haem peroxidase', 'short': 'An_peroxidase'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0617', 'name': 'Peroxidase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 999,\n",
       "    'interactions': 0,\n",
       "    'matches': 34755,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28439,\n",
       "    'proteomes': 2780,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17269, 'rosettafold': 0},\n",
       "    'structures': 215,\n",
       "    'taxa': 11649},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27355,\n",
       "    'alignment:full': 13653,\n",
       "    'alignment:seed': 330},\n",
       "   'cross_references': {}}},\n",
       " 'PF02686': {'metadata': {'accession': 'PF02686',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003837',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glu-tRNAGln amidotransferase C subunit',\n",
       "    'short': 'Glu-tRNAGln'},\n",
       "   'description': ['<p>This is a family of Glu-tRNAGln amidotransferase C subunits. The Glu-tRNA Gln amidotransferase enzyme itself is an important translational fidelity mechanism replacing incorrectly charged Glu-tRNAGln with the correct Gln-tRANGln via transmidation of the misacylated Glu-tRNAGln [PMID:9342321].  This activity supplements the lack  of glutaminyl-tRNA synthetase activity in gram-positive eubacterteria, cyanobacteria, Archaea, and organelles [PMID:9342321].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007932': {'PMID': 9342321,\n",
       "     'ISBN': None,\n",
       "     'volume': '94',\n",
       "     'issue': '22',\n",
       "     'year': 1997,\n",
       "     'title': 'Glu-tRNAGln amidotransferase: a novel heterotrimeric enzyme required for correct decoding of glutamine codons during translation.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '11819-26',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Curnow AW',\n",
       "      'Hong K',\n",
       "      'Yuan R',\n",
       "      'Kim S',\n",
       "      'Martins O',\n",
       "      'Winkler W',\n",
       "      'Henkin TM',\n",
       "      'Soll D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.94.22.11819'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 38963,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38961,\n",
       "    'proteomes': 8822,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 33328, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 36798},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34531,\n",
       "    'alignment:full': 7894,\n",
       "    'alignment:seed': 716},\n",
       "   'cross_references': {}}},\n",
       " 'PF03331': {'metadata': {'accession': 'PF03331',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004463',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'UDP-3-O-acyl N-acetylglycosamine deacetylase',\n",
       "    'short': 'LpxC'},\n",
       "   'description': ['<p>The enzymes in this family catalyse the second step in the biosynthetic pathway for lipid A.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019806': {'PMID': 10753902,\n",
       "     'ISBN': None,\n",
       "     'volume': '275',\n",
       "     'issue': '15',\n",
       "     'year': 2000,\n",
       "     'title': 'Antibacterial agents that target lipid A biosynthesis in gram-negative bacteria. Inhibition of diverse UDP-3-O-(r-3-hydroxymyristoyl)-n-acetylglucosamine deacetylases by substrate analogs containing zinc binding motifs.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '11002-9',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Jackman JE',\n",
       "      'Fierke CA',\n",
       "      'Tumey LN',\n",
       "      'Pirrung M',\n",
       "      'Uchiyama T',\n",
       "      'Tahir SH',\n",
       "      'Hindsgaul O',\n",
       "      'Raetz CR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.275.15.11002'}},\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 54,\n",
       "    'interactions': 0,\n",
       "    'matches': 27683,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27547,\n",
       "    'proteomes': 4742,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 23236, 'rosettafold': 0},\n",
       "    'structures': 86,\n",
       "    'taxa': 22297},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24749,\n",
       "    'alignment:full': 4671,\n",
       "    'alignment:seed': 286},\n",
       "   'cross_references': {}}},\n",
       " 'PF03645': {'metadata': {'accession': 'PF03645',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005334',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tctex-1 family', 'short': 'Tctex-1'},\n",
       "   'description': ['<p>Tctex-1 is a dynein light chain. It has been shown that Tctex-1 can bind to the cytoplasmic tail of rhodopsin. C-terminal rhodopsin mutations responsible for retinitis pigmentosa inhibit this interaction. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007752': {'PMID': 2570638,\n",
       "     'ISBN': None,\n",
       "     'volume': '58',\n",
       "     'issue': '5',\n",
       "     'year': 1989,\n",
       "     'title': 'tctex-1: a candidate gene family for a mouse t complex sterility locus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '969-79',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Lader E', 'Ha HS', \"O'Neill M\", 'Artzt K', 'Bennett D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0092-8674(89)90948-3'},\n",
       "    'PUB00020416': {'PMID': 10399916,\n",
       "     'ISBN': None,\n",
       "     'volume': '97',\n",
       "     'issue': '7',\n",
       "     'year': 1999,\n",
       "     'title': \"Rhodopsin's carboxy-terminal cytoplasmic tail acts as a membrane receptor for cytoplasmic dynein by binding to the dynein light chain Tctex-1.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '877-87',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Tai AW', 'Chuang JZ', 'Bode C', 'Wolfrum U', 'Sung CH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80800-4'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 112,\n",
       "    'interactions': 0,\n",
       "    'matches': 9515,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9498,\n",
       "    'proteomes': 1799,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7806, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 6702},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7194,\n",
       "    'alignment:full': 3998,\n",
       "    'alignment:seed': 286},\n",
       "   'cross_references': {}}},\n",
       " 'PF17759': {'metadata': {'accession': 'PF17759',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041616',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phenylalanyl tRNA synthetase beta chain CLM domain',\n",
       "    'short': 'tRNA_synthFbeta'},\n",
       "   'description': ['<p>This domain corresponds to the catalytic like domain (CLM) in the beta chain of phe tRNA synthetase [PMID:21082706].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0040', 'name': 'tRNA_synt_II'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 137,\n",
       "    'interactions': 0,\n",
       "    'matches': 55442,\n",
       "    'pathways': 0,\n",
       "    'proteins': 55351,\n",
       "    'proteomes': 10954,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46858, 'rosettafold': 0},\n",
       "    'structures': 43,\n",
       "    'taxa': 46363},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50152,\n",
       "    'alignment:full': 10394,\n",
       "    'alignment:seed': 213},\n",
       "   'cross_references': {}}},\n",
       " 'PF00177': {'metadata': {'accession': 'PF00177',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023798',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S7p/S5e', 'short': 'Ribosomal_S7'},\n",
       "   'description': ['<p>This family contains ribosomal protein S7 from prokaryotes and S5 from eukaryotes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005285': {'PMID': 9331418,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '9',\n",
       "     'year': 1997,\n",
       "     'title': 'The structure of ribosomal protein S7 at 1.9 A resolution reveals a beta-hairpin motif that binds double-stranded nucleic acids.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1187-98',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Wimberly BT', 'White SW', 'Ramakrishnan V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00269-4'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 163,\n",
       "    'interactions': 0,\n",
       "    'matches': 72697,\n",
       "    'pathways': 0,\n",
       "    'proteins': 72633,\n",
       "    'proteomes': 11093,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 61264, 'rosettafold': 0},\n",
       "    'structures': 1215,\n",
       "    'taxa': 75729},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 64033,\n",
       "    'alignment:full': 12546,\n",
       "    'alignment:seed': 407},\n",
       "   'cross_references': {}}},\n",
       " 'PF02245': {'metadata': {'accession': 'PF02245',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003180',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Methylpurine-DNA glycosylase (MPG)',\n",
       "    'short': 'Pur_DNA_glyco'},\n",
       "   'description': ['<p>Methylpurine-DNA glycosylase is a base excision-repair protein. It is responsible for the hydrolysis of the deoxyribose N-glycosidic bond, excising 3-methyladenine and 3-methylguanine from damaged DNA.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020209': {'PMID': 9790531,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '2',\n",
       "     'year': 1998,\n",
       "     'title': 'Crystal structure of a human alkylbase-DNA repair enzyme complexed to DNA: mechanisms for nucleotide flipping and base excision.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '249-58',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Lau AY',\n",
       "      'Scharer OD',\n",
       "      'Samson L',\n",
       "      'Verdine GL',\n",
       "      'Ellenberger T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)81755-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 21009,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20449,\n",
       "    'proteomes': 4614,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17609, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 19714},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19386,\n",
       "    'alignment:full': 4195,\n",
       "    'alignment:seed': 331},\n",
       "   'cross_references': {}}},\n",
       " 'PF03733': {'metadata': {'accession': 'PF03733',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005185',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Inner membrane component domain', 'short': 'YccF'},\n",
       "   'description': ['<p>Domain occurs as one or more copies in bacterial and eukaryotic proteins.  These are membrane proteins of four TM regions, two appearing in each of the two copies when both are present. Many of the latter members also carry the sodium/calcium exchanger protein family Pfam:PF01699, which have multipass membrane regions.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 33,\n",
       "    'interactions': 0,\n",
       "    'matches': 20085,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11330,\n",
       "    'proteomes': 3414,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9875, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13515},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19260,\n",
       "    'alignment:full': 5195,\n",
       "    'alignment:seed': 396},\n",
       "   'cross_references': {}}},\n",
       " 'PF12392': {'metadata': {'accession': 'PF12392',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020988',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Collagenase', 'short': 'DUF3656'},\n",
       "   'description': ['<p>This domain family is found in bacteria, archaea and eukaryotes, and is approximately 120 amino acids in length. The family is found in association with Pfam:PF01136.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 11872,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11870,\n",
       "    'proteomes': 2156,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10128, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 9964},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11113,\n",
       "    'alignment:full': 2078,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF03776': {'metadata': {'accession': 'PF03776',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005527',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Septum formation topological specificity factor MinE',\n",
       "    'short': 'MinE'},\n",
       "   'description': ['<p>The E. coli minicell locus was shown to code for three gene products (MinC, MinD, and MinE) whose coordinate action is required for proper placement of the division septum. The minE gene codes for a topological specificity factor that, in wild-type cells, prevents the division inhibitor from acting at internal division sites while permitting it to block septation at polar sites [PMID:2645057].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019863': {'PMID': 2645057,\n",
       "     'ISBN': None,\n",
       "     'volume': '56',\n",
       "     'issue': '4',\n",
       "     'year': 1989,\n",
       "     'title': 'A division inhibitor and a topological specificity factor coded for by the minicell locus determine proper placement of the division septum in E. coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '641-9',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['de Boer PA', 'Crossley RE', 'Rothfield LI.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0092-8674(89)90586-2'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 14,\n",
       "    'interactions': 0,\n",
       "    'matches': 13461,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13460,\n",
       "    'proteomes': 2950,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11923, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 14570},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12789,\n",
       "    'alignment:full': 2861,\n",
       "    'alignment:seed': 251},\n",
       "   'cross_references': {}}},\n",
       " 'PF13305': {'metadata': {'accession': 'PF13305',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025996',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Tetracyclin repressor-like, C-terminal domain',\n",
       "    'short': 'TetR_C_33'},\n",
       "   'description': ['<p>This domain is around 80 amino acids in length.  It is found to the C-terminus of a DNA-binding helix-turn-helix domain. This domain may be involved transcriptional regulation predicted to belong to the TetR family. This domain contains three conserved residues (WHG) near the C-terminus.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0174', 'name': 'TetR_C'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 39673,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39648,\n",
       "    'proteomes': 3977,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 34914, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 15837},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37510,\n",
       "    'alignment:full': 9881,\n",
       "    'alignment:seed': 384},\n",
       "   'cross_references': {}}},\n",
       " 'PF08044': {'metadata': {'accession': 'PF08044',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012551',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF1707)',\n",
       "    'short': 'DUF1707'},\n",
       "   'description': ['<p>This domain is found in a variety of Actinomycetales proteins.  All of  the proteins containing this domain are hypothetical and probably membrane bound or associated. Currently, it is unclear to the function  of this domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0660', 'name': 'SHOCT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 44,\n",
       "    'interactions': 0,\n",
       "    'matches': 14352,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14270,\n",
       "    'proteomes': 1185,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12760, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 3882},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13604,\n",
       "    'alignment:full': 4208,\n",
       "    'alignment:seed': 291},\n",
       "   'cross_references': {}}},\n",
       " 'PF05383': {'metadata': {'accession': 'PF05383',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006630',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'La domain', 'short': 'La'},\n",
       "   'description': ['<p>This presumed domain is found at the N-terminus of La RNA-binding proteins as well as other proteins [PMID:8035818]. The function of this region is uncertain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011367': {'PMID': 8035818,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '8',\n",
       "     'year': 1994,\n",
       "     'title': 'La proteins from Drosophila melanogaster and Saccharomyces cerevisiae: a yeast homolog of the La autoantigen is dispensable for growth.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5412-24',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Yoo CJ', 'Wolin SL.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8035818&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 225,\n",
       "    'interactions': 0,\n",
       "    'matches': 18521,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18507,\n",
       "    'proteomes': 2144,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14234, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 7886},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14324,\n",
       "    'alignment:full': 8659,\n",
       "    'alignment:seed': 343},\n",
       "   'cross_references': {}}},\n",
       " 'PF01654': {'metadata': {'accession': 'PF01654',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002585',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cytochrome bd terminal oxidase subunit I',\n",
       "    'short': 'Cyt_bd_oxida_I'},\n",
       "   'description': [\"<p>This family are the alternative oxidases found in many bacteria which oxidise ubiquinol and reduce oxygen as part of the electron transport chain. This family is the subunit I of the oxidase E. coli has two copies of the oxidase, bo and bd', both of which are represented here In some nitrogen fixing bacteria, e.g. Klebsiella pneumoniae this oxidase is responsible for removing oxygen in microaerobic conditions, making the oxidase required for nitrogen fixation. This subunit binds a single b-haem, through ligands at His186 and Met393 (using SW:P11026 numbering). In addition His19 is a ligand for the haem b found in subunit II</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003598': {'PMID': 9274021,\n",
       "     'ISBN': None,\n",
       "     'volume': '143 ( Pt 8)',\n",
       "     'issue': None,\n",
       "     'year': 1997,\n",
       "     'title': \"The Klebsiella pneumoniae cytochrome bd' terminal oxidase complex and its role in microaerobic nitrogen fixation.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '2673-83',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Juty NS', 'Moshiri F', 'Merrick M', 'Anthony C', 'Hill S.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 33,\n",
       "    'interactions': 0,\n",
       "    'matches': 39176,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38484,\n",
       "    'proteomes': 5933,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34309, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 25010},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38285,\n",
       "    'alignment:full': 7556,\n",
       "    'alignment:seed': 525},\n",
       "   'cross_references': {}}},\n",
       " 'PF03473': {'metadata': {'accession': 'PF03473',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005302',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MOSC domain', 'short': 'MOSC'},\n",
       "   'description': ['<p>The MOSC (MOCO sulfurase C-terminal) domain is a  superfamily of beta-strand-rich domains identified in the molybdenum cofactor sulfurase and several other proteins from both prokaryotes and eukaryotes. These MOSC domains contain an absolutely conserved cysteine and occur either as stand-alone forms such as Swiss:P32157, or fused to other domains such as NifS-like catalytic domain in Molybdenum cofactor sulfurase. The MOSC domain is predicted to be a sulfur-carrier domain that receives sulfur abstracted by the pyridoxal phosphate-dependent NifS-like enzymes, on its conserved cysteine, and delivers it for the formation of diverse sulfur-metal clusters.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019067': {'PMID': 11886751,\n",
       "     'ISBN': None,\n",
       "     'volume': '207',\n",
       "     'issue': '1',\n",
       "     'year': 2002,\n",
       "     'title': 'MOSC domains: ancient, predicted sulfur-carrier domains, present in diverse metal-sulfur cluster biosynthesis proteins including Molybdenum cofactor sulfurases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '55-61',\n",
       "     'medline_journal': 'FEMS Microbiol Lett',\n",
       "     'ISO_journal': 'FEMS Microbiol. Lett.',\n",
       "     'authors': ['Anantharaman V', 'Aravind L.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 255,\n",
       "    'interactions': 0,\n",
       "    'matches': 52617,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52533,\n",
       "    'proteomes': 7388,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 44851, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 30494},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48486,\n",
       "    'alignment:full': 12947,\n",
       "    'alignment:seed': 304},\n",
       "   'cross_references': {}}},\n",
       " 'PF01514': {'metadata': {'accession': 'PF01514',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006182',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Secretory protein of YscJ/FliF family',\n",
       "    'short': 'YscJ_FliF'},\n",
       "   'description': ['<p>This family includes proteins that are related to the YscJ lipoprotein, and the amino terminus of FliF, the flageller M-ring protein.  The members of the YscJ family are thought to be involved in secretion of several proteins.                  The FliF protein ring is thought to be part of the export apparatus for flageller proteins, based on the similarity to YscJ proteins [PMID:10049798].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002206': {'PMID': 1332940,\n",
       "     'ISBN': None,\n",
       "     'volume': '174',\n",
       "     'issue': '23',\n",
       "     'year': 1992,\n",
       "     'title': 'MxiJ, a lipoprotein involved in secretion of Shigella Ipa invasins, is homologous to YscJ, a secretion factor of the Yersinia Yop proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '7661-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Allaoui A', 'Sansonetti PJ', 'Parsot C.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=1332940&action=stream&blobtype=pdf'},\n",
       "    'PUB00003476': {'PMID': 10049798,\n",
       "     'ISBN': None,\n",
       "     'volume': '124',\n",
       "     'issue': '2-3',\n",
       "     'year': 1998,\n",
       "     'title': 'A structural feature in the central channel of the bacterial flagellar FliF ring complex is implicated in type III protein export.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '104-14',\n",
       "     'medline_journal': 'J Struct Biol',\n",
       "     'ISO_journal': 'J. Struct. Biol.',\n",
       "     'authors': ['Suzuki H',\n",
       "      'Yonekura K',\n",
       "      'Murata K',\n",
       "      'Hirai T',\n",
       "      'Oosawa K',\n",
       "      'Namba K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jsbi.1998.4048'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 26875,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26874,\n",
       "    'proteomes': 4219,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23591, 'rosettafold': 0},\n",
       "    'structures': 31,\n",
       "    'taxa': 18183},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26135,\n",
       "    'alignment:full': 4624,\n",
       "    'alignment:seed': 322},\n",
       "   'cross_references': {}}},\n",
       " 'PF02571': {'metadata': {'accession': 'PF02571',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003723',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Precorrin-6x reductase CbiJ/CobK', 'short': 'CbiJ'},\n",
       "   'description': ['<p>This family consists of Precorrin-6x reductase EC:1.3.1.54. This enzyme catalyses the reaction: precorrin-6Y + NADP(+)  <=> precorrin-6X + NADPH.  CbiJ and CobK both catalyse the reduction of macocycle in the colbalmin biosynthesis pathway [PMID:9742225, PMID:8501034].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000553': {'PMID': 9742225,\n",
       "     'ISBN': None,\n",
       "     'volume': '335 ( Pt 1)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'Cobalamin (vitamin B12) biosynthesis: identification and characterization of a Bacillus megaterium cobI operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '159-66',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Raux E', 'Lanois A', 'Warren MJ', 'Rambach A', 'Thermes C.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=9742225&action=stream&blobtype=pdf'},\n",
       "    'PUB00015329': {'PMID': 8501034,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '11',\n",
       "     'year': 1993,\n",
       "     'title': 'Characterization of the cobalamin (vitamin B12) biosynthetic genes of Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3303-16',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Roth JR',\n",
       "      'Lawrence JG',\n",
       "      'Rubenfield M',\n",
       "      'Kieffer-Higgins S',\n",
       "      'Church GM.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8501034'}},\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 41,\n",
       "    'interactions': 0,\n",
       "    'matches': 13751,\n",
       "    'pathways': 0,\n",
       "    'proteins': 13704,\n",
       "    'proteomes': 3121,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12226, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 13277},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13493,\n",
       "    'alignment:full': 3026,\n",
       "    'alignment:seed': 301},\n",
       "   'cross_references': {}}},\n",
       " 'PF03618': {'metadata': {'accession': 'PF03618',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005177',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Kinase/pyrophosphorylase', 'short': 'Kinase-PPPase'},\n",
       "   'description': ['<p>This family of regulatory proteins has ADP-dependent kinase and inorganic phosphate-dependent pyrophosphorylase activity [1-3].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 26,\n",
       "    'interactions': 0,\n",
       "    'matches': 19160,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19132,\n",
       "    'proteomes': 3843,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16860, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 17475},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18347,\n",
       "    'alignment:full': 3961,\n",
       "    'alignment:seed': 298},\n",
       "   'cross_references': {}}},\n",
       " 'PF14319': {'metadata': {'accession': 'PF14319',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR026889',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Transposase zinc-binding domain', 'short': 'Zn_Tnp_IS91'},\n",
       "   'description': ['<p>This domain is likely to be a zinc-binding domain. It is found at the N-terminus of transposases belonging to the IS91 family.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 8764,\n",
       "    'pathways': 0,\n",
       "    'proteins': 8762,\n",
       "    'proteomes': 785,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 7479, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 5359},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8276,\n",
       "    'alignment:full': 1472,\n",
       "    'alignment:seed': 247},\n",
       "   'cross_references': {}}},\n",
       " 'PF02561': {'metadata': {'accession': 'PF02561',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003713',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Flagellar protein FliS', 'short': 'FliS'},\n",
       "   'description': ['<p>FliS is coded for by the FliD operon and is transcribed in conjunction with FliD and FliT, however this protein has no known function.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008199': {'PMID': 8550529,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Negative regulation by fliD, fliS, and fliT of the export of the flagellum-specific anti-sigma factor, FlgM, in Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '899-901',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Yokoseki T', 'Iino T', 'Kutsukake K.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=8550529&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 17,\n",
       "    'interactions': 0,\n",
       "    'matches': 17488,\n",
       "    'pathways': 0,\n",
       "    'proteins': 17488,\n",
       "    'proteomes': 3263,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15358, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 14546},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16700,\n",
       "    'alignment:full': 3508,\n",
       "    'alignment:seed': 364},\n",
       "   'cross_references': {}}},\n",
       " 'PF13559': {'metadata': {'accession': 'PF13559',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025403',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4129)',\n",
       "    'short': 'DUF4129'},\n",
       "   'description': ['<p>This presumed domain is found at the C-terminus of proteins that contain a transglutaminase core domain. The function of this domain is unknown. The domain has a conserved TXXE motif.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 18267,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18267,\n",
       "    'proteomes': 2911,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 15321, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12534},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16343,\n",
       "    'alignment:full': 3670,\n",
       "    'alignment:seed': 344},\n",
       "   'cross_references': {}}},\n",
       " 'PF01288': {'metadata': {'accession': 'PF01288',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000550',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '7,8-dihydro-6-hydroxymethylpterin-pyrophosphokinase (HPPK)',\n",
       "    'short': 'HPPK'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 161,\n",
       "    'interactions': 0,\n",
       "    'matches': 48252,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48026,\n",
       "    'proteomes': 8859,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41419, 'rosettafold': 0},\n",
       "    'structures': 85,\n",
       "    'taxa': 35894},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44661,\n",
       "    'alignment:full': 9204,\n",
       "    'alignment:seed': 854},\n",
       "   'cross_references': {}}},\n",
       " 'PF08029': {'metadata': {'accession': 'PF08029',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013115',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HisG, C-terminal domain', 'short': 'HisG_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0089', 'name': 'GlnB-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 20829,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20825,\n",
       "    'proteomes': 4916,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17953, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 19745},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18707,\n",
       "    'alignment:full': 4587,\n",
       "    'alignment:seed': 341},\n",
       "   'cross_references': {}}},\n",
       " 'PF07715': {'metadata': {'accession': 'PF07715',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012910',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'TonB-dependent Receptor Plug Domain', 'short': 'Plug'},\n",
       "   'description': ['<p>The Plug domain has been shown to be an independently folding subunit of the TonB-dependent receptors ([PMID:15111112]). It acts as the channel gate, blocking the pore until the channel is bound by ligand. At this point it under goes conformational changes opens the channel.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015225': {'PMID': 15111112,\n",
       "     'ISBN': None,\n",
       "     'volume': '564',\n",
       "     'issue': '3',\n",
       "     'year': 2004,\n",
       "     'title': 'The plug domain of a neisserial TonB-dependent transporter retains structural integrity in the absence of its transmembrane beta-barrel.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '294-300',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Oke M',\n",
       "      'Sarra R',\n",
       "      'Ghirlando R',\n",
       "      'Farnaud S',\n",
       "      'Gorringe AR',\n",
       "      'Evans RW',\n",
       "      'Buchanan SK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(04)00196-6'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 647,\n",
       "    'interactions': 0,\n",
       "    'matches': 517380,\n",
       "    'pathways': 0,\n",
       "    'proteins': 517215,\n",
       "    'proteomes': 4554,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 452772, 'rosettafold': 0},\n",
       "    'structures': 125,\n",
       "    'taxa': 21637},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 496957,\n",
       "    'alignment:full': 97946,\n",
       "    'alignment:seed': 617},\n",
       "   'cross_references': {}}},\n",
       " 'PF00149': {'metadata': {'accession': 'PF00149',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004843',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Calcineurin-like phosphoesterase',\n",
       "    'short': 'Metallophos'},\n",
       "   'description': [\"<p>This family includes a diverse range of phosphoesterases [PMID:9685491], including protein phosphoserine phosphatases, nucleotidases, sphingomyelin phosphodiesterases and 2'-3' cAMP phosphodiesterases as well as nucleases such as bacterial SbcD Swiss:P13457 or yeast MRE11 Swiss:P32829. The most conserved regions in this superfamily centre around the metal chelating residues.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019430': {'PMID': 9685491,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '16',\n",
       "     'year': 1998,\n",
       "     'title': 'Phosphoesterase domains associated with DNA polymerases of diverse origins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3746-52',\n",
       "     'medline_journal': 'Nucleic Acids Res',\n",
       "     'ISO_journal': 'Nucleic Acids Res.',\n",
       "     'authors': ['Aravind L', 'Koonin EV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/nar/26.16.3746'}},\n",
       "   'set_info': {'accession': 'CL0163', 'name': 'Calcineurin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4234,\n",
       "    'interactions': 0,\n",
       "    'matches': 434969,\n",
       "    'pathways': 0,\n",
       "    'proteins': 432231,\n",
       "    'proteomes': 12026,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 361147, 'rosettafold': 0},\n",
       "    'structures': 333,\n",
       "    'taxa': 51595},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 388354,\n",
       "    'alignment:full': 117613,\n",
       "    'alignment:seed': 324},\n",
       "   'cross_references': {}}},\n",
       " 'PF07497': {'metadata': {'accession': 'PF07497',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011113',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Rho termination factor, RNA-binding domain',\n",
       "    'short': 'Rho_RNA_bind'},\n",
       "   'description': ['<p>The Rho termination factor disengages newly transcribed RNA from its DNA template at certain, specific transcripts.  It it thought that two copies of Rho bind to RNA and that Rho functions as a hexamer of protomers [PMID:10230401].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014277': {'PMID': 10230401,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': '4',\n",
       "     'year': 1999,\n",
       "     'title': 'The structural basis for terminator recognition by the Rho transcription termination factor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '487-93',\n",
       "     'medline_journal': 'Mol Cell',\n",
       "     'ISO_journal': 'Mol. Cell',\n",
       "     'authors': ['Bogden CE',\n",
       "      'Fass D',\n",
       "      'Bergman N',\n",
       "      'Nichols MD',\n",
       "      'Berger JM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1097-2765(00)80476-1'}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 45,\n",
       "    'interactions': 0,\n",
       "    'matches': 39861,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39825,\n",
       "    'proteomes': 7789,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33990, 'rosettafold': 0},\n",
       "    'structures': 35,\n",
       "    'taxa': 32116},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36168,\n",
       "    'alignment:full': 7592,\n",
       "    'alignment:seed': 246},\n",
       "   'cross_references': {}}},\n",
       " 'PF18075': {'metadata': {'accession': 'PF18075',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040690',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FtsX extracellular domain', 'short': 'FtsX_ECD'},\n",
       "   'description': ['<p>This is the extracellular domain (ECD) found in FtsX enzyme, a homolog of the transmembrane PG-hydrolase regulator. The FtsX extracellular domain binds the PG peptidase Rv2190c/RipC N-terminal segment, causing a conformational change that activates the enzyme ileading to PG hydrolysis in Mycobacterium tuberculosis. Structural analysis of FtsX ECD reveals fold containing two lobes connected by a flexible hinge. Mutations in the hydrophobic cleft between the lobes showed reduction in RipC binding in vitro and inhibition of FtsX function in Mycobacterium smegmatis [PMID:24843173].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0032', 'name': 'Dim_A_B_barrel'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 24,\n",
       "    'interactions': 0,\n",
       "    'matches': 30392,\n",
       "    'pathways': 0,\n",
       "    'proteins': 30303,\n",
       "    'proteomes': 5827,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26081, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 25033},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27827,\n",
       "    'alignment:full': 5609,\n",
       "    'alignment:seed': 420},\n",
       "   'cross_references': {}}},\n",
       " 'PF12686': {'metadata': {'accession': 'PF12686',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024524',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF3800)',\n",
       "    'short': 'DUF3800'},\n",
       "   'description': ['<p>This family of proteins is functionally uncharacterised. This family of proteins is found in bacteria, archaea, eukaryotes and viruses.  Proteins in this family are typically between 215 and 302 amino acids in length. There is a DE motif at the N-terminus and a QXXD motif at the C-terminus that may be functionally important.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 7,\n",
       "    'interactions': 0,\n",
       "    'matches': 9768,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9768,\n",
       "    'proteomes': 1502,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 8404, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 9722},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8952,\n",
       "    'alignment:full': 1638,\n",
       "    'alignment:seed': 226},\n",
       "   'cross_references': {}}},\n",
       " 'PF02978': {'metadata': {'accession': 'PF02978',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004125',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Signal peptide binding domain', 'short': 'SRP_SPB'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020371': {'PMID': 10497032,\n",
       "     'ISBN': None,\n",
       "     'volume': '292',\n",
       "     'issue': '3',\n",
       "     'year': 1999,\n",
       "     'title': 'Crystal structure of the conserved subdomain of human protein SRP54M at 2.1 A resolution: evidence for the mechanism of signal peptide binding.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '697-705',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Clemons WM Jr',\n",
       "      'Gowda K',\n",
       "      'Black SD',\n",
       "      'Zwieb C',\n",
       "      'Ramakrishnan V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1999.3090'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 119,\n",
       "    'interactions': 0,\n",
       "    'matches': 49832,\n",
       "    'pathways': 0,\n",
       "    'proteins': 49800,\n",
       "    'proteomes': 10929,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42523, 'rosettafold': 0},\n",
       "    'structures': 50,\n",
       "    'taxa': 45506},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44714,\n",
       "    'alignment:full': 10504,\n",
       "    'alignment:seed': 742},\n",
       "   'cross_references': {}}},\n",
       " 'PF03727': {'metadata': {'accession': 'PF03727',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022673',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hexokinase', 'short': 'Hexokinase_2'},\n",
       "   'description': ['<p>Hexokinase (EC:2.7.1.1) contains two structurally similar domains represented by this family and Pfam:PF00349. Some members of the family have two copies of each of these domains.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003193': {'PMID': 5133118,\n",
       "     'ISBN': None,\n",
       "     'volume': '61',\n",
       "     'issue': '3',\n",
       "     'year': 1971,\n",
       "     'title': 'Structure of yeast hexokinase-B. I. Preliminary x-ray studies and subunit structure.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '695-700',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(71)90073-8'},\n",
       "    'PUB00003196': {'PMID': 7001032,\n",
       "     'ISBN': None,\n",
       "     'volume': '140',\n",
       "     'issue': '2',\n",
       "     'year': 1980,\n",
       "     'title': 'Structure of a complex between yeast hexokinase A and glucose. II. Detailed comparisons of conformation and active site configuration with the native hexokinase B monomer and dimer.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '211-30',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['Bennett WS Jr', 'Steitz TA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0022-2836(80)90103-5'}},\n",
       "   'set_info': {'accession': 'CL0108', 'name': 'Actin_ATPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 223,\n",
       "    'interactions': 0,\n",
       "    'matches': 23544,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18146,\n",
       "    'proteomes': 2226,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 14354, 'rosettafold': 0},\n",
       "    'structures': 81,\n",
       "    'taxa': 8660},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17865,\n",
       "    'alignment:full': 10684,\n",
       "    'alignment:seed': 403},\n",
       "   'cross_references': {}}},\n",
       " 'PF01975': {'metadata': {'accession': 'PF01975',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002828',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Survival protein SurE', 'short': 'SurE'},\n",
       "   'description': ['<p>E. coli cells with the surE gene disrupted are found to survive poorly in stationary phase [PMID:7928962].  It is suggested that SurE may be involved in stress response. Yeast also contains a member of the family Swiss:P38254.  Swiss:P30887 can complement a mutation in acid phosphatase, suggesting that members of this family could be phosphatases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005703': {'PMID': 1423722,\n",
       "     'ISBN': None,\n",
       "     'volume': '22',\n",
       "     'issue': '5',\n",
       "     'year': 1992,\n",
       "     'title': 'Complementation of Saccharomyces cerevisiae acid phosphatase mutation by a genomic sequence from the yeast Yarrowia lipolytica identifies a new phosphatase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '345-55',\n",
       "     'medline_journal': 'Curr Genet',\n",
       "     'ISO_journal': 'Curr. Genet.',\n",
       "     'authors': ['Treton BY', 'Le Dall MT', 'Gaillardin CM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/BF00352435'},\n",
       "    'PUB00005728': {'PMID': 7928962,\n",
       "     'ISBN': None,\n",
       "     'volume': '176',\n",
       "     'issue': '19',\n",
       "     'year': 1994,\n",
       "     'title': 'A new gene involved in stationary-phase survival located at 59 minutes on the Escherichia coli chromosome.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6015-22',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Li C',\n",
       "      'Ichikawa JK',\n",
       "      'Ravetto JJ',\n",
       "      'Kuo HC',\n",
       "      'Fu JC',\n",
       "      'Clarke S.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=7928962&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 37931,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37845,\n",
       "    'proteomes': 6445,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32011, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 28130},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33763,\n",
       "    'alignment:full': 7882,\n",
       "    'alignment:seed': 646},\n",
       "   'cross_references': {}}},\n",
       " 'PF04327': {'metadata': {'accession': 'PF04327',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007422',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cysteine protease Prp', 'short': 'Peptidase_Prp'},\n",
       "   'description': ['<p>This is a family of cysteine protease that are found to cleave the N-terminus extension of ribosomal subunit L27 in eubacteria. Proteins in this family are distinguished by a pair of invariant histidine and cysteine residues with conserved spacing that form the classic catalytic dyad of a cysteine protease [PMID:25388641].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 4,\n",
       "    'interactions': 0,\n",
       "    'matches': 7374,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7374,\n",
       "    'proteomes': 1740,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6456, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 6841},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6906,\n",
       "    'alignment:full': 1722,\n",
       "    'alignment:seed': 277},\n",
       "   'cross_references': {}}},\n",
       " 'PF00471': {'metadata': {'accession': 'PF00471',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001705',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L33', 'short': 'Ribosomal_L33'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0167', 'name': 'Zn_Beta_Ribbon'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 29,\n",
       "    'interactions': 0,\n",
       "    'matches': 56687,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56683,\n",
       "    'proteomes': 9063,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 49027, 'rosettafold': 0},\n",
       "    'structures': 760,\n",
       "    'taxa': 55673},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50476,\n",
       "    'alignment:full': 9906,\n",
       "    'alignment:seed': 333},\n",
       "   'cross_references': {}}},\n",
       " 'PF17802': {'metadata': {'accession': 'PF17802',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041033',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Prealbumin-like fold domain', 'short': 'SpaA'},\n",
       "   'description': ['<p>This entry contains a prealbumin-like domain from a wide variety of bacterial surface proteins. This entry corresponds to domain 1 and domain 3 of SpaA from Corynebacterium diphtheriae [PMID:19805181]. Some members of this family contain an isopeptide bond.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0287', 'name': 'Transthyretin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2363,\n",
       "    'interactions': 0,\n",
       "    'matches': 76177,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24921,\n",
       "    'proteomes': 1932,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 16533, 'rosettafold': 0},\n",
       "    'structures': 31,\n",
       "    'taxa': 8059},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 82785,\n",
       "    'alignment:full': 11181,\n",
       "    'alignment:seed': 412},\n",
       "   'cross_references': {}}},\n",
       " 'PF01386': {'metadata': {'accession': 'PF01386',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR029751',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal L25p family', 'short': 'Ribosomal_L25p'},\n",
       "   'description': ['<p>Ribosomal protein L25 is an RNA binding protein, that binds 5S rRNA. This family includes Ctc from B. subtilis Swiss:P14194, which is induced by stress.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001319': {'PMID': 9799245,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '21',\n",
       "     'year': 1998,\n",
       "     'title': 'The NMR structure of Escherichia coli ribosomal protein L25 shows homology to general stress proteins and glutaminyl-tRNA synthetases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6377-84',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Stoldt M', 'Wohnert J', 'Gorlach M', 'Brown LR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/17.21.6377'},\n",
       "    'PUB00020252': {'PMID': 10562563,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '22',\n",
       "     'year': 1999,\n",
       "     'title': 'The NMR structure of the 5S rRNA E-domain-protein L25 complex shows preformed and induced recognition.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6508-21',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Stoldt M',\n",
       "      'Wohnert J',\n",
       "      'Ohlenschlager O',\n",
       "      'Gorlach M',\n",
       "      'Brown LR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/18.22.6508'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 40266,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40266,\n",
       "    'proteomes': 7877,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34399, 'rosettafold': 0},\n",
       "    'structures': 706,\n",
       "    'taxa': 34170},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36285,\n",
       "    'alignment:full': 7463,\n",
       "    'alignment:seed': 622},\n",
       "   'cross_references': {}}},\n",
       " 'PF02578': {'metadata': {'accession': 'PF02578',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003730',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Multi-copper polyphenol oxidoreductase laccase',\n",
       "    'short': 'Cu-oxidase_4'},\n",
       "   'description': ['<p>Laccases are multi-copper oxidoreductases able to oxidise a wide variety of phenolic and non-phenolic compounds and are widely distributed among both prokaryotes and eukaryotes. There are two main active catalytic sites with conserved histidines that are capable of binding four copper atoms [PMID:16740638].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044734': {'PMID': 16740638,\n",
       "     'ISBN': None,\n",
       "     'volume': '281',\n",
       "     'issue': '32',\n",
       "     'year': 2006,\n",
       "     'title': 'Novel polyphenol oxidase mined from a metagenome expression library of bovine rumen: biochemical properties, structural analysis, and phylogenetic relationships.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '22933-42',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Beloqui A',\n",
       "      'Pita M',\n",
       "      'Polaina J',\n",
       "      'Martinez-Arias A',\n",
       "      'Golyshina OV',\n",
       "      'Zumarraga M',\n",
       "      'Yakimov MM',\n",
       "      'Garcia-Arellano H',\n",
       "      'Alcalde M',\n",
       "      'Fernandez VM',\n",
       "      'Elborough K',\n",
       "      'Andreu JM',\n",
       "      'Ballesteros A',\n",
       "      'Plou FJ',\n",
       "      'Timmis KN',\n",
       "      'Ferrer M',\n",
       "      'Golyshin PN.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M600577200'}},\n",
       "   'set_info': {'accession': 'CL0663', 'name': 'CNF1_YfiH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 57,\n",
       "    'interactions': 0,\n",
       "    'matches': 36439,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36362,\n",
       "    'proteomes': 7334,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31182, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 30689},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33933,\n",
       "    'alignment:full': 6621,\n",
       "    'alignment:seed': 580},\n",
       "   'cross_references': {}}},\n",
       " 'PF00037': {'metadata': {'accession': 'PF00037',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017896',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '4Fe-4S binding domain', 'short': 'Fer4'},\n",
       "   'description': ['<p>Superfamily includes proteins containing  domains which bind to iron-sulfur clusters. Members  include bacterial ferredoxins, various dehydrogenases, and various reductases.  Structure of the domain is an alpha-antiparallel beta sandwich.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0344', 'name': '4Fe-4S'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2640,\n",
       "    'interactions': 0,\n",
       "    'matches': 178494,\n",
       "    'pathways': 0,\n",
       "    'proteins': 150010,\n",
       "    'proteomes': 9156,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 125562, 'rosettafold': 0},\n",
       "    'structures': 99,\n",
       "    'taxa': 40144},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 152506,\n",
       "    'alignment:full': 31425,\n",
       "    'alignment:seed': 528},\n",
       "   'cross_references': {}}},\n",
       " 'PF00410': {'metadata': {'accession': 'PF00410',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000630',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S8', 'short': 'Ribosomal_S8'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005270': {'PMID': 8805594,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '9',\n",
       "     'year': 1996,\n",
       "     'title': 'Structural evidence for specific S8-RNA and S8-protein interactions within the 30S ribosomal subunit: ribosomal protein S8 from Bacillus stearothermophilus at 1.9 A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1093-104',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Davies C', 'Ramakrishnan V', 'White SW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(96)00115-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 122,\n",
       "    'interactions': 0,\n",
       "    'matches': 65870,\n",
       "    'pathways': 0,\n",
       "    'proteins': 65823,\n",
       "    'proteomes': 11124,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 56563, 'rosettafold': 0},\n",
       "    'structures': 1068,\n",
       "    'taxa': 66882},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57474,\n",
       "    'alignment:full': 11802,\n",
       "    'alignment:seed': 402},\n",
       "   'cross_references': {}}},\n",
       " 'PF00745': {'metadata': {'accession': 'PF00745',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015896',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutamyl-tRNAGlu reductase, dimerisation domain',\n",
       "    'short': 'GlutR_dimer'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019621': {'PMID': 3075378,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '4',\n",
       "     'year': 1988,\n",
       "     'title': 'tRNA(Glu) as a cofactor in delta-aminolevulinate biosynthesis: steps that regulate chlorophyll synthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '139-43',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Kannangara CG',\n",
       "      'Gough SP',\n",
       "      'Bruyant P',\n",
       "      'Hoober JK',\n",
       "      'Kahn A',\n",
       "      'von Wettstein D.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0968-0004(88)90071-0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 53,\n",
       "    'interactions': 0,\n",
       "    'matches': 29464,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29462,\n",
       "    'proteomes': 5979,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25635, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 25758},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27321,\n",
       "    'alignment:full': 5868,\n",
       "    'alignment:seed': 467},\n",
       "   'cross_references': {}}},\n",
       " 'PF01346': {'metadata': {'accession': 'PF01346',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000774',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain amino terminal to FKBP-type peptidyl-prolyl isomerase',\n",
       "    'short': 'FKBP_N'},\n",
       "   'description': ['<p>This family is only found at the amino terminus of Pfam:PF00254. This entry represents the N-terminal domain found in FKBP-type peptidylprolyl isomerases (PPIase). The N-terminal domain forms the dimer interface by the mutual exchange of two beta-strands between monomers [PMID:14672666].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 70,\n",
       "    'interactions': 0,\n",
       "    'matches': 24199,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23743,\n",
       "    'proteomes': 2098,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20893, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 11226},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23683,\n",
       "    'alignment:full': 3613,\n",
       "    'alignment:seed': 335},\n",
       "   'cross_references': {}}},\n",
       " 'PF03367': {'metadata': {'accession': 'PF03367',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004457',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ZPR1 zinc-finger domain', 'short': 'zf-ZPR1'},\n",
       "   'description': ['<p>The zinc-finger protein ZPR1 is ubiquitous among eukaryotes. It is indeed known to be an essential protein in yeast. In quiescent cells, ZPR1 is localised to the cytoplasm. But in proliferating cells treated with EGF or with other mitogens, ZPR1 accumulates in the nucleolus. ZPR1 interacts with the cytoplasmic domain of the inactive EGF receptor (EGFR) and is thought to inhibit the basal protein tyrosine kinase activity of EGFR. This interaction is disrupted when cells are treated with EGF, though by themselves, inactive EGFRs are not sufficient to sequester ZPR1 to the cytoplasm [PMID:9852145, PMID:9763455, PMID:8650580]. Upon stimulation by EGF, ZPR1 directly binds the eukaryotic translation elongation factor-1alpha (eEF-1alpha) to form ZPR1/eEF-1alpha complexes [PMID:9852145]. These move into the nucleus, localising particularly at the nucleolus. Indeed, the interaction between ZPR1 and eEF-1alpha has been shown to be essential for normal cellular proliferation [PMID:9852145], and ZPR1 is thought to be involved in pre-ribosomal RNA expression [PMID:9763455]. The ZPR1 domain consists of an elongation initiation factor 2-like zinc finger and a double-stranded beta helix with a helical hairpin insertion. ZPR1 binds preferentially to GDP-bound eEF1A but does not directly influence the kinetics of nucleotide exchange or GTP hydrolysis [PMID:17704259]. The alignment for this family shows a domain of which there are two copies in ZPR1 proteins. This family also includes several hypothetical archaeal proteins (from both Crenarchaeota and Euryarchaeota), which only contain one copy of the aligned region. This similarity between ZPR1 and archaeal proteins was not previously noted.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00017397': {'PMID': 8650580,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '5269',\n",
       "     'year': 1996,\n",
       "     'title': 'Binding of zinc finger protein ZPR1 to the epidermal growth factor receptor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1797-802',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Galcheva-Gargova Z',\n",
       "      'Konstantinov KN',\n",
       "      'Wu IH',\n",
       "      'Klier FG',\n",
       "      'Barrett T',\n",
       "      'Davis RJ.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/272/5269/1797'},\n",
       "    'PUB00020594': {'PMID': 9852145,\n",
       "     'ISBN': None,\n",
       "     'volume': '143',\n",
       "     'issue': '6',\n",
       "     'year': 1998,\n",
       "     'title': 'Interaction of ZPR1 with translation elongation factor-1alpha in proliferating cells.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1471-84',\n",
       "     'medline_journal': 'J Cell Biol',\n",
       "     'ISO_journal': 'J. Cell Biol.',\n",
       "     'authors': ['Gangwani L', 'Mikrut M', 'Galcheva-Gargova Z', 'Davis RJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1083/jcb.143.6.1471'},\n",
       "    'PUB00020595': {'PMID': 9763455,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '10',\n",
       "     'year': 1998,\n",
       "     'title': 'The cytoplasmic zinc finger protein ZPR1 accumulates in the nucleolus of proliferating cells.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2963-71',\n",
       "     'medline_journal': 'Mol Biol Cell',\n",
       "     'ISO_journal': 'Mol. Biol. Cell',\n",
       "     'authors': ['Galcheva-Gargova Z',\n",
       "      'Gangwani L',\n",
       "      'Konstantinov KN',\n",
       "      'Mikrut M',\n",
       "      'Theroux SJ',\n",
       "      'Enoch T',\n",
       "      'Davis RJ.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=9763455'},\n",
       "    'PUB00045181': {'PMID': 17704259,\n",
       "     'ISBN': None,\n",
       "     'volume': '104',\n",
       "     'issue': '35',\n",
       "     'year': 2007,\n",
       "     'title': 'Structural insights into the interaction of the evolutionarily conserved ZPR1 domain tandem with eukaryotic EF1A, receptors, and SMN complexes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '13930-5',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Mishra AK', 'Gangwani L', 'Davis RJ', 'Lambright DG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.0704915104'}},\n",
       "   'set_info': {'accession': 'CL0167', 'name': 'Zn_Beta_Ribbon'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 71,\n",
       "    'interactions': 0,\n",
       "    'matches': 9167,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5726,\n",
       "    'proteomes': 2192,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4716, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 8720},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7513,\n",
       "    'alignment:full': 3920,\n",
       "    'alignment:seed': 289},\n",
       "   'cross_references': {}}},\n",
       " 'PF02261': {'metadata': {'accession': 'PF02261',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003190',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aspartate decarboxylase', 'short': 'Asp_decarbox'},\n",
       "   'description': ['<p>Decarboxylation of aspartate is the major route of beta-alanine  production in bacteria, and is catalysed by the enzyme aspartate decarboxylase EC:4.1.1.11 which requires a pyruvoyl group for its activity. It is synthesised initially as a proenzyme which is then proteolytically cleaved to an alpha (C-terminal) and beta (N-terminal) subunit and a pyruvoyl group. This family contains both chains of aspartate decarboxylase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019168': {'PMID': 9546220,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '4',\n",
       "     'year': 1998,\n",
       "     'title': 'Crystal structure of aspartate decarboxylase at 2.2 A resolution provides evidence for an ester in protein self-processing.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '289-93',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Albert A',\n",
       "      'Dhanaraj V',\n",
       "      'Genschel U',\n",
       "      'Khan G',\n",
       "      'Ramjee MK',\n",
       "      'Pulido R',\n",
       "      'Sibanda BL',\n",
       "      'von Delft F',\n",
       "      'Witty M',\n",
       "      'Blundell TL',\n",
       "      'Smith AG',\n",
       "      'Abell C.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0498-289'}},\n",
       "   'set_info': {'accession': 'CL0332', 'name': 'AcetylDC-like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 26,\n",
       "    'interactions': 0,\n",
       "    'matches': 23287,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23283,\n",
       "    'proteomes': 4846,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20394, 'rosettafold': 0},\n",
       "    'structures': 31,\n",
       "    'taxa': 21239},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21616,\n",
       "    'alignment:full': 4725,\n",
       "    'alignment:seed': 331},\n",
       "   'cross_references': {}}},\n",
       " 'PF18072': {'metadata': {'accession': 'PF18072',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041609',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Formylglycinamide ribonucleotide amidotransferase linker domain',\n",
       "    'short': 'FGAR-AT_linker'},\n",
       "   'description': ['<p>This is the linker domain found in Formylglycinamide ribonucleotide amidotransferase (FGAR-AT), also known as Phosphoribosylformylglycinamidine synthase (EC:6.3.5.3), PurL and formylglycinamidine ribonucleotide (FGAM) synthase. This enzyme catalyzes the ATP-dependent conversion of formylglycinamide ribonucleotide (FGAR) and glutamine to formylglycinamidine ribonucleotide (FGAM), ADP, Pi, and glutamate in the fourth step of the purine biosynthetic pathway.  The structure analysis of Salmonella typhimurium FGAR-AT reveals that this linker domain is made up of a long hydrophilic belt with an extended conformation [PMID:15301531].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 148,\n",
       "    'interactions': 0,\n",
       "    'matches': 46603,\n",
       "    'pathways': 0,\n",
       "    'proteins': 46600,\n",
       "    'proteomes': 9730,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 29120, 'rosettafold': 0},\n",
       "    'structures': 25,\n",
       "    'taxa': 40008},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42840,\n",
       "    'alignment:full': 9200,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF01027': {'metadata': {'accession': 'PF01027',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006214',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Inhibitor of apoptosis-promoting Bax1',\n",
       "    'short': 'Bax1-I'},\n",
       "   'description': ['<p>Programmed cell-death involves a set of Bcl-2 family proteins, some of which inhibit apoptosis (Bcl-2 and Bcl-XL) and some of which promote it (Bax and Bak). Human Bax inhibitor, BI-1, is an evolutionarily conserved integral membrane protein containing multiple membrane-spanning segments predominantly localised to intracellular membranes. It has 6-7 membrane-spanning domains. The C termini of the mammalian BI-1 proteins are comprised of basic amino acids resembling some nuclear targeting sequences, but otherwise the predicted proteins lack motifs that suggest a function. As plant BI-1 appears to localise predominantly to the ER, we hypothesized that plant BI-1 could also regulate cell death triggered by ER stress [PMID:19704470].  BI-1 appears to exert its effect through an interaction with calmodulin [PMID:19742129]. The budding yeast member of this family has been found unexpectedly to encode a BH3 domain-containing protein (Ybh3p) that regulates the mitochondrial pathway of apoptosis in a phylogenetically conserved manner [PMID:21673659]. Examination of the crystal structure of a bacterial member of this family shows that these proteins mediate a calcium leak across the membrane that is pH-dependent. Calcium homoeostasis balances passive calcium leak with active calcium uptake. The structure exists in a pore-closed and pore-open conformation, at pHs of 8 and 6 respectively, and the pore can be opened by intracrystalline transition; together these findings suggest that pH controls the conformational transition [PMID:24904158].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0453', 'name': 'Apoptosis-Inhib'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 213,\n",
       "    'interactions': 0,\n",
       "    'matches': 42087,\n",
       "    'pathways': 0,\n",
       "    'proteins': 41889,\n",
       "    'proteomes': 6624,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 35334, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 28202},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37631,\n",
       "    'alignment:full': 11686,\n",
       "    'alignment:seed': 276},\n",
       "   'cross_references': {}}},\n",
       " 'PF05013': {'metadata': {'accession': 'PF05013',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007709',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'N-formylglutamate amidohydrolase', 'short': 'FGase'},\n",
       "   'description': ['<p>Formylglutamate amidohydrolase (FGase) catalyses the terminal reaction in the five-step pathway for histidine utilisation in Pseudomonas putida. By this action, N-formyl-L-glutamate (FG) is hydrolysed to produce L-glutamate plus formate [PMID:3308850].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00010158': {'PMID': 3308850,\n",
       "     'ISBN': None,\n",
       "     'volume': '169',\n",
       "     'issue': '10',\n",
       "     'year': 1987,\n",
       "     'title': 'Purification and properties of formylglutamate amidohydrolase from Pseudomonas putida.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4696-702',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Hu L', 'Mulfinger LM', 'Phillips AT.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=3308850&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0035', 'name': 'Peptidase_MH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 38,\n",
       "    'interactions': 0,\n",
       "    'matches': 23773,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23769,\n",
       "    'proteomes': 2795,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 20399, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 11943},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22114,\n",
       "    'alignment:full': 4962,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF01205': {'metadata': {'accession': 'PF01205',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001498',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Uncharacterized protein family UPF0029',\n",
       "    'short': 'UPF0029'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 112,\n",
       "    'interactions': 0,\n",
       "    'matches': 32175,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32144,\n",
       "    'proteomes': 7834,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28254, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 31185},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30262,\n",
       "    'alignment:full': 7903,\n",
       "    'alignment:seed': 688},\n",
       "   'cross_references': {}}},\n",
       " 'PF01657': {'metadata': {'accession': 'PF01657',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002902',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Salt stress response/antifungal',\n",
       "    'short': 'Stress-antifung'},\n",
       "   'description': ['<p>This domain is often found in association with the kinase domains Pfam:PF00069 or Pfam:PF07714. In many proteins it is duplicated. It contains six conserved cysteines which are involved in disulphide bridges [PMID:19603485]. It has a role in salt stress response [PMID:19036832] and has antifungal activity [PMID:17338634].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 428,\n",
       "    'interactions': 0,\n",
       "    'matches': 36981,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19207,\n",
       "    'proteomes': 201,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16065, 'rosettafold': 0},\n",
       "    'structures': 4,\n",
       "    'taxa': 769},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31353,\n",
       "    'alignment:full': 21992,\n",
       "    'alignment:seed': 425},\n",
       "   'cross_references': {}}},\n",
       " 'PF00858': {'metadata': {'accession': 'PF00858',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001873',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Amiloride-sensitive sodium channel', 'short': 'ASC'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 466,\n",
       "    'interactions': 0,\n",
       "    'matches': 24796,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22843,\n",
       "    'proteomes': 921,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 16865, 'rosettafold': 0},\n",
       "    'structures': 30,\n",
       "    'taxa': 3807},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17573,\n",
       "    'alignment:full': 9840,\n",
       "    'alignment:seed': 346},\n",
       "   'cross_references': {}}},\n",
       " 'PF02881': {'metadata': {'accession': 'PF02881',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013822',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SRP54-type protein, helical bundle domain',\n",
       "    'short': 'SRP54_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004250': {'PMID': 9002525,\n",
       "     'ISBN': None,\n",
       "     'volume': '385',\n",
       "     'issue': '6614',\n",
       "     'year': 1997,\n",
       "     'title': 'Crystal structure of the NG domain from the signal-recognition particle receptor FtsY.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '365-8',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Montoya G', 'Svensson C', 'Luirink J', 'Sinning I.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/385365a0'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 235,\n",
       "    'interactions': 0,\n",
       "    'matches': 96148,\n",
       "    'pathways': 0,\n",
       "    'proteins': 96127,\n",
       "    'proteomes': 11073,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 82005, 'rosettafold': 0},\n",
       "    'structures': 92,\n",
       "    'taxa': 46276},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 86713,\n",
       "    'alignment:full': 20239,\n",
       "    'alignment:seed': 1220},\n",
       "   'cross_references': {}}},\n",
       " 'PF01960': {'metadata': {'accession': 'PF01960',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002813',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ArgJ family', 'short': 'ArgJ'},\n",
       "   'description': ['<p>Members of the ArgJ family catalyse the first EC:2.3.1.1 and fifth steps EC:2.3.1.35 in arginine biosynthesis. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005708': {'PMID': 8473852,\n",
       "     'ISBN': None,\n",
       "     'volume': '139',\n",
       "     'issue': '3',\n",
       "     'year': 1993,\n",
       "     'title': 'Primary structure, partial purification and regulation of key enzymes of the acetyl cycle of arginine biosynthesis in Bacillus stearothermophilus: dual function of ornithine acetyltransferase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '393-402',\n",
       "     'medline_journal': 'J Gen Microbiol',\n",
       "     'ISO_journal': 'J. Gen. Microbiol.',\n",
       "     'authors': ['Sakanyan V',\n",
       "      'Charlier D',\n",
       "      'Legrain C',\n",
       "      'Kochikyan A',\n",
       "      'Mett I',\n",
       "      'Pierard A',\n",
       "      'Glansdorff N.'],\n",
       "     'DOI_URL': None}},\n",
       "   'set_info': {'accession': 'CL0635', 'name': 'DmpA_ArgJ'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 61,\n",
       "    'interactions': 0,\n",
       "    'matches': 33103,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33013,\n",
       "    'proteomes': 7262,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 28035, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 28214},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29963,\n",
       "    'alignment:full': 7005,\n",
       "    'alignment:seed': 648},\n",
       "   'cross_references': {}}},\n",
       " 'PF00278': {'metadata': {'accession': 'PF00278',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022643',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pyridoxal-dependent decarboxylase, C-terminal sheet domain',\n",
       "    'short': 'Orn_DAP_Arg_deC'},\n",
       "   'description': ['<p>These pyridoxal-dependent decarboxylases act on ornithine, lysine,  arginine and related substrates.  </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011776': {'PMID': 10378276,\n",
       "     'ISBN': None,\n",
       "     'volume': '7',\n",
       "     'issue': '5',\n",
       "     'year': 1999,\n",
       "     'title': 'Structure of mammalian ornithine decarboxylase at 1.6 A resolution: stereochemical implications of PLP-dependent amino acid decarboxylases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '567-81',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Kern AD', 'Oliveira MA', 'Coffino P', 'Hackert ML.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(99)80073-2'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 231,\n",
       "    'interactions': 0,\n",
       "    'matches': 82442,\n",
       "    'pathways': 0,\n",
       "    'proteins': 82382,\n",
       "    'proteomes': 10464,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 70550, 'rosettafold': 0},\n",
       "    'structures': 57,\n",
       "    'taxa': 42996},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 75196,\n",
       "    'alignment:full': 17889,\n",
       "    'alignment:seed': 430},\n",
       "   'cross_references': {}}},\n",
       " 'PF01713': {'metadata': {'accession': 'PF01713',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002625',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Smr domain', 'short': 'Smr'},\n",
       "   'description': ['<p>This family includes the Smr (Small MutS Related) proteins, and the C-terminal region of the MutS2 protein. It has been suggested that this domain interacts with the MutS1 Swiss:P23909 protein in the case of Smr proteins and with the N-terminal MutS related region of MutS2 Swiss:P94545 [PMID:10431172]. This domain exhibits nicking endonuclease activity that might have a role in mismatch repair or genetic recombination. It shows no significant double strand cleavage or exonuclease activity [PMID:12730195]. The full-length Swiss:Q86UW6 also has the polynucleotide kinase activity.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005494': {'PMID': 10431172,\n",
       "     'ISBN': None,\n",
       "     'volume': '24',\n",
       "     'issue': '8',\n",
       "     'year': 1999,\n",
       "     'title': 'Smr: a bacterial and eukaryotic homologue of the C-terminal region of the MutS2 family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '298-300',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Moreira D', 'Philippe H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(99)01419-X'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 242,\n",
       "    'interactions': 0,\n",
       "    'matches': 45023,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45011,\n",
       "    'proteomes': 8147,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 37838, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 33992},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41214,\n",
       "    'alignment:full': 9589,\n",
       "    'alignment:seed': 565},\n",
       "   'cross_references': {}}},\n",
       " 'PF03797': {'metadata': {'accession': 'PF03797',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005546',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Autotransporter beta-domain', 'short': 'Autotransporter'},\n",
       "   'description': ['<p>Secretion of protein products occurs by a number of different pathways in bacteria.  One of these pathways known as the type V pathway was first described for the IgA1 protease [PMID:3027577]. The protein component that mediates secretion through the outer membrane is contained within the secreted protein itself, hence the proteins secreted in this way are called autotransporters. This family corresponds to the presumed integral membrane beta-barrel domain that transports the protein. This domain is found at the C terminus of the proteins it occurs in.  The N terminus contains the variable passenger domain that is translocated across the membrane.  Once the passenger domain is exported it is cleaved auto-catalytically in some proteins, in others a different protease is used and in some cases no cleavage occurs [PMID:9778731].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008434': {'PMID': 3027577,\n",
       "     'ISBN': None,\n",
       "     'volume': '325',\n",
       "     'issue': '6103',\n",
       "     'year': 1987,\n",
       "     'title': 'Gene structure and extracellular secretion of Neisseria gonorrhoeae IgA protease.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '458-62',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Pohlner J', 'Halter R', 'Beyreuther K', 'Meyer TF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/325458a0'},\n",
       "    'PUB00008435': {'PMID': 9778731,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'The great escape: structure and function of the autotransporter proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '370-8',\n",
       "     'medline_journal': 'Trends Microbiol',\n",
       "     'ISO_journal': 'Trends Microbiol.',\n",
       "     'authors': ['Henderson IR', 'Navarro-Garcia F', 'Nataro JP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0966-842X(98)01318-3'},\n",
       "    'PUB00019184': {'PMID': 11980709,\n",
       "     'ISBN': None,\n",
       "     'volume': '21',\n",
       "     'issue': '9',\n",
       "     'year': 2002,\n",
       "     'title': 'Export of autotransported proteins proceeds through an oligomeric ring shaped by C-terminal domains.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2122-31',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Veiga E',\n",
       "      'Sugawara E',\n",
       "      'Nikaido H',\n",
       "      'de Lorenzo V',\n",
       "      'Fernandez LA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/21.9.2122'},\n",
       "    'PUB00019185': {'PMID': 15014442,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '6',\n",
       "     'year': 2004,\n",
       "     'title': 'Structure of the translocator domain of a bacterial autotransporter.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1257-66',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Oomen CJ',\n",
       "      'van Ulsen P',\n",
       "      'van Gelder P',\n",
       "      'Feijen M',\n",
       "      'Tommassen J',\n",
       "      'Gros P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/sj.emboj.7600148'}},\n",
       "   'set_info': {'accession': 'CL0193', 'name': 'MBB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 831,\n",
       "    'interactions': 0,\n",
       "    'matches': 44869,\n",
       "    'pathways': 0,\n",
       "    'proteins': 44852,\n",
       "    'proteomes': 1878,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 34477, 'rosettafold': 0},\n",
       "    'structures': 22,\n",
       "    'taxa': 10180},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51173,\n",
       "    'alignment:full': 5454,\n",
       "    'alignment:seed': 542},\n",
       "   'cross_references': {}}},\n",
       " 'PF01990': {'metadata': {'accession': 'PF01990',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008218',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase (F/14-kDa) subunit', 'short': 'ATP-synt_F'},\n",
       "   'description': ['<p>This family includes 14-kDa subunit from vATPases [PMID:8682310], which is in the peripheral catalytic part of the complex [PMID:8621738]. The family also includes archaebacterial ATP synthase subunit F [PMID:8702544].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002959': {'PMID': 8702544,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '31',\n",
       "     'year': 1996,\n",
       "     'title': 'Subunit structure and organization of the genes of the A1A0 ATPase from the Archaeon Methanosarcina mazei Go1.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '18843-52',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Wilms R',\n",
       "      'Freiberg C',\n",
       "      'Wegerle E',\n",
       "      'Meier I',\n",
       "      'Mayer F',\n",
       "      'Muller V.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.31.18843'},\n",
       "    'PUB00005757': {'PMID': 8621738,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '6',\n",
       "     'year': 1996,\n",
       "     'title': 'Identification of a 14-kDa subunit associated with the catalytic sector of clathrin-coated vesicle H+-ATPase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3324-7',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Peng SB', 'Crider BP', 'Tsai SJ', 'Xie XS', 'Stone DK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.46.28734'},\n",
       "    'PUB00005760': {'PMID': 8682310,\n",
       "     'ISBN': None,\n",
       "     'volume': '172',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'The Drosophila melanogaster gene vha14 encoding a 14-kDa F-subunit of the vacuolar ATPase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '239-43',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Guo Y', 'Kaiser K', 'Wieczorek H', 'Dow JA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1119(96)00057-1'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 38,\n",
       "    'interactions': 0,\n",
       "    'matches': 10877,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10753,\n",
       "    'proteomes': 2955,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9254, 'rosettafold': 0},\n",
       "    'structures': 101,\n",
       "    'taxa': 12849},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9258,\n",
       "    'alignment:full': 2987,\n",
       "    'alignment:seed': 277},\n",
       "   'cross_references': {}}},\n",
       " 'PF17762': {'metadata': {'accession': 'PF17762',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041468',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HTH domain found in ParB protein', 'short': 'HTH_ParB'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 51,\n",
       "    'interactions': 0,\n",
       "    'matches': 45512,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45494,\n",
       "    'proteomes': 8028,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 38819, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 33123},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 41286,\n",
       "    'alignment:full': 8525,\n",
       "    'alignment:seed': 187},\n",
       "   'cross_references': {}}},\n",
       " 'PF06418': {'metadata': {'accession': 'PF06418',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017456',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CTP synthase N-terminus', 'short': 'CTP_synth_N'},\n",
       "   'description': ['<p>This family consists of the N-terminal region of the CTP synthase protein (EC:6.3.4.2). This family is found in conjunction with Pfam:PF00117 located in the C-terminal region of the protein. CTP synthase catalyses the synthesis of CTP from UTP by amination of the pyrimidine ring at the 4-position [PMID:12522217].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012378': {'PMID': 12522217,\n",
       "     'ISBN': None,\n",
       "     'volume': '278',\n",
       "     'issue': '11',\n",
       "     'year': 2003,\n",
       "     'title': 'Thr-431 and Arg-433 are part of a conserved sequence motif of the glutamine amidotransferase domain of CTP synthases and are involved in GTP activation of the Lactococcus lactis enzyme.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9407-11',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Willemoes M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M212995200'}},\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 161,\n",
       "    'interactions': 0,\n",
       "    'matches': 55174,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54812,\n",
       "    'proteomes': 10701,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46432, 'rosettafold': 0},\n",
       "    'structures': 44,\n",
       "    'taxa': 45351},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49047,\n",
       "    'alignment:full': 11706,\n",
       "    'alignment:seed': 380},\n",
       "   'cross_references': {}}},\n",
       " 'PF03484': {'metadata': {'accession': 'PF03484',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005147',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'tRNA synthetase B5 domain', 'short': 'B5'},\n",
       "   'description': ['<p>This domain is found in phenylalanine-tRNA synthetase beta subunits.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00007363': {'PMID': 10447505,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '8',\n",
       "     'year': 1999,\n",
       "     'title': 'Evolution of aminoacyl-tRNA synthetases--analysis of unique domain architectures and phylogenetic trees reveals a complex history of horizontal gene transfer events.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '689-710',\n",
       "     'medline_journal': 'Genome Res',\n",
       "     'ISO_journal': 'Genome Res.',\n",
       "     'authors': ['Wolf YI', 'Aravind L', 'Grishin NV', 'Koonin EV.'],\n",
       "     'DOI_URL': 'http://www.genome.org/cgi/content/abstract/9/8/689'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 118,\n",
       "    'interactions': 0,\n",
       "    'matches': 54180,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54179,\n",
       "    'proteomes': 10873,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 45929, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 46091},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49119,\n",
       "    'alignment:full': 10156,\n",
       "    'alignment:seed': 1043},\n",
       "   'cross_references': {}}},\n",
       " 'PF02607': {'metadata': {'accession': 'PF02607',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003759',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'B12 binding domain', 'short': 'B12-binding_2'},\n",
       "   'description': ['<p>This B12 binding domain is found in methionine synthase EC:2.1.1.13 Swiss:Q99707, and other shorter proteins that bind to B12. This domain is always found to the N-terminus of Pfam:PF02310. The structure of this domain is known [PMID:7992050], it is a  4 helix bundle. Many of the conserved residues in this domain are involved in B12 binding, such as those in the MXXVG motif.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008237': {'PMID': 7992050,\n",
       "     'ISBN': None,\n",
       "     'volume': '266',\n",
       "     'issue': '5191',\n",
       "     'year': 1994,\n",
       "     'title': 'How a protein binds B12: A 3.0 A X-ray structure of B12-binding domains of methionine synthase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1669-74',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Drennan CL',\n",
       "      'Huang S',\n",
       "      'Drummond JT',\n",
       "      'Matthews RG',\n",
       "      'Lidwig ML.'],\n",
       "     'DOI_URL': 'http://www.sciencemag.org/cgi/content/abstract/266/5191/1669'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 181,\n",
       "    'interactions': 0,\n",
       "    'matches': 54137,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54131,\n",
       "    'proteomes': 7578,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45055, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 31155},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47996,\n",
       "    'alignment:full': 10342,\n",
       "    'alignment:seed': 676},\n",
       "   'cross_references': {}}},\n",
       " 'PF00163': {'metadata': {'accession': 'PF00163',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001912',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S4/S9 N-terminal domain',\n",
       "    'short': 'Ribosomal_S4'},\n",
       "   'description': ['<p>This family includes small ribosomal subunit S9 from prokaryotes and S16 from metazoans.  This domain is predicted to bind to ribosomal RNA [PMID:9707415].  This domain is composed of four helices in the known structure.  However the domain is discontinuous in sequence and the alignment for this family contains only the first three helices.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001317': {'PMID': 9707415,\n",
       "     'ISBN': None,\n",
       "     'volume': '17',\n",
       "     'issue': '16',\n",
       "     'year': 1998,\n",
       "     'title': 'The crystal structure of ribosomal protein S4 reveals a two-domain molecule with an extensive RNA-binding surface: one domain shows structural homology to the ETS DNA-binding motif.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4545-58',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Davies C',\n",
       "      'Gerstner RB',\n",
       "      'Draper DE',\n",
       "      'Ramakrishnan V',\n",
       "      'White SW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1093/emboj/17.16.4545'}},\n",
       "   'set_info': {'accession': 'CL0492', 'name': 'S4'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 133,\n",
       "    'interactions': 0,\n",
       "    'matches': 74736,\n",
       "    'pathways': 0,\n",
       "    'proteins': 74726,\n",
       "    'proteomes': 10712,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 64031, 'rosettafold': 0},\n",
       "    'structures': 1150,\n",
       "    'taxa': 72291},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 66411,\n",
       "    'alignment:full': 12082,\n",
       "    'alignment:seed': 392},\n",
       "   'cross_references': {}}},\n",
       " 'PF01330': {'metadata': {'accession': 'PF01330',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013849',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RuvA N terminal domain', 'short': 'RuvA_N'},\n",
       "   'description': ['<p>The N terminal domain of RuvA has an OB-fold structure. This domain forms the RuvA tetramer contacts [PMID:8832889].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005222': {'PMID': 8832889,\n",
       "     'ISBN': None,\n",
       "     'volume': '274',\n",
       "     'issue': '5286',\n",
       "     'year': 1996,\n",
       "     'title': 'Crystal structure of DNA recombination protein RuvA and a model for its binding to the Holliday junction.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '415-21',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Rafferty JB',\n",
       "      'Sedelnikova SE',\n",
       "      'Hargreaves D',\n",
       "      'Artymiuk PJ',\n",
       "      'Baker PJ',\n",
       "      'Sharples GJ',\n",
       "      'Mahdi AA',\n",
       "      'Lloyd RG',\n",
       "      'Rice DW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.274.5286.415'}},\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 41,\n",
       "    'interactions': 0,\n",
       "    'matches': 43227,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43227,\n",
       "    'proteomes': 8448,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37075, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 35987},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39192,\n",
       "    'alignment:full': 7785,\n",
       "    'alignment:seed': 730},\n",
       "   'cross_references': {}}},\n",
       " 'PF06968': {'metadata': {'accession': 'PF06968',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010722',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Biotin and Thiamin Synthesis associated domain',\n",
       "    'short': 'BATS'},\n",
       "   'description': ['<p>Biotin synthase (BioB), EC:2.8.1.6 , catalyses the last step of the biotin biosynthetic pathway.  The reaction consists in the introduction of a sulphur atom into dethiobiotin. BioB functions as a homodimer [PMID:12482614].  Thiamin synthesis if a complex process involving at least six gene products (ThiFSGH, ThiI and ThiJ). Two of the proteins required for the biosynthesis of the thiazole moiety of thiamine (vitamin B(1)) are ThiG and ThiH (this family) and form a heterodimer[PMID:12650933].  Both of these reactions are thought of involve the binding of co-factors, and both function as dimers [PMID:12482614, PMID:12650933]. This domain therefore may be involved in co-factor binding or dimerisation (Finn, RD personal observation).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012954': {'PMID': 12482614,\n",
       "     'ISBN': None,\n",
       "     'volume': '532',\n",
       "     'issue': '3',\n",
       "     'year': 2002,\n",
       "     'title': 'The PLP-dependent biotin synthase from Escherichia coli: mechanistic studies.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '465-8',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Ollagnier-de-Choudens S', 'Mulliez E', 'Fontecave M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(02)03733-X'},\n",
       "    'PUB00012955': {'PMID': 12650933,\n",
       "     'ISBN': None,\n",
       "     'volume': '539',\n",
       "     'issue': '1-3',\n",
       "     'year': 2003,\n",
       "     'title': 'Thiamine biosynthesis in Escherichia coli: isolation and initial characterisation of the ThiGH complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '95-9',\n",
       "     'medline_journal': 'FEBS Lett',\n",
       "     'ISO_journal': 'FEBS Lett.',\n",
       "     'authors': ['Leonardi R',\n",
       "      'Fairhurst SA',\n",
       "      'Kriek M',\n",
       "      'Lowe DJ',\n",
       "      'Roach PL.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0014-5793(03)00204-7'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 67,\n",
       "    'interactions': 0,\n",
       "    'matches': 40635,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40631,\n",
       "    'proteomes': 6915,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35876, 'rosettafold': 0},\n",
       "    'structures': 35,\n",
       "    'taxa': 29647},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38608,\n",
       "    'alignment:full': 8380,\n",
       "    'alignment:seed': 680},\n",
       "   'cross_references': {}}},\n",
       " 'PF13399': {'metadata': {'accession': 'PF13399',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR027381',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LytR cell envelope-related transcriptional attenuator',\n",
       "    'short': 'LytR_C'},\n",
       "   'description': ['<p>This family appears at the C-terminus of members of the LytR_cpsA_psr, Pfam:PF03816, family</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 128,\n",
       "    'interactions': 0,\n",
       "    'matches': 26239,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26133,\n",
       "    'proteomes': 2439,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22683, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 10544},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24219,\n",
       "    'alignment:full': 5727,\n",
       "    'alignment:seed': 521},\n",
       "   'cross_references': {}}},\n",
       " 'PF01297': {'metadata': {'accession': 'PF01297',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006127',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Zinc-uptake complex component A periplasmic',\n",
       "    'short': 'ZnuA'},\n",
       "   'description': ['<p>ZnuA includes periplasmic solute binding proteins such as TroA that interacts with an ATP-binding cassette transport system in Treponema pallidum [PMID:10404217]. ZnuA is part of the bacterial zinc-uptake complex ZnuABC, whose components are the following families, ZinT, Pfam:PF09223, Pfam:PF00950, Pfam:PF00005, all of which are regulated by the transcription-regulator family FUR, Pfam:PF01475. ZinT acts as a Zn2+-buffering protein that delivers Zn2+ to ZnuA (TroA), a high-affinity zinc-uptake protein.  In Gram-negative bacteria the ZnuABC transporter system ensures an adequate import of zinc in Zn2+-poor environments, such as those encountered by pathogens within the infected host [PMID:21338480, PMID:24128931].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020311': {'PMID': 10404217,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '7',\n",
       "     'year': 1999,\n",
       "     'title': 'Treponema pallidum TroA is a periplasmic zinc-binding protein with a helical backbone.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '628-33',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Lee YH',\n",
       "      'Deka RK',\n",
       "      'Norgard MV',\n",
       "      'Radolf JD',\n",
       "      'Hasemann CA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/10677'}},\n",
       "   'set_info': {'accession': 'CL0043', 'name': 'Chelatase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 82,\n",
       "    'interactions': 0,\n",
       "    'matches': 60232,\n",
       "    'pathways': 0,\n",
       "    'proteins': 59902,\n",
       "    'proteomes': 6868,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 52158, 'rosettafold': 0},\n",
       "    'structures': 84,\n",
       "    'taxa': 29748},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 57351,\n",
       "    'alignment:full': 10474,\n",
       "    'alignment:seed': 945},\n",
       "   'cross_references': {}}},\n",
       " 'PF01504': {'metadata': {'accession': 'PF01504',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002498',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Phosphatidylinositol-4-phosphate 5-Kinase',\n",
       "    'short': 'PIP5K'},\n",
       "   'description': ['<p>This family contains a region from the common kinase core found in the type I phosphatidylinositol-4-phosphate 5-kinase (PIP5K) family as described in [PMID:9535851]. The family consists of various type I, II and III PIP5K enzymes. PIP5K catalyses the formation of phosphoinositol-4,5-bisphosphate via the  phosphorylation of phosphatidylinositol-4-phosphate a precursor in the  phosphinositide signaling pathway.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002977': {'PMID': 8955136,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '51',\n",
       "     'year': 1996,\n",
       "     'title': 'Type I phosphatidylinositol-4-phosphate 5-kinases are distinct members of this novel lipid kinase family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '32937-43',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Loijens JC', 'Anderson RA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.51.32937'},\n",
       "    'PUB00003041': {'PMID': 9535851,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '15',\n",
       "     'year': 1998,\n",
       "     'title': 'Type I phosphatidylinositol-4-phosphate 5-kinases. Cloning of the third isoform and deletion/substitution analysis of members of this novel lipid kinase family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '8741-8',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Ishihara H',\n",
       "      'Shibasaki Y',\n",
       "      'Kizuki N',\n",
       "      'Wada T',\n",
       "      'Yazaki Y',\n",
       "      'Asano T',\n",
       "      'Oka Y.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.15.8741'}},\n",
       "   'set_info': {'accession': 'CL0016', 'name': 'PKinase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 423,\n",
       "    'interactions': 0,\n",
       "    'matches': 29025,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26252,\n",
       "    'proteomes': 2171,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15961, 'rosettafold': 0},\n",
       "    'structures': 51,\n",
       "    'taxa': 7969},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24354,\n",
       "    'alignment:full': 16009,\n",
       "    'alignment:seed': 332},\n",
       "   'cross_references': {}}},\n",
       " 'PF06050': {'metadata': {'accession': 'PF06050',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010327',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '2-hydroxyglutaryl-CoA dehydratase, D-component',\n",
       "    'short': 'HGD-D'},\n",
       "   'description': [\"<p>Degradation of glutamate via the hydroxyglutarate pathway involves the syn-elimination of water from 2-hydroxyglutaryl-CoA. This anaerobic process is catalysed by 2-hydroxyglutaryl-CoA dehydratase, an enzyme with two components (A and D) that reversibly associate during reaction cycles. This component contains one non-reducible [4Fe-4S]2+ cluster and a reduced riboflavin 5'-monophosphate [PMID:11980491].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012237': {'PMID': 11980491,\n",
       "     'ISBN': None,\n",
       "     'volume': '41',\n",
       "     'issue': '18',\n",
       "     'year': 2002,\n",
       "     'title': 'Adenosine triphosphate-induced electron transfer in 2-hydroxyglutaryl-CoA dehydratase from Acidaminococcus fermentans.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5873-82',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Hans M',\n",
       "      'Bill E',\n",
       "      'Cirpus I',\n",
       "      'Pierik AJ',\n",
       "      'Hetzel M',\n",
       "      'Alber D',\n",
       "      'Buckel W.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi020033m'}},\n",
       "   'set_info': {'accession': 'CL0108', 'name': 'Actin_ATPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 32,\n",
       "    'interactions': 0,\n",
       "    'matches': 20876,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20461,\n",
       "    'proteomes': 1597,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 13745, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 7606},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18212,\n",
       "    'alignment:full': 3486,\n",
       "    'alignment:seed': 335},\n",
       "   'cross_references': {}}},\n",
       " 'PF01817': {'metadata': {'accession': 'PF01817',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002701',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Chorismate mutase type II', 'short': 'CM_2'},\n",
       "   'description': ['<p>Chorismate mutase EC:5.4.99.5 catalyses the conversion of chorismate to prephenate in the pathway of tyrosine and phenylalanine biosynthesis. This enzyme is negatively regulated by tyrosine, tryptophan and phenylalanine [PMID:9642265, PMID:9497350].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003018': {'PMID': 9642265,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '27',\n",
       "     'year': 1998,\n",
       "     'title': 'Tyrosine and tryptophan act through the same binding site at the dimer interface of yeast chorismate mutase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17012-7',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Schnappauf G', 'Krappmann S', 'Braus GH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.27.17012'},\n",
       "    'PUB00003037': {'PMID': 9497350,\n",
       "     'ISBN': None,\n",
       "     'volume': '273',\n",
       "     'issue': '11',\n",
       "     'year': 1998,\n",
       "     'title': 'Chorismate mutase-prephenate dehydratase from Escherichia coli. Study of catalytic and regulatory domains using genetically engineered proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6248-53',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Zhang S',\n",
       "      'Pohnert G',\n",
       "      'Kongsaeree P',\n",
       "      'Wilson DB',\n",
       "      'Clardy J',\n",
       "      'Ganem B.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.273.11.6248'},\n",
       "    'PUB00004830': {'PMID': 7971967,\n",
       "     'ISBN': None,\n",
       "     'volume': '91',\n",
       "     'issue': '23',\n",
       "     'year': 1994,\n",
       "     'title': 'The crystal structure of allosteric chorismate mutase at 2.2-A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '10814-8',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Xue Y', 'Lipscomb WN', 'Graf R', 'Schnappauf G', 'Braus G.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=7971967&action=stream&blobtype=pdf'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 151,\n",
       "    'interactions': 0,\n",
       "    'matches': 52375,\n",
       "    'pathways': 0,\n",
       "    'proteins': 52372,\n",
       "    'proteomes': 7999,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 45699, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 33300},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 49409,\n",
       "    'alignment:full': 9719,\n",
       "    'alignment:seed': 775},\n",
       "   'cross_references': {}}},\n",
       " 'PF07556': {'metadata': {'accession': 'PF07556',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011435',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1538)',\n",
       "    'short': 'DUF1538'},\n",
       "   'description': ['<p>This family contains several conserved glycines and phenylalanines.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 13,\n",
       "    'interactions': 0,\n",
       "    'matches': 8918,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6428,\n",
       "    'proteomes': 840,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5200, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 4156},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 7301,\n",
       "    'alignment:full': 1616,\n",
       "    'alignment:seed': 229},\n",
       "   'cross_references': {}}},\n",
       " 'PF13561': {'metadata': {'accession': 'PF13561',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Enoyl-(Acyl carrier protein) reductase',\n",
       "    'short': 'adh_short_C2'},\n",
       "   'description': ['<p>This domain is found in Enoyl-(Acyl carrier protein)  reductases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0063', 'name': 'NADP_Rossmann'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1693,\n",
       "    'interactions': 0,\n",
       "    'matches': 786281,\n",
       "    'pathways': 0,\n",
       "    'proteins': 784017,\n",
       "    'proteomes': 11045,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 677995, 'rosettafold': 0},\n",
       "    'structures': 898,\n",
       "    'taxa': 47336},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 733596,\n",
       "    'alignment:full': 181249,\n",
       "    'alignment:seed': 315},\n",
       "   'cross_references': {}}},\n",
       " 'PF04020': {'metadata': {'accession': 'PF04020',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007165',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Mycobacterial 4 TMS phage holin, superfamily IV',\n",
       "    'short': 'Phage_holin_4_2'},\n",
       "   'description': ['<p>These proteins are predicted transmembrane proteins with probably four transmembrane spans. The 1.E.40 is represented by the mycobacterial 4 phage holin, but it also contains many cyanobacterial. proteobacterial and firmicute proteins. Holins are encoded within the genomes of Gram-positive and Gram-negative bacteria as well as in those of the bacteriophage of these organisms. The primary function of holins appears to be transport of murein hydrolases across the cytoplasmic membrane to the cell wall where these enzymes hydrolyse the cell wall polymer as a prelude to cell lysis. When chromosomally encoded the enzymes are therefore autolysins. Holins may also facilitate leakage of electrolytes and nutrients from the cell cytoplasm, thereby promoting cell death. Some may catalyse export of nucleases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 16,\n",
       "    'interactions': 0,\n",
       "    'matches': 22058,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22054,\n",
       "    'proteomes': 4143,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 18677, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 17166},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19424,\n",
       "    'alignment:full': 4713,\n",
       "    'alignment:seed': 420},\n",
       "   'cross_references': {}}},\n",
       " 'PF07703': {'metadata': {'accession': 'PF07703',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011625',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Alpha-2-macroglobulin bait region domain',\n",
       "    'short': 'A2M_BRD'},\n",
       "   'description': ['<p>Alpha-2-macroglobulins (A2Ms) are plasma proteins that trap and inhibit a broad range of proteases and are major components of the eukaryotic innate immune system. However, A2M-like proteins were identified in pathogenically invasive bacteria and species that colonize higher eukaryotes. This domain is found in eukaryotic and bacterial proteins. In human A2Ms, this domain encompasses macroglobulin-like domain MG5 and 6 including bait region. In Salmonella enterica ser A2Ms, this domain encompasses MG7 and MG8 including the bait region [PMID:25221932] [PMID:22290936]. The Bait region is cleaved by proteases, followed by a large conformational change that blocks the target protease within a cage-like complex. This model of protease entrapment is recognised as the Venus flytrap mechanism [PMID:25221932].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1123,\n",
       "    'interactions': 0,\n",
       "    'matches': 32684,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32555,\n",
       "    'proteomes': 3353,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4092, 'rosettafold': 0},\n",
       "    'structures': 95,\n",
       "    'taxa': 16188},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27097,\n",
       "    'alignment:full': 8356,\n",
       "    'alignment:seed': 527},\n",
       "   'cross_references': {}}},\n",
       " 'PF07562': {'metadata': {'accession': 'PF07562',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011500',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Nine Cysteines Domain of family 3 GPCR',\n",
       "    'short': 'NCD3G'},\n",
       "   'description': ['<p>This conserved sequence contains several highly-conserved Cys residues that are predicted to form disulphide bridges. It is predicted to lie outside the cell membrane, tethered to the Pfam:PF00003 in several receptor proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0607', 'name': 'TNF_receptor'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 202,\n",
       "    'interactions': 0,\n",
       "    'matches': 26827,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26619,\n",
       "    'proteomes': 872,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17534, 'rosettafold': 0},\n",
       "    'structures': 62,\n",
       "    'taxa': 3535},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18535,\n",
       "    'alignment:full': 12738,\n",
       "    'alignment:seed': 429},\n",
       "   'cross_references': {}}},\n",
       " 'PF16640': {'metadata': {'accession': 'PF16640',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR032109',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial Ig-like domain (group 3)', 'short': 'Big_3_5'},\n",
       "   'description': ['<p>This family consists of bacterial domains with an Ig-like fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0159', 'name': 'E-set'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1700,\n",
       "    'interactions': 0,\n",
       "    'matches': 16248,\n",
       "    'pathways': 0,\n",
       "    'proteins': 7312,\n",
       "    'proteomes': 770,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 4620, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 3267},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14231,\n",
       "    'alignment:full': 4575,\n",
       "    'alignment:seed': 281},\n",
       "   'cross_references': {}}},\n",
       " 'PF05161': {'metadata': {'accession': 'PF05161',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007835',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MOFRL family', 'short': 'MOFRL'},\n",
       "   'description': ['<p>MOFRL(multi-organism fragment with rich Leucine) family exists in bacteria and eukaryotes. The function of this domain is not clear, although it  exists in some putative enzymes such as reductases and kinases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 15109,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15101,\n",
       "    'proteomes': 3006,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12681, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 14124},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13318,\n",
       "    'alignment:full': 2776,\n",
       "    'alignment:seed': 288},\n",
       "   'cross_references': {}}},\n",
       " 'PF00137': {'metadata': {'accession': 'PF00137',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002379',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ATP synthase subunit C', 'short': 'ATP-synt_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 237,\n",
       "    'interactions': 0,\n",
       "    'matches': 95923,\n",
       "    'pathways': 0,\n",
       "    'proteins': 81028,\n",
       "    'proteomes': 11073,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 69508, 'rosettafold': 0},\n",
       "    'structures': 331,\n",
       "    'taxa': 66399},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 82416,\n",
       "    'alignment:full': 23587,\n",
       "    'alignment:seed': 929},\n",
       "   'cross_references': {}}},\n",
       " 'PF12002': {'metadata': {'accession': 'PF12002',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021886',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MgsA AAA+ ATPase C terminal', 'short': 'MgsA_C'},\n",
       "   'description': ['<p>The MgsA protein possesses DNA-dependent ATPase and ssDNA annealing activities [PMID:15743409]. MgsA contributes to the recovery of stalled replication forks and therefore prevents genomic instability caused by aberrant DNA replication [PMID:15743409]. Additionally, MgsA may play a role in chromosomal segregation [PMID:15743409]. This is consistent with a report that MgsA co-localises with the replisome and affects chromosome segregation [PMID:15743409].  This domain represents the C terminal region of MgsA.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0604', 'name': 'post-AAA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 47263,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47242,\n",
       "    'proteomes': 9965,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 40598, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 39822},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 43356,\n",
       "    'alignment:full': 9403,\n",
       "    'alignment:seed': 598},\n",
       "   'cross_references': {}}},\n",
       " 'PF02132': {'metadata': {'accession': 'PF02132',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023628',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RecR protein', 'short': 'RecR'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 42,\n",
       "    'interactions': 0,\n",
       "    'matches': 40858,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40856,\n",
       "    'proteomes': 8329,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35035, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 35366},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 37029,\n",
       "    'alignment:full': 7662,\n",
       "    'alignment:seed': 748},\n",
       "   'cross_references': {}}},\n",
       " 'PF04304': {'metadata': {'accession': 'PF04304',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007401',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF454)', 'short': 'DUF454'},\n",
       "   'description': ['<p>Predicted membrane protein.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 34,\n",
       "    'interactions': 0,\n",
       "    'matches': 15706,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15706,\n",
       "    'proteomes': 2506,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13792, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12792},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15433,\n",
       "    'alignment:full': 2544,\n",
       "    'alignment:seed': 234},\n",
       "   'cross_references': {}}},\n",
       " 'PF01268': {'metadata': {'accession': 'PF01268',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000559',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Formate--tetrahydrofolate ligase', 'short': 'FTHFS'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 110,\n",
       "    'interactions': 0,\n",
       "    'matches': 32720,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32366,\n",
       "    'proteomes': 5628,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26814, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 24758},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 28571,\n",
       "    'alignment:full': 6712,\n",
       "    'alignment:seed': 386},\n",
       "   'cross_references': {}}},\n",
       " 'PF02322': {'metadata': {'accession': 'PF02322',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003317',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cytochrome bd terminal oxidase subunit II',\n",
       "    'short': 'Cyt_bd_oxida_II'},\n",
       "   'description': ['<p>This family consists of cytochrome bd type terminal oxidases that catalyse quinol-dependent, Na+-independent oxygen uptake [PMID:8626304]. Members of this family are integral membrane proteins and contain a protohaem IX centre B558. One member of the family Swiss:O05192 is implicated in having an important role in micro-aerobic nitrogen fixation in the enteric bacterium Klebsiella pneumoniae [PMID:9274021]. The family forms an integral functional unit with subunit I, family Bac_Ubq_Cox, Pfam:PF01654.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003598': {'PMID': 9274021,\n",
       "     'ISBN': None,\n",
       "     'volume': '143 ( Pt 8)',\n",
       "     'issue': None,\n",
       "     'year': 1997,\n",
       "     'title': \"The Klebsiella pneumoniae cytochrome bd' terminal oxidase complex and its role in microaerobic nitrogen fixation.\",\n",
       "     'URL': None,\n",
       "     'raw_pages': '2673-83',\n",
       "     'medline_journal': 'Microbiology',\n",
       "     'ISO_journal': 'Microbiology (Reading, Engl.)',\n",
       "     'authors': ['Juty NS', 'Moshiri F', 'Merrick M', 'Anthony C', 'Hill S.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00008048': {'PMID': 8626304,\n",
       "     'ISBN': None,\n",
       "     'volume': '178',\n",
       "     'issue': '6',\n",
       "     'year': 1996,\n",
       "     'title': 'Purification of a cytochrome bd terminal oxidase encoded by the Escherichia coli app locus from a delta cyo delta cyd strain complemented by genes from Bacillus firmus OF4.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1742-9',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Sturr MG', 'Krulwich TA', 'Hicks DB.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/178/6/1742'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 28,\n",
       "    'interactions': 0,\n",
       "    'matches': 36047,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36021,\n",
       "    'proteomes': 5867,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32350, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 24458},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35695,\n",
       "    'alignment:full': 7112,\n",
       "    'alignment:seed': 581},\n",
       "   'cross_references': {}}},\n",
       " 'PF02912': {'metadata': {'accession': 'PF02912',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004188',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aminoacyl tRNA synthetase class II, N-terminal domain',\n",
       "    'short': 'Phe_tRNA-synt_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020104': {'PMID': 7664121,\n",
       "     'ISBN': None,\n",
       "     'volume': '2',\n",
       "     'issue': '7',\n",
       "     'year': 1995,\n",
       "     'title': 'Structure of phenylalanyl-tRNA synthetase from Thermus thermophilus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '537-47',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Mosyak L',\n",
       "      'Reshetnikova L',\n",
       "      'Goldgur Y',\n",
       "      'Delarue M',\n",
       "      'Safro MG.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nsb0795-537'}},\n",
       "   'set_info': {'accession': 'CL0298', 'name': 'tRNA_bind_arm'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 39,\n",
       "    'interactions': 0,\n",
       "    'matches': 43098,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43097,\n",
       "    'proteomes': 8560,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 36955, 'rosettafold': 0},\n",
       "    'structures': 29,\n",
       "    'taxa': 36111},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39302,\n",
       "    'alignment:full': 7884,\n",
       "    'alignment:seed': 711},\n",
       "   'cross_references': {}}},\n",
       " 'PF01535': {'metadata': {'accession': 'PF01535',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002885',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PPR repeat', 'short': 'PPR'},\n",
       "   'description': ['<p>This repeat has no known function. It is about 35 amino acids long and found in up to 18 copies in some proteins.  This family appears to be greatly expanded in plants. This repeat occurs in PET309 Swiss:P32522 that may be involved in RNA stabilisation [PMID:7664742]. This domain occurs in crp1 that is involved in RNA processing [PMID:8039510].  This repeat is associated with a predicted plant protein Swiss:O49549 that has a domain organisation similar to the human BRCA1 protein. The repeat has been called PPR [PMID:10664580].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001258': {'PMID': 8039510,\n",
       "     'ISBN': None,\n",
       "     'volume': '13',\n",
       "     'issue': '13',\n",
       "     'year': 1994,\n",
       "     'title': 'A nuclear mutation in maize blocks the processing and translation of several chloroplast mRNAs and provides evidence for the differential translation of alternative mRNA forms.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3170-81',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Barkan A', 'Walker M', 'Nolasco M', 'Johnson D.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=8039510&action=stream&blobtype=pdf'},\n",
       "    'PUB00001271': {'PMID': 7664742,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '16',\n",
       "     'year': 1995,\n",
       "     'title': 'The product of the nuclear gene PET309 is required for translation of mature mRNA and stability or production of intron-containing RNAs derived from the mitochondrial COX1 locus of Saccharomyces cerevisiae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4031-43',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Manthey GM', 'McEwen JE.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=7664742&action=stream&blobtype=pdf'},\n",
       "    'PUB00017589': {'PMID': 10664580,\n",
       "     'ISBN': None,\n",
       "     'volume': '25',\n",
       "     'issue': '2',\n",
       "     'year': 2000,\n",
       "     'title': 'The PPR motif - a TPR-related motif prevalent in plant organellar proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '46-7',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Small ID', 'Peeters N.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(99)01520-0'}},\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 26945,\n",
       "    'interactions': 0,\n",
       "    'matches': 500289,\n",
       "    'pathways': 0,\n",
       "    'proteins': 163667,\n",
       "    'proteomes': 2068,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 135106, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 9523},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 380234,\n",
       "    'alignment:full': 253857,\n",
       "    'alignment:seed': 458},\n",
       "   'cross_references': {}}},\n",
       " 'PF02138': {'metadata': {'accession': 'PF02138',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000409',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Beige/BEACH domain', 'short': 'Beach'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 498,\n",
       "    'interactions': 0,\n",
       "    'matches': 21283,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20711,\n",
       "    'proteomes': 2097,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 4526, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 7681},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15588,\n",
       "    'alignment:full': 9688,\n",
       "    'alignment:seed': 271},\n",
       "   'cross_references': {}}},\n",
       " 'PF10135': {'metadata': {'accession': 'PF10135',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019301',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Rod binding protein', 'short': 'Rod-binding'},\n",
       "   'description': ['<p>Members of this family are involved in the assembly of the prokaryotic flagellar rod.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 16,\n",
       "    'interactions': 0,\n",
       "    'matches': 15324,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15323,\n",
       "    'proteomes': 2809,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13421, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 12920},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14782,\n",
       "    'alignment:full': 2804,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF08436': {'metadata': {'accession': 'PF08436',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013644',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '1-deoxy-D-xylulose 5-phosphate reductoisomerase C-terminal domain',\n",
       "    'short': 'DXP_redisom_C'},\n",
       "   'description': ['<p>This domain is found to the C-terminus of Pfam:PF02670 domains in bacterial and plant 1-deoxy-D-xylulose 5-phosphate reductoisomerases which catalyse the formation of 2-C-methyl-D-erythritol 4-phosphate from 1-deoxy-D-xylulose-5-phosphate in the presence of NADPH [PMID:9707569].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020977': {'PMID': 9707569,\n",
       "     'ISBN': None,\n",
       "     'volume': '95',\n",
       "     'issue': '17',\n",
       "     'year': 1998,\n",
       "     'title': 'A 1-deoxy-D-xylulose 5-phosphate reductoisomerase catalyzing the formation of 2-C-methyl-D-erythritol 4-phosphate in an alternative nonmevalonate pathway for terpenoid biosynthesis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '9879-84',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Takahashi S', 'Kuzuyama T', 'Watanabe H', 'Seto H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.95.17.9879'}},\n",
       "   'set_info': {'accession': 'CL0139', 'name': 'GADPH_aa-bio_dh'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 69,\n",
       "    'interactions': 0,\n",
       "    'matches': 38894,\n",
       "    'pathways': 0,\n",
       "    'proteins': 38886,\n",
       "    'proteomes': 7594,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33281, 'rosettafold': 0},\n",
       "    'structures': 70,\n",
       "    'taxa': 31589},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35884,\n",
       "    'alignment:full': 7124,\n",
       "    'alignment:seed': 457},\n",
       "   'cross_references': {}}},\n",
       " 'PF03030': {'metadata': {'accession': 'PF03030',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004131',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Inorganic H+ pyrophosphatase', 'short': 'H_PPase'},\n",
       "   'description': ['<p>The H+ pyrophosphatase is an transmembrane proton pump involved in establishing the H+ electrochemical potential difference between the vacuole lumen and the cell cytosol. Vacuolar-type H(+)-translocating inorganic pyrophosphatases have long been considered to be restricted to plants and to a few species of photo-trophic bacteria. However, in recent investigations, these pyrophosphatases have been found in organisms as disparate as thermophilic Archaea and parasitic protists [PMID:11335173].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019667': {'PMID': 11335173,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '5',\n",
       "     'year': 2001,\n",
       "     'title': 'Vacuolar H(+) pyrophosphatases: from the evolutionary backwaters into the mainstream.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '206-11',\n",
       "     'medline_journal': 'Trends Plant Sci',\n",
       "     'ISO_journal': 'Trends Plant Sci.',\n",
       "     'authors': ['Drozdowicz YM', 'Rea PA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S1360-1385(01)01923-9'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 140,\n",
       "    'interactions': 0,\n",
       "    'matches': 33687,\n",
       "    'pathways': 0,\n",
       "    'proteins': 33198,\n",
       "    'proteomes': 4583,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 26748, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 20283},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 27888,\n",
       "    'alignment:full': 5979,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF08494': {'metadata': {'accession': 'PF08494',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013701',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DEAD/H associated', 'short': 'DEAD_assoc'},\n",
       "   'description': ['<p>This domain is found in ATP-dependent helicases as well as a number of hypothetical proteins together with the helicase conserved C-terminal domain (Pfam:PF00270) and the Pfam:PF00271 domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 79,\n",
       "    'interactions': 0,\n",
       "    'matches': 22745,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22735,\n",
       "    'proteomes': 4143,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11674, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 16388},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21169,\n",
       "    'alignment:full': 4006,\n",
       "    'alignment:seed': 283},\n",
       "   'cross_references': {}}},\n",
       " 'PF03729': {'metadata': {'accession': 'PF03729',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005325',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Short repeat of unknown function (DUF308)',\n",
       "    'short': 'DUF308'},\n",
       "   'description': ['<p>Family of short repeats that occurs in a limited number of membrane proteins. It may divide further in short repeats of around 7-10 residues of the pattern G-#-X(2)-#(2)-X  (#=hydrophobic).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 56,\n",
       "    'interactions': 0,\n",
       "    'matches': 51602,\n",
       "    'pathways': 0,\n",
       "    'proteins': 21884,\n",
       "    'proteomes': 3614,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 19397, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 15668},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48881,\n",
       "    'alignment:full': 11076,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF12673': {'metadata': {'accession': 'PF12673',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR024300',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF3794)',\n",
       "    'short': 'DUF3794'},\n",
       "   'description': ['<p>This presumed domain is functionally uncharacterised. This domain family is found in bacteria, and is approximately 90 amino acids in length. The family is found in association with Pfam:PF01476.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 25,\n",
       "    'interactions': 0,\n",
       "    'matches': 7009,\n",
       "    'pathways': 0,\n",
       "    'proteins': 3084,\n",
       "    'proteomes': 615,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 2599, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 2048},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 6380,\n",
       "    'alignment:full': 1775,\n",
       "    'alignment:seed': 376},\n",
       "   'cross_references': {}}},\n",
       " 'PF01455': {'metadata': {'accession': 'PF01455',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001109',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HupF/HypC family', 'short': 'HupF_HypC'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 31,\n",
       "    'interactions': 0,\n",
       "    'matches': 16409,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16409,\n",
       "    'proteomes': 2253,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14039, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 12025},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14451,\n",
       "    'alignment:full': 2786,\n",
       "    'alignment:seed': 361},\n",
       "   'cross_references': {}}},\n",
       " 'PF12833': {'metadata': {'accession': 'PF12833',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR018060',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Helix-turn-helix domain', 'short': 'HTH_18'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1020,\n",
       "    'interactions': 0,\n",
       "    'matches': 638461,\n",
       "    'pathways': 0,\n",
       "    'proteins': 637831,\n",
       "    'proteomes': 7983,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 557871, 'rosettafold': 0},\n",
       "    'structures': 29,\n",
       "    'taxa': 32731},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 634110,\n",
       "    'alignment:full': 130034,\n",
       "    'alignment:seed': 249},\n",
       "   'cross_references': {}}},\n",
       " 'PF06949': {'metadata': {'accession': 'PF06949',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009711',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1292)',\n",
       "    'short': 'DUF1292'},\n",
       "   'description': ['<p>This family consists of several hypothetical bacterial proteins of around 90 residues in length. The function of this family is unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 15,\n",
       "    'interactions': 0,\n",
       "    'matches': 11229,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11175,\n",
       "    'proteomes': 1913,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9771, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 7736},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10506,\n",
       "    'alignment:full': 2927,\n",
       "    'alignment:seed': 242},\n",
       "   'cross_references': {}}},\n",
       " 'PF03457': {'metadata': {'accession': 'PF03457',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005114',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Helicase associated domain', 'short': 'HA'},\n",
       "   'description': ['<p>This short domain is found in multiple copies in bacterial helicase proteins. The domain is predicted to contain 3 alpha helices.  The function of this domain may be to bind nucleic acid.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016669': {'PMID': 12625841,\n",
       "     'ISBN': None,\n",
       "     'volume': '3',\n",
       "     'issue': None,\n",
       "     'year': 2003,\n",
       "     'title': 'New knowledge from old: in silico discovery of novel protein domains in Streptomyces coelicolor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3',\n",
       "     'medline_journal': 'BMC Microbiol',\n",
       "     'ISO_journal': 'BMC Microbiol.',\n",
       "     'authors': ['Yeats C', 'Bentley S', 'Bateman A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/1471-2180-3-3'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 232,\n",
       "    'interactions': 0,\n",
       "    'matches': 15385,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6171,\n",
       "    'proteomes': 560,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 5595, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 2942},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14753,\n",
       "    'alignment:full': 4198,\n",
       "    'alignment:seed': 236},\n",
       "   'cross_references': {}}},\n",
       " 'PF02654': {'metadata': {'accession': 'PF02654',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003805',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cobalamin-5-phosphate synthase', 'short': 'CobS'},\n",
       "   'description': [\"<p>This is family of Colbalmin-5-phosphate synthases, CobS, from bacteria. The CobS enzyme catalyses the synthesis of AdoCbl-5'-p from AdoCbi-GDP  and alpha-ribazole-5'-P [PMID:10518530].  This enzyme is involved in the cobalamin (vitamin B12) biosynthesis pathway in particular the nucleotide loop  assembly stage in conjunction with CobC, CobU and CobT [PMID:10518530].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008256': {'PMID': 10518530,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '21',\n",
       "     'year': 1999,\n",
       "     'title': 'In vitro synthesis of the nucleotide loop of cobalamin by Salmonella typhimurium enzymes.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '11798-803',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Maggio-Hall LA', 'Escalante-Semerena JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.96.21.11798'},\n",
       "    'PUB00015329': {'PMID': 8501034,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '11',\n",
       "     'year': 1993,\n",
       "     'title': 'Characterization of the cobalamin (vitamin B12) biosynthetic genes of Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3303-16',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Roth JR',\n",
       "      'Lawrence JG',\n",
       "      'Rubenfield M',\n",
       "      'Kieffer-Higgins S',\n",
       "      'Church GM.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8501034'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 23,\n",
       "    'interactions': 0,\n",
       "    'matches': 23786,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23770,\n",
       "    'proteomes': 5031,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20912, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 21924},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 22388,\n",
       "    'alignment:full': 4735,\n",
       "    'alignment:seed': 582},\n",
       "   'cross_references': {}}},\n",
       " 'PF08592': {'metadata': {'accession': 'PF08592',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013901',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Anthrone oxygenase', 'short': 'Anthrone_oxy'},\n",
       "   'description': ['<p>This family consists of anthrone oxygenases found in bacteria and fungi, and involved in the synthesis of different products. GedH from Aspergillus terreus is part of the gene cluster that mediates the biosynthesis of geodin [PMID:24009710], EncC from Neosartorya fumigata is involved in the biosynthesis of endocrocin [PMID:22492455], and MdpH from Aspergillus nidulans in the biosynthesis of monodictyphenone [PMID:21351751].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 65,\n",
       "    'interactions': 0,\n",
       "    'matches': 10859,\n",
       "    'pathways': 0,\n",
       "    'proteins': 10858,\n",
       "    'proteomes': 2246,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 9400, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 8955},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 9824,\n",
       "    'alignment:full': 3417,\n",
       "    'alignment:seed': 226},\n",
       "   'cross_references': {}}},\n",
       " 'PF01239': {'metadata': {'accession': 'PF01239',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002088',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein prenyltransferase alpha subunit repeat',\n",
       "    'short': 'PPTA'},\n",
       "   'description': [\"<p>Both farnesyltransferase (FT) and geranylgeranyltransferase 1 (GGT1) recognise a CaaX motif on their substrates where 'a' stands for preferably aliphatic residues, whereas GGT2 recognises a completely different motif. Important substrates for FT include, amongst others, many members of the Ras superfamily. GGT1 substrates include some of the other small GTPases and GGT2 substrates include the Rab family [PMID:12702202].</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000450': {'PMID': 9843427,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '47',\n",
       "     'year': 1998,\n",
       "     'title': 'Crystal structure of farnesyl protein transferase complexed with a CaaX peptide and farnesyl diphosphate analogue.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '16601-11',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Strickland CL',\n",
       "      'Windsor WT',\n",
       "      'Syto R',\n",
       "      'Wang L',\n",
       "      'Bond R',\n",
       "      'Wu Z',\n",
       "      'Schwartz J',\n",
       "      'Le HV',\n",
       "      'Beese LS',\n",
       "      'Weber PC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi981197z'},\n",
       "    'PUB00020171': {'PMID': 12702202,\n",
       "     'ISBN': None,\n",
       "     'volume': '4',\n",
       "     'issue': '4',\n",
       "     'year': 2003,\n",
       "     'title': 'Protein prenyltransferases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '212',\n",
       "     'medline_journal': 'Genome Biol',\n",
       "     'ISO_journal': 'Genome Biol.',\n",
       "     'authors': ['Maurer-Stroh S', 'Washietl S', 'Eisenhaber F.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1186/gb-2003-4-4-212'}},\n",
       "   'set_info': {'accession': 'CL0020', 'name': 'TPR'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 308,\n",
       "    'interactions': 0,\n",
       "    'matches': 36467,\n",
       "    'pathways': 0,\n",
       "    'proteins': 9762,\n",
       "    'proteomes': 2168,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 8014, 'rosettafold': 0},\n",
       "    'structures': 116,\n",
       "    'taxa': 8152},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29649,\n",
       "    'alignment:full': 18228,\n",
       "    'alignment:seed': 612},\n",
       "   'cross_references': {}}},\n",
       " 'PF03466': {'metadata': {'accession': 'PF03466',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005119',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'LysR substrate binding domain',\n",
       "    'short': 'LysR_substrate'},\n",
       "   'description': ['<p>The structure of this domain is known and is similar to the periplasmic binding proteins [PMID:9309218]. This domain binds a variety of ligands that caries in size and structure, such as amino acids, sugar phosphates, organic acids, metal cations, flavonoids, C6-ring carboxylic acids, H2O2, HOCl, homocysteine, NADPH, ATP, sulphate, muropeptides, acetate, salicylate, citrate, phenol- and quinolone derivatives, acetylserines, fatty acid CoA, shikimate, chorismate, homocysteine, indole-3-acetic acid, Na(I), c-di-GMP, ppGpp and hydrogen peroxide (Matilla et. al., FEMS Microbiology Reviews, fuab043, 45, 2021, 1. https://doi.org/10.1093/femsre/fuab043).</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005283': {'PMID': 9309218,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '8',\n",
       "     'year': 1997,\n",
       "     'title': 'The structure of the cofactor-binding fragment of the LysR family member, CysB: a familiar fold with a surprising subunit arrangement.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1017-32',\n",
       "     'medline_journal': 'Structure',\n",
       "     'ISO_journal': 'Structure',\n",
       "     'authors': ['Tyrrell R',\n",
       "      'Verschueren KH',\n",
       "      'Dodson EJ',\n",
       "      'Murshudov GN',\n",
       "      'Addy C',\n",
       "      'Wilkinson AJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0969-2126(97)00254-2'}},\n",
       "   'set_info': {'accession': 'CL0177', 'name': 'PBP'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 370,\n",
       "    'interactions': 0,\n",
       "    'matches': 846976,\n",
       "    'pathways': 0,\n",
       "    'proteins': 845625,\n",
       "    'proteomes': 8313,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 770340, 'rosettafold': 0},\n",
       "    'structures': 146,\n",
       "    'taxa': 35240},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 877632,\n",
       "    'alignment:full': 146371,\n",
       "    'alignment:seed': 294},\n",
       "   'cross_references': {}}},\n",
       " 'PF12019': {'metadata': {'accession': 'PF12019',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022346',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Type II transport protein GspH', 'short': 'GspH'},\n",
       "   'description': ['<p>GspH is involved in bacterial type II export systems [PMID:18241884]. Like all pilins, GspH has an N terminus alpha helix [PMID:18241884]. This helix is followed by nine beta strands forming two beta sheets, one of five antiparallel strands and one of four antiparallel strands [PMID:18241884]. GspH is a minor pseudopilin; it is expressed much less than other pseudopilins in the type II secretion pilus (major pilins) [PMID:18241884].  The function and localisation of minor pseudo-pilins are still to be fully unraveled [PMID:18241884]. It has been suggested that some minor pseudopilins may assemble either into the base or the tip of pili, or both. They function as initiators or regulators of pilus biogenesis and dynamics, and/or as adaptors between various pseudopilin component and other members of the T2SS [PMID:18241884].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0327', 'name': 'Pilus'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 24,\n",
       "    'interactions': 0,\n",
       "    'matches': 19508,\n",
       "    'pathways': 0,\n",
       "    'proteins': 19508,\n",
       "    'proteomes': 1757,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17007, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 8946},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 18791,\n",
       "    'alignment:full': 3169,\n",
       "    'alignment:seed': 273},\n",
       "   'cross_references': {}}},\n",
       " 'PF01924': {'metadata': {'accession': 'PF01924',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002780',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Hydrogenase formation hypA family', 'short': 'HypD'},\n",
       "   'description': ['<p>HypD is involved in hydrogenase formation. It contains many possible metal binding residues, which may bind to nickel. Transposon Tn5 insertions into hypD resulted in R. leguminosarum mutants that lacked any hydrogenase activity in symbiosis with peas [PMID:8326860].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005713': {'PMID': 8326860,\n",
       "     'ISBN': None,\n",
       "     'volume': '8',\n",
       "     'issue': '3',\n",
       "     'year': 1993,\n",
       "     'title': 'Molecular analysis of a microaerobically induced operon required for hydrogenase synthesis in Rhizobium leguminosarum biovar viciae.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '471-81',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Rey L',\n",
       "      'Murillo J',\n",
       "      'Hernando Y',\n",
       "      'Hidalgo E',\n",
       "      'Cabrera E',\n",
       "      'Imperial J',\n",
       "      'Ruiz-Argueso T.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.1993.tb01591.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 28,\n",
       "    'interactions': 0,\n",
       "    'matches': 14398,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14317,\n",
       "    'proteomes': 2201,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12244, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 11681},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12805,\n",
       "    'alignment:full': 2150,\n",
       "    'alignment:seed': 276},\n",
       "   'cross_references': {}}},\n",
       " 'PF03255': {'metadata': {'accession': 'PF03255',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001095',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Acetyl co-enzyme A carboxylase carboxyltransferase alpha subunit',\n",
       "    'short': 'ACCA'},\n",
       "   'description': ['<p>Acetyl co-enzyme A carboxylase carboxyltransferase is composed of an alpha and beta subunit.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019086': {'PMID': 10470374,\n",
       "     'ISBN': None,\n",
       "     'volume': '39',\n",
       "     'issue': None,\n",
       "     'year': 1999,\n",
       "     'title': 'The regulation of acetyl-CoA carboxylase--a potential target for the action of hypolipidemic agents.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '205-34',\n",
       "     'medline_journal': 'Adv Enzyme Regul',\n",
       "     'ISO_journal': 'Adv. Enzyme Regul.',\n",
       "     'authors': ['Munday MR', 'Hemingway CJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0065-2571(98)00016-8'}},\n",
       "   'set_info': {'accession': 'CL0127', 'name': 'ClpP_crotonase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 31982,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31978,\n",
       "    'proteomes': 6237,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 27543, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 27343},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29467,\n",
       "    'alignment:full': 5979,\n",
       "    'alignment:seed': 285},\n",
       "   'cross_references': {}}},\n",
       " 'PF08542': {'metadata': {'accession': 'PF08542',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013748',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Replication factor C C-terminal domain',\n",
       "    'short': 'Rep_fac_C'},\n",
       "   'description': ['<p>This is the C-terminal domain of RFC (replication factor-C) protein of the clamp loader complex which binds to the DNA sliding clamp (proliferating cell nuclear antigen, PCNA).  The five modules of RFC assemble into a right-handed spiral, which results in only three of the five RFC subunits (RFC-A, RFC-B and RFC-C) making contact with PCNA, leaving a wedge-shaped gap between RFC-E and the PCNA clamp-loader complex. The C-terminal is vital for the correct orientation of RFC-E with respect to RFC-A [PMID:15201901].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0604', 'name': 'post-AAA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 198,\n",
       "    'interactions': 0,\n",
       "    'matches': 15141,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15139,\n",
       "    'proteomes': 2510,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12565, 'rosettafold': 0},\n",
       "    'structures': 36,\n",
       "    'taxa': 10453},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 12326,\n",
       "    'alignment:full': 5871,\n",
       "    'alignment:seed': 316},\n",
       "   'cross_references': {}}},\n",
       " 'PF01679': {'metadata': {'accession': 'PF01679',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000612',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Proteolipid membrane potential modulator',\n",
       "    'short': 'Pmp3'},\n",
       "   'description': ['<p>Pmp3 is an evolutionarily conserved proteolipid in the plasma membrane which, in S. pombe, is transcriptionally regulated by the Spc1 stress MAPK (mitogen-activated protein kinases) pathway. It functions to modulate the membrane potential, particularly to resist high cellular cation concentration. In eukaryotic organisms, stress-activated mitogen-activated protein kinases play crucial roles in transmitting environmental signals that will regulate gene expression for allowing the cell to adapt to cellular stress. Pmp3-like proteins are highly conserved in bacteria, yeast, nematode and plants.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 109,\n",
       "    'interactions': 0,\n",
       "    'matches': 14118,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14064,\n",
       "    'proteomes': 2613,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12278, 'rosettafold': 1},\n",
       "    'structures': 2,\n",
       "    'taxa': 12031},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13006,\n",
       "    'alignment:full': 5271,\n",
       "    'alignment:seed': 320},\n",
       "   'cross_references': {}}},\n",
       " 'PF02457': {'metadata': {'accession': 'PF02457',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003390',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DisA bacterial checkpoint controller nucleotide-binding',\n",
       "    'short': 'DAC'},\n",
       "   'description': ['<p>The DisA protein is a bacterial checkpoint protein that dimerises into an octameric complex. The protein consists of three distinct domains. This domain is the first and is a globular, nucleotide-binding region; the next 146-289 residues constitute the DisA-linker family, Pfam:PF10635, that consists of an elongated bundle of three alpha helices (alpha-6, alpha-10, and alpha-11), one side of which carries an additional three helices (alpha7-9), which thus forms a spine like-linker between domains 1 and 3. The C-terminal residues, of domain 3, are represented by family HHH, Pfam:PF00633, the specific DNA-binding domain. The octameric complex thus has structurally linked nucleotide-binding and DNA-binding HhH domains and the nucleotide-binding domains are bound to a cyclic di-adenosine phosphate such that DisA is a specific di-adenylate cyclase. This N-terminal domain has been identified as a diadenylate cyclase (DAC) responsible for producing c-di-AMP from two molecules of ATP [PMID:32095817]. The di-adenylate cyclase activity is strongly suppressed by binding to branched DNA, but not to duplex or single-stranded DNA, suggesting a role for DisA as a monitor of the presence of stalled replication forks or recombination intermediates via DNA structure-modulated c-di-AMP synthesis [PMID:18439896].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 69,\n",
       "    'interactions': 0,\n",
       "    'matches': 27352,\n",
       "    'pathways': 0,\n",
       "    'proteins': 27349,\n",
       "    'proteomes': 4835,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 23043, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 20107},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24155,\n",
       "    'alignment:full': 5407,\n",
       "    'alignment:seed': 501},\n",
       "   'cross_references': {}}},\n",
       " 'PF02570': {'metadata': {'accession': 'PF02570',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003722',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Precorrin-8X methylmutase', 'short': 'CbiC'},\n",
       "   'description': ['<p>This is a family Precorrin-8X methylmutases also known as Precorrin  isomerase, CbiC/CobH,  EC:5.4.1.2.  This enzyme catalyses the reaction: Precorrin-8X <=> hydrogenobyrinate.  This enzyme is part of the Cobalamin (vitamin B12) biosynthetic pathway and catalyses a methyl rearrangement  [PMID:9742225, PMID:8501034].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00000553': {'PMID': 9742225,\n",
       "     'ISBN': None,\n",
       "     'volume': '335 ( Pt 1)',\n",
       "     'issue': None,\n",
       "     'year': 1998,\n",
       "     'title': 'Cobalamin (vitamin B12) biosynthesis: identification and characterization of a Bacillus megaterium cobI operon.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '159-66',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Raux E', 'Lanois A', 'Warren MJ', 'Rambach A', 'Thermes C.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=9742225&action=stream&blobtype=pdf'},\n",
       "    'PUB00015329': {'PMID': 8501034,\n",
       "     'ISBN': None,\n",
       "     'volume': '175',\n",
       "     'issue': '11',\n",
       "     'year': 1993,\n",
       "     'title': 'Characterization of the cobalamin (vitamin B12) biosynthetic genes of Salmonella typhimurium.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3303-16',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Roth JR',\n",
       "      'Lawrence JG',\n",
       "      'Rubenfield M',\n",
       "      'Kieffer-Higgins S',\n",
       "      'Church GM.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=8501034'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 43,\n",
       "    'interactions': 0,\n",
       "    'matches': 16302,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16264,\n",
       "    'proteomes': 3605,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 14334, 'rosettafold': 0},\n",
       "    'structures': 10,\n",
       "    'taxa': 15847},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 15315,\n",
       "    'alignment:full': 3610,\n",
       "    'alignment:seed': 376},\n",
       "   'cross_references': {}}},\n",
       " 'PF01808': {'metadata': {'accession': 'PF01808',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002695',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'AICARFT/IMPCHase bienzyme', 'short': 'AICARFT_IMPCHas'},\n",
       "   'description': ['<p>This is a family of bifunctional enzymes catalysing the last two steps in de novo purine biosynthesis. The bifunctional enzyme is found in both prokaryotes and eukaryotes. The second last step is catalysed by 5-aminoimidazole-4-carboxamide  ribonucleotide formyltransferase EC:2.1.2.3 (AICARFT), this enzyme catalyses the formylation of AICAR with 10-formyl-tetrahydrofolate to yield FAICAR and tetrahydrofolate [PMID:9332377]. This is catalysed by a pair of C-terminal deaminase fold domains in the protein [PMID:21890906], where the active site is formed by the dimeric interface of two monomeric units [PMID:21890906]. The last  step is catalysed by the N-terminal IMP (Inosine monophosphate)  cyclohydrolase domain EC:3.5.4.10 (IMPCHase), cyclizing FAICAR  (5-formylaminoimidazole-4-carboxamide ribonucleotide) to IMP [PMID:9332377]. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001876': {'PMID': 9332377,\n",
       "     'ISBN': None,\n",
       "     'volume': '197',\n",
       "     'issue': '1-2',\n",
       "     'year': 1997,\n",
       "     'title': 'Molecular cloning and expression of a rat cDNA encoding 5-aminoimidazole-4-carboxamide ribonucleotide formyltransferase/IMP cyclohydrolase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '289-93',\n",
       "     'medline_journal': 'Gene',\n",
       "     'ISO_journal': 'Gene',\n",
       "     'authors': ['Akira T',\n",
       "      'Komatsu M',\n",
       "      'Nango R',\n",
       "      'Tomooka A',\n",
       "      'Konaka K',\n",
       "      'Yamauchi M',\n",
       "      'Kitamura Y',\n",
       "      'Nomura S',\n",
       "      'Tsukamoto I.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0378-1119(97)00273-4'},\n",
       "    'PUB00002969': {'PMID': 8567683,\n",
       "     'ISBN': None,\n",
       "     'volume': '271',\n",
       "     'issue': '4',\n",
       "     'year': 1996,\n",
       "     'title': 'The human purH gene product, 5-aminoimidazole-4-carboxamide ribonucleotide formyltransferase/IMP cyclohydrolase. Cloning, sequencing, expression, purification, kinetic analysis, and domain mapping.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2225-33',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Rayl EA', 'Moroson BA', 'Beardsley GP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.271.4.2225'}},\n",
       "   'set_info': {'accession': 'CL0109', 'name': 'CDA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 103,\n",
       "    'interactions': 0,\n",
       "    'matches': 51806,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51590,\n",
       "    'proteomes': 10301,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 43633, 'rosettafold': 0},\n",
       "    'structures': 19,\n",
       "    'taxa': 41947},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 46783,\n",
       "    'alignment:full': 10158,\n",
       "    'alignment:seed': 627},\n",
       "   'cross_references': {}}},\n",
       " 'PF01237': {'metadata': {'accession': 'PF01237',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000648',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Oxysterol-binding protein', 'short': 'Oxysterol_BP'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019972': {'PMID': 11916983,\n",
       "     'ISBN': None,\n",
       "     'volume': '157',\n",
       "     'issue': '1',\n",
       "     'year': 2002,\n",
       "     'title': 'Analysis of oxysterol binding protein homologue Kes1p function in regulation of Sec14p-dependent protein transport from the yeast Golgi complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '63-77',\n",
       "     'medline_journal': 'J Cell Biol',\n",
       "     'ISO_journal': 'J. Cell Biol.',\n",
       "     'authors': ['Li X',\n",
       "      'Rivas MP',\n",
       "      'Fang M',\n",
       "      'Marchena J',\n",
       "      'Mehrotra B',\n",
       "      'Chaudhary A',\n",
       "      'Feng L',\n",
       "      'Prestwich GD',\n",
       "      'Bankaitis VA.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1083/jcb.200201037'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 433,\n",
       "    'interactions': 0,\n",
       "    'matches': 40021,\n",
       "    'pathways': 0,\n",
       "    'proteins': 35040,\n",
       "    'proteomes': 2123,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 25958, 'rosettafold': 0},\n",
       "    'structures': 24,\n",
       "    'taxa': 7729},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29831,\n",
       "    'alignment:full': 19244,\n",
       "    'alignment:seed': 318},\n",
       "   'cross_references': {}}},\n",
       " 'PF18198': {'metadata': {'accession': 'PF18198',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041658',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Dynein heavy chain AAA lid domain',\n",
       "    'short': 'AAA_lid_11'},\n",
       "   'description': ['<p>This family represents the AAA lid domain found neat the C-terminal region of dynein heavy chain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0671', 'name': 'AAA_lid'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1328,\n",
       "    'interactions': 0,\n",
       "    'matches': 24808,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24702,\n",
       "    'proteomes': 1922,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 3524, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 7118},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17272,\n",
       "    'alignment:full': 10242,\n",
       "    'alignment:seed': 357},\n",
       "   'cross_references': {}}},\n",
       " 'PF02559': {'metadata': {'accession': 'PF02559',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003711',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CarD-like/TRCF domain', 'short': 'CarD_CdnL_TRCF'},\n",
       "   'description': ['<p>CarD is a Myxococcus xanthus protein required for the activation of light- and starvation-inducible genes [PMID:8692912]. This family includes the presumed N-terminal domain, CdnL.                 CarD interacts with the zinc-binding protein CarG to form a complex that regulates multiple processes in Myxococcus xanthus [PMID:16879646].  This family also includes a domain to the N-terminal side of the DEAD helicase of TRCF (transcription-repair-coupling factor) proteins. TRCF displaces RNA polymerase stalled at a lesion, binds to the damage recognition protein UvrA, and increases the template strand repair rate during transcription [PMID:7876261]. This domain is involved in binding to the stalled RNA polymerase [PMID:7876261]. The family includes members otherwise referred to as CdnL, for CarD N-terminal like, which differ functionally from CarD. The TRCF domain mentioned above is the RNA polymerase-interacting domain or RID [PMID:20371514].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008194': {'PMID': 8692912,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '14',\n",
       "     'year': 1996,\n",
       "     'title': 'High mobility group I(Y)-like DNA-binding domains on a bacterial transcription factor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6881-5',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Nicolas FJ',\n",
       "      'Cayuela ML',\n",
       "      'Martinez-Argudo IM',\n",
       "      'Ruiz-Vazquez RM',\n",
       "      'Murillo FJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.14.6881'},\n",
       "    'PUB00019252': {'PMID': 11533063,\n",
       "     'ISBN': None,\n",
       "     'volume': '276',\n",
       "     'issue': '45',\n",
       "     'year': 2001,\n",
       "     'title': 'Domain architecture of a high mobility group A-type bacterial transcriptional factor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '41566-75',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Padmanabhan S',\n",
       "      'Elias-Arnanz M',\n",
       "      'Carpio E',\n",
       "      'Aparicio P',\n",
       "      'Murillo FJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M106352200'},\n",
       "    'PUB00019253': {'PMID': 7876261,\n",
       "     'ISBN': None,\n",
       "     'volume': '270',\n",
       "     'issue': '9',\n",
       "     'year': 1995,\n",
       "     'title': 'Structure and function of transcription-repair coupling factor. I. Structural domains and binding properties.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4882-9',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Selby CP', 'Sancar A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.270.9.4882'},\n",
       "    'PUB00044926': {'PMID': 16879646,\n",
       "     'ISBN': None,\n",
       "     'volume': '61',\n",
       "     'issue': '4',\n",
       "     'year': 2006,\n",
       "     'title': 'Recruitment of a novel zinc-bound transcriptional factor by a bacterial HMGA-type protein is required for regulating multiple processes in Myxococcus xanthus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '910-26',\n",
       "     'medline_journal': 'Mol Microbiol',\n",
       "     'ISO_journal': 'Mol. Microbiol.',\n",
       "     'authors': ['Penalver-Mellado M',\n",
       "      'Garcia-Heras F',\n",
       "      'Padmanabhan S',\n",
       "      'Garcia-Moreno D',\n",
       "      'Murillo FJ',\n",
       "      'Elias-Arnanz M.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1365-2958.2006.05289.x'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 90,\n",
       "    'interactions': 0,\n",
       "    'matches': 63320,\n",
       "    'pathways': 0,\n",
       "    'proteins': 63317,\n",
       "    'proteomes': 8599,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 53912, 'rosettafold': 0},\n",
       "    'structures': 41,\n",
       "    'taxa': 35732},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 58490,\n",
       "    'alignment:full': 12067,\n",
       "    'alignment:seed': 759},\n",
       "   'cross_references': {}}},\n",
       " 'PF13959': {'metadata': {'accession': 'PF13959',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025313',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of unknown function (DUF4217)',\n",
       "    'short': 'DUF4217'},\n",
       "   'description': ['<p>This short domain is found at the C-terminus of many helicase proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 219,\n",
       "    'interactions': 0,\n",
       "    'matches': 14537,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14529,\n",
       "    'proteomes': 2176,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 11762, 'rosettafold': 0},\n",
       "    'structures': 27,\n",
       "    'taxa': 7882},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11725,\n",
       "    'alignment:full': 7185,\n",
       "    'alignment:seed': 359},\n",
       "   'cross_references': {}}},\n",
       " 'PF17678': {'metadata': {'accession': 'PF17678',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041371',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glycosyl hydrolase family 92 N-terminal domain',\n",
       "    'short': 'Glyco_hydro_92N'},\n",
       "   'description': ['<p>This domain is found at the N-terminus of family 92 glycosyl hydrolase proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0103', 'name': 'Gal_mutarotase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 259,\n",
       "    'interactions': 0,\n",
       "    'matches': 28205,\n",
       "    'pathways': 0,\n",
       "    'proteins': 28165,\n",
       "    'proteomes': 2470,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24139, 'rosettafold': 0},\n",
       "    'structures': 18,\n",
       "    'taxa': 9695},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26416,\n",
       "    'alignment:full': 7273,\n",
       "    'alignment:seed': 328},\n",
       "   'cross_references': {}}},\n",
       " 'PF13537': {'metadata': {'accession': 'PF13537',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR017932',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutamine amidotransferase domain', 'short': 'GATase_7'},\n",
       "   'description': ['<p>This domain is a class-II glutamine amidotransferase domain found in a variety of enzymes such as asparagine synthetase and glutamine-fructose-6-phosphate transaminase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0052', 'name': 'NTN'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 237,\n",
       "    'interactions': 0,\n",
       "    'matches': 83890,\n",
       "    'pathways': 0,\n",
       "    'proteins': 83875,\n",
       "    'proteomes': 9142,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 70497, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 38579},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 74518,\n",
       "    'alignment:full': 17314,\n",
       "    'alignment:seed': 270},\n",
       "   'cross_references': {}}},\n",
       " 'PF01066': {'metadata': {'accession': 'PF01066',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000462',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CDP-alcohol phosphatidyltransferase',\n",
       "    'short': 'CDP-OH_P_transf'},\n",
       "   'description': ['<p>All of these members have the ability to catalyse the displacement of CMP from a CDP-alcohol by a second alcohol with formation of a phosphodiester bond and concomitant breaking of a phosphoride anhydride bond.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001469': {'PMID': 8917447,\n",
       "     'ISBN': None,\n",
       "     'volume': '241',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'Two-dimensional 1H-NMR of transmembrane peptides from Escherichia coli phosphatidylglycerophosphate synthase in micelles.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '489-97',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Morein S',\n",
       "      'Trouard TP',\n",
       "      'Hauksson JB',\n",
       "      'Rilfors L',\n",
       "      'Arvidson G',\n",
       "      'Lindblom G.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1432-1033.1996.00489.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 435,\n",
       "    'interactions': 0,\n",
       "    'matches': 144263,\n",
       "    'pathways': 0,\n",
       "    'proteins': 143965,\n",
       "    'proteomes': 11145,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 122595, 'rosettafold': 0},\n",
       "    'structures': 17,\n",
       "    'taxa': 46171},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 128791,\n",
       "    'alignment:full': 32088,\n",
       "    'alignment:seed': 230},\n",
       "   'cross_references': {}}},\n",
       " 'PF02365': {'metadata': {'accession': 'PF02365',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003441',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'No apical meristem (NAM) protein', 'short': 'NAM'},\n",
       "   'description': ['<p>This is a family of no apical meristem (NAM) proteins these are plant development proteins.  Mutations in NAM result in the failure to develop a shoot apical meristem in petunia embryos [PMID:8612269].  NAM is indicated as having a role in determining positions of meristems and primordial [PMID:8612269]. One member of this family NAP (NAC-like, activated by AP3/PI) is encoded by the target genes of the AP3/PI transcriptional activators and functions in the transition between growth by cell division and cell expansion in stamens and petals [PMID:9489703].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009489': {'PMID': 8612269,\n",
       "     'ISBN': None,\n",
       "     'volume': '85',\n",
       "     'issue': '2',\n",
       "     'year': 1996,\n",
       "     'title': 'The no apical meristem gene of Petunia is required for pattern formation in embryos and flowers and is expressed at meristem and primordia boundaries.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '159-70',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Souer E',\n",
       "      'van Houwelingen A',\n",
       "      'Kloos D',\n",
       "      'Mol J',\n",
       "      'Koes R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)81093-4'},\n",
       "    'PUB00009490': {'PMID': 9489703,\n",
       "     'ISBN': None,\n",
       "     'volume': '92',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'A homolog of NO APICAL MERISTEM is an immediate target of the floral homeotic genes APETALA3/PISTILLATA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '93-103',\n",
       "     'medline_journal': 'Cell',\n",
       "     'ISO_journal': 'Cell',\n",
       "     'authors': ['Sablowski RW', 'Meyerowitz EM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0092-8674(00)80902-2'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 400,\n",
       "    'interactions': 0,\n",
       "    'matches': 37011,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36396,\n",
       "    'proteomes': 205,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 30881, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 1208},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 31088,\n",
       "    'alignment:full': 21006,\n",
       "    'alignment:seed': 444},\n",
       "   'cross_references': {}}},\n",
       " 'PF00742': {'metadata': {'accession': 'PF00742',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001342',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Homoserine dehydrogenase', 'short': 'Homoserine_dh'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0139', 'name': 'GADPH_aa-bio_dh'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 153,\n",
       "    'interactions': 0,\n",
       "    'matches': 54654,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54624,\n",
       "    'proteomes': 9342,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 47376, 'rosettafold': 0},\n",
       "    'structures': 29,\n",
       "    'taxa': 38774},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50935,\n",
       "    'alignment:full': 10618,\n",
       "    'alignment:seed': 799},\n",
       "   'cross_references': {}}},\n",
       " 'PF14403': {'metadata': {'accession': 'PF14403',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025841',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Circularly permuted ATP-grasp type 2',\n",
       "    'short': 'CP_ATPgrasp_2'},\n",
       "   'description': ['<p>Circularly permuted ATP-grasp prototyped by Roseiflexus RoseRS_2616  that is associated in gene neighborhoods with a GCS2-like COOH-NH2  ligase, alpha/beta hydrolase fold peptidase, GAT-II -like  amidohydrolase, and M20 peptidase. Members of this family are  predicted to be involved in the biosynthesis of small peptides [PMID:20023723].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0179', 'name': 'ATP-grasp'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 26,\n",
       "    'interactions': 0,\n",
       "    'matches': 20964,\n",
       "    'pathways': 0,\n",
       "    'proteins': 20941,\n",
       "    'proteomes': 3345,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 17870, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 13973},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19403,\n",
       "    'alignment:full': 4502,\n",
       "    'alignment:seed': 294},\n",
       "   'cross_references': {}}},\n",
       " 'PF01388': {'metadata': {'accession': 'PF01388',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001606',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'ARID/BRIGHT DNA binding domain', 'short': 'ARID'},\n",
       "   'description': ['<p>This domain is know as ARID for AT-Rich Interaction Domain [PMID:8543152], and also known as the BRIGHT domain [PMID:8622680].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00001923': {'PMID': 8543152,\n",
       "     'ISBN': None,\n",
       "     'volume': '9',\n",
       "     'issue': '24',\n",
       "     'year': 1995,\n",
       "     'title': 'The immunoglobulin heavy-chain matrix-associating regions are bound by Bright: a B cell-specific trans-activator that describes a new DNA-binding protein family.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3067-82',\n",
       "     'medline_journal': 'Genes Dev',\n",
       "     'ISO_journal': 'Genes Dev.',\n",
       "     'authors': ['Herrscher RF',\n",
       "      'Kaplan MH',\n",
       "      'Lelsz DL',\n",
       "      'Das C',\n",
       "      'Scheuermann R',\n",
       "      'Tucker PW.'],\n",
       "     'DOI_URL': 'http://www.genesdev.org/cgi/content/abstract/9/24/3067'},\n",
       "    'PUB00003962': {'PMID': 9808040,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '11',\n",
       "     'year': 1998,\n",
       "     'title': 'A novel DNA-binding motif shares structural homology to DNA replication and repair nucleases and polymerases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '959-64',\n",
       "     'medline_journal': 'Nat Struct Biol',\n",
       "     'ISO_journal': 'Nat. Struct. Biol.',\n",
       "     'authors': ['Yuan YC', 'Whitson RH', 'Liu Q', 'Itakura K', 'Chen Y.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/2934'},\n",
       "    'PUB00019164': {'PMID': 8622680,\n",
       "     'ISBN': None,\n",
       "     'volume': '16',\n",
       "     'issue': '3',\n",
       "     'year': 1996,\n",
       "     'title': 'Characterization of the dead ringer gene identifies a novel, highly conserved family of sequence-specific DNA-binding proteins.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '792-9',\n",
       "     'medline_journal': 'Mol Cell Biol',\n",
       "     'ISO_journal': 'Mol. Cell. Biol.',\n",
       "     'authors': ['Gregory SL', 'Kortschak RD', 'Kalionis B', 'Saint R.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=8622680'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 415,\n",
       "    'interactions': 0,\n",
       "    'matches': 29706,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29672,\n",
       "    'proteomes': 2110,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12634, 'rosettafold': 0},\n",
       "    'structures': 119,\n",
       "    'taxa': 7586},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 21893,\n",
       "    'alignment:full': 13851,\n",
       "    'alignment:seed': 367},\n",
       "   'cross_references': {}}},\n",
       " 'PF09479': {'metadata': {'accession': 'PF09479',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013378',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Listeria-Bacteroides repeat domain (List_Bact_rpt)',\n",
       "    'short': 'Flg_new'},\n",
       "   'description': ['<p>This model describes a conserved core region of about 43 residues [PMID:21345802] which occurs in more than 400 mostly secreted or cell surface-located proteins from over 300 eubacterial and few archaeal species. These proteins contain between one and 34 copies of the domain that often occur in tandem arrays with up to more than 20 copies [PMID:27789707].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0072', 'name': 'Ubiquitin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 3773,\n",
       "    'interactions': 0,\n",
       "    'matches': 37062,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14689,\n",
       "    'proteomes': 1045,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 9102, 'rosettafold': 0},\n",
       "    'structures': 6,\n",
       "    'taxa': 4385},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 34529,\n",
       "    'alignment:full': 8246,\n",
       "    'alignment:seed': 220},\n",
       "   'cross_references': {}}},\n",
       " 'PF02569': {'metadata': {'accession': 'PF02569',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003721',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Pantoate-beta-alanine ligase',\n",
       "    'short': 'Pantoate_ligase'},\n",
       "   'description': ['<p>Pantoate-beta-alanine ligase, also know as pantothenate  synthase, (EC:6.3.2.1) catalyses the formation of pantothenate from pantoate and alanine [PMID:374975].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019988': {'PMID': 374975,\n",
       "     'ISBN': None,\n",
       "     'volume': '62',\n",
       "     'issue': None,\n",
       "     'year': 1979,\n",
       "     'title': 'Pantothenate synthetase from Escherichia coli [D-pantoate: beta-alanine ligase (AMP-forming), EC 6.3.2.1].',\n",
       "     'URL': None,\n",
       "     'raw_pages': '215-9',\n",
       "     'medline_journal': 'Methods Enzymol',\n",
       "     'ISO_journal': 'Meth. Enzymol.',\n",
       "     'authors': ['Miyatake K', 'Nakano Y', 'Kitaoka S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0076-6879(79)62221-8'}},\n",
       "   'set_info': {'accession': 'CL0039', 'name': 'HUP'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 82,\n",
       "    'interactions': 0,\n",
       "    'matches': 39227,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39129,\n",
       "    'proteomes': 8043,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33745, 'rosettafold': 0},\n",
       "    'structures': 61,\n",
       "    'taxa': 32821},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36159,\n",
       "    'alignment:full': 7618,\n",
       "    'alignment:seed': 661},\n",
       "   'cross_references': {}}},\n",
       " 'PF02910': {'metadata': {'accession': 'PF02910',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR015939',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Fumarate reductase flavoprotein C-term',\n",
       "    'short': 'Succ_DH_flav_C'},\n",
       "   'description': ['<p>This family contains fumarate reductases, succinate dehydrogenases and L-aspartate oxidases.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00020385': {'PMID': 10373108,\n",
       "     'ISBN': None,\n",
       "     'volume': '284',\n",
       "     'issue': '5422',\n",
       "     'year': 1999,\n",
       "     'title': 'Structure of the Escherichia coli fumarate reductase respiratory complex.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1961-6',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Iverson TM', 'Luna-Chavez C', 'Cecchini G', 'Rees DC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.284.5422.1961'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 179,\n",
       "    'interactions': 0,\n",
       "    'matches': 86777,\n",
       "    'pathways': 0,\n",
       "    'proteins': 86717,\n",
       "    'proteomes': 9765,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 74093, 'rosettafold': 0},\n",
       "    'structures': 87,\n",
       "    'taxa': 40460},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 79638,\n",
       "    'alignment:full': 17519,\n",
       "    'alignment:seed': 625},\n",
       "   'cross_references': {}}},\n",
       " 'PF05960': {'metadata': {'accession': 'PF05960',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010281',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Bacterial protein of unknown function (DUF885)',\n",
       "    'short': 'DUF885'},\n",
       "   'description': ['<p>This family consists of several hypothetical bacterial proteins several of which are putative membrane proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0126', 'name': 'Peptidase_MA'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 30289,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29829,\n",
       "    'proteomes': 3463,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 25074, 'rosettafold': 0},\n",
       "    'structures': 3,\n",
       "    'taxa': 13081},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26751,\n",
       "    'alignment:full': 6328,\n",
       "    'alignment:seed': 294},\n",
       "   'cross_references': {}}},\n",
       " 'PF01165': {'metadata': {'accession': 'PF01165',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001911',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein S21', 'short': 'Ribosomal_S21'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 46,\n",
       "    'interactions': 0,\n",
       "    'matches': 37866,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37858,\n",
       "    'proteomes': 8214,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32314, 'rosettafold': 0},\n",
       "    'structures': 414,\n",
       "    'taxa': 35400},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33670,\n",
       "    'alignment:full': 8509,\n",
       "    'alignment:seed': 320},\n",
       "   'cross_references': {}}},\n",
       " 'PF01790': {'metadata': {'accession': 'PF01790',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001640',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Prolipoprotein diacylglyceryl transferase',\n",
       "    'short': 'LGT'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 97,\n",
       "    'interactions': 0,\n",
       "    'matches': 57277,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56967,\n",
       "    'proteomes': 8505,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 48075, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 35979},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51406,\n",
       "    'alignment:full': 9745,\n",
       "    'alignment:seed': 366},\n",
       "   'cross_references': {}}},\n",
       " 'PF13976': {'metadata': {'accession': 'PF13976',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025724',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GAG-pre-integrase domain', 'short': 'gag_pre-integrs'},\n",
       "   'description': ['<p>This domain is found associated with retroviral insertion elements and lies just upstream of the integrase region on the polyproteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 2282,\n",
       "    'interactions': 0,\n",
       "    'matches': 29867,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29510,\n",
       "    'proteomes': 477,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 17526, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 2476},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26162,\n",
       "    'alignment:full': 7358,\n",
       "    'alignment:seed': 1083},\n",
       "   'cross_references': {}}},\n",
       " 'PF02575': {'metadata': {'accession': 'PF02575',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004401',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'YbaB/EbfC DNA-binding family', 'short': 'YbaB_DNA_bd'},\n",
       "   'description': ['<p>This is a family of DNA-binding proteins. Members of this family form homodimers which bind DNA via a tweezer-like structure [1-3]. The conformation of the DNA is changed when bound to these proteins [PMID:19208644]. In bacteria, these proteins may play a role in DNA replication-recovery following DNA damage [PMID:12486730].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0717', 'name': 'YbaB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 48,\n",
       "    'interactions': 0,\n",
       "    'matches': 43285,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43254,\n",
       "    'proteomes': 7827,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 37442, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 33452},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39159,\n",
       "    'alignment:full': 9401,\n",
       "    'alignment:seed': 649},\n",
       "   'cross_references': {}}},\n",
       " 'PF06628': {'metadata': {'accession': 'PF06628',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010582',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Catalase-related immune-responsive',\n",
       "    'short': 'Catalase-rel'},\n",
       "   'description': ['<p>This family represents a small conserved region within catalase enzymes (EC:1.11.1.6). All members also contain the Catalase family, Pfam:PF00199 domain. Catalase decomposes hydrogen peroxide into water and oxygen, serving to protect cells from its toxic effects [PMID:11351128]. This domain carries the immune-responsive amphipathic octa-peptide that is recognised by T cells [PMID:15585332].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012765': {'PMID': 11351128,\n",
       "     'ISBN': None,\n",
       "     'volume': '10',\n",
       "     'issue': '3-4',\n",
       "     'year': 2001,\n",
       "     'title': 'Mitochondrial catalase and oxidative injury.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '189-99',\n",
       "     'medline_journal': 'Biol Signals Recept',\n",
       "     'ISO_journal': None,\n",
       "     'authors': ['Bai J', 'Cederbaum AI.'],\n",
       "     'DOI_URL': None},\n",
       "    'PUB00044744': {'PMID': 15585332,\n",
       "     'ISBN': None,\n",
       "     'volume': '96',\n",
       "     'issue': '2',\n",
       "     'year': 2005,\n",
       "     'title': 'Do Th1 or Th2 sequence motifs exist in proteins? Identification of amphipatic immunomodulatory domains in Helicobacter pylori catalase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '261-75',\n",
       "     'medline_journal': 'Immunol Lett',\n",
       "     'ISO_journal': 'Immunol. Lett.',\n",
       "     'authors': ['Guy B',\n",
       "      'Krell T',\n",
       "      'Sanchez V',\n",
       "      'Kennel A',\n",
       "      'Manin C',\n",
       "      'Sodoyer R.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.imlet.2004.09.011'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 123,\n",
       "    'interactions': 0,\n",
       "    'matches': 40377,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40340,\n",
       "    'proteomes': 6315,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 35835, 'rosettafold': 0},\n",
       "    'structures': 134,\n",
       "    'taxa': 26810},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 39623,\n",
       "    'alignment:full': 10387,\n",
       "    'alignment:seed': 557},\n",
       "   'cross_references': {}}},\n",
       " 'PF16870': {'metadata': {'accession': 'PF16870',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031717',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '2-oxoglutarate dehydrogenase C-terminal',\n",
       "    'short': 'OxoGdeHyase_C'},\n",
       "   'description': ['<p>OxoGdeHyase_C is a family found immediately C-terminal to Transket_pyr, Pfam:PF02779. It is found at the C-terminus of 2-oxoglutarate dehydrogenase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0591', 'name': 'TKC_like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 165,\n",
       "    'interactions': 0,\n",
       "    'matches': 37718,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37649,\n",
       "    'proteomes': 8185,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 30503, 'rosettafold': 0},\n",
       "    'structures': 35,\n",
       "    'taxa': 31297},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 33704,\n",
       "    'alignment:full': 9916,\n",
       "    'alignment:seed': 238},\n",
       "   'cross_references': {}}},\n",
       " 'PF17763': {'metadata': {'accession': 'PF17763',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR040919',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glutaminase/Asparaginase C-terminal domain',\n",
       "    'short': 'Asparaginase_C'},\n",
       "   'description': ['<p>This domain is found at the C-terminus of asparaginase enzymes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 134,\n",
       "    'interactions': 0,\n",
       "    'matches': 37319,\n",
       "    'pathways': 0,\n",
       "    'proteins': 37309,\n",
       "    'proteomes': 6684,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 32904, 'rosettafold': 0},\n",
       "    'structures': 122,\n",
       "    'taxa': 29037},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 35639,\n",
       "    'alignment:full': 7669,\n",
       "    'alignment:seed': 275},\n",
       "   'cross_references': {}}},\n",
       " 'PF05792': {'metadata': {'accession': 'PF05792',\n",
       "   'entry_id': None,\n",
       "   'type': 'repeat',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008440',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Candida agglutinin-like (ALS)', 'short': 'Candida_ALS'},\n",
       "   'description': ['<p>This family consists of several agglutinin-like proteins from different Candida species. ALS genes of Candida albicans encode a family of cell-surface glycoproteins with a three-domain structure. Each Als protein has a relatively conserved N-terminal domain, a central domain consisting of a tandemly repeated motif of variable number, and a serine-threonine-rich C-terminal domain that is relatively variable across the family. The ALS family exhibits several types of variability that indicate the importance of considering strain and allelic differences when studying ALS genes and their encoded proteins [PMID:11124701]. Fungal adhesins, which include sexual agglutinins, virulence factors, and flocculins, are surface proteins that mediate cell-cell and cell-environment interactions. It is possible that both the serine/threonine-rich domain and the cysteine residues in the C-terminal and DIPSY Pfam:PF11763 participate in anchoring the terminal domains inside the wall, so that only the inner part of Map4p, including the repeat region, is sticking out as a fold-back loop then able to act in adhesing [PMID:17870620].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011290': {'PMID': 11124701,\n",
       "     'ISBN': None,\n",
       "     'volume': '18',\n",
       "     'issue': '1',\n",
       "     'year': 2001,\n",
       "     'title': 'The ALS5 gene of Candida albicans and analysis of the Als5p N-terminal domain.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '49-60',\n",
       "     'medline_journal': 'Yeast',\n",
       "     'ISO_journal': 'Yeast',\n",
       "     'authors': ['Hoyer LL', 'Hecht JE.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1002/1097-0061(200101)18:1<49::AID-YEA646>3.0.CO;2-M'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 194,\n",
       "    'interactions': 0,\n",
       "    'matches': 11045,\n",
       "    'pathways': 0,\n",
       "    'proteins': 533,\n",
       "    'proteomes': 93,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 175, 'rosettafold': 0},\n",
       "    'structures': 1,\n",
       "    'taxa': 266},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 8673,\n",
       "    'alignment:full': 4721,\n",
       "    'alignment:seed': 445},\n",
       "   'cross_references': {}}},\n",
       " 'PF06965': {'metadata': {'accession': 'PF06965',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004670',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Na+/H+ antiporter 1', 'short': 'Na_H_antiport_1'},\n",
       "   'description': ['<p>This family contains a number of bacterial Na+/H+ antiporter 1 proteins. These are integral membrane proteins that catalyse the exchange of H+ for Na+ in a manner that is highly dependent on the pH [PMID:1657980].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00012952': {'PMID': 1657980,\n",
       "     'ISBN': None,\n",
       "     'volume': '266',\n",
       "     'issue': '32',\n",
       "     'year': 1991,\n",
       "     'title': 'Expression of a sodium proton antiporter (NhaA) in Escherichia coli is induced by Na+ and Li+ ions.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '21753-9',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Karpel R', 'Alon T', 'Glaser G', 'Schuldiner S', 'Padan E.'],\n",
       "     'DOI_URL': 'http://intl.jbc.org/cgi/reprint/266/32/21753.pdf'}},\n",
       "   'set_info': {'accession': 'CL0064', 'name': 'CPA_AT'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 47,\n",
       "    'interactions': 0,\n",
       "    'matches': 24760,\n",
       "    'pathways': 0,\n",
       "    'proteins': 24657,\n",
       "    'proteomes': 4029,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21577, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 17143},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 23538,\n",
       "    'alignment:full': 4911,\n",
       "    'alignment:seed': 337},\n",
       "   'cross_references': {}}},\n",
       " 'PF01926': {'metadata': {'accession': 'PF01926',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006073',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': '50S ribosome-binding GTPase', 'short': 'MMR_HSR1'},\n",
       "   'description': ['<p>The full-length GTPase protein is required for the complete activity of the protein of interacting with the 50S ribosome and binding of both adenine and guanine nucleotides, with a preference for guanine nucleotide.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005720': {'PMID': 8180467,\n",
       "     'ISBN': None,\n",
       "     'volume': '5',\n",
       "     'issue': '2',\n",
       "     'year': 1994,\n",
       "     'title': 'Structure and evolution of a member of a new subfamily of GTP-binding proteins mapping to the human MHC class I region.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '100-5',\n",
       "     'medline_journal': 'Mamm Genome',\n",
       "     'ISO_journal': 'Mamm. Genome',\n",
       "     'authors': ['Vernet C', 'Ribouchon MT', 'Chimini G', 'Pontarotti P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/BF00292335'}},\n",
       "   'set_info': {'accession': 'CL0023', 'name': 'P-loop_NTPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1600,\n",
       "    'interactions': 0,\n",
       "    'matches': 446884,\n",
       "    'pathways': 0,\n",
       "    'proteins': 403632,\n",
       "    'proteomes': 11379,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 342539, 'rosettafold': 0},\n",
       "    'structures': 138,\n",
       "    'taxa': 50144},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 398865,\n",
       "    'alignment:full': 101981,\n",
       "    'alignment:seed': 674},\n",
       "   'cross_references': {}}},\n",
       " 'PF01339': {'metadata': {'accession': 'PF01339',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000673',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'CheB methylesterase', 'short': 'CheB_methylest'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00003348': {'PMID': 7608974,\n",
       "     'ISBN': None,\n",
       "     'volume': '250',\n",
       "     'issue': '2',\n",
       "     'year': 1995,\n",
       "     'title': 'Crystal structure of the catalytic domain of the chemotaxis receptor methylesterase, CheB.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '276-90',\n",
       "     'medline_journal': 'J Mol Biol',\n",
       "     'ISO_journal': 'J. Mol. Biol.',\n",
       "     'authors': ['West AH', 'Martinez-Hackert E', 'Stock AM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1006/jmbi.1995.0376'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 903,\n",
       "    'interactions': 0,\n",
       "    'matches': 47361,\n",
       "    'pathways': 0,\n",
       "    'proteins': 47309,\n",
       "    'proteomes': 4817,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 38812, 'rosettafold': 0},\n",
       "    'structures': 5,\n",
       "    'taxa': 21110},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44011,\n",
       "    'alignment:full': 9409,\n",
       "    'alignment:seed': 879},\n",
       "   'cross_references': {}}},\n",
       " 'PF00883': {'metadata': {'accession': 'PF00883',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR000819',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cytosol aminopeptidase family, catalytic domain',\n",
       "    'short': 'Peptidase_M17'},\n",
       "   'description': ['<p>The two associated zinc ions and the active site are entirely enclosed within the C-terminal catalytic domain in leucine  aminopeptidase.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004713': {'PMID': 2395881,\n",
       "     'ISBN': None,\n",
       "     'volume': '87',\n",
       "     'issue': '17',\n",
       "     'year': 1990,\n",
       "     'title': 'Molecular structure of leucine aminopeptidase at 2.7-A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6878-82',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Burley SK', 'David PR', 'Taylor A', 'Lipscomb WN.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/picrender.fcgi?tool=EBI&pubmedid=2395881&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0035', 'name': 'Peptidase_MH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 192,\n",
       "    'interactions': 0,\n",
       "    'matches': 56413,\n",
       "    'pathways': 0,\n",
       "    'proteins': 56061,\n",
       "    'proteomes': 7968,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 47500, 'rosettafold': 0},\n",
       "    'structures': 63,\n",
       "    'taxa': 33824},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 51436,\n",
       "    'alignment:full': 12121,\n",
       "    'alignment:seed': 650},\n",
       "   'cross_references': {}}},\n",
       " 'PF10576': {'metadata': {'accession': 'PF10576',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003651',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Iron-sulfur binding domain of endonuclease III',\n",
       "    'short': 'EndIII_4Fe-2S'},\n",
       "   'description': ['<p>Escherichia coli endonuclease III (EC 4.2.99.18) [PMID:7664751] is a DNA repair enzyme that acts both as a DNA N-glycosylase, removing oxidised pyrimidines from DNA, and as an apurinic/apyrimidinic (AP) endonuclease, introducing a single-strand nick at the site from which the damaged base was removed. Endonuclease III is an iron-sulfur protein that binds a single 4Fe-4S cluster. The 4Fe-4S cluster does not seem to be important for catalytic activity, but is probably involved in the proper positioning of the enzyme along the DNA strand [PMID:9045706]. The 4Fe-4S cluster is bound by four cysteines which are all located in a 17 amino acid region at the C-terminal end of endonuclease III. A similar region is also present in the central section of mutY and in the C-terminus of ORF-10 and of the Micro-coccus UV endonuclease [PMID:16967954].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00006161': {'PMID': 7664751,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '16',\n",
       "     'year': 1995,\n",
       "     'title': 'Novel DNA binding motifs in the DNA repair enzyme endonuclease III crystal structure.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4108-20',\n",
       "     'medline_journal': 'EMBO J',\n",
       "     'ISO_journal': 'EMBO J.',\n",
       "     'authors': ['Thayer MM',\n",
       "      'Ahern H',\n",
       "      'Xing D',\n",
       "      'Cunningham RP',\n",
       "      'Tainer JA.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/articlerender.cgi?tool=EBI&pubmedid=7664751'},\n",
       "    'PUB00044245': {'PMID': 9045706,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '10',\n",
       "     'year': 1997,\n",
       "     'title': 'Cloning and expression of the cDNA encoding the human homologue of the DNA repair enzyme, Escherichia coli endonuclease III.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '6733-40',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Hilbert TP',\n",
       "      'Chaung W',\n",
       "      'Boorstein RJ',\n",
       "      'Cunningham RP',\n",
       "      'Teebor GW.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.10.6733'},\n",
       "    'PUB00044246': {'PMID': 16967954,\n",
       "     'ISBN': None,\n",
       "     'volume': '128',\n",
       "     'issue': '37',\n",
       "     'year': 2006,\n",
       "     'title': 'Direct electrochemistry of endonuclease III in the presence and absence of DNA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '12082-3',\n",
       "     'medline_journal': 'J Am Chem Soc',\n",
       "     'ISO_journal': 'J. Am. Chem. Soc.',\n",
       "     'authors': ['Gorodetsky AA', 'Boal AK', 'Barton JK.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/ja064784d'},\n",
       "    'PUB00044974': {'PMID': 16096281,\n",
       "     'ISBN': None,\n",
       "     'volume': '280',\n",
       "     'issue': '40',\n",
       "     'year': 2005,\n",
       "     'title': 'Engineering functional changes in Escherichia coli endonuclease III based on phylogenetic and structural analyses.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '34378-84',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Watanabe T', 'Blaisdell JO', 'Wallace SS', 'Bond JP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M504916200'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 46,\n",
       "    'interactions': 0,\n",
       "    'matches': 32180,\n",
       "    'pathways': 0,\n",
       "    'proteins': 32179,\n",
       "    'proteomes': 5717,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 28230, 'rosettafold': 0},\n",
       "    'structures': 16,\n",
       "    'taxa': 24770},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 30107,\n",
       "    'alignment:full': 6573,\n",
       "    'alignment:seed': 392},\n",
       "   'cross_references': {}}},\n",
       " 'PF04982': {'metadata': {'accession': 'PF04982',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007065',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'HPP family', 'short': 'HPP'},\n",
       "   'description': ['<p>These proteins are integral membrane proteins with four transmembrane spanning helices. The most conserved region of the alignment is a motif HPP. The function of these proteins is uncertain but they may be transporters.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0307', 'name': 'FUSC'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 55,\n",
       "    'interactions': 0,\n",
       "    'matches': 14137,\n",
       "    'pathways': 0,\n",
       "    'proteins': 14109,\n",
       "    'proteomes': 3048,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 12497, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 13881},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13506,\n",
       "    'alignment:full': 3389,\n",
       "    'alignment:seed': 303},\n",
       "   'cross_references': {}}},\n",
       " 'PF10825': {'metadata': {'accession': 'PF10825',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR021215',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF2752)',\n",
       "    'short': 'DUF2752'},\n",
       "   'description': ['<p>This family is conserved in bacteria. Many members are annotated as being putative membrane proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 9,\n",
       "    'interactions': 0,\n",
       "    'matches': 11874,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11874,\n",
       "    'proteomes': 2284,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 10025, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 8999},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10517,\n",
       "    'alignment:full': 2499,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF17146': {'metadata': {'accession': 'PF17146',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR033411',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PIN domain of ribonuclease', 'short': 'PIN_6'},\n",
       "   'description': ['<p>This is a PIN domain found in eukaryotic ribonuclease Nob1 and archaeal ribonuclease VapC1 [PMID:22156373]. Budding yeast Nob1 is involved in proteasomal and 40S ribosomal subunit biogenesis [PMID:10675611]. VapC1 is a toxic component and a ribonuclease of a toxin-antitoxin (TA) module [PMID:25391136]. PIN domains are small protein domains identified by the presence of three strictly conserved acidic residues. Apart from these three residues, there is poor sequence conservation [PMID:21036780]. PIN domains are found in eukaryotes, eubacteria and archaea. In eukaryotes they are ribonucleases involved in nonsense mediated mRNA decay [PMID:17053788] and in processing of 18S ribosomal RNA [PMID:19706509]. In prokaryotes, they are the toxic components of toxin-antitoxin (TA) systems, their toxicity arising by virtue of their ribonuclease activity. The PIN domain TA systems are now called VapBC TAs(virulence associated proteins), where VapB is the inhibitor and VapC, the PIN-domain ribonuclease toxin [PMID:21036780].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0280', 'name': 'PIN'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 75,\n",
       "    'interactions': 0,\n",
       "    'matches': 5891,\n",
       "    'pathways': 0,\n",
       "    'proteins': 5877,\n",
       "    'proteomes': 2209,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 5037, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 8997},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4855,\n",
       "    'alignment:full': 1960,\n",
       "    'alignment:seed': 263},\n",
       "   'cross_references': {}}},\n",
       " 'PF04963': {'metadata': {'accession': 'PF04963',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007046',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sigma-54 factor, core binding domain',\n",
       "    'short': 'Sigma54_CBD'},\n",
       "   'description': ['<p>This domain makes a direct interaction with the core RNA polymerase, to form an enhancer dependent holoenzyme [PMID:10894718]. The centre of this domain contains a very weak similarity to a helix-turn-helix motif which may represent the other DNA binding domain.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008721': {'PMID': 10894718,\n",
       "     'ISBN': None,\n",
       "     'volume': '182',\n",
       "     'issue': '15',\n",
       "     'year': 2000,\n",
       "     'title': 'The bacterial enhancer-dependent sigma(54) (sigma(N)) transcription factor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4129-36',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Buck M',\n",
       "      'Gallegos MT',\n",
       "      'Studholme DJ',\n",
       "      'Guo Y',\n",
       "      'Gralla JD.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.182.15.4129-4136.2000'}},\n",
       "   'set_info': {'accession': 'CL0123', 'name': 'HTH'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 54,\n",
       "    'interactions': 0,\n",
       "    'matches': 31367,\n",
       "    'pathways': 0,\n",
       "    'proteins': 31195,\n",
       "    'proteomes': 5168,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 26694, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 22859},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 29068,\n",
       "    'alignment:full': 5195,\n",
       "    'alignment:seed': 412},\n",
       "   'cross_references': {}}},\n",
       " 'PF00766': {'metadata': {'accession': 'PF00766',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR014731',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Electron transfer flavoprotein FAD-binding domain',\n",
       "    'short': 'ETF_alpha'},\n",
       "   'description': ['<p>This domain found at the C-terminus of electron transfer flavoprotein alpha chain and binds to FAD [PMID:8962055]. The fold consists of a five-stranded parallel beta sheet as the core of the domain, flanked by alternating helices.  A small part of this domain is donated by the beta chain [PMID:8962055].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004879': {'PMID': 8962055,\n",
       "     'ISBN': None,\n",
       "     'volume': '93',\n",
       "     'issue': '25',\n",
       "     'year': 1996,\n",
       "     'title': 'Three-dimensional structure of human electron transfer flavoprotein to 2.1-A resolution.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '14355-60',\n",
       "     'medline_journal': 'Proc Natl Acad Sci U S A',\n",
       "     'ISO_journal': 'Proc. Natl. Acad. Sci. U.S.A.',\n",
       "     'authors': ['Roberts DL', 'Frerman FE', 'Kim JJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1073/pnas.93.25.14355'}},\n",
       "   'set_info': {'accession': 'CL0085', 'name': 'FAD_DHS'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 131,\n",
       "    'interactions': 0,\n",
       "    'matches': 54171,\n",
       "    'pathways': 0,\n",
       "    'proteins': 54161,\n",
       "    'proteomes': 8714,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 46592, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 34585},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50055,\n",
       "    'alignment:full': 10857,\n",
       "    'alignment:seed': 427},\n",
       "   'cross_references': {}}},\n",
       " 'PF02225': {'metadata': {'accession': 'PF02225',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR003137',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'PA domain', 'short': 'PA'},\n",
       "   'description': ['<p>The PA (Protease associated) domain is found as an insert domain in diverse proteases. The PA domain is also found in a plant vacuolar sorting receptor Swiss:O22925 and members of the RZF family Swiss:O43567. It has been suggested that this domain forms a lid-like structure that covers the active site in active proteases, and is involved in protein recognition in vacuolar sorting receptors [PMID:11246007].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019976': {'PMID': 11246007,\n",
       "     'ISBN': None,\n",
       "     'volume': '26',\n",
       "     'issue': '3',\n",
       "     'year': 2001,\n",
       "     'title': 'The protease-associated domain: a homology domain associated with multiple classes of proteases.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '147-8',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Luo X', 'Hofmann K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(00)01768-0'}},\n",
       "   'set_info': {'accession': 'CL0364', 'name': 'Leu-IlvD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 1416,\n",
       "    'interactions': 0,\n",
       "    'matches': 70139,\n",
       "    'pathways': 0,\n",
       "    'proteins': 69845,\n",
       "    'proteomes': 5476,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 53673, 'rosettafold': 0},\n",
       "    'structures': 130,\n",
       "    'taxa': 21659},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 59359,\n",
       "    'alignment:full': 25301,\n",
       "    'alignment:seed': 339},\n",
       "   'cross_references': {}}},\n",
       " 'PF17768': {'metadata': {'accession': 'PF17768',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR041122',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'RecJ OB domain', 'short': 'RecJ_OB'},\n",
       "   'description': ['<p>This OB-fold is found in RecJ proteins where is binds to ssDNA [PMID:27058167].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0021', 'name': 'OB'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 39582,\n",
       "    'pathways': 0,\n",
       "    'proteins': 39581,\n",
       "    'proteomes': 6602,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 33569, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 30088},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36238,\n",
       "    'alignment:full': 6395,\n",
       "    'alignment:seed': 523},\n",
       "   'cross_references': {}}},\n",
       " 'PF02781': {'metadata': {'accession': 'PF02781',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR022675',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Glucose-6-phosphate dehydrogenase, C-terminal domain',\n",
       "    'short': 'G6PD_C'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019591': {'PMID': 9485426,\n",
       "     'ISBN': None,\n",
       "     'volume': '37',\n",
       "     'issue': '9',\n",
       "     'year': 1998,\n",
       "     'title': 'On the mechanism of the reaction catalyzed by glucose 6-phosphate dehydrogenase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2759-67',\n",
       "     'medline_journal': 'Biochemistry',\n",
       "     'ISO_journal': 'Biochemistry',\n",
       "     'authors': ['Cosgrove MS',\n",
       "      'Naylor C',\n",
       "      'Paludan S',\n",
       "      'Adams MJ',\n",
       "      'Levy HR.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1021/bi972069y'}},\n",
       "   'set_info': {'accession': 'CL0139', 'name': 'GADPH_aa-bio_dh'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 160,\n",
       "    'interactions': 0,\n",
       "    'matches': 48742,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48353,\n",
       "    'proteomes': 8056,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 41781, 'rosettafold': 0},\n",
       "    'structures': 44,\n",
       "    'taxa': 35228},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45670,\n",
       "    'alignment:full': 11331,\n",
       "    'alignment:seed': 398},\n",
       "   'cross_references': {}}},\n",
       " 'PF12631': {'metadata': {'accession': 'PF12631',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR025867',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MnmE helical domain', 'short': 'MnmE_helical'},\n",
       "   'description': ['<p>The tRNA modification GTPase MnmE consists of three domains. An N-terminal domain, a helical domain and a GTPase domain which is nested within the helical domain. This family represents the helical domain [1-2].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 126,\n",
       "    'interactions': 0,\n",
       "    'matches': 40316,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40293,\n",
       "    'proteomes': 8445,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34296, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 35186},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 36732,\n",
       "    'alignment:full': 7910,\n",
       "    'alignment:seed': 391},\n",
       "   'cross_references': {}}},\n",
       " 'PF01706': {'metadata': {'accession': 'PF01706',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR023087',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'FliG C-terminal domain', 'short': 'FliG_C'},\n",
       "   'description': ['<p>FliG is a component of the flageller rotor, present in about 25 copies per flagellum.  This domain functions specifically in motor rotation.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00004294': {'PMID': 10440379,\n",
       "     'ISBN': None,\n",
       "     'volume': '400',\n",
       "     'issue': '6743',\n",
       "     'year': 1999,\n",
       "     'title': 'Structure of the C-terminal domain of FliG, a component of the rotor in the bacterial flagellar motor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '472-5',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Lloyd SA', 'Whitby FG', 'Blair DF', 'Hill CP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/22794'}},\n",
       "   'set_info': {'accession': 'CL0436', 'name': 'FliG'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 88,\n",
       "    'interactions': 0,\n",
       "    'matches': 22336,\n",
       "    'pathways': 0,\n",
       "    'proteins': 22326,\n",
       "    'proteomes': 4197,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 19245, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 17984},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20751,\n",
       "    'alignment:full': 4441,\n",
       "    'alignment:seed': 295},\n",
       "   'cross_references': {}}},\n",
       " 'PF02104': {'metadata': {'accession': 'PF02104',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002994',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SURF1 family', 'short': 'SURF1'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 66,\n",
       "    'interactions': 0,\n",
       "    'matches': 26556,\n",
       "    'pathways': 0,\n",
       "    'proteins': 26488,\n",
       "    'proteomes': 5942,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 22839, 'rosettafold': 1},\n",
       "    'structures': 1,\n",
       "    'taxa': 21751},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24475,\n",
       "    'alignment:full': 6715,\n",
       "    'alignment:seed': 457},\n",
       "   'cross_references': {}}},\n",
       " 'PF14450': {'metadata': {'accession': 'PF14450',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': None,\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cell division protein FtsA', 'short': 'FtsA'},\n",
       "   'description': ['<p>FtsA is essential for bacterial cell division, and co-localises to  the septal ring with FtsZ. It has been suggested that the interaction of FtsA-FtsZ has arisen through coevolution in different bacterial strains [PMID:9352931].  The FtsA protein contains two structurally related  actin-like ATPase domains which are also structurally related to the ATPase domains of HSP70 (see PF00012). FtsA has a SHS2 domain  PF02491 inserted in to the RnaseH fold PF02491 [PMID:15281131].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0108', 'name': 'Actin_ATPase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 50,\n",
       "    'interactions': 0,\n",
       "    'matches': 55572,\n",
       "    'pathways': 0,\n",
       "    'proteins': 36537,\n",
       "    'proteomes': 5913,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 31253, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 27548},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 50955,\n",
       "    'alignment:full': 9397,\n",
       "    'alignment:seed': 413},\n",
       "   'cross_references': {}}},\n",
       " 'PF05494': {'metadata': {'accession': 'PF05494',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR008869',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'MlaC protein', 'short': 'MlaC'},\n",
       "   'description': ['<p>MlaC is a component of the Mla pathway, an ABC transport system that functions to maintain the asymmetry of the outer membrane [PMID:19383799].  This family of proteins is involved in toluene tolerance, which is mediated by increased cell membrane rigidity resulting from changes in fatty acid and phospholipid compositions, exclusion of toluene from the cell membrane, and removal of intracellular toluene by degradation [2-3]. Many proteins are involved in these processes.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00011501': {'PMID': 9020089,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '7',\n",
       "     'year': 1997,\n",
       "     'title': 'Mechanisms for solvent tolerance in bacteria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3887-90',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Ramos JL',\n",
       "      'Duque E',\n",
       "      'Rodriguez-Herva JJ',\n",
       "      'Godoy P',\n",
       "      'Haidour A',\n",
       "      'Reyes F',\n",
       "      'Fernandez-Barrero A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.7.3887'},\n",
       "    'PUB00011502': {'PMID': 9658016,\n",
       "     'ISBN': None,\n",
       "     'volume': '180',\n",
       "     'issue': '14',\n",
       "     'year': 1998,\n",
       "     'title': 'Isolation and characterization of toluene-sensitive mutants from the toluene-resistant bacterium Pseudomonas putida GM73.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3692-6',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Kim K', 'Lee S', 'Lee K', 'Lim D.'],\n",
       "     'DOI_URL': 'http://ukpmc.ac.uk/picrender.cgi?tool=EBI&pubmedid=9658016&action=stream&blobtype=pdf'}},\n",
       "   'set_info': {'accession': 'CL0051', 'name': 'NTF2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 35,\n",
       "    'interactions': 0,\n",
       "    'matches': 18017,\n",
       "    'pathways': 0,\n",
       "    'proteins': 18009,\n",
       "    'proteomes': 2503,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 15356, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 12278},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 16573,\n",
       "    'alignment:full': 2850,\n",
       "    'alignment:seed': 256},\n",
       "   'cross_references': {}}},\n",
       " 'PF07884': {'metadata': {'accession': 'PF07884',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR012932',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Vitamin K epoxide reductase family', 'short': 'VKOR'},\n",
       "   'description': ['<p>Vitamin K epoxide reductase (VKOR) recycles reduced vitamin K, which is used subsequently as a co-factor in the gamma-carboxylation of glutamic acid residues in blood coagulation enzymes. VKORC1 is a member of a large family of predicted enzymes that are present in vertebrates, Drosophila, plants, bacteria and archaea [PMID:15276181]. Four cysteine residues and one residue, which is either serine or threonine, are identified as likely active-site residues [PMID:15276181]. In some plant and bacterial homologues the VKORC1 homologous domain is fused with domains of the thioredoxin family of oxidoreductases [PMID:15276181].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00016407': {'PMID': 15276181,\n",
       "     'ISBN': None,\n",
       "     'volume': '29',\n",
       "     'issue': '6',\n",
       "     'year': 2004,\n",
       "     'title': 'Vitamin K epoxide reductase: homology, active site and catalytic mechanism.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '289-92',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Goodstadt L', 'Ponting CP.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.tibs.2004.04.004'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 91,\n",
       "    'interactions': 0,\n",
       "    'matches': 16563,\n",
       "    'pathways': 0,\n",
       "    'proteins': 16553,\n",
       "    'proteomes': 3362,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13839, 'rosettafold': 0},\n",
       "    'structures': 15,\n",
       "    'taxa': 15243},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 14239,\n",
       "    'alignment:full': 3693,\n",
       "    'alignment:seed': 233},\n",
       "   'cross_references': {}}},\n",
       " 'PF10150': {'metadata': {'accession': 'PF10150',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019307',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribonuclease E/G family', 'short': 'RNase_E_G'},\n",
       "   'description': ['<p>Ribonuclease E and Ribonuclease G are related enzymes that cleave a wide variety of RNAs [PMID:16237448].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00033641': {'PMID': 16237448,\n",
       "     'ISBN': None,\n",
       "     'volume': '437',\n",
       "     'issue': '7062',\n",
       "     'year': 2005,\n",
       "     'title': 'Structure of Escherichia coli RNase E catalytic domain and implications for RNA turnover.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1187-91',\n",
       "     'medline_journal': 'Nature',\n",
       "     'ISO_journal': 'Nature',\n",
       "     'authors': ['Callaghan AJ',\n",
       "      'Marcaida MJ',\n",
       "      'Stead JA',\n",
       "      'McDowall KJ',\n",
       "      'Scott WG',\n",
       "      'Luisi BF.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1038/nature04084'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 80,\n",
       "    'interactions': 0,\n",
       "    'matches': 51046,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50113,\n",
       "    'proteomes': 7532,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41917, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 31225},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 47354,\n",
       "    'alignment:full': 9546,\n",
       "    'alignment:seed': 501},\n",
       "   'cross_references': {}}},\n",
       " 'PF01197': {'metadata': {'accession': 'PF01197',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR002150',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Ribosomal protein L31', 'short': 'Ribosomal_L31'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 40,\n",
       "    'interactions': 0,\n",
       "    'matches': 48052,\n",
       "    'pathways': 0,\n",
       "    'proteins': 48052,\n",
       "    'proteomes': 8933,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 41675, 'rosettafold': 0},\n",
       "    'structures': 503,\n",
       "    'taxa': 38950},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 44056,\n",
       "    'alignment:full': 9454,\n",
       "    'alignment:seed': 482},\n",
       "   'cross_references': {}}},\n",
       " 'PF16016': {'metadata': {'accession': 'PF16016',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR031968',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'VAD1 Analog of StAR-related lipid transfer domain',\n",
       "    'short': 'VASt'},\n",
       "   'description': [\"<p>The VASt (VAD1 Analog of StAR-related lipid transfer) domain is conserved across eukaryotes and is structurally related to Bet v1-like domains, including START lipid-binding domains. The 190-amino acid VASt domain is predominantly associated with lipid binding domains such as GRAM Pfam:PF02893, C2 and PH domains. The VASt domain is likely to have a function in binding large hydrophobic ligands and may be specific for sterol [PMID:24965341, PMID:26001273]. The predicted structure of the VASt domain is a two-layer sandwich alpha beta fold, also called 'helix grip fold', containing three alpha helices (alpha1 to 3), six beta-sheets (beta1 to 6) and two loops (omega1 and 2) numbered from N to C terminus [PMID:24965341]. Some proteins known to contain a VASt domain are : Plant vascular associated death1 (VAD1), a regulator of programmed cell death (PCD) harboring a GRAM putative lipid-binding domain. Yeast SNF1 Interacting Protein 3 (SIP3), may be involved in sterol transfer between intracellular membranes. Yeast Suicide Protein 1 (YSP1), a mitochondrial protein specifically required for the mitochondrial thread-grain transition, de-energization, and the cell death. May be involved in sterol transfer between intracellular membranes. Yeast Suicide Protein 2 (YSP2), a mitochondrial membrane protein involved in mitochondrial fragmentation and may be involved in sterol transfer between intracellular membranes.It is also found in human GramD1a-c.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0209', 'name': 'Bet_v_1_like'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 175,\n",
       "    'interactions': 0,\n",
       "    'matches': 12751,\n",
       "    'pathways': 0,\n",
       "    'proteins': 11688,\n",
       "    'proteomes': 2051,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 8229, 'rosettafold': 0},\n",
       "    'structures': 11,\n",
       "    'taxa': 7359},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 10199,\n",
       "    'alignment:full': 6693,\n",
       "    'alignment:seed': 315},\n",
       "   'cross_references': {}}},\n",
       " 'PF03737': {'metadata': {'accession': 'PF03737',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR005493',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Aldolase/RraA', 'short': 'RraA-like'},\n",
       "   'description': ['<p>Members of this family include regulator of ribonuclease E activity A (RraA) and 4-hydroxy-4-methyl-2-oxoglutarate (HMG)/4-carboxy- 4-hydroxy-2-oxoadipate (CHA) aldolase, also known as RraA-like protein [PMID:24359411]. RraA acts as a trans-acting modulator of RNA turnover, binding essential endonuclease RNase E and inhibiting RNA processing [PMID:14499605]. RraA-like proteins seem to contain aldolase and/or decarboxylase activity either in place of or in addition to the RNase E inhibitor functions [PMID:24359411].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019851': {'PMID': 9139683,\n",
       "     'ISBN': None,\n",
       "     'volume': '272',\n",
       "     'issue': '19',\n",
       "     'year': 1997,\n",
       "     'title': 'Identification of a novel gene cluster participating in menaquinone (vitamin K2) biosynthesis. Cloning and sequence determination of the 2-heptaprenyl-1,4-naphthoquinone methyltransferase gene of Bacillus stearothermophilus.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '12380-3',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Koike-Takeshita A', 'Koyama T', 'Ogura K.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.272.19.12380'}},\n",
       "   'set_info': {'accession': 'CL0364', 'name': 'Leu-IlvD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 145,\n",
       "    'interactions': 0,\n",
       "    'matches': 45635,\n",
       "    'pathways': 0,\n",
       "    'proteins': 45413,\n",
       "    'proteomes': 5403,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 39468, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 22690},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42783,\n",
       "    'alignment:full': 9950,\n",
       "    'alignment:seed': 353},\n",
       "   'cross_references': {}}},\n",
       " 'PF06172': {'metadata': {'accession': 'PF06172',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009327',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Cupin superfamily (DUF985)', 'short': 'Cupin_5'},\n",
       "   'description': ['<p>Family of uncharacterised proteins found in bacteria and eukaryotes that belongs to the Cupin superfamily.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00019370': {'PMID': 15608122,\n",
       "     'ISBN': None,\n",
       "     'volume': '14',\n",
       "     'issue': '1',\n",
       "     'year': 2005,\n",
       "     'title': 'Crystal structure of the YML079w protein from Saccharomyces cerevisiae reveals a new sequence family of the jelly-roll fold.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '209-15',\n",
       "     'medline_journal': 'Protein Sci',\n",
       "     'ISO_journal': 'Protein Sci.',\n",
       "     'authors': ['Zhou CZ',\n",
       "      'Meyer P',\n",
       "      'Quevillon-Cheruel S',\n",
       "      'De La Sierra-Gallay IL',\n",
       "      'Collinet B',\n",
       "      'Graille M',\n",
       "      'Blondeau K',\n",
       "      'Francois JM',\n",
       "      'Leulliot N',\n",
       "      'Sorel I',\n",
       "      'Poupon A',\n",
       "      'Janin J',\n",
       "      'Van Tilbeurgh H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1110/ps.041121305'}},\n",
       "   'set_info': {'accession': 'CL0029', 'name': 'Cupin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 12241,\n",
       "    'pathways': 0,\n",
       "    'proteins': 12191,\n",
       "    'proteomes': 3326,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 10491, 'rosettafold': 0},\n",
       "    'structures': 7,\n",
       "    'taxa': 13374},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 11133,\n",
       "    'alignment:full': 3293,\n",
       "    'alignment:seed': 267},\n",
       "   'cross_references': {}}},\n",
       " 'PF16113': {'metadata': {'accession': 'PF16113',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR045004',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Enoyl-CoA hydratase/isomerase', 'short': 'ECH_2'},\n",
       "   'description': [\"<p>This family contains a diverse set of enzymes including: enoyl-CoA hydratase, napthoate synthase, carnitate racemase, 3-hydroxybutyryl-CoA dehydratase and dodecanoyl-CoA delta-isomerase. This family differs from Pfam:PF00378 in the structure of it's C-terminus.</p>\"],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0127', 'name': 'ClpP_crotonase'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 157,\n",
       "    'interactions': 0,\n",
       "    'matches': 25900,\n",
       "    'pathways': 0,\n",
       "    'proteins': 25261,\n",
       "    'proteomes': 5160,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 21920, 'rosettafold': 0},\n",
       "    'structures': 20,\n",
       "    'taxa': 20517},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 24352,\n",
       "    'alignment:full': 7548,\n",
       "    'alignment:seed': 251},\n",
       "   'cross_references': {}}},\n",
       " 'PF07516': {'metadata': {'accession': 'PF07516',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR011116',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SecA Wing and Scaffold domain', 'short': 'SecA_SW'},\n",
       "   'description': ['<p>SecA protein binds to the plasma membrane where it interacts with proOmpA to support translocation of proOmpA through the membrane.  SecA protein achieves this translocation, in association with SecY protein, in an ATP  dependent manner.  This family is composed of two C-terminal alpha helical subdomains: the wing and scaffold subdomains [PMID:12242434].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00014329': {'PMID': 12242434,\n",
       "     'ISBN': None,\n",
       "     'volume': '297',\n",
       "     'issue': '5589',\n",
       "     'year': 2002,\n",
       "     'title': 'Nucleotide control of interdomain interactions in the conformational reaction cycle of SecA.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2018-26',\n",
       "     'medline_journal': 'Science',\n",
       "     'ISO_journal': 'Science',\n",
       "     'authors': ['Hunt JF',\n",
       "      'Weinkauf S',\n",
       "      'Henry L',\n",
       "      'Fak JJ',\n",
       "      'McNicholas P',\n",
       "      'Oliver DB',\n",
       "      'Deisenhofer J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1126/science.1074424'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 121,\n",
       "    'interactions': 0,\n",
       "    'matches': 53235,\n",
       "    'pathways': 0,\n",
       "    'proteins': 51927,\n",
       "    'proteomes': 8867,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 44180, 'rosettafold': 0},\n",
       "    'structures': 29,\n",
       "    'taxa': 38591},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 48743,\n",
       "    'alignment:full': 9558,\n",
       "    'alignment:seed': 736},\n",
       "   'cross_references': {}}},\n",
       " 'PF03952': {'metadata': {'accession': 'PF03952',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR020811',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Enolase, N-terminal domain', 'short': 'Enolase_N'},\n",
       "   'description': None,\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0227', 'name': 'Enolase_N'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 199,\n",
       "    'interactions': 0,\n",
       "    'matches': 62527,\n",
       "    'pathways': 0,\n",
       "    'proteins': 62379,\n",
       "    'proteomes': 10912,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 52402, 'rosettafold': 0},\n",
       "    'structures': 108,\n",
       "    'taxa': 49789},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 55445,\n",
       "    'alignment:full': 13645,\n",
       "    'alignment:seed': 632},\n",
       "   'cross_references': {}}},\n",
       " 'PF06724': {'metadata': {'accession': 'PF06724',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR009597',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Domain of Unknown Function (DUF1206)',\n",
       "    'short': 'DUF1206'},\n",
       "   'description': ['<p>This region consists of two a pair of transmembrane helices and occurs three times in each of the family member proteins.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 9,\n",
       "    'interactions': 0,\n",
       "    'matches': 19840,\n",
       "    'pathways': 0,\n",
       "    'proteins': 6953,\n",
       "    'proteomes': 1773,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 6208, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 6756},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 19165,\n",
       "    'alignment:full': 5149,\n",
       "    'alignment:seed': 354},\n",
       "   'cross_references': {}}},\n",
       " 'PF01368': {'metadata': {'accession': 'PF01368',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR001667',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'DHH family', 'short': 'DHH'},\n",
       "   'description': ['<p>It is predicted that this family of proteins all perform a phosphoesterase function. It included the single stranded DNA exonuclease RecJ.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00005478': {'PMID': 9478130,\n",
       "     'ISBN': None,\n",
       "     'volume': '23',\n",
       "     'issue': '1',\n",
       "     'year': 1998,\n",
       "     'title': 'A novel family of predicted phosphoesterases includes Drosophila prune protein and bacterial RecJ exonuclease.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '17-9',\n",
       "     'medline_journal': 'Trends Biochem Sci',\n",
       "     'ISO_journal': 'Trends Biochem. Sci.',\n",
       "     'authors': ['Aravind L', 'Koonin EV.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/S0968-0004(97)01162-6'}},\n",
       "   'set_info': {'accession': 'CL0137', 'name': 'HAD'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 237,\n",
       "    'interactions': 0,\n",
       "    'matches': 95610,\n",
       "    'pathways': 0,\n",
       "    'proteins': 95577,\n",
       "    'proteomes': 9260,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 81600, 'rosettafold': 0},\n",
       "    'structures': 58,\n",
       "    'taxa': 40803},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 85845,\n",
       "    'alignment:full': 17271,\n",
       "    'alignment:seed': 1075},\n",
       "   'cross_references': {}}},\n",
       " 'PF06071': {'metadata': {'accession': 'PF06071',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR013029',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF933)',\n",
       "    'short': 'YchF-GTPase_C'},\n",
       "   'description': ['<p>This domain is found at the C terminus of the YchF GTP-binding protein (Swiss:O13998) and is possibly related to the ubiquitin-like and MoaD/ThiS superfamilies. </p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0072', 'name': 'Ubiquitin'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 138,\n",
       "    'interactions': 0,\n",
       "    'matches': 50142,\n",
       "    'pathways': 0,\n",
       "    'proteins': 50125,\n",
       "    'proteomes': 10466,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 42650, 'rosettafold': 0},\n",
       "    'structures': 9,\n",
       "    'taxa': 44713},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 45014,\n",
       "    'alignment:full': 10545,\n",
       "    'alignment:seed': 353},\n",
       "   'cross_references': {}}},\n",
       " 'PF04043': {'metadata': {'accession': 'PF04043',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006501',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Plant invertase/pectin methylesterase inhibitor',\n",
       "    'short': 'PMEI'},\n",
       "   'description': ['<p>This domain inhibits pectin methylesterases (PMEs) and invertases through formation of a non-covalent 1:1 complex [PMID:8521860]. It has been implicated in the regulation of fruit development, carbohydrate metabolism and cell wall extension (see [PMID:10880981]). It may also be involved in inhibiting microbial pathogen PMEs. It has been observed that it is often expressed as a large inactive preprotein [PMID:8521860]. It is also found at the N-termini of PMEs predicted from DNA sequences (personal obs:C Yeats), suggesting that both PMEs and their inhibitor are expressed as a single polyprotein and subsequently processed. It has two disulphide bridges and is mainly alpha-helical [PMID:10880981].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009644': {'PMID': 10880981,\n",
       "     'ISBN': None,\n",
       "     'volume': '267',\n",
       "     'issue': '14',\n",
       "     'year': 2000,\n",
       "     'title': 'Kiwi protein inhibitor of pectin methylesterase amino-acid sequence and structural importance of two disulfide bridges.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '4561-5',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Camardella L',\n",
       "      'Carratore V',\n",
       "      'Ciardiello MA',\n",
       "      'Servillo L',\n",
       "      'Balestrieri C',\n",
       "      'Giovane A.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1046/j.1432-1327.2000.01510.x'},\n",
       "    'PUB00010585': {'PMID': 8521860,\n",
       "     'ISBN': None,\n",
       "     'volume': '233',\n",
       "     'issue': '3',\n",
       "     'year': 1995,\n",
       "     'title': 'A glycoprotein inhibitor of pectin methylesterase in kiwi fruit. Purification by affinity chromatography and evidence of a ripening-related precursor.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '926-9',\n",
       "     'medline_journal': 'Eur J Biochem',\n",
       "     'ISO_journal': 'Eur. J. Biochem.',\n",
       "     'authors': ['Giovane A',\n",
       "      'Balestrieri C',\n",
       "      'Quagliuolo L',\n",
       "      'Castaldo D',\n",
       "      'Servillo L.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1111/j.1432-1033.1995.926_3.x'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 305,\n",
       "    'interactions': 0,\n",
       "    'matches': 24355,\n",
       "    'pathways': 0,\n",
       "    'proteins': 23983,\n",
       "    'proteomes': 205,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 20412, 'rosettafold': 0},\n",
       "    'structures': 12,\n",
       "    'taxa': 918},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 20280,\n",
       "    'alignment:full': 13776,\n",
       "    'alignment:seed': 629},\n",
       "   'cross_references': {}}},\n",
       " 'PF06477': {'metadata': {'accession': 'PF06477',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR010512',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Protein of unknown function (DUF1091)',\n",
       "    'short': 'DUF1091'},\n",
       "   'description': ['<p>This is a family of uncharacterised proteins. Based on its distant similarity to Pfam:PF02221 and conserved pattern of cysteine residues it is possible that these domains are also lipid binding.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0532', 'name': 'LIG'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 37,\n",
       "    'interactions': 0,\n",
       "    'matches': 4004,\n",
       "    'pathways': 0,\n",
       "    'proteins': 3715,\n",
       "    'proteomes': 76,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 3440, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 514},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 4311,\n",
       "    'alignment:full': 2080,\n",
       "    'alignment:seed': 352},\n",
       "   'cross_references': {}}},\n",
       " 'PF10646': {'metadata': {'accession': 'PF10646',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019606',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Sporulation and spore germination', 'short': 'Germane'},\n",
       "   'description': ['<p>The GerMN domain is a region of approximately 100 residues that is found, duplicated, in the Bacillus GerM protein and is implicated in both sporulation and spore germination. The domain is found in a number of different bacterial species both alone and in association with other domains such as Amidase_3 Pfam:PF01520, Gmad1 and Gmad2.  It is predicted to have a novel alpha-beta fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00044986': {'PMID': 7926687,\n",
       "     'ISBN': None,\n",
       "     'volume': '121',\n",
       "     'issue': '3',\n",
       "     'year': 1994,\n",
       "     'title': 'Molecular genetical and phenotypical analysis of the gerM spore germination gene of Bacillus subtilis 168.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '315-20',\n",
       "     'medline_journal': 'FEMS Microbiol Lett',\n",
       "     'ISO_journal': 'FEMS Microbiol. Lett.',\n",
       "     'authors': ['Slynn GM', 'Sammons RL', 'Smith DA', 'Moir A', 'Corfe BM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1097(94)90310-7'},\n",
       "    'PUB00044987': {'PMID': 14662349,\n",
       "     'ISBN': None,\n",
       "     'volume': '6',\n",
       "     'issue': '6',\n",
       "     'year': 2003,\n",
       "     'title': 'Spore germination.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '550-6',\n",
       "     'medline_journal': 'Curr Opin Microbiol',\n",
       "     'ISO_journal': 'Curr. Opin. Microbiol.',\n",
       "     'authors': ['Setlow P.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/j.mib.2003.10.001'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 84,\n",
       "    'interactions': 0,\n",
       "    'matches': 18973,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15816,\n",
       "    'proteomes': 2968,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 13482, 'rosettafold': 0},\n",
       "    'structures': 2,\n",
       "    'taxa': 11817},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 17108,\n",
       "    'alignment:full': 4383,\n",
       "    'alignment:seed': 488},\n",
       "   'cross_references': {}}},\n",
       " 'PF12680': {'metadata': {'accession': 'PF12680',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR037401',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'SnoaL-like domain', 'short': 'SnoaL_2'},\n",
       "   'description': ['<p>This family contains a large number of proteins that share the SnoaL fold.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': None,\n",
       "   'set_info': {'accession': 'CL0051', 'name': 'NTF2'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 432,\n",
       "    'interactions': 0,\n",
       "    'matches': 102950,\n",
       "    'pathways': 0,\n",
       "    'proteins': 101770,\n",
       "    'proteomes': 6207,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 88861, 'rosettafold': 0},\n",
       "    'structures': 123,\n",
       "    'taxa': 26143},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 96773,\n",
       "    'alignment:full': 22205,\n",
       "    'alignment:seed': 272},\n",
       "   'cross_references': {}}},\n",
       " 'PF04551': {'metadata': {'accession': 'PF04551',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR004588',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'GcpE protein', 'short': 'GcpE'},\n",
       "   'description': ['<p>In a variety of organisms, including plants and several eubacteria, isoprenoids are synthesised by the mevalonate-independent 2-C-methyl-D-erythritol 4-phosphate (MEP) pathway. Although different enzymes of this pathway have been described, the terminal biosynthetic steps of the MEP pathway have not been fully elucidated. GcpE gene of Escherichia coli is involved in this pathway [PMID:11274098].</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00008495': {'PMID': 11274098,\n",
       "     'ISBN': None,\n",
       "     'volume': '183',\n",
       "     'issue': '8',\n",
       "     'year': 2001,\n",
       "     'title': 'GcpE is involved in the 2-C-methyl-D-erythritol 4-phosphate pathway of isoprenoid biosynthesis in Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2411-6',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Altincicek B',\n",
       "      'Kollas AK',\n",
       "      'Sanderbrand S',\n",
       "      'Wiesner J',\n",
       "      'Hintz M',\n",
       "      'Beck E',\n",
       "      'Jomaa H.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JB.183.8.2411-2416.2001'},\n",
       "    'PUB00019604': {'PMID': 1521767,\n",
       "     'ISBN': None,\n",
       "     'volume': '73',\n",
       "     'issue': '1-2',\n",
       "     'year': 1992,\n",
       "     'title': 'Sequence and characterization of the gcpE gene of Escherichia coli.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '175-80',\n",
       "     'medline_journal': 'FEMS Microbiol Lett',\n",
       "     'ISO_journal': 'FEMS Microbiol. Lett.',\n",
       "     'authors': ['Baker J', 'Franklin DB', 'Parker J.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1016/0378-1097(92)90604-M'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 59,\n",
       "    'interactions': 0,\n",
       "    'matches': 42840,\n",
       "    'pathways': 0,\n",
       "    'proteins': 40424,\n",
       "    'proteomes': 7793,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 34420, 'rosettafold': 0},\n",
       "    'structures': 13,\n",
       "    'taxa': 32303},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 38861,\n",
       "    'alignment:full': 8102,\n",
       "    'alignment:seed': 367},\n",
       "   'cross_references': {}}},\n",
       " 'PF01769': {'metadata': {'accession': 'PF01769',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR006667',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Divalent cation transporter', 'short': 'MgtE'},\n",
       "   'description': ['<p>This region is the integral membrane part of the eubacterial MgtE family of magnesium transporters. Related regions are found also in archaebacterial and eukaryotic proteins. All the archaebacterial and eukaryotic examples have two copies of the region. This suggests that the eubacterial examples may act as dimers. Members of this family probably transport Mg2+ or other divalent cations into the cell. The alignment contains two highly conserved aspartates that may be involved in cation binding (Bateman A unpubl.)</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00002262': {'PMID': 7868596,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '5',\n",
       "     'year': 1995,\n",
       "     'title': 'Cloning and characterization of MgtE, a putative new class of Mg2+ transporter from Bacillus firmus OF4.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '1233-8',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Smith RL', 'Thompson LJ', 'Maguire ME.'],\n",
       "     'DOI_URL': 'http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=EBI&pubmedid=7868596'},\n",
       "    'PUB00002279': {'PMID': 7665526,\n",
       "     'ISBN': None,\n",
       "     'volume': '177',\n",
       "     'issue': '18',\n",
       "     'year': 1995,\n",
       "     'title': 'Cloning of the mgtE Mg2+ transporter from Providencia stuartii and the distribution of mgtE in gram-negative and gram-positive bacteria.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '5350-4',\n",
       "     'medline_journal': 'J Bacteriol',\n",
       "     'ISO_journal': 'J. Bacteriol.',\n",
       "     'authors': ['Townsend DE',\n",
       "      'Esenwine AJ',\n",
       "      'George J 3rd',\n",
       "      'Bross D',\n",
       "      'Maguire ME',\n",
       "      'Smith RL.'],\n",
       "     'DOI_URL': 'http://jb.asm.org/cgi/content/abstract/177/18/5350'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 116,\n",
       "    'interactions': 0,\n",
       "    'matches': 49711,\n",
       "    'pathways': 0,\n",
       "    'proteins': 43570,\n",
       "    'proteomes': 6888,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 36444, 'rosettafold': 0},\n",
       "    'structures': 8,\n",
       "    'taxa': 29628},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 42595,\n",
       "    'alignment:full': 11900,\n",
       "    'alignment:seed': 882},\n",
       "   'cross_references': {}}},\n",
       " 'PF10509': {'metadata': {'accession': 'PF10509',\n",
       "   'entry_id': None,\n",
       "   'type': 'domain',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR019539',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'Galactokinase galactose-binding signature',\n",
       "    'short': 'GalKase_gal_bdg'},\n",
       "   'description': ['<p>This is the highly conserved galactokinase signature sequence which appears to be present in all galactokinases irrespective of how many other ATP binding sites, etc that they carry [PMID:10359639]. The function of this domain appears to be to bind galactose [PMID:12796487], and the domain is normally at the N-terminus of the enzymes, EC:2.7.1.6 [PMID:15526155]. This domain is associated with the families GHMP_kinases_C, Pfam:PF08544 and GHMP_kinases_N, Pfam:PF00288.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00015835': {'PMID': 12796487,\n",
       "     'ISBN': None,\n",
       "     'volume': '278',\n",
       "     'issue': '35',\n",
       "     'year': 2003,\n",
       "     'title': 'Molecular structure of galactokinase.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '33305-11',\n",
       "     'medline_journal': 'J Biol Chem',\n",
       "     'ISO_journal': 'J. Biol. Chem.',\n",
       "     'authors': ['Thoden JB', 'Holden HM.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1074/jbc.M304789200'},\n",
       "    'PUB00044187': {'PMID': 10359639,\n",
       "     'ISBN': None,\n",
       "     'volume': '340 ( Pt 3)',\n",
       "     'issue': None,\n",
       "     'year': 1999,\n",
       "     'title': 'The trappin gene family: proteins defined by an N-terminal transglutaminase substrate domain and a C-terminal four-disulphide core.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '569-77',\n",
       "     'medline_journal': 'Biochem J',\n",
       "     'ISO_journal': 'Biochem. J.',\n",
       "     'authors': ['Schalkwijk J', 'Wiedow O', 'Hirose S.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1042/0264-6021:3400569'},\n",
       "    'PUB00044277': {'PMID': 15526155,\n",
       "     'ISBN': None,\n",
       "     'volume': '61',\n",
       "     'issue': '19-20',\n",
       "     'year': 2004,\n",
       "     'title': 'Galactokinase: structure, function and role in type II galactosemia.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2471-84',\n",
       "     'medline_journal': 'Cell Mol Life Sci',\n",
       "     'ISO_journal': 'Cell. Mol. Life Sci.',\n",
       "     'authors': ['Holden HM', 'Thoden JB', 'Timson DJ', 'Reece RJ.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1007/s00018-004-4160-6'}},\n",
       "   'set_info': {'accession': 'CL0329', 'name': 'S5'},\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 139,\n",
       "    'interactions': 0,\n",
       "    'matches': 29283,\n",
       "    'pathways': 0,\n",
       "    'proteins': 29268,\n",
       "    'proteomes': 6237,\n",
       "    'sets': 1,\n",
       "    'structural_models': {'alphafold': 24871, 'rosettafold': 0},\n",
       "    'structures': 33,\n",
       "    'taxa': 24806},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 26207,\n",
       "    'alignment:full': 7191,\n",
       "    'alignment:seed': 463},\n",
       "   'cross_references': {}}},\n",
       " 'PF04294': {'metadata': {'accession': 'PF04294',\n",
       "   'entry_id': None,\n",
       "   'type': 'family',\n",
       "   'go_terms': None,\n",
       "   'source_database': 'pfam',\n",
       "   'member_databases': None,\n",
       "   'integrated': 'IPR007391',\n",
       "   'hierarchy': None,\n",
       "   'name': {'name': 'VanW like protein', 'short': 'VanW'},\n",
       "   'description': ['<p>Family members include vancomycin resistance protein W (VanW). Genes encoding members of this family have been found in vancomycin  resistance gene clusters vanB [PMID:11376048] and vanG [PMID:11036060].  The function of VanW is  unknown.</p>'],\n",
       "   'wikipedia': None,\n",
       "   'literature': {'PUB00009941': {'PMID': 11376048,\n",
       "     'ISBN': None,\n",
       "     'volume': '39',\n",
       "     'issue': '6',\n",
       "     'year': 2001,\n",
       "     'title': 'High prevalence of VanB2 vancomycin-resistant Enterococcus faecium in Taiwan.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '2140-5',\n",
       "     'medline_journal': 'J Clin Microbiol',\n",
       "     'ISO_journal': 'J. Clin. Microbiol.',\n",
       "     'authors': ['Lu JJ', 'Perng CL', 'Ho MF', 'Chiueh TS', 'Lee WH.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/JCM.39.6.2140-2145.2001'},\n",
       "    'PUB00009942': {'PMID': 11036060,\n",
       "     'ISBN': None,\n",
       "     'volume': '44',\n",
       "     'issue': '11',\n",
       "     'year': 2000,\n",
       "     'title': 'Genetic characterization of vanG, a novel vancomycin resistance locus of Enterococcus faecalis.',\n",
       "     'URL': None,\n",
       "     'raw_pages': '3224-8',\n",
       "     'medline_journal': 'Antimicrob Agents Chemother',\n",
       "     'ISO_journal': 'Antimicrob. Agents Chemother.',\n",
       "     'authors': ['McKessar SJ',\n",
       "      'Berry AM',\n",
       "      'Bell JM',\n",
       "      'Turnidge JD',\n",
       "      'Paton JC.'],\n",
       "     'DOI_URL': 'http://dx.doi.org/10.1128/AAC.44.11.3224-3228.2000'}},\n",
       "   'set_info': None,\n",
       "   'overlaps_with': None,\n",
       "   'counters': {'subfamilies': 0,\n",
       "    'domain_architectures': 62,\n",
       "    'interactions': 0,\n",
       "    'matches': 15032,\n",
       "    'pathways': 0,\n",
       "    'proteins': 15019,\n",
       "    'proteomes': 2263,\n",
       "    'sets': 0,\n",
       "    'structural_models': {'alphafold': 12700, 'rosettafold': 1},\n",
       "    'structures': 0,\n",
       "    'taxa': 9548},\n",
       "   'entry_annotations': {'hmm': 0,\n",
       "    'logo': 0,\n",
       "    'alignment:uniprot': 13531,\n",
       "    'alignment:full': 3162,\n",
       "    'alignment:seed': 397},\n",
       "   'cross_references': {}}}}"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "base",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.13"
  },
  "orig_nbformat": 4
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
